{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "provenance": [
     {
      "last_modified": "2019-02-26T21:10:34.129Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-02-26T21:23:37.218Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-02-26T21:29:42.927Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-02-26T21:29:49.331Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-11T13:02:36.154Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-11T13:58:18.648Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-11T14:13:25.127Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-11T14:28:11.358Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-11T14:55:50.293Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-11T15:21:59.351Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-11T15:50:15.879Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-11T16:55:37.863Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-11T18:10:00.910Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-12T17:57:50.106Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-12T18:01:00.923Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-12T19:29:14.676Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-12T20:33:52.907Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-12T20:38:04.391Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-12T20:49:16.980Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-12T21:02:53.226Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-12T21:07:48.517Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-12T21:41:35.957Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-12T22:18:50.792Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T09:45:38.340Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T09:47:22.661Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T09:48:57.366Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T09:49:05.414Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T09:56:41.225Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T10:03:19.244Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T10:06:32.401Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T10:07:26.390Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T10:45:50.106Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T10:50:54.338Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T12:17:40.880Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T12:26:19.966Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T13:09:14.146Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T13:27:43.417Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T16:06:20.219Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T16:43:54.326Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T17:56:24.512Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T19:43:53.038Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T20:44:19.133Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T22:04:09.636Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-13T22:50:18.325Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-14T11:10:48.986Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-14T12:56:15.174Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-14T13:04:00.532Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-14T13:16:54.161Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     },
     {
      "last_modified": "2019-03-14T15:38:06.791Z",
      "source": "## One search can sum up to 140 search results and up to 1200 relationships\n## If duplicates are removed the number of relationships can be docked down to around 500"
     }
    ]
   },
   "source": [
    "## One search can sum up to 140 search results and up to 1200 relationships\n",
    "## If duplicates are removed the number of relationships can be docked down to around 500"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecutionTime": {
     "end_time": "2019-03-14T16:08:05.490714Z",
     "start_time": "2019-03-14T16:08:03.676795Z"
    },
    "provenance": [
     {
      "end_time": "Unknown",
      "execution_time": "Unknown",
      "outputs": [],
      "source": "",
      "start_time": "Unknown"
     },
     {
      "end_time": "2019-02-26T15:08:49.442Z",
      "execution_time": "3.67s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-02-26T15:08:45.771Z"
     },
     {
      "end_time": "2019-02-26T15:09:40.435Z",
      "execution_time": "3.71s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-02-26T15:09:36.729Z"
     },
     {
      "end_time": "2019-02-26T15:11:56.291Z",
      "execution_time": "2.86s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-02-26T15:11:53.431Z"
     },
     {
      "end_time": "2019-02-26T16:54:38.452Z",
      "execution_time": "3.18s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-02-26T16:54:35.272Z"
     },
     {
      "end_time": "2019-02-26T17:00:20.226Z",
      "execution_time": "25.7s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-02-26T16:59:54.524Z"
     },
     {
      "end_time": "2019-03-11T13:01:52.229Z",
      "execution_time": "103ms",
      "outputs": [
       {
        "ename": "IndentationError",
        "evalue": "expected an indented block (Search.py, line 87)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-2-4f7b86e9866f>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from Search import Search\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m87\u001b[0m\n\u001b[0;31m    TYPE = prop[1]\u001b[0m\n\u001b[0m       ^\u001b[0m\n\u001b[0;31mIndentationError\u001b[0m\u001b[0;31m:\u001b[0m expected an indented block\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:01:52.126Z"
     },
     {
      "end_time": "2019-03-11T13:02:45.110Z",
      "execution_time": "21ms",
      "outputs": [
       {
        "ename": "IndentationError",
        "evalue": "expected an indented block (Search.py, line 87)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-4f7b86e9866f>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from Search import Search\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m87\u001b[0m\n\u001b[0;31m    TYPE = prop[1]\u001b[0m\n\u001b[0m       ^\u001b[0m\n\u001b[0;31mIndentationError\u001b[0m\u001b[0;31m:\u001b[0m expected an indented block\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:02:45.089Z"
     },
     {
      "end_time": "2019-03-11T13:03:11.196Z",
      "execution_time": "4.89s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:03:06.308Z"
     },
     {
      "end_time": "2019-03-11T13:04:42.023Z",
      "execution_time": "3.24s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:04:38.786Z"
     },
     {
      "end_time": "2019-03-11T13:26:48.764Z",
      "execution_time": "8.44s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:26:40.326Z"
     },
     {
      "end_time": "2019-03-11T13:37:10.590Z",
      "execution_time": "6.83s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:37:03.760Z"
     },
     {
      "end_time": "2019-03-11T13:42:24.239Z",
      "execution_time": "4.04s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:42:20.201Z"
     },
     {
      "end_time": "2019-03-11T13:45:39.034Z",
      "execution_time": "8.78s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:45:30.254Z"
     },
     {
      "end_time": "2019-03-11T13:47:22.645Z",
      "execution_time": "2.96s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:47:19.690Z"
     },
     {
      "end_time": "2019-03-11T13:50:25.715Z",
      "execution_time": "3.40s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:50:22.319Z"
     },
     {
      "end_time": "2019-03-11T13:58:01.842Z",
      "execution_time": "3.73s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:57:58.117Z"
     },
     {
      "end_time": "2019-03-11T13:58:34.907Z",
      "execution_time": "3.24s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T13:58:31.669Z"
     },
     {
      "end_time": "2019-03-11T14:02:56.594Z",
      "execution_time": "2.92s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:02:53.677Z"
     },
     {
      "end_time": "2019-03-11T14:03:38.455Z",
      "execution_time": "5.49s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:03:32.970Z"
     },
     {
      "end_time": "2019-03-11T14:05:25.198Z",
      "execution_time": "3.49s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:05:21.710Z"
     },
     {
      "end_time": "2019-03-11T14:06:37.196Z",
      "execution_time": "3.01s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:06:34.182Z"
     },
     {
      "end_time": "2019-03-11T14:08:35.536Z",
      "execution_time": "2.21s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:08:33.326Z"
     },
     {
      "end_time": "2019-03-11T14:09:04.862Z",
      "execution_time": "2.25s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:09:02.611Z"
     },
     {
      "end_time": "2019-03-11T14:10:55.136Z",
      "execution_time": "3.00s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:10:52.136Z"
     },
     {
      "end_time": "2019-03-11T14:13:26.431Z",
      "execution_time": "201ms",
      "outputs": [
       {
        "ename": "NameError",
        "evalue": "name 'logging' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-4f7b86e9866f>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfrom\u001b[0m \u001b[0mSearch\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\u001b[0m in \u001b[0;36m__init__\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     23\u001b[0m         \"all\"]\n\u001b[1;32m     24\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 25\u001b[0;31m         logging.basicConfig(level=logging.DEBUG,\n\u001b[0m\u001b[1;32m     26\u001b[0m                             format='(%(threadName)-10s) %(message)s',)\n\u001b[1;32m     27\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mNameError\u001b[0m: name 'logging' is not defined"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:13:26.230Z"
     },
     {
      "end_time": "2019-03-11T14:13:54.331Z",
      "execution_time": "3.20s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:13:51.134Z"
     },
     {
      "end_time": "2019-03-11T14:16:12.177Z",
      "execution_time": "3.19s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:16:08.988Z"
     },
     {
      "end_time": "2019-03-11T14:20:17.459Z",
      "execution_time": "3.67s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:20:13.793Z"
     },
     {
      "end_time": "2019-03-11T14:23:31.871Z",
      "execution_time": "5.56s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:23:26.311Z"
     },
     {
      "end_time": "2019-03-11T14:24:03.438Z",
      "execution_time": "141ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:24:03.297Z"
     },
     {
      "end_time": "2019-03-11T14:26:32.167Z",
      "execution_time": "166ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:26:32.001Z"
     },
     {
      "end_time": "2019-03-11T14:26:36.039Z",
      "execution_time": "11ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:26:36.028Z"
     },
     {
      "end_time": "2019-03-11T14:28:12.392Z",
      "execution_time": "119ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:28:12.273Z"
     },
     {
      "end_time": "2019-03-11T14:40:06.223Z",
      "execution_time": "112ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:40:06.111Z"
     },
     {
      "end_time": "2019-03-11T14:57:18.889Z",
      "execution_time": "136ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T14:57:18.753Z"
     },
     {
      "end_time": "2019-03-11T15:00:42.344Z",
      "execution_time": "116ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:00:42.228Z"
     },
     {
      "end_time": "2019-03-11T15:02:14.739Z",
      "execution_time": "115ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:02:14.624Z"
     },
     {
      "end_time": "2019-03-11T15:02:48.238Z",
      "execution_time": "116ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:02:48.122Z"
     },
     {
      "end_time": "2019-03-11T15:03:38.956Z",
      "execution_time": "160ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:03:38.796Z"
     },
     {
      "end_time": "2019-03-11T15:04:09.899Z",
      "execution_time": "123ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:04:09.776Z"
     },
     {
      "end_time": "2019-03-11T15:04:48.738Z",
      "execution_time": "134ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:04:48.604Z"
     },
     {
      "end_time": "2019-03-11T15:05:54.518Z",
      "execution_time": "111ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:05:54.407Z"
     },
     {
      "end_time": "2019-03-11T15:12:01.144Z",
      "execution_time": "103ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:12:01.041Z"
     },
     {
      "end_time": "2019-03-11T15:12:42.398Z",
      "execution_time": "147ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:12:42.251Z"
     },
     {
      "end_time": "2019-03-11T15:13:05.527Z",
      "execution_time": "91ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:13:05.436Z"
     },
     {
      "end_time": "2019-03-11T15:13:07.765Z",
      "execution_time": "1.15s",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:13:06.613Z"
     },
     {
      "end_time": "2019-03-11T15:14:03.460Z",
      "execution_time": "116ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:14:03.344Z"
     },
     {
      "end_time": "2019-03-11T15:14:39.336Z",
      "execution_time": "111ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:14:39.225Z"
     },
     {
      "end_time": "2019-03-11T15:17:40.455Z",
      "execution_time": "195ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:17:40.260Z"
     },
     {
      "end_time": "2019-03-11T15:18:25.196Z",
      "execution_time": "125ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:18:25.071Z"
     },
     {
      "end_time": "2019-03-11T15:20:19.455Z",
      "execution_time": "120ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:20:19.335Z"
     },
     {
      "end_time": "2019-03-11T15:20:52.617Z",
      "execution_time": "104ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:20:52.513Z"
     },
     {
      "end_time": "2019-03-11T15:21:35.521Z",
      "execution_time": "152ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:21:35.369Z"
     },
     {
      "end_time": "2019-03-11T15:22:01.144Z",
      "execution_time": "123ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:22:01.021Z"
     },
     {
      "end_time": "2019-03-11T15:23:37.656Z",
      "execution_time": "123ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:23:37.533Z"
     },
     {
      "end_time": "2019-03-11T15:25:51.257Z",
      "execution_time": "114ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:25:51.143Z"
     },
     {
      "end_time": "2019-03-11T15:26:33.597Z",
      "execution_time": "120ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:26:33.477Z"
     },
     {
      "end_time": "2019-03-11T15:27:03.402Z",
      "execution_time": "122ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:27:03.280Z"
     },
     {
      "end_time": "2019-03-11T15:27:56.595Z",
      "execution_time": "121ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:27:56.474Z"
     },
     {
      "end_time": "2019-03-11T15:28:31.370Z",
      "execution_time": "123ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:28:31.247Z"
     },
     {
      "end_time": "2019-03-11T15:28:58.386Z",
      "execution_time": "114ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:28:58.272Z"
     },
     {
      "end_time": "2019-03-11T15:33:38.253Z",
      "execution_time": "103ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:33:38.150Z"
     },
     {
      "end_time": "2019-03-11T15:34:40.824Z",
      "execution_time": "119ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:34:40.705Z"
     },
     {
      "end_time": "2019-03-11T15:40:23.325Z",
      "execution_time": "158ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:40:23.167Z"
     },
     {
      "end_time": "2019-03-11T15:43:26.762Z",
      "execution_time": "130ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:43:26.632Z"
     },
     {
      "end_time": "2019-03-11T15:44:39.029Z",
      "execution_time": "107ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:44:38.922Z"
     },
     {
      "end_time": "2019-03-11T15:46:26.192Z",
      "execution_time": "33ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (ReadObject.py, line 52)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "  File \u001b[1;32m\"<ipython-input-1-4f7b86e9866f>\"\u001b[0m, line \u001b[1;32m1\u001b[0m, in \u001b[1;35m<module>\u001b[0m\n    from Search import Search\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m2\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from ReadObject import ReadObject\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/ReadObject.py\"\u001b[0;36m, line \u001b[0;32m52\u001b[0m\n\u001b[0;31m    print(\"\\n\" + str(val) \"=\" + str(key) + \"\\n\")\u001b[0m\n\u001b[0m                            ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:46:26.159Z"
     },
     {
      "end_time": "2019-03-11T15:46:36.087Z",
      "execution_time": "170ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:46:35.917Z"
     },
     {
      "end_time": "2019-03-11T15:46:54.455Z",
      "execution_time": "101ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:46:54.354Z"
     },
     {
      "end_time": "2019-03-11T15:46:55.552Z",
      "execution_time": "419ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:46:55.133Z"
     },
     {
      "end_time": "2019-03-11T15:47:56.982Z",
      "execution_time": "194ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:47:56.788Z"
     },
     {
      "end_time": "2019-03-11T15:51:52.971Z",
      "execution_time": "207ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:51:52.764Z"
     },
     {
      "end_time": "2019-03-11T15:52:11.193Z",
      "execution_time": "197ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:52:10.996Z"
     },
     {
      "end_time": "2019-03-11T15:52:30.510Z",
      "execution_time": "202ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:52:30.308Z"
     },
     {
      "end_time": "2019-03-11T15:54:31.399Z",
      "execution_time": "195ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:54:31.204Z"
     },
     {
      "end_time": "2019-03-11T15:56:50.210Z",
      "execution_time": "207ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:56:50.003Z"
     },
     {
      "end_time": "2019-03-11T15:58:13.287Z",
      "execution_time": "142ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T15:58:13.145Z"
     },
     {
      "end_time": "2019-03-11T16:02:48.856Z",
      "execution_time": "201ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:02:48.655Z"
     },
     {
      "end_time": "2019-03-11T16:03:46.439Z",
      "execution_time": "109ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:03:46.330Z"
     },
     {
      "end_time": "2019-03-11T16:12:56.306Z",
      "execution_time": "103ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:12:56.203Z"
     },
     {
      "end_time": "2019-03-11T16:13:27.219Z",
      "execution_time": "96ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:13:27.123Z"
     },
     {
      "end_time": "2019-03-11T16:14:30.944Z",
      "execution_time": "123ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:14:30.821Z"
     },
     {
      "end_time": "2019-03-11T16:16:02.327Z",
      "execution_time": "123ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:16:02.204Z"
     },
     {
      "end_time": "2019-03-11T16:16:21.886Z",
      "execution_time": "113ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:16:21.773Z"
     },
     {
      "end_time": "2019-03-11T16:19:17.271Z",
      "execution_time": "12ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:19:17.259Z"
     },
     {
      "end_time": "2019-03-11T16:20:48.702Z",
      "execution_time": "24ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (Search.py, line 86)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-4f7b86e9866f>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from Search import Search\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m86\u001b[0m\n\u001b[0;31m    for item in self.data.:\u001b[0m\n\u001b[0m                          ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:20:48.678Z"
     },
     {
      "end_time": "2019-03-11T16:21:15.627Z",
      "execution_time": "120ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:21:15.507Z"
     },
     {
      "end_time": "2019-03-11T16:22:14.266Z",
      "execution_time": "117ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:22:14.149Z"
     },
     {
      "end_time": "2019-03-11T16:25:40.266Z",
      "execution_time": "114ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:25:40.152Z"
     },
     {
      "end_time": "2019-03-11T16:34:12.938Z",
      "execution_time": "114ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:34:12.824Z"
     },
     {
      "end_time": "2019-03-11T16:38:11.437Z",
      "execution_time": "108ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:38:11.329Z"
     },
     {
      "end_time": "2019-03-11T16:40:05.856Z",
      "execution_time": "111ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:40:05.745Z"
     },
     {
      "end_time": "2019-03-11T16:47:21.382Z",
      "execution_time": "116ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:47:21.266Z"
     },
     {
      "end_time": "2019-03-11T16:48:19.622Z",
      "execution_time": "113ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:48:19.509Z"
     },
     {
      "end_time": "2019-03-11T16:48:52.691Z",
      "execution_time": "111ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:48:52.580Z"
     },
     {
      "end_time": "2019-03-11T16:50:19.225Z",
      "execution_time": "118ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:50:19.107Z"
     },
     {
      "end_time": "2019-03-11T16:52:47.023Z",
      "execution_time": "119ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:52:46.904Z"
     },
     {
      "end_time": "2019-03-11T16:55:36.010Z",
      "execution_time": "115ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:55:35.895Z"
     },
     {
      "end_time": "2019-03-11T16:56:31.719Z",
      "execution_time": "103ms",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()",
      "start_time": "2019-03-11T16:56:31.616Z"
     },
     {
      "end_time": "2019-03-11T17:23:23.057Z",
      "execution_time": "4.86s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-11T17:23:18.193Z"
     },
     {
      "end_time": "2019-03-11T17:26:12.414Z",
      "execution_time": "3.27s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-11T17:26:09.139Z"
     },
     {
      "end_time": "2019-03-11T17:55:39.876Z",
      "execution_time": "8.28s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allstudies\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=studies HTTP/1.1\" 302 249\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=studies HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=studies HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-11T17:55:31.597Z"
     },
     {
      "end_time": "2019-03-11T18:00:53.588Z",
      "execution_time": "6.98s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allinvestigations\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=investigations HTTP/1.1\" 302 256\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=investigations HTTP/1.1\" 302 252\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=investigations HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-11T18:00:46.609Z"
     },
     {
      "end_time": "2019-03-11T18:04:04.952Z",
      "execution_time": "4.58s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allprojects\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=projects HTTP/1.1\" 302 250\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=projects HTTP/1.1\" 302 246\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=projects HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-11T18:04:00.374Z"
     },
     {
      "end_time": "2019-03-11T18:06:18.895Z",
      "execution_time": "4.74s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-11T18:06:14.160Z"
     },
     {
      "end_time": "2019-03-11T18:10:02.417Z",
      "execution_time": "16ms",
      "outputs": [
       {
        "ename": "IndentationError",
        "evalue": "expected an indented block (ReadObject.py, line 66)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "  File \u001b[1;32m\"<ipython-input-1-6a184f23f92a>\"\u001b[0m, line \u001b[1;32m1\u001b[0m, in \u001b[1;35m<module>\u001b[0m\n    from Search import Search\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m2\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from ReadObject import ReadObject\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/ReadObject.py\"\u001b[0;36m, line \u001b[0;32m66\u001b[0m\n\u001b[0;31m    print(\"\\n\" + self.data.attributes.description)\u001b[0m\n\u001b[0m        ^\u001b[0m\n\u001b[0;31mIndentationError\u001b[0m\u001b[0;31m:\u001b[0m expected an indented block\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-11T18:10:02.401Z"
     },
     {
      "end_time": "2019-03-11T18:10:15.405Z",
      "execution_time": "3.91s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-11T18:10:11.499Z"
     },
     {
      "end_time": "2019-03-12T16:57:13.447Z",
      "execution_time": "3.54s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T16:57:09.902Z"
     },
     {
      "end_time": "2019-03-12T16:57:32.958Z",
      "execution_time": "3.57s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T16:57:29.388Z"
     },
     {
      "end_time": "2019-03-12T17:18:45.372Z",
      "execution_time": "4.58s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T17:18:40.789Z"
     },
     {
      "end_time": "2019-03-12T17:53:03.411Z",
      "execution_time": "3.71s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T17:52:59.696Z"
     },
     {
      "end_time": "2019-03-12T19:29:23.063Z",
      "execution_time": "2.91s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T19:29:20.152Z"
     },
     {
      "end_time": "2019-03-12T19:32:50.213Z",
      "execution_time": "2.55s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T19:32:47.667Z"
     },
     {
      "end_time": "2019-03-12T20:14:42.648Z",
      "execution_time": "4.13s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T20:14:38.517Z"
     },
     {
      "end_time": "2019-03-12T20:28:49.605Z",
      "execution_time": "3.61s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T20:28:45.998Z"
     },
     {
      "end_time": "2019-03-12T20:34:01.816Z",
      "execution_time": "3.12s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T20:33:58.696Z"
     },
     {
      "end_time": "2019-03-12T20:38:12.710Z",
      "execution_time": "2.61s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T20:38:10.099Z"
     },
     {
      "end_time": "2019-03-12T20:43:22.478Z",
      "execution_time": "3.20s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T20:43:19.280Z"
     },
     {
      "end_time": "2019-03-12T20:46:02.987Z",
      "execution_time": "2.82s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T20:46:00.168Z"
     },
     {
      "end_time": "2019-03-12T20:49:25.685Z",
      "execution_time": "3.62s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T20:49:22.069Z"
     },
     {
      "end_time": "2019-03-12T20:58:32.269Z",
      "execution_time": "3.17s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T20:58:29.096Z"
     },
     {
      "end_time": "2019-03-12T21:03:04.287Z",
      "execution_time": "3.22s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T21:03:01.071Z"
     },
     {
      "end_time": "2019-03-12T21:08:00.170Z",
      "execution_time": "3.04s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T21:07:57.129Z"
     },
     {
      "end_time": "2019-03-12T21:19:50.473Z",
      "execution_time": "3.14s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T21:19:47.333Z"
     },
     {
      "end_time": "2019-03-12T21:35:56.024Z",
      "execution_time": "3.71s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T21:35:52.315Z"
     },
     {
      "end_time": "2019-03-12T21:41:43.476Z",
      "execution_time": "3.16s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T21:41:40.315Z"
     },
     {
      "end_time": "2019-03-12T21:46:55.390Z",
      "execution_time": "3.05s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T21:46:52.341Z"
     },
     {
      "end_time": "2019-03-12T21:48:04.881Z",
      "execution_time": "3.71s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T21:48:01.167Z"
     },
     {
      "end_time": "2019-03-12T21:50:13.069Z",
      "execution_time": "3.21s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T21:50:09.856Z"
     },
     {
      "end_time": "2019-03-12T22:09:46.987Z",
      "execution_time": "3.49s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T22:09:43.496Z"
     },
     {
      "end_time": "2019-03-12T22:19:03.446Z",
      "execution_time": "5.39s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T22:18:58.054Z"
     },
     {
      "end_time": "2019-03-12T22:20:29.468Z",
      "execution_time": "2.85s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T22:20:26.618Z"
     },
     {
      "end_time": "2019-03-12T22:21:33.956Z",
      "execution_time": "3.20s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T22:21:30.756Z"
     },
     {
      "end_time": "2019-03-12T22:22:13.505Z",
      "execution_time": "3.16s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T22:22:10.344Z"
     },
     {
      "end_time": "2019-03-12T22:32:18.230Z",
      "execution_time": "3.54s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T22:32:14.689Z"
     },
     {
      "end_time": "2019-03-12T22:47:32.885Z",
      "execution_time": "2.79s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T22:47:30.097Z"
     },
     {
      "end_time": "2019-03-12T22:49:25.022Z",
      "execution_time": "2.72s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-12T22:49:22.301Z"
     },
     {
      "end_time": "2019-03-13T09:45:50.355Z",
      "execution_time": "5.71s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T09:45:44.647Z"
     },
     {
      "end_time": "2019-03-13T09:47:31.554Z",
      "execution_time": "2.99s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T09:47:28.566Z"
     },
     {
      "end_time": "2019-03-13T09:49:14.174Z",
      "execution_time": "2.75s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T09:49:11.422Z"
     },
     {
      "end_time": "2019-03-13T09:56:49.353Z",
      "execution_time": "3.72s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T09:56:45.629Z"
     },
     {
      "end_time": "2019-03-13T10:01:44.145Z",
      "execution_time": "9.53s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T10:01:34.611Z"
     },
     {
      "end_time": "2019-03-13T10:06:41.421Z",
      "execution_time": "3.67s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T10:06:37.748Z"
     },
     {
      "end_time": "2019-03-13T10:32:28.497Z",
      "execution_time": "3.76s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T10:32:24.735Z"
     },
     {
      "end_time": "2019-03-13T10:37:15.929Z",
      "execution_time": "6.95s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T10:37:08.984Z"
     },
     {
      "end_time": "2019-03-13T10:46:04.438Z",
      "execution_time": "3.16s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T10:46:01.275Z"
     },
     {
      "end_time": "2019-03-13T10:51:04.302Z",
      "execution_time": "3.31s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T10:51:00.992Z"
     },
     {
      "end_time": "2019-03-13T11:15:02.393Z",
      "execution_time": "11.7s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T11:14:50.732Z"
     },
     {
      "end_time": "2019-03-13T11:17:20.787Z",
      "execution_time": "3.29s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T11:17:17.500Z"
     },
     {
      "end_time": "2019-03-13T12:18:25.695Z",
      "execution_time": "9.12s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \ncell\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=cell&search_type=all HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=cell&search_type=all HTTP/1.1\" 302 240\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=cell&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:18:16.574Z"
     },
     {
      "end_time": "2019-03-13T12:20:48.860Z",
      "execution_time": "4.94s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \ncell\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=cell&search_type=all HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=cell&search_type=all HTTP/1.1\" 302 240\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=cell&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:20:43.923Z"
     },
     {
      "end_time": "2019-03-13T12:26:31.132Z",
      "execution_time": "5.01s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:26:26.117Z"
     },
     {
      "end_time": "2019-03-13T12:28:25.752Z",
      "execution_time": "4.87s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:28:20.880Z"
     },
     {
      "end_time": "2019-03-13T12:31:06.239Z",
      "execution_time": "2.67s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:31:03.567Z"
     },
     {
      "end_time": "2019-03-13T12:34:41.543Z",
      "execution_time": "3.50s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:34:38.048Z"
     },
     {
      "end_time": "2019-03-13T12:37:27.607Z",
      "execution_time": "4.52s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:37:23.084Z"
     },
     {
      "end_time": "2019-03-13T12:39:23.398Z",
      "execution_time": "3.78s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:39:19.621Z"
     },
     {
      "end_time": "2019-03-13T12:44:55.466Z",
      "execution_time": "3.59s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:44:51.872Z"
     },
     {
      "end_time": "2019-03-13T12:47:24.076Z",
      "execution_time": "3.04s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:47:21.032Z"
     },
     {
      "end_time": "2019-03-13T12:49:07.288Z",
      "execution_time": "4.25s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T12:49:03.033Z"
     },
     {
      "end_time": "2019-03-13T13:17:04.440Z",
      "execution_time": "2.53s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T13:17:01.912Z"
     },
     {
      "end_time": "2019-03-13T13:17:16.814Z",
      "execution_time": "4.17s",
      "outputs": [
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(Thread 0  ) https://fairdomhub.org:443 \"GET /assays/751 HTTP/1.1\" 200 None\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T13:17:12.643Z"
     },
     {
      "end_time": "2019-03-13T13:20:10.714Z",
      "execution_time": "10.8s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \ncell\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=cell&search_type=all HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=cell&search_type=all HTTP/1.1\" 302 240\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=cell&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T13:19:59.961Z"
     },
     {
      "end_time": "2019-03-13T13:45:05.251Z",
      "execution_time": "3.81s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T13:45:01.438Z"
     },
     {
      "end_time": "2019-03-13T13:47:12.333Z",
      "execution_time": "2.85s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 248\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=assays HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T13:47:09.485Z"
     },
     {
      "end_time": "2019-03-13T13:48:01.688Z",
      "execution_time": "2.59s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T13:47:59.102Z"
     },
     {
      "end_time": "2019-03-13T14:38:05.904Z",
      "execution_time": "4.54s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T14:38:01.362Z"
     },
     {
      "end_time": "2019-03-13T14:50:48.288Z",
      "execution_time": "3.35s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T14:50:44.937Z"
     },
     {
      "end_time": "2019-03-13T14:53:28.760Z",
      "execution_time": "5.61s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T14:53:23.154Z"
     },
     {
      "end_time": "2019-03-13T15:05:52.770Z",
      "execution_time": "6.02s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T15:05:46.750Z"
     },
     {
      "end_time": "2019-03-13T15:06:07.476Z",
      "execution_time": "5.75s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \ncell\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=cell&search_type=all HTTP/1.1\" 302 244\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=cell&search_type=all HTTP/1.1\" 302 240\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=cell&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T15:06:01.725Z"
     },
     {
      "end_time": "2019-03-13T15:10:53.246Z",
      "execution_time": "3.41s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T15:10:49.835Z"
     },
     {
      "end_time": "2019-03-13T16:06:28.782Z",
      "execution_time": "4.33s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:06:24.453Z"
     },
     {
      "end_time": "2019-03-13T16:13:08.860Z",
      "execution_time": "4.82s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:13:04.037Z"
     },
     {
      "end_time": "2019-03-13T16:24:04.866Z",
      "execution_time": "3.21s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:24:01.655Z"
     },
     {
      "end_time": "2019-03-13T16:27:45.742Z",
      "execution_time": "5.88s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:27:39.864Z"
     },
     {
      "end_time": "2019-03-13T16:30:37.152Z",
      "execution_time": "3.97s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:30:33.182Z"
     },
     {
      "end_time": "2019-03-13T16:32:57.034Z",
      "execution_time": "4.27s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:32:52.766Z"
     },
     {
      "end_time": "2019-03-13T16:35:49.843Z",
      "execution_time": "4.17s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:35:45.676Z"
     },
     {
      "end_time": "2019-03-13T16:38:44.534Z",
      "execution_time": "3.16s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:38:41.376Z"
     },
     {
      "end_time": "2019-03-13T16:41:44.077Z",
      "execution_time": "4.32s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:41:39.754Z"
     },
     {
      "end_time": "2019-03-13T16:43:59.712Z",
      "execution_time": "4.20s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:43:55.511Z"
     },
     {
      "end_time": "2019-03-13T16:54:28.429Z",
      "execution_time": "4.12s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:54:24.305Z"
     },
     {
      "end_time": "2019-03-13T16:58:17.811Z",
      "execution_time": "2.49s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T16:58:15.320Z"
     },
     {
      "end_time": "2019-03-13T17:02:14.324Z",
      "execution_time": "3.84s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:02:10.482Z"
     },
     {
      "end_time": "2019-03-13T17:18:20.934Z",
      "execution_time": "3.95s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:18:16.984Z"
     },
     {
      "end_time": "2019-03-13T17:21:04.699Z",
      "execution_time": "3.13s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:21:01.571Z"
     },
     {
      "end_time": "2019-03-13T17:30:44.673Z",
      "execution_time": "3.66s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:30:41.014Z"
     },
     {
      "end_time": "2019-03-13T17:39:35.346Z",
      "execution_time": "2.92s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:39:32.430Z"
     },
     {
      "end_time": "2019-03-13T17:40:53.988Z",
      "execution_time": "3.19s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:40:50.802Z"
     },
     {
      "end_time": "2019-03-13T17:51:49.180Z",
      "execution_time": "17ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (ReadObject.py, line 12)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "  File \u001b[1;32m\"<ipython-input-1-6a184f23f92a>\"\u001b[0m, line \u001b[1;32m1\u001b[0m, in \u001b[1;35m<module>\u001b[0m\n    from Search import Search\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m2\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from ReadObject import ReadObject\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/ReadObject.py\"\u001b[0;36m, line \u001b[0;32m12\u001b[0m\n\u001b[0;31m    \"Accept-Charset\": \"ISO-8859-1\"}\u001b[0m\n\u001b[0m                    ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:51:49.163Z"
     },
     {
      "end_time": "2019-03-13T17:52:51.180Z",
      "execution_time": "3.31s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:52:47.874Z"
     },
     {
      "end_time": "2019-03-13T17:53:25.175Z",
      "execution_time": "3.21s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:53:21.967Z"
     },
     {
      "end_time": "2019-03-13T17:56:33.599Z",
      "execution_time": "3.26s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:56:30.337Z"
     },
     {
      "end_time": "2019-03-13T17:59:31.160Z",
      "execution_time": "4.95s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 245\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 302 241\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=liver&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T17:59:26.211Z"
     },
     {
      "end_time": "2019-03-13T19:35:31.804Z",
      "execution_time": "9.01s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \ncell\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, alldata_files\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=cell&search_type=data_files HTTP/1.1\" 302 251\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=cell&search_type=data_files HTTP/1.1\" 302 247\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=cell&search_type=data_files HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T19:35:22.792Z"
     },
     {
      "end_time": "2019-03-13T19:41:31.299Z",
      "execution_time": "7.74s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \ndata table with cell counts\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       },
       {
        "name": "stderr",
        "output_type": "stream",
        "text": "(MainThread) Starting new HTTP connection (1): www.fairdomhub.org\n(MainThread) http://www.fairdomhub.org:80 \"GET /search?q=data+table+with+cell+counts&search_type=all HTTP/1.1\" 302 267\n(MainThread) Starting new HTTPS connection (1): www.fairdomhub.org\n(MainThread) https://www.fairdomhub.org:443 \"GET /search?q=data+table+with+cell+counts&search_type=all HTTP/1.1\" 302 263\n(MainThread) Starting new HTTPS connection (1): fairdomhub.org\n(MainThread) https://fairdomhub.org:443 \"GET /search?q=data+table+with+cell+counts&search_type=all HTTP/1.1\" 200 None\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T19:41:23.561Z"
     },
     {
      "end_time": "2019-03-13T19:44:01.202Z",
      "execution_time": "4.84s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \ncell\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T19:43:56.357Z"
     },
     {
      "end_time": "2019-03-13T19:50:13.258Z",
      "execution_time": "3.58s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T19:50:09.673Z"
     },
     {
      "end_time": "2019-03-13T19:51:06.815Z",
      "execution_time": "3.20s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T19:51:03.618Z"
     },
     {
      "end_time": "2019-03-13T19:52:01.400Z",
      "execution_time": "3.15s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n135 results\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T19:51:58.247Z"
     },
     {
      "end_time": "2019-03-13T19:55:26.671Z",
      "execution_time": "4.10s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n"
       },
       {
        "ename": "NameError",
        "evalue": "name 's' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     52\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     53\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mParallelSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 54\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m2\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     56\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mNameError\u001b[0m: name 's' is not defined"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T19:55:22.573Z"
     },
     {
      "end_time": "2019-03-13T19:55:52.527Z",
      "execution_time": "2.99s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n"
       },
       {
        "ename": "NameError",
        "evalue": "name 's' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     52\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     53\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mParallelSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 54\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m2\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     56\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mNameError\u001b[0m: name 's' is not defined"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T19:55:49.534Z"
     },
     {
      "end_time": "2019-03-13T19:56:06.048Z",
      "execution_time": "4.27s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait...........................501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/70\n..........................501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/15\n.........................501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/68\n......................501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/58\n...................................."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T19:56:01.775Z"
     },
     {
      "end_time": "2019-03-13T20:03:53.352Z",
      "execution_time": "1m 18.4s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait....................................501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/15\n................501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/70.\n.................................501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/58\n.............501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/68\n....................................."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T20:02:34.915Z"
     },
     {
      "end_time": "2019-03-13T20:31:14.716Z",
      "execution_time": "3.74s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/15\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.NoneNone\n.\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/70\n.None\n.None\n.None\n.NoneNone\n.\nNone\n..None\n.None\n.None\n.None\n.None\n.None\n.NoneNone\n.\n.None\n.None\n.None\n.None\n.501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/58\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.501 Server Error: Not Implemented for url: https://fairdomhub.org/sample_types/68\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n.None\n."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T20:31:10.975Z"
     },
     {
      "end_time": "2019-03-13T20:33:24.512Z",
      "execution_time": "2.93s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.501\nstring indices must be integers\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n200\n.200\n.501.\nstring indices must be integers\n.200200\n.\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n200\n..200\n.200\n.200\n.200\n.200\n.501\n200string indices must be integers\n.\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.501\nstring indices must be integers\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n.200\n."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T20:33:21.584Z"
     },
     {
      "end_time": "2019-03-13T20:38:05.002Z",
      "execution_time": "4.19s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait................................................................Internal Error(result ommited)\n................Internal Error(result ommited)\n............................Internal Error(result ommited)\n........Internal Error(result ommited).\n..................."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T20:38:00.816Z"
     },
     {
      "end_time": "2019-03-13T20:38:37.694Z",
      "execution_time": "3.13s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait...............................................................\nInternal Error(result ommited).....\nInternal Error(result ommited)................................\nInternal Error(result ommited)....\nInternal Error(result ommited)................................"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T20:38:34.560Z"
     },
     {
      "end_time": "2019-03-13T20:44:28.198Z",
      "execution_time": "3.08s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait."
       },
       {
        "ename": "RuntimeError",
        "evalue": "cannot join current thread",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mRuntimeError\u001b[0m                             Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     52\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     53\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mParallelSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 54\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m2\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     56\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/ParallelSearch.py\u001b[0m in \u001b[0;36mparallelRequest\u001b[0;34m(self, requests, requestPerThread)\u001b[0m\n\u001b[1;32m     46\u001b[0m             \u001b[0mnewThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mthreading\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mThread\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m\"Thread \"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcurrentThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtarget\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmakeRequests\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0margs\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mcurrentThread\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0mrequestPerThread\u001b[0m \u001b[0;34m:\u001b[0m \u001b[0mrightArrayBound\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     47\u001b[0m             \u001b[0mnewThread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstart\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 48\u001b[0;31m             \u001b[0mmainThread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     49\u001b[0m             \u001b[0mnewThread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     50\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/threading.py\u001b[0m in \u001b[0;36mjoin\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m   1051\u001b[0m             \u001b[0;32mraise\u001b[0m \u001b[0mRuntimeError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"cannot join thread before it is started\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1052\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mself\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0mcurrent_thread\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1053\u001b[0;31m             \u001b[0;32mraise\u001b[0m \u001b[0mRuntimeError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"cannot join current thread\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1054\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1055\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mtimeout\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mRuntimeError\u001b[0m: cannot join current thread"
        ]
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": ".."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T20:44:25.122Z"
     },
     {
      "end_time": "2019-03-13T20:45:52.087Z",
      "execution_time": "49.6s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.................................................................."
       },
       {
        "ename": "KeyboardInterrupt",
        "evalue": "",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                        Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     52\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     53\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mParallelSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 54\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m5\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     56\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/ParallelSearch.py\u001b[0m in \u001b[0;36mparallelRequest\u001b[0;34m(self, requests, requestPerThread)\u001b[0m\n\u001b[1;32m     47\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     48\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 49\u001b[0;31m     \u001b[0;32mdef\u001b[0m \u001b[0mmakeRequests\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestsList\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     50\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     51\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/threading.py\u001b[0m in \u001b[0;36mjoin\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m   1054\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1055\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mtimeout\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1056\u001b[0;31m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_wait_for_tstate_lock\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1057\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1058\u001b[0m             \u001b[0;31m# the behavior of a negative timeout isn't documented, but\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/threading.py\u001b[0m in \u001b[0;36m_wait_for_tstate_lock\u001b[0;34m(self, block, timeout)\u001b[0m\n\u001b[1;32m   1070\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mlock\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m  \u001b[0;31m# already determined that the C code is done\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1071\u001b[0m             \u001b[0;32massert\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_is_stopped\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1072\u001b[0;31m         \u001b[0;32melif\u001b[0m \u001b[0mlock\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0macquire\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mblock\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1073\u001b[0m             \u001b[0mlock\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelease\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1074\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_stop\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
        ]
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T20:45:02.514Z"
     },
     {
      "end_time": "2019-03-13T20:47:35.932Z",
      "execution_time": "1m 39.0s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.................................................................................................................................\nInternal Error(result ommited).\nInternal Error(result ommited).\nInternal Error(result ommited).\nInternal Error(result ommited)...."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T20:45:56.916Z"
     },
     {
      "end_time": "2019-03-13T21:16:37.381Z",
      "execution_time": "24.1s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait........................."
       },
       {
        "ename": "KeyboardInterrupt",
        "evalue": "",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                        Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     52\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     53\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mParallelSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 54\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m5\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     56\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/ParallelSearch.py\u001b[0m in \u001b[0;36mparallelRequest\u001b[0;34m(self, requests, requestPerThread)\u001b[0m\n\u001b[1;32m     43\u001b[0m             \u001b[0mnewThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mthreading\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mThread\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m\"Thread \"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcurrentThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtarget\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmakeRequests\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0margs\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mcurrentThread\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0mrequestPerThread\u001b[0m \u001b[0;34m:\u001b[0m \u001b[0mrightArrayBound\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     44\u001b[0m             \u001b[0mnewThread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstart\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 45\u001b[0;31m             \u001b[0mnewThread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     46\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     47\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/threading.py\u001b[0m in \u001b[0;36mjoin\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m   1054\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1055\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mtimeout\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1056\u001b[0;31m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_wait_for_tstate_lock\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1057\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1058\u001b[0m             \u001b[0;31m# the behavior of a negative timeout isn't documented, but\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/threading.py\u001b[0m in \u001b[0;36m_wait_for_tstate_lock\u001b[0;34m(self, block, timeout)\u001b[0m\n\u001b[1;32m   1070\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mlock\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m  \u001b[0;31m# already determined that the C code is done\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1071\u001b[0m             \u001b[0;32massert\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_is_stopped\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1072\u001b[0;31m         \u001b[0;32melif\u001b[0m \u001b[0mlock\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0macquire\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mblock\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1073\u001b[0m             \u001b[0mlock\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelease\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1074\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_stop\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
        ]
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T21:16:13.328Z"
     },
     {
      "end_time": "2019-03-13T21:17:01.518Z",
      "execution_time": "3.78s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.........\nInternal Error(result ommited)....................\nInternal Error(result ommited).......................................................\nInternal Error(result ommited)................\nInternal Error(result ommited)...................................."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T21:16:57.735Z"
     },
     {
      "end_time": "2019-03-13T21:58:08.328Z",
      "execution_time": "23ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "positional argument follows keyword argument (Search.py, line 54)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-6a184f23f92a>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from Search import Search\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m54\u001b[0m\n\u001b[0;31m    ps.parallelRequest(self.requestList, requestPerThread=5, numberOfResults)\u001b[0m\n\u001b[0m                                                            ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m positional argument follows keyword argument\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T21:58:08.305Z"
     },
     {
      "end_time": "2019-03-13T21:58:36.710Z",
      "execution_time": "93ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "non-default argument follows default argument (ParallelSearch.py, line 18)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "  File \u001b[1;32m\"<ipython-input-2-6a184f23f92a>\"\u001b[0m, line \u001b[1;32m1\u001b[0m, in \u001b[1;35m<module>\u001b[0m\n    from Search import Search\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m2\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from ParallelSearch import ParallelSearch\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSEEKNotebook/ParallelSearch.py\"\u001b[0;36m, line \u001b[0;32m18\u001b[0m\n\u001b[0;31m    def parallelRequest(self, requests, requestPerThread = PROT_REQUESTS_PER_THREAD, total):\u001b[0m\n\u001b[0m                       ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m non-default argument follows default argument\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T21:58:36.617Z"
     },
     {
      "end_time": "2019-03-13T21:59:21.050Z",
      "execution_time": "3.05s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.\nInternal Error(result ommited)\nInternal Error(result ommited)\nInternal Error(result ommited)\nInternal Error(result ommited)"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T21:59:18.001Z"
     },
     {
      "end_time": "2019-03-13T22:00:59.197Z",
      "execution_time": "3.00s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.\nInternal Error(result ommited)\nInternal Error(result ommited)\nInternal Error(result ommited)\nInternal Error(result ommited)"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:00:56.192Z"
     },
     {
      "end_time": "2019-03-13T22:01:45.337Z",
      "execution_time": "3.00s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.\nInternal Error(result ommited)\nInternal Error(result ommited)\nInternal Error(result ommited)\nInternal Error(result ommited)"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:01:42.341Z"
     },
     {
      "end_time": "2019-03-13T22:02:38.568Z",
      "execution_time": "2.40s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.\nInternal Error(result ommited)\nInternal Error(result ommited)\nInternal Error(result ommited)\nInternal Error(result ommited)"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:02:36.167Z"
     },
     {
      "end_time": "2019-03-13T22:04:14.446Z",
      "execution_time": "2.58s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.,\n,\n,\n,\n,\n,\n,,\n\n\nInternal Error(result ommited),\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n\nInternal Error(result ommited),\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,,,\n\n,\n\n,\n,\n,\n,\n,\n,,\n\n,\n,,\n\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,,\n\n,\n,\n,\n,\n,\n,\n,\n\nInternal Error(result ommited),\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n\nInternal Error(result ommited),\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:04:11.864Z"
     },
     {
      "end_time": "2019-03-13T22:10:23.051Z",
      "execution_time": "3.06s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.,\n,\n,\n,\n\nInternal Error(result ommited),\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n\nInternal Error(result ommited),\n,\n,\n,\n,\n,\n,,\n\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,,\n\n,\n,,\n\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n\nInternal Error(result ommited),\n,\n,\n,\n,,\n\n,\n,\n,\n,\n,\n,\n,\n,\n,,\n,\n\n\nInternal Error(result ommited),\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n,\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:10:19.989Z"
     },
     {
      "end_time": "2019-03-13T22:14:54.620Z",
      "execution_time": "5.62s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \niver\n"
       },
       {
        "ename": "KeyboardInterrupt",
        "evalue": "",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                        Traceback (most recent call last)",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36m_input_request\u001b[0;34m(self, prompt, ident, parent, password)\u001b[0m\n\u001b[1;32m    728\u001b[0m             \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 729\u001b[0;31m                 \u001b[0mident\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mreply\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstdin_socket\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    730\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mException\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/jupyter_client/session.py\u001b[0m in \u001b[0;36mrecv\u001b[0;34m(self, socket, mode, content, copy)\u001b[0m\n\u001b[1;32m    802\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 803\u001b[0;31m             \u001b[0mmsg_list\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msocket\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv_multipart\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmode\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    804\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mzmq\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mZMQError\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/zmq/sugar/socket.py\u001b[0m in \u001b[0;36mrecv_multipart\u001b[0;34m(self, flags, copy, track)\u001b[0m\n\u001b[1;32m    465\u001b[0m         \"\"\"\n\u001b[0;32m--> 466\u001b[0;31m         \u001b[0mparts\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mflags\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtrack\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mtrack\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    467\u001b[0m         \u001b[0;31m# have first part already, only loop while more to receive\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket.Socket.recv\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket.Socket.recv\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket._recv_copy\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/zmq/backend/cython/checkrc.pxd\u001b[0m in \u001b[0;36mzmq.backend.cython.checkrc._check_rc\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: ",
         "\nDuring handling of the above exception, another exception occurred:\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                        Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     36\u001b[0m         \u001b[0mchoice\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     37\u001b[0m         \u001b[0;32mwhile\u001b[0m \u001b[0mchoice\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchChoices\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 38\u001b[0;31m             \u001b[0mchoice\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0minput\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Please enter one of: \"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m', '\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchChoices\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     39\u001b[0m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchType\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mchoice\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     40\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36mraw_input\u001b[0;34m(self, prompt)\u001b[0m\n\u001b[1;32m    702\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_parent_ident\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    703\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_parent_header\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 704\u001b[0;31m             \u001b[0mpassword\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mFalse\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    705\u001b[0m         )\n\u001b[1;32m    706\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36m_input_request\u001b[0;34m(self, prompt, ident, parent, password)\u001b[0m\n\u001b[1;32m    732\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    733\u001b[0m                 \u001b[0;31m# re-raise KeyboardInterrupt, to truncate traceback\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 734\u001b[0;31m                 \u001b[0;32mraise\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    735\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    736\u001b[0m                 \u001b[0;32mbreak\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:14:48.999Z"
     },
     {
      "end_time": "2019-03-13T22:15:00.292Z",
      "execution_time": "3.10s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.......\nInternal Error(result ommited),\n.......................\nInternal Error(result ommited),\n.........................................\nInternal Error(result ommited),\n...........\nInternal Error(result ommited),\n.................................................."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:14:57.196Z"
     },
     {
      "end_time": "2019-03-13T22:18:03.816Z",
      "execution_time": "3.45s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait......\nInternal Error(result ommited)..................\nInternal Error(result ommited).............................................\nInternal Error(result ommited)..........\nInternal Error(result ommited)....................................................."
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:18:00.367Z"
     },
     {
      "end_time": "2019-03-13T22:18:35.602Z",
      "execution_time": "3.31s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.unsupported operand type(s) for /: 'int' and 'str'\n\nInternal Error(result ommited)unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n\nInternal Error(result ommited)unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n\nInternal Error(result ommited)unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n\nInternal Error(result ommited)unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:18:32.290Z"
     },
     {
      "end_time": "2019-03-13T22:19:42.317Z",
      "execution_time": "3.38s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.unsupported operand type(s) for /: 'int' and 'str'\n\nInternal Error(result ommited)unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n\nInternal Error(result ommited)unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n\nInternal Error(result ommited)unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n\nInternal Error(result ommited)unsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:19:38.939Z"
     },
     {
      "end_time": "2019-03-13T22:20:43.592Z",
      "execution_time": "5.34s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n"
       },
       {
        "ename": "ValueError",
        "evalue": "invalid literal for int() with base 10: '\\n135 results found'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mValueError\u001b[0m                               Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSEEKNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     52\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     53\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mParallelSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 54\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnumberOfResults\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m5\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     56\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mValueError\u001b[0m: invalid literal for int() with base 10: '\\n135 results found'"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:20:38.253Z"
     },
     {
      "end_time": "2019-03-13T22:22:58.849Z",
      "execution_time": "2.95s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading results. Please wait.Loading 0.0%Loading 0.7407407407407408%Loading 1.4814814814814816%Loading 2.2222222222222223%Loading 2.9629629629629632%Loading 3.7037037037037033%Loading 4.444444444444445%\nInternal Error(result ommited)Loading 5.9259259259259265%Loading 6.666666666666667%Loading 7.4074074074074066%Loading 7.4074074074074066%Loading 8.88888888888889%Loading 9.62962962962963%Loading 10.37037037037037%Loading 10.37037037037037%Loading 11.851851851851853%Loading 12.592592592592592%Loading 13.333333333333334%Loading 14.074074074074074%Loading 14.814814814814813%Loading 15.555555555555555%Loading 16.296296296296298%Loading 17.037037037037038%Loading 17.77777777777778%Loading 18.51851851851852%Loading 19.25925925925926%Loading 20.0%Loading 20.74074074074074%Loading 21.48148148148148%Loading 22.22222222222222%Loading 22.962962962962962%\nInternal Error(result ommited)Loading 24.444444444444443%Loading 25.185185185185183%Loading 25.925925925925924%Loading 26.666666666666668%Loading 27.40740740740741%Loading 28.14814814814815%Loading 28.888888888888886%Loading 29.629629629629626%Loading 30.37037037037037%Loading 31.11111111111111%Loading 31.851851851851855%Loading 32.592592592592595%Loading 33.33333333333333%Loading 34.074074074074076%Loading 34.81481481481482%Loading 35.55555555555556%Loading 36.2962962962963%Loading 37.03703703703704%Loading 37.77777777777778%Loading 38.51851851851852%Loading 39.25925925925926%Loading 40.0%Loading 40.74074074074074%Loading 41.48148148148148%Loading 42.22222222222222%Loading 42.96296296296296%Loading 43.7037037037037%Loading 44.44444444444444%Loading 45.18518518518518%Loading 45.925925925925924%Loading 45.925925925925924%Loading 47.40740740740741%Loading 48.148148148148145%Loading 48.888888888888886%Loading 49.629629629629626%Loading 50.37037037037037%Loading 51.11111111111111%Loading 51.85185185185185%Loading 52.59259259259259%Loading 53.333333333333336%Loading 54.074074074074076%Loading 54.81481481481482%Loading 55.55555555555556%Loading 56.2962962962963%Loading 57.03703703703704%Loading 57.77777777777777%Loading 58.51851851851851%Loading 59.25925925925925%Loading 60.0%Loading 60.74074074074074%\nInternal Error(result ommited)Loading 62.22222222222222%Loading 62.96296296296296%Loading 63.70370370370371%Loading 64.44444444444444%Loading 65.18518518518519%Loading 65.92592592592592%Loading 66.66666666666666%Loading 67.4074074074074%Loading 68.14814814814815%Loading 68.88888888888889%Loading 69.62962962962963%Loading 70.37037037037037%Loading 71.11111111111111%Loading 71.11111111111111%Loading 72.5925925925926%Loading 73.33333333333333%Loading 74.07407407407408%Loading 74.81481481481481%Loading 75.55555555555556%Loading 76.29629629629629%Loading 77.03703703703704%Loading 77.77777777777779%\nInternal Error(result ommited)Loading 79.25925925925927%Loading 80.0%Loading 80.74074074074075%Loading 81.48148148148148%Loading 82.22222222222221%Loading 82.96296296296296%Loading 83.7037037037037%Loading 84.44444444444444%Loading 85.18518518518519%Loading 85.92592592592592%Loading 86.66666666666667%Loading 87.4074074074074%Loading 88.14814814814815%Loading 88.88888888888889%Loading 89.62962962962962%Loading 90.37037037037037%Loading 91.11111111111111%Loading 91.85185185185185%Loading 92.5925925925926%Loading 93.33333333333333%Loading 94.07407407407408%Loading 94.81481481481482%Loading 95.55555555555556%Loading 96.29629629629629%Loading 97.03703703703704%Loading 97.77777777777777%Loading 98.51851851851852%Loading 99.25925925925925%"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:22:55.899Z"
     },
     {
      "end_time": "2019-03-13T22:23:50.311Z",
      "execution_time": "4.17s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 2.2222222222222223%Loading 2.2222222222222223%\nLoading 17.037037037037038%ed)Loading 4.444444444444445%\nLoading 62.96296296296296%%oading 47.40740740740741%852%Loading 38.51851851851852%Loading 48.888888888888886%\nLoading 79.25925925925927%Loading 77.03703703703704%Loading 70.37037037037037%\nLoading 99.25925925925925%ted)Loading 80.74074074074075%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:23:46.141Z"
     },
     {
      "end_time": "2019-03-13T22:26:02.206Z",
      "execution_time": "3.03s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 0.7407407407407408%\nLoading 15.555555555555555%ed)Loading 2.2222222222222223%\nLoading 58.51851851851851%%oading 51.11111111111111%7038%Loading 17.037037037037038%\nLoading 73.33333333333333%Loading 64.44444444444444%Loading 61.48148148148148%\nLoading 99.25925925925925%Loading 80.74074074074075%481%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:25:59.179Z"
     },
     {
      "end_time": "2019-03-13T22:27:26.587Z",
      "execution_time": "2.41s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 0.0%\u001b[F\nInternal Error(result ommited)Loading 1.4814814814814816%\u001b[FLoading 2.2222222222222223%\u001b[FLoading 2.9629629629629632%\u001b[FLoading 3.7037037037037033%\u001b[FLoading 4.444444444444445%\u001b[FLoading 5.185185185185185%\u001b[FLoading 5.9259259259259265%\u001b[FLoading 6.666666666666667%\u001b[FLoading 7.4074074074074066%\u001b[FLoading 7.4074074074074066%\u001b[FLoading 8.88888888888889%\u001b[FLoading 9.62962962962963%\u001b[FLoading 10.37037037037037%\u001b[FLoading 11.11111111111111%\u001b[FLoading 11.851851851851853%\u001b[FLoading 12.592592592592592%\u001b[FLoading 13.333333333333334%\u001b[FLoading 14.074074074074074%\u001b[FLoading 14.814814814814813%\u001b[FLoading 15.555555555555555%\u001b[FLoading 16.296296296296298%\u001b[FLoading 17.037037037037038%\u001b[FLoading 17.77777777777778%\u001b[FLoading 18.51851851851852%\u001b[F\nInternal Error(result ommited)Loading 20.0%\u001b[FLoading 20.74074074074074%\u001b[FLoading 21.48148148148148%\u001b[FLoading 22.22222222222222%\u001b[FLoading 22.962962962962962%\u001b[FLoading 23.703703703703706%\u001b[FLoading 24.444444444444443%\u001b[FLoading 25.185185185185183%\u001b[FLoading 25.925925925925924%\u001b[FLoading 26.666666666666668%\u001b[FLoading 27.40740740740741%\u001b[FLoading 28.14814814814815%\u001b[FLoading 28.888888888888886%\u001b[FLoading 29.629629629629626%\u001b[FLoading 30.37037037037037%\u001b[FLoading 31.11111111111111%\u001b[FLoading 31.851851851851855%Loading 31.851851851851855%\u001b[F\u001b[FLoading 33.33333333333333%\u001b[FLoading 34.074074074074076%\u001b[FLoading 34.81481481481482%\u001b[FLoading 35.55555555555556%\u001b[FLoading 36.2962962962963%\u001b[FLoading 37.03703703703704%\u001b[FLoading 37.77777777777778%\u001b[FLoading 38.51851851851852%\u001b[FLoading 39.25925925925926%\u001b[FLoading 40.0%\u001b[FLoading 40.74074074074074%\u001b[FLoading 41.48148148148148%\u001b[FLoading 42.22222222222222%\u001b[FLoading 42.96296296296296%\u001b[FLoading 43.7037037037037%\u001b[FLoading 44.44444444444444%\u001b[FLoading 45.18518518518518%\u001b[FLoading 45.925925925925924%\u001b[FLoading 46.666666666666664%\u001b[FLoading 47.40740740740741%\u001b[FLoading 48.148148148148145%\u001b[FLoading 48.888888888888886%Loading 48.888888888888886%\u001b[F\u001b[FLoading 50.37037037037037%\u001b[FLoading 51.11111111111111%Loading 51.11111111111111%\u001b[F\u001b[FLoading 52.59259259259259%\u001b[FLoading 53.333333333333336%\u001b[FLoading 54.074074074074076%\u001b[FLoading 54.81481481481482%\u001b[FLoading 55.55555555555556%\u001b[FLoading 56.2962962962963%\u001b[F\nInternal Error(result ommited)Loading 57.77777777777777%\u001b[FLoading 58.51851851851851%\u001b[FLoading 59.25925925925925%\u001b[FLoading 60.0%Loading 60.0%\u001b[F\u001b[FLoading 61.48148148148148%\u001b[FLoading 62.22222222222222%\u001b[FLoading 62.96296296296296%\u001b[FLoading 63.70370370370371%\u001b[FLoading 63.70370370370371%\u001b[FLoading 65.18518518518519%\u001b[FLoading 65.92592592592592%Loading 65.92592592592592%Loading 65.92592592592592%\u001b[F\u001b[F\u001b[FLoading 68.14814814814815%\u001b[FLoading 68.88888888888889%\u001b[FLoading 69.62962962962963%\u001b[FLoading 69.62962962962963%\u001b[FLoading 71.11111111111111%\u001b[F\nInternal Error(result ommited)Loading 71.85185185185186%\u001b[FLoading 73.33333333333333%\u001b[FLoading 74.07407407407408%\u001b[FLoading 74.81481481481481%\u001b[FLoading 75.55555555555556%\u001b[FLoading 76.29629629629629%\u001b[FLoading 77.03703703703704%\u001b[FLoading 77.77777777777779%\u001b[FLoading 78.51851851851852%\u001b[FLoading 79.25925925925927%\u001b[FLoading 80.0%\u001b[FLoading 80.74074074074075%\u001b[FLoading 81.48148148148148%\u001b[FLoading 82.22222222222221%\u001b[FLoading 82.96296296296296%\u001b[FLoading 83.7037037037037%\u001b[FLoading 84.44444444444444%\u001b[FLoading 85.18518518518519%\u001b[FLoading 85.92592592592592%\u001b[FLoading 86.66666666666667%\u001b[FLoading 87.4074074074074%\u001b[FLoading 88.14814814814815%\u001b[FLoading 88.88888888888889%\u001b[FLoading 89.62962962962962%\u001b[FLoading 90.37037037037037%\u001b[FLoading 91.11111111111111%\u001b[FLoading 91.85185185185185%\u001b[FLoading 92.5925925925926%\u001b[FLoading 93.33333333333333%\u001b[FLoading 94.07407407407408%\u001b[FLoading 94.81481481481482%\u001b[FLoading 94.81481481481482%\u001b[FLoading 96.29629629629629%\u001b[FLoading 97.03703703703704%\u001b[FLoading 97.77777777777777%\u001b[FLoading 98.51851851851852%\u001b[FLoading 99.25925925925925%\u001b[F"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:27:24.174Z"
     },
     {
      "end_time": "2019-03-13T22:28:22.153Z",
      "execution_time": "2.37s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 3.7037037037037033%\nLoading 17.037037037037038%ed)Loading 5.185185185185185%\nLoading 53.333333333333336%Loading 49.629629629629626%Loading 24.444444444444443%Loading 44.44444444444444%\nLoading 68.14814814814815%ted)Loading 54.81481481481482%\nLoading 99.25925925925925%Loading 94.07407407407408%963%Loading 95.55555555555556%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:28:19.787Z"
     },
     {
      "end_time": "2019-03-13T22:42:21.214Z",
      "execution_time": "2.95s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 2.9629629629629632%\nLoading 13.333333333333334%ed)Loading 4.444444444444445%\nLoading 51.11111111111111%%Loading 31.851851851851855%Loading 26.666666666666668%2%Loading 33.33333333333333%\nLoading 60.0%925925925925%%ed)\nLoading 99.25925925925925%ted)Loading 61.48148148148148%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:42:18.264Z"
     },
     {
      "end_time": "2019-03-13T22:44:14.327Z",
      "execution_time": "2.90s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 4.44%\nLoading 19.26%Loading 13.33%d)Loading 5.93%\nLoading 51.11%oading 40.0%8%d)Loading 20.74%\nLoading 61.48%Loading 53.33%d)Loading 52.59%\nLoading 99.26%(result ommited)Loading 62.96%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:44:11.425Z"
     },
     {
      "end_time": "2019-03-13T22:45:00.128Z",
      "execution_time": "2.67s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n\nInternal Error(result ommited)\nLoading 9.63%\nInternal Error(result ommited)\nLoading 58.52%Loading 39.26%\nInternal Error(result ommited)\nLoading 65.93%\nInternal Error(result ommited)\nLoading 99.26%Loading 70.37%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:44:57.458Z"
     },
     {
      "end_time": "2019-03-13T22:48:27.903Z",
      "execution_time": "2.58s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 0.0%\nLoading 0.74%\nLoading 1.48%\n\nInternal Error(result ommited)\nLoading 2.96%\nLoading 3.7%\nLoading 4.44%\nLoading 5.19%\nLoading 5.93%\nLoading 6.67%\nLoading 7.41%\nLoading 8.15%\nLoading 8.89%\nLoading 9.63%\nLoading 10.37%\nLoading 11.11%\nLoading 11.85%\nLoading 12.59%\nLoading 13.33%\nLoading 14.07%\nLoading 14.81%\nLoading 15.56%\n\nInternal Error(result ommited)\nLoading 17.04%\nLoading 17.78%\nLoading 18.52%\nLoading 18.52%\nLoading 19.26%\nLoading 20.0%\nLoading 21.48%\nLoading 22.22%\nLoading 22.96%\nLoading 23.7%\nLoading 24.44%\nLoading 25.19%\nLoading 25.93%\nLoading 26.67%\nLoading 27.41%\nLoading 28.15%\nLoading 28.89%\nLoading 29.63%\nLoading 30.37%\nLoading 31.11%\nLoading 31.85%\nLoading 32.59%\nLoading 33.33%\nLoading 34.07%\nLoading 34.07%\nLoading 35.56%\nLoading 36.3%\nLoading 37.04%\nLoading 37.78%\nLoading 38.52%\nLoading 39.26%\nLoading 40.0%\nLoading 40.74%\nLoading 41.48%\nLoading 42.22%\nLoading 42.96%\nLoading 43.7%\nLoading 44.44%\nLoading 45.19%\nLoading 45.93%\nLoading 46.67%\nLoading 47.41%\nLoading 48.15%\nLoading 48.89%\nLoading 49.63%\nLoading 50.37%\nLoading 51.11%\nLoading 51.85%\nLoading 52.59%\nLoading 53.33%\nLoading 54.07%\nLoading 54.81%\n\nInternal Error(result ommited)\nLoading 56.3%\nLoading 57.04%\nLoading 57.78%\nLoading 58.52%\nLoading 59.26%\nLoading 59.26%\nLoading 60.74%\nLoading 61.48%\nLoading 62.22%\nLoading 62.96%\n\nInternal Error(result ommited)\nLoading 64.44%\nLoading 65.19%\nLoading 65.93%\nLoading 66.67%\nLoading 67.41%\nLoading 68.15%\nLoading 68.89%\nLoading 69.63%\nLoading 70.37%\nLoading 71.11%\nLoading 71.85%\nLoading 72.59%\nLoading 73.33%\nLoading 73.33%\nLoading 74.81%\nLoading 75.56%\nLoading 76.3%\nLoading 77.04%\nLoading 77.78%\nLoading 77.78%\nLoading 79.26%\nLoading 80.0%\nLoading 80.74%\nLoading 81.48%\nLoading 82.22%\nLoading 82.96%\nLoading 83.7%\nLoading 84.44%\nLoading 85.19%\nLoading 85.93%\nLoading 86.67%\nLoading 87.41%\nLoading 88.15%\nLoading 88.89%\nLoading 89.63%\nLoading 90.37%\nLoading 91.11%\nLoading 91.85%\nLoading 92.59%\nLoading 93.33%\nLoading 94.07%\nLoading 94.81%\nLoading 95.56%\nLoading 96.3%\nLoading 97.04%\nLoading 97.78%\nLoading 98.52%\nLoading 99.26%\n\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:48:25.324Z"
     },
     {
      "end_time": "2019-03-13T22:49:35.333Z",
      "execution_time": "2.95s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 1.48%\nInternal Error(result ommited)\nLoading 14.07%oading 7.41%Loading 9.63%\nInternal Error(result ommited)\nLoading 51.85%oading 16.3%Loading 20.0%Loading 26.67%Loading 40.0%Loading 44.44%Loading 44.44%Loading 51.11%\nInternal Error(result ommited)\nLoading 62.96%Loading 62.96%\nInternal Error(result ommited)\nLoading 97.04%Loading 67.41%Loading 68.15%Loading 75.56%Loading 81.48%Loading 82.96%Loading 86.67%Loading 88.89%Loading 91.11%Loading 92.59%Loading 93.33%Loading 94.07%Loading 96.3%Loading 97.78%Loading 98.52%Loading 99.26%"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:49:32.386Z"
     },
     {
      "end_time": "2019-03-13T22:50:24.028Z",
      "execution_time": "2.98s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 6.67%\nInternal Error(result ommited)\nLoading 20.74%Loading 19.26%\nInternal Error(result ommited)\nLoading 52.59%Loading 22.22%\nInternal Error(result ommited)\nLoading 62.96%\nInternal Error(result ommited)\nLoading 99.26%Loading 88.89%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:50:21.051Z"
     },
     {
      "end_time": "2019-03-13T22:52:11.720Z",
      "execution_time": "3.25s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 7.41%\nInternal Error(result ommited)\nLoading 16.3%%Loading 12.59%\nInternal Error(result ommited)\nLoading 49.63%Loading 40.74%\nInternal Error(result ommited)\nLoading 57.04%\nInternal Error(result ommited)\nLoading Completedding 68.89%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:52:08.475Z"
     },
     {
      "end_time": "2019-03-13T22:52:50.060Z",
      "execution_time": "6.13s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 0.0%                     \nInternal Error(result ommited)\nLoading 14.07%                     \nInternal Error(result ommited)\nLoading 58.52%                     oading 56.3%                       \nInternal Error(result ommited)\nLoading 64.44%                     \nInternal Error(result ommited)\nLoading Completed                  Loading 90.37%                     \r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:52:43.928Z"
     },
     {
      "end_time": "2019-03-13T22:53:24.494Z",
      "execution_time": "5.20s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 4.44%                                                       \nInternal Error(result ommited)\nLoading 17.04%                                                       Loading 10.37%                                                       \nInternal Error(result ommited)\nLoading 54.07%                                                       Loading 39.26%                                                       \nInternal Error(result ommited)\nLoading 61.48%                                                       Loading 57.78%                                                       \nInternal Error(result ommited)\nLoading Completed                                                    Loading 65.19%                                                       \r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:53:19.290Z"
     },
     {
      "end_time": "2019-03-13T22:54:02.357Z",
      "execution_time": "4.44s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 4.44%\nInternal Error(result ommited)\nLoading 20.0%Loading 20.0%9%Loading 14.07%\nInternal Error(result ommited)\nLoading 55.56%Loading 34.07%\nInternal Error(result ommited)\nLoading 63.7%%\nInternal Error(result ommited)\nLoading 98.52%Loading 88.15%Loading 71.85%Loading 89.63%\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:53:57.917Z"
     },
     {
      "end_time": "2019-03-13T22:55:13.575Z",
      "execution_time": "5.83s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \ncell\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n810 results found\nLoading 43.46%Loading 41.73%ading 7.9%Loading 31.6%Loading 38.02%Loading 41.98%\nInternal Error(result ommited)\nLoading 98.89%Loading 93.46%Loading 49.14%Loading 55.43%Loading 82.84%\nLoading Completed\n\nLoading Completed\n\nLoading Completed\n\nLoading Completed\n\nLoading Completed\n\nLoading Completed\n\nLoading Completed\n\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:55:07.743Z"
     },
     {
      "end_time": "2019-03-13T22:59:01.475Z",
      "execution_time": "3.00s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 0.0%\nInternal Error(result ommited)\nLoading 14.81%oading 4.44%\nInternal Error(result ommited)\nLoading 56.3%%Loading 40.74%\nInternal Error(result ommited)\nLoading 65.19%\nInternal Error(result ommited)\nLoading 99.26%Loading 84.44%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:58:58.473Z"
     },
     {
      "end_time": "2019-03-13T22:59:32.488Z",
      "execution_time": "2.61s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 2.96%\nInternal Error(result ommited)\nLoading 14.81%Loading 12.59%\nInternal Error(result ommited)\nLoading 52.59%Loading 51.11%Loading 36.3%\nInternal Error(result ommited)\nLoading 60.74%\nInternal Error(result ommited)\nLoading 99.26%Loading 82.22%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T22:59:29.874Z"
     },
     {
      "end_time": "2019-03-13T23:03:00.648Z",
      "execution_time": "4.63s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 0.74%\nInternal Error(result ommited)\nLoading 14.07%\nInternal Error(result ommited)\nLoading 57.04%Loading 49.63%Loading 51.11%\nInternal Error(result ommited)\nLoading 65.93%Loading 65.93%\nInternal Error(result ommited)\nLoading 98.52%\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:02:56.019Z"
     },
     {
      "end_time": "2019-03-13T23:03:55.320Z",
      "execution_time": "2.81s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 0.0%\nInternal Error(result ommited)\nLoading 13.33%\nInternal Error(result ommited)\nLoading 54.07%Loading 26.67%\nInternal Error(result ommited)\nLoading 65.93%\nInternal Error(result ommited)\nLoading 99.26%\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:03:52.506Z"
     },
     {
      "end_time": "2019-03-13T23:08:17.415Z",
      "execution_time": "2.74s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 0.0Loading 0.74Loading 1.48Loading 2.22Loading 2.96Loading 3.7\nInternal Error(result ommited)\nLoading 5.19Loading 5.93Loading 6.67Loading 7.41Loading 8.15Loading 8.89Loading 9.63Loading 10.37Loading 11.11Loading 11.85Loading 12.59Loading 13.33Loading 14.07Loading 14.81\nInternal Error(result ommited)\nLoading 16.3Loading 17.04Loading 17.78Loading 18.52Loading 19.26Loading 20.0Loading 20.74Loading 21.48Loading 22.22Loading 22.96Loading 23.7Loading 24.44Loading 25.19Loading 25.93Loading 26.67Loading 27.41Loading 28.15Loading 28.89Loading 29.63Loading 30.37Loading 31.11Loading 31.85Loading 32.59Loading 33.33Loading 34.07Loading 34.81%Loading 34.8Loading 36.3Loading 37.04Loading 37.78%Loading 37.7Loading 39.26Loading 40.0Loading 40.74Loading 41.48Loading 42.22Loading 42.96Loading 43.7Loading 44.44Loading 45.19Loading 45.93%Loading 45.9Loading 47.41\nInternal Error(result ommited)\nLoading 48.89Loading 49.63Loading 50.37Loading 51.11Loading 51.85Loading 52.59Loading 53.33Loading 54.07Loading 54.81Loading 55.56\nInternal Error(result ommited)\nLoading 57.04Loading 57.78Loading 58.52Loading 59.26Loading 60.0Loading 60.74Loading 61.48Loading 62.22Loading 62.96%Loading 62.9Loading 64.44Loading 65.19Loading 65.93Loading 66.67Loading 67.41Loading 68.15%Loading 68.1Loading 69.63Loading 70.37Loading 71.11Loading 71.85Loading 72.59Loading 73.33Loading 74.07Loading 74.81Loading 75.56Loading 76.3Loading 77.04Loading 77.78Loading 78.52Loading 79.26Loading 80.0Loading 80.74Loading 81.48%Loading 81.4Loading 82.96Loading 83.7Loading 84.44Loading 85.19Loading 85.93Loading 86.67Loading 87.41Loading 88.15Loading 88.89Loading 89.63Loading 90.37Loading 91.11Loading 91.85Loading 92.59Loading 93.33Loading 94.07Loading 94.81Loading 95.56Loading 96.3Loading 97.04Loading 97.78Loading 98.52Loading 99.26%\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:08:14.678Z"
     },
     {
      "end_time": "2019-03-13T23:08:44.449Z",
      "execution_time": "3.09s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n\nInternal Error(result ommited)\nLoading 12.59%\nInternal Error(result ommited)\nLoading 57.04%Loading 46.67%\nInternal Error(result ommited)\nLoading 63.7%%\nInternal Error(result ommited)\nLoading 99.26%Loading 94.81%Loading 84.44%Loading 96.3%\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:08:41.361Z"
     },
     {
      "end_time": "2019-03-13T23:09:24.194Z",
      "execution_time": "2.98s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \na\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n0 results found\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:09:21.210Z"
     },
     {
      "end_time": "2019-03-13T23:09:39.888Z",
      "execution_time": "13.4s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nbio\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n1853 results found\nLoading 68.54%Loading 67.94%Loading 50.35%Loading 41.5%Loading 51.21%Loading 54.72%\nInternal Error(result ommited)\n\nInternal Error(result ommited)\n\nInternal Error(result ommited)\nLoading 69.24%\nInternal Error(result ommited)\n\nInternal Error(result ommited)\nLoading 88.77%oading 87.1%8%Loading 75.01%\nInternal Error(result ommited)\nLoading 90.72%Loading 90.34%\nInternal Error(result ommited)\nLoading 99.95%Loading 97.36%\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:09:26.440Z"
     },
     {
      "end_time": "2019-03-13T23:12:38.085Z",
      "execution_time": "7.03s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 2.22%\nInternal Error(result ommited)\nLoading 15.56%\nInternal Error(result ommited)\nLoading 57.78%Loading 49.63%\nInternal Error(result ommited)\nLoading 67.41%\nInternal Error(result ommited)\nLoading 99.26%\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:12:31.055Z"
     },
     {
      "end_time": "2019-03-13T23:13:18.726Z",
      "execution_time": "10.4s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nbio\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n1853 results found\nLoading 14.73%Loading 14.73%ading 4.43%Loading 14.25%Loading 14.25%Loading 14.25%Loading 14.25%\nInternal Error(result ommited)\nLoading 16.24%Loading 15.43%\nInternal Error(result ommited)\nLoading 53.1%%Loading 52.51%Loading 34.97%Loading 41.88%Loading 42.15%Loading 44.95%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:13:08.297Z"
     },
     {
      "end_time": "2019-03-13T23:14:08.684Z",
      "execution_time": "5.63s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n\nLoading 0.0%\nLoading 0.74%\nLoading 1.48%\nInternal Error(result ommited)\n\nLoading 2.96%\nLoading 3.7%\nLoading 4.44%\nLoading 5.19%\nLoading 5.93%\nLoading 6.67%\nLoading 7.41%\nLoading 8.15%\nLoading 8.89%\nLoading 9.63%\nLoading 10.37%\nLoading 11.11%\nLoading 11.85%\nLoading 12.59%\nLoading 13.33%\nLoading 14.07%\nLoading 14.07%\nLoading 14.81%\nLoading 16.3%\nInternal Error(result ommited)\n\nLoading 17.78%\nLoading 18.52%\nLoading 19.26%\nLoading 20.0%\nLoading 20.74%\nLoading 21.48%\nLoading 22.22%\nLoading 22.96%\nLoading 23.7%\nLoading 24.44%\nLoading 25.19%\nLoading 25.93%\nLoading 26.67%\nLoading 27.41%\nLoading 28.15%\nLoading 28.89%\nLoading 29.63%\nLoading 30.37%\nLoading 31.11%\nLoading 31.85%\nLoading 32.59%\nLoading 33.33%\nLoading 34.07%\nLoading 34.81%\nLoading 35.56%\nLoading 36.3%\nLoading 37.04%\nLoading 37.78%\nLoading 38.52%\nLoading 39.26%\nLoading 40.0%\nLoading 40.74%\nLoading 41.48%\nLoading 42.22%\nLoading 42.96%\nLoading 42.96%\nLoading 44.44%\nLoading 45.19%\nLoading 45.19%\nLoading 46.67%\nLoading 47.41%\nLoading 48.15%\nLoading 48.89%\nInternal Error(result ommited)\n\nLoading 50.37%\nLoading 51.11%\nLoading 51.85%\nLoading 52.59%\nLoading 53.33%\nLoading 54.07%\nLoading 54.81%\nLoading 55.56%\nLoading 56.3%\nLoading 57.04%\nLoading 57.78%\nLoading 58.52%\nLoading 59.26%\nInternal Error(result ommited)\n\nLoading 60.74%\nLoading 61.48%\nLoading 61.48%\nLoading 62.22%\nLoading 63.7%\nLoading 64.44%\nLoading 65.19%\nLoading 65.93%\nLoading 66.67%\nLoading 67.41%\nLoading 68.15%\nLoading 68.89%\nLoading 69.63%\nLoading 70.37%\nLoading 71.11%\nLoading 71.85%\nLoading 72.59%\nLoading 73.33%\nLoading 74.07%\nLoading 74.81%\nLoading 75.56%\nLoading 76.3%\nLoading 77.04%\nLoading 77.78%\nLoading 78.52%\nLoading 79.26%\nLoading 80.0%\nLoading 80.74%\nLoading 81.48%\nLoading 82.22%\nLoading 82.96%\nLoading 83.7%\nLoading 84.44%\nLoading 85.19%\nLoading 85.93%\nLoading 86.67%\nLoading 87.41%\nLoading 88.15%\nLoading 88.89%\nLoading 89.63%\nLoading 90.37%\nLoading 91.11%\nLoading 91.85%\nLoading 92.59%\nLoading 93.33%\nLoading 94.07%\nLoading 94.81%\nLoading 95.56%\nLoading 96.3%\nLoading 97.04%\nLoading 97.78%\nLoading 99.26%\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:14:03.059Z"
     },
     {
      "end_time": "2019-03-13T23:15:48.080Z",
      "execution_time": "2.42s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n\nLoading 0.0%\nInternal Error(result ommited)\n\nLoading 1.48%\nLoading 2.22%\nLoading 2.96%\nLoading 3.7%\nLoading 4.44%\nLoading 5.19%\nLoading 5.19%\nLoading 5.93%\nLoading 6.67%\nLoading 8.15%\nLoading 8.89%\nLoading 9.63%\nLoading 10.37%\nLoading 11.11%\nLoading 11.85%\nLoading 12.59%\nLoading 13.33%\nLoading 13.33%\nLoading 14.81%\nLoading 15.56%\nInternal Error(result ommited)\n\nLoading 17.04%\nLoading 17.04%\nLoading 18.52%\nLoading 19.26%\nLoading 20.0%\nLoading 20.0%\nLoading 21.48%\nLoading 22.22%\nLoading 22.96%\nLoading 23.7%\nLoading 24.44%\nLoading 25.19%\nLoading 25.93%\nLoading 25.93%\nLoading 27.41%\nLoading 28.15%\nLoading 28.89%\nLoading 29.63%\nLoading 30.37%\nLoading 31.11%\nLoading 31.85%\nLoading 32.59%\nLoading 33.33%\nLoading 34.07%\nLoading 34.81%\nLoading 35.56%\nLoading 36.3%\nLoading 37.04%\nLoading 37.78%\nLoading 38.52%\nLoading 39.26%\nLoading 40.0%\nLoading 40.74%\nLoading 41.48%\nLoading 42.22%\nLoading 42.96%\nLoading 43.7%\nLoading 44.44%\nLoading 45.19%\nLoading 45.93%\nLoading 46.67%\nLoading 47.41%\nLoading 48.15%\nLoading 48.89%\nLoading 49.63%\nLoading 50.37%\nLoading 51.11%\nLoading 51.85%\nLoading 52.59%\nLoading 53.33%\nLoading 54.07%\nLoading 54.81%\nLoading 55.56%\nLoading 56.3%\nInternal Error(result ommited)\n\nLoading 57.78%\nLoading 58.52%\nLoading 59.26%\nLoading 60.0%\nLoading 60.74%\nLoading 61.48%\nLoading 62.22%\nLoading 62.96%\nLoading 63.7%\nLoading 64.44%\nLoading 65.19%\nLoading 65.93%\nInternal Error(result ommited)\n\nLoading 67.41%\nLoading 68.15%\nLoading 68.89%\nLoading 69.63%\nLoading 70.37%\nLoading 71.11%\nLoading 71.85%\nLoading 72.59%\nLoading 73.33%\nLoading 74.07%\nLoading 74.81%\nLoading 75.56%\nLoading 76.3%\nLoading 77.04%\nLoading 77.78%\nLoading 78.52%\nLoading 79.26%\nLoading 80.0%\nLoading 80.74%\nLoading 81.48%\nLoading 82.22%\nLoading 82.96%\nLoading 83.7%\nLoading 84.44%\nLoading 85.19%\nLoading 85.93%\nLoading 86.67%\nLoading 87.41%\nLoading 88.15%\nLoading 88.89%\nLoading 89.63%\nLoading 90.37%\nLoading 91.11%\nLoading 91.85%\nLoading 92.59%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:15:45.656Z"
     },
     {
      "end_time": "2019-03-13T23:15:57.877Z",
      "execution_time": "3.94s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "\nLoading 93.33%\nLoading 94.07%\nLoading 94.81%\nLoading 95.56%\nLoading 96.3%\nLoading 97.04%\nEnter your search: \nliver\n\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\nLoading 99.26%\nLoading Completed\n\n135 results found\n\nLoading 0.0%\nLoading 0.74%\nLoading 1.48%\nLoading 2.22%\nLoading 2.96%\nLoading 3.7%\nLoading 4.44%\nInternal Error(result ommited)\n\nLoading 5.93%\nLoading 6.67%\nLoading 7.41%\nLoading 8.15%\nLoading 8.15%\nLoading 9.63%\nLoading 10.37%\nLoading 11.11%\nLoading 11.85%\nLoading 12.59%\nLoading 13.33%\nLoading 14.07%\nLoading 14.81%\nLoading 15.56%\nInternal Error(result ommited)\n\nLoading 16.3%\nLoading 17.78%\nLoading 18.52%\nLoading 19.26%\nLoading 20.0%\nLoading 20.74%\nLoading 21.48%\nLoading 22.22%\nLoading 22.96%\nLoading 23.7%\nLoading 24.44%\nLoading 25.19%\nLoading 25.93%\nLoading 26.67%\nLoading 27.41%\nLoading 28.15%\nLoading 28.89%\nLoading 29.63%\nLoading 30.37%\nLoading 31.11%\nLoading 31.85%\nLoading 32.59%\nLoading 33.33%\nLoading 34.07%\nLoading 34.81%\nLoading 35.56%\nLoading 36.3%\nLoading 37.04%\nLoading 37.78%\nLoading 38.52%\nLoading 39.26%\nLoading 40.0%\nLoading 40.74%\nLoading 41.48%\nLoading 42.22%\nLoading 42.96%\nLoading 43.7%\nLoading 44.44%\nLoading 45.19%\nLoading 45.93%\nLoading 46.67%\nLoading 47.41%\nLoading 48.15%\nLoading 48.89%\nLoading 49.63%\nLoading 50.37%\nLoading 51.11%\nLoading 51.85%\nLoading 52.59%\nLoading 53.33%\nLoading 54.07%\nLoading 54.81%\nLoading 55.56%\nLoading 56.3%\nInternal Error(result ommited)\n\nLoading 57.78%\nLoading 58.52%\nLoading 59.26%\nLoading 60.0%\nLoading 60.74%\nLoading 61.48%\nLoading 61.48%\nLoading 62.96%\nLoading 63.7%\nLoading 64.44%\nLoading 65.19%\nLoading 65.93%\nLoading 66.67%\nLoading 67.41%\nLoading 68.15%\nInternal Error(result ommited)\n\nLoading 69.63%\nLoading 70.37%\nLoading 71.11%\nLoading 71.85%\nLoading 72.59%\nLoading 73.33%\nLoading 74.07%\nLoading 74.81%\nLoading 75.56%\nLoading 76.3%\nLoading 77.04%\nLoading 77.78%\nLoading 78.52%\nLoading 79.26%\nLoading 80.0%\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:15:53.932Z"
     },
     {
      "end_time": "2019-03-13T23:16:11.578Z",
      "execution_time": "3.01s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 2.96%\nInternal Error(result ommited)\nLoading 14.07%\nInternal Error(result ommited)\nLoading 51.85%Loading 33.33%Loading 34.81%\nInternal Error(result ommited)\nLoading 64.44%oading 60.0%\nInternal Error(result ommited)\nLoading 99.26%Loading 69.63%\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-13T23:16:08.572Z"
     },
     {
      "end_time": "2019-03-14T11:10:54.259Z",
      "execution_time": "2.49s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 99.26%\nLoading Completed\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:10:51.772Z"
     },
     {
      "end_time": "2019-03-14T11:13:01.639Z",
      "execution_time": "3.47s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nThe number of relationships in this search = 0\n"
       },
       {
        "ename": "TypeError",
        "evalue": "parallelRequest() missing 1 required positional argument: 'total'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     64\u001b[0m         \u001b[0;31m# request all the relationships\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     65\u001b[0m         \u001b[0mPS\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mParallelSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 66\u001b[0;31m         \u001b[0mPS\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnon_Cached\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m5\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     67\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     68\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: parallelRequest() missing 1 required positional argument: 'total'"
        ]
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Loading 99.26%\nLoading Completed\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:12:58.166Z"
     },
     {
      "end_time": "2019-03-14T11:14:22.302Z",
      "execution_time": "2.39s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nThe number of relationships in this search = 0\nLoading 99.26%\nLoading Completed\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:14:19.915Z"
     },
     {
      "end_time": "2019-03-14T11:21:55.300Z",
      "execution_time": "2.49s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n"
       },
       {
        "ename": "AttributeError",
        "evalue": "'list' object has no attribute 'is_alive'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                           Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     56\u001b[0m         \u001b[0misWating\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mFalse\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     57\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 58\u001b[0;31m         \u001b[0;32mwhile\u001b[0m \u001b[0;32mTrue\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mthreadList\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mis_alive\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     59\u001b[0m             \u001b[0misWating\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mTrue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     60\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mAttributeError\u001b[0m: 'list' object has no attribute 'is_alive'"
        ]
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Loading 99.26%\nLoading Completed\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:21:52.812Z"
     },
     {
      "end_time": "2019-03-14T11:28:33.194Z",
      "execution_time": "14.1s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 99.26%\nfinnishedompleted\n\n557 relationships found\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "unsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:28:19.059Z"
     },
     {
      "end_time": "2019-03-14T11:29:45.041Z",
      "execution_time": "12.6s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 99.26%\nLoading Completed\n557 relationships found\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'unsupported operand type(s) for /: 'int' and 'str'\n\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\nunsupported operand type(s) for /: 'int' and 'str'\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:29:32.457Z"
     },
     {
      "end_time": "2019-03-14T11:46:18.856Z",
      "execution_time": "13.0s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 99.26%\nLoading Completed\n557 relationships found\n"
       },
       {
        "ename": "ValueError",
        "evalue": "invalid literal for int() with base 10: '557 relat'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mValueError\u001b[0m                               Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     70\u001b[0m         \u001b[0;31m# request all the relationships\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     71\u001b[0m         \u001b[0mPS\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mParallelSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 72\u001b[0;31m         \u001b[0mPS\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnon_Cached\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnumberOfRelationships\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m14\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m5\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     73\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     74\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mValueError\u001b[0m: invalid literal for int() with base 10: '557 relat'"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:46:05.866Z"
     },
     {
      "end_time": "2019-03-14T11:47:22.912Z",
      "execution_time": "12.7s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 99.26%\nLoading Completed\n557 relationships found\nLoading 99.82%\nLoading Completed\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:47:10.215Z"
     },
     {
      "end_time": "2019-03-14T11:48:44.228Z",
      "execution_time": "11.3s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nSubstituting relationships into original search results: \nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nObject has no relationships\n\n________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nObject has no relationships\n\n________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nObject has no relationships\n\n________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nObject has no relationships\n\n________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nObject has no relationships\n\n________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nObject has no relationships\n\n________________________________________________________\n\nLoading 96.0%\nLoading Completed\r"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:48:32.900Z"
     },
     {
      "end_time": "2019-03-14T11:49:28.332Z",
      "execution_time": "13.9s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nLoading 96.0%\nSubstituting relationships into original search results: \n..............................................Galactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:49:14.430Z"
     },
     {
      "end_time": "2019-03-14T11:51:39.490Z",
      "execution_time": "42.9s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 99.26%\nLoading Completed\n557 relationships found\nLoading 99.82%\nSubstituting relationships into original search results: \n............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n\nSEARCH RESULTS\n\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hnicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1 axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias Knig\n\n________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\n\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\n\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n________________________________________________________\n\nKollevg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias Knig\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nUte Spiekerktter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias Knig\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias Knig\n\n________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n________________________________________________________\n\nUrsula Klingmller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nAn update on SEEK BioSamples\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29  30 November 2012, Heidelberg\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n________________________________________________________\n\nLiver chemistry_Kollevg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\n\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Mller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhh\nDorota Kurek\nUte Spiekerktter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\nErik Schrder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n________________________________________________________\n\nTBARS_Kollevg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nUrsula Klingmller\nJens Timmer\nMartin Golebiewski\nMartin Bhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n________________________________________________________\n\nOliver Ebenhh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\n\nMore information available from\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n________________________________________________________\n\nqPCR liver_Kollevg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\n\n<b>Topics include:</b>\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\n- Comprehensive and unambiguous data description and annotation\n- Standard formats and terminologies for data and metadata \n- Guidelines for reporting data (Minimum information standards) \n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\n- Using SEEK & FAIRDOMhub for integrated data management\n\n<b>Registration:</b> http://bit.ly/2ArOC7w\n(please state if you will attend on both days or only on one day)\n\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\n\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hnfeld will be refunded.\n\n<b>Organization:</b>\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Model Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n________________________________________________________\n\nMatthias Knig(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\r\nRead more about the LiSyM junior group at: www.livermetabolism.com\r\n\r\nJunior Group Leader\r\nDr. Matthias Knig\r\n\r\nInstitute for Theoretical Biology\r\nHumboldt-University Berlin\r\nInvalidenstrae 43, 10117 Berlin, Germany\r\nphone +49 30 2093-8450\r\nkonigmatt@googlemail.com\r\n\r\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\r\n\r\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universitt zu Berlin\nCharit University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universitt Dresden.\r\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\r\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\r\n\r\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\r\n\r\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias Knig\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias Knig\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias Knig\n\n________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nWolfgang Mller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\r\n\r\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\r\n\r\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\r\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\r\nMESI-STRAT Coordinator\r\nLeader WP8  Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seil Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \r\n19th-20th of March, 2012, Vienna, Austria\r\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Mller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\r\n\r\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \r\n\r\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\r\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias Knig\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Brries\nBernhard Steiert\nChristian Tnsing\nUte Spiekerktter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\n\n________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dchert\nMartin Bhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\r\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\r\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universitt zu Berlin\nUniversity of Freiburg\nCharit University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nMatthias Knig\nJohannes Bausch\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n________________________________________________________\n\nMartin Bhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n________________________________________________________\n\nLeibniz-Institut fr Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\r\n\r\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \r\n\r\nKey questions:\r\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\r\n\r\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\r\n\r\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\r\n\r\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharit University Medicine Berlin\nGriffin Discoveries BV\nUniversittsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Bhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Bhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Bhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Bhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n________________________________________________________\n\nCharit University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias Knig\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\r\nMy current research deals with the integration of different metabolic pathways. Research lines include:\r\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\r\n- The relation between metabolism and biologial ageing;\r\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemnde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP  EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemnde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\r\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\nSiberian SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n________________________________________________________\n\nPablo Jos Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndr Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nUrsula Klingmller\nJens Timmer\nMartin Golebiewski\nMartin Bhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n________________________________________________________\n\nHumboldt-Universitt zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias Knig\nJudith Wodke\nClemens Khn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\r\n\r\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\r\n\r\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Mller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \r\n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Mller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universitt zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universitt Dresden\nGerman Cancer Research Center (DKFZ)\nCharit University Medicine Berlin\nHumboldt University Berlin\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nUniversittsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Mller\nMaja Rey\nUrsula Klingmller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias Knig\nMartin Bhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schrder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schnfels\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hnfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Sderstrm\nLibe Aranguren Abada\nSiri fsthus Goksyr\nOdd Andr Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\r\n\r\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmons ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmons body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\r\n\r\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and fed in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seil Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Rsg\nSahar Hassani\nMarius Andr Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigacin Prncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversit catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Mller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\n\n________________________________________________________\n\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:50:56.541Z"
     },
     {
      "end_time": "2019-03-14T11:52:42.807Z",
      "execution_time": "14.8s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nLoading 96.0%\nSubstituting relationships into original search results: \n..............................................\n\nSEARCH RESULTS\n\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:52:28.001Z"
     },
     {
      "end_time": "2019-03-14T11:54:25.034Z",
      "execution_time": "14.3s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nLoading 96.0%\nSubstituting relationships into original search results: \n..............................................\n\n                    --SEARCH RESULTS--\n\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:54:10.731Z"
     },
     {
      "end_time": "2019-03-14T11:55:04.556Z",
      "execution_time": "14.7s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nLoading 96.0%\nSubstituting relationships into original search results: \n..............................................\n\n                                        --SEARCH RESULTS--\n\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:54:49.873Z"
     },
     {
      "end_time": "2019-03-14T11:57:16.449Z",
      "execution_time": "13.6s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nLoading 96.0%\nSubstituting relationships into original search results: \n"
       },
       {
        "ename": "TypeError",
        "evalue": "must be str, not int",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-6a184f23f92a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     75\u001b[0m             \u001b[0mthread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     76\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 77\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msubstituteRelationships\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mPS\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     78\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     79\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\\n\\n                                        --SEARCH RESULTS--\\n\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/ParallelSearch.py\u001b[0m in \u001b[0;36msubstituteRelationships\u001b[0;34m(self, relationshipsList)\u001b[0m\n\u001b[1;32m    158\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    159\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 160\u001b[0;31m                                         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Loading \"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpercentageLoaded\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"%\\r\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mend\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m''\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    161\u001b[0m                                         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpercentageLoaded\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpercentageLoaded\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;36m1\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    162\u001b[0m                                         \u001b[0;32mif\u001b[0m \u001b[0mhasattr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrelation\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'newData'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: must be str, not int"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:57:02.871Z"
     },
     {
      "end_time": "2019-03-14T11:58:42.025Z",
      "execution_time": "45.4s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 99.26%\nLoading Completed\n557 relationships found\nLoading 99.82%\nSubstituting relationships into original search results: \nLoading 258.55855855855856%%\n\n                                        --SEARCH RESULTS--\n\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias Knig\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1 axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hnicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nUte Spiekerktter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nKollevg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nUrsula Klingmller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29  30 November 2012, Heidelberg\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias Knig\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\n\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\n\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nErik Schrder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\n\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Mller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhh\nDorota Kurek\nUte Spiekerktter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nOliver Ebenhh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\n\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias Knig\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias Knig\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nTBARS_Kollevg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nUrsula Klingmller\nJens Timmer\nMartin Golebiewski\nMartin Bhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nqPCR liver_Kollevg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMatthias Knig(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\nRead more about the LiSyM junior group at: www.livermetabolism.com\n\nJunior Group Leader\nDr. Matthias Knig\n\nInstitute for Theoretical Biology\nHumboldt-University Berlin\nInvalidenstrae 43, 10117 Berlin, Germany\nphone +49 30 2093-8450\nkonigmatt@googlemail.com\n\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\n\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universitt zu Berlin\nCharit University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\n\nMore information available from\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\n\n<b>Topics include:</b>\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\n- Comprehensive and unambiguous data description and annotation\n- Standard formats and terminologies for data and metadata \n- Guidelines for reporting data (Minimum information standards) \n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\n- Using SEEK & FAIRDOMhub for integrated data management\n\n<b>Registration:</b> http://bit.ly/2ArOC7w\n(please state if you will attend on both days or only on one day)\n\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\n\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hnfeld will be refunded.\n\n<b>Organization:</b>\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\n\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nWolfgang Mller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\n\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\n\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \n19th-20th of March, 2012, Vienna, Austria\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Mller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universitt Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Bhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Bhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Bhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias Knig\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seil Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\n\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \n\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dchert\nMartin Bhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\nMESI-STRAT Coordinator\nLeader WP8  Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nMartin Bhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Brries\nBernhard Steiert\nChristian Tnsing\nUte Spiekerktter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\nMy current research deals with the integration of different metabolic pathways. Research lines include:\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\n- The relation between metabolism and biologial ageing;\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universitt zu Berlin\nUniversity of Freiburg\nCharit University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nMatthias Knig\nJohannes Bausch\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nLeibniz-Institut fr Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nPablo Jos Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\n\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \n\nKey questions:\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\n\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\n\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\n\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharit University Medicine Berlin\nGriffin Discoveries BV\nUniversittsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Bhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nCharit University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias Knig\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Mller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nHumboldt-Universitt zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias Knig\nJudith Wodke\nClemens Khn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndr Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nUrsula Klingmller\nJens Timmer\nMartin Golebiewski\nMartin Bhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Description: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemnde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP  EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemnde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\r\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\nSiberian SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\r\n\r\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\r\n\r\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Mller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universitt zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universitt Dresden\nGerman Cancer Research Center (DKFZ)\nCharit University Medicine Berlin\nHumboldt University Berlin\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nUniversittsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Mller\nMaja Rey\nUrsula Klingmller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias Knig\nMartin Bhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schrder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schnfels\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \r\n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Sderstrm\nLibe Aranguren Abada\nSiri fsthus Goksyr\nOdd Andr Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hnfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\r\n\r\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmons ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmons body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\r\n\r\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and fed in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seil Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Rsg\nSahar Hassani\nMarius Andr Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigacin Prncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversit catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Mller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T11:57:56.614Z"
     },
     {
      "end_time": "2019-03-14T12:03:07.250Z",
      "execution_time": "113ms",
      "outputs": [
       {
        "ename": "IndentationError",
        "evalue": "expected an indented block (ParallelSearch.py, line 21)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "  File \u001b[1;32m\"<ipython-input-1-6a184f23f92a>\"\u001b[0m, line \u001b[1;32m1\u001b[0m, in \u001b[1;35m<module>\u001b[0m\n    from Search import Search\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m2\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from ParallelSearch import ParallelSearch\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/ParallelSearch.py\"\u001b[0;36m, line \u001b[0;32m21\u001b[0m\n\u001b[0;31m    def parallelRequest(self, requests, total, requestPerThread = PROT_REQUESTS_PER_THREAD):\u001b[0m\n\u001b[0m      ^\u001b[0m\n\u001b[0;31mIndentationError\u001b[0m\u001b[0;31m:\u001b[0m expected an indented block\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T12:03:07.137Z"
     },
     {
      "end_time": "2019-03-14T12:03:34.030Z",
      "execution_time": "21ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (ParallelSearch.py, line 163)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "  File \u001b[1;32m\"<ipython-input-2-6a184f23f92a>\"\u001b[0m, line \u001b[1;32m1\u001b[0m, in \u001b[1;35m<module>\u001b[0m\n    from Search import Search\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m2\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from ParallelSearch import ParallelSearch\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/ParallelSearch.py\"\u001b[0;36m, line \u001b[0;32m163\u001b[0m\n\u001b[0;31m    self.percentageLoaded = self.percentageLoaded + 1\u001b[0m\n\u001b[0m       ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T12:03:34.009Z"
     },
     {
      "end_time": "2019-03-14T12:04:22.699Z",
      "execution_time": "7ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (ParallelSearch.py, line 41)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "  File \u001b[1;32m\"<ipython-input-3-6a184f23f92a>\"\u001b[0m, line \u001b[1;32m1\u001b[0m, in \u001b[1;35m<module>\u001b[0m\n    from Search import Search\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m2\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from ParallelSearch import ParallelSearch\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/ParallelSearch.py\"\u001b[0;36m, line \u001b[0;32m41\u001b[0m\n\u001b[0;31m    \"\"\"relationshipsList\u001b[0m\n\u001b[0m                        \n^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T12:04:22.692Z"
     },
     {
      "end_time": "2019-03-14T12:04:35.511Z",
      "execution_time": "113ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (ParallelSearch.py, line 41)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "  File \u001b[1;32m\"<ipython-input-1-6a184f23f92a>\"\u001b[0m, line \u001b[1;32m1\u001b[0m, in \u001b[1;35m<module>\u001b[0m\n    from Search import Search\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m2\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from ParallelSearch import ParallelSearch\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/ParallelSearch.py\"\u001b[0;36m, line \u001b[0;32m41\u001b[0m\n\u001b[0;31m    \"\"\"relationshipsList\u001b[0m\n\u001b[0m                        \n^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T12:04:35.398Z"
     },
     {
      "end_time": "2019-03-14T12:05:11.091Z",
      "execution_time": "14.4s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nLoading 96.0%\nLoading Completed\nSubstituting relationships into original search results: \nLoading 195.65%\n\n                                        --SEARCH RESULTS--\n\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T12:04:56.641Z"
     },
     {
      "end_time": "2019-03-14T12:06:46.881Z",
      "execution_time": "136ms",
      "outputs": [
       {
        "ename": "IndentationError",
        "evalue": "unindent does not match any outer indentation level (ParallelSearch.py, line 175)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "  File \u001b[1;32m\"<ipython-input-1-6a184f23f92a>\"\u001b[0m, line \u001b[1;32m1\u001b[0m, in \u001b[1;35m<module>\u001b[0m\n    from Search import Search\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m2\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from ParallelSearch import ParallelSearch\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/ParallelSearch.py\"\u001b[0;36m, line \u001b[0;32m175\u001b[0m\n\u001b[0;31m    for item in relationshipsList:\u001b[0m\n\u001b[0m                                  ^\u001b[0m\n\u001b[0;31mIndentationError\u001b[0m\u001b[0;31m:\u001b[0m unindent does not match any outer indentation level\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T12:06:46.745Z"
     },
     {
      "end_time": "2019-03-14T12:07:39.377Z",
      "execution_time": "44.9s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 99.26%\nLoading Completed\n557 relationships found\nLoading 99.82%\nLoading Completed\nSubstituting relationships into original search results: \n1oading 4.86%\n1oading 5.23%\n1oading 7.21%\n1oading 7.93%\n1oading 60.72%\n1oading 61.08%\n1oading 81.08%\n1oading 109.73%\n1oading 133.33%\n1oading 133.69%\n1oading 155.32%\n1oading 155.68%\n1oading 194.41%\n1oading 194.95%\nLoading 258.56%\n\n                                        --SEARCH RESULTS--\n\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hnicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias Knig\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1 axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29  30 November 2012, Heidelberg\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\n\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\n\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nUte Spiekerktter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nKollevg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nUrsula Klingmller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias Knig\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\n\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Mller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhh\nDorota Kurek\nUte Spiekerktter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nErik Schrder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTBARS_Kollevg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nOliver Ebenhh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\n\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nUrsula Klingmller\nJens Timmer\nMartin Golebiewski\nMartin Bhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\n\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias Knig\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias Knig\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\n\n<b>Topics include:</b>\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\n- Comprehensive and unambiguous data description and annotation\n- Standard formats and terminologies for data and metadata \n- Guidelines for reporting data (Minimum information standards) \n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\n- Using SEEK & FAIRDOMhub for integrated data management\n\n<b>Registration:</b> http://bit.ly/2ArOC7w\n(please state if you will attend on both days or only on one day)\n\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\n\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hnfeld will be refunded.\n\n<b>Organization:</b>\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\n\nMore information available from\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nqPCR liver_Kollevg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nMatthias Knig(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\nRead more about the LiSyM junior group at: www.livermetabolism.com\n\nJunior Group Leader\nDr. Matthias Knig\n\nInstitute for Theoretical Biology\nHumboldt-University Berlin\nInvalidenstrae 43, 10117 Berlin, Germany\nphone +49 30 2093-8450\nkonigmatt@googlemail.com\n\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\n\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universitt zu Berlin\nCharit University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universitt Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\nMESI-STRAT Coordinator\nLeader WP8  Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \n19th-20th of March, 2012, Vienna, Austria\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Mller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nWolfgang Mller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\n\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\n\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seil Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\n\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \n\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Bhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Bhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Bhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dchert\nMartin Bhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias Knig\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\n\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \n\nKey questions:\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\n\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\n\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\n\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharit University Medicine Berlin\nGriffin Discoveries BV\nUniversittsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Bhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nMartin Bhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Brries\nBernhard Steiert\nChristian Tnsing\nUte Spiekerktter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universitt zu Berlin\nUniversity of Freiburg\nCharit University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nMatthias Knig\nJohannes Bausch\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nLeibniz-Institut fr Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\nMy current research deals with the integration of different metabolic pathways. Research lines include:\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\n- The relation between metabolism and biologial ageing;\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nCharit University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias Knig\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemnde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP  EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemnde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\nSiberian SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nPablo Jos Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Mller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndr Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nUrsula Klingmller\nJens Timmer\nMartin Golebiewski\nMartin Bhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nHumboldt-Universitt zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias Knig\nJudith Wodke\nClemens Khn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\n\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\n\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Mller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universitt zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universitt Dresden\nGerman Cancer Research Center (DKFZ)\nCharit University Medicine Berlin\nHumboldt University Berlin\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nUniversittsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Mller\nMaja Rey\nUrsula Klingmller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias Knig\nMartin Bhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schrder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schnfels\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hnfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\n\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmons ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmons body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\n\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and fed in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seil Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Rsg\nSahar Hassani\nMarius Andr Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Sderstrm\nLibe Aranguren Abada\nSiri fsthus Goksyr\nOdd Andr Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigacin Prncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversit catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Mller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T12:06:54.473Z"
     },
     {
      "end_time": "2019-03-14T12:12:06.977Z",
      "execution_time": "17.6s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nLoading 96.0%\nLoading Completed\nSubstituting relationships into original search results: \n1oading 3.92%\n1oading 7.84%\n1oading 13.73%\n1oading 21.57%\n1oading 31.37%\n1oading 35.29%\n1oading 47.06%\n1oading 50.98%\n1oading 60.78%\n1oading 64.71%\n1oading 78.43%\n1oading 84.31%\nLoading 88.24%\n\n                                        --SEARCH RESULTS--\n\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T12:11:49.344Z"
     },
     {
      "end_time": "2019-03-14T12:15:24.922Z",
      "execution_time": "15.1s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nLoading 96.0%\nLoading Completed\nSubstituting relationships into original search results: \nLoading 88.24%\n\n                                        --SEARCH RESULTS--\n\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.search()",
      "start_time": "2019-03-14T12:15:09.857Z"
     },
     {
      "end_time": "2019-03-14T12:23:27.817Z",
      "execution_time": "1m 15.8s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Search multithreading\nDecide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once1\nHow many relationships should be requested at once10\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T12:22:11.977Z"
     },
     {
      "end_time": "2019-03-14T12:24:20.435Z",
      "execution_time": "8.72s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Search multithreading\nDecide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 1\nHow many relationships should be requested at once: 3\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T12:24:11.710Z"
     },
     {
      "end_time": "2019-03-14T12:24:52.025Z",
      "execution_time": "5.61s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Search multithreading\nDecide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 1\nHow many relationships should be requested at once: 1\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T12:24:46.412Z"
     },
     {
      "end_time": "2019-03-14T12:48:26.732Z",
      "execution_time": "6.87s",
      "outputs": [
       {
        "data": {
         "text/html": "<h1 color=\"navy\">Search multithreading</h1>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\n"
       },
       {
        "ename": "KeyboardInterrupt",
        "evalue": "",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                        Traceback (most recent call last)",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36m_input_request\u001b[0;34m(self, prompt, ident, parent, password)\u001b[0m\n\u001b[1;32m    728\u001b[0m             \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 729\u001b[0;31m                 \u001b[0mident\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mreply\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstdin_socket\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    730\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mException\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/jupyter_client/session.py\u001b[0m in \u001b[0;36mrecv\u001b[0;34m(self, socket, mode, content, copy)\u001b[0m\n\u001b[1;32m    802\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 803\u001b[0;31m             \u001b[0mmsg_list\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msocket\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv_multipart\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmode\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    804\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mzmq\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mZMQError\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/zmq/sugar/socket.py\u001b[0m in \u001b[0;36mrecv_multipart\u001b[0;34m(self, flags, copy, track)\u001b[0m\n\u001b[1;32m    465\u001b[0m         \"\"\"\n\u001b[0;32m--> 466\u001b[0;31m         \u001b[0mparts\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mflags\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtrack\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mtrack\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    467\u001b[0m         \u001b[0;31m# have first part already, only loop while more to receive\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket.Socket.recv\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket.Socket.recv\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket._recv_copy\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/zmq/backend/cython/checkrc.pxd\u001b[0m in \u001b[0;36mzmq.backend.cython.checkrc._check_rc\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: ",
         "\nDuring handling of the above exception, another exception occurred:\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                        Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-5bc72e443646>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchAdvancedSetup\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearchAdvancedSetup\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     97\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Search results are usually less and the search requires less to be requested at once.\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     98\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Relationships are usually a lot more, therefore it requires more requests at once\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 99\u001b[0;31m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchResultsPerThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"How many search results should be requested at once: \"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    100\u001b[0m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelationshipsPerThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"How many relationships should be requested at once: \"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    101\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36mraw_input\u001b[0;34m(self, prompt)\u001b[0m\n\u001b[1;32m    702\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_parent_ident\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    703\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_parent_header\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 704\u001b[0;31m             \u001b[0mpassword\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mFalse\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    705\u001b[0m         )\n\u001b[1;32m    706\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36m_input_request\u001b[0;34m(self, prompt, ident, parent, password)\u001b[0m\n\u001b[1;32m    732\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    733\u001b[0m                 \u001b[0;31m# re-raise KeyboardInterrupt, to truncate traceback\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 734\u001b[0;31m                 \u001b[0;32mraise\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    735\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    736\u001b[0m                 \u001b[0;32mbreak\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T12:48:19.859Z"
     },
     {
      "end_time": "2019-03-14T12:52:46.413Z",
      "execution_time": "24ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (Search.py, line 95)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-5bc72e443646>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from Search import Search\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m95\u001b[0m\n\u001b[0;31m    display(HTML('<h3 style=\"color: navy\"'>Search multithreading</h3>'))\u001b[0m\n\u001b[0m                                                               ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T12:52:46.389Z"
     },
     {
      "end_time": "2019-03-14T12:53:55.407Z",
      "execution_time": "12ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (Search.py, line 95)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-2-5bc72e443646>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from Search import Search\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m95\u001b[0m\n\u001b[0;31m    display(HTML('<h3 style=\"color: navy\"'>Search multithreading</h3>'))\u001b[0m\n\u001b[0m                                                               ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T12:53:55.395Z"
     },
     {
      "end_time": "2019-03-14T12:53:58.263Z",
      "execution_time": "12ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (Search.py, line 95)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-3-5bc72e443646>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from Search import Search\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\"\u001b[0;36m, line \u001b[0;32m95\u001b[0m\n\u001b[0;31m    display(HTML('<h3'>Search multithreading</h3>'))\u001b[0m\n\u001b[0m                                           ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T12:53:58.251Z"
     },
     {
      "end_time": "2019-03-14T12:56:14.560Z",
      "execution_time": "2.12s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: liver\n"
       },
       {
        "ename": "ValueError",
        "evalue": "invalid literal for int() with base 10: 'liver'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mValueError\u001b[0m                               Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-5bc72e443646>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchAdvancedSetup\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearchAdvancedSetup\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     98\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Search results are usually less and the search requires less to be requested at once.\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     99\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Relationships are usually a lot more, therefore it requires more requests at once\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 100\u001b[0;31m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchResultsPerThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"How many search results should be requested at once: \"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    101\u001b[0m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelationshipsPerThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"How many relationships should be requested at once: \"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    102\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mValueError\u001b[0m: invalid literal for int() with base 10: 'liver'"
        ]
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T12:56:12.441Z"
     },
     {
      "end_time": "2019-03-14T12:56:29.272Z",
      "execution_time": "3.57s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 1\nHow many relationships should be requested at once: 1\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T12:56:25.702Z"
     },
     {
      "end_time": "2019-03-14T12:57:13.586Z",
      "execution_time": "1.76s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 1\nHow many relationships should be requested at once: 1\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T12:57:11.824Z"
     },
     {
      "end_time": "2019-03-14T13:04:16.857Z",
      "execution_time": "1.68s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 1\nHow many relationships should be requested at once: 1\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T13:04:15.177Z"
     },
     {
      "end_time": "2019-03-14T13:18:43.782Z",
      "execution_time": "1m 33.0s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 10\nHow many relationships should be requested at once: 10\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T13:17:10.799Z"
     },
     {
      "end_time": "2019-03-14T15:05:53.766Z",
      "execution_time": "2.07s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 1\nHow many relationships should be requested at once: 1\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T15:05:51.699Z"
     },
     {
      "end_time": "2019-03-14T15:16:10.917Z",
      "execution_time": "1.78s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 1\nHow many relationships should be requested at once: 1\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T15:16:09.140Z"
     },
     {
      "end_time": "2019-03-14T15:38:09.542Z",
      "execution_time": "1.37s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 1\nHow many relationships should be requested at once: 1\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T15:38:08.172Z"
     },
     {
      "end_time": "2019-03-14T15:39:02.805Z",
      "execution_time": "54.6s",
      "outputs": [],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T15:38:08.172Z"
     },
     {
      "end_time": "2019-03-14T15:39:12.807Z",
      "execution_time": "7.61s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 3\nHow many relationships should be requested at once: 10\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T15:39:05.202Z"
     },
     {
      "end_time": "2019-03-14T15:39:38.076Z",
      "execution_time": "2.42s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 3\nHow many relationships should be requested at once: 10\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T15:39:35.651Z"
     },
     {
      "end_time": "2019-03-14T16:08:05.490Z",
      "execution_time": "1.81s",
      "outputs": [
       {
        "data": {
         "text/html": "<h3>Search multithreading</h3>",
         "text/plain": "<IPython.core.display.HTML object>"
        },
        "metadata": {},
        "output_type": "display_data",
        "transient": {}
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Decide how fast will the search run\n\nSearch results are usually less and the search requires less to be requested at once.\nRelationships are usually a lot more, therefore it requires more requests at once\nHow many search results should be requested at once: 3\nHow many relationships should be requested at once: 10\n"
       }
      ],
      "source": "from Search import Search\n\ns = Search()\ns.searchAdvancedSetup()",
      "start_time": "2019-03-14T16:08:03.676Z"
     }
    ],
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<h3>Search multithreading</h3>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Decide how fast will the search run\n",
      "\n",
      "Search results are usually less and the search requires less to be requested at once.\n",
      "Relationships are usually a lot more, therefore it requires more requests at once\n",
      "How many search results should be requested at once: 3\n",
      "How many relationships should be requested at once: 10\n"
     ]
    }
   ],
   "source": [
    "from Search import Search\n",
    "\n",
    "s = Search()\n",
    "s.searchAdvancedSetup()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "ExecutionTime": {
     "end_time": "2019-03-14T16:08:48.842554Z",
     "start_time": "2019-03-14T16:08:37.935051Z"
    },
    "provenance": [
     {
      "end_time": "2019-03-14T12:23:57.281Z",
      "execution_time": "26.5s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading 99.26%\nLoading Completed\n557 relationships found\nLoading 60.32%\r"
       },
       {
        "ename": "KeyboardInterrupt",
        "evalue": "",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                        Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     73\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     74\u001b[0m         \u001b[0;32mfor\u001b[0m \u001b[0mthread\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mPS\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mthreadList\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 75\u001b[0;31m             \u001b[0mthread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     76\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     77\u001b[0m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msubstituteRelationships\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mPS\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtotalNumberRelationships\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/threading.py\u001b[0m in \u001b[0;36mjoin\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m   1054\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1055\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mtimeout\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1056\u001b[0;31m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_wait_for_tstate_lock\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1057\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1058\u001b[0m             \u001b[0;31m# the behavior of a negative timeout isn't documented, but\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/threading.py\u001b[0m in \u001b[0;36m_wait_for_tstate_lock\u001b[0;34m(self, block, timeout)\u001b[0m\n\u001b[1;32m   1070\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mlock\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m  \u001b[0;31m# already determined that the C code is done\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1071\u001b[0m             \u001b[0;32massert\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_is_stopped\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1072\u001b[0;31m         \u001b[0;32melif\u001b[0m \u001b[0mlock\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0macquire\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mblock\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1073\u001b[0m             \u001b[0mlock\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelease\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1074\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_stop\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
        ]
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Loading 63.73%\r"
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T12:23:30.801Z"
     },
     {
      "end_time": "2019-03-14T12:24:00.796Z",
      "execution_time": "2.14s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Loading 73.97%\r"
       },
       {
        "ename": "KeyboardInterrupt",
        "evalue": "",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                        Traceback (most recent call last)",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36m_input_request\u001b[0;34m(self, prompt, ident, parent, password)\u001b[0m\n\u001b[1;32m    728\u001b[0m             \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 729\u001b[0;31m                 \u001b[0mident\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mreply\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstdin_socket\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    730\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mException\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/jupyter_client/session.py\u001b[0m in \u001b[0;36mrecv\u001b[0;34m(self, socket, mode, content, copy)\u001b[0m\n\u001b[1;32m    802\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 803\u001b[0;31m             \u001b[0mmsg_list\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msocket\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv_multipart\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmode\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    804\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mzmq\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mZMQError\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/zmq/sugar/socket.py\u001b[0m in \u001b[0;36mrecv_multipart\u001b[0;34m(self, flags, copy, track)\u001b[0m\n\u001b[1;32m    465\u001b[0m         \"\"\"\n\u001b[0;32m--> 466\u001b[0;31m         \u001b[0mparts\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mflags\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtrack\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mtrack\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    467\u001b[0m         \u001b[0;31m# have first part already, only loop while more to receive\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket.Socket.recv\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket.Socket.recv\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket._recv_copy\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/zmq/backend/cython/checkrc.pxd\u001b[0m in \u001b[0;36mzmq.backend.cython.checkrc._check_rc\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: ",
         "\nDuring handling of the above exception, another exception occurred:\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                        Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-3-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     32\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     33\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 34\u001b[0;31m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchTerm\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0minput\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Enter your search: \\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     35\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     36\u001b[0m         \u001b[0mchoice\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36mraw_input\u001b[0;34m(self, prompt)\u001b[0m\n\u001b[1;32m    702\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_parent_ident\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    703\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_parent_header\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 704\u001b[0;31m             \u001b[0mpassword\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mFalse\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    705\u001b[0m         )\n\u001b[1;32m    706\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36m_input_request\u001b[0;34m(self, prompt, ident, parent, password)\u001b[0m\n\u001b[1;32m    732\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    733\u001b[0m                 \u001b[0;31m# re-raise KeyboardInterrupt, to truncate traceback\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 734\u001b[0;31m                 \u001b[0;32mraise\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    735\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    736\u001b[0m                 \u001b[0;32mbreak\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
        ]
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Loading 83.84%\r"
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T12:23:58.652Z"
     },
     {
      "end_time": "2019-03-14T12:24:35.045Z",
      "execution_time": "12.6s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nLoading 96.0%\nLoading Completed\nSubstituting relationships into original search results: \nLoading 88.24%\n\n                                        --SEARCH RESULTS--\n\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T12:24:22.426Z"
     },
     {
      "end_time": "2019-03-14T12:25:02.234Z",
      "execution_time": "8.71s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 66.67%\n25 relationships found\nLoading 96.0%\nLoading Completed\nSubstituting relationships into original search results: \nLoading 88.24%\n\n                                        --SEARCH RESULTS--\n\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T12:24:53.519Z"
     },
     {
      "end_time": "2019-03-14T12:56:37.002Z",
      "execution_time": "6.41s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\n'ParallelSearch' object has no attribute 'requestList'\n'ParallelSearch' object has no attribute 'requestList'\n'ParallelSearch' object has no attribute 'requestList'\n'ParallelSearch' object has no attribute 'requestList'\n'ParallelSearch' object has no attribute 'requestList'\n'ParallelSearch' object has no attribute 'requestList'\n"
       },
       {
        "ename": "AttributeError",
        "evalue": "'ParallelSearch' object has no attribute 'requestList'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                           Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-3-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     63\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     64\u001b[0m         \u001b[0;31m# Get the total number of places where there is a relationship\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 65\u001b[0;31m         \u001b[0mtotalNumberRelationships\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcreateRelationshipList\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     66\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     67\u001b[0m         \u001b[0;31m# Remove duplicates and split into cache-matching and non-cache-matching\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/ParallelSearch.py\u001b[0m in \u001b[0;36mcreateRelationshipList\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    107\u001b[0m         \u001b[0;31m# ommited 'investigation' object!!!\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    108\u001b[0m         \u001b[0mrelations\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 109\u001b[0;31m         \u001b[0;32mfor\u001b[0m \u001b[0mrequest\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    110\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    111\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mhasattr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'relationships'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mAttributeError\u001b[0m: 'ParallelSearch' object has no attribute 'requestList'"
        ]
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T12:56:30.594Z"
     },
     {
      "end_time": "2019-03-14T12:57:21.418Z",
      "execution_time": "6.34s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\n'ParallelSearch' object has no attribute 'requestList'\n'ParallelSearch' object has no attribute 'requestList'\n'ParallelSearch' object has no attribute 'requestList'\n'ParallelSearch' object has no attribute 'requestList'\n'ParallelSearch' object has no attribute 'requestList'\n'ParallelSearch' object has no attribute 'requestList'\n"
       },
       {
        "ename": "AttributeError",
        "evalue": "'ParallelSearch' object has no attribute 'requestList'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m--------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                           Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     63\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     64\u001b[0m         \u001b[0;31m# Get the total number of places where there is a relationship\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 65\u001b[0;31m         \u001b[0mtotalNumberRelationships\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcreateRelationshipList\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     66\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     67\u001b[0m         \u001b[0;31m# Remove duplicates and split into cache-matching and non-cache-matching\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/ParallelSearch.py\u001b[0m in \u001b[0;36mcreateRelationshipList\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    107\u001b[0m         \u001b[0;31m# ommited 'investigation' object!!!\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    108\u001b[0m         \u001b[0mrelations\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 109\u001b[0;31m         \u001b[0;32mfor\u001b[0m \u001b[0mrequest\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    110\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    111\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mhasattr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'relationships'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mAttributeError\u001b[0m: 'ParallelSearch' object has no attribute 'requestList'"
        ]
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T12:57:15.078Z"
     },
     {
      "end_time": "2019-03-14T13:04:28.400Z",
      "execution_time": "10.1s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading 83.33%\nLoading Completed\n25 relationships found\nLoading 96.0%\nLoading Completed\nSubstituting relationships into original search results: \nLoading 88.24%\n\n                                        --SEARCH RESULTS--\n\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T13:04:18.271Z"
     },
     {
      "end_time": "2019-03-14T13:23:17.674Z",
      "execution_time": "4m 32s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \ncell\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n810 results found\nLoading 99.88%\nLoading Completed\n2143 relationships found\nLoading 99.95%\nLoading Completed\nSubstituting relationships into original search results: \nLoading 99.84%\n\n                                        --SEARCH RESULTS--\n\n\nFermentation of S. solfataricus at 70 and 80C in a batch fermenter(id: 13 | type: assays)\n\nDescription: In order to get uniform data from all geographically separated partners in a consortium, central fermentation facility has been set-up. Based on empirical data, standard procedures for growing S. solfataricus cells have been developed.\n\n\nINVESTIGATION\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature\nORGANISMS\nSulfolobus solfataricus\nPEOPLE\nPawel Sierocinski\nPROJECTS\nSulfoSys\nSOPS\nSulfosys - SOP fermentation\nSTUDY\nPilot experiment - S. solfataricus grown at 70 and 80 C.\nSUBMITTER\nPawel Sierocinski\n\n____________________________________________________________________________\n\nAriane Scheuren(id: 558 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nETH Zurich\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nMarek Ostaszewski(id: 665 | type: people)\n\nDescription: Scientific Project Manager at Luxembourg Centre for Systems Biomedicine, University of Luxembourg\nhttp://lcsb.uni.lu\n\n\nINSTITUTIONS\nKinetics on the move  Workshop at HITS\nPROJECTS\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nPatrice D. Cani(id: 785 | type: people)\n\nDescription: Professor Patrice D. Cani is researcher from the Belgian Fund for Scientific Research and group leader in the Metabolism and Nutrition lab at the Louvain Drug Research Institute from the UCL, Brussels, Belgium. He is WELBIO investigator and recipient of an ERC Starting Grant 2013 and a PoC ERC Grant 2016. He is laureate of the Baillet-Latour grant for medical research and the international prize of Physiology Lucien Dautrebande. His main research interests are the investigation of interactions between the gut microbiota, the host and specific biological systems such as the endocannabinoid system and the innate immune system in the context of obesity, type 2 diabetes and metabolic inflammation. In 2007, he published the discovery of the concept of metabolic endotoxemia, and in 2008 the first experimental evidence that gut barrier alteration contribute to metabolic alteration during obesity and diabetes. More recently, he discovered the beneficial role of the bacteria Akkermansia muciniphila on obesity and cardiometabolic risk factors, and isolated active metabolic compounds from this bacteria. He is author and co-author of more than 170 scientific research publications, member of several international associations; he is member of the Royal Belgian Academy of Sciences.\n\n\nINSTITUTIONS\nUniversit catholique de Louvain\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nModelling sustained glycolytic oscillations in individual isolated yeast cells (Gustavsson models)(id: 218 | type: assays)\n\nDescription: missing\n\n\nINVESTIGATION\nYeast Glycolytic Oscillations\nMODELS\nSustained glycolytic oscillations in individual isolated yeast cells - IV\nSustained glycolytic oscillations in individual isolated yeast cells - III\nSustained glycolytic oscillations in individual isolated yeast cells - II\nSustained glycolytic oscillations in individual isolated yeast cells - I\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nPUBLICATIONS\nSustained glycolytic oscillations in individual isolated yeast cells\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction\nSTUDY\nSustained glycolytic oscillations in individual isolated yeast cells\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nData table with cell counts (combined both rounds of analysis)(id: 1435 | type: data_files)\n\nDescription: just cell counts, the area of the grains is separate.\n\n\nCREATORS\nAntoine Buetti-Dinh\nPEOPLE\nAntoine Buetti-Dinh\nSUBMITTER\nAntoine Buetti-Dinh\n\n____________________________________________________________________________\n\nlist of publications 2006 - 2016(id: 1461 | type: data_files)\n\nDescription: List of publications from 2006 till 2016. \nNote: I retired from the University of Amsterdam in 2010\n\n\nCREATORS\nRoel Van Driel\nPEOPLE\nRoel Van Driel\nPROJECTS\nHUMET Startup\nSUBMITTER\nRoel Van Driel\n\n____________________________________________________________________________\n\nProteome of Clostridium acetobutylicum growing at steady-state pH 5.5 to pH 4.7(id: 836 | type: data_files)\n\nDescription: Representation of all prepared 2D PAGE analyses of cytosolic proteins of Clostridium acetobutylicum growing continuously at pH 5.5, 5.3, 5.1, 4.9 and 4.7.\n\n\n\nASSAYS\nComparison of the proteome between pH 5.7 (acidogenesis) and pH 4.5 (solventogenesis).\nCREATORS\nHolger Janssen\nINVESTIGATIONS\nThe effect of pH upon the metabolic shift in Clostridium acetobutylicum in continuous culture.\nPEOPLE\nHolger Janssen\nPROJECTS\nCOSMIC\nPUBLICATIONS\nIntegrative modelling of pH-dependent enzyme activity and transcriptomic regulation of the acetone-butanol-ethanol fermentation of\n              Clostridium acetobutylicum\n              in continuous culture\nSTUDIES\nEffect of pH on the proteome\nSUBMITTER\nHolger Janssen\n\n____________________________________________________________________________\n\nLactobacillus plantarum JDM1 protein list(id: 1064 | type: data_files)\n\nDescription: Lactobacillus plantarum JDM1 differentially expressed proteins, comparison between low and high growth rates\n\n\nCREATORS\nAnette McLeod\nPEOPLE\nAnette McLeod\nPROJECTS\nSysMO-LAB\nSUBMITTER\nAnette McLeod\n\n____________________________________________________________________________\n\nProteomics Template (empty) from PRIDE(id: 10 | type: data_files)\n\nDescription: An example of a blank data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of MIAPE compliant data.\n\nThe shreadsheet contains macros to help you annotate your data with terms from controlled vocabularies and ontologies\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nZucAt: FBA solution of the model under light growth conditions (II)(id: 1335 | type: data_files)\n\nDescription: The solution of Flux Balance Analysis (FBA) represents metabolic flux distribution in ZucAt model under light growth conditions. In this solution, (i) the ratio photorespiration / photosynthesis has been fixed to 0.25; (ii) and ATP transport between plastid and cytoplasm has been set to 0. The last constraint allows finding the ratio between fluxes through FQR (ferredoxin-plastoquinone reductase) and FRN (ferredoxin-NADP oxidoreductase) under which the ATP balance in plastid becomes self-sufficient (due to cyclic electron flux through FQR), correspondingly plastid neither imports nor exports ATP to/from cytoplasm. Flux through FQR represents cyclic electron flow through the light photosynthesis system. Flux through FRN represents non-cyclic electron flow through the light photosynthesis system. \n\n\nASSAYS\nFlux Balance Analysis of multi-compartment metabolic model of growing Arabidopsis thaliana\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nZucAt\nSTUDIES\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nEntire transcriptional dataset - anaerobic to aerobic .csv(id: 47 | type: data_files)\n\nDescription: This .csv file contains the entire transcriptional dataset of the anaerobic to aerobic transition. The values shown are the gene-expression ratio at 2, 5, 10, 15 and 20 minutes relative to the 0 minute timepoint.\n\n\nASSAYS\nTranscriptional profiling of E. coli during anaerobic to aerobic and aerobic to anaerobic transitions\nCREATORS\nMatthew Rolfe\nINVESTIGATIONS\nDynamical studies for different oxygen availability in Escherichia coli\nPEOPLE\nMatthew Rolfe\nPROJECTS\nSUMO\nSTUDIES\nThe Escherichia coli dynamical response to oxygen: from molecular interactions to regulatory modules\n\n____________________________________________________________________________\n\nETCmutants_mRNA(id: 1130 | type: data_files)\n\nDescription: Gene expression levels in mutants with linear ETC were determined by RealTime RT-PCR\n\n\nASSAYS\nGene expression analysis of mutants with linear electron transport chain at different aerobiosis levels\nCREATORS\nKatja Bettenbrock\nINVESTIGATIONS\nAnalysis of Escherichia coli with linear electron transport chain\nPEOPLE\nKatja Bettenbrock\nPROJECTS\nSUMO\nSTUDIES\nAnalysis of Escherichia coli strains with linear respiratory chain\nSUBMITTER\nKatja Bettenbrock\n\n____________________________________________________________________________\n\nLive Cell Imaging of <em>Bacillus subtilis</em> and <em>Streptococcus pneumoniae</em> using Automated Time-lapse Microscopy(id: 128 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nLive Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy.\nDATA_FILES\nRNA-seq wild type\nPEOPLE\nOscar Kuipers\nImke De Jong\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nIdentification of Cell Type-Specific Differences in Erythropoietin Receptor Signaling in Primary Erythroid and Lung Cancer Cells(id: 301 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nWP3: Cell Fate Decisions of Erythroid Progenitor Cells\nPEOPLE\nMaja Rey\nBernhard Steiert\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nArabidopsis_clock_2011 - PLM_43, version 5(id: 827 | type: assays)\n\nDescription: \tAlexandra Pokhilko's model of the Arabidopsis clock, private drafts created in preparation for publication (Mol. Syst. Biol.), or as working versions with various modifications after publication.\t\tThe published model version is also in PlaSMo as <a href=\"http://www.plasmo.ed.ac.uk/plasmo/models/model.shtml?accession=PLM_64\">PLM_64 here</a>.<br><br><strong>Comments</strong></strong><p>\n\tMatlab files are attached to version 1</p>\n 2012-01-31 11:08:51 3 amillar2 andrew.millar@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\n\tthis version has a graphical representation in Cell Designer. It runs in Cell Designer, Copasi and SBSI, but not run optimization in SBSI. The model is submitted to Biomodels database (accession number MODEL1109200000)</p>\n<br><br><strong>Originally submitted to PLaSMo on 2011-07-16 12:31:04</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nINVESTIGATION\nPokhilko, Alexandra\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nArabidopsis_clock_2011 - PLM_43\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nThe role of dynamic stimulation pattern in the analysis of bistable intracellular networks(id: 60 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlaf Wolkenhauer\nUwe Voelker\nThomas Millat\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nEnzyme activity tests for S. solfataricus(id: 14 | type: assays)\n\nDescription: Based on initial cell material, series of SOPs connected with assays of enzymes involved in Central Carbon Metabolism of S. solfataricus have been developed.\n\n\nINVESTIGATION\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature\nORGANISMS\nSulfolobus solfataricus\nPEOPLE\nPawel Sierocinski\nPROJECTS\nSulfoSys\nPUBLICATIONS\n\"Hot standards\" for the thermoacidophilic archaeon Sulfolobus solfataricus\nSTUDY\nPilot experiment - S. solfataricus grown at 70 and 80 C.\nSUBMITTER\nPawel Sierocinski\n\n____________________________________________________________________________\n\nBas Teusink(id: 45 | type: people)\n\nDescription: Since August 2008 I am professor in Systems Biology at the VU University Amsterdam. My Systems Bioinformatics group  focusses on systems biology with a special focus on integrative bioinformatics. It aims at forming bridges between the classical bottom-up approaches in systems biology and the more data-driven approaches in classical bioinformatics. We combine experimental, modeling and theoretical approaches to study cellular physiology, with an emphasis on metabolic networks. \n\n\nDATA_FILES\npresentation in Oxford 2011\nINSTITUTIONS\nVU University Amsterdam\nWageningen University & Research\nMODELS\nGenome-scale metabolic model of Enterococcus faecalis V583\nPROJECTS\nSysMO-LAB\nSysMilk\nIMOMESIC\nPUBLICATIONS\nRole of phosphate in the central metabolism of two lactic acid bacteria - a comparative systems biology approach\nUsing a Genome-Scale Metabolic Model of Enterococcus faecalis V583 To Assess Amino Acid Uptake and Its Impact on Central Metabolism\n\n____________________________________________________________________________\n\nGlucose pulsed S pyogenes(id: 69 | type: data_files)\n\nDescription: Glucose pulsed S pyogenes with 5 mM glucose in 100 mM MES buffer at pH 6.5 and followed intracellular metabolites in time\n\n\nASSAYS\nGlucose pulsed S. pyogenes\nCREATORS\nMartijn Bekker\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSTUDIES\nComparative modeling and phosphate dependence flux distributions and glucose consumption rates of two lactic acid bacteria S. pyogenes and L. lactis\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nTest created 1552504664537(id: 962 | type: assays)\n\nDescription: <br><br><strong>Related Publications</strong></strong><br>Me and Helena (2014). some book. <br>Moore A, Zielinski T, Millar AJ (2014). Online period estimation and determination of rhythmicity in circadian data, using the BioDare data infrastructure.. Methods in molecular biology (Clifton, N.J.). Retrieved from: doi.org/10.1007/978-1-4939-0700-7_2<br>Stein JM (1975). The effect of adrenaline and of alpha- and beta-adrenergic blocking agents on ATP concentration and on incorporation of 32Pi into ATP in rat fat cells.. Biochemical pharmacology. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/12<br><br><strong>Comments</strong></strong><p>\n\tNew comment after 2nd version</p>\n 2019-01-10 11:59:51 201 zielu tomasz.zielinski@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\n\tCommentsdfa</p>\n<br><br><strong>Originally submitted to PLaSMo on 2014-07-25 18:01:50</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nTomasz Zielinski\nINVESTIGATION\nZielinski, Tomasz\nPEOPLE\nTomasz Zielinski\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\n091016 quinones complete data magdeburg(id: 64 | type: data_files)\n\nDescription: This is the csv version of the file: 091016 quinones complete data magdeburg\n\nThis document contains data for quinone concentrations and ratios of ubichinon and menachinon.These data have been generated for analysis of MG1655 at steady-state conditions in Infors-Multifors-Bioreactors. All measurements have been done in Amsterdam at the group of Prof. Joost Teixeira de Mattos.\n\nDuplicates of samples for quinones were taken after 20 hours of steady state conditions. Chemostat experiments were carried out in Infors-Bioreactors at 0, 20, 50, 80, 100 and 120% aerobiosis. All data for concentrations are shown in [mol / g DCW].\n\n\nASSAYS\nDetermination of intracellular redox state by means of ubiquinones (oxd/red-ratio)\nSteady State Oxygen Response of E. coli WT and two Electron Transport Chain Mutants\nCREATORS\nStefan Stagge\nAlexander Ter Beek\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nPEOPLE\nStefan Stagge\nAlexander Ter Beek\nPROJECTS\nSUMO\nSTUDIES\nThe Escherichia coli steady state response to oxygen: from molecular interactions to regulatory modules\n\n____________________________________________________________________________\n\nYeast Glycolytic Oscillations(id: 50 | type: investigations)\n\nDescription: Investigating oscillations at the level of yeast populations and individual cells\n\n\nASSAYS\nModelling sustained glycolytic oscillations in individual isolated yeast cells (Gustavsson models)\nDetailed kinetic model of yeast glycolytic oscillation\nMetabolite concentrations in yeast glycolytic oscillations\nDATA_FILES\nSustained glycolytic oscillations in yeast\nRichard (1996) Concentrations of oscillating glycolytic intermediates in yeast cultures\nDas (1991) NADH and ATP concentrations determined during different phases of glycolytic oscillation in yeast extracts\nDas (1991) Time varying concentrations for metabolites and cofactors in upper glycolysis measured in oscillating yeast extracts\nMODELS\nDetailed kinetic model of yeast glycolytic oscillation - I\nDetailed kinetic model of yeast glycolytic oscillation - II\nDetailed kinetic model of yeast glycolytic oscillation - III\nDetailed kinetic model of yeast glycolytic oscillation - IV\nDetailed kinetic model of yeast glycolytic oscillation - V\nDetailed kinetic model of yeast glycolytic oscillation - VI\nDetailed kinetic model of yeast glycolytic oscillation - VII\nSustained glycolytic oscillations in individual isolated yeast cells - IV\nSustained glycolytic oscillations in individual isolated yeast cells - III\nSustained glycolytic oscillations in individual isolated yeast cells - II\nSustained glycolytic oscillations in individual isolated yeast cells - I\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nPUBLICATIONS\nSustained glycolytic oscillations in individual isolated yeast cells\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction\nSTUDIES\nSustained glycolytic oscillations in individual isolated yeast cells\nDetailed kinetics of yeast glycolytic oscillation\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nPGI Kinetic characterisation(id: 193 | type: sops)\n\nDescription: SOP for measurement of PGI activity in extracts.\n\n\nASSAYS\nPGI\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nSustained glycolytic oscillations in individual isolated yeast cells - IV(id: 81 | type: models)\n\nDescription: Using optical tweezers to position yeast cells in a microfluidic chamber, we were able to observe sustained oscillations in individual isolated cells. Using a detailed kinetic model for the cellular reactions, we simulated the heterogeneity in the response of the individual cells, assuming small differences in a single internal parameter. By operating at two different flow rates per experiment, we observe four of categories of cell behaviour. The present model (gustavsson4) predicts the steady-state behaviour observed in 25.7% of the cell population. \n\n\nASSAYS\nModelling sustained glycolytic oscillations in individual isolated yeast cells (Gustavsson models)\nCREATORS\nFranco Du Preez\nJacky Snoep\nDawie Van Niekerk\nINVESTIGATIONS\nYeast Glycolytic Oscillations\nPEOPLE\nJacky Snoep\nFranco Du Preez\nDawie Van Niekerk\nPROJECTS\nSysMO DB\nPUBLICATIONS\nSustained glycolytic oscillations in individual isolated yeast cells\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction\nSTUDIES\nSustained glycolytic oscillations in individual isolated yeast cells\nSUBMITTER\nFranco Du Preez\n\n____________________________________________________________________________\n\nSalazar2009_FloweringPhotoperiod - PLM_9, version 1(id: 886 | type: assays)\n\nDescription: \tThe model shows how the <em>CONSTANS</em> gene and protein in <em>Arabidopsis thaliana</em> forms a day-length sensor. It corresponds to Model 3 in the publication of Salazar et al. 2009.\tMatlab versions of all the models in the paper are attached to this record as a ZIP archive, as are all the data waveforms curated from the literature to constrain the model. Further information may be available via links from the authors web site (<a href=\"http://www.amillar.org\">www.amillar.org</a>).\tSimulation notes for SBML version of Model3 from Salazar et al., Cell, 2009.\tThe interlocking-loop circadian clock in this model reaches its entrained limit cycle rather slowly. To ensure that simulations show the stable phase under any light:dark cycle, the model should be run for 720 hours (30 days) before evaluating its behaviour in light:dark cycles. Using the boundary value solver in Matlab, as described in the paper, avoids this issue.\tSBML prepared by Dr. Treenut Saithong, based on Matlab by Dr. Domingo Salazar. Model depositor, updates to version 2, Prof. Andrew Millar.\tNote comment from Rob Smith on Figure 5.<br><br><strong>Related Publications</strong></strong><br>Salazar JD, Saithong T, Brown PE, Foreman J, Locke JC, Halliday KJ, Carr IA, Rand DA, Millar AJ. (2009). Prediction of photoperiodic regulators from quantitative gene circuit models. Cell. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/20005809<br><br><strong>Comments</strong></strong><p>\n\tThe <em>FT</em> expression in Figure 5 in Salazar et al. (2009) was simulated using Model 3b. Model 3b is a slight modification of Model 3. Instead of having <em>nTOC1</em> (nuclear TOC1 protein) replacing <em>CO</em> mRNA directly (as in equation 4 in the Supplementary), Model 3b contains an additional ODE for <em>CO</em> mRNA as follows:</p>\n<p>\n\t&nbsp;d<em>COm</em>/dt = d<em>nTOC1</em>/dt - 0.0004<em>L</em>.<em>vnTOC1</em>.<em>nTOC1</em></p>\n<p>\n\t&nbsp;<em>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</em></p>\n<p>\n\twhere <em>COm</em> = <em>CO</em> mRNA concentration, <em>L</em> = Light and <em>vnTOC1</em> = nuclear TOC1 protein degradation rate. The additional term represents <em>CO </em>mRNA degradation rate, which was slightly increased (by 0.04% of its original value) during the light interval. This was done to improve fit for training data set 8.</p>\n<p>\n\t&nbsp;</p>\n<p>\n\tThus, equation 4 in the Supplementary becomes</p>\n<p>\n\t&nbsp;d<em>COp</em>/dt = <em>vCOm</em>.<em>COm</em> - (1 - <em>L</em>).<em>vCOp</em>.<em>COp</em>/(<em>kCOp</em> + <em>COp</em>)</p>\n<p>\n\t&nbsp;</p>\n<p>\n\tAll parameter values remain the same.</p>\n 2011-08-19 12:35:48 121 s0921165 s0921165@sms.ed.ac.uk<br><p>\n\t&nbsp;The <em>FT</em> expression in Figure 5 in Salazar et al. (2009) was simulated using Model 3b. Model 3b is a slight modification of Model 3. Instead of having <em>nTOC1</em> (nuclear TOC1 protein) replacing <em>CO</em> mRNA directly (as in equation 4 in the Supplementary), Model 3b contains an additional ODE for <em>CO</em> mRNA as follows:</p>\n<p>\n\t&nbsp;</p>\n<p>\n\t</p>\n<p>\n\t&nbsp;</p>\n<p>\n\t<em>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</em></p>\n<p>\n\twhere <em>COm</em> = <em>CO</em> mRNA concentration, <em>L</em> = Light and <em>vnTOC1</em> = nuclear TOC1 protein degradation rate. The additional term represents <em>CO </em>mRNA degradation rate, which was slightly increased (by 0.04% of its original value) during the light interval. This was done to improve fit for training data set 8.</p>\n<p>\n\t&nbsp;</p>\n<p>\n\tThus, equation 4 in the Supplementary becomes</p>\n<p>\n\t&nbsp;</p>\n<p>\n\t</p>\n<p>\n\t&nbsp;</p>\n<p>\n\tAll parameter values remain the same.</p>\n 2011-08-19 12:23:46 121 s0921165 s0921165@sms.ed.ac.uk<br><br><strong>Originally submitted to PLaSMo on 2010-05-05 15:16:35</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nAndrew Millar\nDATA_FILES\nZIP archive of Matlab model files for the models described in the paper, PLM_9_ 1\nFigure 1 of the paper, with a cartoon of the model. Model3 is provided as SBML. This is the sum of model 1(b) and model 2(b) shown in the cartoon., PLM_9_ 1\nINVESTIGATION\nMillar, Andrew\nMODELS\nSalazar2009_FloweringPhotoperiod - PLM_9, version 1, SUBMITTED\nSalazar2009_FloweringPhotoperiod - PLM_9, version 1, SIMPLIFIED\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nSalazar2009_FloweringPhotoperiod - PLM_9\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nMetabolic efficiency in yeast Saccharomyces cerevisiae in relation to temperature dependent growth and biomass yield(id: 243 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMaksim Zakhartsev\nMatthias Reuss\nPROJECTS\nMOSES\n\n____________________________________________________________________________\n\nTroein Ostreococcus clock 1-loop - PLM_7, version 2, SIMPLIFIED(id: 542 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_7, version 2 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nTroein Ostreococcus clock 1-loop - PLM_7, version 2\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nTroein Ostreococcus clock 1-loop - PLM_7\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nG3PDH Kinetic characterisation(id: 207 | type: sops)\n\nDescription: SOP for measurement of G3PDH activity in extracts.\n\n\nASSAYS\nG3PDH\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nEin Beitrag zu einer Theorie lebender Zellen (A Contribution towards a Theory of Living Cells)(id: 92 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlaf Wolkenhauer\nUlf Liebal\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nHands-On: Tyson1991 - Cell Cycle 6 var(id: 156 | type: studies)\n\nDescription: missing\n\n\nASSAYS\nBIOMD0000000005 - Tyson1991 - Cell Cycle 6 var\nCREATORS\nRon Henkel\nDagmar Waltemath\nDATA_FILES\nSEDML for BIOMD0000000005\nINVESTIGATION\nHands-on: Model Management in SEEK\nMODELS\nBIOMD0000000005 from BioModels Database\nPEOPLE\nRon Henkel\nDagmar Waltemath\nPROJECTS\nKinetics on the move - Workshop 2016\nPUBLICATIONS\nModeling the cell division cycle: cdc2 and cyclin interactions\nSUBMITTER\nRon Henkel\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nRNA isolation of Clostridium acetobutylicum(id: 105 | type: sops)\n\nDescription: missing\n\n\nCREATORS\nHolger Janssen\nPEOPLE\nHolger Janssen\nPROJECTS\nCOSMIC\n\n____________________________________________________________________________\n\nMaike  Bartholomae(id: 291 | type: people)\n\nDescription: missing\n\n\nASSAYS\nBinding constants for CcpA with HPrSer46P interacting with various cre-elements\nDATA_FILES\nAssociation and dissociation kinetics of the CcpA-HPrSer46P-complex binding to various cre elements\nINSTITUTIONS\nUniversity of Erlangen-Nuernberg\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nStructures of carbon catabolite protein A-(HPr-Ser46-P) bound to diverse catabolite response element sites reveal the basis for high-affinity binding to degenerate DNA operators\nCcpA forms complexes with CodY and RpoA in Bacillus subtilis\n\n____________________________________________________________________________\n\nMeasured steady state concentrations of extracellular metabolites in yeast Saccharomyces cerevisiae anaerobic chemostat(id: 77 | type: data_files)\n\nDescription: Steady state concentrations of extracellular metabolites in yeast Saccharomyces cerevisiae anaerobic chemostat at D = 0.1 h-1 on minimal medium. All metabolite concentrations are in mmol/L(R) except CO2, which is in parts of the partial pressure. \n\n\nASSAYS\nSteady state concentrations of extracellular metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nMOSES: dynamic model of glucose pulse\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nSteady state metabolic fluxes and metabolite concentrations in anaerobic chemostat of yeast Saccharomyces cerevisiae\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nPUBLICATIONS\nFast sampling for quantitative microbial metabolomics: new aspects on cold methanol quenching: metabolite co-precipitation\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSteady state concentrations of metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nPotassium starvation protocol(id: 107 | type: sops)\n\nDescription: Purpose: \n\n\n\nCREATORS\nJose Ramos\nPEOPLE\nJose Ramos\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nAbsolute copy number per locus tag, M. pneumoniae at different times of growth(id: 1732 | type: data_files)\n\nDescription: Contains the absolute copy number per locus tag during growth between 0.25 and 96hours of growth\nGrowth in batch, cells attached to the bottom of the flask, non-aerated, non-stirred\n\n\nASSAYS\nTranscriptomics assay of M. pneumoniae at diferent times of growth\nCREATORS\nLuis Serrano\nMaria Lluch\nEva Yus\nPEOPLE\nNiels Zondervan\nLuis Serrano\nMaria Lluch\nEva Yus\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nPUBLICATIONS\nLarge-scale metabolome analysis and quantitative integration with genomics and proteomics data in Mycoplasma pneumoniae.\nSTUDIES\nTranscriptomics of M. pneumoniae at different times of growth\nSUBMITTER\nNiels Zondervan\n\n____________________________________________________________________________\n\nHigh- and low-affinity cre boxes for CcpA binding in Bacillus subtilis revealed by genome-wide analysis(id: 1068 | type: data_files)\n\nDescription: In Bacillus subtilis and its relatives carbon catabolite control, a mechanism enabling to reach maximal efficiency of carbon and energy sources metabolism, is achieved by the global regulator CcpA (carbon catabolite protein A). CcpA in a complex with HPr-Ser-P (seryl-phosphorylated form of histidine-containing protein, HPr) binds to operator sites called catabolite responsive elements, cre. Depending on the cre box position relative to the promoter, the CcpA/HPr-Ser-P complex can either act as a positive or a negative regulator. The cre boxes are highly degenerate semi-palindromes with a lowly conserved consensus sequence. So far, studies aimed at revealing how CcpA can bind such diverse sites were focused on the analysis of single cre boxes. In this study, a genome-wide analysis of cre sites was performed in order to identify differences in cre sequence and position, which determine their binding affinity.\nThe transcriptomes of B. subtilis cultures with three different CcpA expression levels were compared. The higher the amount of CcpA in the cells, the more operons possessing cre sites were differentially regulated. The cre boxes that mediated regulation at low CcpA levels were designated as strong (high affinity) and those which responded only to high amounts of CcpA, as weak (low affinity). Differences in the sequence and position in relation to the transcription start site between strong and weak cre boxes were revealed.\nCertain residues at specific positions in the cre box as well as, to a certain extent, a more palindromic nature of cre sequences and the location of cre in close vicinity to the transcription start site contribute to the strength of CcpA-dependent regulation. The main factors contributing to cre regulatory efficiencies, enabling subtle differential control of various subregulons of the CcpA regulon, are identified.\n\n\nCREATORS\nOscar Kuipers\nPEOPLE\nOscar Kuipers\nLeif Steil\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nHigh- and low-affinity cre boxes for CcpA binding in Bacillus subtilis revealed by genome-wide analysis\nSUBMITTER\nLeif Steil\n\n____________________________________________________________________________\n\nStandard operating procedure to analyze the intracellular proteome of Clostridium acetobutylicum.(id: 147 | type: sops)\n\nDescription: missing\n\n\nASSAYS\nComparison of the proteome between pH 5.7 (acidogenesis) and pH 4.5 (solventogenesis).\nCREATORS\nHolger Janssen\nTomas Fiedler\nINVESTIGATIONS\nThe effect of pH upon the metabolic shift in Clostridium acetobutylicum in continuous culture.\nPEOPLE\nTomas Fiedler\nHolger Janssen\nPROJECTS\nCOSMIC\nSTUDIES\nEffect of pH on the proteome\nSUBMITTER\nHolger Janssen\n\n____________________________________________________________________________\n\nGenome-wide gene expression analysis of the switch between acidogenesis and solventogenesis in continuous cultures of Clostridium acetobutylicum(id: 142 | type: publications)\n\nDescription: missing\n\n\nMODELS\nModel of the pH-induced metabolic shift in phosphate-limited continuous cultures of C. acetobutylicum assuming a phenotypic switch\nPEOPLE\nArmin Ehrenreich\nHubert Bahl\nPeter Drre\nRalf-Jrg Fischer\nThomas Millat\nDsire Krausse\nHolger Janssen\nPRESENTATIONS\nThe pH-induced metabolic shift from acidogenesis to solventogenesis in Clostridium acetobutylicum - From experiments to models\nPROJECTS\nCOSMIC\n\n____________________________________________________________________________\n\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling(id: 321 | type: models)\n\nDescription: \n\nThe model is adapted from A.P. Kupinski, I. Raabe, M. Michel, D. Ail, L. Brusch, T. Weidemann, C. Bkel (2013) Phosphorylation of the Smo tail is controlled by membrane localization and is dispensable for clustering, J. Cell Sci., 126, 20, 4684-4697 doi: 10.1242/jcs.128926\n\nThe model format is MorpheusML that can readily be loaded and run in Morpheus: https://imc.zih.tu-dresden.de//wiki/morpheus\n\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nSebastian Aljoscha Wahl(id: 581 | type: people)\n\nDescription: Assistant professor working on industrial and environmental microorganisms\n\n\nINSTITUTIONS\nDelft University of Technology (DUT)\nPROJECTS\nXyloCut\n\n____________________________________________________________________________\n\nUpdate on proteome data in chemostat and shake flask experiments(id: 65 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nPraveen Kumar Sappa\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nGeneration of uniformly 13C-labelled E. coli extract for absolute quantification(id: 155 | type: sops)\n\nDescription: Labelling and extraction procedure for uniformly 13C-labelled E. coli (MG1655) for absolute quantification using isotope dilution technique by LC-MS\n\n\nASSAYS\nGeneration of uniformly 13C-labelled E. coli extract\nCREATORS\nDong-Hyun Kim\nINVESTIGATIONS\nMetabolite profiling, quantification and flux quantification\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSTUDIES\nMetabolic flux measurement\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nContious growth_1.2M NaCl_SMM_Transcritome_Nimblegen(id: 94 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nTiling array analysis of continuous growth stress conditions in SMM\nCREATORS\nPraveen Kumar Sappa\nINVESTIGATIONS\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSTUDIES\nTranscriptome of continuously stressed B. subtilis\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nMaaruthy Yelleswarapu(id: 551 | type: people)\n\nDescription: I am a first year graduate student in the lab of Prof. Wilhelm Huck at the Radboud University Nijmegen. I am working on a project to create an artificial cell. This involves implementing complex genetic networks in cell-free systems which are far from equilibrium. My project involves designing, quantifying, modeling such networks. I am a molecular biologist by training and have basic modeling skills. I am looking to expand my skill-set to include microscopy, microfluidics and a little bit of chemistry.\n\n\nINSTITUTIONS\nRadboud University Nijmegen\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nSampling of biomass(id: 95 | type: sops)\n\nDescription: Biomass sampling - SOP for sampling of biomass\n\n\nASSAYS\nCellular size and granularity during glucose pulse\nSteady state extracellular fluxes in anaerobic yeast Saccharomyces cerevisiae\nSteady state concentrations of extracellular metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nSteady state concentrations of intracellular metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nBiomass weight during glucose pulse\nDynamics of extracellular metabolites during glucose pulse\nDynamics of intracellular metabolites during glucose pulse\nDynamics of macromolecules during glucose pulse\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nSteady state metabolic fluxes and metabolite concentrations in anaerobic chemostat of yeast Saccharomyces cerevisiae\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSteady state fluxes in yeast Saccharomyces cerevisiae in anaerobic chemostat\nSteady state concentrations of metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nProgress meeting Berlin April 5th 2012(id: 33 | type: presentations)\n\nDescription: Presentation on project progress - meeting Berlin April 5th 2012\n\n\nCREATORS\nJohn Raedts\nPEOPLE\nJohn Raedts\nPROJECTS\nCOSMIC\nSUBMITTER\nJohn Raedts\n\n____________________________________________________________________________\n\nAuromega(id: 134 | type: projects)\n\nDescription: A joint NTNU and SINTEF project to establish a knowledge platform on DHA synthesis and lipid accumulation in the native DHA-producing thraustochytrids, and to develop these into high productivity omega-3 fatty acid producing cell factories.\n\n\n\nINSTITUTIONS\nNorwegian University of Science and Technology\nSINTEF\nORGANISMS\nAurantiochytrium sp. T66\nPEOPLE\nHelga Ertesvg\nEivind Almaas\nTonje Heggeset\nPer Bruheim\nMarit H Stafsnes\nZdenka Bartosova\nSimone Balzer Le\nAndre Voigt\nVetle Simensen\nPROGRAMMES\nAuromega\nPUBLICATIONS\nDraft genome sequence of the docosahexaenoic acid producing thraustochytrid Aurantiochytrium sp. T66.\n\n____________________________________________________________________________\n\nAllosteric regulation of phosphofructokinase controls the emergence of glycolytic oscillations in isolated yeast cells(id: 376 | type: studies)\n\nDescription: missing\n\n\nASSAYS\ngustavsson5 model\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATION\nGlycolytic oscillations in individual isolated yeast cells\nMODELS\ngustavsson5 model\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nPUBLICATIONS\nAllosteric regulation of phosphofructokinase controls the emergence of glycolytic oscillations in isolated yeast cells\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nAutomated time-lapse microscopy(id: 81 | type: studies)\n\nDescription: During the last few years scientists became increasingly aware that average data obtained from microbial population based experiments are not representative of the behavior, status or phenotype of single cells. Due to this new insight the number of single cell studies rises continuously (for recent reviews see (1,2,3)). However, many of the single cell techniques applied do not allow monitoring the development and behavior of one specific single cell in time (e.g. flow cytometry or standard microscopy). Here, we provide a detailed description of a microscopy method used in several recent studies (4, 5, 6, 7), which allows following and recording (fluorescence of) individual bacterial cells of Bacillus subtilis and Streptococcus pneumoniae through growth and division for many generations. The resulting movies can be used to construct phylogenetic lineage trees by tracing back the history of a single cell within a population that originated from one common ancestor. This time-lapse fluorescence microscopy method cannot only be used to investigate growth, division and differentiation of individual cells, but also to analyze the effect of cell history and ancestry on specific cellular behavior. Furthermore, time-lapse microscopy is ideally suited to examine gene expression dynamics and protein localization during the bacterial cell cycle. The method explains how to prepare the bacterial cells and construct the microscope slide to enable the outgrowth of single cells into a microcolony. In short, single cells are spotted on a semi-solid surface consisting of growth medium supplemented with agarose on which they grow and divide under a fluorescence microscope within a temperature controlled environmental chamber. Images are captured at specific intervals and are later analyzed using the open source software ImageJ.\n\n\nASSAYS\nLive Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy.\nINVESTIGATION\nWetlab approach to transcription fidelity\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\nPUBLICATIONS\nSMC is recruited to oriC by ParB and promotes chromosome segregation in Streptococcus pneumoniae\nLive Cell Imaging of <em>Bacillus subtilis</em> and <em>Streptococcus pneumoniae</em> using Automated Time-lapse Microscopy\nControl of cell division in Streptococcus pneumoniae by the conserved Ser/Thr protein kinase StkP\nSingle cell analysis of gene expression patterns during carbon starvation in Bacillus subtilis reveals large phenotypic variation\nSUBMITTER\nJan-Willem Veening\n\n____________________________________________________________________________\n\nAlison Graham(id: 198 | type: people)\n\nDescription: Post-doctoral research associate working in Sheffield in the SUMO consortium.\n\n\nASSAYS\nMeasurement of cytochrome numbers\nINSTITUTIONS\nUniversity of Sheffield\nPROJECTS\nSUMO\nPUBLICATIONS\nTranscript profiling and inference of Escherichia coli K-12 ArcA activity across the range of physiologically relevant oxygen concentrations\nModelling complex biological systems using an agent-based approach\nSOPS\nSimons method for making monolysogenic lacZ operon fusions on the E. coli chromosome\nQuantifying cytochromes\n\n____________________________________________________________________________\n\nUniversity of Newcastle(id: 44 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nColin Harwood\nJoseph Newman\nHenrik Strahl Von Schulten\nLeendert Hamoen\nRick Lewis\nNikolay Zenkin\nYulia Yuzenkova\nCiaran Welsh\nDaryl Shanley\nKrutik Patel\nHannah O'Keefe\nAlvaro Martinez Guimera\nJames Wordsworth\nPROJECTS\nBaCell-SysMO\nNoisy-Strep\nICYSB 2015 - International Practical Course in Systems Biology\nMESI-STRAT\nSystems modelling age-related changes in the maintenance of dermal extra-cellular matrix: mechanisms and interventions\nAre microRNAs key mediators of cartilage destruction in osteoarthritis?\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nRaju Prasad Sharma(id: 905 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nVU University Amsterdam\nUniversitat Rovira i Virgili\nPROJECTS\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\n\n____________________________________________________________________________\n\nRegulation of Trk-dependent potassium transport by the calcineurin pathway involves the Hal5 kinase(id: 24 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoaquin Ario\nJose Ramos\nFalko Krause\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nSEEK: Systems Biology Data and Models Sharing(id: 35 | type: presentations)\n\nDescription: An overview of the SEEK for the HCLS Systems Biology Task force meeting\n\n\nCREATORS\nKaty Wolstencroft\nStuart Owen\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nRole of phosphate in the central metabolism of two lactic acid bacteria - a comparative systems biology approach(id: 136 | type: publications)\n\nDescription: missing\n\n\nMODELS\nPublished kinetic model of glycolysis in L. lactis\nPublished kinetic model of glycolysis in S. pyogenes\nPEOPLE\nWillem De Vos\nBas Teusink\nBernd Kreikemeyer\nJeroen Hugenholtz\nMark Musters\nMartijn Bekker\nTomas Fiedler\nUrsula Kummer\nJennifer Levering\nDomenico Bellomo\nPROJECTS\nSysMO-LAB\n\n____________________________________________________________________________\n\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.(id: 365 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nAgent-based model of response of E.coli to different levels of oxygen supply (with FNR and ArcBA)(id: 105 | type: models)\n\nDescription: The model presents the response of E.coli to different levels of oxygen supply, in which the oxidases, Cyo and Cyd, and their regulators, FNR and ArcBA systems, are included. The initial file 0.xml and supporting documents are for the model with FNR only. Four 0.xml files provided are at AAU level 31, 85, 115 and 217 respectively. The ArcBA system can be activated by revising the number of agents, ArcB, ArcA dimer, ArcA monomer, ArcA tetramer and ArcA octamer, in the initial file. The model needs to be compiled and run in the FLAME environment. The related software of FLAME can be obtained from http://www.flame.ac.uk/download/.\n\n\nCREATORS\nHao Bai\nPEOPLE\nHao Bai\nPROJECTS\nSUMO\nSUBMITTER\nHao Bai\n\n____________________________________________________________________________\n\nNew surveyor tools for charting microbial metabolic maps(id: 11 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nRainer Breitling\nJay Moore\nPROJECTS\nSTREAM\n\n____________________________________________________________________________\n\nQuantifying cytochromes(id: 48 | type: sops)\n\nDescription: This assay uses a dual-wavelength spectrophotometer to quantify cytochromes present in the E. coli respiratory chain.\n\n\nCREATORS\nAlison Graham\nRobert Poole\nJeff Green\nPEOPLE\nJeff Green\nRobert Poole\nAlison Graham\nPROJECTS\nSUMO\nSUBMITTER\nAlison Graham\n\n____________________________________________________________________________\n\nMetabolic flux measurement(id: 104 | type: studies)\n\nDescription: Flux will be measured using the metabolomics platforms based on absolute quantification method (isotope ratio based MS technique) by LC-MS, using heavy-isotope labelled precursors of the metabolites of interest. For example, 15N labelled cysteine, glycine and glutamate will be used to determine rates of synthesis of glutathione. 15N-labelled methionine to measure S-adenosyl methionine (and its decarboxylated form, as well as methionine cycle intermediates). 15N labelled arginine is used as precursor for ornithine, putrescine, and spermidine and trypanothione synthesis. 13C labelled glucose will be used to measure production of glucose 6-phosphate, 6-phosphogluconate and pentose-phosphate. Experiments will involve adding these precursors to medium and then following their distribution into the different metabolites under normal physiological conditions and comparing that distribution over time with cells perturbed with the different stress inducing agents and also in selected RNAi and gene knockout cells defective in various enzymes.\n\n\nASSAYS\nGeneration of uniformly 13C-labelled E. coli extract\nLC-MS based absolute quantification of intracellular metabolites\nLC-MS based absolute quantification of extracellular metabolites\nDATA_FILES\nAbsolute intracellular concentrations of metabolites extracted from T. b. brucei 427 wild type\nConcentrations of extracellular metabolites from T. b. brucei 427\nINVESTIGATION\nMetabolite profiling, quantification and flux quantification\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSOPS\nSample preparation for intracellular metabolite extraction in T. b. brucei 427\nGeneration of uniformly 13C-labelled E. coli extract for absolute quantification\nSample preparation procedure for footprint analysis on T. b. brucei 427 using LC/MS\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universitt zu Berlin\nUniversity of Freiburg\nCharit University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nMatthias Knig\nJohannes Bausch\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nfermentation and product spectrum of Clostridium acetobutylicum adhE::int(id: 1026 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nThomas Rimpf\nPEOPLE\nThomas Rimpf\nPROJECTS\nCOSMIC\nSUBMITTER\nThomas Rimpf\n\n____________________________________________________________________________\n\nMeasuring the mobility of soluble and aggergating protein using fluorescence recovery after photobleaching (FRAP) in normal and osmotically stressed cells.(id: 17 | type: sops)\n\nDescription: FRAP experiments are used for studying cytoplasmic diffusion in cells and cells membrane. \n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nKlaus Winzer(id: 21 | type: people)\n\nDescription: A microbiologist with interest in microbial physiology and metabolism. Current research focuses on the biology of pathogenic and non-pathogenic clostridia, in particluar in vivo metabolism and cell-to-cell communication. Also active in metabolic engineering, using both systems and synthetic biology approaches. \n\n\nINSTITUTIONS\nUniversity of Nottingham\nPROJECTS\nCOSMIC\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nopenBIS: Framework for Data Management and Analysis in Bio Research(id: 227 | type: presentations)\n\nDescription: The presentation was given during the PALs meeting in Munich on Dec. 10th, 2015\nIt talks about openBIS, an open, distributed system for managing biological information.\nThe main goal of openBIS  is to support biological research data workflows from the source (i.e. the measurement instruments) to data processing and analyzing facilities.  \n\n\nCREATORS\nRostyslav Kuzyakiv\nEVENTS\n2nd FAIRDOM PALs meeting \nPEOPLE\nRostyslav Kuzyakiv\nPROJECTS\nFAIRDOM\nSUBMITTER\nRostyslav Kuzyakiv\n\n____________________________________________________________________________\n\nSheffield steady state cytochrome measurements(id: 56 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMatthew Rolfe\nPEOPLE\nMatthew Rolfe\nPROJECTS\nSUMO\n\n____________________________________________________________________________\n\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range(id: 248 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nBachmann et al (2011) Molecular Systems Biology 7:516\nPEOPLE\nDawie Van Niekerk\nUrsula Klingmller\nJens Timmer\nMartin Bhm\nMarcel Schilling\nPROJECTS\nMolecular Systems Biology\n\n____________________________________________________________________________\n\nProcessed proteomics data from comparison of E. faecalis wt and deltaEF2933(id: 1012 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMargrete Solheim\nPEOPLE\nMargrete Solheim\nPROJECTS\nSysMO-LAB\nSUBMITTER\nMargrete Solheim\n\n____________________________________________________________________________\n\nRebecca Wade(id: 55 | type: people)\n\nDescription: Group Leader, Molecular and Cellular Modeling Group, EML Research, Heidelberg\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nMODELS\nPYK structural models\nPRESENTATIONS\nComparative computational analysis of enzyme allosteric sites and the binding of allosteric effectors.\nPROJECTS\nSysMO-LAB\nPUBLICATIONS\nRegulation of the Activity of Lactate Dehydrogenases from Four Lactic Acid Bacteria\nOrganism-Adapted Specificity of the Allosteric Regulation of Pyruvate Kinase in Lactic Acid Bacteria\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29  30 November 2012, Heidelberg\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nVolume(id: 80 | type: assays)\n\nDescription: What is the relationship between the activity of Trk1,2 system and cell volume? Is cell volume affected by lack of the main potassium uptake systems? Comparison of cell volume in wild type and TRK mutants.\n\n\nINVESTIGATION\nTRK1,2 Transport Systems of Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nKey Cell-Physiological Parameters  of TRK1,2 Transport Systems Investigation\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nBIOMD0000000005 - Tyson1991 - Cell Cycle 6 var(id: 327 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nRon Henkel\nDagmar Waltemath\nDATA_FILES\nSEDML for BIOMD0000000005\nINVESTIGATION\nHands-on: Model Management in SEEK\nMODELS\nBIOMD0000000005 from BioModels Database\nPEOPLE\nRon Henkel\nDagmar Waltemath\nPROJECTS\nKinetics on the move - Workshop 2016\nPUBLICATIONS\nModeling the cell division cycle: cdc2 and cyclin interactions\nSTUDY\nHands-On: Tyson1991 - Cell Cycle 6 var\nSUBMITTER\nRon Henkel\n\n____________________________________________________________________________\n\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemnde/Rostock 21.05.2012)(id: 40 | type: presentations)\n\nDescription: Presented by Olga Krebs at All Hands PALs meeting on 21-22 of May 2012 in Warnemnde/Rostock\n\n\nCREATORS\nOlga Krebs\nEVENTS\n5th annual SySMO PALs All Hands meeting\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nLocalization of Nha1-GFP(id: 884 | type: data_files)\n\nDescription: Confocal microscopy of Nha1-GFP cells after 60 min at 0 or 50 mM KCl or at 1 M NaCl.\n\n\nCREATORS\nPaul Heusden\nPEOPLE\nPaul Heusden\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nPaul Heusden\n\n____________________________________________________________________________\n\nEntire transcriptional dataset - aerobic to anaerobic transition .csv(id: 46 | type: data_files)\n\nDescription: This .csv file contains the entire transcriptional dataset of the aerobic to anaerobic transition. The values shown are the gene-expression ratio at 2, 5, 10, 15 and 20 minutes relative to the 0 minute timepoint.\n\n\nASSAYS\nTranscriptional profiling of E. coli during anaerobic to aerobic and aerobic to anaerobic transitions\nCREATORS\nMatthew Rolfe\nINVESTIGATIONS\nDynamical studies for different oxygen availability in Escherichia coli\nPEOPLE\nMatthew Rolfe\nPROJECTS\nSUMO\nSTUDIES\nThe Escherichia coli dynamical response to oxygen: from molecular interactions to regulatory modules\n\n____________________________________________________________________________\n\nArabidopsis - starch and the circadian clock, Model 3 (Seaton et al., 2013) - PLM_79, version 1, SIMPLIFIED(id: 378 | type: models)\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "\nDescription: Simplified model file for PLaSMo accession ID PLM_79, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis - starch and the circadian clock, Model 3 (Seaton et al., 2013) - PLM_79, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nSeaton, Daniel\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis - starch and the circadian clock, Model 3 (Seaton et al., 2013) - PLM_79\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature(id: 8 | type: investigations)\n\nDescription: The Sulfolobus systems biology (SulfoSYS)-project represented the first (hyper-)thermophilic Systems Biology project, funded within the European trans-national research initiative Systems Biology of Microorganisms. Within the SulfoSYS-project, focus lies on studying the effect of temperature variation on the central carbohydrate metabolism (CCM) of S. solfataricus that is characterized by the branched EntnerDoudoroff (ED)-like pathway for sugar (glucose, galactose) degradation and the EmbdenMeyerhofParnas (EMP)-like pathway, which is employed during gluconeogenesis. The effect of temperature changes on the CCM network of S. solfataricus is analyzed by the tight integration of bioinformatics, genome, transcriptome, proteome, metabolome, and enzymatic data, with all omic and biochemical data being produced from identical batches of biomass. Beside providing experimental data, one main part of this highly integrative project is the in silico analysis of the CCM network, including the design of a sufficiently precise model according to the silicon cell type model . This model will allow for the computation of the S. solfataricus CCM, and in particular to investigate its robustness to changes in temperature at the system level.\n\n\nASSAYS\nComparison of transcriptome of S. solfataricus at 70 and 80C\nComparison of proteome of S. solfataricus at 70 and 80C\nFermentation of S. solfataricus at 70 and 80C in a batch fermenter\nEnzyme activity tests for S. solfataricus\nIntracellular metabolomics of S. solfataricus at 70 and 80C\nPEOPLE\nPawel Sierocinski\nPROJECTS\nSulfoSys\nPUBLICATIONS\n\"Hot standards\" for the thermoacidophilic archaeon Sulfolobus solfataricus\nSOPS\nSulfosys - SOP proteomics\nSulfosys - SOP metabolomics\nSulfosys - SOP microarray\nSulfosys - SOP fermentation\nSTUDIES\nPilot experiment - S. solfataricus grown at 70 and 80 C.\nSUBMITTER\nPawel Sierocinski\n\n____________________________________________________________________________\n\nZucAt: FBA solution of the model under light growth conditions (I)(id: 1334 | type: data_files)\n\nDescription: The solution of Flux Balance Analysis (FBA) represents metabolic flux distribution in ZucAt model under light growth conditions. In this solution, (i) the ratio photorespiration / photosynthesis has been fixed to 0.25; (ii) and cyclic electron flow through FQR (ferredoxin-plastoquinone reductase) has been set to 0. Under this constraints, ATP formed by non-cyclic photophosphorylation is not sufficient to fulfill ATP/NADPH ratio for carbon fixation, therefore plastid imports ATP from cytoplasm. \n\n\nASSAYS\nFlux Balance Analysis of multi-compartment metabolic model of growing Arabidopsis thaliana\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nZucAt\nSTUDIES\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nFlis et al, 2016, Supplemental Table S4, Global expression profiles(id: 2012 | type: data_files)\n\nDescription: Microarray data at end of day (ED) and end of night (EN) in 4, 6, 8, 12, and 18h photoperiods.\n\n\nASSAYS\nStitt lab, TiMet photoperiod microarrays\nCREATORS\nDaniel Seaton\nINVESTIGATIONS\nPhotoperiodic control of the Arabidopsis proteome reveals a translational coincidence mechanism\nPEOPLE\nDaniel Seaton\nPROJECTS\nMillar group\nTiMet\nPUBLICATIONS\nPhotoperiod-dependent changes in the phase of core clock transcripts and global transcriptional outputs at dawn and dusk in Arabidopsis.\nSTUDIES\nRhythmic and photoperiod-specific transcriptome datasets for Arabidopsis\nSUBMITTER\nDaniel Seaton\n\n____________________________________________________________________________\n\nAbsolute intracellular concentrations of metabolites extracted from T. b. brucei 427 exposed to methylene blue(id: 1022 | type: data_files)\n\nDescription: Metabolic changes of 26 intracellular metabolites extracted from T. b. brucei 427 wild type and arginine kinase knockout cells exposed to methylene blue\n\n\nASSAYS\nIntracellular metabolite concentrations in T. brucei exposed to oxidative stress\nCREATORS\nDong-Hyun Kim\nINVESTIGATIONS\nMetabolite profiling, quantification and flux quantification\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSTUDIES\nUntargeted metabolite analysis\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nBoolean Network Simulation(id: 78 | type: assays)\n\nDescription: A boolean network was created using booleannet (after experimenting with Squad and CellNetAnalyzer). This network can be simulated and visualized using additional software components that will be part of the pyMantis CMS that is developed by the Translucent project.\n\n\nINVESTIGATION\nENA1 Transcription in Saccharomyces cerevisiae\nORGANISMS\nSaccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nModelling of the Signaling network regulating the ENA1-gene transcription\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nPhotoperiod-dependent changes in the phase of core clock transcripts and global transcriptional outputs at dawn and dusk in Arabidopsis.(id: 351 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nStitt lab, TiMet photoperiod microarrays\nDATA_FILES\nFlis et al, 2016, Supplemental Table S4, Global expression profiles\nPEOPLE\nDaniel Seaton\nPROJECTS\nMillar group\nTiMet\n\n____________________________________________________________________________\n\nSugars and circadian regulation make major contributions to the global regulation of diurnal gene expression in Arabidopsis.(id: 353 | type: publications)\n\nDescription: missing\n\n\nDATA_FILES\nBlasing et al, 2005, diurnal microarray dataset in 12L:12D\nPEOPLE\nDaniel Seaton\nPROJECTS\nMillar group\n\n____________________________________________________________________________\n\nMichael Holcombe(id: 32 | type: people)\n\nDescription: Professor of Computer Science, University of Sheffield. FBCS, FIMA, CEng, C.Math, CITP.\r\nI have been involved in the use of computational techniques for modelling biological systems since 1980. More recently I have developed a technique of agent-based modelling based on the framework FLAME which is the only such system that can be run on supercomputers. We have made significant new biological discoveries using this approach: The approach models the location and activity of millions of individual components such as molecules, receptors; cells in tissue, individual insects in ecologies. We are able to represent intricate biological behaviour and the emergence of structure and function in realistic ways.\r\n1. our model of the NFkappaB system (part of the innate immune system) is probably the most detailed model of this pathway and has led to the prediction of the actinisation of IkappaBalpha - this has been confirmed experimentally\r\n2. models of epithelial tissue have been built and the role of stem cells in wound healing has been predicted and confirmed experimentally\r\n3. models of foraging strategies of ant communities - Monomorium pharoensis - has uncovered the geometric structure of pheromone trails and why this is\r\n4 .In SUMO we are modelling the oxygen system in E. coli by representing every relevant molecule and its location within a single cell and studying the interplay between the transcription factors and genes and how these interact within a complex control system. All of the parameters are besed on experimental data from the SUMO consortium. We have predicted the possible existence of a new transcription factor and its properties - other than Fnr and ArcA - and this is being investigated experimentally.\r\n\n\n\nINSTITUTIONS\nUniversity of Sheffield\nPROJECTS\nSUMO\nPUBLICATIONS\nModelling complex biological systems using an agent-based approach\nTowards a Systems Level Understanding of the Oxygen Response of Escherichia coli\nAgent-based modeling of oxygen-responsive transcription factors in Escherichia coli\n\n____________________________________________________________________________\n\nGene expression levels in mutants with linear respiratory chain(id: 1129 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nGene expression analysis of mutants with linear electron transport chain at different aerobiosis levels\nCREATORS\nKatja Bettenbrock\nINVESTIGATIONS\nAnalysis of Escherichia coli with linear electron transport chain\nPEOPLE\nKatja Bettenbrock\nPROJECTS\nSUMO\nSTUDIES\nAnalysis of Escherichia coli strains with linear respiratory chain\nSUBMITTER\nKatja Bettenbrock\n\n____________________________________________________________________________\n\nCellPublisher: a web platform for the intuitive visualization and sharing of metabolic, signalling and regulatory pathways(id: 156 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoerg Stuelke\nLeif Steil\nLope A. Florez\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2011 - PLM_64, version 3, SIMPLIFIED(id: 428 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_64, version 3 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis_clock_P2011 - PLM_64, version 3\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis_clock_P2011 - PLM_64\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nSubtiWiki--a comprehensive community resource for the model organism Bacillus subtilis(id: 168 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoerg Stuelke\nLeif Steil\nLope A. Florez\nUlrike Mder\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2012 - PLM_70, version 2, SIMPLIFIED(id: 434 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_70, version 2 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis_clock_P2012 - PLM_70, version 2\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis_clock_P2012 - PLM_70\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nPhysical interactions between tricarboxylic acid cycle enzymes in Bacillus subtilis: Evidence for a metabolon(id: 94 | type: publications)\n\nDescription: missing\n\n\nDATA_FILES\nProtein report on identification of interaction partners of TCA cycle enzymes in a SPINE experiment\nPEOPLE\nJoerg Stuelke\nUwe Voelker\nLeif Steil\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nChip-chip Array Template (Affymetrix)(id: 929 | type: data_files)\n\nDescription: This Excel template is for use with affy Chip-chip data where the results of the primary analysis are reported in BAR and BED files. It was created from a template on the GEO web site (http://www.ncbi.nlm.nih.gov/geo/info/geo_affy.html) and modified to conform to the SysMO JERM for transcriptomics.\r\n\n\n\nASSAYS\nAffy Transcriptomics Templates\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nModeling metabolic networks including gene expression and uncertainties(id: 342 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nSteffen Waldherr\nHenning Lindhorst\nPROJECTS\nROBUSTYEAST\n\n____________________________________________________________________________\n\nModeling of cellular processes: methods, data, and requirements(id: 99 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlaf Wolkenhauer\nHubert Bahl\nRalf-Jrg Fischer\nThomas Millat\nPROJECTS\nBaCell-SysMO\nCOSMIC\n\n____________________________________________________________________________\n\nComparison of transcriptome of S. solfataricus at 70 and 80C(id: 11 | type: assays)\n\nDescription: The pilot experiment was conducted in order to create SOPs and to gain insight in transcriptome of cells grown at 70 and 80C \n\n\nINVESTIGATION\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature\nORGANISMS\nSulfolobus solfataricus\nPEOPLE\nPawel Sierocinski\nPROJECTS\nSulfoSys\nSOPS\nSulfosys - SOP microarray\nSTUDY\nPilot experiment - S. solfataricus grown at 70 and 80 C.\nSUBMITTER\nPawel Sierocinski\n\n____________________________________________________________________________\n\nTranscriptomics Template (ArrayExpress Format)(id: 8 | type: data_files)\n\nDescription: An example template for MAGE-TAB compliant transcriptomics data. This example only covers the Investigation Description Format (IDF) and the Sample and Data Relationship Format (SDRF) worksheets (i.e. the metadata from a microarray experiment).\r\nThis template was generated automatically using the ArrayExpress submission help tool (http://www.ebi.ac.uk/cgi-bin/microarray/magetab.cgi)\n\n\nASSAYS\nGeneral Transcriptomics Templates\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nETC_mutants_ArcA(id: 1128 | type: data_files)\n\nDescription: The ArcA phosphorylation state was determined for mutants with linear respiratory chain at defined aerobiosis levels.\n\n\nASSAYS\nDeternination of ArcA phosphroylation level in mutants with linear ETC at different defined aerobiosis levels\nCREATORS\nKatja Bettenbrock\nINVESTIGATIONS\nAnalysis of Escherichia coli with linear electron transport chain\nPEOPLE\nKatja Bettenbrock\nPROJECTS\nSUMO\nSTUDIES\nAnalysis of Escherichia coli strains with linear respiratory chain\nSUBMITTER\nKatja Bettenbrock\n\n____________________________________________________________________________\n\nLeendert Hamoen(id: 296 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Newcastle\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nLocalization of general and regulatory proteolysis in Bacillus subtilis cells\nConnecting parts with processes: SubtiWiki and SubtiPathways integrate gene and pathway annotation for Bacillus subtilis\n\n____________________________________________________________________________\n\namino acid auxotrophies E. faecalis(id: 1050 | type: data_files)\n\nDescription: raw data OD600\n\n\nCREATORS\nKoen Van Grinsven\nPEOPLE\nKoen Van Grinsven\nPROJECTS\nSysMO-LAB\nSUBMITTER\nKoen Van Grinsven\n\n____________________________________________________________________________\n\nPGM Kinetic characterisation(id: 206 | type: sops)\n\nDescription: SOP for measurement of PGM activity in extracts.\n\n\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nTranscriptome of shocked B. subtilis cells(id: 26 | type: studies)\n\nDescription: BMM EtOH, 16, 57\r\nSMM NaCl\n\n\nASSAYS\nTiling Array analysis of shocked B. subtilis cells\nDATA_FILES\nThe condition-dependent transcriptome of Bacillus subtilis 168\nINVESTIGATION\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nCondition-dependent transcriptome reveals high-level regulatory architecture in Bacillus subtilis\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nJae Il Joo(id: 727 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nKAIST\nPROJECTS\nFAIRDOM user meeting\n\n____________________________________________________________________________\n\nImke De Jong(id: 281 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nHow mathematical modelling elucidates signalling in B. subtilis\nHeterochronic phosphorelay gene expression as a source of heterogeneity in Bacillus subtilis spore formation\nLive Cell Imaging of <em>Bacillus subtilis</em> and <em>Streptococcus pneumoniae</em> using Automated Time-lapse Microscopy\nBet hedging or not? A guide to proper classification of microbial survival strategies\nSingle cell analysis of gene expression patterns during carbon starvation in Bacillus subtilis reveals large phenotypic variation\n\n____________________________________________________________________________\n\nArabidopsis_clock_2011 - PLM_43, version 6(id: 828 | type: assays)\n\nDescription: \tAlexandra Pokhilko's model of the Arabidopsis clock, private drafts created in preparation for publication (Mol. Syst. Biol.), or as working versions with various modifications after publication.\t\tThe published model version is also in PlaSMo as <a href=\"http://www.plasmo.ed.ac.uk/plasmo/models/model.shtml?accession=PLM_64\">PLM_64 here</a>.<br><br><strong>Comments</strong></strong><p>\n\tMatlab files are attached to version 1</p>\n 2012-01-31 11:08:51 3 amillar2 andrew.millar@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\n\tthis version is similar to version 5 (but csymbol time is replaced to t), it has a graphical representation in Cell Designer, but does not run simulation in Cell Designer. It runs simulation in Copasi and SBSI and runs optimization in SBSI</p>\n<br><br><strong>Originally submitted to PLaSMo on 2011-07-16 12:31:04</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nINVESTIGATION\nPokhilko, Alexandra\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nArabidopsis_clock_2011 - PLM_43\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nNew and upcoming features in SEEK(id: 28 | type: presentations)\n\nDescription: Slide for an overview of new, and future upcoming features being developed in SEEK - presented at the Berlin SEEK Users Meeting March 2012.\n\n\nCREATORS\nStuart Owen\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nSUMO B-galactosidase assay(id: 46 | type: sops)\n\nDescription: This SOP describes the SUMO procedure for determining B-galactosidase activities.\n\n\nASSAYS\nFNR activity at different aerobiosis levels (steady state)\nCREATORS\nMatthew Rolfe\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nPEOPLE\nMatthew Rolfe\nPROJECTS\nSUMO\nSTUDIES\nThe Escherichia coli steady state response to oxygen: from molecular interactions to regulatory modules\nSUBMITTER\nMatthew Rolfe\n\n____________________________________________________________________________\n\nB. subtilis_SysMo2_Chemostat_growhtrate-salt_cell size(id: 893 | type: data_files)\n\nDescription: B. subtilis was grown in minimal media in a chemostat at different growth rates (= max, =0.1, =0.4) and in the presence of 1.2M NaCl (=0.1) with or without glycinebetaine. Here you'll find cell sizes for every sample.\n\n\nASSAYS\nAbsolute quantification of proteins by the AQUA-technology\nCREATORS\nSandra Maass\nMichael Kohlstedt\nINVESTIGATIONS\nMultiomics study of Bacillus subtilis under osmotic stress\nPEOPLE\nSandra Maass\nMichael Kohlstedt\nPROJECTS\nBaCell-SysMO\nSTUDIES\nB. subtilis_SysMo2_Chemostat_growthrate-salt\nSUBMITTER\nSandra Maass\n\n____________________________________________________________________________\n\nTest created 1552504857803(id: 963 | type: assays)\n\nDescription: <br><br><strong>Related Publications</strong></strong><br>Me and Helena (2014). some book. <br>Moore A, Zielinski T, Millar AJ (2014). Online period estimation and determination of rhythmicity in circadian data, using the BioDare data infrastructure.. Methods in molecular biology (Clifton, N.J.). Retrieved from: doi.org/10.1007/978-1-4939-0700-7_2<br>Stein JM (1975). The effect of adrenaline and of alpha- and beta-adrenergic blocking agents on ATP concentration and on incorporation of 32Pi into ATP in rat fat cells.. Biochemical pharmacology. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/12<br><br><strong>Comments</strong></strong><p>\n\tNew comment after 2nd version</p>\n 2019-01-10 11:59:51 201 zielu tomasz.zielinski@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\r\n\tCommentsdfa</p>\r\n<br><br><strong>Originally submitted to PLaSMo on 2014-07-25 18:01:50</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nTomasz Zielinski\nINVESTIGATION\nZielinski, Tomasz\nPEOPLE\nTomasz Zielinski\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nProteome turnover in the green alga Ostreococcus tauri by time course 15N metabolic labeling mass spectrometry.(id: 356 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nMartin et al, 2012, Ostreococcus N15 labelling proteomics data\nDATA_FILES\nMartin et al, 2012, Ostreococcus N15 labelling proteomics\nPEOPLE\nDaniel Seaton\nPROJECTS\nMillar group\n\n____________________________________________________________________________\n\nValidation experiments(id: 203 | type: sops)\n\nDescription: SOP for the determination of external metabolites (Glc, Pyr, Gly, Lac) in intact trophozoite incubations, and for the determination of intracellular metabolite concentrations.\n\n\nASSAYS\nGLC incubation\nSteady state\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel validation\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nMetabolic model of Sulfolobus solfataricus(id: 225 | type: models)\n\nDescription: Metabolic model of Sulfolobus solfataricus P2 in the SBML (xml) and metano (txt, sce, fba) format. Scenarios are specific for growth on D-glucose or L-fucose as sole carbon source. Different theoretical routes of L-fucose degradation were modeled (E. coli-like, Xanthomonas-like and lactaldehyde-forming). Highest overall agreement between the model and experimental data was observed for the lactaldehyde-forming route.\n\n\nASSAYS\nMetabolic model of Sulfolobus solfataricus\nCREATORS\nJacqueline Wolf\nHelge Stark\nDietmar Schomburg\nINVESTIGATIONS\nL-fucose degradation in Sulfolobus solfataricus P2\nPEOPLE\nDietmar Schomburg\nHelge Stark\nJacqueline Wolf\nPROJECTS\nSulfoSys - Biotec\nSTUDIES\nComparison of S. solfataricus grown on l-fucose and d-glucose\nSUBMITTER\nJacqueline Wolf\n\n____________________________________________________________________________\n\nA transcriptional study of acidogenic chemostat cells of Clostridium acetobutylicumcellular behavior in adaptation to n-butanol(id: 146 | type: publications)\n\nDescription: missing\n\n\nMODELS\nModel of the pH-induced metabolic shift in phosphate-limited continuous cultures of C. acetobutylicum assuming a phenotypic switch\nPEOPLE\nArmin Ehrenreich\nServ Kengen\nKatrin Schwarz\nWouter Kuit\nPROJECTS\nCOSMIC\n\n____________________________________________________________________________\n\nAdlung et al (2017) Molecular Systems Biology(id: 121 | type: investigations)\n\nDescription: Protein abundance of AKT and ERK pathway components governs cell-type- specific regulation of proliferation\n\n\nASSAYS\nSource data for Figure 2: Experimental time-resolved quantitative immunoblot data\nMathematical models of the Epo-induced AKT, ERK and S6 activation.\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nDATA_FILES\nSource data for Figure 2B\nSource data for Figure 2C\nSource data for Figure 2D\nSource data for Figure 2E\nSource data for Figure 2F BaF3\nSource data for Figure 2F mCFU-E\nSource data for Figure 2G BaF3\nSource data for Figure 2G mCFU-E\nMODELS\nBaF3 model\nmCFU-E model\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nMolecular Systems Biology\nPUBLICATIONS\nProtein abundance of AKT and ERK pathway components governs cell type-specific regulation of proliferation\nSTUDIES\nFigure 2: Model calibration with time-resolved quantitative immunoblot data of mCFU-E cells and BaF3-EpoR cells.\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nJacek Mika(id: 164 | type: people)\n\nDescription: Kosmobac, WP3, looking at diffusion of macromolecules in vivo (in E.coli cells) and cell responses to osmotic shock using confocal (fluorescence) microscopy especially pulsed - FRAP\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nKOSMOBAC\nPUBLICATIONS\nMolecular sieving properties of the cytoplasm of Escherichia coli and consequences of osmotic stress\nMacromolecule diffusion and confinement in prokaryotic cells\nProtein mobility and diffusive barriers in Escherichia coli: consequences of osmotic stress\nThe role of biomacromolecular crowding, ionic strength, and physicochemical gradients in the complexities of life's emergence\nStructural basis for the enhanced activity of cyclic antimicrobial peptides: The case of BPC194\nThe molecular basis for antimicrobial activity of pore-forming cyclicpeptides\n\n____________________________________________________________________________\n\nProtocol for glucose perturbations(id: 61 | type: sops)\n\nDescription: This protocol for applying glucose perturbations works for Lactococcus lactis and Enterococcus faecalis\n\n\nASSAYS\nGlucose pulsed S. pyogenes\nCREATORS\nMartijn Bekker\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSTUDIES\nComparative modeling and phosphate dependence flux distributions and glucose consumption rates of two lactic acid bacteria S. pyogenes and L. lactis\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nArabidopsis - starch and the circadian clock, Model 1 (Seaton et al., 2013) - PLM_77, version 1, SIMPLIFIED(id: 374 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_77, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis - starch and the circadian clock, Model 1 (Seaton et al., 2013) - PLM_77, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nSeaton, Daniel\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis - starch and the circadian clock, Model 1 (Seaton et al., 2013) - PLM_77\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nMeasured steady state metabolic fluxes(id: 76 | type: data_files)\n\nDescription: Steady state metabolic fluxes measured in glucose-limited chemostat of Saccharomyces cerevisiae at D = 0.1 h-1 growing on minimal medium. Fluxes are: glucose, ethanol, glycerol, acetate, succinate, pyruvate, lactate, citrate, malate, a-ketoglutarate, fumarate\r\n\n\n\nASSAYS\nSteady state extracellular fluxes in anaerobic yeast Saccharomyces cerevisiae\nMOSES: dynamic model of glucose pulse\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nSteady state metabolic fluxes and metabolite concentrations in anaerobic chemostat of yeast Saccharomyces cerevisiae\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nPUBLICATIONS\nFast sampling for quantitative microbial metabolomics: new aspects on cold methanol quenching: metabolite co-precipitation\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSteady state fluxes in yeast Saccharomyces cerevisiae in anaerobic chemostat\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nATP levels after benzoic acid pretreatments(id: 51 | type: studies)\n\nDescription: Effect of benzoate treatment (high concentrations) on ATP levels and Pdr12 expression after pretreatment of cells with low concentrations of benzoic acid.\n\n\nINVESTIGATION\nEffect of Benzoic Acid on ATP Levels\nPEOPLE\nMartin Valachovic\nPROJECTS\nMOSES\nSUBMITTER\nMartin Valachovic\n\n____________________________________________________________________________\n\n003 Biomolecular Extractions from LAO (Qiagen AllPrep)(id: 215 | type: sops)\n\nDescription: Protocol for the extraction of biomolecules (DNA,RNA,protein,metabolties) from the same sample. Originally intended for lipid accumulating organisms (LAO) from wastewater treatment plant samples.\r\nSo far succesfully used for pelleted cells from planktonic cultures. \r\n\n\n\nASSAYS\nExperimental methods\nCREATORS\nMalte Herold\nINVESTIGATIONS\nMulti -omics reveal lifestyle of acidophile, mineral-oxidizing model species Leptospirillum ferriphilumT\nPEOPLE\nMalte Herold\nPROJECTS\nSysMetEx\nSTUDIES\nOmics_data_analysis\nSUBMITTER\nMalte Herold\n\n____________________________________________________________________________\n\nDetailed kinetic model of yeast glycolytic oscillation - VI(id: 79 | type: models)\n\nDescription: An existing detailed kinetic model for the steady-state behavior of yeast glycolysis was tested for its ability to simulate dynamic behavior. This model (dupreez6) is an oscillating version of the model published by Teusink et al., 2000 (PMID: 10951190), which describes data for glycolytic intermediates in cell free extracts of oscillating yeast cultures reported by Das and Busse, 1991 (PMCID: 1260073).\n\n\nASSAYS\nDetailed kinetic model of yeast glycolytic oscillation\nCREATORS\nFranco Du Preez\nJacky Snoep\nINVESTIGATIONS\nYeast Glycolytic Oscillations\nPEOPLE\nJacky Snoep\nFranco Du Preez\nPROJECTS\nSysMO DB\nPUBLICATIONS\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction\nSTUDIES\nDetailed kinetics of yeast glycolytic oscillation\nSUBMITTER\nFranco Du Preez\n\n____________________________________________________________________________\n\nProteomic signatures uncover thiol-specific electrophile resistance mechanisms in Bacillus subtilis(id: 50 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\n\n____________________________________________________________________________\n\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism(id: 225 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nBarbara Bakker\nJurgen  Haanstra\nPROJECTS\nSilicoTryp\n\n____________________________________________________________________________\n\nJrg Schaber(id: 445 | type: people)\n\nDescription: My group investigates dynamic regulation and control mechanisms of cellular signal transduction networks by a combination of theoretical, experimental and computational methods. We seek to make sense of our biological data with the help of mathematical models, which ideally enable us to make valid predictions for new experiments, thereby generating novel biological insights.\n\n\nINSTITUTIONS\nOtto-von-Guericke University Magdeburg\nUniversity of Gothenburg\nPROJECTS\nSulfoSys - Biotec\nICYSB 2015 - International Practical Course in Systems Biology\nPUBLICATIONS\nModelling reveals novel roles of two parallel signalling pathways and homeostatic feedbacks in yeast\nNested autoinhibitory feedbacks alter the resistance of homeostatic adaptive biochemical networks\nSystems biology of the modified branched Entner-Doudoroff pathway in Sulfolobus solfataricus.\n\n____________________________________________________________________________\n\nProteomic Studies(id: 32 | type: studies)\n\nDescription: What is the proteome of starved cells. Main characteristics? \n\n\nASSAYS\n2D-Gel Electrophoresis\nProtein Identification - MS\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nThomas Sauter(id: 35 | type: people)\n\nDescription: From 2005 to 2008 I was group leader at the Institute for System Dynamics at the University of Stuttgart. Since 2008 I am now Professor of Systems Biology at the University of Luxembourg.\r\n\r\nThe research of the Systems Biology Group at the University of Luxembourg is focussed in the area of experimental and theoretical systems biology. We are applying different modelling techniques (mainly ODE and logical) to biological systems to develop suitable computational models. The analysis of these models using system theoretical methods helps gaining insight into dynamic and stationary properties of the investigated biological networks which in turn is the basis for the suggestion of further promising experiments. We are also performing experimental validation of the models using cell cultivation facilities.\r\n\r\nWithin the area of mammalian signal transduction one main focus lays in the development of mathematical models of pro- and anti-apoptotic signalling by death receptors in mammalian cells. We are interested in understanding and influencing the signalling protein networks that govern the decision between life and programmed cell death. The second main focus is on the inflammatory signalling networks which play a crucial role within many aging-related diseases like cancer, Alzheimers and atherosclerosis.\r\n\r\nIn addition experimental validation using mammalian cell cultivation is performed within our own cell cultivation facility in close collaboration with experimental groups within the Life Sciences Research Unit. \n\n\nINSTITUTIONS\nUniversity of Stuttgart\nPROJECTS\nSUMO\nPUBLICATIONS\nBasic Regulatory Principles of Escherichia coli's Electron Transport Chain for Varying Oxygen Conditions\nA mathematical model of metabolism and regulation provides a systems-level view of how Escherichia coli responds to oxygen\n\n____________________________________________________________________________\n\nMathematical Modelling of the pH-induced Metabolic Shift in C. aceobutylicum Unravels a Heterogeneous Phase Transition(id: 73 | type: presentations)\n\nDescription: The acetone-butanol-ethanol (ABE) fermentation of Clostridium acetobutylicum attracts new attention because it provides a potential alternative for the synthesis of value added chemicals to petroleum and other fossil reserves. This fermentative metabolic process comprises two distinct metabolic states that differ in their product spectrum. Growing on starch or sugars the predominant fermentation products are acetate and butyrate during acidogenesis (high pH). In contrast, C. acetobutylicum produces the solvents acetone and butanol during solventogenesis (low pH). In a continuous culture under phosphate limitation, the shift between both metabolic states can be induced by changes of the external pH level. Existing models were unable to reproduce the dynamics of this phase transition.\r\nHere, we present a kinetic model of ABE fermentation in continuous culture that incorporates metabolic reactions, changes of the proteomic composition, pH-dependent kinetics, and population growth. Our analyses of three independent shift experiments led us to the hypothesis that the pH-induced metabolic shift is a heterogeneous process governed by two sub-populations. The extension of our kinetic model with respect to this hypothesis better reproduces the experimental data and led us to the conclusion that an acidogenic and a solventogenic phenotype coexist during the transition from acidogenesis and solventogenesis and vice versa. Furthermore, we show that the measured evolution of the optical density (OD600) represents a transition between these two phenotypes.\r\nFurther analysis revealed that the metabolic shift is governed by three different time scales under the experimental conditions. The rapid decline of the external pH level affects the kinetic properties of the ABE fermentation. Interestingly, the acidogenic population seems to be able to respond to these environmental changes until the pH drops below a critical value. Then, the fermentation pathway is seriously disturbed, so that the acidogenic cells are incapable to grow further. Simultaneously, a low level solventogenic population is increasing. The rise of the solventogenic phenotype starts around 20 hours after the pH-shift and is completed after approximate 50 hours.\r\nThe origin of the solventogenic cells is still unsettled. We suggest two potential explanations: First, the clostridial population is heterogeneous during both metabolic states. Second, few acidogenic cells, selected by unknown criteria, are able to shift their phenotype. Intracellular variability inherent to microbial populations might cause this phenomenon. \r\n\n\n\nCREATORS\nThomas Millat\nPEOPLE\nThomas Millat\nPROJECTS\nCOSMIC\nPUBLICATIONS\nA proteomic and transcriptional view of acidogenic and solventogenic steady-state cells of Clostridium acetobutylicum in a chemostat culture\nA systems biology approach to investigate the effect of pH-induced gene regulation on solvent production by Clostridium acetobutylicum in continuous culture\nSUBMITTER\nThomas Millat\n\n____________________________________________________________________________\n\nContious growth_57C_SMM_Transcritome_Nimblegen(id: 92 | type: data_files)\n\nDescription: Data for three biological replicates of control culture and 37C and 57C in the file. \n\n\nASSAYS\nTiling array analysis of continuous growth stress conditions in SMM\nCREATORS\nPraveen Kumar Sappa\nINVESTIGATIONS\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSTUDIES\nTranscriptome of continuously stressed B. subtilis\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nTargeted disruption and over expression of critical enzymes(id: 92 | type: studies)\n\nDescription: Key enzymes of critical points in the pathways will be targeted for disruption by the generation of RNAi cell lines and lines which drive tetracycline-regulatable ectopic over expression of either wild type enzyme or, if appropriate, dominant-negative or mis-targeted mutants of these. In all cases perturbed lines will be analysed with respect to the mRNA, protein or enzymatic activities of other components of the subsystem, this being directed iteratively by the predictions from systems modelling.\n\n\nASSAYS\nGeneration of RNAi cell lines in T.brucei brucei 2T1 cell line\nInduction of RNA interference in T.brucei brucei 2T1 cell line\nDATA_FILES\nInduction of RNA interference in T.brucei\nRNAi growth curves\nINVESTIGATION\nPerturbing the biological system: developmental adaptation and experimental manipulation of trypanosome biology\nPEOPLE\nFederico Rojas\nPROJECTS\nSilicoTryp\nSOPS\nGeneration of RNAi cell lines in T.brucei brucei 2T1 cell line\nSUBMITTER\nFederico Rojas\n\n____________________________________________________________________________\n\nGenome-scale model of Streptococcus pyogenes(id: 71 | type: models)\n\nDescription: Preliminary metabolic network of S. pyogenes including primary metabolism, polysaccharide metabolism, purine and pyrimidine biosoynthesis, teichoic acid biosynthesis, fatty acid and phospholipid bioynthesis, amino acid metabolism, vitamins and cofactors. The model still needs to be validated.\n\n\nASSAYS\nGenome-scale model of Streptococcus pyogenes\nCREATORS\nJennifer Levering\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nJennifer Levering\nPROJECTS\nSysMO-LAB\nSTUDIES\nReconstructing the metabolic pathways of S. pyogenes and E. faecalis from their genome sequence\nSUBMITTER\nJennifer Levering\n\n____________________________________________________________________________\n\nGrowth rate dependent control in Enterococcus faecalis: effects on the transcriptome, proteome and strong regulation of lactate dehydrogenase(id: 129 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nIngolf Nes\nMartijn Bekker\nTomas Fiedler\nHelge Holo\nIbrahim Mehmeti\nPROJECTS\nSysMO-LAB\n\n____________________________________________________________________________\n\nEnvironmental salinity determines the specificity and need for Tat-dependent secretion of the YwbN protein in Bacillus subtilis(id: 165 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nJan Maarten Van Dijl\nColin Harwood\nPraveen Kumar Sappa\nUlrike Mder\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nLocal sensitivity analysis based on 40 samples(id: 2333 | type: data_files)\n\nDescription: Local sensitivity analysis based on 40 samples using 1000x sampling from measurement distribution. The control shown is the control over flux through glycolysis represented by flux through PRK. Th plot summarized the control for each parameter over all observed metabolite concentrations encountered for the 40 samples. As such the metabolic control analysis is local but shows the distribution taking into account measurement error as well as biological variation over the 40 samples.\n\n\nPEOPLE\nNiels Zondervan\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nSUBMITTER\nNiels Zondervan\n\n____________________________________________________________________________\n\nIntroduction into SABIO-RK(id: 545 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nAndreas Weidemann\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nAndreas Weidemann\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nMetabolic changes by oxidative stress in T. b. brucei 427(id: 923 | type: data_files)\n\nDescription: Untargeted and targeted metabolic analysis on T. b. brucei 427 grown under oxidative stress with methylene blue has been carried out. This work has been completed with 11 bio-reps and found significant metabolic changes as you can see in the IDEOM file attached. 'Comparison' tab in the data spread sheet shows heat maps and fold change analysis regarding different metabolite levels (T: T brucei, TMB: T. brucei exposed to methylene blue, numbers: time points, 0, 5, 60 & 120min). If you double click any metabolite name (column E), then you will be able to see the graph showing the metabolite level. If you want to understand more about IDEOM, please see the below reference as IDEOM has been recently published in Bioinformatics. Through multivariate and univariate analysis,  a potential oxidative stress maker has been found since the level of ariginine phosphate dramatically decreased as methylene blue concentration increased. The participation of the phosphagen system related to arginine kinase in oxidative stress responses has been already reported (C. A. Pereira et al., 2003, FEBS Letters, 554, 201-205). Arginine kinase kock-out (ArgK KO) cells have been already produced and it is under investigation if there are any different drug responses between ArgK KO and wild type cells.\r\n\r\nFor IDEOM: D. J. Creek et al., 2012, IDEOM: An Excel interface for analysis of LC-MS based metabolomics data, Bioinformatics.\r\n\r\nhttp://bioinformatics.oxfordjournals.org/content/early/2012/02/04/bioinformatics.bts069.abstract?ikey=AVa4c4dfHWRqjU6&keytype=ref%253Chttp%3A%2F%2Fbioinformatics.oxfordjournals.org%2Fcgi%2Fcontent%2Fabstract%2Fbts069%3Fijkey%3DAVa4c4dfHWRqjU6&keytype=ref%253E\r\n\r\n\n\n\nASSAYS\nMetabolite profiling on T. brucei exposed to oxidative stress\nCREATORS\nDong-Hyun Kim\nINVESTIGATIONS\nMetabolite profiling, quantification and flux quantification\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSTUDIES\nUntargeted metabolite analysis\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nIntegration of sensitivity and bifurcation analysis to detect critical processes in a model combining signalling and cell population dynamics(id: 90 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlaf Wolkenhauer\nUlf Liebal\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2011 - PLM_64, version 3(id: 840 | type: assays)\n\nDescription: \tThis model is termed P2011 and derives from the article: <strong>The clock gene circuit in Arabidopsis includes a repressilator with additional feedback loops</strong><strong>. </strong>Alexandra Pokhilko, Aurora Pias Fernndez, Kieron D Edwards, Megan M Southern, Karen J Halliday &amp; Andrew J Millar <em>Mol. Syst. Biol. </em>2012; 8: 574, submitted 9 Aug 2011 and published 6 March 2012. <a href=\"http://www.nature.com/msb/journal/v8/n1/synopsis/msb20126.html\">Link</a>\tLink to <a href=\"http://www.nature.com/msb/journal/v8/n1/extref/msb20126-s1.pdf\">Supplementary Information</a>, including equations. Minor errors in the published Supplementary Information are described in a file attached to version 1 of this model (the published SBML is correct).\t&nbsp;The model describes the circuit depicted in Fig. 1 of the paper (GIF attached). It updates the Pokhilko et al. 2010 model (termed P2010), <a href=\"http://www.plasmo.ed.ac.uk/plasmo/models/model.shtml?accession=PLM_6\">PLM_6</a>, by including:<ol><li>\t\tthe Evening Complex genes (ELF4, ELF3, LUX),</li><li>\t\tlight-regulated degradation of ELF3 by COP1,</li><li>\t\tTOC1 as a repressor rather than an activator of LHY/CCA1.</li></ol>\tThese changes allowed the removal of hypothetical components TOC1mod (or X) and Y from the earlier models. They also reveal that the central loop of the model is a triple-repressor ring oscillator, or 'repressilator' (illustrated in Fig. 8, GIF attached).\tSBML curation notes (please see Comments for each version): Compared to the model version submitted to the Biomodels database, version 1 here slightly alters the names of some variables and uses an SBML AssignmentRule for the light input. A MATLAB version is attached to version 1. In version 2, the rule is used with a generic SBML StepFunction (Adams et al. J. Biol. Rhythms Aug 2012) to describe the light-dark cycle more flexibly. Versions 1 and 2 have multiple sinks and sources created by CellDesigner. These are removed in versions 3 &amp; 4. Version 3 has the published lightfunction, and an equivalent COPASI file attached. Version 4 has the StepFunction.<br><br><strong>Related Publications</strong></strong><br>Alexandra Pokhilko, Aurora Pias Fernndez, Kieron D Edwards, Megan M Southern, Karen J Halliday & Andrew J Millar (2012). The clock gene circuit in Arabidopsis includes a repressilator with additional feedback loops.. Mol. Syst. Biol. 8: 574.. Retrieved from: http://www.nature.com/msb/journal/v8/n1/synopsis/msb20126.html<br><br><strong>Comments</strong></strong><p>\n\tVersion 2 of the model includes the StepFunction. Note that we have not yet removed the original light function parameter names, they are now non-functional.</p>\n 2012-06-01 17:29:46 3 amillar2 andrew.millar@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\r\n\tThe SBML for this version uses the lightfunction (as in Mol Syst Biol 2012 paper) to represent the light-dark cycle. It is assigned to parameter name L, which is used in the model reactions.</p>\r\n<p>\r\n\tHowever, the internal parameters of lightfunction have default Copasi ID&#39;s, parameter_1 to parameter_6, with names that match the publication (photperiod, etc.). &nbsp;A COPASI file for this model is also attached.</p>\r\n<p>\r\n\tThis model is equivalent to the published version, referred to as P2011.1.1 in our later paper (Flis et al. RSOB 2015).</p>\r\n<br><br><strong>Originally submitted to PLaSMo on 2012-03-07 12:15:12</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nAndrew Millar\nDATA_FILES\nFig. 1, outline of P2011 model, with P2010 inset,PLM_64_ 3\nFig. 8, cartoon illustrating repressilator structure,PLM_64_ 3\nODE file for Matlab version of P2011 clock model, PLM_64_ 3\nMatlab version of P2011 clock model, PLM_64_ 3\nCopasi file corresponding to PLM_64 version 3, PLM_64_ 3\nINVESTIGATION\nMillar, Andrew\nMODELS\nArabidopsis_clock_P2011 - PLM_64, version 3, SUBMITTED\nArabidopsis_clock_P2011 - PLM_64, version 3, SIMPLIFIED\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nArabidopsis_clock_P2011 - PLM_64\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nLocalization of Ena1-GFP(id: 882 | type: data_files)\n\nDescription: Confocal microscopy of Ena1-GFP cells after 60 min at 0 or 50 mM KCl or at 1 M NaCl.\n\n\nCREATORS\nPaul Heusden\nPEOPLE\nPaul Heusden\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nPaul Heusden\n\n____________________________________________________________________________\n\nNovel activities of glycolytic enzymes in Bacillus subtilis: interactions with essential proteins involved in mRNA processing(id: 71 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoerg Stuelke\nUwe Voelker\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nMathematical Modelling of the Sporulation-Initiation Network in Bacillus Subtilis Revealing the Dual Role of the Putative Quorum-Sensing Signal Molecule PhrA(id: 34 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMathematical modelling of the sporulation-initiation network in Bacillus subtilis.\nMathematical Modelling of the Sporulation-Initiation Network in Bacillus Subtilis Revealing the Dual Role of the Putative Quorum-Sensing Signal Molecule PhrA\nPEOPLE\nJohn King\nJohn Heap\nSara Jabbari\nPROJECTS\nCOSMIC\n\n____________________________________________________________________________\n\nWP3: Cell Fate Decisions of Erythroid Progenitor Cells(id: 104 | type: investigations)\n\nDescription: missing\n\n\nCREATORS\nLorenz Adlung\nMelania Barile\nMarcel Schilling\nPatricia Sauer\nChristian Tnsing\nMarie-Christine Wagner\nPEOPLE\nMaja Rey\nLorenz Adlung\nMelania Barile\nMarcel Schilling\nPatricia Sauer\nChristian Tnsing\nMarie-Christine Wagner\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\nPUBLICATIONS\nIdentification of Cell Type-Specific Differences in Erythropoietin Receptor Signaling in Primary Erythroid and Lung Cancer Cells\nProtein abundance of AKT and ERK pathway components governs cell type-specific regulation of proliferation\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nMetabolomics perturbation samples preparation(id: 287 | type: sops)\n\nDescription: Metabolomics perturbation sample preparation and description of how the exact details of the perturbations.\r\nPerturbations:\r\n-Glucose starvation\r\n-Amino acid starvation\r\n-Fe2+ depletion\r\n-Oxidative stress via H2O2\r\n-Glycerol addition to the medium\n\n\nCREATORS\nNiels Zondervan\nLuis Serrano\nMaria Lluch\nEva Yus\nPEOPLE\nNiels Zondervan\nLuis Serrano\nMaria Lluch\nEva Yus\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nSUBMITTER\nNiels Zondervan\n\n____________________________________________________________________________\n\nPlerixafor induces the rapid and transient release of stromal cell-derived factor-1 alpha from human mesenchymal stromal cells and influences the migration behavior of human hematopoietic progenitor cells(id: 280 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nPEOPLE\nMaja Rey\nRainer Saffrich\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nEvaluation of GMP-compliant culture media for in vitro expansion of human bone marrow mesenchymal stromal cells(id: 273 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nPEOPLE\nMaja Rey\nRainer Saffrich\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nZucAt: FBA solution of the model under light growth conditions (III)(id: 1336 | type: data_files)\n\nDescription: The solution of Flux Balance Analysis (FBA) represents metabolic flux distribution in ZucAt model under light growth conditions. In this solution, (i) the ratio photorespiration / photosynthesis has been fixed to 0.25; and (ii) cyclic electron flow through FQR (ferredoxin-plastoquinone reductase) has been set 0.5 from non-cyclic flow through FRN (ferredoxin-NADP oxidoreductase). Under this constraints, ATP is over-produced in plastid and a surplus is exported to cytoplasm. Flux through FQR represents cyclic electron flow through the light photosynthesis system. Flux through FRN represents non-cyclic electron flow through the light photosynthesis system.\n\n\nASSAYS\nFlux Balance Analysis of multi-compartment metabolic model of growing Arabidopsis thaliana\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nZucAt\nSTUDIES\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOM Data Management : SEEK and openBIS(id: 472 | type: presentations)\n\nDescription: Talk given by Olga Krebs at  FAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23  in  Hnfeld (D)\r\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\nPEOPLE\nOlga Krebs\nPROJECTS\nFAIRDOM\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nProtein report on identification of interaction partners of CitG in a SPINE experiment(id: 1057 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nJoerg Stuelke\nPEOPLE\nJoerg Stuelke\nLeif Steil\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nMalate-mediated carbon catabolite repression in Bacillus subtilis involves the HPrK/CcpA pathway\nSUBMITTER\nLeif Steil\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hnfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nThe role of biomacromolecular crowding, ionic strength, and physicochemical gradients in the complexities of life's emergence(id: 122 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nBert Poolman\nJacek Mika\nPROJECTS\nKOSMOBAC\n\n____________________________________________________________________________\n\nTrophozoite Isolation and Lysate Preparation(id: 268 | type: assays)\n\nDescription: Assay for the isolation of intact Plasmodium falciparum trophozoites from infected red blood cells and the preparation of a cell free extract that can be used for kinetic analyses.\n\n\nINVESTIGATION\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDY\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nSulfoSYS_BIOTECH: RNA isolation and sequencing(id: 209 | type: sops)\n\nDescription: missing\n\n\nASSAYS\nRNA sequencing:l-fuc/d-glu\nRNA sequencing: Casaminoacids vs D-glc\nCREATORS\nTheresa Kouril\nINVESTIGATIONS\nL-fucose degradation in Sulfolobus solfataricus P2\nAmino acid degradation in Sulfolobus solfataricus P2\nPEOPLE\nTheresa Kouril\nPROJECTS\nSulfoSys - Biotec\nSTUDIES\nComparison of S. solfataricus grown on l-fucose and d-glucose\nComparison of Sulfolobus solfataricus P2 grown on caseinhydrolysate and D-glucose\nSUBMITTER\nTheresa Kouril\n\n____________________________________________________________________________\n\n[Mesenchymal stroma cells (MSCs) for the treatment of rheumatic disease](id: 283 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nPEOPLE\nMaja Rey\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets(id: 329 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nFinn Bacall\nQuyen Nguyen\nPROJECTS\nFAIRDOM\nExample use cases\n\n____________________________________________________________________________\n\nSteady state transcriptional profiles at different oxygen concentrations(id: 54 | type: data_files)\n\nDescription: Transcriptional profiles of steady-state chemostat cultures were measured at a range of oxygen levels using microarrays. Oxygen levels were defined by the aerobiosis scale of Alexeeva et al. (2002). \n\n\nASSAYS\nTranscriptional profiling of steady states at different aerobiosis levels\nCREATORS\nMatthew Rolfe\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nPEOPLE\nMatthew Rolfe\nPROJECTS\nSUMO\nPUBLICATIONS\nTranscript profiling and inference of Escherichia coli K-12 ArcA activity across the range of physiologically relevant oxygen concentrations\nSTUDIES\nThe Escherichia coli steady state response to oxygen: from molecular interactions to regulatory modules\nSUBMITTER\nMatthew Rolfe\n\n____________________________________________________________________________\n\nCellular size and granularity during glucose pulse(id: 23 | type: assays)\n\nDescription: Cellular size and granularity (measured by FACS) during glucose pulse. Glucose pulse was performed in anaerobically growing yeast Saccharomyces cerevisiae in steady state chemostat (D = 0.1 h-1) and transent concentrations of the extra- and intracellular metabolites from central carbon metabolism (e.g. glycolysis, PPP, glycerol, purines, etc) were measured.\n\n\nDATA_FILES\nCellular size and granularity during glucose pulse\nINVESTIGATION\nKinetic analysis of metabolic system using transient perturbations\nORGANISMS\nSaccharomyces cerevisiae\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nPUBLICATIONS\nCell size and morphological properties of yeast Saccharomyces cerevisiae in relation to growth temperature\nSOPS\nAnaerobic media composition\nSampling of biomass\nPerturbation conditions\nDetermination of intracellular metabolites\nYeast strains\nSTUDY\nMetabolic perturbation of the steady state culture by glucose pulse\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nSulman Shafeeq(id: 409 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nNoisy-Strep\nPUBLICATIONS\nControl of transcription elongation by GreA determines rate of gene expression in Streptococcus pneumoniae\n\n____________________________________________________________________________\n\nAlexander Mazein(id: 755 | type: people)\n\nDescription: POSITON\r\nSenior Researcher at the European Institute for Systems Biology and Medicine (EISBM), Lyon, France\r\n\r\nRESEARCH\r\nThe Disease Maps Project: a community effort for comprehensive representation of disease mechanisms and omics data interpretation in connection to EU projects U-BIOPRED (Unbiased BIOmarkers in Prediction of REspiratory Disease outcomes) and eTRIKS (european TRanslational Information and Knowledge Management Services) funded by IMI (Innovative Medicines Initiative).\n\n\nINSTITUTIONS\nEuropean Institute for Systems Biology and Medicine\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nProcessing Streptomyces coelicolor cell pellets(id: 138 | type: sops)\n\nDescription: missing\n\n\nCREATORS\nLouise Thomas\nMaggie Smith\nPEOPLE\nMaggie Smith\nJay Moore\nLouise Thomas\nPROJECTS\nSTREAM\nSUBMITTER\nJay Moore\n\n____________________________________________________________________________\n\nBiomass weight during glucose pulse(id: 80 | type: data_files)\n\nDescription: Biomass weight during glucose pulse. Glucose pulse was performed in anaerobically growing yeast Saccharomyces cerevisiae in steady state chemostat (D = 0.1 h-1) and transent concentrations of the extra- and intracellular metabolites from central carbon metabolism (e.g. glycolysis, PPP, glycerol, purines, etc) were measured.\n\n\nASSAYS\nBiomass weight during glucose pulse\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nTranscription of glucose uptake transport systems in a mutant background under aerobic conditions(id: 1071 | type: data_files)\n\nDescription: Transcription of glucose uptake transport systems in a mutant background under aerobic batch conditions.\n\n\nASSAYS\nAerobic and anaerobic characterization of MG1655 and mutant strains with regard to transcription profiles.\nCREATORS\nSonja Steinsiek\nINVESTIGATIONS\nAnalysis of the glucose transport in Escherichia coli under aerobic and anaerobic conditions\nPEOPLE\nSonja Steinsiek\nPROJECTS\nSUMO\nPUBLICATIONS\nGlucose Transport in Escherichia coli Mutant Strains with Defects in Sugar Transport Systems\nSTUDIES\nCharacterization of mutant strains with defects in sugar transport systems\nSUBMITTER\nSonja Steinsiek\n\n____________________________________________________________________________\n\nLorenz Adlung(id: 763 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nWP3: Cell Fate Decisions of Erythroid Progenitor Cells\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\nPUBLICATIONS\nData2Dynamics: a modeling environment tailored to parameter estimation in dynamical systems\nIdentification of isoform-specific dynamics in phosphorylation-dependent STAT5 dimerization by quantitative mass spectrometry and mathematical modeling\nProtein abundance of AKT and ERK pathway components governs cell type-specific regulation of proliferation\n\n____________________________________________________________________________\n\nCell survival under different benzoic acid concentrations(id: 52 | type: studies)\n\nDescription: Cell survival was determined under different benzoic acid concentrations\n\n\nINVESTIGATION\nEffect of Benzoic Acid on ATP Levels\nPEOPLE\nMartin Valachovic\nPROJECTS\nMOSES\nSUBMITTER\nMartin Valachovic\n\n____________________________________________________________________________\n\nJoerg Stuelke(id: 5 | type: people)\n\nDescription: missing\n\n\nDATA_FILES\nTrancriptome data - RNA processing in Bacillus subtilis: identification of targetsof the essential RNase Y\nProtein report on identification of interaction partners of TCA cycle enzymes in a SPINE experiment\nPhosphorylation site of the HPr protein\nEnzymatic activities of the RocG variants\nProtein report on identification of interaction partners of CitG in a SPINE experiment\nThe condition-dependent transcriptome of Bacillus subtilis 168\nCatabolite repression of -xylosidase by different carbon sources in various mutants\nCatabolite repression of -xylosidase by different carbon sources in various mutants\nINSTITUTIONS\nUniversity of Goettingen\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nGlutamate metabolism in Bacillus subtilis: gene expression and enzyme activities evolved to avoid futile cycles and to allow rapid responses to perturbations of the system\nCarbon catabolite repression in bacteria: many ways to make the most out of nutrients\nCarbon catabolite repression in Bacillus subtilis: quantitative analysis of repression exerted by different carbon sources\nIn vitro phosphorylation of key metabolic enzymes from Bacillus subtilis: PrkC phosphorylates enzymes from different branches of basic metabolism\nNovel activities of glycolytic enzymes in Bacillus subtilis: interactions with essential proteins involved in mRNA processing\nA community-curated consensual annotation that is continuously updated: the Bacillus subtilis centred wiki SubtiWiki\nConnecting parts with processes: SubtiWiki and SubtiPathways integrate gene and pathway annotation for Bacillus subtilis\nPhysical interactions between tricarboxylic acid cycle enzymes in Bacillus subtilis: Evidence for a metabolon\nCondition-dependent transcriptome reveals high-level regulatory architecture in Bacillus subtilis\nTrigger enzymes: bifunctional proteins active in metabolism and in controlling gene expression\nSPABBATS: A pathway-discovery method based on Boolean satisfiability that facilitates the characterization of suppressor mutants\nCellPublisher: a web platform for the intuitive visualization and sharing of metabolic, signalling and regulatory pathways\nCarbon source control of the phosphorylation state of the Bacillus subtilis carbon-flux regulator Crh in vivo\nRNase Y in Bacillus subtilis: a Natively disordered protein that is the functional equivalent of RNase E from Escherichia coli\nRNA processing in Bacillus subtilis: identification of targets of the essential RNase Y\nSubtiWiki--a comprehensive community resource for the model organism Bacillus subtilis\nMalate-mediated carbon catabolite repression in Bacillus subtilis involves the HPrK/CcpA pathway\nA high-frequency mutation in Bacillus subtilis: requirements for the decryptification of the gudB glutamate dehydrogenase gene\nMalate metabolism in Bacillus subtilis: distinct roles for three classes of malate-oxidizing enzymes\nRNA degradation in Bacillus subtilis: an interplay of essential endo- and exoribonucleases\nThe DEAD-box RNA helicases in Bacillus subtilis have multiple functions and act independent from each other\n\n____________________________________________________________________________\n\nSample preparation for mRNA decay study(id: 170 | type: sops)\n\nDescription: For the study of mRNA decay rates, transcription was inhibited with ActinomycinD, and RNA splicing with Sinefungin, at different time points, in the Matthews lab. rRNA depleted RNA was extracted from each of the samples in the Clayton lab, and sent for deep sequencing at the BioQuant facility in Heidelberg\n\n\nASSAYS\nmRNA decay assay\nCREATORS\nFederico Rojas\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nFederico Rojas\nPROJECTS\nSilicoTryp\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\r\nphosphate pathway enzymes\nSUBMITTER\nFederico Rojas\n\n____________________________________________________________________________\n\ntest file(id: 1665 | type: data_files)\n\nDescription: test file\n\n\nCREATORS\nOlga Krebs\nVnia Pobre\nPEOPLE\nOlga Krebs\nVnia Pobre\nPROJECTS\nFAIRDOM user meeting\nEmPowerPutida\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nArabidopsis_clock_extend - PLM_6, version 3, SIMPLIFIED(id: 422 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_6, version 3 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis_clock_extend - PLM_6, version 3\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nPokhilko, Alexandra\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis_clock_extend - PLM_6\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nData of glucose-pulsed E. faecalis(id: 603 | type: data_files)\n\nDescription: E. faecalis was gucose-pulsed after resuspension in 100 mM MES buffer at pH 6.5\r\nIntra- and extracellular metabolites concentrations were followed in time\n\n\nCREATORS\nMartijn Bekker\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nShake flask cultivations of Bacillus subtilis 168 trp+ on M9-DHB medium(id: 156 | type: sops)\n\nDescription: missing\n\n\nCREATORS\nMichael Kohlstedt\nPEOPLE\nMichael Kohlstedt\nPROJECTS\nBaCell-SysMO\nSUBMITTER\nMichael Kohlstedt\n\n____________________________________________________________________________\n\nSOP - RNA/Protein sampling for continuous culture(id: 214 | type: sops)\n\nDescription: steps conducted to pellet cells for RNA and protein extraction\n\n\nASSAYS\nExperimental methods\nCREATORS\nStephan Christel\nINVESTIGATIONS\nMulti -omics reveal lifestyle of acidophile, mineral-oxidizing model species Leptospirillum ferriphilumT\nPEOPLE\nStephan Christel\nPROJECTS\nSysMetEx\nSTUDIES\nOmics_data_analysis\nSUBMITTER\nStephan Christel\n\n____________________________________________________________________________\n\nA mechanism for cell cycle regulation of sporulation initiation in Bacillus subtilis(id: 107 | type: publications)\n\nDescription: missing\n\n\nDATA_FILES\nRNA-seq wild type\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nSofie Sderstrm(id: 1125 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nA shift in the dominant phenotype governs the pH-induced metabolic switch of Clostridium acetobutylicumin phosphate-limited continuous cultures(id: 207 | type: publications)\n\nDescription: missing\n\n\nMODELS\nEstimation of an acidogenic and solventogenic population from optical density data observed in 'forward'-shift experiments using continuous cultures of C. acetobutylicum\nModel of the pH-induced metabolic shift in phosphate-limited continuous cultures of C. acetobutylicum assuming a phenotypic switch\nPEOPLE\nOlaf Wolkenhauer\nHubert Bahl\nJohn King\nRalf-Jrg Fischer\nThomas Millat\nHolger Janssen\nGraeme Thorn\nPROJECTS\nCOSMIC\n\n____________________________________________________________________________\n\nSheffield chemostat parameters .csv(id: 52 | type: data_files)\n\nDescription: This .csv file contains growth parameters measured from chemostat runs of the Sheffield Infors chemostats. The runs were carried out for acetate calibration purposes and for sampling for the steady-state transcriptional profiles.\r\n\r\nColumns:\r\nDate\r\nO2_input (% total)\r\nAerobiosis level (%AAU)\r\nOD600\r\nRedox (mV)\r\npO2 (M)\r\nDCW (g/L)\r\nViable cell counts (CFU/ml)\r\nRespRate (mmol/h/gDCW)\r\nRespRate STDEV\r\n[Acetate] (mM)\r\nQacet (mmoles/h/gDCW)\r\n\n\n\nASSAYS\nPhysiological measurements from Sheffield chemostat\nCREATORS\nMatthew Rolfe\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nPEOPLE\nMatthew Rolfe\nPROJECTS\nSUMO\nSTUDIES\nThe Escherichia coli steady state response to oxygen: from molecular interactions to regulatory modules\n\n____________________________________________________________________________\n\nMOSES: dynamic model of glucose pulse(id: 132 | type: assays)\n\nDescription: The dynamic model describes response of yeast metabolic network on metabolic perturbation (i.e. glucose-pulse). One compartmental ODE-based model of yeast anaerobic metabolism includes: glycolysis, pentose phosphate reactions, purine de novo synthesis pathway, purine salvage reactions, redox reactions and biomass growth. The model describes metabolic perturbation of steady state growing cells in chemostat. \n\n\nDATA_FILES\nMeasured steady state metabolic fluxes\nMeasured steady state concentrations of extracellular metabolites in yeast Saccharomyces cerevisiae anaerobic chemostat\nMeasured steady state concentrations of intracellular metabolites in yeast Saccharomyces cerevisiae anaerobic chemostat\nOff-gas monitoring during glucose pulse\nDynamics of extracellular metabolites during glucose pulse\nDynamics of intracellular metabolites during glucose pulse\nINVESTIGATION\nKinetic analysis of metabolic system using transient perturbations\nMODELS\nSBML model of yeast central carbon metabolism\nORGANISMS\nSaccharomyces cerevisiae\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nSTUDY\nMetabolic perturbation of the steady state culture by glucose pulse\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nIdentification of isoform-specific dynamics in phosphorylation-dependent STAT5 dimerization by quantitative mass spectrometry and mathematical modeling(id: 277 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nLorenz Adlung\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nEffect of Benzoic Acid on ATP Levels(id: 22 | type: investigations)\n\nDescription: This investigation examines the effect of benzoic acid, a food preservative, on the ATP levels of yeast cells. \n\n\nASSAYS\nCompare ATP extraction methods\nATP measurement under 10mM benzoic acid stress\nATP measurement under 30mM benzoic acid stress\nATP measurement under 50mM benzoic acid stress\nPEOPLE\nMartin Valachovic\nPROJECTS\nMOSES\nSOPS\nATP determination with ROCHE kit  ATP Bioluminescence Assay Kit CLS II\nSTUDIES\nTest different ATP extraction protocols and conditions\nATP levels under 10 mM benzoic acid stress\nATP levels under 30 mM and 50 mM benzoic acid stress\nATP levels after benzoic acid pretreatments\nCell survival under different benzoic acid concentrations\nSUBMITTER\nMartin Valachovic\n\n____________________________________________________________________________\n\nNiels Zondervan(id: 873 | type: people)\n\nDescription: I am a researcher  (PhD student) working at Wageningen University & Research as bioinformatician and modeller. I am working as part of the MycoSynVac (http://www.mycosynvac.eu/) project on dynamic modelling of central carbon metabolism in M. pneumoniae, to be extended to full dynamic modelling of metabolism to be implemented in a whole cell model. \r\nI am also looking into possibilities to improve standards in model generation using semantic technologies, improving automatic generation, annotation and provenance of dynamic models as well as retrieval of enzyme kinetic information from online resources such as Brenda and SabioRK to improve parameter estimation.\n\n\nASSAYS\nComparison of Kcat values from the model and values from literature\nConstruction of a Genome Scale Metabolitic model of M. hyopneumoniae\nTranscriptomics assay of M. pneumoniae at diferent times of growth\n40 samples data analysis - metabolite correlation\nDATA_FILES\nModel training, parameter estimation\nComparison of Kcat values model and values from literature\nInternal metabolite concentraitons for mutants, perturbation and time series samples including mutant enzyme fold change\nComparison of model SS metabolite concentrations with measured values using sampling\nMean Absolute Percentage Error between measured and simulated metabolite concentrations using 1000x sampling\nDouble mutants and perturbations\nMaster file, metabolite concentration, protein copy number and flux estimates to train the model\nExternal metabolite concentration times series\nProteomics, protein copy number measured over time\nMetabolites all experiments, relative measurements\nAbsolute copy number per locus tag, M. pneumoniae at different times of growth\nSymmetric mean absolute percentage error per sample\nParameter estmimation for model with addition of NoxE\n40 samples internal metabolite concentrations FC values used for model simmulations\nParameter scan for the model with addition of oxygen inhibition of LDH\n40 samples metabolite correlation analysis - heatmaps, PCA, correlation network\nGlobal sensitivity analysis\nGlobal sensitivity analysis - correlation between parameters\nLocal sensitivity analysis based on 40 samples\nGlobal sensitivity analysis - tab seperated file\nLocal sensitivity analysis based on 40 samples - tab seperated file\nDynamic modelling pipeline\nINSTITUTIONS\nWageningen University & Research\nMODELS\nGenome Scale Metabolic model of M. hyopneumoniae\nDynamic model of glycolysis, pyruvate metabolism and ATPase in M. pneumonaie with innitial parameter values and annotation\nCore model of glycolysis, pyruvate metabolism ATPase + NoxE reaction\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nWURSynBio\nPUBLICATIONS\nDissecting the energy metabolism in Mycoplasma pneumoniae through genome-scale metabolic modeling.\nLarge-scale metabolome analysis and quantitative integration with genomics and proteomics data in Mycoplasma pneumoniae.\nSOPS\nMetabolomics perturbation samples preparation\nSOP for generating a Genome Scale Metabolic model of M. hyopneumoniae\nSOP  for dynamic core model construction and parameter estimation\nSTUDIES\nProteomics analysis\nGenome-scale, constraint-based metabolic modeling of M. hyopneumonia\nTranscriptomics of M. pneumoniae at different times of growth\n\n____________________________________________________________________________\n\nGlucose perturbations in S. pyogenes(id: 62 | type: sops)\n\nDescription: Protocol for applying a glucose perturbation in Streptococcus pyogenes. \n\n\nCREATORS\nMartijn Bekker\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nDeborah Croteau(id: 1263 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nNational Institute on Aging\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nSimplified absolute metabolite quantification by gas chromatographyisotope dilution mass spectrometry on the basis of commercially available source material(id: 137 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nSteady state concentrations of extracellular metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nDynamics of extracellular metabolites during glucose pulse\nDATA_FILES\nMeasured steady state concentrations of extracellular metabolites in yeast Saccharomyces cerevisiae anaerobic chemostat\nDynamics of extracellular metabolites during glucose pulse\nPEOPLE\nMaksim Zakhartsev\nMatthias Reuss\nPROJECTS\nMOSES\n\n____________________________________________________________________________\n\nBiphasic response as a mechanism against mutant takeover in tissue homeostasis circuits(id: 384 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nKarin et al (2017) Molecular Systems Biology\nPEOPLE\nJacky Snoep\nPROJECTS\nMolecular Systems Biology\n\n____________________________________________________________________________\n\nTest quenching buffers(id: 171 | type: assays)\n\nDescription: We developed a new metabolomics protocol, which involved a comparison of different harvesting techniques, quenching solutions and extraction methods. Cell leakage and metabolite recovery was determined by ATP measurements\n\n\nINVESTIGATION\nIdentification of regulatory metabolites in the central pathway of the ABE fermentation\nPEOPLE\nJohn Raedts\nPROJECTS\nCOSMIC\nSTUDY\nDevelopement metabolomics protocol\nSUBMITTER\nJohn Raedts\n\n____________________________________________________________________________\n\nGrowth, by-product formation and glucose consumption rates and cAMP-concentrations under aerobic conditions(id: 1069 | type: data_files)\n\nDescription: Measurement of by-product formation rates, substrate consumption rates and extracellular concentrations of cAMP under aerobic batch conditions with glucose as substrate.\n\n\nASSAYS\nAerobic and anaerobic characterization of MG1655 and mutant strains with regard to growth, by-product formation and cAMP concentrations.\nCREATORS\nSonja Steinsiek\nINVESTIGATIONS\nAnalysis of the glucose transport in Escherichia coli under aerobic and anaerobic conditions\nPEOPLE\nSonja Steinsiek\nPROJECTS\nSUMO\nPUBLICATIONS\nGlucose Transport in Escherichia coli Mutant Strains with Defects in Sugar Transport Systems\nSTUDIES\nCharacterization of mutant strains with defects in sugar transport systems\nSUBMITTER\nSonja Steinsiek\n\n____________________________________________________________________________\n\nTranscriptomic Profiling of 14-3-3 Mutants(id: 41 | type: studies)\n\nDescription: Studies on the effect of deletion of genes encoding 14-3-3 proteins on the transcriptional response of yeast cells to potassium starvation.\n\n\nINVESTIGATION\nTRK1,2 Transport Systems of Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nFigure 4C: Biphasic control can provide mutant resistance to stem-cell homeostatic circuits.(id: 388 | type: studies)\n\nDescription: C A mutated stem cell with a strong inactivation of the sensing of y has a\ngrowth advantage (differentiates less), and therefore, it invades the stem- cell population. As a result, both the stem-cell pool and the number of terminally differentiated cells increase. \r\n\r\nSED-ML simulation\r\nhttps://jjj.bio.vu.nl/models/experiments/karin2017_fig4c/simulate\n\n\nASSAYS\nBiphasic control can provide mutant resistance to stem-cell homeostatic circuits.\nCREATORS\nJacky Snoep\nINVESTIGATION\nKarin et al (2017) Molecular Systems Biology\nPEOPLE\nJacky Snoep\nPROJECTS\nMolecular Systems Biology\nSUBMITTER\nJacky Snoep\n\n____________________________________________________________________________\n\nLarge-scale metabolome analysis and quantitative integration with genomics and proteomics data in Mycoplasma pneumoniae.(id: 336 | type: publications)\n\nDescription: missing\n\n\nDATA_FILES\nMetabolites all experiments, relative measurements\nAbsolute copy number per locus tag, M. pneumoniae at different times of growth\nPEOPLE\nNiels Zondervan\nLuis Serrano\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nSTUDIES\nTranscriptomics of M. pneumoniae at different times of growth\n\n____________________________________________________________________________\n\nStable membrane potential(id: 76 | type: assays)\n\nDescription: The membrane potential of cells grown overnight in a certain concentration of KCl will be determined, to reveal a relation between external KCl and a possibly stable membrane potential. See also \"2D-Gel Electrophoresis \" Assay for further details.\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nExperimental approaches to determine the dependence of volume, pH and membrane potential on external potassium\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nHands-on Model Management in SEEK: Example Paper(id: 249 | type: presentations)\n\nDescription: The paper by Tyson for the Hands-on Model Management in SEEK. Please refer to the related task sheet: https://fairdomhub.org/presentations/251\n\n\nCREATORS\nRon Henkel\nDagmar Waltemath\nPEOPLE\nRon Henkel\nDagmar Waltemath\nSUBMITTER\nRon Henkel\n\n____________________________________________________________________________\n\nAbsolute intracellular concentrations of metabolites extracted from T. b. brucei 427 under pH stress(id: 1023 | type: data_files)\n\nDescription: Metabolic changes of 26 intracellular metabolites extracted from T. b. brucei 427 wild type and arginine kinase knockout cells under high pH stress (pH8.7)\n\n\nASSAYS\nIntracellular metabolite concentrations in T. brucei under pH stress\nCREATORS\nDong-Hyun Kim\nINVESTIGATIONS\nMetabolite profiling, quantification and flux quantification\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSTUDIES\nTargeted metabolite analysis\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nTacita Nye(id: 303 | type: people)\n\nDescription: I am a first year PhD student, working with Professor Robert Poole (University of Sheffield), Professor Jeff Green (University of Sheffield) and Dr Jamie Wood (University of York) using a systems biology approach to study respiration in Escherichia coli.  \n\n\nINSTITUTIONS\nUniversity of Sheffield\nPROJECTS\nSUMO\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nTest created 1552505193451(id: 964 | type: assays)\n\nDescription: <br><br><strong>Related Publications</strong></strong><br>Me and Helena (2014). some book. <br>Moore A, Zielinski T, Millar AJ (2014). Online period estimation and determination of rhythmicity in circadian data, using the BioDare data infrastructure.. Methods in molecular biology (Clifton, N.J.). Retrieved from: doi.org/10.1007/978-1-4939-0700-7_2<br>Stein JM (1975). The effect of adrenaline and of alpha- and beta-adrenergic blocking agents on ATP concentration and on incorporation of 32Pi into ATP in rat fat cells.. Biochemical pharmacology. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/12<br><br><strong>Comments</strong></strong><p>\n\tNew comment after 2nd version</p>\n 2019-01-10 11:59:51 201 zielu tomasz.zielinski@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\r\n\tCommentsdfa</p>\r\n<br><br><strong>Originally submitted to PLaSMo on 2014-07-25 18:01:50</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nTomasz Zielinski\nINVESTIGATION\nZielinski, Tomasz\nPEOPLE\nTomasz Zielinski\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nProteomics data comparing LXX cultures cultivated on mineral and in continuous culture(id: 1926 | type: data_files)\n\nDescription: Maxquant output.\r\n\r\nProteomics data comparing LXX cultures cultivated on mineral and in continuous culture:\r\nmineral samples:\r\nLNU-LXX9-Si00-14B-P-B-Pr\r\nLNU-LXX9-Si00-14B-P-C-Pr\r\nLNU-LXX9-Si00-14B-P-D-Pr\r\nconitnuous culture samples:\r\nLNU-LXX9-Si00-CnA-P-B2-Pr\r\nLNU-LXX9-Si00-CnA-P-B3-Pr\r\nLNU-LXX9-Si00-CnA-P-B4-Pr\r\nLNU-LXX9-Si00-CnA-P-B5-Pr\r\nLNU-LXX9-Si00-CnA-P-B6-Pr\r\nLNU-LXX9-Si00-CnA-P-B7-Pr\n\n\nASSAYS\nProteomics\nCREATORS\nMohamed El Hajjami\nINVESTIGATIONS\nMulti -omics reveal lifestyle of acidophile, mineral-oxidizing model species Leptospirillum ferriphilumT\nPEOPLE\nMohamed El Hajjami\nPROJECTS\nSysMetEx\nSTUDIES\nOmics_data_analysis\nSUBMITTER\nMohamed El Hajjami\n\n____________________________________________________________________________\n\nPlasmid transformation lactococci(id: 90 | type: sops)\n\nDescription: Method for transformation of plasmids into Lactococcus lactis\n\n\nCREATORS\nMartijn Bekker\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\n\n____________________________________________________________________________\n\nDnaA and ORC: more than DNA replication initiators(id: 104 | type: publications)\n\nDescription: missing\n\n\nDATA_FILES\nRNA-seq wild type\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nGene expression in Trypanosoma brucei(id: 33 | type: investigations)\n\nDescription: Aim: To provide quantitative data that will allow modeling of gene expression for all enzymes of redox metabolism and the pentose phosphate pathway. Modeling will be used to predict enzyme levels based on the integration of an RNA degradation model with translation and protein degradation rates. \r\n\r\nPlan: The amounts of a protein in a cell can be determined by the rates of transcription, mRNA processing, translation, mRNA turnover and protein degradation. In trypanosomes analysis is simpler because the transcription rates of individual genes are (with few exceptions) not subject to control (Clayton and Shapira, 2007). This investigation will involve two main studies:\r\n\r\n1. Determination of the abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all enzymes of interest. For this goal RNA will be extracted at several times points of trypanosomes treated with Actinomycin D and Sinefungin, to inhibit RNAprocessing and transcription, and sequenced with Next Generation Sequencing (RNAseq).\r\n\r\n2. Determination of the rate of translation. This will be based on ribosomal loading: we will fractionate extracts on sucrose gradients and perform RNASeq across the polysome profile. From this we can predict protein synthesis rates.\n\n\nASSAYS\nmRNA decay assay\npre-mRNA processing rate\nModelling the gene expression cascade with length-dependent processes\nDATA_FILES\nmRNA decay\nHalf life values for enzymes of redox and polyamine pathways as provided by Alejandro\nmRNA half-lives\nprecursor mRNA half-lives\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSOPS\nSample preparation for mRNA decay study\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\r\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\r\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nNonlinear fitness landscape of a molecular pathway(id: 133 | type: publications)\n\nDescription: missing\n\n\nDATA_FILES\nRNA-seq wild type\nPEOPLE\nJan-Willem Veening\nJohannes Berg\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nGAPDH Kinetic characterisation(id: 197 | type: sops)\n\nDescription: SOP for measurement of GAPDH activity in extracts.\n\n\nASSAYS\nGAPDH\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nProtein mobility and diffusive barriers in Escherichia coli: consequences of osmotic stress(id: 121 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nBert Poolman\nJacek Mika\nPROJECTS\nKOSMOBAC\n\n____________________________________________________________________________\n\nEfficient, global-scale quantification of absolute protein amounts by integration of targeted mass spectrometry and two-dimensional gel-based proteomics(id: 166 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\nLeif Steil\nPraveen Kumar Sappa\nSandra Maass\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nMicrocavity arrays as an in vitro model system of the bone marrow niche for hematopoietic stem cells(id: 275 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nPEOPLE\nMaja Rey\nRainer Saffrich\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nSynthesis, release, and recapture of compatible solute proline by osmotically stressed Bacillus subtilis cells(id: 198 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLeif Steil\nTamara Hoffmann\nErhard Bremer\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nArabidopsis clock model P2011.3.1 - PLM_1041, version 1, SIMPLIFIED(id: 381 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_1041, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis clock model P2011.3.1 - PLM_1041, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis clock model P2011.3.1 - PLM_1041\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nGraph based storage and retrieval of computational models(id: 64 | type: presentations)\n\nDescription: Presented by Ron Henkel at SySMO PALs meeting 29-30 November 2012 in Heidelberg\n\n\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nArabidopsis clock models P2011.1.2 and P2011.2.1 - PLM_71, version 2, SIMPLIFIED(id: 391 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_71, version 2 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis clock models P2011.1.2 and P2011.2.1 - PLM_71, version 2\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis clock models P2011.1.2 and P2011.2.1 - PLM_71\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nEntrainment of heterogeneous glycolytic oscillations in single cells(id: 374 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nSTUDIES\nEntrainment of heterogeneous glycolytic oscillations in single cells\n\n____________________________________________________________________________\n\nRT-PCR Excel Template(id: 930 | type: data_files)\n\nDescription: An example of a JERM-compliant template for RT-PCR data\r\n\r\nThis template was taken from the GEO website (http://www.ncbi.nlm.nih.gov/geo/info/spreadsheet.html) and modified to conform to the SysMO-JERM (Just enough Results Model) for transcriptomics. \n\n\nASSAYS\nRT-PCR Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nJERM templates: overview & recent development.(id: 98 | type: presentations)\n\nDescription: Talk given by Olga Krebs at SysMO PALs meeting on 18 - 20 September 2013 in Paris\n\n\nCREATORS\nOlga Krebs\nEVENTS\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nSOP 7 Glucose-pulse experiments(id: 84 | type: sops)\n\nDescription: missing\n\n\nASSAYS\nGlucose pulsed L. lactis\nKinetic behavior of intracellular metabolites of E. faecalis upon a glucose-pulse\nKinetic behavior of intracellular metabolites of L. lactis upon a glucose-pulse\nCREATORS\nMartijn Bekker\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nThe Attic\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSTUDIES\nComparative modeling and phosphate dependence flux distributions and glucose consumption rates of two lactic acid bacteria S. pyogenes and L. lactis\nPre-liminary data from Martijn Bekker\n\n____________________________________________________________________________\n\nNachiket Vartak(id: 807 | type: people)\n\nDescription: missing\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nZucAt: FBA solution of the model under light growth conditions (IV)(id: 1338 | type: data_files)\n\nDescription: The solution of Flux Balance Analysis (FBA) represents metabolic flux distribution in the model under light growth conditions. In this solution, (i) the photorespiration was set to 0; and (ii) cyclic electron flow through FQR (ferredoxin-plastoquinone reductase) has been set of 0.1 of flow through FRN (ferredoxin-NADP oxidoreductase). Under this constraints, ATP is under-produced in plastid and therefore is additionally imported to cytoplasm. Flux through FQR represents cyclic electron flow through the light photosynthesis system. Flux through FRN represents non-cyclic electron flow through the light photosynthesis system.\n\n\nASSAYS\nFlux Balance Analysis of multi-compartment metabolic model of growing Arabidopsis thaliana\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nZucAt\nSTUDIES\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nMetabolic model of Sulfolobus solfataricus(id: 231 | type: models)\n\nDescription: Metabolic model of Sulfolobus solfataricus P2 in the SBML (sbml) and metano (txt, sce, fba) format. Scenarios are specific for growth on D-glucose or caseinhydrolysate as sole carbon source.\n\n\nASSAYS\nMetabolic modelling of S. solfataricus during growth on casaminoacids\nCREATORS\nHelge Stark\nINVESTIGATIONS\nAmino acid degradation in Sulfolobus solfataricus P2\nPEOPLE\nHelge Stark\nPROJECTS\nSulfoSys - Biotec\nPUBLICATIONS\nA systems biology approach reveals major metabolic changes in the thermoacidophilic archaeon Sulfolobus solfataricus in response to the carbon source L-fucose versus D-glucose\nSTUDIES\nComparison of Sulfolobus solfataricus P2 grown on caseinhydrolysate and D-glucose\nSUBMITTER\nHelge Stark\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\r\n\r\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nfermentation of C. acetobutylicum ptb::int(id: 1080 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nThomas Rimpf\nPEOPLE\nThomas Rimpf\nPROJECTS\nCOSMIC\nSUBMITTER\nThomas Rimpf\n\n____________________________________________________________________________\n\nMetabolic modeling and analysis of the metabolic switch in Streptomyces coelicolor(id: 7 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nDavid Hodgson\nElizabeth Wellington\nEriko Takano\nRainer Breitling\nM. Tauqeer Alam\nJay Moore\nMaria Elena Merlo\nPROJECTS\nSTREAM\n\n____________________________________________________________________________\n\ngustavsson5 model(id: 668 | type: assays)\n\nDescription: Model of glycolytic oscillations in individual yeast cells in microfluidic flow chamber\n\n\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATION\nGlycolytic oscillations in individual isolated yeast cells\nMODELS\ngustavsson5 model\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nSTUDY\nAllosteric regulation of phosphofructokinase controls the emergence of glycolytic oscillations in isolated yeast cells\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nQuantitative analysis of drug effects at the whole-body level: a case study for glucose metabolism in malaria patients(id: 269 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nKathleen Green\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\n\n____________________________________________________________________________\n\nLack of main K(+) uptake systems in Saccharomyces cerevisiae cells affects yeast performance in both potassium-sufficient and potassium-limiting conditions(id: 23 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nHana Sychrova\nJoaquin Ario\nJose Ramos\nLina Patricia Barreto Parra\nFalko Krause\nSilvia Petrezselyova\nJaromir Zahrdka\nClara Navarrete\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nRainer Breitling(id: 115 | type: people)\n\nDescription: I am currently Professor of Systems Biology at the University of Manchester. My research interests focus on the development of innovative computational approaches for post-genomic systems biology, statistical methods for high-throughput biological experimentation and the dynamic modelling of cellular systems. This work is highly interdisciplinary and usually involves close collaboration with experimental biologists and clinicians. A recurrently theme is the study of complex cellular networks at various molecular levels. Recently, a large part of my work has focused on exploiting high-accuracy mass spectrometry for metabolomics systems biology, for example in studies of the metabolic networks of protozoan parasites and the synthetic biology of antibiotic production in actinomycetes. \n\n\nDATA_FILES\nFermentor 199 transcriptomics\nINSTITUTIONS\nUniversity of Groningen\nINVESTIGATIONS\nMetabolism of Streptomyces coelicolor (SysMO STREAM)\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSTREAM\nSilicoTryp\nPUBLICATIONS\nMetabolic modeling and analysis of the metabolic switch in Streptomyces coelicolor\nThe dynamic architecture of the metabolic switch in Streptomyces coelicolor\nNew surveyor tools for charting microbial metabolic maps\nMetaNetter: inference and visualization of high-resolution metabolomic networks\nProbabilistic assignment of formulas to mass peaks in metabolomics experiments\nIncreasing the mass accuracy of high-resolution LC-MS data using background ions: a case study on the LTQ-Orbitrap\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nSTUDIES\nTimeseries 1\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2012 - PLM_70, version 1, SIMPLIFIED(id: 432 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_70, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis_clock_P2012 - PLM_70, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis_clock_P2012 - PLM_70\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nAndreas Kremling(id: 37 | type: people)\n\nDescription: I'm an engineer at the MPI Magdeburg and I'm working in the field of mathematical modeling, model verification, parameter identification, model analysis and experimental design. I'm involved in two projects, KOsmoBac and PSYSMO. \n\n\nINSTITUTIONS\nMax Planck Institute for Dynamics of Complex Technical Systems\nPROJECTS\nKOSMOBAC\nPSYSMO\n\n____________________________________________________________________________\n\nVilhelm Bohr(id: 1264 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nNational Institute on Aging\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nZucAt: the model documentation(id: 1337 | type: data_files)\n\nDescription: The ASCII file includes complete information on used transformers / compounds / genes and their inter-connection in the model. The Transformer information includes: Identifier name, Trivial name, Stoichiometric equation, Compartment, EC number, Pathway, Associated genes). The Compound information includes: Identifier name, Trivial name, Kegg ID, Compartment. The Gene information includes: ATG code of the genes whose products participate in transformations accounted in the model, Association with Identifier name of the Transformer.  \n\n\nASSAYS\nFlux Balance Analysis of multi-compartment metabolic model of growing Arabidopsis thaliana\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nZucAt\nSTUDIES\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nLF_proteomics_maxquant_perseus_filtered(id: 1802 | type: data_files)\n\nDescription: LFQ intensities from maxquant from combined search (mineral and continuous samples), filtered (at least 3 values in each condition), log-transformed\n\n\nASSAYS\nProteomics\nCREATORS\nMalte Herold\nAnsgar Poetsch\nMohamed El Hajjami\nINVESTIGATIONS\nMulti -omics reveal lifestyle of acidophile, mineral-oxidizing model species Leptospirillum ferriphilumT\nPEOPLE\nMalte Herold\nAnsgar Poetsch\nMohamed El Hajjami\nPROJECTS\nSysMetEx\nSTUDIES\nOmics_data_analysis\nSUBMITTER\nMalte Herold\n\n____________________________________________________________________________\n\nCell free extract activity measurements: L-fuc/d-glc(id: 310 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nJacqueline Wolf\nKatharina Kruse\nTheresa Kouril\nBettina Siebers\nDATA_FILES\nActivity assay L-fuc vs D-glc\nINVESTIGATION\nL-fucose degradation in Sulfolobus solfataricus P2\nORGANISMS\nSulfolobus solfataricus\nPEOPLE\nBettina Siebers\nTheresa Kouril\nKatharina Kruse\nJacqueline Wolf\nPROJECTS\nSulfoSys - Biotec\nSTUDY\nComparison of S. solfataricus grown on l-fucose and d-glucose\nSUBMITTER\nJacqueline Wolf\n\n____________________________________________________________________________\n\n FAIRDOM approach for semantic interoperability of systems biology data and models(id: 106 | type: presentations)\n\nDescription: Poster presented at SWAT4LS - Semantic web applications and tools for life science- in Berlin at 10 of december 2014 by Olga Krebs\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nBernd  Rinn\nWolfgang Mller\nQuyen Nguyen\nJacky Snoep\nStuart Owen\nNatalie Stanford\nPeter Kunszt\nEVENTS\nInternational SWAT4LS Workshop\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nQuyen Nguyen\nNatalie Stanford\nPeter Kunszt\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nLC-MS based absolute quantification of intracellular metabolites(id: 204 | type: assays)\n\nDescription: Intracellular metabolites in T. brucei at different stage of cell growth have been quantified absolutely by isotope ratio based MS technique using uniformly 13C-labelled E. coli extract. This is the case study for method development of absolute quantification for metabolic flux analysis.\n\n\nDATA_FILES\nAbsolute intracellular concentrations of metabolites extracted from T. b. brucei 427 wild type\nINVESTIGATION\nMetabolite profiling, quantification and flux quantification\nORGANISMS\nTrypanosoma brucei\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSOPS\nSample preparation for intracellular metabolite extraction in T. b. brucei 427\nSTUDY\nMetabolic flux measurement\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2011 - PLM_64, version 1(id: 838 | type: assays)\n\nDescription: \tThis model is termed P2011 and derives from the article: <strong>The clock gene circuit in Arabidopsis includes a repressilator with additional feedback loops</strong><strong>. </strong>Alexandra Pokhilko, Aurora Pias Fernndez, Kieron D Edwards, Megan M Southern, Karen J Halliday &amp; Andrew J Millar <em>Mol. Syst. Biol. </em>2012; 8: 574, submitted 9 Aug 2011 and published 6 March 2012. <a href=\"http://www.nature.com/msb/journal/v8/n1/synopsis/msb20126.html\">Link</a>\tLink to <a href=\"http://www.nature.com/msb/journal/v8/n1/extref/msb20126-s1.pdf\">Supplementary Information</a>, including equations. Minor errors in the published Supplementary Information are described in a file attached to version 1 of this model (the published SBML is correct).\t&nbsp;The model describes the circuit depicted in Fig. 1 of the paper (GIF attached). It updates the Pokhilko et al. 2010 model (termed P2010), <a href=\"http://www.plasmo.ed.ac.uk/plasmo/models/model.shtml?accession=PLM_6\">PLM_6</a>, by including:<ol><li>\t\tthe Evening Complex genes (ELF4, ELF3, LUX),</li><li>\t\tlight-regulated degradation of ELF3 by COP1,</li><li>\t\tTOC1 as a repressor rather than an activator of LHY/CCA1.</li></ol>\tThese changes allowed the removal of hypothetical components TOC1mod (or X) and Y from the earlier models. They also reveal that the central loop of the model is a triple-repressor ring oscillator, or 'repressilator' (illustrated in Fig. 8, GIF attached).\tSBML curation notes (please see Comments for each version): Compared to the model version submitted to the Biomodels database, version 1 here slightly alters the names of some variables and uses an SBML AssignmentRule for the light input. A MATLAB version is attached to version 1. In version 2, the rule is used with a generic SBML StepFunction (Adams et al. J. Biol. Rhythms Aug 2012) to describe the light-dark cycle more flexibly. Versions 1 and 2 have multiple sinks and sources created by CellDesigner. These are removed in versions 3 &amp; 4. Version 3 has the published lightfunction, and an equivalent COPASI file attached. Version 4 has the StepFunction.<br><br><strong>Related Publications</strong></strong><br>Alexandra Pokhilko, Aurora Pias Fernndez, Kieron D Edwards, Megan M Southern, Karen J Halliday & Andrew J Millar (2012). The clock gene circuit in Arabidopsis includes a repressilator with additional feedback loops.. Mol. Syst. Biol. 8: 574.. Retrieved from: http://www.nature.com/msb/journal/v8/n1/synopsis/msb20126.html<br><br><strong>Comments</strong></strong><p>\n\tVersion 2 of the model includes the StepFunction. Note that we have not yet removed the original light function parameter names, they are now non-functional.</p>\n 2012-06-01 17:29:46 3 amillar2 andrew.millar@ed.ac.uk<br><br><strong>Originally submitted to PLaSMo on 2012-03-07 12:15:12</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nAndrew Millar\nDATA_FILES\nFig. 1, outline of P2011 model, with P2010 inset,PLM_64_ 1\nFig. 8, cartoon illustrating repressilator structure,PLM_64_ 1\nODE file for Matlab version of P2011 clock model, PLM_64_ 1\nMatlab version of P2011 clock model, PLM_64_ 1\nDescribes minor errors in published Supplementary Information (published SBML is correct)., PLM_64_ 1\nINVESTIGATION\nMillar, Andrew\nMODELS\nArabidopsis_clock_P2011 - PLM_64, version 1, SUBMITTED\nArabidopsis_clock_P2011 - PLM_64, version 1, SIMPLIFIED\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nArabidopsis_clock_P2011 - PLM_64\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nFigure 2: Model calibration with time-resolved quantitative immunoblot data of mCFU-E cells and BaF3-EpoR cells.(id: 209 | type: studies)\n\nDescription: Growth-factor deprived mCFU-E cells (5x106 cells per condition) and BaF3-EpoR cells (1x107 cells per condition) were stimulated with different Epo doses and absolute concentrations were determined for pEpoR (B), pAKT (C), ppERK (D). The scale for pS6 (E) was estimated in arbitrary units. GTP-Ras (F) and ppERK were determined upon stimulation with indicated, color-coded Epo doses. pEpoR was analyzed by immunoprecipitation followed by immunoblotting, GTP-Ras was analyzed after pulldown using a fusion protein harboring GST fused to the Ras binding domain of Raf-1 followed by detection by quantitative immunoblotting. For pAkt and ppERK, cellular lysates were subjected to quantitative immunoblotting. Calibrator proteins were used for EpoR, AKT, GTP- Ras and ERK to facilitate the conversion to nM concentrations. \r\n\r\nSED-ML simulations\r\nFig. 2B - 2E: https://jjj.bio.vu.nl/models/experiments/adlung2017_fig2bto2e/simulate\r\nFig. 2F: https://jjj.bio.vu.nl/models/experiments/adlung2017_fig2f/simulate\r\nFig. 2G: https://jjj.bio.vu.nl/models/experiments/adlung2017_fig2g/simulate\n\n\nASSAYS\nSource data for Figure 2: Experimental time-resolved quantitative immunoblot data\nMathematical models of the Epo-induced AKT, ERK and S6 activation.\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nDATA_FILES\nSource data for Figure 2B\nSource data for Figure 2C\nSource data for Figure 2D\nSource data for Figure 2E\nSource data for Figure 2F BaF3\nSource data for Figure 2F mCFU-E\nSource data for Figure 2G BaF3\nSource data for Figure 2G mCFU-E\nINVESTIGATION\nAdlung et al (2017) Molecular Systems Biology\nMODELS\nBaF3 model\nmCFU-E model\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nMolecular Systems Biology\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nClosTron Mutagenesis(id: 165 | type: sops)\n\nDescription: A refined and streamlined procedure to generate mutant in a wide range of different clostridial species, using group II intron retargeting methodologies. \n\n\nCREATORS\nYing Zhang\nNigel Minton\nPEOPLE\nNigel Minton\nYing Zhang\nPROJECTS\nCOSMIC\nSUBMITTER\nYing Zhang\n\n____________________________________________________________________________\n\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation(id: 143 | type: studies)\n\nDescription: we describe a multi-compartmental model consisting of a mesophyll cell with plastid and mitochondrion, a phloem cell, as well as a root cell with mitochondrion. In this model, the phloem was considered as a non-growing transport compartment, the mesophyll compartment was considered as both autotrophic (growing on CO2 under light) and heterotrophic (growing on starch in darkness), and the root was always considered as heterotrophic tissue completely dependent on sucrose supply from the mesophyll compartment. In total, the model includes 380 balanced compounds interconnected by 372 transformers. The structured metabolic model accounts for central carbon metabolism, photosynthesis, photorespiration, carbohydrate metabolism, energy and redox metabolisms, proton metabolism, biomass growth, nutrients uptake, proton gradient generation and sucrose translocation between tissues. Biochemical processes in the model were associated with gene-products (742 ORFs) which were validated using mass-spectrometry based proteomic data. Flux Balance Analysis (FBA) of the model resulted in balanced carbon, proton, energy and redox states under both light and dark conditions. Based on the FBA, the model predictes that in light conditions the plastid produced enough ATP to serve the Calvin-Benson cycle and starch formation, and the excess of photosynthetic potential (in form of reducing equivalents) was exported to the mitochondria via malate/oxaloacetate shuttle for ATP synthesis. The main H+-fluxes were reconstructed and sucrose translocation matched with them. \n\n\nASSAYS\nFlux Balance Analysis of multi-compartment metabolic model of growing Arabidopsis thaliana\nCREATORS\nMaksim Zakhartsev\nOlga Krebs\nDATA_FILES\nZucAt: FBA constraints for light conditions\nZucAt: FBA constraints for dark conditions\nZucAt: FBA solution of the model under dark growth conditions\nZucAt: The stoichiometric matrix of the model\nZucAt: The compound database\nZucAt: The gene database\nZucAt: The transformers database\nZucAt: FBA solution of the model under light growth conditions (I)\nZucAt: FBA solution of the model under light growth conditions (II)\nZucAt: FBA solution of the model under light growth conditions (III)\nZucAt: the model documentation\nZucAt: FBA solution of the model under light growth conditions (IV)\nZucAt: FBA solution of the model under light growth conditions (V)\nINVESTIGATION\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nPROJECTS\nZucAt\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nPreparation of cell extracts of recombinant E.coli(id: 188 | type: sops)\n\nDescription: Preparation of cell free extracts of the recombinant E. coli strains expressing the gluconeogenic S. solfataricus enzymes.\n\n\nASSAYS\nPGK\nGAPDH\nTPI\nFBPAase\nCREATORS\nJacky Snoep\nTheresa Kouril\nINVESTIGATIONS\nCentral Carbon Metabolism of Sulfolobus solfataricus\nPEOPLE\nJacky Snoep\nTheresa Kouril\nPROJECTS\nSulfoSys\nSTUDIES\nModel Gluconeogenesis\nSUBMITTER\nJacky Snoep\n\n____________________________________________________________________________\n\nSOP - Bioleaching in flasks (LNU)(id: 239 | type: sops)\n\nDescription: Initial configuration and parameters of the Bioleaching\n\n\nASSAYS\nExperimental methods\nCREATORS\nStephan Christel\nINVESTIGATIONS\nMulti -omics reveal lifestyle of acidophile, mineral-oxidizing model species Leptospirillum ferriphilumT\nPEOPLE\nStephan Christel\nPROJECTS\nSysMetEx\nSTUDIES\nOmics_data_analysis\nSUBMITTER\nStephan Christel\n\n____________________________________________________________________________\n\nSOP for cultivation of B.Subtilis(id: 83 | type: sops)\n\nDescription: SOP for shake flask cultivation of B.Subtilis in Bacell-Sysmo \n\n\nASSAYS\nIPTG induction of sigmaB in BSA115\nTiling Array analysis of glucose strarved B. subtilis cells\nCREATORS\nPraveen Kumar Sappa\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSTUDIES\nControlled sigmaB induction in shake flask\nTranscriptome analysis of glucose starvation in B. subtilis\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nMalate-mediated carbon catabolite repression in Bacillus subtilis involves the HPrK/CcpA pathway(id: 186 | type: publications)\n\nDescription: missing\n\n\nDATA_FILES\nProtein report on identification of interaction partners of CitG in a SPINE experiment\nCatabolite repression of -xylosidase by different carbon sources in various mutants\nPEOPLE\nJoerg Stuelke\nLeif Steil\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nATP levels under 10 mM benzoic acid stress(id: 49 | type: studies)\n\nDescription: To see changes in ATP levels in cells with induced ABC transporters. Cells with Pdr12 pump by 10 mM benzoic acid are used.\n\n\nASSAYS\nATP measurement under 10mM benzoic acid stress\nINVESTIGATION\nEffect of Benzoic Acid on ATP Levels\nPEOPLE\nMartin Valachovic\nPROJECTS\nMOSES\nSOPS\nATP determination with ROCHE kit  ATP Bioluminescence Assay Kit CLS II\nSUBMITTER\nMartin Valachovic\n\n____________________________________________________________________________\n\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes(id: 226 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nBarbara Bakker\nJurgen  Haanstra\nPROJECTS\nSilicoTryp\n\n____________________________________________________________________________\n\nT-box-mediated control of the anabolic proline biosynthetic genes of Bacillus subtilis(id: 150 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLeif Steil\nTamara Hoffmann\nErhard Bremer\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nMetabolome Data(id: 1117 | type: data_files)\n\nDescription: SysMo2:\r\nIntra- and extracellular metabolome data of the chemostat experiments: nitrogen limitation, nitrogen limitation+ NaCl, nitrogen limitation + glucose\r\n \r\n\r\n\n\n\nCREATORS\nHanna Meyer\nPEOPLE\nHanna Meyer\nPROJECTS\nBaCell-SysMO\nSUBMITTER\nHanna Meyer\n\n____________________________________________________________________________\n\nMOSES(id: 8 | type: projects)\n\nDescription: MOSES (Micro Organism Systems biology: Energy and Saccharomyces cerevisiae) develops a new Systems Biology approach, which is called 'domino systems biology'. It uses this to unravel the role of cellular free energy ('ATP') in the control and regulation of cell function. MOSES operates though continuous iterations between partner groups through a new systems-biology driven data-management workflow. MOSES also tries to serve as a substrate for three or more other SYSMO programs. \n\n\nASSAYS\nCellular size and granularity during glucose pulse\nSteady state extracellular fluxes in anaerobic yeast Saccharomyces cerevisiae\nSteady state concentrations of extracellular metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nSteady state concentrations of intracellular metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nBiomass weight during glucose pulse\nDynamics of extracellular metabolites during glucose pulse\nDynamics of intracellular metabolites during glucose pulse\nDynamics of macromolecules during glucose pulse\nCompare ATP extraction methods\nATP measurement under 10mM benzoic acid stress\nATP measurement under 30mM benzoic acid stress\nATP measurement under 50mM benzoic acid stress\nMOSES: dynamic model of glucose pulse\nDATA_FILES\nMeasured steady state metabolic fluxes\nMeasured steady state concentrations of extracellular metabolites in yeast Saccharomyces cerevisiae anaerobic chemostat\nMeasured steady state concentrations of intracellular metabolites in yeast Saccharomyces cerevisiae anaerobic chemostat\nCellular size and granularity during glucose pulse\nBiomass weight during glucose pulse\nOff-gas monitoring during glucose pulse\nDynamics of extracellular metabolites during glucose pulse\nDynamics of intracellular metabolites during glucose pulse\nDynamics of macromolecules during glucose pulse\nINSTITUTIONS\nMedical University Vienna\nVU University Amsterdam\nUniversity of Heidelberg\nManchester Centre for Integrative Systems Biology, University of Manchester\nUniversity of Stavanger\nUniversity of Stuttgart\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nSteady state metabolic fluxes and metabolite concentrations in anaerobic chemostat of yeast Saccharomyces cerevisiae\nEffect of Benzoic Acid on ATP Levels\nMODELS\nSBML model of yeast central carbon metabolism\nLAB with ATP-driven glucose import of yeast\nORGANISMS\nSaccharomyces cerevisiae\nPEOPLE\nJacky Snoep\nHans Westerhoff\nAna Kitanovic\nFemke Mensonides\nHanan Messiha\nKarl Kuchler\nMaksim Zakhartsev\nMalkhey Verma\nMartin Valachovic\nPeter Ruoff\nRalf Steuer\nStefan Wlfl\nWalter Glaser\nMatthias Reuss\nKieran Sharkey\nSteve Wilkinson\nPedro Mendes\nCornelia Klein\nFred Boogerd\nEttore Murabito\nPRESENTATIONS\nMy.PAL history & experience\nPROGRAMMES\nSysMO\nPUBLICATIONS\nComputer controlled automated assay for comprehensive studies of enzyme kinetic parameters\nSystems biology: the elements and principles of life\nSimplified absolute metabolite quantification by gas chromatographyisotope dilution mass spectrometry on the basis of commercially available source material\nMiniaturized device for agitating a high-density yeast suspension that is suitable for in vivo nuclear magnetic resonance applications\nDomino systems biology and the A of ATP\nMetabolic efficiency in yeast Saccharomyces cerevisiae in relation to temperature dependent growth and biomass yield\nFast sampling for quantitative microbial metabolomics: new aspects on cold methanol quenching: metabolite co-precipitation\nCell size and morphological properties of yeast Saccharomyces cerevisiae in relation to growth temperature\nSOPS\nIsolation of total RNA from yeast Saccharomyces cerevisiae for microarrays\nAnaerobic media composition\nSampling of biomass\nPerturbation conditions\nDetermination of intracellular metabolites\nDetermination of (cytosolic) enzyme activities\nYeast strains\nDetermination of the kinetic parameters of the purified glycolytic isoenzymes of Saccharomyces cerevisiae\nATP determination with ROCHE kit  ATP Bioluminescence Assay Kit CLS II\nAssays for measuring the activities of the individual glycolytic isoenzymes of Saccharomyces cerevisiae\nGeneral protocol for measuring the effects of metabolites (including ATP, ADP, AMP, and cAMP) on the activity of glycolytic isoenzymes\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSteady state fluxes in yeast Saccharomyces cerevisiae in anaerobic chemostat\nSteady state concentrations of metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nTest different ATP extraction protocols and conditions\nATP levels under 10 mM benzoic acid stress\nATP levels under 30 mM and 50 mM benzoic acid stress\nATP levels after benzoic acid pretreatments\nCell survival under different benzoic acid concentrations\n\n____________________________________________________________________________\n\nLocke2006_CircadianClock_tanh - PLM_10, version 1, SIMPLIFIED(id: 474 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_10, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nLocke2006_CircadianClock_tanh - PLM_10, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nLocke2006_CircadianClock_tanh - PLM_10\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nVitor Martins Dos Santos(id: 77 | type: people)\n\nDescription: My research activities has been to use mathematical models and Computational Biology to answer biological questions, intertwining in silico and experimental methods at all stages. I have a strong interest in exploring the interfaces between Fundamental Biology and bona fide Engineering, specifically in the realm of environmental and industrial problems. The research goals of my group are to contribute to the elucidation of mechanisms underlying basic cellular processes, evolution and ecological interactions of microbes, and to translate this knowledge into Synthetic Biology applications of biomedical, biotechnological and environmental interest.\n\n\nDATA_FILES\nList of recent publications & patents\nINSTITUTIONS\nHelmholtz Centre for Infection Research Braunscheig\nWageningen University & Research\nPROJECTS\nPSYSMO\nGenoSysFat\nDigiSal\nHUMET Startup\nEmPowerPutida\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nSAFE-Aqua\nINDIE - Biotechnological production of sustainable indole\nPUBLICATIONS\nGenome-scale reconstruction and analysis of the Pseudomonas putida KT2440 metabolic network facilitates applications in biotechnology\nThe metabolic response of P. putida KT2442 producing high levels of polyhydroxyalkanoate under single- and multiple-nutrient-limited growth: highlights from a multi-level omics approach\nReconciliation of Genome-Scale Metabolic Reconstructions for Comparative Systems Analysis\n\n____________________________________________________________________________\n\nRNA isolation and quality check by Agilent bioanalyser(id: 51 | type: sops)\n\nDescription: Cells were harvested from culture keeping the cells cold to quench the physiological condition of RNA and the cells were mechanically disrupted. RNA was isolated from the cells by conventional acid-phenol method and the quality was checked by Agilent bioanalyser.\n\n\nASSAYS\nGene expression(Transcriptome)\nTranscriptome data for chemostat cultivated samples\nTranscritome data_Identification of targets of the essential RNase Y of Bacillus subtilis\nCREATORS\nPraveen Kumar Sappa\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nMultiomics study of Bacillus subtilis under osmotic stress\nIdentification of targets of the essential RNase Y of Bacillus subtilis\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSTUDIES\nBatchfermentation exp-starv01_090204\nB. subtilis_SysMo2_Chemostat_growthrate-salt\nIdentification of targets of the essential RNase Y of Bacillus subtilis\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nCarbon source control of the phosphorylation state of the Bacillus subtilis carbon-flux regulator Crh in vivo(id: 159 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoerg Stuelke\nLeif Steil\nWolfgang Hillen\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nPhotosynthate partitioning to starch in Arabidopsis thaliana is insensitive to light intensity but sensitive to photoperiod due to a restriction on growth in the light in short photoperiods.(id: 372 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nAndrew Romanowski\nVirginie Mengin\nPROJECTS\nPHYTOCAL: Phytochrome Control of Resource Allocation and Growth in Arabidopsis and in Brassicaceae crops\n\n____________________________________________________________________________\n\nStable pH(id: 75 | type: assays)\n\nDescription: The pH of cells grown overnight in a certain concentration of KCl will be determined, to reveal a functional relationship between external KCl and a stable pH. See also \"2D-Gel Electrophoresis\" Assay for further details.\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nExperimental approaches to determine the dependence of volume, pH and membrane potential on external potassium\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nSysMO-DB2 - Review and Update(id: 85 | type: presentations)\n\nDescription: The talk presented by Carole Goble and Wolfgang Mueller at the SysMO evaluation conference 2013 in Berlin.\n\n\nCREATORS\nStuart Owen\nCarole Goble\nWolfgang Mller\nEVENTS\nSysMO Evaluation 2013\nPEOPLE\nCarole Goble\nStuart Owen\nWolfgang Mller\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nPresentation Krzysztof Gizynski(id: 92 | type: presentations)\n\nDescription: missing\n\n\nEVENTS\nCOSMIC Final Meeting 2013\nPEOPLE\nSebastian Curth\nPROJECTS\nCOSMIC\nSUBMITTER\nSebastian Curth\n\n____________________________________________________________________________\n\nHenrik Strahl Von Schulten(id: 295 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Newcastle\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nCovering a broad dynamic range: information processing at the erythropoietin receptor(id: 284 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nData Management and Data Standardization\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nControl of cell division in Streptococcus pneumoniae by the conserved Ser/Thr protein kinase StkP(id: 143 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nLive Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy.\nDATA_FILES\nRNA-seq wild type\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nSulfosys - SOP fermentation(id: 72 | type: sops)\n\nDescription: SOPs related with medium composition, fermentation and stocking of S. solfataricus.\n\n\nASSAYS\nFermentation of S. solfataricus at 70 and 80C in a batch fermenter\nCREATORS\nPawel Sierocinski\nINVESTIGATIONS\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature\nPEOPLE\nPawel Sierocinski\nPROJECTS\nSulfoSys\nSTUDIES\nPilot experiment - S. solfataricus grown at 70 and 80 C.\nSUBMITTER\nPawel Sierocinski\n\n____________________________________________________________________________\n\nContious growth_16C_SMM_Transcritome_Nimblegen(id: 93 | type: data_files)\n\nDescription: missing\n\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "\nASSAYS\nTiling array analysis of continuous growth stress conditions in SMM\nCREATORS\nPraveen Kumar Sappa\nINVESTIGATIONS\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSTUDIES\nTranscriptome of continuously stressed B. subtilis\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nGlycolytic oscillations in individual isolated yeast cells(id: 213 | type: investigations)\n\nDescription: missing\n\n\nASSAYS\ngustavsson5 model\ngustavsson1-4 models\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nMODELS\ngustavsson5 model\ngustavsson1-4 models\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nPUBLICATIONS\nAllosteric regulation of phosphofructokinase controls the emergence of glycolytic oscillations in isolated yeast cells\nHeterogeneity of glycolytic oscillatory behaviour in individual yeast cells\nSustained glycolytic oscillations in individual isolated yeast cells\nSTUDIES\nAllosteric regulation of phosphofructokinase controls the emergence of glycolytic oscillations in isolated yeast cells\nHeterogeneity of glycolytic oscillatory behaviour in individual yeast cells\nSustained glycolytic oscillations in individual isolated yeast cells\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nRightField: embedding ontology annotation in spreadsheets(id: 139 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nAffy Transcriptomics Templates\nDATA_FILES\n3' or Whole Gene Expression Array Template (Affymetrix)\nChip-chip Array Template (Affymetrix)\nPEOPLE\nJacky Snoep\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nFranco Du Preez\nPRESENTATIONS\nRightField presentation for Virtual Liver Network\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nCalculation of amino acid uptake rates(id: 1463 | type: data_files)\n\nDescription: The calculation of amino acid uptake rates for cells of Sulfolobus solfataricus P2 grown on caseinhydrolysate was performed based on the relative consumption of individual amino acids from the growth medium and the previously published absolute concentration of amino acids in the used growth medium.\n\n\nASSAYS\nMetabolome analysis: Casaminoacids versus D-Glc\nMetabolic modelling of S. solfataricus during growth on casaminoacids\nCREATORS\nHelge Stark\nINVESTIGATIONS\nAmino acid degradation in Sulfolobus solfataricus P2\nPEOPLE\nHelge Stark\nPROJECTS\nSulfoSys - Biotec\nSTUDIES\nComparison of Sulfolobus solfataricus P2 grown on caseinhydrolysate and D-glucose\nSUBMITTER\nHelge Stark\n\n____________________________________________________________________________\n\nPlaSMo project description(id: 36 | type: documents)\n\nDescription: <h3><a name=\"Plasmowebpages-WhatisPlaSMo?\"></a>What is PlaSMo?</h3>\r\n<p>PlaSMo stands for Plant Systems-biology Modelling; Ensuring the achievements of yesterday's Mathematical Modellers will be available for the Systems Biologists of tomorrow.\r\n</p>\r\n\r\n<h4>The original PlaSMo project description is included within the attached document.</h4>\r\n\r\n<h3><a name=\"Plasmowebpages-Ouraims\"></a>Our aims</h3>\r\n<ul>\r\n\t<li>To identify plant mathematical models useful to the UK plant systems biology community, which are currently in a variety of legacy formats and in danger of being lost</li>\r\n\r\n\t<li>To represent these models in a declarative XML-based format, which is closer to the systems biology standard SBML</li>\r\n\t<li>To evaluate the behaviour of the models represented in this format against the output of the original models</li>\r\n\t<li>To produce a portal to disseminate the models, and examples of their output, in XML format; so that they can be used in the programming language and computating environment of a scientist's choice</li>\r\n\t<li>To provide an interface to enable researchers to share reviews, annotations and enhancements of models</li>\r\n</ul>\r\n\r\n\r\n<h3><a name=\"Plasmowebpages-Targetaudience\"></a>Who is it for?</h3>\r\n\r\n<ul>\r\n\t<li>Plant scientists wishing to apply systems biology approaches to the study of plant growth and development</li>\r\n\t<li>Crop scientists and the plant breeding and agricultural industries</li>\r\n\t<li>Current and retired mathematical modellers wishing to safeguard their models and make them more widely available</li>\r\n\t<li>Those researching plant responses to climate change</li>\r\n\t<li>Those needing to model the growth of new crops or new uses of existing crops, such as bioenergy production</li>\r\n</ul>\r\n\r\n\r\n<h3><a name=\"Plasmowebpages-Ourfunders\"></a>Our funders</h3>\r\n\r\n<p>PlaSMo was establised with funding from the BBSRC's <a href=\"http://www.bbsrc.ac.uk/funding/opportunities/2008/bioinformatics_biological_resources.html\" rel=\"nofollow\">Bioinformatics and Biological Resources</a> initiative and is maintained with funding from the European Commission's FP7 Collaborative Project <a href=\"http://www.timing-metabolism.eu\" rel=\"nofollow\">TiMet</a>.</p>\r\n\r\n\r\n<h3><a name=\"Plasmowebpages-Peopleandcollaboratinginstitutions\"></a>People and collaborating institutions</h3>\r\n\r\n<p>At Aberystwyth University's <a href=\"http://www.aber.ac.uk/en/ibers/\" rel=\"nofollow\">Institute of Biological, Environmental and Rural Sciences (IBERS)</a></p>\r\n<ul>\r\n\t<li>Helen Ougham</li>\r\n\t<li>Chris Davey</li>\r\n</ul>\r\n\r\n<p>At the University of Edinburgh's <a href=\"http://csbe.bio.ed.ac.uk/\" rel=\"nofollow\">Centre for Systems Biology (CSBE)</a></p>\r\n<ul>\r\n\t<li>Andrew Millar</li>\r\n\t<li>Chris Tindal</li>\r\n        <li>Tomasz Zielinski</li>\r\n\t<li>BioDare2 - <a href=\"https://biodare2.ed.ac.uk\">period analysis service for circadian data</a></li>\r\n</ul>\r\n\r\n\r\n<p>At <a href=\"http://www.simulistics.com\" rel=\"nofollow\">Simulistics Ltd, Edinburgh</a></p>\r\n\r\n<ul>\r\n\t<li>Robert Muetzelfeldt</li>\r\n\t<li>Jonathan Massheder</li>\r\n\t<li>Jasper Taylor</li>\r\n\t<li>Mark Muetzelfeldt</li>\r\n</ul>\r\n\r\n\n\n\nCREATORS\nBioData SynthSys\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nLocalization of general and regulatory proteolysis in Bacillus subtilis cells(id: 57 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nUwe Voelker\nLeendert Hamoen\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nCharacterization of flux distribution and general physiology of S. pyogenes in continuous cultures(id: 599 | type: data_files)\n\nDescription: S. pyogenes was grown in C-limited cultures at pH 6.7 and 7.5 and at a growth rate of 0.05 and 0.15\n\n\nASSAYS\nPhysiological characterization of Lactic acid bacteria grown in C-limited continuous cultures\nCREATORS\nMartijn Bekker\nTomas Fiedler\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nMartijn Bekker\nTomas Fiedler\nPROJECTS\nSysMO-LAB\nPUBLICATIONS\nPhysiological characterization of three lactic acid bacteria and their isogenic ldh deletion mutants shows that these organisms are optimized for Yatp and not ATP formation rates at their physiological pH\nSTUDIES\nStudy of the physiological characterization of three lactic acid bacteria and their isogenic ldh deletion mutants\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nPK Kinetic characterisation(id: 201 | type: sops)\n\nDescription: SOP for measurement of PK activity in extracts.\n\n\nASSAYS\nPK\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nMOA2010-05_3C_sRNAomics_bioinformatics_PHAS-loci(id: 2787 | type: data_files)\n\nDescription: PHAS loci identified by genome and transcriptome-wide phasing analysis. List of genome locations for the all identified non-coding PHAS loci or transcript identifiers (IDs) for protein-coding PHAS loci together with the lengths of their producing phasiRNAs. For the protein-coding PHAS loci, transcript full description, protein domains (obtained from PFAM database; Finn et al. 2016), MapMan ontology annotation (from GoMapMan;(Ramak et al. 2014)) and log2 ratio of gene expression between PVYNTN-infected and mock samples measured in Dsire and NahG-Dsire plants 3 dpi were given. Yellow  upregulated; blue  downregulated;  statistically not significant (FDR p > 0.05); NA  not analyzed. # - non-coding loci do not have MapMan ontology assignments. D  Dsire; NahG  NahG-Dsire; M  mock; P  PVYNTN.\n\n\nASSAYS\nsRNAomics (bioinformatics)\nINVESTIGATIONS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nPEOPLE\nZiva Ramsak\nPROJECTS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nSTUDIES\nMOA2010-05\nSUBMITTER\nZiva Ramsak\n\n____________________________________________________________________________\n\nCOSMIC2: Growth of model-driven Clostridium acetobutylicum mutant strains using standardized chemostat culture conditions(id: 72 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nThomas Rimpf\nPEOPLE\nThomas Rimpf\nPROJECTS\nCOSMIC\nSUBMITTER\nThomas Rimpf\n\n____________________________________________________________________________\n\nLocalization of Arl1-GFP(id: 883 | type: data_files)\n\nDescription: Confocal microscopy of Arl1-GFP cells after 60 min at 0 or 50 mM KCl or at 1 M NaCl.\n\n\nCREATORS\nPaul Heusden\nPEOPLE\nPaul Heusden\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nPaul Heusden\n\n____________________________________________________________________________\n\nSOP 8 Glucose-pulse experiments (adapted for S. pyogenes)(id: 85 | type: sops)\n\nDescription: missing\n\n\nASSAYS\nGlucose pulsed S. pyogenes\nCREATORS\nMartijn Bekker\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSTUDIES\nComparative modeling and phosphate dependence flux distributions and glucose consumption rates of two lactic acid bacteria S. pyogenes and L. lactis\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\r\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nSMC is recruited to oriC by ParB and promotes chromosome segregation in Streptococcus pneumoniae(id: 127 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nLive Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy.\nDATA_FILES\nRNA-seq wild type\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nETCmutants_metabolites(id: 1127 | type: data_files)\n\nDescription: by-product formation rates and glucose uptake rates of mutants with linear electron transport chain at different aerobiosis levels\n\n\nASSAYS\nDetermination of by-product formation and glucose uptake of mutants with linear respiratory chain at different aerobiosis levels\nCREATORS\nKatja Bettenbrock\nINVESTIGATIONS\nAnalysis of Escherichia coli with linear electron transport chain\nPEOPLE\nKatja Bettenbrock\nPROJECTS\nSUMO\nSTUDIES\nAnalysis of Escherichia coli strains with linear respiratory chain\nSUBMITTER\nKatja Bettenbrock\n\n____________________________________________________________________________\n\nCarolyn Alia Joko(id: 540 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nbo Akademi University\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nSeverin Ehret(id: 547 | type: people)\n\nDescription: I am working at the boundary of wet-labs and mathematical modeling, trying to understand the regulation of the eukaryotic cell cycle. I started working with yeast in a group of Systems Biology (Prof. Edda Klipp, Humboldt University, Berlin). Before that I studied biology, engineering and did research on the nuclear pore complex.\n\n\nINSTITUTIONS\nHumboldt-Universitt zu Berlin\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\n090714 chemostat overview fluxes & quinones wt(id: 57 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nAlexander Ter Beek\nKlaas Hellingwerf\nPEOPLE\nAlexander Ter Beek\nKlaas Hellingwerf\nPROJECTS\nSUMO\n\n____________________________________________________________________________\n\nAlvaro Martinez Guimera(id: 1256 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Newcastle\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nArabidopsis clock model P2011.4.1 - PLM_1042, version 1, SIMPLIFIED(id: 383 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_1042, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis clock model P2011.4.1 - PLM_1042, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis clock model P2011.4.1 - PLM_1042\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nStable volume(id: 77 | type: assays)\n\nDescription: The volume of cells grown overnight in a certain concentration of KCl will be determined, to reveal a relation between the osmotic effects of external KCl and the cell volume. See also 1.4.1. for further details.\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nExperimental approaches to determine the dependence of volume, pH and membrane potential on external potassium\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nExternal Potassium Concentration(id: 64 | type: assays)\n\nDescription: External concentration changes under the conditions of Navarrete et al. (2010) will be estimated from the internal concentration changes and the volume ratio of cell sample to medium.\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nKey Cell-Physiological Parameters\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nColin Harwood(id: 282 | type: people)\n\nDescription: Optimisation of Bacillus subtilis for the secretion of heterologous proteins Therapeutic proteins (including those required for experimental purposes and clinical trials) are major products of biomanufacturing processes and considerable time and expense are expended to maximise the yield and quality of proteins produced in heterologous hosts. The production host of choice is the Gram-negative bacterium Escherichia coli for which many strains and expression systems have been developed. However, one size does not fit all: E. coli is not suitable for the production of many proteins, either because it is not able to carry out appropriate post-translational modifications (e.g. glycosylation) or because it does not facilitate their folding into a native (i.e. functional) configuration. The former can be overcome by use of more expensive eukaryotic host production systems, while the latter can often be overcome by secreting proteins from the cytoplasm. Secretion has three major potential advantages over intracellular accumulation: the secreted target protein is usually natively folded; yields can be as high or higher than that obtained from intracellular E. coli host/vector systems; there is a reduced requirement for expensive extraction and purification procedures, with reduced risks of contamination with host proteins and nucleic acids.\r\n\r\nAlthough various microbial protein secretion systems have been developed, all currently have limitations that restrict their use and value as a tool for the Biomanufacturing industry. These systems include secretion into the periplasm of E. coli and subsequent selective release and purification; production in the yeast Pichia pastoris; production in Bacillus subtilis and close relatives. In all cases, the yields are relatively low compared with that of the intracellular E. coli systems (mg rather than g per litre).\r\n\r\nBacillus subtilis is widely used for the commercial production of industrial enzymes, particularly for the food and detergents industries. Its advantage is its capacity to secrete native Bacillus proteins into the culture medium at concentrations in excess of ten of grams per litre. However, experience with the use of this bacterium for the production of heterologous proteins has been mixed with, in most cases, generally low yields. The aim of this proposal is to exploit knowledge of the genomics and systems biology of B. subtilis to develop this bacterium as a host/vector system for the production of high quality heterologous proteins for clinical trials and for high-throughput X-ray crystallographic studies. The aim is not to replace E. coli as the primary host for heterologous protein production, but to expand the range of host/vector systems that are available for problematical proteins.\r\n\r\nResponse of Bacillus anthracis to stress in the environment and in host cells Infectious disease is the culmination of a battle between the pathogen and its host. Central to the hosts innate defence mechanism are phagocytic cells; macrophages and dendritic cells. Phagocytes express a series of antibacterial factors, including reactive oxygen and nitrogen species, antibacterial peptides and enzymes. They can also impose increased acidification. In spite of these defence mechanisms, several clinically important pathogens can proliferate inside macrophages. The interaction between B. anthracis and host macrophages is central to its pathogenesis. However, an understanding of the cellular and molecular interactions between B. anthracis and the immune system is far from complete. B. anthracis avoids and even breaches cells of the immune system: vegetative cells avoid macrophages by virtue of their possession of a poly-D-glutamic acid capsule, while engulfment of spores by alveolar macrophages and their subsequent efficient germination within the macrophage is an important component of its ability to cause infection. However, it is not clear if B. anthracis can multiply in macrophages. We are dissecting the B. anthracis-macrophage relationship through a combined cellular, genetic and molecular approach.\n\n\nDATA_FILES\nThe condition-dependent transcriptome of Bacillus subtilis 168\nINSTITUTIONS\nUniversity of Newcastle\nPRESENTATIONS\nBiophysical analysis of putative interactions between TCA cycle enzymes of B. subtilis\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nCondition-dependent transcriptome reveals high-level regulatory architecture in Bacillus subtilis\nEnvironmental salinity determines the specificity and need for Tat-dependent secretion of the YwbN protein in Bacillus subtilis\nDissection of the network of interactions that links RNA processing with glycolysis in the Bacillus subtilis degradosome\n\n____________________________________________________________________________\n\nArabidopsis clock model P2011.6.1 - PLM_1044, version 1, SIMPLIFIED(id: 387 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_1044, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis clock model P2011.6.1 - PLM_1044, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis clock model P2011.6.1 - PLM_1044\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nProtocol for glyoxalase II extracts and kinetic assays.(id: 32 | type: sops)\n\nDescription: Method of how to perform the purification of glyoxalase II as well as kinetics assays.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nPoster presentation Netherlands Biotechnology Congress(id: 36 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nJohn Raedts\nServ Kengen\nPEOPLE\nServ Kengen\nJohn Raedts\nPROJECTS\nCOSMIC\nSUBMITTER\nJohn Raedts\n\n____________________________________________________________________________\n\nGrowth, yield and cAMP excretion under anaerobic conditions(id: 1070 | type: data_files)\n\nDescription: Measurement of by-product formation rates, substrate consumption rates and extracellular concentrations of cAMP under anaerobic batch conditions with glucose as substrate.\n\n\nASSAYS\nAerobic and anaerobic characterization of MG1655 and mutant strains with regard to growth, by-product formation and cAMP concentrations.\nCREATORS\nSonja Steinsiek\nINVESTIGATIONS\nAnalysis of the glucose transport in Escherichia coli under aerobic and anaerobic conditions\nPEOPLE\nSonja Steinsiek\nPROJECTS\nSUMO\nPUBLICATIONS\nGlucose Transport in Escherichia coli Mutant Strains with Defects in Sugar Transport Systems\nSTUDIES\nCharacterization of mutant strains with defects in sugar transport systems\nSUBMITTER\nSonja Steinsiek\n\n____________________________________________________________________________\n\nCarbon Core deFBA Model(id: 557 | type: models)\n\nDescription: A small model representing the core carbon network in each cell. For more detail on the model creation see [1]. The model is written in SBML using the RAM extension for use in deFBA. Compatible python software for simulation can be found at https://tinyurl.com/yy8xu4v7\r\n\r\n[1] S. Waldherr, D. A. Oyarzn, A. Bockmayr. Dynamic optimization of metabolic networks coupled with gene expression. In: Journal of Theoretical Biology, 365(0): 469 - 485.\n\n\nCREATORS\nHenning Lindhorst\nSteffen Waldherr\nPEOPLE\nSteffen Waldherr\nHenning Lindhorst\nPROJECTS\nROBUSTYEAST\nSUBMITTER\nHenning Lindhorst\n\n____________________________________________________________________________\n\nMolecular biological correlation of fluorine-18 fluorodeoxyglucose uptake in esophageal squamous cell carcinoma(id: 259 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Kohl\nPROJECTS\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nKatja Bettenbrock(id: 31 | type: people)\n\nDescription: I am interested in the coupling of global regulation and metabolism in E. coli. To analyze this I construct and analyze defined mutant strains. These strains are characterized in bioreactor experiments of different types (batch, conti, pulse ...) and measurements on the level of metabolites, mRNA, and protein are applied. For all projects there are cooperation partners that use the data in modeling approaches either from the MPI Magdeburg or from the SUMO consortium.\n\n\nASSAYS\nDetermination of by-product formation and glucose uptake of mutants with linear respiratory chain at different aerobiosis levels\nGene expression analysis of mutants with linear electron transport chain at different aerobiosis levels\nDeternination of ArcA phosphroylation level in mutants with linear ETC at different defined aerobiosis levels\nDATA_FILES\nETCmutants_metabolites\nETC_mutants_ArcA\nGene expression levels in mutants with linear respiratory chain\nETCmutants_mRNA\nGene expression rates at different aerobiosis levels via RT-PCR\nINSTITUTIONS\nMax Planck Institute for Dynamics of Complex Technical Systems\nINVESTIGATIONS\nAnalysis of Escherichia coli with linear electron transport chain\nAnalysis of population heterogeneity in chemostat cultures with limited oxygen supply\nPROJECTS\nSUMO\nEraCoBiotech 2 nd call proposal preparation\nPUBLICATIONS\nCharacterization of E. coli MG1655 and frdA and sdhC mutants at various aerobiosis levels\nGlucose Transport in Escherichia coli Mutant Strains with Defects in Sugar Transport Systems\nAnalysis of Escherichia coli mutants with a linear respiratory chain\nKinase activity of ArcB from Escherichia coli is subject to regulation by both ubiquinone and demethylmenaquinone\nTowards a Systems Level Understanding of the Oxygen Response of Escherichia coli\nBasic Regulatory Principles of Escherichia coli's Electron Transport Chain for Varying Oxygen Conditions\nA mathematical model of metabolism and regulation provides a systems-level view of how Escherichia coli responds to oxygen\nSOPS\nEvans Medium\nSTUDIES\nAnalysis of Escherichia coli strains with linear respiratory chain\n\n____________________________________________________________________________\n\nSample preparation procedure for footprint analysis on T. b. brucei 427 using LC/MS(id: 182 | type: sops)\n\nDescription: Sample preparation procedure for metabolic footprint analysis on T. b. brucei 427 using LC/MS\n\n\nASSAYS\nLC-MS based absolute quantification of extracellular metabolites\nCREATORS\nDong-Hyun Kim\nINVESTIGATIONS\nMetabolite profiling, quantification and flux quantification\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSTUDIES\nMetabolic flux measurement\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nMetabolic switches and adaptations deduced from the proteomes of Streptomyces coelicolor wild type and phoP mutant grown in batch culture(id: 295 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nAlexander Wentzel\nPROJECTS\nSTREAM\nSTUDIES\nTimeseries 1\n\n____________________________________________________________________________\n\nData DNA microarray analyses of C. acetobutylicum::trxB(29)(id: 1018 | type: data_files)\n\nDescription: WT (Cy5) vs. trxB (Cy3)_exp. growth phase_DyeFlip\n\n\nASSAYS\nTranscriptional analyses of a thioredoxin (trxB, encoded by CAC1548) knock-out mutant in CGM batch cultures.\nCREATORS\nDaniel Hnicke\nINVESTIGATIONS\nAltering the expression pattern in Clostridium acetobutylicum mutants in the fermentative pathways\nPEOPLE\nDaniel Hnicke\nPROJECTS\nCOSMIC\nSTUDIES\nInfluence of cysteine/sulfate negative mutants of Clostridium acetobutylicum on the expression pattern of batch and chemostat cultures\nSUBMITTER\nDaniel Hnicke\n\n____________________________________________________________________________\n\nNew and upcoming features in SEEK(id: 41 | type: presentations)\n\nDescription: Presented at the annual PALs meeting in Rostock\n\n\nCREATORS\nStuart Owen\nEVENTS\n5th annual SySMO PALs All Hands meeting\nPEOPLE\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nLC-MS based absolute quantification of extracellular metabolites(id: 208 | type: assays)\n\nDescription: Extracellular metabolites in T. brucei at different stage of cell growth have been quantified absolutely by isotope ratio based MS technique using uniformly 13C-labelled E. coli extract. This is the case study for method development of absolute quantification for metabolic flux analysis.\n\n\nDATA_FILES\nConcentrations of extracellular metabolites from T. b. brucei 427\nINVESTIGATION\nMetabolite profiling, quantification and flux quantification\nORGANISMS\nTrypanosoma brucei\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSOPS\nSample preparation procedure for footprint analysis on T. b. brucei 427 using LC/MS\nSTUDY\nMetabolic flux measurement\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nComparison of Sulfolobus solfataricus P2 grown on caseinhydrolysate and D-glucose(id: 169 | type: studies)\n\nDescription: To investigate amino acid degradation pathways in Sulfolobus solfataricus transcriptome, proteome and metabolome analyses were performed on cells grown on caseinhydrolysate as carbon source. Cells grown with glucose served as reference condition. Metabolic modelling was used to compare the efficiency of different degradation routes.\n\n\nASSAYS\nRNA sequencing: Casaminoacids vs D-glc\nProteome analysis: Casaminoacids versus D-Glc\nMetabolome analysis: Casaminoacids versus D-Glc\nMetabolic modelling of S. solfataricus during growth on casaminoacids\nCREATORS\nJacqueline Wolf\nHelge Stark\nDietmar Schomburg\nTrong Khoa Pham\nPhil Wright\nBettina Siebers\nTheresa Kouril\nStefan Albaum\nDATA_FILES\nSSO Genome reference\nRNA-Seq raw data - Sulfolobus solfataricus P2 - casaminoacids and d-glucose\nRNA-Seq mappings - Sulfolobus solfataricus P2 - casaminoacids\nRNA-Seq mappings - Sulfolobus solfataricus P2 - d-glucose\nMetabolomics Casaminoacids vs Glc\nCalculation of amino acid uptake rates\nProteomics_Casaminoacids vs Glucose_Phenyx search output\nProteomics_Casaminoacids vs Glucose_results\nRaw MS data Casaminoacids vs glucose  - Main Zip archive\nRaw MS data Casaminoacids vs glucose - Supplementary file 1\nRaw MS data Casaminoacids vs Glucose - Supplementary file 2\nRaw MS data Casaminoacids vs Glucose - Supplementary file 3\nBiomass composition of S. solfataricus P2 grown on caseinhydrolysate\nDescription of changes made to the model compared to the first publication in 2012\nAmino acid depletion profile of S. solfataricus during growth on 1 % casaminoacids\nDistribution of enzymes involved in oxidative Stickland reactions among archaea\nIntegrated summary of transcriptomics and proteomics results\nINVESTIGATION\nAmino acid degradation in Sulfolobus solfataricus P2\nMODELS\nMetabolic model of Sulfolobus solfataricus\nPEOPLE\nDietmar Schomburg\nBettina Siebers\nPhil Wright\nTheresa Kouril\nTrong Khoa Pham\nStefan Albaum\nHelge Stark\nJacqueline Wolf\nPROJECTS\nSulfoSys - Biotec\nPUBLICATIONS\nOxidative Stickland reactions in an obligate aerobic organism - amino acid catabolism in the Crenarchaeon\n              Sulfolobus solfataricus\nSOPS\nSulfoSYS_BIOTECH: RNA isolation and sequencing\nSULFOSYSbiotech_Proteome analysis\nSUBMITTER\nJacqueline Wolf\n\n____________________________________________________________________________\n\nThe role of the 5'-3' exoribonuclease XRNA in transcriptome-wide mRNA degradation(id: 171 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nAbeer Fadda\nChristine Clayton\nPROJECTS\nSilicoTryp\n\n____________________________________________________________________________\n\nPyruvate formate-lyase (PFL): literature review, structure analysis and suggestions for experiments.(id: 936 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nPyruvate formate-lyase (PFL): literature review, structure analysis and suggestions for experiments.\nCREATORS\nStefan Henrich\nINVESTIGATIONS\nThe Attic\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSTUDIES\nPyruvate formate-lyase (PFL)\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nFrequency-dependent selection of mutant pancreatic beta cells.(id: 679 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nJacky Snoep\nINVESTIGATION\nKarin et al (2017) Molecular Systems Biology\nPEOPLE\nJacky Snoep\nPROJECTS\nMolecular Systems Biology\nSTUDY\nFigure 2: Frequency-dependent selection of mutant pancreatic beta cells.\nSUBMITTER\nJacky Snoep\n\n____________________________________________________________________________\n\nCHecking if all works - PLM_1000, version 111(id: 796 | type: assays)\n\nDescription: <br><br><strong>Related Publications</strong></strong><br>Me and Helena (2014). some book. <br>Moore A, Zielinski T, Millar AJ (2014). Online period estimation and determination of rhythmicity in circadian data, using the BioDare data infrastructure.. Methods in molecular biology (Clifton, N.J.). Retrieved from: doi.org/10.1007/978-1-4939-0700-7_2<br>Stein JM (1975). The effect of adrenaline and of alpha- and beta-adrenergic blocking agents on ATP concentration and on incorporation of 32Pi into ATP in rat fat cells.. Biochemical pharmacology. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/12<br><br><strong>Comments</strong></strong><p>\n\tNew comment after 2nd version</p>\n 2019-01-10 11:59:51 201 zielu tomasz.zielinski@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\r\n\tCommentsdfa</p>\r\n<br><br><strong>Originally submitted to PLaSMo on 2014-07-25 18:01:50</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nTomasz Zielinski\nINVESTIGATION\nZielinski, Tomasz\nPEOPLE\nTomasz Zielinski\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nActivity assay L-fuc vs D-glc(id: 1432 | type: data_files)\n\nDescription: Activity assays in cell free extracts of S. solfataricus grown on either L-fucose or D-glucose and activity assays with recombinant proteins\n\n\nASSAYS\nCell free extract activity measurements: L-fuc/d-glc\nCREATORS\nJacqueline Wolf\nKatharina Kruse\nBettina Siebers\nTheresa Kouril\nINVESTIGATIONS\nL-fucose degradation in Sulfolobus solfataricus P2\nPEOPLE\nBettina Siebers\nTheresa Kouril\nKatharina Kruse\nJacqueline Wolf\nPROJECTS\nSulfoSys - Biotec\nPUBLICATIONS\nA systems biology approach reveals major metabolic changes in the thermoacidophilic archaeon Sulfolobus solfataricus in response to the carbon source L-fucose versus D-glucose\nSTUDIES\nComparison of S. solfataricus grown on l-fucose and d-glucose\nSUBMITTER\nJacqueline Wolf\n\n____________________________________________________________________________\n\nMarcel Schilling(id: 772 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nWP2: Link of Signal Transduction to Gene Regulatory Network\nWP3: Cell Fate Decisions of Erythroid Progenitor Cells\nWP4: Cross-talk with Modulators - Epo with TGF and Novel Factors\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation\nData2Dynamics: a modeling environment tailored to parameter estimation in dynamical systems\nIdentification of isoform-specific dynamics in phosphorylation-dependent STAT5 dimerization by quantitative mass spectrometry and mathematical modeling\nCovering a broad dynamic range: information processing at the erythropoietin receptor\nComputational processing and error reduction strategies for standardized quantitative data in biological networks\nIdentification of Cell Type-Specific Differences in Erythropoietin Receptor Signaling in Primary Erythroid and Lung Cancer Cells\nProtein abundance of AKT and ERK pathway components governs cell type-specific regulation of proliferation\n\n____________________________________________________________________________\n\nKey Cell-Physiological Parameters  of TRK1,2 Transport Systems Investigation(id: 37 | type: studies)\n\nDescription: missing\n\n\nASSAYS\nVolume\nInternal pH\nMembrane Potential\nProtein Concentration\nPotassium Concentration\nINVESTIGATION\nTRK1,2 Transport Systems of Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nStaphylococcal PknB as the first prokaryotic representative of the proline-directed kinases(id: 116 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nThe role of transcription factor GreA in transcription fidelity and processivity in Streptococcus pneumoniae(id: 72 | type: studies)\n\nDescription: For cells to accurately read out the genomic content, high fidelity during transcription is required. This is mainly established by the accuracy of the active centre of RNA polymerase (RNAP). Based on in vitro experiments with Escherichia coli RNAP it was also suggested that proofreading of transcription via RNA hydrolysis by RNAP may contribute to overall fidelity and processivity. RNAPs intrinsic cleavage activity is stimulated by the highly conserved Gre factors suggesting that Gre factors in this way further contribute to transcription fidelity. The in vivo role of Gre however, remains poorly understood. Furthermore, limited functional studies have been performed in bacteria outside E. coli. Here we examine the role of the Gre factor of the Gram-positive human pathogen Streptococcus pneumoniae, GreA. We show that GreA is not essential in S. pneumoniae although greA cells show a slower growth rate and morphological defects. To examine if transcription fidelity is affected by greA deletion in vivo, we constructed a plasmid that contains a constitutively expressed lacZ gene containing a stop codon mutation. Functional LacZ will thus be only produced if errors in transcription or translation are regularly made. Using this in vivo fidelity system we show that cells mutated for greA show significant levels of -galactosidase activity, indicating that GreA promotes transcription fidelity in vivo. To explore the mechanism of GreA-dependent transcription fidelity in more detail, we have purified S. pneumoniae GreA and S. pneumoniae RNAP and performed in vitro transcription assays. We show that S. pneumoniae GreA indeed improves transcription fidelity in vitro. Interestingly, S. pneumoniae RNAP seems to be less processive than other bacterial RNAPs, forming strong species-specific pauses. S. pneumoniae GreA helps in overcoming these pauses and thereby improves RNAPs processivity, thus increasing RNAPs overall elongation rate. Genetic switches that depend on critical thresholds in concentration of regulator protein should be especially responsive to changes in transcription fidelity and rate. In line with this idea, the greA mutant was unable to become naturally competent for DNA uptake, a developmental process that is under the control of a sensitive genetic switch. Our results therefore suggest that fidelity/processivity of transcription facilitated by GreA is crucial for cell well being, and may play an important role in the control of genetic switches. \n\n\nASSAYS\nRNA-Seq\nDATA_FILES\nRNA-seq wild type\nRNA-seq delta greA\nINVESTIGATION\nWetlab approach to transcription fidelity\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\nPUBLICATIONS\nControl of transcription elongation by GreA determines rate of gene expression in Streptococcus pneumoniae\nSUBMITTER\nJan-Willem Veening\n\n____________________________________________________________________________\n\nContributions of the pre- and pro-regions of a Staphylococcus hyicus lipase to secretion of a heterologous protein by Bacillus subtilis(id: 77 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nUwe Voelker\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nGeneration of RNAi cell lines in T.brucei brucei 2T1 cell line(id: 153 | type: sops)\n\nDescription: We routinely select specific RNAi gene targets (400600 bp) and primers using the RNAit software http://trypanofan.path.cam.ac.uk/software/RNAit.html. A single pair of PCR primers are designed that incorporate four selected restriction sites (not present in the RNAi target fragment) such that a single PCR product can be differentially digested and sequentially cloned. For example, using MCS1/2, the following primers could be used to clone antisense followed by sense fragments: Primer 1, XbaIBamHI-5 sequence; Primer 2, ApaIKpnI-3 sequence. The antisense and sense fragments are generated by BamHIKpnI and XbaIApaI digestion respectively.\r\nSelectable marker (HYG) transcription is then constitutive (PEP: procyclin promoter) and independent of the Tet-regulated cassette (PRRNA: RRNA promoter).\r\nWe maintain 2T1 T. brucei cells (Lister 427, MITat1.2, clone 221a), that also express the Tn10 Tet repressor (TetR::BLE::TUB), in HMI-11 growth medium containing phleomycin and puromycin (0.5 g ml1 and 0.2 g ml1 respectively). We use electroporation (Amaxa) with 2.5  107 cells and 3 g of each AscI-linearised construct in duplicate; each mixture is transferred to 36 ml of medium in a 25 cm2 culture flask. After 6 h, the appropriate drug selection is applied (2.5 g ml1 hygromycin) and cells are distributed into 12- or 24-well plates. Each electroporation experiment typically, and conveniently, yields 25 transformed clones using this approach and these are visible after 5 days (100,000 cells). To screen clones for simple, double-crossover integration and loss of the PAC gene, we remove 5% of each day-5, hygR culture and grow with or without puromycin (2 g ml1 ) in a 24-well plate for another 2448 h. Typically, 50% of clones are puromycin sensitive and two of these are retained for further analysis. Note that these strains are now resistant to phleomycin (TetR::BLE::TUB) and hygromycin (HYG::RPa::RRNA). \n\n\nASSAYS\nGeneration of RNAi cell lines in T.brucei brucei 2T1 cell line\nCREATORS\nFederico Rojas\nINVESTIGATIONS\nPerturbing the biological system: developmental adaptation and experimental manipulation of trypanosome biology\nPEOPLE\nFederico Rojas\nPROJECTS\nSilicoTryp\nSTUDIES\nTargeted disruption and over expression of critical enzymes\nSUBMITTER\nFederico Rojas\n\n____________________________________________________________________________\n\nSulfosys - SOP proteomics(id: 64 | type: sops)\n\nDescription: Standard operating procedures regarding iTraq based proteomics.\n\n\nASSAYS\nComparison of proteome of S. solfataricus at 70 and 80C\nCREATORS\nPawel Sierocinski\nINVESTIGATIONS\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature\nPEOPLE\nPawel Sierocinski\nPROJECTS\nSulfoSys\nSTUDIES\nPilot experiment - S. solfataricus grown at 70 and 80 C.\nSUBMITTER\nPawel Sierocinski\n\n____________________________________________________________________________\n\nSABIO-RK documentation web services(id: 1392 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nGene Knock-out Through Allelic Exchange(id: 168 | type: sops)\n\nDescription: A protocol to improve conventional, recombination-based gene knock-out methodologies thtough the provision of negative selection markers, pyrE or codA.\n\n\nCREATORS\nYing Zhang\nNigel Minton\nPEOPLE\nNigel Minton\nYing Zhang\nPROJECTS\nCOSMIC\nSUBMITTER\nYing Zhang\n\n____________________________________________________________________________\n\nTrigger enzymes: bifunctional proteins active in metabolism and in controlling gene expression(id: 151 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoerg Stuelke\nLeif Steil\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nMolecular diffusion in E. coli(id: 29 | type: sops)\n\nDescription: Pulsed-FRAP  measure the diffusion constants in confined volumes in small cells like E. coli and other bacteria or cellular organelles. \n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nAnalysis of the genome-wide distribution of RNA polymerase upon methylglyoxal stress in Escherichia coli using ChIP-chip(id: 36 | type: sops)\n\nDescription: A procedure to analyse the genomic interaction of E.coli RNA polymerase (RNAP) upon methylglyoxal (MG) stress, a toxic keto-aldehyde by-product of metabolism.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nInsights into Streptococcus pyogenes pathogenesis from transcriptome studies(id: 101 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nBernd Kreikemeyer\nTomas Fiedler\nPROJECTS\nSysMO-LAB\n\n____________________________________________________________________________\n\nTowards the development of Bacillus subtilis as a cell factory for membrane proteins and protein complexes(id: 64 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nThomas Hentrich(id: 538 | type: people)\n\nDescription: Trained as a Computer Scientist at the University of Jena, Germany, my interests drifted more and more towards \"computing life\" during my PhD at Simon Fraser University and the University of British Columbia, Canada. On one hand, that term captures that I got fascinated by the idea to understand life as a form of computation and to describe, reprogram, and reassemble parts of cells as described in many brilliant experiments from a spectrum of disciplines ranging from DNA computing to Synthetic biology. On the other hand \"computing life\" stands for the fact that I really like to use empirical data to derive and describe the principles that govern and exhibit cellular functions of living systems. That's the range of (bio-)informatics of my work.\r\nIn particular, I am focusing on epigenetic mechanisms and the epigenome as a cellular computation/regulation layer, which is capable of integrating (external) signals into a genetic regulatory program and which exhibits great plasticity to adjust cell functions in health and disease states. Since my PostDoc time at the University of Tbingen, Germany, I am eager to explore the role of epigenetics in the realm of complex neurological disorders, which seem to be heavily influenced by environmental factors regarding onset and progression of the disease. In a systems-approach, I am employing bioinformatics to identify molecular parts of that system that fail and trigger the disease, and potentially identify therapeutic targets that can be exploited to fix that \"erroneous\" computation and restore a healthy system state.\n\n\nINSTITUTIONS\nUniversity of Tbingen\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nCcpA forms complexes with CodY and RpoA in Bacillus subtilis(id: 160 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nUwe Voelker\nPraveen Kumar Sappa\nGerald Seidel\nMaike  Bartholomae\nWolfgang Hillen\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nSustained glycolytic oscillations in individual isolated yeast cells(id: 379 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nSTUDIES\nSustained glycolytic oscillations in individual isolated yeast cells\n\n____________________________________________________________________________\n\nCompare ATP extraction methods(id: 102 | type: assays)\n\nDescription: We will compare two different procedures to extract ATP from yeast cells: Standard kit procedure (hot Tris/EDTA) and Serrano procedure (cold perchloric acid). In addition we have tested different condition as it turned out that some are important.\n\n\nINVESTIGATION\nEffect of Benzoic Acid on ATP Levels\nORGANISMS\nSaccharomyces cerevisiae\nPEOPLE\nWalter Glaser\nPROJECTS\nMOSES\nSOPS\nATP determination with ROCHE kit  ATP Bioluminescence Assay Kit CLS II\nSTUDY\nTest different ATP extraction protocols and conditions\nSUBMITTER\nWalter Glaser\n\n____________________________________________________________________________\n\nMOA2010-05_2_transcriptomics_qPCR(id: 2773 | type: data_files)\n\nDescription: Validation of microarray results by RT-qPCR. RT-qPCR measurements for eight genes: biologically relevant for photosynthesis chlorophyll a-b binding protein (CAB, CAB_NEW) and RuBisCO activase (RA); for defense response three classes of -1,3-glucanases (Glu-I, Glu-II, Glu-III) and pathogenesis-related protein1b (PR-1b); and for sugar metabolism granule bound starch synthase I (GBSSI) and cell wall invertase (INV).\n\n\nASSAYS\nTranscriptomics (qPCR)\nINVESTIGATIONS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nPEOPLE\nZiva Ramsak\nPROJECTS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nSTUDIES\nMOA2010-05\nSUBMITTER\nZiva Ramsak\n\n____________________________________________________________________________\n\nGenome-scale reconstruction and analysis of the Pseudomonas putida KT2440 metabolic network facilitates applications in biotechnology(id: 2 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nKenneth Timmis\nVitor Martins Dos Santos\nCarolyn Lam\nPROJECTS\nPSYSMO\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nInduction of RNAi in Trypanosoma brucei(id: 183 | type: sops)\n\nDescription: missing\n\n\nCREATORS\nFederico Rojas\nPEOPLE\nFederico Rojas\nPROJECTS\nSilicoTryp\nSUBMITTER\nFederico Rojas\n\n____________________________________________________________________________\n\nChemostat Fermentation B(id: 948 | type: data_files)\n\nDescription: acetic acid puls during acidogenesis\r\nshift to solventogensis\r\nback-shift to acidogenesis\r\nbutyric acid puls during acidogenesis\n\n\nASSAYS\nMetabolome quantification after butyric acid pulse\nMetabolome quantification after acetic acid pulse\nCREATORS\nSebastian Curth\nINVESTIGATIONS\nInvestigation on the effect of pulsed metabolites during a chemostat cultivation of ATCC824\nPEOPLE\nSebastian Curth\nPROJECTS\nCOSMIC\nSTUDIES\nEffect of acid pulses during acidogenesis and solventogenesis\nSUBMITTER\nSebastian Curth\n\n____________________________________________________________________________\n\nMaster file, metabolite concentration, protein copy number and flux estimates to train the model(id: 1704 | type: data_files)\n\nDescription: Master file, aggregates metabolite concentrations inside and outside the cell, protein copy number and flux estimates for metabolites in the core model. Based on all internal metabolite concentrations, external metabolite concentrations from growth curve data, flux of glucose, lactate and acetate based on growth curve data and protein copy number data for enzyme concentrations. Combines absolute and relative measurements and metabolomics measurements from different experiment to get an as complete as possible set of metabolite, enzyme and flux measurements to train the model. Contains data for 6,24 and 48h of adherent (bound to the bottom of the glass) growth in non stirred, non aerated conditions with only oxygen present of the initial medium.\r\nMetabolite concentration and protein concentration Mean and SD values are based on multiple measurements.\n\n\nCREATORS\nNiels Zondervan\nPEOPLE\nNiels Zondervan\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nSTUDIES\nProteomics analysis\nSUBMITTER\nNiels Zondervan\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nMOA2010-05_4_degradomics_Degradome-Seq_targets-vsiRNA(id: 2793 | type: data_files)\n\nDescription: Targets of PVY-derived siRNAs identified by Degradome-Seq. For each predicted interaction, PVYNTN-derived siRNAs (vsiRNAs), the target transcript identifiers, representative gene identifier, full descriptions and MapMan ontology annotations (GoMapMan; (Ramak et al., 2014)) are shown. Short names for potato genes were inferred from Arabidopsis thaliana orthologs where applicable, else the StNIB_v1 gene identifier was set (Ramak et al., 2014). For each vsiRNA-target interaction, degradome category (Addo-Quaye et al., 2009), its p-value (the chance is the total number of peaks of the given category divided by the effective transcriptome size, where 'n' is the number of transcripts), T_Start (one-based start position of the alignment within the transcript, T_Stop (one-based stop position of the alignment within the transcript), T_SLice (one-based position of the alignment opposite position 10 of the query, MFE_perfect (Minimum free energy of a perfectly matched site), MFE_site (Minimum free energy of the alignment), MFE_ratio (MFE_site / MFE_perfect) and number of mismatches are given. Additionally, the log2FC ratios of target transcripts between PVYNTN and mock inoculated Dsire or NahG-Dsire plants; yellow  upregulated; blue  downregulated;  statistically not significant (FDR p > 0.05). NA  not analyzed or not available. D  Dsire; NahG  NahG-Dsire; M  mock; P  PVYNTN.\n\n\nASSAYS\nDegradomics\nINVESTIGATIONS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nPEOPLE\nZiva Ramsak\nPROJECTS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nSTUDIES\nMOA2010-05\nSUBMITTER\nZiva Ramsak\n\n____________________________________________________________________________\n\nIrina Balaur(id: 801 | type: people)\n\nDescription: Researcher at the European Institute for Systems Biology and Medicine, working on network-based data analytics within the EU eTRIKS Project.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nLene Juel Rasmussen(id: 1258 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Copenhagen\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nSimone Keller(id: 552 | type: people)\n\nDescription: I am a postdoc at the Technical University in Munich. When I studied nutritional science I got more and more interested in molecular cancer research. My PhD thesis was in the field of molecular cancer research. In our lab we are interested in molecular basic research on cell culture level, especially regarding gastric cancer. \n\n\nINSTITUTIONS\nInstitute of Pathology\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nA mathematical investigation of the effects of inhibitor therapy on three putative phosphorylation cascades governing the two-component system of the agr operon(id: 36 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJohn King\nSara Jabbari\nPROJECTS\nCOSMIC\n\n____________________________________________________________________________\n\nLocalization of Pma1-GFP(id: 885 | type: data_files)\n\nDescription: Confocal microscopy of Pma1-GFP cells after 60 min at 0 or 50 mM KCl .\n\n\nCREATORS\nPaul Heusden\nPEOPLE\nPaul Heusden\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nPaul Heusden\n\n____________________________________________________________________________\n\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria(id: 11 | type: investigations)\n\nDescription: Challenge: Comparative analyses, as demonstrated by comparative genomics and bioinformatics, are extremely powerful for (i) transfer of information from (experimentally) well-studied organisms to the other organisms, and (ii) when coupled to functional and phenotypic information, insight in the relative importance of components to the observed differences and simalities. The central principle of this proposal is that important aspects of the functional differences between organisms derive not only from the differences in genetic components (which underlies comparative genomics) but also from the interactions between their components (see Fig. 1). Therefore, this project will develop Comparative Systems Biology (CSB).\r\nModel system: The project focuses on three relatively simple and highly related microorganisms, which nevertheless exhibit stark and important differences in their functional relationship with human beings: These organisms are homofermentative lactic acid bacteria, namely Lactococcus lactis, the major microorganism used in the dairy industry, Enterococcus faecalis, a major (fecal) contaminant in food and water as well as a \r\ncontributor to food fermentation, and Streptococcus pyogenes, an important human pathogen. These organisms have similar primary metabolism, but persist in completely different environments (milk, faeces, blood). Lactococcus lactis will be used as the reference microorganism, since: (i) it is by far the best studied lactic acid bacterium, (ii) three different genomes have been sequenced, (iii) a kinetic model has been developed for its complete glycolysis including some branching pathways, (iv) genetic and metabolic engineering tools are available and (v) a functional genomics platform has been set-up including DNA microarray and metabolic databases.\r\nDeliverables: The specific deliverables of this project will be \r\n(i) a thorough understanding of the differences and similarities in systems biology (components and how these components interact) between these three organisms and the extent to which these differences contribute to the organisms being different functionally (Fig. 1).\r\n(ii) generic methods and tools for transfer of systems biology knowledge from one organism to another by comparative analysis. \r\nThis multidisciplinary approach will result in mathematical, comprehensive, models that quantitatively describe the catabolic (product formation) response of lactic acid bacteria under different, industrially relevant, conditions. These models will help in the design of strategies that optimise functional activities such as acidification and flavour production, but also strategies to control or reduce growth of undesirable contaminants and pathogens. \r\nMethodology: The project will focus on carbon metabolism and its response to aeration, change of sugar-source, addition of heme and elimination of the crucial metabolic enzyme, lactate dehydrogenase. Cells will be grown in a non-nutrient-limited turbidostat that will allow maximal growth under defined conditions. The major regulatory events at the genetic level (adaptive mutations), the transcription, translation, enzymatic and metabolic level up to the final output (functional) level, will be quantified and then integrated in iterations between experimentation (all~omics) and modeling (network structure, flux balances, dynamics, control, regulation). The detailed knowledge from the reference strain L lactis will be used to accelerate the model development of the other two, less-well studied, organisms (contributing to deliverable ii in that process).\r\n\n\n\nASSAYS\nGlucose pulsed S. pyogenes\nBIOLOG substrate utilization assay\nGlucose pulsed L. lactis\nKinetics of L-lactate dehydrogenase from S. pyogenes, E. faecalis, and L. plantarum\nKinetics of L-lactate dehydrogenase from L. lactis\nGlobal sensitivity analysis\nModel of L. lactis glycolysis\nMaximal specific growth rates of the three lactic acid bacteria and their ldh deletions\nPhysiological characterization of Lactic acid bacteria grown in C-limited continuous cultures\nGenome-scale model of Streptococcus pyogenes\nGenome-Scale Model Enterococcus faecalis V583\nRegulation of the activity of lactate dehydrogenases from four lactic acid bacteria\nDATA_FILES\nGlucose pulsed S pyogenes\nGlucose pulsed L. lactis\nGlucose pulsed L. lactis 2\nMaximal specific growth rates of the three lactic acid bacteria and their ldh deletions.\nCharacterization of flux distribution and general physiology of S. pyogenes in continuous cultures\nCharacterization of flux distribution and general physiology of E. faecalis in continuous cultures\nCharacterization of flux distribution and general physiology of L. lactis in continuous cultures\nKinetic behavior of intracellular metabolites of L. lactis upon a glucose-pulse\nComparison of maximal build-up of metabolites in lactococcus lactis upon a 20 mM glucose-pulse\nMeasurements on kinetics of L-LDHs from different latic acid bacteria at pH 6.0 and pH 7.0.\nThe effects of fructose-1,6-bisphosphate, phosphate and ionic strength on enzyme activity differ for the three LDHs from three Lactic acid bacterias.\n3D-sturctures of Lactate Dehydrogenase from 4 Lactic Acid Bacteria predicted by Comparative Modeling Tool\nElectrostatic potentials of four LDH enzymes from LAB\nComparison of electrostatic potentials of LDH enzymes from fours LABs.\nExample for the binding energy computation of phosphate ion on the surface of LDH from Lactococcus lactis type 1\nActivation / Inhibition of 4 LDH enzymes in the presence of phosphate ions (Pi).\nMODELS\nGenome-scale model of Streptococcus pyogenes\nPart 1: Comparative modeling of 3D structures of LDH enzymes\nPart 3: Comparison of electrostatic potentials for FBP binding\nPart 2: Computation of electrostatic potentials for 4 LDH enzymes\nPart 4: Activation / Inhibition Effect on LDHs function in the presence of Phosphate ions\nPart 5: Algorithm to computationally estimate the effect of Pi for 4 LDH enzymes\nGenome-scale metabolic model of Enterococcus faecalis V583\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nPUBLICATIONS\nPhysiological characterization of three lactic acid bacteria and their isogenic ldh deletion mutants shows that these organisms are optimized for Yatp and not ATP formation rates at their physiological pH\nRegulation of the Activity of Lactate Dehydrogenases from Four Lactic Acid Bacteria\nSOPS\nAnalysis of organic acid by HPLC\nQuenching of lactic acid bacteria\nProtocol for glucose perturbations\nMeasurement of LDH activity\nSOP 7 Glucose-pulse experiments\nSOP 8 Glucose-pulse experiments (adapted for S. pyogenes)\nSOP 5 Analysis of intracellular metabolites\nSOP 6 Analysis of intracellular metabolites\nSOP 4B  Quenching and extraction of intracellular metabolites\nSOP 2.1 Method for synthesis of LAB-medium used for the SYSMO-LAB project\nSTUDIES\nComparative modeling and phosphate dependence flux distributions and glucose consumption rates of two lactic acid bacteria S. pyogenes and L. lactis\nReconstructing the metabolic pathways of S. pyogenes and E. faecalis from their genome sequence\nStudy of the physiological characterization of three lactic acid bacteria and their isogenic ldh deletion mutants\nKinetics of L-lactate dehydrogenase from S. pyogenes, E. faecalis and L. plantarum\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nGLCtr Kinetic characterisation(id: 191 | type: sops)\n\nDescription: SOP for measurement of glucose transport activity in intact trophozoites.\n\n\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nMorgiane Richard(id: 157 | type: people)\n\nDescription: My background is physics engineering & biomedical engineering. I did my PhD in Surrey on the modelling of response of mammalian cells to radiation of different qualities.\r\nI have been working at the University of Aberdeen since November 2007 as a theoreticien research fellow of the KOSMOBAC project. We are investigating the homeostasis of ions in bacteria E. coli. I have been working at a model of the buffering capacity of the cytopplasm, arising from the presence of weak acids and bases. We have shown that the buffering capacity plays a significant role during metabolic processes and during processes involving flux and protons across the membrane.\n\n\nINSTITUTIONS\nUniversity of Aberdeen\nPROJECTS\nKOSMOBAC\n\n____________________________________________________________________________\n\nHands-on Model Management in SEEK - Task sheet(id: 251 | type: presentations)\n\nDescription: Here you find the Tasks for todays Hands-on Model Management in SEEK session\n\n\nCREATORS\nRon Henkel\nDagmar Waltemath\nPEOPLE\nRon Henkel\nDagmar Waltemath\nSUBMITTER\nRon Henkel\n\n____________________________________________________________________________\n\nPAH metabolites in bile Kollevg(id: 2250 | type: data_files)\n\nDescription: Measurements of PAH metabolites in bile in fish caged in Kollevg.\n\n\nASSAYS\nChemical analyses\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nintracellular metabolites GC/MS(id: 922 | type: data_files)\n\nDescription: intracellular metabolite measured by GC/MS\n\n\nASSAYS\nintracellular metabolites\nCREATORS\nHanna Meyer\nINVESTIGATIONS\nMultiomics study of Bacillus subtilis under osmotic stress\nPEOPLE\nHanna Meyer\nPROJECTS\nBaCell-SysMO\nSTUDIES\nB. subtilis_SysMo2_Chemostat_growthrate-salt\nSUBMITTER\nHanna Meyer\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2011 - PLM_64, version 2(id: 839 | type: assays)\n\nDescription: \tThis model is termed P2011 and derives from the article: <strong>The clock gene circuit in Arabidopsis includes a repressilator with additional feedback loops</strong><strong>. </strong>Alexandra Pokhilko, Aurora Pias Fernndez, Kieron D Edwards, Megan M Southern, Karen J Halliday &amp; Andrew J Millar <em>Mol. Syst. Biol. </em>2012; 8: 574, submitted 9 Aug 2011 and published 6 March 2012. <a href=\"http://www.nature.com/msb/journal/v8/n1/synopsis/msb20126.html\">Link</a>\tLink to <a href=\"http://www.nature.com/msb/journal/v8/n1/extref/msb20126-s1.pdf\">Supplementary Information</a>, including equations. Minor errors in the published Supplementary Information are described in a file attached to version 1 of this model (the published SBML is correct).\t&nbsp;The model describes the circuit depicted in Fig. 1 of the paper (GIF attached). It updates the Pokhilko et al. 2010 model (termed P2010), <a href=\"http://www.plasmo.ed.ac.uk/plasmo/models/model.shtml?accession=PLM_6\">PLM_6</a>, by including:<ol><li>\t\tthe Evening Complex genes (ELF4, ELF3, LUX),</li><li>\t\tlight-regulated degradation of ELF3 by COP1,</li><li>\t\tTOC1 as a repressor rather than an activator of LHY/CCA1.</li></ol>\tThese changes allowed the removal of hypothetical components TOC1mod (or X) and Y from the earlier models. They also reveal that the central loop of the model is a triple-repressor ring oscillator, or 'repressilator' (illustrated in Fig. 8, GIF attached).\tSBML curation notes (please see Comments for each version): Compared to the model version submitted to the Biomodels database, version 1 here slightly alters the names of some variables and uses an SBML AssignmentRule for the light input. A MATLAB version is attached to version 1. In version 2, the rule is used with a generic SBML StepFunction (Adams et al. J. Biol. Rhythms Aug 2012) to describe the light-dark cycle more flexibly. Versions 1 and 2 have multiple sinks and sources created by CellDesigner. These are removed in versions 3 &amp; 4. Version 3 has the published lightfunction, and an equivalent COPASI file attached. Version 4 has the StepFunction.<br><br><strong>Related Publications</strong></strong><br>Alexandra Pokhilko, Aurora Pias Fernndez, Kieron D Edwards, Megan M Southern, Karen J Halliday & Andrew J Millar (2012). The clock gene circuit in Arabidopsis includes a repressilator with additional feedback loops.. Mol. Syst. Biol. 8: 574.. Retrieved from: http://www.nature.com/msb/journal/v8/n1/synopsis/msb20126.html<br><br><strong>Comments</strong></strong><p>\n\tVersion 2 of the model includes the StepFunction. Note that we have not yet removed the original light function parameter names, they are now non-functional.</p>\n 2012-06-01 17:29:46 3 amillar2 andrew.millar@ed.ac.uk<br><br><strong>Originally submitted to PLaSMo on 2012-03-07 12:15:12</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATION\nMillar, Andrew\nMODELS\nArabidopsis_clock_P2011 - PLM_64, version 2, SUBMITTED\nArabidopsis_clock_P2011 - PLM_64, version 2, SIMPLIFIED\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nArabidopsis_clock_P2011 - PLM_64\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nA metabolomics and proteomics study of the adaptation of Staphylococcus aureus to glucose starvation(id: 163 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\nLeif Steil\nMichael Lalk\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemnde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP  EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemnde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\r\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\nSiberian SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universitt Dresden.\r\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\r\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nSystems analysis of transcription factor activities in environments with stable and dynamic oxygen concentrations(id: 177 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nTranscriptional profiling of E. coli during anaerobic to aerobic and aerobic to anaerobic transitions\nDATA_FILES\nDissolved oxygen tension aerobic to anaerobic .pdf\nDissolved oxygen tension anaerobic to aerobic .pdf\nEntire transcriptional dataset - aerobic to anaerobic transition .csv\nEntire transcriptional dataset - anaerobic to aerobic .csv\nFiltered datset - aerobic to anaerobic transition .csv\nFiltered dataset - anaerobic to aerobic .csv\nMODELS\nTFInfer: a tool for probabilistic inference of transcription factor activities\nPEOPLE\nJeff Green\nRobert Poole\nMatthew Rolfe\nGuido Sanguinetti\nPROJECTS\nSUMO\n\n____________________________________________________________________________\n\nMathematical Modelling of the Sporulation-Initiation Network in Bacillus Subtilis Revealing the Dual Role of the Putative Quorum-Sensing Signal Molecule PhrA(id: 59 | type: models)\n\nDescription: Bacillus subtilis cells may opt to forgo normal cell division and instead form spores if subjected to certain environmental stimuli, for example nutrient deficiency or extreme temperature. The gene regulation net-work governing sporulation initiation accordingly incorporates a variety of signals and is of significant complexity. The present model (Bulletin of Mathematical Biology (2011) 73:181211)  includes four of these signals: nutrient levels, DNA damage, the products of the competence genes, and cell population size. An interesting prediction derived from the model is that PhrA, which is secreted from the cell and subsequently re-imported can act simultaneously as a quorum-sensing signal and as a timing mechanism, delaying sporulation when the cell has damaged DNA, possibly thereby allowing the cell time to repair its DNA before forming a spore.\r\n\r\nSimulation link at JWS Online: http://jjj.mib.ac.uk/webMathematica/Examples/run.jsp?modelName=jabbari\n\n\nCREATORS\nSara Jabbari\nJohn Heap\nJohn King\nPEOPLE\nJohn King\nJohn Heap\nSara Jabbari\nFranco Du Preez\nPROJECTS\nSysMO DB\nPUBLICATIONS\nMathematical Modelling of the Sporulation-Initiation Network in Bacillus Subtilis Revealing the Dual Role of the Putative Quorum-Sensing Signal Molecule PhrA\nSUBMITTER\nFranco Du Preez\n\n____________________________________________________________________________\n\nMOA2010-05_2_transcriptomics_qPCR_MIQE(id: 2774 | type: data_files)\n\nDescription: RT-qPCR assays for microarray validation described using the MIQE standard.\n\n\nASSAYS\nTranscriptomics (qPCR)\nINVESTIGATIONS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nPEOPLE\nZiva Ramsak\nPROJECTS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nSTUDIES\nMOA2010-05\nSUBMITTER\nZiva Ramsak\n\n____________________________________________________________________________\n\nLDH Kinetic characterisation(id: 202 | type: sops)\n\nDescription: SOP for measurement of LDH activity in extracts.\n\n\nASSAYS\nLDH\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nCellular Adjustments of Bacillus subtilis and Other Bacilli to Fluctuating Salinities(id: 170 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLeif Steil\nErhard Bremer\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems(id: 303 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nBernhard Steiert\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nKatharina Robichon(id: 516 | type: people)\n\nDescription: missing\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nATP levels under 30 mM and 50 mM benzoic acid stress(id: 50 | type: studies)\n\nDescription: ATP levels of cells stressed with higher concentrations of benzoic acid (30 mM and 50 mM).\r\n\r\n\n\n\nASSAYS\nATP measurement under 30mM benzoic acid stress\nATP measurement under 50mM benzoic acid stress\nINVESTIGATION\nEffect of Benzoic Acid on ATP Levels\nPEOPLE\nMartin Valachovic\nPROJECTS\nMOSES\nSOPS\nATP determination with ROCHE kit  ATP Bioluminescence Assay Kit CLS II\nSUBMITTER\nMartin Valachovic\n\n____________________________________________________________________________\n\nProcessed microarray data from comparison of E. faecalis wt and deltaEF2933 mutant(id: 1010 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMargrete Solheim\nPEOPLE\nMargrete Solheim\nPROJECTS\nSysMO-LAB\nSUBMITTER\nMargrete Solheim\n\n____________________________________________________________________________\n\npH-BASED ESTIMATION OF YEAST ATPase ACTIVITY(id: 148 | type: sops)\n\nDescription: missing\n\n\nCREATORS\nLina Patricia Barreto Parra\nPEOPLE\nLina Patricia Barreto Parra\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nLina Patricia Barreto Parra\n\n____________________________________________________________________________\n\nThe transition from growing to non-growing Bacillus subtilis cells(id: 2 | type: investigations)\n\nDescription: The objective of this project is an integrated understanding the metabolic, proteomic and genetic network that controls the transition from growth to glucose starvation. This transition is a fundamental ecophysiological response that serves as a scientific model for environmental signal integration and is pivotal for industrial fermentations of Bacillus that occur predominantly under nutrient starvation.\r\n\r\nKeywords:\r\nGlucose starvation, Transcriptomics, Proteomics, Metabolomics,Bacillus subtilis, \n\n\nASSAYS\nGene expression(Transcriptome)\nmetabolome-LCMS\n2D-gelbased analysis of intracellular proteins\nRelative quantification of proteins by metabolic labeling\nAbsolute quantification of proteins by the AQUA-technology\nIPTG induction of sigmaB in BSA115\nB. subtilis Transcription Factor Competition - theoretical interpretation\nFermentation-BM5_SysMo\nB. subtilis Transcription Factor Competition - theoretical interpretation\nIPTG induction of sigmaB in BSA115\nStressosome activation dynamics\nDATA_FILES\n0804_shake-flask-sigB-starvation\nbatchfermentation exp-starv01 transcriptome_Microarray Agilent platform_090204\nbatchfermentation exp-starv01 quant-proteomics\nB. subtilis Transcription Factor Competition - computational data\nbatchfermentation exp-starv01 transcriptome_Microarray Nimblegen platform_090204\nBatchfermentation_Exp-Starv01_Fermentationdetails_090204\nBatchfermentation_exp-starv01_2D-proteomics\n20090915_BSA115-IPTG-assay\nModel-representations-BSA115-Experiments\nStressosome activation and structural representations\nMODELS\nBSA115-sigB-adaptation\nBSA115 Post-transcriptional instability\nBSA115-transcription inhibition model\nBSA115-lacz-synth\nStressosome-dynamics\nMonomer Stressosome Model\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nProteolysis of beta-galactosidase following SigmaB activation in Bacillus subtilis\nSimulations of stressosome activation emphasize allosteric interactions between RsbR and RsbT\nSOPS\nSOP for cDNA labelling and Microarray hybridizaton\nRNA isolation and quality check by Agilent bioanalyser\n-Galactosidase assay\nSOP for cultivation of B.Subtilis\nSTUDIES\nBiphase Batch Fermentation(2009/02/04)\nTransition to starvation in shake flask\nControlled sigmaB induction in shake flask\nB. subtilis Transcription Factor Competition\nBatchfermentation exp-starv01_090204\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nStable potassium concentration(id: 74 | type: assays)\n\nDescription: The potassium content of cells grown overnight in a certain amount (0.1, 0.2, 0.5, 1, 5, 20, 50, 200 mM) of KCl will be determined. Additionally the potassium content of cells grown overnight in high potassium and shifted to the amounts of potassium used in the former experiment. After growing overnight again in the lower potassium, the cells should contain finally a comparable potassium concentration than the cells grown in the respecting KCl in the first experiment.\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nExperimental approaches to determine the dependence of volume, pH and membrane potential on external potassium\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, TOC1 degr L - PLM_89, version 1, SIMPLIFIED(id: 526 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_89, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, TOC1 degr L - PLM_89, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, TOC1 degr L - PLM_89\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nJose Ramos(id: 98 | type: people)\n\nDescription: Professor of Microbiology\n\n\nINSTITUTIONS\nUniversity of Cordoba\nPROJECTS\nTRANSLUCENT\nPUBLICATIONS\nLack of main K(+) uptake systems in Saccharomyces cerevisiae cells affects yeast performance in both potassium-sufficient and potassium-limiting conditions\nRegulation of Trk-dependent potassium transport by the calcineurin pathway involves the Hal5 kinase\nAlkali metal cation transport and homeostasis in yeasts\nRef2, a regulatory subunit of the yeast protein phosphatase 1, is a novel component of cation homoeostasis\nAlkali-metal-cation influx and efflux systems in nonconventional yeast species\nSystems Biology of Monovalent Cation Homeostasis in Yeast\nSOPS\nPotassium starvation protocol\n\n____________________________________________________________________________\n\nModel of pH buffering capacity(id: 31 | type: sops)\n\nDescription: Mathematical model of pH buffering, which allows the prediction of how the buffering capacity depends on the cytoplasm's composition, for any number of buffers.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nGeneration of RNAi cell lines in T.brucei brucei 2T1 cell line(id: 188 | type: assays)\n\nDescription: missing\n\n\nINVESTIGATION\nPerturbing the biological system: developmental adaptation and experimental manipulation of trypanosome biology\nPEOPLE\nFederico Rojas\nPROJECTS\nSilicoTryp\nSOPS\nGeneration of RNAi cell lines in T.brucei brucei 2T1 cell line\nSTUDY\nTargeted disruption and over expression of critical enzymes\nSUBMITTER\nFederico Rojas\n\n____________________________________________________________________________\n\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation.(id: 381 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nSTUDIES\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \r\nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nDifferences between healthy hematopoietic progenitors and leukemia cells with respect to CD44 mediated rolling versus adherence behavior on hyaluronic acid coated surfaces(id: 281 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nPEOPLE\nMaja Rey\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nProtein transport across and into cell membranes in bacteria and archaea(id: 82 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nUwe Voelker\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nGeneral model of translation elongation(id: 42 | type: sops)\n\nDescription: A model for translation elongation, which allows the prediction of how different conditions and parameters affect the rate and throughput of translation.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nMOA2010-05_4_degradomics_Degradome-Seq_targets-sRNA(id: 2792 | type: data_files)\n\nDescription: Experimentally validated targets of endogenous sRNAs by Degradome-Seq. For each identified interaction, miRNAs/phasiRNA ID, the target transcript identifiers, representative gene identifier, full descriptions, and MapMan ontology annotations (GoMapMan; (Ramak et al., 2014)) are shown. Short names for potato genes were inferred from Arabidopsis thaliana orthologs where applicable, else the StNIB_v1 gene identifier was set (Ramak et al., 2014). For each sRNA-target interaction, degradome category are given: Category 0 is defined as>1 raw read at the position, with abundance at a position equal to the maximum on the transcript, and with only one maximum on the transcript. Category 1 is described as>1 raw read at the position, with abundance at the position equal to the maximum on the transcript, and more than one maximum position on the transcript. Category 2 includes>1 raw read at the position, and abundance at the position less than the maximum but higher than the median for the transcript. Category 3 comprised the transcripts with>1 raw read at the position, and abundance at the position equal to or less than the median for the transcript (Addo-Quaye et al., 2009). For each sRNA-target interaction also p-value (the chance is the total number of peaks of the given category divided by the effective transcriptome size, where 'n' is the number of transcripts), T_Start (one-based start position of the alignment within the transcript, T_Stop (one-based stop position of the alignment within the transcript), T_SLice (one-based position of the alignment opposite position 10 of the query, MFE_perfect (Minimum free energy of a perfectly matched site), MFE_site (Minimum free energy of the alignment), MFE_ratio (MFE_site / MFE_perfect) and number of mismatches are given.  Additionally, for each sRNA the log2FC and p-values among different comparisons are given and the log2FC ratios of target transcripts between PVYNTN and mock inoculated Dsire or NahG-Dsire plants; yellow  upregulated, blue  downregulated,  statistically not significant (FDR p > 0.05). NA  not analyzed or not available. D  Dsire; NahG  NahG-Dsire; M  mock; P  PVYNTN.\n\n\nASSAYS\nDegradomics\nINVESTIGATIONS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nPEOPLE\nZiva Ramsak\nPROJECTS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nSTUDIES\nMOA2010-05\nSUBMITTER\nZiva Ramsak\n\n____________________________________________________________________________\n\nLutz Brusch(id: 795 | type: people)\n\nDescription: Lutz Brusch is heading the research group \"Spatio-temporal pattern formation in cells and tissues\" at the Centre for Information Services and High Performance Computing of TU Dresden, Germany. The group is co-developing the multi-cellular modelling and simulation framework Morpheus (https://imc.zih.tu-dresden.de/wiki/morpheus/) and is collaborating with experimental labs on questions of tissue morphogenesis and regeneration. \n\n\nINSTITUTIONS\nTechnische Universitt Dresden\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nCOMBINE Multicellular Modelling\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\n\n____________________________________________________________________________\n\nHeterogeneity of glycolytic oscillatory behaviour in individual yeast cells(id: 378 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nSTUDIES\nHeterogeneity of glycolytic oscillatory behaviour in individual yeast cells\n\n____________________________________________________________________________\n\nRoger Adan(id: 806 | type: people)\n\nDescription: Professor in Neuroscience and Molecular Pharmacology at UMCU, Utrecht, NL\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nAna Serrano Puebla(id: 1277 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Copenhagen\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nSustained glycolytic oscillations in individual isolated yeast cells(id: 110 | type: studies)\n\nDescription: missing\n\n\nASSAYS\nModelling sustained glycolytic oscillations in individual isolated yeast cells (Gustavsson models)\nDATA_FILES\nSustained glycolytic oscillations in yeast\nINVESTIGATION\nYeast Glycolytic Oscillations\nMODELS\nSustained glycolytic oscillations in individual isolated yeast cells - IV\nSustained glycolytic oscillations in individual isolated yeast cells - III\nSustained glycolytic oscillations in individual isolated yeast cells - II\nSustained glycolytic oscillations in individual isolated yeast cells - I\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nPUBLICATIONS\nSustained glycolytic oscillations in individual isolated yeast cells\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling(id: 97 | type: investigations)\n\nDescription: Research in Systems Biology involves integrating data and knowledge about the dynamic processes in biological systems in order to understand and model them. By connecting fields such as genomics, proteomics, bioinformatics, mathematics, cell biology, genetics, mathematics, engineering and computer sciences, Systems Biology enables discovery of yet unknown principles underlying the functioning of living cells. At the same time, testable and predictive models of complex cellular pathways and eventually of whole cells and organisms are generated as tools for research. Our workshop will provide a hands-on experience in integrating experimentation and mathematical modelling.\n\n\nASSAYS\nHow to use templates to structure experimental data\nCREATORS\nOlga Krebs\nPEOPLE\nOlga Krebs\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSTUDIES\nExperimental study for hands on session\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nBogumila Marciniak(id: 293 | type: people)\n\nDescription: PhD student. Analyzing CcpA affinity to cre boxes (catabolite responsive elements) and response of B. subtilis to membrane protein overproduction stress.\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nPenicillin-binding protein folding is dependent on the PrsA peptidyl-prolyl cis-trans isomerase in Bacillus subtilis\nHigh- and low-affinity cre boxes for CcpA binding in Bacillus subtilis revealed by genome-wide analysis\n\n____________________________________________________________________________\n\nLocke2005_CircadianClock_tanh - PLM_8, version 1, SIMPLIFIED(id: 472 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_8, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nLocke2005_CircadianClock_tanh - PLM_8, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nLocke2005_CircadianClock_tanh - PLM_8\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nThe DEAD-box RNA helicases in Bacillus subtilis have multiple functions and act independent from each other(id: 192 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoerg Stuelke\nOscar Kuipers\nLeif Steil\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nStandard-based Excel template for metabolomics data(id: 1 | type: assays)\n\nDescription: Dear SEEK users, this Assay is just an example  Excel sheet for intracellular metabolites concentration measurements performed using cell culture growing in chemostat\n\n\nCREATORS\nOlga Krebs\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nThe FAIR Guiding Principles for scientific data management and stewardship\nINVESTIGATION\nCreating data sheet template for 'omics data\nORGANISMS\nSaccharomyces cerevisiae\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nFAIRDOM\nFAIRDOM user meeting\nSTUDY\nCreating template for metabolomics data\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nRAM: An annotation standard for SBML Level 3. Specification Documents(id: 304 | type: sops)\n\nDescription: Metabolic networks with gene expression are researched under very different banners with different  techniques. For example, there are  the dynamic enzyme-cost Flux Balance Analysis (deFBA) [1], conditional Flux Balance Analysis [2], Metabolism and Expression models (ME models) [3], Resource Balance Analysis [4], etc. At their core, these methods can all understood as Resource Allocation Models (RAM) and while investigating their potential and their results, we encountered the problem of sharing these types of models between workgroups. The usual goto-standard is the Systems Biology Markup Language (SBML) but even with the latest available packages some resource allocation model parameters cannot be integrated.\r\n\r\nThus, we have developed the 'ram'-standard, which utilizes the <annotation> field available inside the species and reactions element to solve this problem. Naturally, the standard includes a few more conventions for describing RAM. This document contains all these specifications.\r\n\r\nYou can use the latest version of libsbml (version >5.13) [5] and directly use the 'ram' standard without additional software. At the moment we support interfaces from 'ram' to a Python implementation ([6]) and a MATLAB implementation (link). Below you can also find a few examples of models in the 'ram' format.\r\n\r\nIn this first version we focused on the parameters needed for deFBA models, but we hope to support more model types in the near future. If you want to contribute, you can do so via the issue manager of the bitbucket copy of the specifications [7].\r\n\r\n[1] Waldherr, S.; Oyarzn, D.A.; Bockmayr, A. Dynamic optimization of metabolic networks coupled with gene expression. Journal of Theoretical Biology 2015, 365, 469485.\r\n[2] Reimers, A.M.; Knoop, H.; Bockmayr, A.; Steuer, R. Cellular trade-offs and optimal resource allocation during cyanobacterial diurnal growth. Proceedings of the National Academy of Sciences 2017 (in press).\r\n[3] Lerman, J.A.; Hyduke, D.R.; Latif, H.; Portnoy, V.A.; others. In silico method for modelling metabolism and gene product expression at genome scale. Nature Communications 2012.\r\n[4] Goelzer, A.; Fromion, V.; Scorletti, G. Cell design in bacteria as a convex optimization problem. Automatica 2011, 47, 12101218\r\n[5] http://sbml.org/Software/libSBML/Downloading_libSBML\r\n[6] https://bitbucket.org/hlindhor/defba-python-package\r\n[7] https://bitbucket.org/hlindhor/sbml-ram-specifications.\n\n\nCREATORS\nHenning Lindhorst\nAlexandra-M. Reimers\nPEOPLE\nHenning Lindhorst\nAlexandra-M. Reimers\nPROJECTS\nROBUSTYEAST\nSUBMITTER\nHenning Lindhorst\n\n____________________________________________________________________________\n\nATP determination with ROCHE kit  ATP Bioluminescence Assay Kit CLS II(id: 143 | type: sops)\n\nDescription: The kit is used for the quantitative detection of ATP in yeast cells by luciferase driven \r\nbioluminescence. ATP extraction is done by boiling cells in TE buffer. \n\n\nASSAYS\nCompare ATP extraction methods\nATP measurement under 10mM benzoic acid stress\nATP measurement under 30mM benzoic acid stress\nATP measurement under 50mM benzoic acid stress\nCREATORS\nMartin Valachovic\nCornelia Klein\nINVESTIGATIONS\nEffect of Benzoic Acid on ATP Levels\nPEOPLE\nMartin Valachovic\nWalter Glaser\nCornelia Klein\nPROJECTS\nMOSES\nSTUDIES\nTest different ATP extraction protocols and conditions\nATP levels under 10 mM benzoic acid stress\nATP levels under 30 mM and 50 mM benzoic acid stress\nSUBMITTER\nWalter Glaser\n\n____________________________________________________________________________\n\nHow does internal potassium change (/decrease) during several hours of potassium starvation.(id: 62 | type: assays)\n\nDescription: How does internal potassium changes (decreases) during several hours of potassium starvation. Is there a limit for internal potassium decrease? Sampling potassium content in cells during starvation\r\nThe relative membrane potential will be measured according to the conditions of Navarrete et al. (2010). Various mutants will be tested for their effect.\r\nTo estimate the relative changes of plasma-membrane potential, the diSC3(3) fluorescent probe was used and the changes in cell fluorescence monitored during 5 hours of incubation in YNB-F growth media without added potassium.\r\n\r\n\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nKey Cell-Physiological Parameters\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nDomino systems biology and the A of ATP(id: 179 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nSteady state concentrations of intracellular metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nDynamics of intracellular metabolites during glucose pulse\nDATA_FILES\nMeasured steady state concentrations of intracellular metabolites in yeast Saccharomyces cerevisiae anaerobic chemostat\nDynamics of intracellular metabolites during glucose pulse\nMODELS\nSBML model of yeast central carbon metabolism\nPEOPLE\nHans Westerhoff\nMaksim Zakhartsev\nMalkhey Verma\nMatthias Reuss\nPROJECTS\nMOSES\n\n____________________________________________________________________________\n\nModels of Redox and Carbon Metabolism in Escherichia coli: Integrating Different Types of Data(id: 83 | type: presentations)\n\nDescription: Presentation by Michael Ederer and Klaas Hellingwerf at the SysMO Conference 2013 (Feb 2013, Berlin)\n\n\nCREATORS\nMichael Ederer\nEVENTS\nSysMO Evaluation 2013\nPEOPLE\nMichael Ederer\nPROJECTS\nSUMO\nSUBMITTER\nMichael Ederer\n\n____________________________________________________________________________\n\nThe FAIR Guiding Principles for scientific data management and stewardship(id: 2754 | type: data_files)\n\nDescription: Here published artikel about The FAIR Guiding Principles for scientific data management and stewardship\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nOlga Krebs\nPROJECTS\nFAIRDOM user meeting\nSTUDIES\nCreating template for metabolomics data\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nSophie Kussauer(id: 1291 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRostock University Medical Centre\nPROJECTS\niRhythmics\n\n____________________________________________________________________________\n\nWhole genome marker frequency analysis on strain MK350(id: 1429 | type: data_files)\n\nDescription: Strain MK350 was grown under the same conditions as used when growing cells for microscopy analysis; cells of OD600 = 0.4 were diluted 100-fold in C+Y medium with 0.1 mM ZnCl2 and incubated for 2.5 hours until OD600 = 0.15. Cells were then harvested by centrifugation for 5 min at 6500 x g at 4C. Genomic DNA was isolated using the Wizard Genomic DNA Purification Kit (Promega) as described previously (Slager et al. 2014 Cell). Fragmentation was performed using Covaris instrument, and libraries were prepared using NEBNext Ultra DNA Library Kit for Illumina prior to sequencing on an Illuminia HiSeq 2000 Platform (all performed at the Genomics Core Facility, EMBL, Heidelberg, Germany). \n\n\nASSAYS\nWhole genome sequencing (Illumina)\nCREATORS\nJan-Willem Veening\nINVESTIGATIONS\nChromosome segregation and cell division in Streptococcus pneumoniae\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\nSTUDIES\nGenome wide marker frequency analysis on new strains to image genetic loci in Streptococcus pneumoniae\nSUBMITTER\nJan-Willem Veening\n\n____________________________________________________________________________\n\nDissection of the network of interactions that links RNA processing with glycolysis in the Bacillus subtilis degradosome(id: 169 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLeif Steil\nColin Harwood\nJoseph Newman\nRick Lewis\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nSalazar2009_FloweringPhotoperiod - PLM_9(id: 528 | type: studies)\n\nDescription: \tThe model shows how the <em>CONSTANS</em> gene and protein in <em>Arabidopsis thaliana</em> forms a day-length sensor. It corresponds to Model 3 in the publication of Salazar et al. 2009.\tMatlab versions of all the models in the paper are attached to this record as a ZIP archive, as are all the data waveforms curated from the literature to constrain the model. Further information may be available via links from the authors web site (<a href=\"http://www.amillar.org\">www.amillar.org</a>).\tSimulation notes for SBML version of Model3 from Salazar et al., Cell, 2009.\tThe interlocking-loop circadian clock in this model reaches its entrained limit cycle rather slowly. To ensure that simulations show the stable phase under any light:dark cycle, the model should be run for 720 hours (30 days) before evaluating its behaviour in light:dark cycles. Using the boundary value solver in Matlab, as described in the paper, avoids this issue.\tSBML prepared by Dr. Treenut Saithong, based on Matlab by Dr. Domingo Salazar. Model depositor, updates to version 2, Prof. Andrew Millar.\tNote comment from Rob Smith on Figure 5.<br><br><strong>Related Publications</strong></strong><br>Salazar JD, Saithong T, Brown PE, Foreman J, Locke JC, Halliday KJ, Carr IA, Rand DA, Millar AJ. (2009). Prediction of photoperiodic regulators from quantitative gene circuit models. Cell. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/20005809<br><br><strong>Originally submitted to PLaSMo on 2010-05-05 15:16:35</strong></strong>\n\n\nASSAYS\nSalazar2009_FloweringPhotoperiod - PLM_9, version 1\nSalazar2009_FloweringPhotoperiod - PLM_9, version 2\nCREATORS\nBioData SynthSys\nAndrew Millar\nDATA_FILES\nZIP archive of Matlab model files for the models described in the paper, PLM_9_ 1\nFigure 1 of the paper, with a cartoon of the model. Model3 is provided as SBML. This is the sum of model 1(b) and model 2(b) shown in the cartoon., PLM_9_ 1\nINVESTIGATION\nMillar, Andrew\nMODELS\nSalazar2009_FloweringPhotoperiod - PLM_9, version 1, SUBMITTED\nSalazar2009_FloweringPhotoperiod - PLM_9, version 1, SIMPLIFIED\nSalazar2009_FloweringPhotoperiod - PLM_9, version 2, SUBMITTED\nSalazar2009_FloweringPhotoperiod - PLM_9, version 2, SIMPLIFIED\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nSteroid hormones plasma Kollevg(id: 2251 | type: data_files)\n\nDescription: Concentrations of E2 and T in cod plasma of female cod caged in Kollevg measured by enzyme immunoassay.\n\n\nASSAYS\nEnzyme immunoassay\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nDaniel Samaga(id: 167 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMax Planck Institute for Dynamics of Complex Technical Systems\nPROJECTS\nPSYSMO\n\n____________________________________________________________________________\n\nNewcastle University Systems Modelling of Ageing(id: 56 | type: programmes)\n\nDescription: Systems modelling molecular mechanisms of ageing\n\n\nADMINISTRATORS\nDaryl Shanley\nINSTITUTIONS\nUniversity of Newcastle\nDurham University\nUniversity of Copenhagen\nNational Institute on Aging\nPEOPLE\nCiaran Welsh\nDaryl Shanley\nKrutik Patel\nCraig Manning\nStefan Przyborski\nHannah O'Keefe\nAlvaro Martinez Guimera\nLene Juel Rasmussen\nDeborah Croteau\nVilhelm Bohr\nAna Serrano Puebla\nJames Wordsworth\nPROJECTS\nSystems modelling age-related changes in the maintenance of dermal extra-cellular matrix: mechanisms and interventions\nAre microRNAs key mediators of cartilage destruction in osteoarthritis?\nDynamic interplay between nuclei and mitochondria in ageing cells\nPUBLICATIONS\nNAD + Replenishment Improves Lifespan and Healthspan in Ataxia Telangiectasia Models via Mitophagy and DNA Repair\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1 axis\n\n____________________________________________________________________________\n\nExperimental approaches to determine the dependence of volume, pH and membrane potential on external potassium(id: 34 | type: studies)\n\nDescription: Are there one or several stationary states for physiological parameters dependent on external KCl? For which range of external potassium the cell is able to maintain a constant pH, potassium content, membrane potential and volume?\n\n\nASSAYS\nStable potassium concentration\nStable pH\nStable membrane potential\nStable volume\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nArabidopsis clock model P2011.5.1 - PLM_1043, version 1, SIMPLIFIED(id: 385 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_1043, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis clock model P2011.5.1 - PLM_1043, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis clock model P2011.5.1 - PLM_1043\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nBIOMD0000000005 from BioModels Database(id: 196 | type: models)\n\nDescription: This is BIOMD0000000005.\n\n\nASSAYS\nBIOMD0000000005 - Tyson1991 - Cell Cycle 6 var\nCREATORS\nRon Henkel\nDagmar Waltemath\nINVESTIGATIONS\nHands-on: Model Management in SEEK\nPEOPLE\nRon Henkel\nDagmar Waltemath\nPROJECTS\nKinetics on the move - Workshop 2016\nPUBLICATIONS\nModeling the cell division cycle: cdc2 and cyclin interactions\nSTUDIES\nHands-On: Tyson1991 - Cell Cycle 6 var\nSUBMITTER\nRon Henkel\n\n____________________________________________________________________________\n\nmRNA half-lives(id: 1118 | type: data_files)\n\nDescription: This files contains the parameter values, life-times, half-lives and errors associated with modeling the decay of the transcriptome, based on 3 models described in Deneke et al. \"Complex degradation processes lead to non-exponential decay patters and age-dependent decay rates of messenger RNA\". PLoS One. 2013;8(2):e55442\r\n\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\r\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nKinetic behavior of intracellular metabolites of L. lactis upon a glucose-pulse(id: 821 | type: data_files)\n\nDescription: In this experiment we glucose-pulsed an L. lactiss cultures re-suspended in 100 mM MES buffer at pH 6.5. Samples were taken in time to study intra- and extracellular metabolites. These data are used to construct a kinetic model of the catabolism of E. L. lactis\n\n\nASSAYS\nGlucose pulsed L. lactis\nCREATORS\nMartijn Bekker\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSTUDIES\nComparative modeling and phosphate dependence flux distributions and glucose consumption rates of two lactic acid bacteria S. pyogenes and L. lactis\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nProteomic analysis(id: 40 | type: studies)\n\nDescription: Does the general proteome of yeast cells change in a strain lacking Trk1,2 transporter? Under which conditions?\n\n\nASSAYS\n2D Gel Analysis\nINVESTIGATION\nTRK1,2 Transport Systems of Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nInternal pH(id: 81 | type: assays)\n\nDescription: To estimate the differnces of internal pH between the wild type, single trk1 and trk2 or double trk1 trk2 mutants, the pH-dependent variant of GFP pHluorin was expressed in cells from a multicopy plasmid, cells were grown under various conditions, and the cell fluorescence was monitored.\n\n\nINVESTIGATION\nTRK1,2 Transport Systems of Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nKey Cell-Physiological Parameters  of TRK1,2 Transport Systems Investigation\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nAgent-based modeling of oxygen-responsive transcription factors in Escherichia coli(id: 232 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJeff Green\nMichael Holcombe\nRobert Poole\nMatthew Rolfe\nWenjing Jia\nHao Bai\nPROJECTS\nSUMO\n\n____________________________________________________________________________\n\nSample preparation for intracellular metabolite extraction in T. b. brucei 427(id: 154 | type: sops)\n\nDescription: Sample preparation procedure for metabolic analysis on T. b. brucei 427 using LC/MS\n\n\nASSAYS\nLC-MS based absolute quantification of intracellular metabolites\nMetabolite profiling on T. brucei exposed to oxidative stress\nIntracellular metabolite concentrations in T. brucei exposed to oxidative stress\nIntracellular metabolite concentrations in T. brucei under pH stress\nCREATORS\nDong-Hyun Kim\nINVESTIGATIONS\nMetabolite profiling, quantification and flux quantification\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSTUDIES\nTargeted metabolite analysis\nUntargeted metabolite analysis\nMetabolic flux measurement\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1 axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nList of recent publications & patents(id: 1465 | type: data_files)\n\nDescription: Publications and patents from the last 6 years\n\n\nCREATORS\nVitor Martins Dos Santos\nPEOPLE\nVitor Martins Dos Santos\nSUBMITTER\nVitor Martins Dos Santos\n\n____________________________________________________________________________\n\nCell dry weight (CDW) determination of Streptomyces coelicolor culture(id: 133 | type: sops)\n\nDescription: missing\n\n\nASSAYS\nOnline/offline measurements\nCREATORS\nPer Bruheim\nTrond Ellingsen\nSunniva Hoel\nyvind Jakobsen\nHvard Sletta\nAlexander Wentzel\nAnders verby\nINVESTIGATIONS\nMetabolism of Streptomyces coelicolor (SysMO STREAM)\nPEOPLE\nTrond Ellingsen\nyvind Jakobsen\nJay Moore\nPer Bruheim\nHvard Sletta\nAnders verby\nSunniva Hoel\nAlexander Wentzel\nPROJECTS\nSTREAM\nSTUDIES\nTimeseries 1\nSUBMITTER\nJay Moore\n\n____________________________________________________________________________\n\nDetermination of protein phosphorylation levels using Phos-tag gels(id: 185 | type: sops)\n\nDescription: The phosphorylation level of a particular protein can be determined using a procedure based upon western immunoblotting, with Phos-tag reagent present in the SDS-PAGE gel. The Phos-tag reagent, supplied in the form of Phos-tag acrylamide (Wako Pure Chemical Industries, AAL-107), causes proteins to be resolved both on the basis of size and phosphorylation state. This means that phosphorylated and de-phosphorylated forms of the same protein can be distinguished. \n\n\nASSAYS\nArcA phosphorylation at different aerobiosis levels (steady states)\nCREATORS\nMatthew Rolfe\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nPEOPLE\nMatthew Rolfe\nPROJECTS\nSUMO\nSTUDIES\nThe Escherichia coli steady state response to oxygen: from molecular interactions to regulatory modules\nSUBMITTER\nMatthew Rolfe\n\n____________________________________________________________________________\n\nMeasurement of intracellular pH using the radio-labeled weak acid distribution method in E. coli.(id: 35 | type: sops)\n\nDescription: Measurement of the transmembrane pH gradient and thus pHi, when the external pH is known.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nSaccharomyces cerevisiae BY4741 and W303-1A laboratory strains differ in salt tolerance(id: 31 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nHana Sychrova\nFalko Krause\nSilvia Petrezselyova\nJaromir Zahrdka\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nTiling Array analysis of glucose strarved B. subtilis cells(id: 53 | type: assays)\n\nDescription: B. subtilis was grown in M9 media with glucose as carbon source and the samples were harvested during exponential phase (OD600nm- 0.4), early stationary phase(OD600nm- 1.3), late stationary phase(OD600nm- 1.0).\r\nAll the samples were analysed for transcriptome as biological triplicates.\r\n\n\n\nDATA_FILES\nM9_shake_flask_exp_trans_stat_transcriptome_nimblegen\nThe condition-dependent transcriptome of Bacillus subtilis 168\nINVESTIGATION\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nORGANISMS\nBacillus subtilis\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nCondition-dependent transcriptome reveals high-level regulatory architecture in Bacillus subtilis\nSOPS\nTotal RNA isolation from B.Subtilis\nSOP for cultivation of B.Subtilis\nSTUDY\nTranscriptome analysis of glucose starvation in B. subtilis\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nSulfosys - SOP metabolomics(id: 65 | type: sops)\n\nDescription: Standard procedure regarding intracellular metabolome analysis using GC-MS.\n\n\nASSAYS\nIntracellular metabolomics of S. solfataricus at 70 and 80C\nCREATORS\nPawel Sierocinski\nINVESTIGATIONS\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature\nPEOPLE\nPawel Sierocinski\nPROJECTS\nSulfoSys\nSTUDIES\nPilot experiment - S. solfataricus grown at 70 and 80 C.\nSUBMITTER\nPawel Sierocinski\n\n____________________________________________________________________________\n\nAnalysis of population heterogeneity in chemostat cultures with limited oxygen supply(id: 53 | type: investigations)\n\nDescription: Cultures grown under standard SUMO conditions were analyzed with respect to heterogeneity in gene expression. To this end GFP reporter strains were constructed and GFP expression at single cell level was monitored by flow cytometry.\n\n\nPEOPLE\nKatja Bettenbrock\nPROJECTS\nSUMO\nSUBMITTER\nKatja Bettenbrock\n\n____________________________________________________________________________\n\nPresentation Armin Ehrenreich(id: 94 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nArmin Ehrenreich\nEVENTS\nCOSMIC Final Meeting 2013\nPEOPLE\nArmin Ehrenreich\nSebastian Curth\nPROJECTS\nCOSMIC\nSUBMITTER\nSebastian Curth\n\n____________________________________________________________________________\n\nB. subtilis_SysMo2_Chemostat_growthrate-salt_salt_+GB_proteome_relative quantification(id: 859 | type: data_files)\n\nDescription: B. subtilis was grown in minimal media in a chemostat at growth rate =0.1, with 1.2M NaCl and glycine betaine. Relative quantification for the proteome was done using metabolic labeling.\n\n\nASSAYS\nRelative quantification of proteins by metabolic labeling\nCREATORS\nSandra Maass\nINVESTIGATIONS\nMultiomics study of Bacillus subtilis under osmotic stress\nPEOPLE\nSandra Maass\nPROJECTS\nBaCell-SysMO\nSTUDIES\nB. subtilis_SysMo2_Chemostat_growthrate-salt\nSUBMITTER\nSandra Maass\n\n____________________________________________________________________________\n\nProtein abundance of AKT and ERK pathway components governs cell type-specific regulation of proliferation(id: 305 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nWP2: Link of Signal Transduction to Gene Regulatory Network\nWP3: Cell Fate Decisions of Erythroid Progenitor Cells\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nAdlung et al (2017) Molecular Systems Biology\nPEOPLE\nMaja Rey\nSajib Chakraborty\nLorenz Adlung\nHauke Busch\nMarcel Schilling\nMarie-Christine Wagner\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nArabidopsis_clock_2012_TOC1_repressor - PLM_49, version 2(id: 833 | type: assays)\n\nDescription: \tMillar lab working model, extends the Arabidopsis clock model by incorporating multiple sites of inhibition of clock gene expression by TOC1. Model is included into submitted publication \"Global Mapping at the Core of the Arabidopsis Circadian Clock Defines a Novel Network Structure of the Oscillator\" with Paloma Mas\tVersion 1 has two errors corrected in version 2.\tThis private record is now superseded by the published version, which is public as PLM_70.<br><br><strong>Version Comments</strong></strong><p>\n\tThis is a tidied-up and corrected SBML file, lacking the CellDesigner sinks and sources of version 1. Two minor errors from version 1 are also corrected:</p>\n<ol>\n\t<li>\n\t\tM37 was used twice, in error</li>\n\t<li>\n\t\tM9 and m10 were local to cE3n_degr, not global.</li>\n</ol>\n<p>\n\tVersion 2 will simulate in Copasi and in SBSI, and will also compile for optimisation on SBSI.</p>\n<p>\n\tA corresponding Copasi file is attached.</p>\n<br><br><strong>Originally submitted to PLaSMo on 2011-12-29 14:36:56</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nINVESTIGATION\nPokhilko, Alexandra\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nArabidopsis_clock_2012_TOC1_repressor - PLM_49\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2011 - PLM_64, version 4(id: 841 | type: assays)\n\nDescription: \tThis model is termed P2011 and derives from the article: <strong>The clock gene circuit in Arabidopsis includes a repressilator with additional feedback loops</strong><strong>. </strong>Alexandra Pokhilko, Aurora Pias Fernndez, Kieron D Edwards, Megan M Southern, Karen J Halliday &amp; Andrew J Millar <em>Mol. Syst. Biol. </em>2012; 8: 574, submitted 9 Aug 2011 and published 6 March 2012. <a href=\"http://www.nature.com/msb/journal/v8/n1/synopsis/msb20126.html\">Link</a>\tLink to <a href=\"http://www.nature.com/msb/journal/v8/n1/extref/msb20126-s1.pdf\">Supplementary Information</a>, including equations. Minor errors in the published Supplementary Information are described in a file attached to version 1 of this model (the published SBML is correct).\t&nbsp;The model describes the circuit depicted in Fig. 1 of the paper (GIF attached). It updates the Pokhilko et al. 2010 model (termed P2010), <a href=\"http://www.plasmo.ed.ac.uk/plasmo/models/model.shtml?accession=PLM_6\">PLM_6</a>, by including:<ol><li>\t\tthe Evening Complex genes (ELF4, ELF3, LUX),</li><li>\t\tlight-regulated degradation of ELF3 by COP1,</li><li>\t\tTOC1 as a repressor rather than an activator of LHY/CCA1.</li></ol>\tThese changes allowed the removal of hypothetical components TOC1mod (or X) and Y from the earlier models. They also reveal that the central loop of the model is a triple-repressor ring oscillator, or 'repressilator' (illustrated in Fig. 8, GIF attached).\tSBML curation notes (please see Comments for each version): Compared to the model version submitted to the Biomodels database, version 1 here slightly alters the names of some variables and uses an SBML AssignmentRule for the light input. A MATLAB version is attached to version 1. In version 2, the rule is used with a generic SBML StepFunction (Adams et al. J. Biol. Rhythms Aug 2012) to describe the light-dark cycle more flexibly. Versions 1 and 2 have multiple sinks and sources created by CellDesigner. These are removed in versions 3 &amp; 4. Version 3 has the published lightfunction, and an equivalent COPASI file attached. Version 4 has the StepFunction.<br><br><strong>Related Publications</strong></strong><br>Alexandra Pokhilko, Aurora Pias Fernndez, Kieron D Edwards, Megan M Southern, Karen J Halliday & Andrew J Millar (2012). The clock gene circuit in Arabidopsis includes a repressilator with additional feedback loops.. Mol. Syst. Biol. 8: 574.. Retrieved from: http://www.nature.com/msb/journal/v8/n1/synopsis/msb20126.html<br><br><strong>Comments</strong></strong><p>\n\tVersion 2 of the model includes the StepFunction. Note that we have not yet removed the original light function parameter names, they are now non-functional.</p>\n 2012-06-01 17:29:46 3 amillar2 andrew.millar@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\n\tThe SBML for this version uses the StepFunction (as in Adams et al. J. Biol Rhythms 2012) to represent the light-dark cycle. It has parameter name Lightstep, ID step1. Parameter name L, which is used in the model reactions, is assigned the value of Lightstep.</p>\n<p>\n\tNote version 3 has a Copasi file attached.</p>\n<br><br><strong>Originally submitted to PLaSMo on 2012-03-07 12:15:12</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nAndrew Millar\nDATA_FILES\nFig. 1, outline of P2011 model, with P2010 inset,PLM_64_ 4\nFig. 8, cartoon illustrating repressilator structure,PLM_64_ 4\nODE file for Matlab version of P2011 clock model, PLM_64_ 4\nMatlab version of P2011 clock model, PLM_64_ 4\nINVESTIGATION\nMillar, Andrew\nMODELS\nArabidopsis_clock_P2011 - PLM_64, version 4, SUBMITTED\nArabidopsis_clock_P2011 - PLM_64, version 4, SIMPLIFIED\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nArabidopsis_clock_P2011 - PLM_64\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nKirsten Jung(id: 42 | type: people)\n\nDescription: I have the chair for Microbiology at the Ludwig-Maximilians-Universitt Mnchen since 2004. My lab has a long-standing interest in stress response and transmembrane signal transduction in E. coli and V. harveyi. Our work is focused on the molecular mechanisms of membrane-integrated receptors, and the systems biological analysis of regulatory circuits. \n\n\nINSTITUTIONS\nLudwig-Maximilians University of Munich\nPROJECTS\nKOSMOBAC\n\n____________________________________________________________________________\n\nEntrainment of heterogeneous glycolytic oscillations in single cells(id: 375 | type: studies)\n\nDescription: missing\n\n\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATION\nPhase shift responses in isolated yeast glycolytic oscillators\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nPUBLICATIONS\nEntrainment of heterogeneous glycolytic oscillations in single cells\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nMap with simulation results(id: 1061 | type: data_files)\n\nDescription: The abscissa of the plots shows the percentage of aerobiosis that is a physiological measure for oxygen availability (http://www.ncbi.nlm.nih.gov/pubmed/11844770).\r\n\r\n1) Grey Boxes: Enzymes & Reactions\r\nblue lines/symbols: flux in mmol per gramm dry cell weight an hour\r\nred lines/symbols: mRNA levels\r\n\r\n2) White Boxes: Intracellular and extracellular metabolites\r\nblue lines/symbols: concentration of the metabolites (extracellular: mM, intracellular: AU)\r\n\r\n3) Yellow Boxes:\r\nAggregated Quantities as yield, percentage of reduction of quinones, etc.\r\n\r\n\n\n\nASSAYS\nSteady State Oxygen Response of E. coli WT and two Electron Transport Chain Mutants\nCREATORS\nMichael Ederer\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nPEOPLE\nMichael Ederer\nPROJECTS\nSUMO\nSTUDIES\nThe Escherichia coli steady state response to oxygen: from molecular interactions to regulatory modules\nSUBMITTER\nMichael Ederer\n\n____________________________________________________________________________\n\nProtein report on identification of interaction partners of TCA cycle enzymes in a SPINE experiment(id: 1052 | type: data_files)\n\nDescription: List of protein identified proteins in SPINE experiment.\n\n\nCREATORS\nJoerg Stuelke\nUwe Voelker\nPEOPLE\nJoerg Stuelke\nUwe Voelker\nLeif Steil\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nPhysical interactions between tricarboxylic acid cycle enzymes in Bacillus subtilis: Evidence for a metabolon\nSUBMITTER\nLeif Steil\n\n____________________________________________________________________________\n\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation(id: 175 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nModelling sustained glycolytic oscillations in individual isolated yeast cells (Gustavsson models)\nDetailed kinetic model of yeast glycolytic oscillation\nDATA_FILES\nRichard (1996) Concentrations of oscillating glycolytic intermediates in yeast cultures\nDas (1991) NADH and ATP concentrations determined during different phases of glycolytic oscillation in yeast extracts\nDas (1991) Time varying concentrations for metabolites and cofactors in upper glycolysis measured in oscillating yeast extracts\nMODELS\nDetailed kinetic model of yeast glycolytic oscillation - I\nDetailed kinetic model of yeast glycolytic oscillation - II\nDetailed kinetic model of yeast glycolytic oscillation - III\nDetailed kinetic model of yeast glycolytic oscillation - IV\nDetailed kinetic model of yeast glycolytic oscillation - V\nDetailed kinetic model of yeast glycolytic oscillation - VI\nDetailed kinetic model of yeast glycolytic oscillation - VII\nSustained glycolytic oscillations in individual isolated yeast cells - IV\nSustained glycolytic oscillations in individual isolated yeast cells - III\nSustained glycolytic oscillations in individual isolated yeast cells - II\nSustained glycolytic oscillations in individual isolated yeast cells - I\nPEOPLE\nJacky Snoep\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nJan-Willem Veening(id: 342 | type: people)\n\nDescription: The Veening lab is interested in phenotypic bi-stability in Streptococcus pneumoniae and its importance in virulence of this human pathogen. \n\n\nASSAYS\nParB-GFP ChIP on chip\nRNA-Seq\nLive Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy.\nWhole genome sequencing (Illumina)\nDATA_FILES\nChIP-chip ParB-GFP\nRNA-seq wild type\nRNA-seq delta greA\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nWhole genome marker frequency analysis on strain MK350\nWhole genome marker frequency analysis on strain MK422\nWhole genome marker frequency analysis on strain MK423\nDCI15\nMK453\nMK454\nINSTITUTIONS\nUniversity of Groningen\nINVESTIGATIONS\nWetlab approach to transcription fidelity\nModeling approach to transcription fidelity\nChromosome segregation and cell division in Streptococcus pneumoniae\nPROJECTS\nNoisy-Strep\nPUBLICATIONS\nHeterochronic phosphorelay gene expression as a source of heterogeneity in Bacillus subtilis spore formation\nStepwise mechanism for transcription fidelity\nGene position within a long transcript as a determinant for stochastic switching in bacteria\nDnaA and ORC: more than DNA replication initiators\nTransformation of environmental Bacillus subtilis isolates by transiently inducing genetic competence\nCellular localization of choline-utilization proteins in Streptococcus pneumoniae using novel fluorescent reporter systems\nA mechanism for cell cycle regulation of sporulation initiation in Bacillus subtilis\nControlled interplay between trigger loop and Gre factor in the RNA polymerase active centre\nSMC is recruited to oriC by ParB and promotes chromosome segregation in Streptococcus pneumoniae\nLive Cell Imaging of <em>Bacillus subtilis</em> and <em>Streptococcus pneumoniae</em> using Automated Time-lapse Microscopy\nA new basal promoter element recognized by RNA polymerase core enzyme\nFactor-independent transcription pausing caused by recognition of the RNA-DNA hybrid sequence\nIn vitro experimental system for analysis of transcription-translation coupling\nNonlinear fitness landscape of a molecular pathway\nControl of cell division in Streptococcus pneumoniae by the conserved Ser/Thr protein kinase StkP\nHypothesis: emergence of translation as a result of RNA helicase evolution\nMultiple active centers of multi-subunit RNA polymerases\nSingle cell analysis of gene expression patterns during carbon starvation in Bacillus subtilis reveals large phenotypic variation\nGraphAlignment: Bayesian pairwise alignment of biological networks\nBenchmarking various green fluorescent protein variants in Bacillus subtilis, Streptococcus pneumoniae, and Lactococcus lactis for live cell imaging\nControl of transcription elongation by GreA determines rate of gene expression in Streptococcus pneumoniae\nWhat makes the lac-pathway switch: identifying the fluctuations that trigger phenotype switching in gene regulatory systems\nSTUDIES\nChromosome segregation in S. pneumoniae\nThe role of transcription factor GreA in transcription fidelity and processivity in Streptococcus pneumoniae\nAutomated time-lapse microscopy\nGenome wide marker frequency analysis on new strains to image genetic loci in Streptococcus pneumoniae\n\n____________________________________________________________________________\n\nComputational processing and error reduction strategies for standardized quantitative data in biological networks(id: 286 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nData Management and Data Standardization\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nSysMetEx Poster @IBS2015(id: 186 | type: presentations)\n\nDescription: Stephan's poster for presentation at IBS 2015\n\n\nCREATORS\nStephan Christel\nPEOPLE\nStephan Christel\nPROJECTS\nSysMetEx\nSUBMITTER\nStephan Christel\n\n____________________________________________________________________________\n\nJan Maarten Van Dijl(id: 276 | type: people)\n\nDescription: The main area of my expertise concerns protein sorting and secretion in Gram-positive bacteria, such as Bacillus subtilis and Staphylococcus aureus.\r\n\r\nThe Gram-positive bacterium B. subtilis is well known for its high capacity to secrete proteins into the extracellular milieu, which has led to its exploitation as a \"cell factory\" for secreted proteins. Nevertheless, the secretion of heterologous proteins of pharmaceutical importance is frequently inefficient. This applied problem has been a major incentive for fundamental research on the mechanisms of protein transport in B. subtilis. For this purpose, molecular biological, phylogenetic, genomic and proteomic techniques were employed in an integrated way to study the mechanisms of protein secretion in B. subtilis. More recently Systems Biological approaches were implemented for the research on protein secretion, including so-called live cell array technology. Important research outcomes were: (i) characterisation of a Gram-positive Tat pathway for protein transport; (ii) novel insights in the roles of lipid-modification in membrane-attachment and function of lipoproteins of Gram-positive bacteria; (iii) a strategy for improved disulphide bond formation in secreted proteins; (iv) description of the B. subtilis response to protein overproduction and secretion stress; and (v) bioinformatic and proteomic descriptions of the B. subtilis secretome. Ongoing research is focused on the dynamic behaviour of the secretome in response to changing environmental conditions.\r\n\r\nStaphylococcus aureus is an important hospital- and community-acquired opportunistic pathogen. Recent research on protein sorting and secretion in S. aureus has yielded a state-of-the-art roadmap of the S. aureus secretome. The secretome map includes both protein transport pathways and the extracytoplasmic proteins. This has resulted in overviews of the exported virulence factors, pathways for protein transport, signals for cellular protein retention or secretion, and the exoproteomes of clinical S. aureus isolates. Importantly, the results include a first definition of the core and variant secretomes of S. aureus. While the core secretome seems to be largely employed for general house-keeping functions, necessary to thrive in particular niches provided by the human host, the variant secretome seems to contain the \"gadgets\" that S. aureus needs to conquer these well-protected niches (Sibbald et al., 2006). Ongoing research is aimed at analysing how the different secretion pathways of S. aureus impact on exoproteome biogenesis, fitness, resistance to antibiotics, adhesion to biomaterials and cell components, biofilm formation and virulence.\r\n\n\n\nDATA_FILES\nThe condition-dependent transcriptome of Bacillus subtilis 168\nINSTITUTIONS\nUniversity Medical Centre Groningen\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nThe twin-arginine translocation (Tat) systems from Bacillus subtilis display a conserved mode of complex organization and similar substrate recognition requirements\nInterchangeable modules in bacterial thiol-disulfide exchange pathways\nModulation of thiol-disulfide oxidoreductases for increased production of disulfide-bond-containing proteins in Bacillus subtilis\nGenetic or chemical protease inhibition causes significant changes in the Bacillus subtilis exoproteome\nTowards the development of Bacillus subtilis as a cell factory for membrane proteins and protein complexes\nImmunity to the bacteriocin sublancin 168 Is determined by the SunI (YolF) protein of Bacillus subtilis\nOverflow of a hyper-produced secretory protein from the Bacillus Sec pathway into the Tat pathway for protein secretion as revealed by proteogenomics\nMscL of Bacillus subtilis prevents selective release of cytoplasmic proteins in a hypotonic environment\nThe large mechanosensitive channel MscL determines bacterial susceptibility to the bacteriocin sublancin 168\nApplications of thiol-disulfide oxidoreductases for optimized in vivo production of functionally active proteins in Bacillus\nContributions of the pre- and pro-regions of a Staphylococcus hyicus lipase to secretion of a heterologous protein by Bacillus subtilis\nProtein transport across and into cell membranes in bacteria and archaea\nStress-responsive systems set specific limits to the overproduction of membrane proteins in Bacillus subtilis\nRecombination between ccrC genes in a type V (5C2&5) staphylococcal cassette chromosome mec (SCCmec) of Staphylococcus aureus ST398 leads to conversion from methicillin resistance to methicillin susceptibility in vivo\nRequirement of the agr locus for colony spreading of Staphylococcus aureus\nStaphylococcal PknB as the first prokaryotic representative of the proline-directed kinases\nCondition-dependent transcriptome reveals high-level regulatory architecture in Bacillus subtilis\nFunctional analysis of the sortase YhcS in Bacillus subtilis\nEnvironmental salinity determines the specificity and need for Tat-dependent secretion of the YwbN protein in Bacillus subtilis\nNovel twin-arginine translocation pathway-dependent phenotypes of Bacillus subtilis unveiled by quantitative proteomics\n\n____________________________________________________________________________\n\nMalate metabolism in Bacillus subtilis: distinct roles for three classes of malate-oxidizing enzymes(id: 188 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoerg Stuelke\nLeif Steil\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nWhat makes the lac-pathway switch: identifying the fluctuations that trigger phenotype switching in gene regulatory systems(id: 229 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nRobin Sorg\nJan-Willem Veening\nJohannes Berg\nPrasanna    Bhogale\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nTrophozoite Isolation and Lysate Preparation(id: 205 | type: sops)\n\nDescription: SOP for the isolation of intact Plasmodium falciparum trophozoites from infected red blood cells and the preparation of a cell free extract that can be used for kinetic analyses.\n\n\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nsigB-response_starvation_shakeflask(id: 20 | type: models)\n\nDescription: The model can simulate the the dynamics of sigB dependent transcription at the transition to starvation. It is was developed along the comic in <Data> 'sigB-activation-comic_vol1'. Parameters were partly taken from Delumeau et al., 2002, J. Bact. and Igoshin et al., 2007, JMB. Parameter estimation was performed using experimental data from <Assay> '0804_shake-flask'. \r\nUse the .m-file with matlab as:\r\n% reading initial conditions from the file:\r\ninic = sigb_model_liebal; \r\n\r\n% performing the simulation:\r\n[t,y] = ode15s(@sigb_model_liebal,[0 800],inic);\r\n\r\n% plotting Biomass and LacZ activity\r\n% (blue by default biomass, green LacZ activity)\r\nplotyy(t/60,y(:,1),t/60,y(:,10))\n\n\nCREATORS\nUlf Liebal\nPEOPLE\nUlf Liebal\nPROJECTS\nBaCell-SysMO\nSUBMITTER\nUlf Liebal\n\n____________________________________________________________________________\n\nGlucose pulsed L. lactis(id: 72 | type: data_files)\n\nDescription: Glucose pulsed L. lactis with 20 mM glucose in 100 mM MES buffer at pH 6.5 and followed intracellular metabolites in time\n\n\nASSAYS\nGlucose pulsed L. lactis\nCREATORS\nMartijn Bekker\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSTUDIES\nComparative modeling and phosphate dependence flux distributions and glucose consumption rates of two lactic acid bacteria S. pyogenes and L. lactis\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nVolume determination during starvation at different times.(id: 60 | type: assays)\n\nDescription: Is cell volume affected by potassium starvation? \r\n\r\nThe volume reduction as a response to the shift of cells from a medium of high potassium to potassium free medium will be studied in mutants lacking certain membrane transporters like Trk1,2 Nha1, etc. The conditions for the experiments follow Navarrete et al. (2010). Additionally knockouts of related regulation proteins (SAP155, SAP185) will be tested. For each mutant several time points will be measured to generate time courses. \n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nKey Cell-Physiological Parameters\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "SUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nThomas Millat(id: 26 | type: people)\n\nDescription: Modelling of cellular signalling, Dynamic Motifs and Feedback, Quantitative Measures, Theoretical Aspects of Modelling Biological Systems\n\n\nINSTITUTIONS\nUniversity of Rostock\nMODELS\npH-dependent steady states of ABE fermetation in C. acetobutylicum\nParameter estimation for a function describing the changing pH level during the pH-induced metabolic shift\nEstimation of an acidogenic and solventogenic population from optical density data observed in 'forward'-shift experiments using continuous cultures of C. acetobutylicum\nModel of the pH-induced metabolic shift in phosphate-limited continuous cultures of C. acetobutylicum assuming a phenotypic switch\nPRESENTATIONS\nAutomated Flow Cytometry Data Analysis\nThe pH-induced metabolic shift from acidogenesis to solventogenesis in Clostridium acetobutylicum - From experiments to models\nMathematical Modelling of the pH-induced Metabolic Shift in C. aceobutylicum Unravels a Heterogeneous Phase Transition\nConclusion of the Stressosome Models\nButyrate re-assimilation in C. acetobutylicum re-evaluated using modeling\nPROJECTS\nBaCell-SysMO\nCOSMIC\nPUBLICATIONS\nHow mathematical modelling elucidates signalling in B. subtilis\nHow quantitative measures unravel design principles in multi-stage phosphorylation cascades\nThe role of dynamic stimulation pattern in the analysis of bistable intracellular networks\nDynamics of receptor and protein transducer homodimerisation\nA mathematical analysis of nuclear intensity dynamics for Mig1-GFP under consideration of bleaching effects and background noise in Saccharomyces cerevisiae\nA systems biology approach to investigate the effect of pH-induced gene regulation on solvent production by Clostridium acetobutylicum in continuous culture\nModeling of cellular processes: methods, data, and requirements\nGenome-wide gene expression analysis of the switch between acidogenesis and solventogenesis in continuous cultures of Clostridium acetobutylicum\nProteolysis of beta-galactosidase following SigmaB activation in Bacillus subtilis\nSimulations of stressosome activation emphasize allosteric interactions between RsbR and RsbT\nIntegrative modelling of pH-dependent enzyme activity and transcriptomic regulation of the acetone-butanol-ethanol fermentation of\n              Clostridium acetobutylicum\n              in continuous culture\nA shift in the dominant phenotype governs the pH-induced metabolic switch of Clostridium acetobutylicumin phosphate-limited continuous cultures\n\n____________________________________________________________________________\n\nAllosteric regulation of phosphofructokinase controls the emergence of glycolytic oscillations in isolated yeast cells(id: 375 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nSTUDIES\nAllosteric regulation of phosphofructokinase controls the emergence of glycolytic oscillations in isolated yeast cells\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, CCA1 degr D - PLM_86, version 1, SIMPLIFIED(id: 512 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_86, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, CCA1 degr D - PLM_86, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, CCA1 degr D - PLM_86\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nMesenchymal stromal cells inhibit proliferation of virus-specific CD8(+) T cells(id: 278 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nPEOPLE\nMaja Rey\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nRaffaele Nicastro(id: 536 | type: people)\n\nDescription: I am a PostDoc working on yeast metabolomics. During my PhD I studied the interplay between metabolism, cell cycle and signalling, mainly focusing on the Snf1/AMPK pathway. I am currently interested in studying metabolic rewiring caused by different nutrients, generating high-throughput data suitable for modelling.\n\n\nINSTITUTIONS\nUniversity of Milano-Bicocca\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nStefan Stagge(id: 203 | type: people)\n\nDescription: missing\n\n\nASSAYS\nDetermination of intracellular metabolite concentrations\nDetermination of intracellular redox state by means of NAD/NADH ratio\nDetermination of intracellular redox state by means of ubiquinones (oxd/red-ratio)\nDATA_FILES\n091016 quinones complete data magdeburg\n091016 quinones complete data magdeburg\nINSTITUTIONS\nMax Planck Institute for Dynamics of Complex Technical Systems\nPROJECTS\nSUMO\nPUBLICATIONS\nCharacterization of E. coli MG1655 and frdA and sdhC mutants at various aerobiosis levels\nAnalysis of Escherichia coli mutants with a linear respiratory chain\nBasic Regulatory Principles of Escherichia coli's Electron Transport Chain for Varying Oxygen Conditions\nA mathematical model of metabolism and regulation provides a systems-level view of how Escherichia coli responds to oxygen\nSOPS\nQuenching and Extraction of intracellular metabolites\n\n____________________________________________________________________________\n\nDesigning a new way to predict engineering strategies in cells.(id: 58 | type: investigations)\n\nDescription: missing\n\n\nASSAYS\nRobOKoD applied to e.coli for butanol production.\nOptKnock applied to e.coli for butanol production.\nRobustKnock applied to e.coli for butanol production.\nDATA_FILES\nFBA result of RobOKoD designed e.coli strain.\nMODELS\niNS142 RobOKoD Redesigned Butanol Producing.\nPEOPLE\nNatalie Stanford\nPROJECTS\nManchester Institute for Biotechnology\nPUBLICATIONS\nRobOKoD: microbial strain design for (over)production of target compounds\nSOPS\nRobOKoD SOP for Redesigning Butanol Producing Strain of E.coli.\nOptKnock SOP for Redesigning Butanol Producing Strain of E.coli.\nRobustKnock SOP for Redesigning Butanol Producing Strain of E.coli.\nSTUDIES\nPredicting engineering strategies for overproduction of butanol in ecoli.\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nMonomer Stressosome Model(id: 88 | type: models)\n\nDescription: The zip folder contains files that allow simulation of stressosome dynamics. The models are based on a cellular automaton approach. Each protein of RsbR and RsbS is located in the crystal structure of the stressosome. The proteins can be phosphorylated or not and these states determine the future of neighbouring proteins. To simulate the model open the file 'liebal_stressosome-model_12_workflow-matlab.m' in Matlab. It is written in the cell-model, put the cursor into a cell that you wish to evaluate and press 'Ctrl + Enter' to simulate.\n\n\nASSAYS\nStressosome activation dynamics\nCREATORS\nUlf Liebal\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nPEOPLE\nUlf Liebal\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nSimulations of stressosome activation emphasize allosteric interactions between RsbR and RsbT\nSTUDIES\nControlled sigmaB induction in shake flask\nSUBMITTER\nUlf Liebal\n\n____________________________________________________________________________\n\nCloning and heterologous expression in E. coli(id: 186 | type: sops)\n\nDescription: Cloning and heterologous expression of gluconeogenic enzymes from S. solfataricus in E. coli\n\n\nASSAYS\nPGK\nGAPDH\nTPI\nFBPAase\nCREATORS\nJacky Snoep\nTheresa Kouril\nINVESTIGATIONS\nCentral Carbon Metabolism of Sulfolobus solfataricus\nPEOPLE\nJacky Snoep\nTheresa Kouril\nPROJECTS\nSulfoSys\nSTUDIES\nModel Gluconeogenesis\nSUBMITTER\nJacky Snoep\n\n____________________________________________________________________________\n\nCation content analysis - uco(id: 108 | type: sops)\n\nDescription: Purpose: Cation content analysis - protocol\n\n\n\nCREATORS\nClara Navarrete\nPEOPLE\nClara Navarrete\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nBaCell-SysMO(id: 1 | type: projects)\n\nDescription: BaCell-SysMO 2\r\nModelling carbon core metabolism in Bacillus subtilis  Exploring the contribution of protein complexes in core carbon and nitrogen metabolism.\r\n\r\nBacillus subtilis is a prime model organism for systems biology approaches because it is one of the most advanced models for functional genomics. Furthermore, comprehensive information on cell and molecular biology, physiology and genetics is available and the European Bacillus community (BACELL) has a well-established reputation for applying this information and technologies to address fundamental issues in life sciences. \r\nBeyond its scientific relevance B. subtilis is an industrial workhorse of white biotechnology as a main producer of technical enzymes and other products.\r\nIn the first funding period we have used a systems biology approach to analyze the transition from exponential phase to stationary phase. Within the context of this project we have analyzed both modules of gene expression as well as metabolic changes. One of the major findings was the discovery that metabolic and regulatory activities do not appear to be dispersed in the cell, but rather exist in defined and distinct protein complexes. The complexes that we have detected in B. subtilis include the glycolytic enzymes PFK, PGM, and ENO (the glycosome) and an RNA degrading machine, the degradosome. These findings contrast with the text-book view that many processes occur in dedicated compartments in eukaryotic cells, whereas little internal structure is present in prokaryotes. However, the existence of such complexes has major consequences for our understanding of the biology of B. subtilis and other bacteria since they may affect the local concentrations of not just the metabolites, but also the proteins themselves within a given pathway and thus impact on the reaction kinetics. \r\nTherefore, we want to concentrate our efforts in the second funding period onto the elucidation of the contribution of protein complexes to the function of central carbon and nitrogen metabolism during nutrient shifts and environmental adaptations. The analysis of the impact of metabolic complexes on cellular physiology is highly relevant and an excellent topic for systems biology. So far, it has been possible to model metabolism of B. subtilis with the inherent assumption that both metabolites and enzymes are more or less evenly distributed in the cell. However, since the existence of enzyme complexes would have a dramatic impact on the kinetics of the reactions and the relevant local concentrations of metabolites, we would like to address the so far largely unexplored but important issue of (micro)compartmentalisation using a new set of experimental and modelling approaches.  We will combine experimental monitoring of in vivo complexes with conventional metabolic modelling to identify nodes in which protein complexes are functionally important. These complexes and their dynamics will then be studied in detail by a combination of in vivo and in vitro approaches up to crystallization of selected complexes to provide data which will allow refinement of the models generated.\r\nMain goals\r\n-\tElucidation of the functional importance of protein complexes for the physiology of B. subtilis using its central metabolism as example\r\n-\tContribution to the general understanding of metabolic compartments in bacteria, leading to new strategies for metabolic modeling that take the existence and importance of such complexes into account\r\n-\tProviding a framework for the analysis of functionally important protein complexes that can then be extended to other bacteria including related low-GC Gram-positive pathogens\r\n\n\n\nASSAYS\nGene expression(Transcriptome)\nmetabolome-LCMS\n2D-gelbased analysis of intracellular proteins\nRelative quantification of proteins by metabolic labeling\nAbsolute quantification of proteins by the AQUA-technology\nIPTG induction of sigmaB in BSA115\nB. subtilis Transcription Factor Competition - theoretical interpretation\nFermentation-BM5_SysMo\nTiling array analysis of continuous growth stress conditions in SMM\nTiling Array analysis of shocked B. subtilis cells\nTiling Array analysis of glucose strarved B. subtilis cells\nRelative quantification of proteins by metabolic labeling\nAbsolute quantification of proteins by the AQUA-technology\nB. subtilis Transcription Factor Competition - theoretical interpretation\nIPTG induction of sigmaB in BSA115\nBinding constants for CcpA with HPrSer46P interacting with various cre-elements\nTranscriptome data for chemostat cultivated samples\n13C Metabolic Flux Analysis of Bacillus subtilis 168 in continuous high-salinity culture\nAbsolute quantification of proteins using QconCAT technology\nextracellular metabolites\nintracellular metabolites\nTranscritome data_Identification of targets of the essential RNase Y of Bacillus subtilis\nStressosome activation dynamics\nDATA_FILES\n0804_shake-flask-sigB-starvation\nbatchfermentation exp-starv01 transcriptome_Microarray Agilent platform_090204\nbatchfermentation exp-starv01 quant-proteomics\nB. subtilis Transcription Factor Competition - computational data\nTrancriptome 20080827\nbatchfermentation exp-starv01 transcriptome_Microarray Nimblegen platform_090204\nBatchfermentation_Exp-Starv01_Fermentationdetails_090204\nContious growth_57C_SMM_Transcritome_Nimblegen\nContious growth_16C_SMM_Transcritome_Nimblegen\nContious growth_1.2M NaCl_SMM_Transcritome_Nimblegen\nM9_shake_flask_exp_trans_stat_transcriptome_nimblegen\nContinuous growth_1.2M NaCl_SMM_Growth Curve\nContinuous growth_16C_SMM_Growth curve\nContinuous growth_57C_SMM_Growth curve\nContiuous growth_37C_SMM(control)_Growth curve.\nBatchfermentation_exp-starv01_2D-proteomics\n090204 batchfermentation metabolomics\nB. subtilis_SysMo2_Chemostat_growthrate-salt_salt_+GB_proteome_relative quantification\nB. subtilis_SysMo2_Chemostat_growthrate-salt_salt_-GB_proteome_relative quantification\nB. subtilis_SysMo2_Chemostat_growhtrate-salt_cell count\n20090915_BSA115-IPTG-assay\nModel-representations-BSA115-Experiments\nAssociation and dissociation kinetics of the CcpA-HPrSer46P-complex binding to various cre elements\nFlux_analysis_B_subtilis_chemostat_reference\nFlux_analysis_B_subtilis_chemostat_osmoprotection\nFlux_analysis_B_subtilis_chemostat_stress\nB. subtilis_SysMo2_Chemostat_growhtrate-salt_cell size\nextracellular metabolites\nintracellular metabolites LC/MS\nintracellular metabolites GC/MS\nTrancriptome data - RNA processing in Bacillus subtilis: identification of targetsof the essential RNase Y\nProtein report on identification of interaction partners of TCA cycle enzymes in a SPINE experiment\nPhosphorylation site of the HPr protein\nEnzymatic activities of the RocG variants\nRNA processing in Bacillus subtilis: Identification of targets of the essential RNase Y\nmRNAs with decreased abundance in RNaseY mutant\nProtein report on identification of interaction partners of CitG in a SPINE experiment\nThe condition-dependent transcriptome of Bacillus subtilis 168\nCatabolite repression of -xylosidase by different carbon sources in various mutants\nCatabolite repression of -xylosidase by different carbon sources in various mutants\nHigh- and low-affinity cre boxes for CcpA binding in Bacillus subtilis revealed by genome-wide analysis\nMetabolome Data\nStressosome activation and structural representations\nEVENTS\nBacell-meeting 1109-Groningen\nBaCell Conference\nINSTITUTIONS\nDSM Nutritional Products Ltd.\nETH Zurich & Basel / The Scientific IT Services (SIS) division\nInstitut National de la Recherche Agronomique\nUniversity Medical Centre Groningen\nUniversity of Braunschweig\nUniversity of Erlangen-Nuernberg\nUniversity of Goettingen\nUniversity of Greifswald\nUniversity of Groningen\nManchester Centre for Integrative Systems Biology, University of Manchester\nUniversity of Newcastle\nUniversity of Newcastle-upon-Tyne\nUniversity of Rostock\nUniversity of Stuttgart\nUniversity of Marburg\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nA dynamic model of the CcpA regulation network\nMultiomics study of Bacillus subtilis under osmotic stress\nIdentification of targets of the essential RNase Y of Bacillus subtilis\nMODELS\nsigB-response_starvation_shakeflask\nBSA115-sigB-adaptation\nTranscription factor competition theoretical analysis\nBSA115 Post-transcriptional instability\nBSA sigB-proteolysis model\nBSA115-transcription inhibition model\nBSA115-lacz-synth\ne.coli-central-carbon_chassagnole-et-al\nStressosome-dynamics\nMonomer Stressosome Model\nORGANISMS\nBacillus subtilis\nPEOPLE\nJoerg Stuelke\nMarkus Wyss\nMichael Hecker\nOlaf Wolkenhauer\nOscar Kuipers\nUwe Voelker\nThomas Millat\nMalkhey Verma\nMatthias Reuss\nLeif Steil\nJan Maarten Van Dijl\nUlf Liebal\nImke De Jong\nColin Harwood\nBeate Knoke\nPraveen Kumar Sappa\nLope A. Florez\nGerald Seidel\nSandra Maass\nNadine Steinert\nMaike  Bartholomae\nMaria Nikolakakou\nBogumila Marciniak\nJoseph Newman\nHenrik Strahl Von Schulten\nLeendert Hamoen\nRick Lewis\nChristoph Wittmann\nPriyanka Nannapaneni\nHanna Meyer\nWolfgang Hillen\nMichael Lalk\nRuben Mars\nTamara Hoffmann\nErhard Bremer\nMichael Kohlstedt\nFei He\nUlrike Mder\nAdrienne Zaprasis\nRaphael Michna\nPRESENTATIONS\nFeed-back regulation of the anabolic (ProB) and osmoadaptive (ProJ) glutamate kinases: predicted role of a flexible protein loop.\nBalancing the osmotic gradient: adaptation of the cytoplasmic solute pool via de novo synthesis and compatible solute uptake in response to hyper-osmolarity.\nTowards system understanding of B. subtilis - integrating fluxes and cellular components under salt stress\nAutomated Flow Cytometry Data Analysis\nEffect of concentration changes for channeling enzymes\nBiophysical analysis of putative interactions between TCA cycle enzymes of B. subtilis\nUpdate on proteome data in chemostat and shake flask experiments\nConclusion of the Stressosome Models\nsigB-activation-comic_vol1\nPROGRAMMES\nSysMO\nPUBLICATIONS\nHow mathematical modelling elucidates signalling in B. subtilis\nPenicillin-binding protein folding is dependent on the PrsA peptidyl-prolyl cis-trans isomerase in Bacillus subtilis\nClpP modulates the activity of the Bacillus subtilis stress response transcription factor, sigmaB\nGlutamate metabolism in Bacillus subtilis: gene expression and enzyme activities evolved to avoid futile cycles and to allow rapid responses to perturbations of the system\nMonitoring of changes in the membrane proteome during stationary phase adaptation of Bacillus subtilis using in vivo labeling techniques\nHow quantitative measures unravel design principles in multi-stage phosphorylation cascades\nCarbon catabolite repression in bacteria: many ways to make the most out of nutrients\nGel-based proteomics of Gram-positive bacteria: a powerful tool to address physiological questions\nLocalization of general and regulatory proteolysis in Bacillus subtilis cells\nInterchangeable modules in bacterial thiol-disulfide exchange pathways\nModulation of thiol-disulfide oxidoreductases for increased production of disulfide-bond-containing proteins in Bacillus subtilis\nThe role of dynamic stimulation pattern in the analysis of bistable intracellular networks\nCarbon catabolite repression in Bacillus subtilis: quantitative analysis of repression exerted by different carbon sources\nThe Spx paralogue MgsR (YqgZ) controls a subregulon within the general stress response of Bacillus subtilis\nGenetic or chemical protease inhibition causes significant changes in the Bacillus subtilis exoproteome\nTowards the development of Bacillus subtilis as a cell factory for membrane proteins and protein complexes\nImmunity to the bacteriocin sublancin 168 Is determined by the SunI (YolF) protein of Bacillus subtilis\nGenome-wide responses to carbonyl electrophiles in Bacillus subtilis: control of the thiol-dependent formaldehyde dehydrogenase AdhA and cysteine proteinase YraA by the MerR-family regulator YraB (AdhR)\nOverflow of a hyper-produced secretory protein from the Bacillus Sec pathway into the Tat pathway for protein secretion as revealed by proteogenomics\nMscL of Bacillus subtilis prevents selective release of cytoplasmic proteins in a hypotonic environment\nIn vitro phosphorylation of key metabolic enzymes from Bacillus subtilis: PrkC phosphorylates enzymes from different branches of basic metabolism\nMINOMICS: visualizing prokaryote transcriptomics and proteomics data in a genomic context\nNovel activities of glycolytic enzymes in Bacillus subtilis: interactions with essential proteins involved in mRNA processing\nHeterochronic phosphorelay gene expression as a source of heterogeneity in Bacillus subtilis spore formation\nA community-curated consensual annotation that is continuously updated: the Bacillus subtilis centred wiki SubtiWiki\nThe large mechanosensitive channel MscL determines bacterial susceptibility to the bacteriocin sublancin 168\nApplications of thiol-disulfide oxidoreductases for optimized in vivo production of functionally active proteins in Bacillus\nContributions of the pre- and pro-regions of a Staphylococcus hyicus lipase to secretion of a heterologous protein by Bacillus subtilis\nConnecting parts with processes: SubtiWiki and SubtiPathways integrate gene and pathway annotation for Bacillus subtilis\nA rapid microwave-assisted derivatization of bacterial metabolome samples for gas chromatography/mass spectrometry analysis\nDiamide triggers mainly S Thiolations in the cytoplasmic proteomes of Bacillus subtilis and Staphylococcus aureus\nMechanisms and evolution of control logic in prokaryotic transcriptional regulation\nProtein transport across and into cell membranes in bacteria and archaea\nStress-responsive systems set specific limits to the overproduction of membrane proteins in Bacillus subtilis\nFunctional dissection of a trigger enzyme: mutations of the bacillus subtilis glutamate dehydrogenase RocG that affect differentially its catalytic activity and regulatory properties\nPhysiological proteomics and stress/starvation responses in Bacillus subtilis and Staphylococcus aureus\nIntegration of sensitivity and bifurcation analysis to detect critical processes in a model combining signalling and cell population dynamics\nDynamics of receptor and protein transducer homodimerisation\nEin Beitrag zu einer Theorie lebender Zellen (A Contribution towards a Theory of Living Cells)\nPhysical interactions between tricarboxylic acid cycle enzymes in Bacillus subtilis: Evidence for a metabolon\nModeling of cellular processes: methods, data, and requirements\nStructures of carbon catabolite protein A-(HPr-Ser46-P) bound to diverse catabolite response element sites reveal the basis for high-affinity binding to degenerate DNA operators\nRecombination between ccrC genes in a type V (5C2&5) staphylococcal cassette chromosome mec (SCCmec) of Staphylococcus aureus ST398 leads to conversion from methicillin resistance to methicillin susceptibility in vivo\nRequirement of the agr locus for colony spreading of Staphylococcus aureus\nStaphylococcal PknB as the first prokaryotic representative of the proline-directed kinases\nCondition-dependent transcriptome reveals high-level regulatory architecture in Bacillus subtilis\nProteolysis of beta-galactosidase following SigmaB activation in Bacillus subtilis\nT-box-mediated control of the anabolic proline biosynthetic genes of Bacillus subtilis\nTrigger enzymes: bifunctional proteins active in metabolism and in controlling gene expression\nSignal Transduction by Trigger Enzymes: Bifunctional Enzymes and Transporters Controlling Gene Expression\nBet hedging or not? A guide to proper classification of microbial survival strategies\nFunctional analysis of the sortase YhcS in Bacillus subtilis\nSPABBATS: A pathway-discovery method based on Boolean satisfiability that facilitates the characterization of suppressor mutants\nCellPublisher: a web platform for the intuitive visualization and sharing of metabolic, signalling and regulatory pathways\nMetabolic fluxes and beyond-systems biology understanding and engineering of microbial metabolism\nCarbon source control of the phosphorylation state of the Bacillus subtilis carbon-flux regulator Crh in vivo\nCcpA forms complexes with CodY and RpoA in Bacillus subtilis\nRNase Y in Bacillus subtilis: a Natively disordered protein that is the functional equivalent of RNase E from Escherichia coli\nRNA processing in Bacillus subtilis: identification of targets of the essential RNase Y\nA metabolomics and proteomics study of the adaptation of Staphylococcus aureus to glucose starvation\nDepletion of thiol-containing proteins in response to quinones in Bacillus subtilis\nEnvironmental salinity determines the specificity and need for Tat-dependent secretion of the YwbN protein in Bacillus subtilis\nEfficient, global-scale quantification of absolute protein amounts by integration of targeted mass spectrometry and two-dimensional gel-based proteomics\nComprehensive identification and quantification of microbial transcriptomes by genome-wide unbiased methods\nSubtiWiki--a comprehensive community resource for the model organism Bacillus subtilis\nDissection of the network of interactions that links RNA processing with glycolysis in the Bacillus subtilis degradosome\nCellular Adjustments of Bacillus subtilis and Other Bacilli to Fluctuating Salinities\nMalate-mediated carbon catabolite repression in Bacillus subtilis involves the HPrK/CcpA pathway\nA high-frequency mutation in Bacillus subtilis: requirements for the decryptification of the gudB glutamate dehydrogenase gene\nMalate metabolism in Bacillus subtilis: distinct roles for three classes of malate-oxidizing enzymes\nRNA degradation in Bacillus subtilis: an interplay of essential endo- and exoribonucleases\nHigh- and low-affinity cre boxes for CcpA binding in Bacillus subtilis revealed by genome-wide analysis\nNovel twin-arginine translocation pathway-dependent phenotypes of Bacillus subtilis unveiled by quantitative proteomics\nThe DEAD-box RNA helicases in Bacillus subtilis have multiple functions and act independent from each other\nOsmotic control of opuA expression in Bacillus subtilis and its modulation in response to intracellular glycine betaine and proline pools\nOsmotically controlled synthesis of the compatible solute proline is critical for cellular defense of Bacillus subtilis against high osmolarity\nSynthesis, release, and recapture of compatible solute proline by osmotically stressed Bacillus subtilis cells\nOsmoprotection of Bacillus subtilis through Import and Proteolysis of Proline-Containing Peptides\nSimulations of stressosome activation emphasize allosteric interactions between RsbR and RsbT\nSOPS\nSOP for cDNA labelling and Microarray hybridizaton\nRNA isolation and quality check by Agilent bioanalyser\nTotal RNA isolation from B.Subtilis\n-Galactosidase assay\nSOP for cultivation of B.Subtilis\nNMR measurment\nShake flask cultivations of Bacillus subtilis 168 trp+ on M9-DHB medium\nSTUDIES\nBiphase Batch Fermentation(2009/02/04)\nTransition to starvation in shake flask\nControlled sigmaB induction in shake flask\nB. subtilis Transcription Factor Competition\nBatchfermentation exp-starv01_090204\nTranscriptome of continuously stressed B. subtilis\nTranscriptome of shocked B. subtilis cells\nTranscriptome analysis of glucose starvation in B. subtilis\nB. subtilis_SysMo2_Chemostat_growthrate-salt\nGlucose starvation specific responses  Quantification of thermodynamic and kinetic parameters of the CcpA regulation network\nFluxome analysis of Bacillus subtilis 168 under osmotic stress\nIdentification of targets of the essential RNase Y of Bacillus subtilis\n\n____________________________________________________________________________\n\nModelling reveals novel roles of two parallel signalling pathways and homeostatic feedbacks in yeast(id: 237 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJrg Schaber\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nNeurospora Circadian Clock 3-variable model - sinusoidal light oscillations - PLM_52, version 1, SUBMITTED(id: 487 | type: models)\n\nDescription: Originally submitted model file for PLaSMo accession ID PLM_52, version 1\n\n\nASSAYS\nNeurospora Circadian Clock 3-variable model - sinusoidal light oscillations - PLM_52, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nAdams, Richard\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nNeurospora Circadian Clock 3-variable model - sinusoidal light oscillations - PLM_52\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nCharacterization of chemostats from S. pyogenes wild type vs. arcA- and glnA-deletion mutants(id: 947 | type: data_files)\n\nDescription: S. pyogenes was grown in C-limited cultures at pH 6.5 and 7.5 and at a growth rate of 0.05\r\nThe glnA mutant strain shows decreased growth in low glutamine and excess glutamate conditions and no growth at all in low glutamine and low glutamate conditions. Preliminary results of glucose-limited chemostat cultures indicate a reversion of the pH dependency of the shift from homolactic to more mixed acid fermentation:\r\nwild type - lactate/formate ratio \t                at pH 6.5 = 11.8, at pH 7.5 = 2.8\r\nglnA mutant - lactate/formate ratio \tat pH 6.5 = 6,      at pH 7.5 = 22.9\r\nIn glucose limited chemostat cultures the arcA mutant strain shows more homolactic fermentation than the wild type under pH 6.5 and pH 7.5, although in both cases at lower pH there is a higher flux through lactate than at higher pH:\r\npH6.5 - lactate/formate ratio \t               wt = 11.9, arcA deletion = 24.1\r\npH7.5 - lactate/formate ratio \t               wt = 2.05, arcA deletion = 8.05\r\n\n\n\nASSAYS\nCharacterization of flux distribution of S. pyogenes M9 wild type and the arcA- and glnA-knockout mutant in continuous cultures\nCREATORS\nAntje Sieg\nSilvio Hering\nMartijn Bekker\nINVESTIGATIONS\nAmino acid metabolism of four LAB species: Streptococcus pyogenes, Lactococcus lactis, Enterococcus faecalis and Lactobacillus plantarum.\nPEOPLE\nMartijn Bekker\nAntje Sieg\nSilvio Hering\nPROJECTS\nSysMO-LAB\nSTUDIES\nArginine and Glutamine metabolism in S. pyogenes\nSUBMITTER\nAntje Sieg\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, TOC1 act L - PLM_91, version 1, SIMPLIFIED(id: 522 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_91, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, TOC1 act L - PLM_91, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, TOC1 act L - PLM_91\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nProtocol for growing cells for purification of KefF(id: 18 | type: sops)\n\nDescription: Growing Escherichia coli to express KefF by adding IPTG for purification and kinetics experiments.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nCecilia Brnnmark(id: 533 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nBernd Kreikemeyer(id: 46 | type: people)\n\nDescription: PD Dr. rer. nat. habil. Bernd Kreikemeyer \r\nis the head of a research group at the Inst. of Medical Microbiology and Hospital Hygiene focusing on the pathogenesis of the human LAB Streptococcus pyogenes, microbial biofilm biology, and oral microbiology. Streptococcus pyogenes research, which is the relevant part for this BMBF proposal, is focused on 1. Identification of GAS virulence factors, 2. Mechanisms of GAS host cell adherence, host cell internalization, cytotoxicity of GAS towards host cells (apoptosis, necrosis), and host cell persistence, 3. Genetic control of virulence factor expression, 4. Characterization of stand alone transcriptional response regulators, quorum-sensing mechanisms, and two-component signal transduction systems, 5. Uncovering of GAS regulatory networks (input signals, output), and 6. Characterization of the central metabolism in relationship to virulence. Since 1999 Dr. Kreikemeyer has published a total number of 42 publications and under his leadership the group has made many important and internationally recognized contributions to the GAS pathogenesis field.\r\n\r\nCurriculum Vitae\t\r\nPD Dr. rer. nat. habil. Bernd Kreikemeyer\r\nborn 16th August 1966 in Braunschweig, Germany\r\nEducation and Professional Development\t\r\n1987-1993\tStudies in biology at the echnical University Braunschweig\r\n1992-1993\tDiploma thesis: Functional characterization of the IgA-binding -antigen from group B streptococci\r\n1993-1996\tPhD thesis: Structural and functional characterization of the fibronectin-binding serum opacity factor from Streptococcus pyogenes (group A streptococci)\r\n11/2004\tHabilitation: Importance of stand alone transcriptional regulators, two-component signal transduction systems and regulatory networks for the human pathogenic bacterium Streptococcus pyogenes\r\nVenia Legendi: General and Molecular Microbiology\r\nSince 2001\tIndependent group leader of section Molecular pathogenesis of Gram-positive pathogens at the Institute of Medical Microbiology, University of Rostock\r\nAugust 2009\tOffer to accept a W2-faculty position as Professor for Molecular Bacteriology at the University of Rostock. Negotiations currently under way\r\n\r\nRecent publications\r\n\r\n(37)\tM. Oey, M. Lohse, B. Kreikemeyer, R. Bock (2009) Exhaustion of the chloroplast protein synthesis capacity by massive expression of a highly stable protein antibiotic. Plant J 57(3):436-45 \r\n(38)\tM. Nakata, T. Kller, K. Moritz, D. Ribardo, L. Jonas, K. S. McIver, T. Sumitomo, Y. Terao, S. Kawabata, A. Podbielski and B. Kreikemeyer* (2009) Mode of expression and functional characterization of the FCT-3 pilus region encoded proteins in the Streptococcus pyogenes serotype M49. Infect  Immun 77(1): 32-44 \r\n(39)\tH. Courtney, I. Ofek, T. Penfound, V. Nizet, M.A. Pence, B. Kreikemeyer, A. Podbielski, D. L Hasty, J. B Dale (2009) Relationship Between Expression of the Family of M proteins and Lipoteichoic Acid to Hydrophobicity and Biofilm Formation in Streptococcus pyogenes. PLoS One. 4(1):e4166\r\n(40)\tM. Oey, M. Lohse, L.B. Scharff, B. Kreikemeyer, R. Bock (2009) Plastid production of protein antibiotics against pneumonia via a new strategy for high-level expression of antimicrobial proteins. Proc Natl Acad Sci U S A. 106(16):6579-84 \r\n(41)\tT. Fiedler, B. Kreikemeyer*, V. Sugareva, S. Redanz, R. Arlt, K. Standar, A. Podbielski (2009) Impact of the Streptococcus pyogenes Mga regulator on human matrix protein binding and interaction with eukaryotic cells. Int J Med Microbiol (in press). \r\n(42)\tD.N. Wood , K.E. Weinstein, A. Podbielski , B. Kreikemeyer, J. P. Gaughan, S. Valentine, B. A. Buttaro. (2009) Generation of Metabolically Diverse Strains of Streptococcus pyogenes during Survival in Stationary Phase. J Bacteriol. 2009 Aug 7. [Epub ahead of print] PMID: 19666718\r\n\r\n\r\n\n\n\nINSTITUTIONS\nUniversity of Rostock\nPROJECTS\nSysMO-LAB\nPUBLICATIONS\nPhysiological characterization of three lactic acid bacteria and their isogenic ldh deletion mutants shows that these organisms are optimized for Yatp and not ATP formation rates at their physiological pH\nInsights into Streptococcus pyogenes pathogenesis from transcriptome studies\nRole of phosphate in the central metabolism of two lactic acid bacteria - a comparative systems biology approach\nRegulation of the Activity of Lactate Dehydrogenases from Four Lactic Acid Bacteria\nKinetic characterization of arginine deiminase and carbamate kinase from Streptococcus pyogenes M49\n\n____________________________________________________________________________\n\nBatchfermentation_Exp-Starv01_Fermentationdetails_090204(id: 91 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nFermentation-BM5_SysMo\nCREATORS\nPraveen Kumar Sappa\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSTUDIES\nBatchfermentation exp-starv01_090204\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nbatchfermentation exp-starv01 transcriptome_Microarray Nimblegen platform_090204(id: 90 | type: data_files)\n\nDescription: Array hybridisation was done at Roche Nimblegen Inc. It cannot be displayed over here.\n\n\nASSAYS\nGene expression(Transcriptome)\nFermentation-BM5_SysMo\nCREATORS\nPraveen Kumar Sappa\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSTUDIES\nBatchfermentation exp-starv01_090204\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nSustained glycolytic oscillations in individual isolated yeast cells(id: 174 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nModelling sustained glycolytic oscillations in individual isolated yeast cells (Gustavsson models)\nMODELS\nSustained glycolytic oscillations in individual isolated yeast cells - IV\nSustained glycolytic oscillations in individual isolated yeast cells - III\nSustained glycolytic oscillations in individual isolated yeast cells - II\nSustained glycolytic oscillations in individual isolated yeast cells - I\nPEOPLE\nJacky Snoep\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nProtein expression Cyp3a+Mt Western Blot_Kollevg(id: 2272 | type: data_files)\n\nDescription: Relative protein expression of the proteins Cyp3a and Mt in cod from the Kollevg study\n\n\nASSAYS\nWestern blot - protein expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nModel of K+ uptake by the Kdp system of E.coli(id: 49 | type: sops)\n\nDescription: A model to describe the effect of K+ uptake by KdpFABC on the two-component system KdpD/KdpE, which enables the identification of control principles of the Kdp system\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nTranscriptomics Gene Expression Excel Template (NimbleGen)(id: 933 | type: data_files)\n\nDescription: This template is for recording gene expression data from the NimbleGen platform. \r\nThis template was taken from the GEO website (http://www.ncbi.nlm.nih.gov/geo/info/spreadsheet.html) and modified to conform to the SysMO-JERM (Just enough Results Model) for transcriptomics.\r\nUsing these templates will mean easier submission to GEO/ArrayExpress and greater consistency of data in SEEK.\n\n\nASSAYS\nNimbleGen Transcriptomics Templates\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nMaria Nikolakakou(id: 292 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Erlangen-Nuernberg\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nTotal RNA isolation from B.Subtilis(id: 67 | type: sops)\n\nDescription: Isolation of total RNA from Bacillus Subtilis using phenol-chloroform extraction method by maintaining cryogenec conditions initailly to prevent RNA degradation. Quality of the obtained RNA is then tested with Agilent Bioanalyser before proceeding for gene expression analysis.\n\n\nASSAYS\nTiling Array analysis of glucose strarved B. subtilis cells\nCREATORS\nPraveen Kumar Sappa\nINVESTIGATIONS\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSTUDIES\nTranscriptome analysis of glucose starvation in B. subtilis\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nPamela Gamba(id: 370 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nNoisy-Strep\nPUBLICATIONS\nControl of transcription elongation by GreA determines rate of gene expression in Streptococcus pneumoniae\n\n____________________________________________________________________________\n\nSimultaneous Clostridial fermentation, lipase-catalyzed esterification, and ester extraction to enrich diesel with butyl butyrate.(id: 399 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nSimone Balzer Le\nPROJECTS\nBESTER\n\n____________________________________________________________________________\n\nTiling Array analysis of shocked B. subtilis cells(id: 52 | type: assays)\n\nDescription: missing\n\n\nDATA_FILES\nThe condition-dependent transcriptome of Bacillus subtilis 168\nINVESTIGATION\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nORGANISMS\nBacillus subtilis\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nCondition-dependent transcriptome reveals high-level regulatory architecture in Bacillus subtilis\nSTUDY\nTranscriptome of shocked B. subtilis cells\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nTranscriptomics of M. pneumoniae at different times of growth(id: 261 | type: studies)\n\nDescription: Contains copy number per locus tag at different times of Growth between 0.25h and 96 hours.\r\nM. pneumoniae was grown in Batch, cells attached to the bottom of the flask (single cell layer), non stirred, non aerated.\n\n\nASSAYS\nTranscriptomics assay of M. pneumoniae at diferent times of growth\nCREATORS\nNiels Zondervan\nLuis Serrano\nMaria Lluch\nEva Yus\nDATA_FILES\nAbsolute copy number per locus tag, M. pneumoniae at different times of growth\nPEOPLE\nNiels Zondervan\nLuis Serrano\nMaria Lluch\nEva Yus\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nPUBLICATIONS\nLarge-scale metabolome analysis and quantitative integration with genomics and proteomics data in Mycoplasma pneumoniae.\nSUBMITTER\nNiels Zondervan\n\n____________________________________________________________________________\n\nApplications of thiol-disulfide oxidoreductases for optimized in vivo production of functionally active proteins in Bacillus(id: 76 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nUwe Voelker\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nWhole genome marker frequency analysis on strain MK422(id: 1430 | type: data_files)\n\nDescription: Strain MK422 was grown under the same conditions as used when growing cells for microscopy analysis; cells of OD600 = 0.4 were diluted 100-fold in C+Y medium with 0.1 mM ZnCl2 and incubated for 2.5 hours until OD600 = 0.15. Cells were then harvested by centrifugation for 5 min at 6500 x g at 4C. Genomic DNA was isolated using the Wizard Genomic DNA Purification Kit (Promega) as described previously (Slager et al. 2014 Cell). Fragmentation was performed using Covaris instrument, and libraries were prepared using NEBNext Ultra DNA Library Kit for Illumina prior to sequencing on an Illuminia HiSeq 2000 Platform (all performed at the Genomics Core Facility, EMBL, Heidelberg, Germany). \n\n\nASSAYS\nWhole genome sequencing (Illumina)\nCREATORS\nJan-Willem Veening\nINVESTIGATIONS\nChromosome segregation and cell division in Streptococcus pneumoniae\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\nSTUDIES\nGenome wide marker frequency analysis on new strains to image genetic loci in Streptococcus pneumoniae\nSUBMITTER\nJan-Willem Veening\n\n____________________________________________________________________________\n\nProteomic Analysis of the Streptomyces coelicolor Proteome(id: 142 | type: sops)\n\nDescription: missing\n\n\nCREATORS\nLouise Thomas\nMaggie Smith\nPEOPLE\nMaggie Smith\nJay Moore\nLouise Thomas\nPROJECTS\nSTREAM\nSUBMITTER\nJay Moore\n\n____________________________________________________________________________\n\nGlucose perturbations(id: 52 | type: sops)\n\nDescription: Perturbation of starved cells with glucose. Concentrations of intra- and extracellular metabolites are followed in time.\n\n\nCREATORS\nMartijn Bekker\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nB. subtilis_SysMo2_Chemostat_growthrate-salt_salt_-GB_proteome_relative quantification(id: 860 | type: data_files)\n\nDescription: B. subtilis was grown in minimal media in a chemostat at growth rate =0.1, with 1.2M NaCl, without glycine betaine. Relative quantification for the proteome was done using metabolic labeling.\n\n\nASSAYS\nRelative quantification of proteins by metabolic labeling\nCREATORS\nSandra Maass\nINVESTIGATIONS\nMultiomics study of Bacillus subtilis under osmotic stress\nPEOPLE\nSandra Maass\nPROJECTS\nBaCell-SysMO\nSTUDIES\nB. subtilis_SysMo2_Chemostat_growthrate-salt\nSUBMITTER\nSandra Maass\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nCharacterization of the Weimberg Pathway in Caulobacter crescentus(id: 368 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nGunnar Liden\nLisa Wasserstrom\nCelina Tufvegren\nHenrik Almqvist\nPROJECTS\nXyloCut\n\n____________________________________________________________________________\n\nDynamic proteomic profiling of a unicellular cyanobacterium Cyanothece ATCC51142 across light-dark diurnal cycles.(id: 355 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nAryal et al, 2011, metabolic labelling of Cyanothece protein synthesis\nDATA_FILES\nAryal et al, 2011, metabolic labelling of Cyanothece protein synthesis\nPEOPLE\nDaniel Seaton\nPROJECTS\nMillar group\n\n____________________________________________________________________________\n\nAuxSim full - PLM_30, version 1(id: 852 | type: assays)\n\nDescription: \tA cell-level model of the Arabidopsis root elongation zone. This spatial model is divided up into biological cells which are further divided into simulation boxes. The original model was designed to investigate how canal cells can accumulate auxin over time rather than to investigate the transport of auxin through the canal cells per se. The main outputs of the simulations in the original paper were the steady state ratios of auxin in the canal cell protoplasts to that in the parenchyma cell protoplasts for different cell sizes and AUX/LAX auxin carrier distributions.\tNote that the working version of the model is in the Simile (.sml) version rather than in XML (which is a temporary holding dummy file).&nbsp;<br><br><strong>Related Publications</strong></strong><br>Kramer E.M. (2004). PIN and AUX/LAX proteins: their role in auxin accumulation. . TRENDS in Plant Sciences, 9, 578-582 . <br><br><strong>Originally submitted to PLaSMo on 2011-01-11 20:52:13</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nAndrew Millar\nDATA_FILES\nWorking Simile model, PLM_30_ 1\nSimile screenshot, PLM_30_ 1\nfiles to help specify tissue config for Simile model, PLM_30_ 1\nexample files to run Simile model, PLM_30_ 1\nINVESTIGATION\nMillar, Andrew\nMODELS\nAuxSim full - PLM_30, version 1, SUBMITTED\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nAuxSim full - PLM_30\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nAryal et al, 2011, metabolic labelling of Cyanothece protein synthesis(id: 2049 | type: data_files)\n\nDescription: Quantitative proteomic analysis of Cyanothece ATCC51142 grown in 12L:12D light:dark cycles, using partial metabolic labeling and LC-MS analysis.\n\n\nASSAYS\nAryal et al, 2011, metabolic labelling of Cyanothece protein synthesis\nEstimation of rates of translation and turnover from proteomics datasets\nCREATORS\nDaniel Seaton\nAndrew Millar\nINVESTIGATIONS\nPhotoperiodic control of the Arabidopsis proteome reveals a translational coincidence mechanism\nPEOPLE\nAndrew Millar\nDaniel Seaton\nPROJECTS\nMillar group\nPUBLICATIONS\nDynamic proteomic profiling of a unicellular cyanobacterium Cyanothece ATCC51142 across light-dark diurnal cycles.\nSTUDIES\nProteome and translation rate data for the Ostreococcus alga and for cyanobacteria\nSUBMITTER\nDaniel Seaton\n\n____________________________________________________________________________\n\nSUMO RNA isolation from E. coli(id: 44 | type: sops)\n\nDescription: This SOP describes the preparation of E. coli RNA\n\n\nASSAYS\nTranscriptional profiling of E. coli during anaerobic to aerobic and aerobic to anaerobic transitions\nTranscriptional profiling of steady states at different aerobiosis levels\nCREATORS\nMatthew Rolfe\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nDynamical studies for different oxygen availability in Escherichia coli\nPEOPLE\nMatthew Rolfe\nPROJECTS\nSUMO\nSTUDIES\nThe Escherichia coli dynamical response to oxygen: from molecular interactions to regulatory modules\nThe Escherichia coli steady state response to oxygen: from molecular interactions to regulatory modules\nSUBMITTER\nMatthew Rolfe\n\n____________________________________________________________________________\n\nTest created 1552502361596(id: 957 | type: assays)\n\nDescription: <br><br><strong>Related Publications</strong></strong><br>Me and Helena (2014). some book. <br>Moore A, Zielinski T, Millar AJ (2014). Online period estimation and determination of rhythmicity in circadian data, using the BioDare data infrastructure.. Methods in molecular biology (Clifton, N.J.). Retrieved from: doi.org/10.1007/978-1-4939-0700-7_2<br>Stein JM (1975). The effect of adrenaline and of alpha- and beta-adrenergic blocking agents on ATP concentration and on incorporation of 32Pi into ATP in rat fat cells.. Biochemical pharmacology. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/12<br><br><strong>Comments</strong></strong><p>\n\tNew comment after 2nd version</p>\n 2019-01-10 11:59:51 201 zielu tomasz.zielinski@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\r\n\tCommentsdfa</p>\r\n<br><br><strong>Originally submitted to PLaSMo on 2014-07-25 18:01:50</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nTomasz Zielinski\nINVESTIGATION\nZielinski, Tomasz\nPEOPLE\nTomasz Zielinski\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nMario Vera(id: 508 | type: people)\n\nDescription: My research is intended to contribute to the elucidation of the physiological and molecular processes involved in the biofilm formation of acidophilic leaching bacteria with emphasis in their cell-cell communication mechanisms. \r\nIn SysMetEx, our role is to understand biofilm formation at a microscopical and OMICS levels, in order to optimize it. \n\n\nINSTITUTIONS\nUniversitity Duisburg-Essen\nPROJECTS\nSysMetEx\nSOPS\nDNA, RNA, Protein extraction from mineral samples\n\n____________________________________________________________________________\n\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.(id: 326 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nDagmar Waltemath\nMartin Golebiewski\nPROJECTS\nLiSyM Core Infrastructure and Management (LiSyM-PD)\n\n____________________________________________________________________________\n\nAdvanced Lecture Course on Systems Biology(id: 24 | type: events)\n\nDescription: Systems Biology is one of the most prominent newly emerging interdisciplinary science areas. By connecting fields such as genomics, proteomics, cell biology, medicine, molecular biology, biochemistry and genetics, with mathematics, bioinformatics, engineering and computation, it enables discovery of yet unknown principles underlying the functioning of living cells. At the same time testable and predictive models of complex cellular pathways and eventually of whole cells are generated, which are useful for efficient experimental design and bioengineering and the network-based design of drugs and therapies. This advanced lecture course will cover essential and state-of-the-art aspects of SysBio from principles and methods through the modeling of living systems to applications in biotechnology and medicine. \r\nCourse opens the doors for regular student training in Systems Biology in Europe, provide a basis for a human network of young Systems Biology experts, and provide a centre for other Systems Biology discussions between scientists and funders.\r\nSysMO DB team will represent Data mangement in praxis.\n\n\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nMetabolic fluxes and beyond-systems biology understanding and engineering of microbial metabolism(id: 158 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLeif Steil\nChristoph Wittmann\nMichael Kohlstedt\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nGene knock-outs by recombineering in E.coli.(id: 20 | type: sops)\n\nDescription: Generating gene knock-outs using DY330 expressing the defective lambdoid prophage recombination system  red.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nA gating model for the potassium efflux systems KefB and KefC(id: 40 | type: sops)\n\nDescription: A kinetic model that describess the activation of a dimeric efflux system that could bind either GSH or SLG\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nNested autoinhibitory feedbacks alter the resistance of homeostatic adaptive biochemical networks(id: 238 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJrg Schaber\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nAutomated Flow Cytometry Data Analysis(id: 5 | type: presentations)\n\nDescription: Presentation by Thomas Millat of a tool for automatized analysis of flow cytometry data. It allows characterisation of different growth modalities, statistical measures of mean and variance and estimation of population overlap all along with appropriate visualization.\n\n\nCREATORS\nThomas Millat\nEVENTS\nBacell-meeting 1109-Groningen\nPEOPLE\nThomas Millat\nUlf Liebal\nPROJECTS\nBaCell-SysMO\nSUBMITTER\nUlf Liebal\n\n____________________________________________________________________________\n\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\r\nphosphate pathway enzymes(id: 76 | type: studies)\n\nDescription: Since over 40 enzymes will be investigated for their mRNA abundance, processing, and degradation kinetics, the less tedious and more accurate Next Generation Sequencing of the entire mRNA repertoire of the cell is employed. To optimise the proportion of useful sequence, while including RNA fragments that are products of of degradation, rRNA is depleted using the eukaryotic Ribominus kit (Ambion). Two biological replicates are treated with Sinefungin and Actinomycin D to inhibit RNA processing and transcription, for different time periods. RNA is collected before treatment, 5 min after the addition of Sinefungin, 5 min after addition of ActD (+Sin), then 10 min after, 20, 30, 60 and 120 min. The samples are sequenced using the Illumina standard protocol, and the sequenced reads are aligned to the reference genome TREU927, and the number of reads aligned to each ORF are determined. \n\n\nASSAYS\nmRNA decay assay\npre-mRNA processing rate\nModelling the gene expression cascade with length-dependent processes\nDATA_FILES\nmRNA decay\nHalf life values for enzymes of redox and polyamine pathways as provided by Alejandro\nmRNA half-lives\nprecursor mRNA half-lives\nINVESTIGATION\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nChristine Clayton\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSOPS\nSample preparation for mRNA decay study\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, CCA1 degr L - PLM_85, version 1, SIMPLIFIED(id: 514 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_85, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, CCA1 degr L - PLM_85, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, CCA1 degr L - PLM_85\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nK+ Starvation in Saccharomyces cerevisiae(id: 10 | type: investigations)\n\nDescription: Experimental approaches to evaluate cellular response to Potassium limitation. For this purpose, a series of isogenic strains derived from BY4741 and lacking one (TRK1 or TRK2) or both TRK genes encoding specific potassium uptake systems was constructed using the homologous recombination and Cre-lox system.\n\n\nASSAYS\nTime Course Micro Array Experiment\nRT-PCR\nLacZ Reporters\nVolume determination during starvation at different times.\nTime courses of the internal pH changes\nHow does internal potassium change (/decrease) during several hours of potassium starvation.\nHow starvation affect protein content in yeast cells\nExternal Potassium Concentration\nExternal pH changes\nProtein Concentration\nPotassium fluxes\nProton fluxes\nAmmonium fluxes\n2D-Gel Electrophoresis\nProtein Identification - MS\nNMR\nHPLC-MS\nStable potassium concentration\nStable pH\nStable membrane potential\nStable volume\nMembrane Potential\nPEOPLE\nSimon Borger\nPROJECTS\nTRANSLUCENT\nSTUDIES\nAnalysis of proton and potassium fluxes\nTranscriptional Profile\nKey Cell-Physiological Parameters\nIon Flux Changes\nProteomic Studies\nLong Term Metabolomic Profile\nExperimental approaches to determine the dependence of volume, pH and membrane potential on external potassium\nPysiological Model\nSUBMITTER\nSimon Borger\n\n____________________________________________________________________________\n\nProtocol for methylglyoxal disappearance(id: 33 | type: sops)\n\nDescription: A method of how to measure methylglyoxal present in the growth medium.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nTroein Ostreococcus clock 1-loop - PLM_7, version 1(id: 912 | type: assays)\n\nDescription: \tThis is a model of the circadian clock of Ostreococcus tauri, with a single negative feedback loop between TOC1 and CCA1 (a.k.a. LHY), and multiple light inputs. It was used and described in Troein et al., Plant Journal (2011).\tThe model has been tested in Copasi, where it reproduces the behaviour of the original (which consisted of equations loaded from a text file by a more or less custom C++ program).<br><br><strong>Comments</strong></strong><p>\n\tNot formulated to easily allow addition of the ISSF to replace the present light function.</p>\n 2013-02-27 12:34:55 3 amillar2 andrew.millar@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\n\tThis is a model of the circadian clock of Ostreococcus tauri, with a negative feedback loop between TOC1 and CCA1 (a.k.a. LHY) and multiple light inputs. It was used and described in Troein et al., Plant Journal (2011).\n\tThe model incorporates luciferase reporters, and in this SBML model the four different versions of the model for transcriptional and translational reporter lines (pTOC1::LUC, pCCA1::LUC, TOC1-LUC and CCA1-LUC) are all accessible by setting one of the rep_X parameters to 1 and the others to 0. You can also set all four to 0 to only simulate the non-reporter core of the system.\n\tInput to the system should be provided by modifying the &quot;light&quot; function. The functions &quot;ox_cca1&quot; and &quot;ox_toc1&quot; can be altered to add overexpression of CCA1 and TOC1. Setting either to x gives additional, constitutive transcription at x times the transcription rate that the native gene has when the Shea-Ackers quotient equals 1. The functions &quot;copies_toc1&quot; and &quot;copies_cca1&quot; are normally 1 but can be lowered to simulate knockdown experiments. The functions &quot;transcription&quot;, &quot;translation&quot; and &quot;proteasome&quot; can be used to modify the overall rate of transcription, translation and protein degradation.\n\tThe parameters were fitted specifically to data from transgenic reporter lines TOC8, pTOC3, LHY7 and pLHY7 (Corellou et al., Plant Cell 2009). Parameters that begin with &quot;effcopies&quot; describe the effective number of copies of CCA1 or TOC1 in the respective translational fusion lines, with anything above 1 due to the fusion proteins.\n\tFor the model fitting, the initial values were fitted to the data in the various time courses. The initial values given here correspond to the limit cycle of the system in LD 12:12. The system converges to the limit cycle in just a few days under most light conditions, so these initial values are biologically meaningful.\n\tThe species cca1luc_c and cca1luc_n have been merged into cca1luc (which corresponds to the observable signal), because Copasi refused to run the system otherwise. For TOC1-LUC, the predicted signal is the sum of toc1luc_1 and toc1luc_2.</p>\n<br><br><strong>Originally submitted to PLaSMo on 2010-05-04 11:27:33</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nINVESTIGATION\nTroein, Carl\nMODELS\nTroein Ostreococcus clock 1-loop - PLM_7, version 1, SUBMITTED\nTroein Ostreococcus clock 1-loop - PLM_7, version 1, SIMPLIFIED\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nTroein Ostreococcus clock 1-loop - PLM_7\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nCompartimented ODE model of oxygen diffusion near E. coli cells(id: 91 | type: models)\n\nDescription: This ordinary-differential equation model is a spatially lumped model showing the behaviour of oxygen in the three compartments medium, membrane and cytoplasm and its impact on FNR inactivation, hereby showing the effects of different oxygen concentrations, diffusion coefficients and reaction rates. The model was created with the Matlab SimBiology toolbox. \n\n\nCREATORS\nSamantha Nolan\nPEOPLE\nDavid Knies\nSamantha Nolan\nPROJECTS\nSUMO\nSUBMITTER\nDavid Knies\n\n____________________________________________________________________________\n\nMiniaturized device for agitating a high-density yeast suspension that is suitable for in vivo nuclear magnetic resonance applications(id: 144 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\n\n____________________________________________________________________________\n\nNorman Ertych(id: 1261 | type: people)\n\nDescription: missing\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nbatchfermentation exp-starv01 quant-proteomics(id: 12 | type: data_files)\n\nDescription: batch fermatation - The transition from growing to non-growing Bacillus subtilis cells\n\n\nASSAYS\nRelative quantification of proteins by metabolic labeling\nCREATORS\nSandra Maass\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nPEOPLE\nSandra Maass\nPROJECTS\nBaCell-SysMO\nSTUDIES\nBatchfermentation exp-starv01_090204\n\n____________________________________________________________________________\n\nWolfgang Hillen(id: 304 | type: people)\n\nDescription: CURRICULUM VITAE\r\n\r\n\r\nWolfgang Hillen\r\nFull Professor and Chairman of Microbiology\r\nLehrstuhl fr Mikrobiologie\r\nInstitut fr Biologie\r\nFriedrich-Alexander Universitt\r\nErlangen-Nrnberg\r\nStaudtstr. 5\r\n91058 Erlangen, Fed. Rep. of Germany\r\n\r\n\r\nDate of Birth:\tApril 24, 1948\r\nPlace of Birth:\tOsnabrck, FRG\r\nNationality:\tGerman\r\nStatus:\tMarried\r\nChildren:\tHauke Sven Hillen - May 17, 1987\t\r\n\r\nRESEARCH AND PROFESSIONAL EXPERIENCE\r\n\r\nPresent \tFull Professor and Chairman of Microbiology at the Institute of Biology, Friedrich-Alexander University, Erlangen, FRG.\r\nProtein-DNA Recognition, Regulation of Tetracycline Resistance Gene Expression, Tetracycline Dependent Gene Regulation in Eukaryotes (Mammals), Catabolite Repression in Bacilli\r\n\r\n2006/2007\tSabbatical Research at the MD Anderson Cancer Center, University of Texas, Houston, with Dr. Richard G. Brennan\r\n\tStructure analysis of transcription factor complexes.\r\n\r\n1999/2000\tSabbatical Research at the Dept. of Biology, University of California at San Diego, with Dr. Milton H. Saier\t\r\nCatabolite repression in Gram positive bacteria.\r\n\r\n1993 \tSabbatical Research at American Cyanamid, Lederle Research Laboratories, Pearl River, NY with Dr. Frank Tally and Dr. George A. Ellestad\r\nDevelopment and analysis of new tetracyclines with activities against resistant bacteria.\r\n\r\n1984-1986\tFull Professor for Physical Biology, Institute of Physical Biology, University of Dsseldorf, Dsseldorf, FRG.\r\nRegulation of Tetracycline Resistance Determinants, Bacterial Expression Systems.\r\n\r\n1980-1984\tP.I., Institute of Biochemistry, Technische Hochschule Darmstadt, Darmstadt, FRG.\r\nStructural and Genetic Analysis of Tetracycline Resistance Determinants.\r\nEarned Degree: Dr. rer. nat. habil.\r\n\r\n1978-1980\tPostdoctoral Research Fellow, Dept. of Biochemistry, University of Wisconsin-Madison, Madison, Wi.\r\nStructural and Dynamic Properties of DNA Sequences from the lac Regulatory Region.\r\n\r\n1977-1978\tPostdoctoral Research Fellow, Institute of Biochemistry, Technische Hochschule Darmstadt, Darmstadt, FRG.\r\nSynthesis and Crystallography of Modified Nucleosides.\r\n\r\n1974-1977\tGraduate Student, Institute of Biochemistry, Westflische-Wilhelms Universitt, Mnster, FRG.\r\nCodon-Anticodon Recognition.\r\nEarned Degree: Dr. rer. nat. (summa cum laude)\r\n\r\n1969-1974\tUndergraduate Student, Department of Chemistry, Westflische-Wilhelms Universitt, Mnster, FRG.\r\nEarned Degree: Diplomchemiker (sehr gut)\r\n\r\nEDUCATION\r\n\r\nGymnasium Carolinum, Osnabrck, Germany\t1960-1967\r\nMilitary Service (drafted)\t1967-1969\r\nDepartment of Chemistry, Mnster, Germany\t1969-1974\r\nInstitute of Biochemistry Mnster, Germany\t1974-1977\r\n\r\n\r\n\r\n\r\nDegrees\r\n\r\nDiplomchemiker (sehr gut)\t1974\t\r\nDr. rer. nat. (summa cum laude)\t1977\t\r\nDr. rer. nat. habil. (Biochemistry)\t1984\t\r\n\r\nFellowships\r\n\r\nGraduate Fellowship, University of Mnster\t1974-1977\r\nPostdoctoral Fellowship, Max Kade Foundation, New York\t1978-1979\r\nPostdoctoral Fellowship, DFG, Bonn\t1979-1980\t\r\nAppointments offered\r\n\r\nFull Professor of Biochemistry, University Bayreuth (declined)\t\t       1984 \t\r\nFull Professor of Biophysics, University Dsseldorf (accepted)\t\t       1984\r\nChair of Microbiology, University Erlangen-Nrnberg (accepted)\t\t       1986\r\nChair of Microbiology, University Bochum (declined)\t\t\t\t       1996\r\nChair of Department of Microbiology, Medical School,\r\nUniversity of Alabama at Birmingham, USA (declined)\t\t\t        2000 \r\n\r\nPROFESSIONAL DUTIES\r\n\r\nSpeaker of Study Section \"Recombinant DNA Technology\", GBCh\t\r\n(German Association of Biological Chemists)\t1985-1987\r\nSpeaker of Research Group \"Cis and Trans Active Elements for\r\nTranscription\"\t1989-1995\r\nOrganizer of Annual Meeting of GBCh\t\r\n(German Association of Biological Chemists) Study Group\t1986,1987\r\nCo-Organizer of Annual Meeting of GBCh\t\r\n(German Association of Biological Chemists)\t1987\t\r\nMember of the DECHEMA (German Chemical Engineering Association)\t\r\nBiotechnology Committee\t1988-1992\r\nSpeaker of Forschergruppe \"Transkriptionskontrolle und Regulatorproteine\"\t1989-1996\r\nSpeaker of Sonderforschungsbereich 473 \"Schaltvorgaenge der Transkription\"\t1997-present\r\nVice President of the VAAM (German Society of Microbiology)\t1999-2001\r\nSpeaker of Bavaria California Technology Center\t2000-present\r\nMember of the Board of Trustees of the Fonds of Chemical Industry\t2001-present\r\nPresident of the VAAM (German Society of Microbiology)\t2001-2003\r\nVice President of the VAAM (German Society of Microbiology) 1999-2001 &\t2003-2005\r\n\r\n\r\nPROFESSIONAL ORGANIZATIONS\r\n\r\nGesellschaft Deutscher Chemiker (German Association of Chemists)\r\nGesellschaft fr Biologische Chemie (German Association of Biological Chemists)\r\nVereinigung Allgemeiner und Angewandter Mikrobiologen (German Association of Microbiologists)\r\nAmerican Society of Microbiology\r\nAmerican Association for the Advancement of Science \r\nSociety for General Microbiology, U.K.\r\n\r\nSCIENTIFIC AWARDS\r\n\r\nCarl Duesberg Preis der GDCh (German Association of Chemists)\t1986\t\r\nMax Planck Forschungspreis fuer Intern. Kooperation (Alexander v.\r\nHumboldt Foundation)\t1997\r\nMember of the Leopoldina Academy of Sciences, Halle, Germany\t2003\r\n\r\n\r\n\r\nEDITOR\r\n\r\nApplied Microbiology and Biotechnology\t1993-2005\r\nEuropean Journal of Biochemistry, since 2004 The FEBS Journal\t1998-present\r\n\r\nCOMMERCIAL APPOINTMENTS\r\n\r\nCo-founder of the \"Arbeitsgemeinschaft Gentechnik\" consisting of\t\t        1982\r\nTH Darmstadt, Institut fr Biochemie; Merck KG, Darmstadt; Rhm, Darmstadt; Grnenthal, Stolberg. \t\r\n\r\nConsultant for Grnenthal, Stolberg: Development of prourokinase \t   1982-1986\r\nproduction for blood clot dissolvement \r\n\r\nCollaboration with Henkel KGaA, Dsseldorf: Development of expression\t   1984-1994\r\nsystems for Bacilli; and Isolation of improved starch hydrolases from\r\nGram positive bacteria \r\n\r\nConsultant for formerly Amercian Cyanamide, then American \t   1992-1996\r\nHome Products, now Wyeth Inc., Pearl River, New York: Development \r\nof new antibacterials. \r\n\r\nConsultant for Cubist Pharmaceuticals, Boston, Massachussetts: \t   1997-2001\r\nDevelopment of the VITATM technology for target validation \r\n\r\nConsultant for Elitra Inc., San Diego: Development of expression \t   2001-2000 systems for target validation and screening of antiinfectives. \r\n\r\nCofounder of TET Systems GmbH & Co KG, Heidelberg\t            2004\r\n\r\n\r\n \r\n\n\n\nINSTITUTIONS\nUniversity of Erlangen-Nuernberg\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nStructures of carbon catabolite protein A-(HPr-Ser46-P) bound to diverse catabolite response element sites reveal the basis for high-affinity binding to degenerate DNA operators\nCarbon source control of the phosphorylation state of the Bacillus subtilis carbon-flux regulator Crh in vivo\nCcpA forms complexes with CodY and RpoA in Bacillus subtilis\nHigh- and low-affinity cre boxes for CcpA binding in Bacillus subtilis revealed by genome-wide analysis\n\n____________________________________________________________________________\n\nAnna Skorska(id: 1290 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRostock University Medical Centre\nPROJECTS\niRhythmics\n\n____________________________________________________________________________\n\npre-mRNA processing rate(id: 235 | type: assays)\n\nDescription: Genes are transcribed in polysictronic messages (pre-mRNA) that are destined for either maturation into mRNAs, or degradation. Since transcription regulation is non-existent with few exceptions, the rate of pre-mRNA processing, together with mRNA decay and translation rates, are believed to control gene expression. In this assay, 2T1 blood form trypanosomes are subject to treatment by ActinomycinD for 5 minutes, inhibiting transcription. The cells are harvested, depleted for ribosomal RNA, and sent for deep sequencing. 2 biological replicates have been processed. Reads that align to 20 bp just upstream of 5' splice sites, are believed to correspond to pre-mRNA. The number of reads is compared to a control sample, and the rate of processing is calculated assuming an exponential profile. \r\nThe results for blood form Trypanosomes will be compared to the results from 2 replicates for the procyclic form. Differences will be linked to differences in mature mRNA abundances between the two forms.\n\n\nINVESTIGATION\nGene expression in Trypanosoma brucei\nORGANISMS\nTrypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDY\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\r\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nAnalysis of proton and potassium fluxes(id: 12 | type: studies)\n\nDescription: Cells are starved. Potassium and subsequently glucose are added to the medium. Proton and potassium fluxes across the plasma membrane are recorded before and after these events for WT and mutants lacking specific transporter proteins.\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nSimon Borger\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nSimon Borger\n\n____________________________________________________________________________\n\nfermentation and product spectrum of Clostridium acetobutylicum ctfB::int(id: 1028 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nThomas Rimpf\nPEOPLE\nThomas Rimpf\nPROJECTS\nCOSMIC\nSUBMITTER\nThomas Rimpf\n\n____________________________________________________________________________\n\nSulfosys - SOP microarray(id: 66 | type: sops)\n\nDescription: Protocol for RNA isolation, cDNA synthesis and labeling and hybridization and cleanup of Sulfolobus solfataricus microarray\n\n\nASSAYS\nComparison of transcriptome of S. solfataricus at 70 and 80C\nCREATORS\nPawel Sierocinski\nINVESTIGATIONS\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature\nPEOPLE\nPawel Sierocinski\nPROJECTS\nSulfoSys\nSTUDIES\nPilot experiment - S. solfataricus grown at 70 and 80 C.\nSUBMITTER\nPawel Sierocinski\n\n____________________________________________________________________________\n\nArabidopsis clock models P2011.1.2 and P2011.2.1 - PLM_71, version 1, SIMPLIFIED(id: 389 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_71, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis clock models P2011.1.2 and P2011.2.1 - PLM_71, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis clock models P2011.1.2 and P2011.2.1 - PLM_71\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nLactobacillus plantarum ATCC 14917 protein list(id: 1065 | type: data_files)\n\nDescription: Lactobacillus plantarum ATCC 14917 differentially expressed proteins, comparison between low and high growth rates\n\n\nCREATORS\nAnette McLeod\nPEOPLE\nAnette McLeod\nPROJECTS\nSysMO-LAB\nSUBMITTER\nAnette McLeod\n\n____________________________________________________________________________\n\n14-3-3 Proteins: insights from genome-wide studies in yeast(id: 28 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nFalko Krause\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nPreliminary overview of the effect of KCl stress on protein interaction networks involved in the regulation of the potassium transporter Trk1.(id: 30 | type: models)\n\nDescription: missing\n\n\nCREATORS\nPaul Heusden\nPEOPLE\nPaul Heusden\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nThe metabolic response of P. putida KT2442 producing high levels of polyhydroxyalkanoate under single- and multiple-nutrient-limited growth: highlights from a multi-level omics approach(id: 220 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nDietmar Schomburg\nVitor Martins Dos Santos\nCarolyn Lam\nAuxi Prieto\nIgnacio Poblete Castro\nPROJECTS\nPSYSMO\n\n____________________________________________________________________________\n\nWhole genome marker frequency analysis on strain MK423(id: 1431 | type: data_files)\n\nDescription: Strain MK423 was grown under the same conditions as used when growing cells for microscopy analysis; cells of OD600 = 0.4 were diluted 100-fold in C+Y medium with 0.1 mM ZnCl2 and incubated for 2.5 hours until OD600 = 0.15. Cells were then harvested by centrifugation for 5 min at 6500 x g at 4C. Genomic DNA was isolated using the Wizard Genomic DNA Purification Kit (Promega) as described previously (Slager et al. 2014 Cell). Fragmentation was performed using Covaris instrument, and libraries were prepared using NEBNext Ultra DNA Library Kit for Illumina prior to sequencing on an Illuminia HiSeq 2000 Platform (all performed at the Genomics Core Facility, EMBL, Heidelberg, Germany). \n\n\nASSAYS\nWhole genome sequencing (Illumina)\nCREATORS\nJan-Willem Veening\nINVESTIGATIONS\nChromosome segregation and cell division in Streptococcus pneumoniae\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\nSTUDIES\nGenome wide marker frequency analysis on new strains to image genetic loci in Streptococcus pneumoniae\nSUBMITTER\nJan-Willem Veening\n\n____________________________________________________________________________\n\nInduction of RNA interference in T.brucei brucei 2T1 cell line(id: 200 | type: assays)\n\nDescription: missing\n\n\nDATA_FILES\nInduction of RNA interference in T.brucei\nRNAi growth curves\nINVESTIGATION\nPerturbing the biological system: developmental adaptation and experimental manipulation of trypanosome biology\nORGANISMS\nTrypanosoma brucei\nPEOPLE\nFederico Rojas\nPROJECTS\nSilicoTryp\nSTUDY\nTargeted disruption and over expression of critical enzymes\nSUBMITTER\nFederico Rojas\n\n____________________________________________________________________________\n\nSingle cell analysis of gene expression patterns during carbon starvation in Bacillus subtilis reveals large phenotypic variation(id: 183 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nLive Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy.\nDATA_FILES\nRNA-seq wild type\nPEOPLE\nOscar Kuipers\nImke De Jong\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nMacromolecule diffusion and confinement in prokaryotic cells(id: 120 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nBert Poolman\nJacek Mika\nPROJECTS\nKOSMOBAC\n\n____________________________________________________________________________\n\nTime courses of the internal pH changes(id: 61 | type: assays)\n\nDescription: Time courses of the internal pH changes under the conditions of Navarrete et al. (2010) will be obtained. The usage of different mutants (transport systems and regulation factors) will reveal the influence and key systems of pH regulation.\r\n\r\nTo estimate the changes of internal pH, the pH-dependent variant of GFP pHluorin is expressed in cells from a multicopy plasmid, and the changes in cell fluorescence are monitored during 5 hours of incubation in YNB-F growth media without added potassium.\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nKey Cell-Physiological Parameters\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nCharacterization of flux distribution and general physiology of L. lactis in continuous cultures(id: 601 | type: data_files)\n\nDescription: L. lactis was grown in C-limited cultures at pH 6.7 and 7.5 and at a growth rate of 0.05, 0.15 and 0.40\n\n\nASSAYS\nPhysiological characterization of Lactic acid bacteria grown in C-limited continuous cultures\nCREATORS\nMartijn Bekker\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nPUBLICATIONS\nPhysiological characterization of three lactic acid bacteria and their isogenic ldh deletion mutants shows that these organisms are optimized for Yatp and not ATP formation rates at their physiological pH\nSTUDIES\nStudy of the physiological characterization of three lactic acid bacteria and their isogenic ldh deletion mutants\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nRoel Van Driel(id: 749 | type: people)\n\nDescription: POSITION\r\nI am an emeritus professor in Biochemistry at the University of Amsterdam (retired 2010). \r\n\r\nRESEARCH\r\nMy research focussed on the human chromatin in its natural environment, i.e. the nucleus of cultured living human cells.\r\nAspects, such as the dynamic folding of the chromatin fiber inside the nucleus and local chemical modification of histones and DNA at genetic loci, are the physical and chemical basis for epigenetic regulation of gene expression. In my group we worked parallel on human cells and plant cells, making use of their similarity in mechanisms and their complementary experimental approaches. In addition we have analyse the dynamics and molecular mechanisms of the formation in living cells of multi-protein complexes on the DNA that are responsible for DNA repair after UV damage.\r\nIn these studies we collaborated with different physicists in making predictive computational models based on quantitative experimental data. Such models gave remarkable insight into underlying molecular dynamic prosesses.\r\n\r\nPUBLICATION\r\nA list of publication for the period 2006 - 2016 can be found as a data file.\r\n\r\nEXPERTISE\r\nCell biology, quantitative confocal microscopy, chromatin structure and dynamics, nuclear organisation, epigenetics\r\n\r\nCURRENT ACTIVITIES\r\nI have been and still am involved in a number of national and European activities in the field of systems biology. This includes particpating in the the building of a European ESFRI research infrastructure for systems biology (ISBE: Infrastructure for Systems Biology Europe: http://project.isbe.eu/).\n\n\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nChromosome segregation and cell division in Streptococcus pneumoniae(id: 93 | type: investigations)\n\nDescription: missing\n\n\nASSAYS\nWhole genome sequencing (Illumina)\nCREATORS\nJan-Willem Veening\nDATA_FILES\nWhole genome marker frequency analysis on strain MK350\nWhole genome marker frequency analysis on strain MK422\nWhole genome marker frequency analysis on strain MK423\nDCI15\nMK453\nMK454\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\nSTUDIES\nGenome wide marker frequency analysis on new strains to image genetic loci in Streptococcus pneumoniae\nSUBMITTER\nJan-Willem Veening\n\n____________________________________________________________________________\n\nCOMBINE (Computational Modeling in Biology Network)(id: 37 | type: programmes)\n\nDescription: This is the exchange platform of the COMBINE network.\r\n\r\nCOMBINE is an initiative to coordinate the development of various community standards and formats for computational models: BioPax, CellML, NeuroML, Synthetic Biology Open Language (SBOL), Systems Biology Graphical Notation (SBGN), Systems Biology Markup Language (SBML), Simulation Experiment Description Markup Language (SED-ML).\n\n\nADMINISTRATORS\nMartin Golebiewski\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universitt zu Berlin\nNorwegian University of Science and Technology\nUniversity of Heidelberg\nUniversity of Newcastle-upon-Tyne\nUniversity of Rostock\nTechnische Universitt Dresden\nStellenbosch University\nUniversity Maastricht\nAuckland Bioengineering Institute\nUniversity College London (UCL)\nBioinformatics MIlan Bicocca (BIMIB) - Universit degli Studi di Milano Bicocca\nIndiana University Bloomington\nPEOPLE\nJacky Snoep\nDagmar Waltemath\nMartin Golebiewski\nVasundra Toure\nMatthias Knig\nJudith Wodke\nLutz Brusch\nChris Evelo\nJames Alastair McLaughlin\nMarco Nobile\nClemens Khn\nDavid Nickerson\nPadraig Gleeson\nJames Sluka\nRandy Heiland\nJrgen Pahle\nPROJECTS\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nMeasurements on kinetics of L-LDHs from L.lactis at pH 7.5(id: 1075 | type: data_files)\n\nDescription: Heterologous Expression of LDH from L.actis (MG1363)  in E. coli DH5. \r\nAssessment of kinetic parameters of LDH to include in a catabolic model\n\n\nCREATORS\nWayne Aubrey\nPEOPLE\nWayne Aubrey\nPROJECTS\nSysMO-LAB\nSUBMITTER\nWayne Aubrey\n\n____________________________________________________________________________\n\nWorkshop on Reproducible and Citable Data and Models, Rostock-Warnemnde, September 14th-16th(id: 39 | type: events)\n\nDescription: This workshop is for computational biologists and experimentalists who wish to learn about standards, citeable data, about how to make scientific results sustainable, available through open repositories, and about how to find and reuse other peoples works. It comprises of a mixture of lectures and hands-on sessions. We will teach you the major standards and tools that support these standards, as well as steps towards encoding and publishing reproducible modeling results. Topics include data identifiers and data identity authorities, licences, public data stores, linking data with publications, how to cite others data, standards for model encoding (SBML, CellML), annotation of model parts with terms from bio-ontologies, and encoding virtual experiments in SED-ML.\r\nParticipants are encouraged to bring their own piece of model and put their hands on transforming it into a standard-compliant and publishable piece of research.\n\n\nDATA_FILES\nReproducible and Citable Data and Models Workshop\nPRESENTATIONS\nFAIRDOMHUB / SEEK user documentation\nLicensing, Citation and Sustainability.\nReproducible and citable data and models: an introduction.\nPublishing data and code openly\nCiting data in research articles: principles, implementation, challenges - and the benefits of changing our ways.\nCapturing the context: one small(ish) step for modellers, one giant leap for mankind.\nReproducibility of model-based results: standards, infrastructure, and recognition.\nImproving the management of computational models.\nMaking your data good enough for sharing.\nThe FAIRDOM Commons for Systems Biology\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nSystems Biology of Monovalent Cation Homeostasis in Yeast(id: 223 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nHana Sychrova\nJoaquin Ario\nJose Ramos\nJost Ludwig\nMaik Kschischo\nJesus Jorrin\nMatthias Kahm\nFalko Krause\nSilvia Petrezselyova\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nChristiane Rennert(id: 818 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nThe pH-induced metabolic shift from acidogenesis to solventogenesis in Clostridium acetobutylicum - From experiments to models(id: 27 | type: presentations)\n\nDescription: This presentation given at the 5th International Beilstein Symposium presents the development of a model for the pH-induced shift in C. acetobutylicum. Starting from experimental information, the model construction and underlying assumptions are explained. Furthermore, the required additions to the metabolic and genetic network caused by the used experimentals setup are established.\r\n\r\nThe established metabolic and genetic model is compared to dynamic shift experiments starting at acidogenesis and solventogenesis. We show that this phase transition of C. acetobutylicum is associated to a cellular adaptation of the proteome. However, the dynamics during those transition is reproduced insufficiently.\r\n\r\nThe external pH is crucial for the behaviour of C. acetobutylicum, hence implications to the pH-dependent product spectrum of branch points in the ABE fermentation pathway are discussed. The comparison between simulations and experimental data reveals that further information about the pH-dependent kinetic regulation of the pathway are required for an improvement of the model.\n\n\nCREATORS\nThomas Millat\nPEOPLE\nThomas Millat\nPROJECTS\nCOSMIC\nPUBLICATIONS\nA proteomic and transcriptional view of acidogenic and solventogenic steady-state cells of Clostridium acetobutylicum in a chemostat culture\nA systems biology approach to investigate the effect of pH-induced gene regulation on solvent production by Clostridium acetobutylicum in continuous culture\nGenome-wide gene expression analysis of the switch between acidogenesis and solventogenesis in continuous cultures of Clostridium acetobutylicum\nSUBMITTER\nThomas Millat\n\n____________________________________________________________________________\n\nThe Glyoxalase pathway model(id: 39 | type: sops)\n\nDescription: The model describes the series of chemical reactions in MG detoxification pathway, allowing one to predict the flux of all compounds produced during detoxification.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nBert Poolman(id: 38 | type: people)\n\nDescription: Bert Poolman is professor in biochemistry and program director of the Centre for Synthetic Biology. His research focuses on the elucidation of the mechanisms by which signals are transduced and small molecules are translocated across cellular membranes. Cells are reengineered for the production of correctly folded membrane proteins, and methods are developed to reconstitute complex molecular assemblies in synthetic membranes and to analyze their functional and structural properties. The in vitro mechanistic studies are used to strengthen in vivo physiological measurements and vice versa. Research topics in his group include: (i) the mechanisms of translocation catalysis by ATP-binding cassette transporters; (ii) the molecular basis of cell volume regulation, incl. a comprehensive understanding of the role of lipid-protein interactions, (macro)molecular crowding and other generic physicochemical factors in transmembrane signaling and transport; and (iii) membrane proteomics with a focus on nuclear and vacuolar membranes and the basis for molecule transport through the nuclear pore complex.\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nKOSMOBAC\nPUBLICATIONS\nMolecular sieving properties of the cytoplasm of Escherichia coli and consequences of osmotic stress\nMacromolecule diffusion and confinement in prokaryotic cells\nProtein mobility and diffusive barriers in Escherichia coli: consequences of osmotic stress\nThe role of biomacromolecular crowding, ionic strength, and physicochemical gradients in the complexities of life's emergence\nStructural basis for the enhanced activity of cyclic antimicrobial peptides: The case of BPC194\nThe molecular basis for antimicrobial activity of pore-forming cyclicpeptides\n\n____________________________________________________________________________\n\nFunctional potentials of human hematopoietic progenitor cells are maintained by mesenchymal stromal cells and not impaired by plerixafor(id: 282 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nPEOPLE\nMaja Rey\nRainer Saffrich\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nTest created 1552502791700(id: 958 | type: assays)\n\nDescription: <br><br><strong>Related Publications</strong></strong><br>Me and Helena (2014). some book. <br>Moore A, Zielinski T, Millar AJ (2014). Online period estimation and determination of rhythmicity in circadian data, using the BioDare data infrastructure.. Methods in molecular biology (Clifton, N.J.). Retrieved from: doi.org/10.1007/978-1-4939-0700-7_2<br>Stein JM (1975). The effect of adrenaline and of alpha- and beta-adrenergic blocking agents on ATP concentration and on incorporation of 32Pi into ATP in rat fat cells.. Biochemical pharmacology. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/12<br><br><strong>Comments</strong></strong><p>\n\tNew comment after 2nd version</p>\n 2019-01-10 11:59:51 201 zielu tomasz.zielinski@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\r\n\tCommentsdfa</p>\r\n<br><br><strong>Originally submitted to PLaSMo on 2014-07-25 18:01:50</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nTomasz Zielinski\nINVESTIGATION\nZielinski, Tomasz\nPEOPLE\nTomasz Zielinski\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nStandardization of Good Manufacturing Practice-compliant production of bone marrow-derived human mesenchymal stromal cells for immunotherapeutic applications(id: 279 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nPEOPLE\nMaja Rey\nRainer Saffrich\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nGenome-scale metabolic model of Enterococcus faecalis V583(id: 180 | type: models)\n\nDescription: A reconstruction of the cellular metabolism of the opportunistic human pathogen Enterococcus faecalis V583 represented as stoichiometric model and analysed using constraint-based modelling approaches\n\n\nASSAYS\nGenome-Scale Model Enterococcus faecalis V583\nCREATORS\nNadine Veith\nMargrete Solheim\nKoen Van Grinsven\nJennifer Levering\nJeroen Hugenholtz\nHelge Holo\nIngolf Nes\nBas Teusink\nUrsula Kummer\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nBas Teusink\nIngolf Nes\nJeroen Hugenholtz\nUrsula Kummer\nJennifer Levering\nHelge Holo\nNadine Veith\nKoen Van Grinsven\nMargrete Solheim\nPROJECTS\nSysMO-LAB\nPUBLICATIONS\nUsing a Genome-Scale Metabolic Model of Enterococcus faecalis V583 To Assess Amino Acid Uptake and Its Impact on Central Metabolism\nSTUDIES\nReconstructing the metabolic pathways of S. pyogenes and E. faecalis from their genome sequence\nSUBMITTER\nNadine Veith\n\n____________________________________________________________________________\n\nPFK Kinetic characterisation(id: 194 | type: sops)\n\nDescription: SOP for measurement of PFK activity in extracts.\n\n\nASSAYS\nPFK\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nExternal metabolite concentration times series(id: 1705 | type: data_files)\n\nDescription: Contains growth curve data such as Glucose uptake rate, lactate and acetate production at different time points.\r\nGrowth curve A was used train the model with external glucose concentration as well as external lactate, acetate concentration and estimated glucose acetate and lactate flux\n\n\nCREATORS\nNiels Zondervan\nPEOPLE\nNiels Zondervan\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nSUBMITTER\nNiels Zondervan\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, CCA1 prod D - PLM_88, version 1, SIMPLIFIED(id: 516 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_88, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, CCA1 prod D - PLM_88, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, CCA1 prod D - PLM_88\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nDaryl Shanley(id: 1088 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Newcastle\nPROJECTS\nMESI-STRAT\nSystems modelling age-related changes in the maintenance of dermal extra-cellular matrix: mechanisms and interventions\nAre microRNAs key mediators of cartilage destruction in osteoarthritis?\nDynamic interplay between nuclei and mitochondria in ageing cells\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nNAD + Replenishment Improves Lifespan and Healthspan in Ataxia Telangiectasia Models via Mitophagy and DNA Repair\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1 axis\n\n____________________________________________________________________________\n\nSEDML for BIOMD0000000005(id: 1354 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nBIOMD0000000005 - Tyson1991 - Cell Cycle 6 var\nCREATORS\nRon Henkel\nDagmar Waltemath\nINVESTIGATIONS\nHands-on: Model Management in SEEK\nPEOPLE\nRon Henkel\nDagmar Waltemath\nSTUDIES\nHands-On: Tyson1991 - Cell Cycle 6 var\nSUBMITTER\nRon Henkel\n\n____________________________________________________________________________\n\nFigure 2: Frequency-dependent selection of mutant pancreatic beta cells.(id: 387 | type: studies)\n\nDescription: Mathematical simulation of a tamoxifen-induced conditional knock-in of a sixfold activating GCK mutant in beta cells. (C) The percentage of beta cells with mutated GCK increases to ~25% after 3 days, but then decreases and is eliminated after a few weeks. (D) Glucose levels initially decrease after the tamoxifen injection, but return to normal after a few weeks. Insets: Experimental results of Tornovsky-Babeay et al (2014). \r\n\r\nSED-ML simulation\r\nhttps://jjj.bio.vu.nl/models/experiments/karin2017_fig2/simulate\n\n\nASSAYS\nFrequency-dependent selection of mutant pancreatic beta cells.\nCREATORS\nJacky Snoep\nINVESTIGATION\nKarin et al (2017) Molecular Systems Biology\nPEOPLE\nJacky Snoep\nPROJECTS\nMolecular Systems Biology\nSUBMITTER\nJacky Snoep\n\n____________________________________________________________________________\n\nfermentation and product spectrum of Clostridium acetobutylicum wildtype(id: 1025 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nThomas Rimpf\nPEOPLE\nThomas Rimpf\nPROJECTS\nCOSMIC\nSUBMITTER\nThomas Rimpf\n\n____________________________________________________________________________\n\nTotal RNA Quantification(id: 140 | type: sops)\n\nDescription: missing\n\n\nCREATORS\nLouise Thomas\nMaggie Smith\nPEOPLE\nMaggie Smith\nJay Moore\nLouise Thomas\nPROJECTS\nSTREAM\nSUBMITTER\nJay Moore\n\n____________________________________________________________________________\n\npSSAlib: The partial-propensity stochastic chemical network simulator(id: 367 | type: publications)\n\nDescription: missing\n\n\nMODELS\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nPilot experiment - S. solfataricus grown at 70 and 80 C.(id: 11 | type: studies)\n\nDescription: The pilot experiment has been set up in order to develop uniform SOPs for the Sulfosys consortium. It comprises proteomics, transcriptomics, metabolomics as well as enzymatic essays. Cells for all the members of the consortium have been obtained from the same batch fermentations according to fermentation SOP of Sulfosys. The pilot resulted in creating a procedures for all the techniques used in consortium.\n\n\nASSAYS\nComparison of transcriptome of S. solfataricus at 70 and 80C\nComparison of proteome of S. solfataricus at 70 and 80C\nFermentation of S. solfataricus at 70 and 80C in a batch fermenter\nEnzyme activity tests for S. solfataricus\nIntracellular metabolomics of S. solfataricus at 70 and 80C\nINVESTIGATION\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature\nPEOPLE\nPawel Sierocinski\nPROJECTS\nSulfoSys\nPUBLICATIONS\n\"Hot standards\" for the thermoacidophilic archaeon Sulfolobus solfataricus\nSOPS\nSulfosys - SOP proteomics\nSulfosys - SOP metabolomics\nSulfosys - SOP microarray\nSulfosys - SOP fermentation\nSUBMITTER\nPawel Sierocinski\n\n____________________________________________________________________________\n\nExternal concentration changes(id: 45 | type: studies)\n\nDescription: How does the transport of potassium and hydrogen ions effect the concentrations in the near surrounding of the cell.\n\n\nASSAYS\nPotassium changes\nProton changes\nINVESTIGATION\nPotassium uptake in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction.(id: 382 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nSTUDIES\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction\n\n____________________________________________________________________________\n\nOsmotically controlled synthesis of the compatible solute proline is critical for cellular defense of Bacillus subtilis against high osmolarity(id: 197 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLeif Steil\nTamara Hoffmann\nErhard Bremer\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nAnalysis of organic acid by HPLC(id: 58 | type: sops)\n\nDescription: This HPLC method uses a isocratic method and a RI detector to identify and quantify almost all excreted catabolic metabolites.\n\n\nASSAYS\nGlucose pulsed S. pyogenes\nCharacterization of flux distribution of S. pyogenes M9 wild type and the arcA- and glnA-knockout mutant in continuous cultures\nCREATORS\nMartijn Bekker\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nAmino acid metabolism of four LAB species: Streptococcus pyogenes, Lactococcus lactis, Enterococcus faecalis and Lactobacillus plantarum.\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSTUDIES\nComparative modeling and phosphate dependence flux distributions and glucose consumption rates of two lactic acid bacteria S. pyogenes and L. lactis\nArginine and Glutamine metabolism in S. pyogenes\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nLactobacillus plantarum WCFS1 protein list(id: 1062 | type: data_files)\n\nDescription: Lactobacillus plantarum WCFS1 differentially expressed proteins, comparison between low and high growth rates\n\n\nCREATORS\nAnette McLeod\nPEOPLE\nAnette McLeod\nPROJECTS\nSysMO-LAB\nSUBMITTER\nAnette McLeod\n\n____________________________________________________________________________\n\nParB-GFP ChIP on chip(id: 130 | type: assays)\n\nDescription: Cells were grown to mid-exponential phase (OD600nm ~0.2) in GM17 medium at 37C (with 0.15 mM ZnSO4 where relevant) and 84 ml of culture was mixed by inverting with 8.4 ml of fixing solution (50 mM Tris pH 8.0, 100 mM NaCl, 0.5 mM EGTA, 1 mM EDTA, 30% (v/v) formaldehyde) and incubated at room temperature for 30 min. Cells were disrupted and crosslinked DNA was sheared by sonication. Antibodies coupled to magnetic beads were used to pull down cross-linked complexes. DNA was purified, amplified, labelled and hybridised to a DNA-Microarray. See GEO submission for more details.\n\n\nDATA_FILES\nChIP-chip ParB-GFP\nINVESTIGATION\nWetlab approach to transcription fidelity\nORGANISMS\nStreptococcus pneumoniae\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\nSTUDY\nChromosome segregation in S. pneumoniae\nSUBMITTER\nJan-Willem Veening\n\n____________________________________________________________________________\n\nGeneration of cell material from Streptomyces coelicolor culture for transcriptome analysis(id: 135 | type: sops)\n\nDescription: missing\n\n\nASSAYS\nOnline/offline measurements\nCREATORS\nPer Bruheim\nTrond Ellingsen\nSunniva Hoel\nyvind Jakobsen\nHvard Sletta\nAlexander Wentzel\nAnders verby\nINVESTIGATIONS\nMetabolism of Streptomyces coelicolor (SysMO STREAM)\nPEOPLE\nTrond Ellingsen\nyvind Jakobsen\nJay Moore\nPer Bruheim\nHvard Sletta\nAnders verby\nSunniva Hoel\nAlexander Wentzel\nPROJECTS\nSTREAM\nSTUDIES\nTimeseries 1\nSUBMITTER\nJay Moore\n\n____________________________________________________________________________\n\nA bistable gene switch for antibiotic biosynthesis: the butyrolactone regulon in Streptomyces coelicolor(id: 9 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nEriko Takano\nJay Moore\nPROJECTS\nSTREAM\n\n____________________________________________________________________________\n\nPraveen Kumar Sappa(id: 284 | type: people)\n\nDescription: I am  PhD student at Prof.Uwe Voelker lab in Department of Functional Genomics. My area of research is microbial functional genomics in particular analysing the whole transcriptome(by microarray and other molecular biolology methods) of B.subtilis under various stress conditions. \r\nI use QconCAT strategy for absolute quantification of carbon metabolic enzymes via MRM(multiple reaction monitoring) by LC-MS/MS.\r\nI also perofrm experiments for understanding of dynamics of SigmaB network for modelling. \n\n\nASSAYS\nGene expression(Transcriptome)\nFermentation-BM5_SysMo\nTiling array analysis of continuous growth stress conditions in SMM\nTiling Array analysis of shocked B. subtilis cells\nTiling Array analysis of glucose strarved B. subtilis cells\nTranscriptome data for chemostat cultivated samples\nAbsolute quantification of proteins using QconCAT technology\nTranscritome data_Identification of targets of the essential RNase Y of Bacillus subtilis\nDATA_FILES\nbatchfermentation exp-starv01 transcriptome_Microarray Agilent platform_090204\nTrancriptome 20080827\nbatchfermentation exp-starv01 transcriptome_Microarray Nimblegen platform_090204\nBatchfermentation_Exp-Starv01_Fermentationdetails_090204\nContious growth_57C_SMM_Transcritome_Nimblegen\nContious growth_16C_SMM_Transcritome_Nimblegen\nContious growth_1.2M NaCl_SMM_Transcritome_Nimblegen\nM9_shake_flask_exp_trans_stat_transcriptome_nimblegen\nContinuous growth_1.2M NaCl_SMM_Growth Curve\nContinuous growth_16C_SMM_Growth curve\nContinuous growth_57C_SMM_Growth curve\nContiuous growth_37C_SMM(control)_Growth curve.\n090204 batchfermentation metabolomics\nTrancriptome data - RNA processing in Bacillus subtilis: identification of targetsof the essential RNase Y\nThe condition-dependent transcriptome of Bacillus subtilis 168\nINSTITUTIONS\nUniversity of Greifswald\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nIdentification of targets of the essential RNase Y of Bacillus subtilis\nPRESENTATIONS\nUpdate on proteome data in chemostat and shake flask experiments\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nCondition-dependent transcriptome reveals high-level regulatory architecture in Bacillus subtilis\nProteolysis of beta-galactosidase following SigmaB activation in Bacillus subtilis\nCcpA forms complexes with CodY and RpoA in Bacillus subtilis\nEnvironmental salinity determines the specificity and need for Tat-dependent secretion of the YwbN protein in Bacillus subtilis\nEfficient, global-scale quantification of absolute protein amounts by integration of targeted mass spectrometry and two-dimensional gel-based proteomics\nSOPS\nSOP for cDNA labelling and Microarray hybridizaton\nRNA isolation and quality check by Agilent bioanalyser\nTotal RNA isolation from B.Subtilis\n-Galactosidase assay\nSOP for cultivation of B.Subtilis\nSTUDIES\nBiphase Batch Fermentation(2009/02/04)\nBatchfermentation exp-starv01_090204\nTranscriptome of continuously stressed B. subtilis\nTranscriptome of shocked B. subtilis cells\nTranscriptome analysis of glucose starvation in B. subtilis\nIdentification of targets of the essential RNase Y of Bacillus subtilis\n\n____________________________________________________________________________\n\nJurgen  Haanstra(id: 387 | type: people)\n\nDescription: missing\n\n\nASSAYS\nCreating kinetic model of Trypanothione Synthetase\nModelling the gene expression cascade with length-dependent processes\nDATA_FILES\nTryS: entire dataset with simulations\nTryS: extended model description\nINSTITUTIONS\nVU University Amsterdam\nUniversity of Groningen\nINVESTIGATIONS\nDynamic modelling of redox metabolism and gene expression\nMODELS\nFinal model of TryS\nPROJECTS\nSilicoTryp\nIMOMESIC\nPUBLICATIONS\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nEvolution, dynamics and specialized functions of glycosomes in metabolism and development of trypanosomatids\nSTUDIES\nModelling of redox metabolism.\nModelling of gene expression and Regulation Analysis.\nIterative cycles of model improvement and extension.\nKinetic modelling of Trypanothione Synthetase to elucidate the enzyme mechanism and an overall rate equation\n\n____________________________________________________________________________\n\nMetabolomic systems biology of trypanosomes(id: 45 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nRainer Breitling\nMike Barrett\nBarbara Bakker\nPROJECTS\nSilicoTryp\n\n____________________________________________________________________________\n\nPROTEIN EXTRACTION FOR 2-DE  BASED PROTEOMIC EXPERIMENTS. 2. Mem(id: 110 | type: sops)\n\nDescription: Purpose: PROTEIN EXTRACTION FOR 2-DE  BASED PROTEOMIC EXPERIMENTS. 2. Membrane proteins \n\n\n\nCREATORS\nMiguel Curto Rubio\nPEOPLE\nMiguel Curto Rubio\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nTejas Borwankar(id: 168 | type: people)\n\nDescription: I am a PhD student working the group of Zoya Ignatova. Cellular and extracellular changes like crowding and osmotic stress conditions play a major role in protein aggregation. A change in the cytoplasmic composition is the result of an interplay between high osmotic pressures outside the cell volume and the cellular response to it in terms of uptake of K+ and secondary organic osmolytes. My research focuses on elucidating the role of natural osmolytes (known also as chemical chaperones or compatible solutes) that modulate protein aggregation propensity. I am using fluorescent recovery after photobleaching (FRAP) and several biochemical and biophysical methods to understand the effect of different osmolytes to counteract cellular crowing and their influence on the mobility of different protein aggregate species. \r\n\r\n\n\n\nINSTITUTIONS\nMax-Planck-Institute of Biochemistry Martinsried\nPROJECTS\nKOSMOBAC\n\n____________________________________________________________________________\n\nAnalysis of potassium efflux from Escherichia coli using flame photometry(id: 34 | type: sops)\n\nDescription: Measurements of K+ retained in the cytoplasm using flame photometry.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nTools for systems biology modeling and data exchange 2018(id: 117 | type: events)\n\nDescription: During this 3-day course, attendees will learn basic techniques for modeling of biochemical networks including data access and storage due to the FAIR principles. The first day introduces principles of stoichiometric and constraint-based modeling coupled with hands on exercises using CellNetAnalyzer. The second day continues with kinetic modeling techniques which will be illustrated and exercised with COPASI. The third day SABIO-RK is used as a resource for kinetic data and FAIRDOM Hub/SEEK is introduced as a data and model management platform fitted to the needs of systems biologists. The hands on exercises throughout the three days will ensure that attendees become familiar with the software tools and with analyzing, creating, editing, importing, simulating and storing biochemical networks.\n\n\nDATA_FILES\nSABIO-RK tasks for hands on training\nSABIO-RK results for hands on training\nSEEK tasks for hands on training\nPRESENTATIONS\nAgenda: Data and Model management 2018-04-26\nIntroduction into SABIO-RK\nIntroduction into data management using FAIRDOMHub/SEEK\nSEEK slides for hands on training\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nOsmotic control of opuA expression in Bacillus subtilis and its modulation in response to intracellular glycine betaine and proline pools(id: 193 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nUwe Voelker\nLeif Steil\nTamara Hoffmann\nErhard Bremer\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nSOP for generating a Genome Scale Metabolic model of M. hyopneumoniae(id: 288 | type: sops)\n\nDescription: Standard Operating Procedure describing the process and software used in generating a Genome Scale Metabolic model of M. hyopneumoniae.\r\nUsed software:\r\nPathway tools \r\nPathLogic\r\nCobrapy\r\nthe Cobra Toolbox\r\nlibSBML\r\n\n\n\nASSAYS\nConstruction of a Genome Scale Metabolitic model of M. hyopneumoniae\nCREATORS\nNiels Zondervan\nPEOPLE\nNiels Zondervan\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nSTUDIES\nGenome-scale, constraint-based metabolic modeling of M. hyopneumonia\nSUBMITTER\nNiels Zondervan\n\n____________________________________________________________________________\n\nCarolyn Lam(id: 220 | type: people)\n\nDescription: I am an engineer with a PhD degree in Chemical Engineering and had been working on dynamic modeling of mammalian cell culture fermentation in London for three years before moving into simulation of microbial systems. In this PSYSMO project I am mainly involved in modeling of PHAs synthesis. I am also a PAL since May 2009 - 2011 to coordinate data management and general communication among all 17 partners.\n\n\nINSTITUTIONS\nHelmholtz Centre for Infection Research Braunscheig\nPROJECTS\nPSYSMO\nPUBLICATIONS\nGenome-scale reconstruction and analysis of the Pseudomonas putida KT2440 metabolic network facilitates applications in biotechnology\nFunctional analysis of new transporters involved in stress tolerance inPseudomonas putidaDOT-T1E\nIdentification and characterization of the PhhR regulon inPseudomonas putida\nThe metabolic response of P. putida KT2442 producing high levels of polyhydroxyalkanoate under single- and multiple-nutrient-limited growth: highlights from a multi-level omics approach\nPseudomonas putida KT2440 genome update by cDNA sequencing and microarray transcriptomics\nFunctional genomics of the initial phase of cold adaptation of Pseudomonas putida KT2440\n\n____________________________________________________________________________\n\nPhillipp Schmidt(id: 537 | type: people)\n\nDescription: I am a beginning PhD student at the VU in Amsterdam and study the heterogeneity of yeast cells at near zero growth conditions. I have a versatile background in Biophysics and Systems Biology.\n\n\nINSTITUTIONS\nVU University Amsterdam\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nDetermination of intracellular metabolites(id: 97 | type: sops)\n\nDescription: Sample preparation - SOP for sampling, preparation of cell extracts, and general assay set-up\n\n\nASSAYS\nCellular size and granularity during glucose pulse\nSteady state concentrations of intracellular metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nDynamics of intracellular metabolites during glucose pulse\nDynamics of macromolecules during glucose pulse\nCREATORS\nFemke Mensonides\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nSteady state metabolic fluxes and metabolite concentrations in anaerobic chemostat of yeast Saccharomyces cerevisiae\nPEOPLE\nFemke Mensonides\nMaksim Zakhartsev\nPROJECTS\nMOSES\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSteady state concentrations of metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nAnalysis of mRNA decay by deep sequencing reveals the roles of exoribonucleases in control of the trypanosome transcriptome(id: 25 | type: presentations)\n\nDescription: Poster presented during the ICSB 2011 conference in Mannheim\n\n\nCREATORS\nAbeer Fadda\nChristine Clayton\nPEOPLE\nAbeer Fadda\nChristine Clayton\nPROJECTS\nSilicoTryp\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nCalculation of qATP for assessment of maintenance and qATP at the maximal growth rate(id: 868 | type: data_files)\n\nDescription: qATP values were calculated based on fermentation product formation and expected used biochemical pathways. These were averaged and used for calculation of the ATP required for maintenance and for growth. These data were subsequently used to calculate the qATP at the maximal growth rate by extrapolation\n\n\nCREATORS\nMartijn Bekker\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nPUBLICATIONS\nPhysiological characterization of three lactic acid bacteria and their isogenic ldh deletion mutants shows that these organisms are optimized for Yatp and not ATP formation rates at their physiological pH\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nUntargeted metabolite analysis(id: 103 | type: studies)\n\nDescription: In addition to the highly targeted quantification of metabolites already known to play major roles in oxidative stress, to provide data directly compatible with current models, we will also take an untargeted metabolomics approach. This will enable us to identify other areas of the metabolome influenced by, or influencing, oxidative stress and will allow us to compare changes in each of the stress-inducing stimuli. We have recently pioneered untargeted metabolite profiling of T. brucei using ultra-high resolution mass spectrometry (Kamleh et al. 2008 PMID: 18470888). This will link directly to the modelling in WP1 by providing the high-accuracy metabolome data for the ab inito networks. We have, so far, been able to identify in the order of 400 metabolites in trypanosomes using a single, simple sample preparation protocol. The untargeted metabolite approach enables perturbations to multiple parts of the cellular metabolome to be assessed in a single experiment and also point to changes in metabolism secondary to inhibition of target enzymes. For example, changes to NADP/NADPH balance can affect other biochemical pathways i.e. secondary effects will ripple out through the metabolome as a consequence of primary changes. Data obtained on changes to metabolite levels will be compared to dynamic changes in transcript and protein levels. In this manner we hope to extend our knowledge of pathways involved in oxidative stress defence, beyond the already established pathways involved in trypanothione production.\n\n\nASSAYS\nMetabolite profiling on T. brucei exposed to oxidative stress\nIntracellular metabolite concentrations in T. brucei exposed to oxidative stress\nDATA_FILES\nMetabolic changes by oxidative stress in T. b. brucei 427\nAbsolute intracellular concentrations of metabolites extracted from T. b. brucei 427 exposed to methylene blue\nINVESTIGATION\nMetabolite profiling, quantification and flux quantification\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSOPS\nSample preparation for intracellular metabolite extraction in T. b. brucei 427\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nSUMO mRNA isolation Epicentre(id: 80 | type: sops)\n\nDescription: This file describes how to isolate mRNA from E. coli using the kit from Epicentre, for gene expression analysis via RT-PCR\n\n\nASSAYS\nAnalysis of gene expression rates at different aerobiosis levels via RT-PCR\nCharacterization of MG1655 and mutant strains under conditions of glucose excess and limitation.\nAerobic and anaerobic characterization of MG1655 and mutant strains with regard to transcription profiles.\nGene expression analysis of mutants with linear electron transport chain at different aerobiosis levels\nCREATORS\nSonja Steinsiek\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nAnalysis of the glucose transport in Escherichia coli under aerobic and anaerobic conditions\nAnalysis of Escherichia coli with linear electron transport chain\nPEOPLE\nSonja Steinsiek\nPROJECTS\nSUMO\nSTUDIES\nDetermination of the impact of specific enzyme reactions and regulatory proteins using isogenic E. coli mutant strains\nCharacterization of mutant strains with defects in sugar transport systems\nAnalysis of Escherichia coli strains with linear respiratory chain\nSUBMITTER\nSonja Steinsiek\n\n____________________________________________________________________________\n\nDynamic interplay between nuclei and mitochondria in ageing cells(id: 136 | type: projects)\n\nDescription: We will systematically analyse large datasets of multiple types to: (i) identify key components\r\naffected by age or experimental perturbation; (ii) establish networks of interaction; (iii) develop dynamic\r\ncomputational models based on these networks; (iv) use model selection methods to discriminate between\r\nalternative network topologies and generate predictive models. To characterise the data, we will apply an\r\nensemble of methods, including frameworks in R/Bioconductor and toolboxes connected via APIs, algorithms for\r\nmachine learning and deep learning, mutual information, Bayesian inference and various cluster analysis. Each\r\nstrategy will be assessed for quality of output, efficiency, ease of use and maintainability. We will develop full\r\ninteraction networks by employing a different set of multi-strategy workflows on enhanced data sets\r\nsupplemented with data generated from cell cultures and in-vitro model systems upon perturbation. This will\r\nrequire the development of novel multi-workflows to enable data integration and analysis of temporal\r\nrelationships across age and/or biological processes. Strategies will employ suites of apps within Cytoscape\r\nas well as emerging technologies such as Active Interaction Mapping where network\r\nconnections are driven by function. The outputs will be used to inform the network topology of computational\r\nmodels of molecular mechanisms and cellular processes.\n\n\nINSTITUTIONS\nUniversity of Newcastle\nUniversity of Copenhagen\nNational Institute on Aging\nPEOPLE\nDaryl Shanley\nAlvaro Martinez Guimera\nLene Juel Rasmussen\nDeborah Croteau\nVilhelm Bohr\nAna Serrano Puebla\nPROGRAMMES\nNewcastle University Systems Modelling of Ageing\nPUBLICATIONS\nNAD + Replenishment Improves Lifespan and Healthspan in Ataxia Telangiectasia Models via Mitophagy and DNA Repair\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1 axis\n\n____________________________________________________________________________\n\nRichard (1996) Concentrations of oscillating glycolytic intermediates in yeast cultures(id: 998 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nDetailed kinetic model of yeast glycolytic oscillation\nMetabolite concentrations in yeast glycolytic oscillations\nCREATORS\nFranco Du Preez\nJacky Snoep\nINVESTIGATIONS\nYeast Glycolytic Oscillations\nPEOPLE\nJacky Snoep\nFranco Du Preez\nPROJECTS\nSysMO DB\nPUBLICATIONS\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction\nSTUDIES\nDetailed kinetics of yeast glycolytic oscillation\nSUBMITTER\nFranco Du Preez\n\n____________________________________________________________________________\n\nRNA-Seq(id: 143 | type: assays)\n\nDescription: S.pneumoniae D39 cells (wild type and delta greA) were grown in C+Y medium and cells were harvested for total RNA isolation at mid-exponential growth (OD600nm 0.3 for wt, 0.25 for delta greA). Total RNA was isolated as described before (Kloosterman et al 2006).\r\nThe total RNA samples were examined by capillary electrophoresis.\r\ndephosphorylated with antarctic phosphatase followed by treatment with polynucleotide kinase (PNK).\r\nAfterwards, samples were poly(A)-tailed using poly(A) polymerase. Then a RNA adapter was ligated to\r\nthe 5-phosphate of the RNA fragments. First-strand cDNA synthesis was performed using an\r\noligo(dT)-adapter primer and the M-MLV reverse transcriptase. The resulting cDNAs were PCRamplified\r\nto about 30 ng/l using a high fidelity DNA polymerase. PCR cycles performed and barcode\r\nsequences, which are part of the 3' TruSeq sequencing adapter, are included in Table 2. The cDNAs\r\nwere purified using the Agencourt AMPure XP kit (Beckman Coulter Genomics) and analyzed by\r\ncapillary electrophoresis.\r\nFor Illumina sequencing, the cDNA in the size range of 300 - 500 bp was eluted from preparative\r\nagarose gels. Aliquots of the size fractionated cDNA were analyzed by capillary electrophoresis.\r\nThe cDNAs are double stranded and have a size of about 300  500 bp. The primers used for PCR\r\namplification were designed for TruSeq sequencing according to the instructions of Illumina.\r\nThe following adapter sequences flank the cDNA inserts:\r\nTrueSeq_Sense_primer\r\n5- AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT-3\r\nTrueSeq_Antisense_primer Barcode\r\n5-CAAGCAGAAGACGGCATACGAGAT-NNNNNN-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATC(dT25)-3\r\nThe combined length of the flanking sequences is 146 bases.\r\nProperties of cDNA samples\r\nNo. 1 2\r\nSample WT greA\r\nBarcode ATCACG CGATGT\r\nPCR cycles 12 12\r\n\r\nFor sequencing, the two cDNA samples were pooled in approximately equimolar amounts.The cDNA\r\npool was then sequenced on a Illumina HiSeq 2000 system.\r\n\r\nWT 110901_SN865_B_L006_R1_GQC-28-\r\nATCACG.fastq.gz\r\n100 19.624.852 1,29\r\ngreA 110901_SN865_B_L006_R1_GQC-28-\r\nCGATGT.fastq.gz\r\n100 20.632.085 1,31\r\n\r\nllumina fastq format\r\n4 lines for each sequence:\r\n1- Unique identifier, with the following format:\r\n@<machine_id>:<lane>:<tile>:<x_coord>:<y_coord>#<index>/<read_#>\r\n2- Sequence (A, T, C ,G or N (undetermined) only)\r\n3- Orientation (always forward without mapping)\r\n4- Quality value for each base, corresponding to a Phred-like score encoded in ASCII format, with an\r\noffset of of 33 (e.g. J gives a value of 41) and is in accordance with sanger FASTQ format.\r\nThe sequence file is compressed as tar.gz archive. The archive can be uncompressed using a tar\r\n-zxvf command.\n\n\nDATA_FILES\nRNA-seq wild type\nRNA-seq delta greA\nINVESTIGATION\nWetlab approach to transcription fidelity\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\nPUBLICATIONS\nControl of transcription elongation by GreA determines rate of gene expression in Streptococcus pneumoniae\nSTUDY\nThe role of transcription factor GreA in transcription fidelity and processivity in Streptococcus pneumoniae\nSUBMITTER\nJan-Willem Veening\n\n____________________________________________________________________________\n\nInduction of RNA interference in T.brucei(id: 1024 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nInduction of RNA interference in T.brucei brucei 2T1 cell line\nCREATORS\nFederico Rojas\nINVESTIGATIONS\nPerturbing the biological system: developmental adaptation and experimental manipulation of trypanosome biology\nPEOPLE\nFederico Rojas\nPROJECTS\nSilicoTryp\nSTUDIES\nTargeted disruption and over expression of critical enzymes\nSUBMITTER\nFederico Rojas\n\n____________________________________________________________________________\n\nCaroline Green(id: 526 | type: people)\n\nDescription: Hi, this is Caroline from Creative Biomart, a world leading biotech company that specialized in providing protein, recombinant protein, cell lines, and GMP proteins.\r\nvisit http://www.creativebiomart.net/ to see more details.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nINBioPharm(id: 55 | type: projects)\n\nDescription: INBioPharm - Integrated Novel Natural Product Discovery and Production Platform for Accelerated Biopharmaceutical Innovation from Microbial Biodiversity project  will develop a new, generic technology platform for the more efficient discovery of novel bioactive compounds with improved prospects to become medical products. A unique national collection of marine microorganisms at SINTEF/NTNU will be used as the basis for the development of different new, complementary molecular biology, analytical and screening methods and their combination into one platform that can access and exploit the huge potential of microorganisms in nature to synthesize useful bioactive natural products in a more efficient way. Central to this effort will be the development of the actinobacterium Streptomyces coelicolor into a microbial cell factory that is capable of producing diverse bioactive compounds of other natural microorganisms, including the huge majority of bacteria that cannot be cultivated in the laboratory, and the large number of compounds of cultivable bacteria that are not produced under laboratory conditions.\r\nThe INBioPharm project is funded by the Research Council of Norway with 27.5 million NOK as part of the 'Centre for Digital Life Norway' established in 2016.\n\n\nINSTITUTIONS\nNorwegian University of Science and Technology\nSINTEF\nMODELS\niKS1317.xml\nORGANISMS\nStreptomyces coelicolor\nPEOPLE\nTrond Ellingsen\nPer Bruheim\nHvard Sletta\nSven Even Borgos\nAlexander Wentzel\nEivind Almaas\nTonje Heggeset\nGiang-Son Nguyen\nDino Van Dissel\nSnorre Sulheim\nAnna Lewin\nKanhaiya Kumar\nTjasa Kumelj\nKristin Degnes\nAnna Nordborg\nGeir Klinkenberg\nPROGRAMMES\nIndependent Projects\nPUBLICATIONS\nPredicting strain engineering strategies using iKS1317: a genomescale metabolic model of Streptomyces coelicolor\n\n____________________________________________________________________________\n\nMiniaturized and automated adaptive laboratory evolution: Evolving Corynebacterium glutamicum towards an improved d-xylose utilization.(id: 344 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nStephan Noack\nNiklas Tenhaef\nChristian Brsseler\nJan Marienhagen\nPROJECTS\nXyloCut\n\n____________________________________________________________________________\n\nIntegrated summary of transcriptomics and proteomics results(id: 1646 | type: data_files)\n\nDescription: Cells of S. solfataricus were grown on either caseinhydrolysate or D-glucose (reference) as sole carbon source.\n\n\nASSAYS\nProteome analysis: Casaminoacids versus D-Glc\nMetabolome analysis: Casaminoacids versus D-Glc\nCREATORS\nHelge Stark\nINVESTIGATIONS\nAmino acid degradation in Sulfolobus solfataricus P2\nPEOPLE\nHelge Stark\nPROJECTS\nSulfoSys - Biotec\nSTUDIES\nComparison of Sulfolobus solfataricus P2 grown on caseinhydrolysate and D-glucose\nSUBMITTER\nHelge Stark\n\n____________________________________________________________________________\n\nIntracellular Metabolite Pool Changes in Response to Nutrient Depletion Induced Metabolic Switching in Streptomyces coelicolor(id: 298 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nAlexander Wentzel\nPROJECTS\nSTREAM\nSTUDIES\nTimeseries 1\n\n____________________________________________________________________________\n\nMarkus Stepath(id: 492 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nPUBLICATIONS\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nMeasurement of intracellular metabolite pools upon methylglyoxal stress in E. coli using LC-MS-MS.(id: 22 | type: sops)\n\nDescription: A procedure to measure the levels of GSH and SLG before and after exposure to MG using formic acid. A similar expreimental set up as for potassium efflux experiments is used to which a silicon oil centrifugation step is incorporated. Samples are analysed by LC-MS-MS in order to quantify intracellular GSH and SLG levels.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nKinetic understanding of the T. brucei trypanothione synthesis pathway(id: 41 | type: investigations)\n\nDescription: Aim. To provide critical quantitative parameter information and to model redox balance by determining the cellular concentration of all enzymes involved in the trypanothione-dependent hydroperoxide detoxification system of trypanosomes and by performing the kinetic characterization of the involved enzymes under pseudo-physiological conditions.\n\n\nASSAYS\nTrypanothione synthetase ATP consumption steady state data\nCreating kinetic model of Trypanothione Synthetase\nTrypanothione synthetase Gsp and T(SH)2 production measured by HPLC\nDATA_FILES\nActivity of TbTryS under different substrate/product concentrations measured by the spectrophotometric assay\nTryS: entire dataset with simulations\nTryS: extended model description\nActivity of TbTryS measured by HPLC\nMODELS\nFinal model of TryS\nPEOPLE\nAlejandro Leroux\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nSOPS\nIn vivo-like buffer for enzymatic measurements\nTrypanothione synthetase enzymatic assay\nDerivatization of Gsp and T(SH)2 produced in the trypanothione synthetase enzymatic assay\nSTUDIES\nKinetic characterization of trypanothione-dependent enzymes\nDetermination of the redox state and the total concentration of the trypanothione-dependent detoxification enzymes in vivo\nKinetic modelling of Trypanothione Synthetase to elucidate the enzyme mechanism and an overall rate equation\nSUBMITTER\nAlejandro Leroux\n\n____________________________________________________________________________\n\nBerlin Progress Report(id: 32 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nSebastian Curth\nEVENTS\nBerlin Meeting 2012/04/05\nPEOPLE\nSebastian Curth\nPROJECTS\nCOSMIC\nSUBMITTER\nSebastian Curth\n\n____________________________________________________________________________\n\nHow starvation affect protein content in yeast cells(id: 63 | type: assays)\n\nDescription: Determination of protein content at several times.\r\n\r\nThe potassium content measurements of Navarrete et al. (2010) are the basis for potassium content analysis in various mutants (Nha1, Trk1, Trk2).\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nKey Cell-Physiological Parameters\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\n JERM templates presentation  at SEEK Users WS 2012 in Berlin(id: 39 | type: presentations)\n\nDescription: Presentation of JERM templates at SEEK Users WS 2012 in Berlin (Olga Krebs)\n\n\nCREATORS\nOlga Krebs\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nExtraction and quantitative analysis of proteins from FFPE tissue(id: 253 | type: sops)\n\nDescription: A Standard Operating Procedure (SOP) is a document consisting of step-by-step information on how to execute a task. An existing SOP may need to just be modified and updated. \n\n\nCREATORS\nOlga Krebs\nPEOPLE\nOlga Krebs\nPROJECTS\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nMolecular sieving properties of the cytoplasm of Escherichia coli and consequences of osmotic stress(id: 119 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nBert Poolman\nJacek Mika\nPROJECTS\nKOSMOBAC\n\n____________________________________________________________________________\n\nFederico Rojas(id: 348 | type: people)\n\nDescription: I am a Postdoc at Keith Matthews lab in the Institute of Immunology and Infection Research, Edinburgh University. As part of the SilicoTryp project we are in charge of performing  Targeted disruption and Overexpression of critical enzymes of Trypanosoma brucei redox metabolism enzymes and developmental perturbations to provide part of the necessary data for the construction of the model. Also generate consistent samples, so that data can be integrated and quantification results are guarateed to be consistent and comparable across different laboratories and experiments\n\n\nASSAYS\nGeneration of RNAi cell lines in T.brucei brucei 2T1 cell line\nInduction of RNA interference in T.brucei brucei 2T1 cell line\nDATA_FILES\nInduction of RNA interference in T.brucei\nRNAi growth curves\nINSTITUTIONS\nUniversity of Edinburgh\nINVESTIGATIONS\nPerturbing the biological system: developmental adaptation and experimental manipulation of trypanosome biology\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSOPS\nGeneration of RNAi cell lines in T.brucei brucei 2T1 cell line\nSample preparation for mRNA decay study\nInduction of RNAi in Trypanosoma brucei\nSTUDIES\nTargeted disruption and over expression of critical enzymes\n\n____________________________________________________________________________\n\nHeterochronic phosphorelay gene expression as a source of heterogeneity in Bacillus subtilis spore formation(id: 72 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nUwe Voelker\nImke De Jong\nJan-Willem Veening\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona (id: 53 | type: events)\n\nDescription: Good data and model management improves the longevity and impact of your interdisciplinary research. FAIRDOM offers software and expertise to support you in better managing your interdisciplinary life-science projects, particularly in systems and synthetic biology. If you have never heard of data and model management, or are curious about it, or you are an expert keen to exchange ideas, our user meeting is the place for you! \r\nAt the meeting you can\r\n- Learn why data and model management is important, and how it can improve the longevity and impact of your research. \r\n- Learn how other research groups formulate their data and model management using FAIRDOM software. \r\n- Have direct contact with the FAIRDOM Community and Tech Team, who can give you personalised advice on your data and model management, and using FAIRDOM. \r\n- Learn how users have extended the platforms to include e.g. electronic lab notebooks. \r\n- Be among the first people to hear about how to publish citable data and models using FAIRDOM!\r\n\r\nWorkshop agenda\r\n\r\n09:00  09:10\tWelcome \r\nExploring data and model management \r\n09:10  10:40\tInvited Talk: Gareth Knight on Rewarding open science practice in funded research \r\nResearch Data Manager, London School of Hygiene & Tropical Medicine \r\n09:40  10:10\tFAIRDOM introduction, Carole Goble \r\nThe University of Manchester, UK \r\n10:10  10:25\tFlash talks  3 minutes one slide (slots still available  register here) \r\nAlexey Kolodkin  Dynamic network modelling: from experimental data to design principles and personalised therapies \r\nNikolaus Sonnenschein  DD-DeCaF: Bioinformatics Services for Data-Driven Design of Cell Factories and Communities \r\nMarie Vidal  Upcoming CORBEL Open Call for advanced research projects \r\nIvelize Rocha Bernardo  Sharing and Reusing Spreadsheets through a Discovery of a Common Vocabulary/Schema \r\n10:25  10:35\tMartin Golebiewski \r\n10:35  11:00\tCoffee \r\nStar use cases \r\n11:00  11:15\tSiberian SEEK, Sergey Lashin, Institute of Cytology and Genetics SB RAS, Russia \r\n11:15  11:30\tSynthSys, Tomasz Zielinski, University of Edinburgh, UK \r\n11:30  11:45\tDigital Salmon, Jon Olav Vik, Norwegian University of Life Sciences, Norway \r\n11:45  12:00\tIMOMESIC, Martin Bhm, DKFZ, Heidelberg, Germany \r\n12.00  12.15\tLiSyM, Matthias Knig, Charite  Universittsmediz Berlin, Germany \r\n12:15  12:30\tMOSES, ZucAt and ExtremoPharm, Maksim Zakhartsev, Hohenheim University, Stuttgart, Germany \r\n12:30  13:00\tDiscussion \r\n13:00  13:45\tLunch \r\nParallel Session \r\n13:45  15:15\t\r\nFAIRDOM Training and helpdesk \r\nGroup discussions \r\n15:15  15:45\tcoffee \r\nLooking to the future \r\n15:45  16:15\tInvited talk: Yannick Wurm \r\nSenior Lecturer in Bioinformatics, Queen Mary University London, UK \r\n16:15  16:45\tPublishing a FAIR study with citable research assets, Jacky Snoep \r\nStellenbosch University, South Africa \r\n16:45  17:00\tWrap up \r\n18:00\tGroup dinner\n\n\nPRESENTATIONS\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nMy.PAL history & experience\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\nA major transition in social evolution & some data tidbits\nSiberian SEEK: Multiscale modeling of microbial communities\nFAIRDOM Introduction\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nNeurospora Circadian Clock 3-variable model - PLM_51, version 1, SIMPLIFIED(id: 486 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_51, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nNeurospora Circadian Clock 3-variable model - PLM_51, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nAdams, Richard\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nNeurospora Circadian Clock 3-variable model - PLM_51\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction(id: 176 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nModelling sustained glycolytic oscillations in individual isolated yeast cells (Gustavsson models)\nDetailed kinetic model of yeast glycolytic oscillation\nDATA_FILES\nSustained glycolytic oscillations in yeast\nRichard (1996) Concentrations of oscillating glycolytic intermediates in yeast cultures\nDas (1991) NADH and ATP concentrations determined during different phases of glycolytic oscillation in yeast extracts\nDas (1991) Time varying concentrations for metabolites and cofactors in upper glycolysis measured in oscillating yeast extracts\nMODELS\nDetailed kinetic model of yeast glycolytic oscillation - I\nDetailed kinetic model of yeast glycolytic oscillation - II\nDetailed kinetic model of yeast glycolytic oscillation - III\nDetailed kinetic model of yeast glycolytic oscillation - IV\nDetailed kinetic model of yeast glycolytic oscillation - V\nDetailed kinetic model of yeast glycolytic oscillation - VI\nDetailed kinetic model of yeast glycolytic oscillation - VII\nSustained glycolytic oscillations in individual isolated yeast cells - IV\nSustained glycolytic oscillations in individual isolated yeast cells - III\nSustained glycolytic oscillations in individual isolated yeast cells - II\nSustained glycolytic oscillations in individual isolated yeast cells - I\nPEOPLE\nJacky Snoep\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nSiberian SEEK: Multiscale modeling of microbial communities(id: 301 | type: presentations)\n\nDescription: Talk given by Sergey Lashin from Institute of Cytology and Genetics, Novosibirsk,  Russia\n\n\nCREATORS\nSergey Lashin\nAlexandra Klimenko\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nAlexandra Klimenko\nSergey Lashin\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nPROTEIN EXTRACTION FOR 2-DE  BASED PROTEOMIC EXPERIMENTS. 1. Cit(id: 109 | type: sops)\n\nDescription: Purpose: PROTEIN EXTRACTION FOR 2-DE  BASED PROTEOMIC EXPERIMENTS. 1. Citosoluble proteins\n\n\n\nCREATORS\nMiguel Curto Rubio\nPEOPLE\nMiguel Curto Rubio\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\r\nMESI-STRAT Coordinator\r\nLeader WP8  Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nSTREAM(id: 10 | type: projects)\n\nDescription: Global metabolic switching in Streptomyces coelicolor\r\n\r\n\r\nAntibiotics are made during the second phase of growth when there is a transition in metabolism from primary to secondary metabolism. Primary metabolism is growth related and involves all the normal cellular activities associated with cell growth and division. Whereas secondary metabolism is non-growth linked and is non-essential but many important activities occur during this phase which help the bacterium survive. \n\n\nASSAYS\nOnline/offline measurements\ntranscriptomics\nmetabolomics\nproteomics\nMetabolic pathway curation\nDATA_FILES\nFermentor 199 transcriptomics\nFermentor 199 cultivation data\nScoCyc 14.6 release bundle\nScoCyc 14.5 release bundle\nINSTITUTIONS\nINBIOTEC Leon\nSINTEF\nUniversity of Aberdeen\nUniversity of Groningen\nUniversity of London UCL\nUniversity of Tuebingen\nUniversity of Warwick\nINVESTIGATIONS\nMetabolism of Streptomyces coelicolor (SysMO STREAM)\nMODELS\nScoCyc 14.5 release bundle\nORGANISMS\nStreptomyces coelicolor\nPEOPLE\nDavid Hodgson\nDavid Rand\nDavid Wild\nElizabeth Wellington\nEriko Takano\nGregory Challis\nJuan-Francisco Martin\nKay Nieselt\nLubbert Dijkhuizen\nMaggie Smith\nNigel Burroughs\nPreben Krabben\nRainer Breitling\nTrond Ellingsen\nWolfgang Wohlleben\nWalid Omara\nM. Tauqeer Alam\nWilliam Gaze\nyvind Jakobsen\nBrent Kiernan\nRoxane Legaie\nJay Moore\nPer Bruheim\nAnthony Palathingal\nR Jansen\nJens Reuther\nAntonio Rodrguez-Garca\nLouise Thomas\nHvard Sletta\nAnders verby\nSunniva Hoel\nSven Even Borgos\nMorris Swertz\nMaria Elena Merlo\nAlexander Wentzel\nPROGRAMMES\nSysMO\nPUBLICATIONS\nMetabolic modeling and analysis of the metabolic switch in Streptomyces coelicolor\nThe dynamic architecture of the metabolic switch in Streptomyces coelicolor\nA bistable gene switch for antibiotic biosynthesis: the butyrolactone regulon in Streptomyces coelicolor\nChapter 6. Regulation of antibiotic production by bacterial hormones\nNew surveyor tools for charting microbial metabolic maps\nMetaNetter: inference and visualization of high-resolution metabolomic networks\nProbabilistic assignment of formulas to mass peaks in metabolomics experiments\nIncreasing the mass accuracy of high-resolution LC-MS data using background ions: a case study on the LTQ-Orbitrap\nPhosphate-dependent regulation of the low- and high-affinity transport systems in the model actinomycete Streptomyces coelicolor\nPhosphate control over nitrogen metabolism in Streptomyces coelicolor: direct and indirect negative control of glnR, glnA, glnII and amtB expression by the response regulator PhoP\nThe roles of the nitrate reductase NarGHJI, the nitrite reductase NirBD and the response regulator GlnR in nitrate assimilation of Mycobacterium tuberculosis\nThe Streptomyces coelicolor GlnR regulon: identification of new GlnR targets and evidence for a central role of GlnR in nitrogen metabolism in actinomycetes\nAntibiotic resistance in the environment, with particular reference to MRSA\nPrevalence of sulfonamide resistance genes in bacterial isolates from manured agricultural soils and pig slurry in the United Kingdom\nAn approach to pathway reconstruction using whole genome metabolic models and sensitive sequence searching\nCross-talk between two global regulators in Streptomyces: PhoP and AfsR interact in the control of afsS, pstS and phoRP transcription\nA technical platform for generating reproducible expression data from Streptomyces coelicolor batch cultivations\nExtracting regulator activity profiles by integration of de novo motifs and expression data: characterizing key regulators of nutrient depletion responses in Streptomyces coelicolor\nMetabolic switches and adaptations deduced from the proteomes of Streptomyces coelicolor wild type and phoP mutant grown in batch culture\nThe PII protein GlnK is a pleiotropic regulator for morphological differentiation and secondary metabolism in Streptomyces coelicolor\nOptimized submerged batch fermentation strategy for systems scale studies of metabolic switching in Streptomyces coelicolor A3(2)\nIntracellular Metabolite Pool Changes in Response to Nutrient Depletion Induced Metabolic Switching in Streptomyces coelicolor\nSOPS\nPreparation of inoculum for Streptomyces coelicolor cultivation\nSampling from Streptomyces coelicolor culture for subsequent offline analyses\nCell dry weight (CDW) determination of Streptomyces coelicolor culture\nGeneration of cell material from Streptomyces coelicolor culture for proteome analysis\nGeneration of cell material from Streptomyces coelicolor culture for transcriptome analysis\nGeneration of cell material from Streptomyces coelicolor culture for metabolome analysis\nContamination test of Streptomyces coelicolor culture\nProcessing Streptomyces coelicolor cell pellets\nTotal Protein Quantification\nTotal RNA Quantification\nTotal DNA Quantification\nProteomic Analysis of the Streptomyces coelicolor Proteome\nMCF derivatization\nMetabolome extraction\nSpore batch generation\nSTUDIES\nTimeseries 1\nScoCyc metabolic pathway curation\n\n____________________________________________________________________________\n\nDong-Hyun Kim(id: 360 | type: people)\n\nDescription: missing\n\n\nASSAYS\nGeneration of uniformly 13C-labelled E. coli extract\nLC-MS based absolute quantification of intracellular metabolites\nMetabolite profiling on T. brucei exposed to oxidative stress\nIntracellular metabolite concentrations in T. brucei exposed to oxidative stress\nIntracellular metabolite concentrations in T. brucei under pH stress\nLC-MS based absolute quantification of extracellular metabolites\nDATA_FILES\nMetabolic changes by oxidative stress in T. b. brucei 427\nAbsolute intracellular concentrations of metabolites extracted from T. b. brucei 427 wild type\nConstruct of Arginine kinase gene knockout T. b. brucei 427\nConcentrations of extracellular metabolites from T. b. brucei 427\nAbsolute intracellular concentrations of metabolites extracted from T. b. brucei 427 exposed to methylene blue\nAbsolute intracellular concentrations of metabolites extracted from T. b. brucei 427 under pH stress\nINSTITUTIONS\nUniversity of Glasgow\nINVESTIGATIONS\nMetabolite profiling, quantification and flux quantification\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nSOPS\nSample preparation for intracellular metabolite extraction in T. b. brucei 427\nGeneration of uniformly 13C-labelled E. coli extract for absolute quantification\nSample preparation procedure for footprint analysis on T. b. brucei 427 using LC/MS\nSTUDIES\nTargeted metabolite analysis\nUntargeted metabolite analysis\nMetabolic flux measurement\n\n____________________________________________________________________________\n\nSupplementary Information on FMv1 model(id: 2210 | type: data_files)\n\nDescription: Supplementary information file from Chew et al. PNAS 2014, including full model description for Arabidopsis Framework Model v1, model simulations and experimental validations.\n\n\nASSAYS\nArabidopsis Framework Model v1, Matlab and Simile version\nCREATORS\nAndrew Millar\nYin Hoon Chew\nINVESTIGATIONS\nArabidopsis Framework Model v1, predicting rosette growth with a multi-model ensemble\nPEOPLE\nAndrew Millar\nYin Hoon Chew\nPROJECTS\nMillar group\nTiMet\nPUBLICATIONS\nMultiscale digital Arabidopsis predicts individual organ and whole-organism growth\nSTUDIES\nConstruction of Framework Model v1\nSUBMITTER\nAndrew Millar\n\n____________________________________________________________________________\n\nComparison of Kcat values model and values from literature(id: 1695 | type: data_files)\n\nDescription: Comparison of Kcat values model and values from literature. Model values are based on Vmax enzyme parameters (maximum activity per enzyme molecule).\r\nLiterature values are largely based on whole cell enzyme extract assays and do not take into account allosteric control. In addition activity is measured at varying time points and varying conditions. The error based on differences in enzyme concentrations at different time points and the error in protein copy number measurements is taken into account in the comparison. In addition Sensitivity analysis show the importance of different Vmax values on flux through glycolysis.\n\n\nASSAYS\nComparison of Kcat values from the model and values from literature\nCREATORS\nNiels Zondervan\nPEOPLE\nNiels Zondervan\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nSUBMITTER\nNiels Zondervan\n\n____________________________________________________________________________\n\nProbabilistic assignment of formulas to mass peaks in metabolomics experiments(id: 13 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nRainer Breitling\nJay Moore\nPROJECTS\nSTREAM\n\n____________________________________________________________________________\n\nSalazar_et_al_2009_Photoperiodism_Model - PLM_74(id: 529 | type: studies)\n\nDescription: \tThis is the SimileXML for the Salazar2009_FloweringPhotoperiod model in PlaSMo. It corresponds to Model 3 in the publication of Salazar et al 2009. The Simile version of this model is also attached here.\t<strong>Instructions to run the Photoperiodism Model in Simile</strong>\t1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Save all the files into the same folder.\t2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Copy and paste the attached lightfunction.pl file in the following folder:\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Program File &gt; Simile6.0 (or other software version)&gt; Functions\t3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Download the attached Simile file or import the XML into Simile:\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; File &gt; Import &gt; XML Model Description\t4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; To run the model:\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Model &gt; Run or click on the Play (triangle) button\t5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; When you run the model for the first time, you will be prompted to load the parameter file. To do so:\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Load the Salazar model.spf file at the TOP LEVEL\t6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The time unit and time step also need to be adjusted as follows:\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The time step can be set in the run display window. The time step #1 should be set to 0.001\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; b.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The time unit should be changed to hour, which can be done in the Run settings tab in the run display window<br><br><strong>Related Publications</strong></strong><br>Salazar JD, Saithong T, Brown PE, Foreman J, Locke JC, Halliday KJ, Carr IA, Rand DA, Millar AJ.  (2009). Prediction of photoperiodic regulators from quantitative gene circuit models . Cell. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/20005809  <br><br><strong>Originally submitted to PLaSMo on 2013-11-12 15:06:57</strong></strong>\n\n\nASSAYS\nSalazar_et_al_2009_Photoperiodism_Model - PLM_74, version 1\nCREATORS\nBioData SynthSys\nDATA_FILES\nSimile file for the Salazar et al 2009 Photoperiodism Model, PLM_74_ 1\nParameter file to run the model in Simile, PLM_74_ 1\nA Simile function file for light input into the clock model , PLM_74_ 1\nINVESTIGATION\nHoon, Yin\nMODELS\nSalazar_et_al_2009_Photoperiodism_Model - PLM_74, version 1, SUBMITTED\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nLuis Serrano(id: 909 | type: people)\n\nDescription: Luis Serrano did his PhD at the CBM (Madrid, Spain) on Cell Biology. Then he spent 4 years in the laboratory of Prof. A.R. Fehrs (MRC, UK) working in protein folding. In 1993, he became Group Leader at the EMBL (Heidelberg, Germany) working in Protein Folding and design. Ten years later, he was appointed head of the Structural & Computational Biology programme at the EMBL and he started to work on Systems Biology. By the end of 2006 he moved back to Spain to lead a programme working on Systems Biology, where he was appointed vice-director before finally becoming the CRG director on July 2011. His group is currently focused on Synthetic Biology, engineering and designing of biological systems. He is EMBO and RACEFyN member and received the Marie Curie Excellence Award. He has also been awarded with the prestigious ERC Advanced Grant and participates as Principal Investigators in many research projects financed both by the European Commission (through the 6th and 7th Framework Programmes\n\n\nASSAYS\nTranscriptomics assay of M. pneumoniae at diferent times of growth\nDATA_FILES\nMetabolites all experiments, relative measurements\nAbsolute copy number per locus tag, M. pneumoniae at different times of growth\nINSTITUTIONS\nCRG Centre for Genomic Regulation\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nPUBLICATIONS\nDissecting the energy metabolism in Mycoplasma pneumoniae through genome-scale metabolic modeling.\nLarge-scale metabolome analysis and quantitative integration with genomics and proteomics data in Mycoplasma pneumoniae.\nSOPS\nMetabolomics perturbation samples preparation\nSTUDIES\nTranscriptomics of M. pneumoniae at different times of growth\n\n____________________________________________________________________________\n\nOxidative Stickland reactions in an obligate aerobic organism - amino acid catabolism in the Crenarchaeon\n              Sulfolobus solfataricus(id: 339 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nAmino acid degradation in Sulfolobus solfataricus P2\nPEOPLE\nDietmar Schomburg\nBettina Siebers\nPhil Wright\nTrong Khoa Pham\nHelge Stark\nJacqueline Wolf\nAndreas Albersmeier\nPROJECTS\nSulfoSys - Biotec\nSTUDIES\nComparison of Sulfolobus solfataricus P2 grown on caseinhydrolysate and D-glucose\n\n____________________________________________________________________________\n\nTotal DNA Quantification(id: 141 | type: sops)\n\nDescription: missing\n\n\nCREATORS\nLouise Thomas\nMaggie Smith\nPEOPLE\nMaggie Smith\nJay Moore\nLouise Thomas\nPROJECTS\nSTREAM\nSUBMITTER\nJay Moore\n\n____________________________________________________________________________\n\nCell size and morphological properties of yeast Saccharomyces cerevisiae in relation to growth temperature(id: 373 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nCellular size and granularity during glucose pulse\nDATA_FILES\nCellular size and granularity during glucose pulse\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nPEOPLE\nMaksim Zakhartsev\nMatthias Reuss\nPROJECTS\nMOSES\nSTUDIES\nSteady state concentrations of metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\n\n____________________________________________________________________________\n\nModified Locke Arabadopsis 3 loop Circadian Clock - PLM_66, version 1, SIMPLIFIED(id: 483 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_66, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nModified Locke Arabadopsis 3 loop Circadian Clock - PLM_66, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nAdams, Richard\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nModified Locke Arabadopsis 3 loop Circadian Clock - PLM_66\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nM9_shake_flask_exp_trans_stat_transcriptome_nimblegen(id: 95 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nTiling Array analysis of glucose strarved B. subtilis cells\nCREATORS\nPraveen Kumar Sappa\nINVESTIGATIONS\nRedefining the Complete Transcriptome of Bacillus subtilis using Tiling Array Technology.\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSTUDIES\nTranscriptome analysis of glucose starvation in B. subtilis\nSUBMITTER\nPraveen Kumar Sappa\n\n____________________________________________________________________________\n\nNational Institute on Aging(id: 268 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDeborah Croteau\nVilhelm Bohr\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nKinase activity of ArcB from Escherichia coli is subject to regulation by both ubiquinone and demethylmenaquinone(id: 217 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nKatja Bettenbrock\nPROJECTS\nSUMO\n\n____________________________________________________________________________\n\nLive Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy.(id: 157 | type: assays)\n\nDescription: 1. Preparation of B. subtilis cultures\r\n\r\n    Inoculate cells from -80C stocks in 10 ml time-lapse microscopy (TLM) medium (62 mM K2HPO4 , 44mM KH2PO4, 15 mM (NH4)2SO4, 6.5 mM sodium citrate, 0.8 mM MgSO4, 0.02 % casamino acids, 27.8 mM glucose, 0.1 mM L-tryptophan, the pH was set to 7 using a KOH solution) supplemented with antibiotics, if necessary.\r\n    Grow the cells overnight in a shake flask (30C, 225 rpm).\r\n    The following morning, dilute the cells 1:10 in pre-warmed chemically defined medium (CDM) (62 mM K2HPO4, 44mM KH2PO4, 15 mM (NH4)2SO4, 6.5 mM sodium citrate, 0.8 mM MgSO4, 2.2mM glucose, 2.1 mM L-glutamic acid, 6 M L-tryptophan, 7.5 M MnCl2, 0.15 x metal mix (prepare 50x MT mix stock (ref 8) which contains: 0.2 M MgCl2, 70 mM CaCl2, 5 mM MnCI2, 0.1 mM ZnCl2, 0.2 mM thiamin-hydrochloride, 2 mM HCl, 0.5 mM FeCl3 (add last)) without antibiotics.\r\n    Grow the B. subtilis cells to mid-exponential phase (30C, 225 rpm). Usually, this takes approximately four hours. Importantly, prepare the agarose slide one hour before the cells reach mid-exponential phase (see section 2).\r\n    Measure the absorbance of the culture at 600nm (A600) and dilute the cells to an approximate A600 of 0.035 using CDM. This OD ensures that single cells with appropriate spacing are spotted on the microscope slide for time-lapse microscopy.\r\n\r\n2. Preparation of the microscope sample (also see Figure 2)\r\n\r\nOne hour before cells reach mid-exponential growth, prepare the microscope slide as follows:\r\n\r\n    Clean two microscope glass slides (e.g. Knittel Glass, 7.6 x 2.6 cm,) with 70% ethanol and water.\r\n    Take a gene frame (ABgene; 1.7 x 2.8 cm) and carefully remove one of the plastic foils from the gene frame without causing disassembly of the plastic cover on the other side of the gene frame.\r\n    Attach the gene frame in the middle of one of the glass slides by first facilitating contact on just one side, followed by guided attachment of the remaining gene frame with a fingernail. Prevent air bubbles while attaching the gene frame to the glass slide.\r\n    Use a microwave to dissolve 150 mg (1.5%) high-resolution low-melting agarose (Sigma) in 10 ml CDM. The agarose needs to be fully dissolved to obtain minimal background required for the time-lapse microscopy experiments. If required, supplement the agarose-CDM with inducer or other compounds at this time\r\n    Transfer 500 l of the warm agarose-CDM in the middle of the gene frame. Make sure the whole area including (the borders) is fully covered.\r\n    The following steps (2.6 - 2.10) have to be carried out quickly to prevent excessive drying of the agarose-CDM.\r\n    Place the second glass slide on the agarose-CDM filled gene frame. Try to avoid air bubbles. Place the sandwiched slides in a horizontal position for 45 min at 4C in the refrigerator to allow the agarose-CDM to solidify sufficiently.\r\n    Carefully slide off the upper glass slide. Use a razor blade to cut out agar strips of ~5 mm width within the gene frame, on which the cells will be grown. A maximum of three strips can be used per slide, separated by ~4 mm space to either side. These spaces will provide air which is essential for B. subtilis growth. If four different strains need to be followed in time, two strips can be made and be cut in half to result in four small squares. Remove any residual solid medium.\r\n    Carefully remove the second and final plastic cover from the gene frame to expose the sticky side of the gene frame\r\n    Load single cells (from step 1.5) on the solid medium without touching it with the pipet tip. Use 2.5 l for a whole strip, or 1 l for a small square. Always start on top of the agarose pad and allow the liquid to disperse equally on its assigned growth area by turning the slide up and down. The slide is ready, as soon as the edges of the liquid become corrugated and movement of the liquid is no longer visible when turning the slide.\r\n    Place a clean microscope slide cover slip (24 x 50 mm) on the gene frame from one side to the other (avoid air bubbles). Assure complete attachment by applying pressure on the cover slip along the gene frame with your fingernail. If the cover slip is placed on the cells without allowing them to dry long enough, cells tend to grow on top of each other during the experiment. Also be careful not to wait too long before applying the cover slip, since the agarose will then be too dry.\r\n    Pre-warm the slide for 1 h at 30C. If the slide would directly be placed in the pre-warmed environmental chamber (see step 3.1) of the microscope, the temperature fluctuations might cause autofocus problems in the first hours of the experiment.\r\n\r\n3. Time-lapse fluorescence microscopy (also see Figure 3 and Movie 1)\r\n\r\n    Pre-warm the environmental chamber on time (in our hands at least 2h before the start of the experiment) in order to prevent autofocus problems after starting the experiment. The time required depends on the environmental chamber used as well as the heating system and the microscope.\r\n    Select the appropriate objective, filters and dichroic mirror according to your experimental set-up. For long experiments make sure that a UV-filter is placed between the light source and the sample. Also, if possible, block some of the excitation light using neutral density filters to minimize exposure.\r\n\r\nThe following equipment (provided by DeltaVision, UK) was used for the time-lapse microscopy experiments published in de Jong et al. 2010 5: IX71 Microscope (Olympus), CoolSNAP HQ2 camera (Princeton Instruments), 300W Xenon Light Source, 60x bright field objective (1.25 NA), GFP filterset (Chroma, excitation at 470/40 nm, emission 525/50 nm), mCherry filterset (Chroma, excitation at 572/35 nm, emission 632/60 nm). Autofocus was performed using diascopic light and using the autofocus routine present in Deltavision's Softworx software. It should be noted that there are now a number of other autofocus systems that are also suitable such as the Zeiss Definite Focus, the Nikon Perfect Focus System and the Leica Adaptive Focus Control.\r\n\r\n    Program your experiment according to your experimental set-up. It is wise to determine the amount of light required for specific constructs, as well as the autofocus settings for other time-lapse microscopes or bacteria prior to the actual experiment. Shorter exposure times and less amount of excitation light will minimize bleaching and phototoxicity. Use diascopic light for the autofocus routine.\r\n\r\nThe following settings were used for the time-lapse microscopy experiments published in de Jong et al. 2010 5: snapshots for movies were taken at intervals of 8 or 12 minutes using 10 % APLLC White LED light and 0.05 s exposure for bright field pictures, 10% Xenon light and 0.5 s exposure for GFP detection, and 32% Xenon light and 0.8 s exposure for mCherry detection, respectively. Raw data were stored using softWoRx 3.6.0 (Applied Presicion). The autofocus was programmed for 0.06 m steps and a total range of 1.2 m.\r\n\r\n    Place the prepared slide (section 2) in the pre-warmed environmental chamber of the microscope and monitor the outgrowth of single cells into a microcolony monolayer at 30C.\r\n\r\nSpecific tips:\r\n\r\n    Select single cells that are located in the middle of the agar pad. The edges of the agar pad dry out more easily. Store the X,Y,Z position using the microscope's software.\r\n    Large movements of the stage in the X,Y and Z direction might perturb the agarose and consequently hamper identification of cells by the autofocus routine. In general, to minimize X,Y,Z movement, we do not select more than 10 positions per experiment even if a slide contains multiple strains.\r\n    After having selected the first cell, only adjust the Z-focus with the software. From this point, do not alter the focus manually on the microscope body using the \"Z-knob\" unless this is digitally encoded. Ensure that after each autofocus routine the new X,Y,Z position is stored by the software.\r\n    Check whether the autofocus settings are appropriate for the experiment, before starting the run. The use of phase-contrast microscopy might improve the autofocus routine compared to using brightfield or DIC microscopy, due to enhanced contrast. However, the phase ring in phase-contrast objectives makes them less sensitive (approximately 10%) in collecting fluorescence light. So for weak fluorescent samples, an objective without phase ring is better suited.\r\n    Check whether the selected cells are still in focus every half hour, until the experiment is running stably. When cells at this point are out of focus, adjust manually. Due to temperature variations, as well as poorly dried samples, this may be necessary during the first hours. Furthermore, due to enhanced contrast, the autofocus works better when more cells are in the field of view.\r\n\r\n    After the experiment has finished, separate the different channels of the movie and safe them as separate files (i.e. phase contrast, GFP, mCherry) if required (certain acquisition packages will place all channels in a single stacked file). For publication, pictures can be enhanced by 2D deconvolution, which is especially useful for protein localization studies. Deconvolve the images using your microscope's software or with a commercial package such as Huygens (www.svi.nl).\r\n    Analyze the data using ImageJ (http://rsbweb.nih.gov/ij/) (use the raw, unprocessed image files for this) and Microsoft Excel or Sigma Plot. Stacks can for example be saved as an \".avi\" movie file in ImageJ. A detailed description of how fluorescence of single cells can be measured in time is given below.\r\n\r\n4. Data analysis of promoter activity dynamics using ImageJ\r\n\r\nWe note that other good software packages are available which are specialized in analyzing time-lapse microscopy images such as BHV software 9, 4, Schnitzcell 10, PSICIC 11, and Microbe-Tracker 12, but here we focus on the freely available ImageJ package.\r\n\r\n    Download ImageJ (http://rsbweb.nih.gov/ij/) and (if necessary) the correct plugin to open your (stacked) file. For instance, movies recorded using a deltavision microscope can only be opened in ImageJ with the deltavision opener plugin. Copy the DV-plugin in the ImageJ plugin folder and start the program. Change the memory capacity in edit/option/memory&threads to 1250. This allows working with bigger files such as those obtained from time-lapse movies.\r\n    To assess the cell history of one single cell, open the original phase contrast movie of one microcolony and scroll to the last frame of interest in the movie. Select the \"segmented lines\" selection button in the menu.\r\n    Draw a line in the background and press \"CTRL\" + \"T\". This will open the regions of interest (ROI) manager. (The corresponding background fluorescence value can be used to manually substract the background - see below. Alternatively, use the background subtraction routine present within ImageJ.) Also draw a line in the cell of interest and add the ROI to the ROI manager. Since we are investigating promoter-GFP fusions in this case study and the GFP is diffused throughout the cytoplasm, the whole cell should have similar fluorescence values for each single pixel across the length of the cell. Scroll one frame back in time and select a new ROI in the same cell of interest. Save this third ROI and continue with the procedure until the corresponding ROI in the first frame of the movie has been saved.\r\n    Be aware that the fluorescence of daughter cells can vary greatly after cell division. Either use a membrane dye which can be applied together with fluorescence proteins produced by the cells (such as the combination of the red membrane dye FM 5-95 (Invitrogen) and GFP) to visualize septum formation. In this case the corresponding fluorescence channel and not the phase contrast movie should be used to follow cells in time. Alternatively, one can stay on the safe side by selecting ROI's in only one half of a cell.\r\n    Within the ROI manager, click \"save\". If the ending of the file is \".roi\", then one roi is selected in the list, and only this one will be saved. If the ending of the file is \".zip\", then the whole set will be saved (required).\r\n    Close the phase contrast movie and open the original fluorescence movie (e.g. GFP). Click \"show all\" and \"measure\" in the ROI manager. A new window will open (Results). Copy the results into an Excel sheet and subtract the mean fluorescence for each single cell from the background fluorescence of the medium. The resulting net fluorescence can be plotted against the time to reveal the promoter activity of the cell of interest in time.\r\n    Alternatively, the fluorescence of all cells of a microcolony can be analyzed at a specific time-point. In order to do this select and save ROI's for the background and every single cell in one frame as described above, copy the fluorescence values to Excel and produce histograms using the \"histogram\" function in the \"tool\" menu.\r\n    To obtain pictures from single frames for publication, select the frame of interest, select \"image\" - \"duplicate image\" and change the type of image to \"RGB\" or \"8-bit\" by selecting \"image\" - \"type\" - \"RGB color\" or \"8-bit\". Save the duplicated frame as \".tiff\". The RGB/8-bit pictures can be opened by conventional drawing programs such as CorelDraw or Adobe Illustrator. If required, the pictures may be adapted using \"image\" - \"adjust\" - \"brightness/contrast\" or \"image\" - \"adjust\" - \"window/level\" in ImageJ.\r\n\r\n5. Producing movies for publication with ImageJ\r\n\r\n    Open the original phase contrast and the corresponding fluorescence movie(s) in ImageJ as described above. Select the rectangular selections button (1st on the left) and draw a rectangle ROI in the first frame in such a way, that the developing microcolony is enclosed by the ROI throughout the whole movie. Select \"image\" - \"crop\" and safe this smaller version of the movie under a new name. Select the same ROI via the ROI manager in the fluorescence movie and proceed as before.\r\n    To combine the movies either horizontally or vertically, select \"plugin\" - \"stack combiner\". If desired, a time stamper can be added via \"plugin\" - \"time stamper\". Save the combined stack once as \".dv\" or \".tiff\" and once as \".avi\" or quicktime movie.\r\n\r\nAlternative Protocol Adaptations for Streptococcus pneumoniae (Figure 4 and Movie 2):\r\n\r\n6. Preparation of S. pneumoniae cultures\r\n\r\n    Grow S. pneumoniae cells (capsulated strain D39 13, or unencapsulated strain R6 14 as standing cultures in C+Y medium 15 at 37C until an OD at A600nm of approximately 0.4 is reached. Centrifuge the cells for 2 min at 14000 rpm and resuspend the cell pellet in a volume of fresh C+Y medium containing 14.5% glycerol (v/v) that would result in an A600nm of exactly 0.4. Aliquot the cells and store them at -80C for future use.\r\n    For time-lapse microscopy, take an aliquot of previously cultured S. pneumoniae cells. Inoculate 4 ml of fresh C+Y medium 1:100 with cells from the -80C aliquot. Grow the cells until mid-exponential phase to an OD of A600nm of 0.1 - 0.2. Usually, this takes about 2 hours when using cells from -80C aliquots.\r\n\r\n7. Preparation of the microscope sample\r\n\r\n    Prepare a microscope slide as described above for B. subtilis but make sure that the agarose contains complex C+Y medium. Since S. pneumoniae is a microaerophile, the air pockets between agarose strips should be smaller than for B. subtilis (~1 mm space on both sides).\r\n    Measure the absorbance of the culture at 600 nm (A600), dilute the exponentially growing S. pneumoniae cells to an approximate A600 of 0.05 using C+Y medium and use this dilution to load the agarose slide.\r\n\r\n8. Time-lapse phase-contrast microscopy\r\n\r\nAdjust the microscope settings for S. pneumoniae: use phase-contrast microscopy since S. pneumoniae is difficult to identify using bright-field microscopy. Continue the protocol as described for B. subtilis (follow steps 2.9 - 3.7). S. pneumoniae cells can be grown at either 30C or 37C (they grow faster at 37C).\r\n\r\n9. Representative results:\r\n\r\nThe time-lapse fluorescence experiment has been carried out successfully, if the bacteria grew into a microcolony monolayer, which is completely located within the field of view at the end of the experiment (see Figure 5A-C). If cells grew on top of each other, it is not only impossible to trace back their history accurately, but also the fluorescence levels of overlapping cells can not be measured correctly. Cells tend to grow on top of each other, if the spotted cells were not dried sufficiently (step 2.9) or if the medium composition needs to be adjusted to obtain slower growth. If a microcolony grew out of view, then the distribution of fluorescence signals within one colony can not be determined. Causes for \"microcolony movement\" can be insufficient drying of spotted cells (step 2.9), or if the software was not programmed to track the microcolony during development. Furthermore, it is important that local patches of increased fluorescence are not detectable in the medium as this obscures the fluorescence signals coming from the cells (see Figure 5D-F). Background related problems can arise from media compounds, airbubbles or undissolved agarose clumps. To visualize this, we show in Fig. 5F background signals of this specific slide when the image was taken using excitation/emission filters for red fluorescent dyes. As seen, bright autofluorescent spots are present which could hamper imaging. To prevent such spots, make sure the agarose is completely dissolved and there are no airbubbles when placing the coverslip on the microscope slide.\r\n\r\nDiscussion:\r\nIn contrast to many other single cell techniques, the time-lapse fluorescence microscopy method described here can be used to follow the history of a specific cell with regard to its ancestors, its behaviour, and division events. In combination with fluorescently labelled target promoters or proteins, specific developmental pathway activation can be followed in time and protein localization as well as protein dynamics can be monitored during bacterial development.\r\n\r\nAs indicated above, studies concentrating on different bacterial species can be performed by adapting the growth conditions according to the requirements for a specific bacterium. The only limitations we encountered are related to the growth conditions and sample size. Due to a sealed environment, medium conditions cannot be changed during the experiment. Furthermore, a maximum of four strains per experiment can be monitored efficiently.\r\n\r\nConsidering a few critical steps, the single cell analysis method described here can easily be applied using any automated microscope. In the following, an overview of these critical steps will be given. Detailed information can be found in the main text. General preparation: It is wise to check the autofocus settings required for a specific bacterium prior to the experiment. Likewise, approximate optimal settings for the visualization of fluorescence should be determined in advance, if possible. Furthermore, following a prepared time-line helps to have all material ready to use in time (pre-warming the microscope chamber, programming the microscope settings, preparing the slide one hour before the cells are in the desired growth phase, see Figure 1). Growth of B. subtilis in TLM and CDM: TLM and CDM are chemically defined starvation media in which B. subtilis only grows slowly. The time-period in which the cells are grown in the media might have to be prolonged depending on the specific strain. The slow growth prevents cells from piling up on each other. Preparation of the microscope sample: Air bubbles between the gene frame, the glass slide and the cover slip have to be prevented to prevent extensive drying of the agarose-based medium. The same holds for the medium/cover slip interface. It is crucial to let the cells dry enough, to prevent swimming and/or multiple layer growth. Time-lapse fluorescence microscopy: Pre-warming of the slide as well as the environmental chamber is crucial to prevent major autofocus problems. Cells should be selected in the middle of an agar pad, since these have the highest chance to stay in field and focus during the experiment (provided the sample was dried well enough). A maximum of 10 locations per experiments still works properly. After having selected the first cell of interest only use the software to adjust the focus (see text for details). Check whether the cells are still in focus during the first hours of the experiment in 30 min intervals. Analysis: It is important to check prior to extended analysis procedures whether the background of the medium has similar values in the fluorescence channels. Small dust particles, medium components, dirty lenses or tiny agarose clumps can contribute to locally increased fluorescence, making the movie difficult or impossible to analyse. Trouble shooting: If the cells grow on top of each other, this might either indicate that the coverslip was attached too soon or that the medium is not suited for growth of microcolony monolayers. If cells of interest continuously die prematurely, whereas other cells on the slide divide happily, you might want to check whether you put the UV-filter in position. It might also help to decrease the exposure time or light intensity during long experiments.\r\n\r\n\n\n\nINVESTIGATION\nWetlab approach to transcription fidelity\nORGANISMS\nStreptococcus pneumoniae\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\nPUBLICATIONS\nSMC is recruited to oriC by ParB and promotes chromosome segregation in Streptococcus pneumoniae\nLive Cell Imaging of <em>Bacillus subtilis</em> and <em>Streptococcus pneumoniae</em> using Automated Time-lapse Microscopy\nControl of cell division in Streptococcus pneumoniae by the conserved Ser/Thr protein kinase StkP\nSingle cell analysis of gene expression patterns during carbon starvation in Bacillus subtilis reveals large phenotypic variation\nSTUDY\nAutomated time-lapse microscopy\nSUBMITTER\nJan-Willem Veening\n\n____________________________________________________________________________\n\nGeneration of cell material from Streptomyces coelicolor culture for metabolome analysis(id: 136 | type: sops)\n\nDescription: missing\n\n\nASSAYS\nOnline/offline measurements\nCREATORS\nPer Bruheim\nTrond Ellingsen\nSunniva Hoel\nyvind Jakobsen\nHvard Sletta\nAlexander Wentzel\nAnders verby\nINVESTIGATIONS\nMetabolism of Streptomyces coelicolor (SysMO STREAM)\nPEOPLE\nTrond Ellingsen\nyvind Jakobsen\nJay Moore\nPer Bruheim\nHvard Sletta\nAnders verby\nSunniva Hoel\nAlexander Wentzel\nPROJECTS\nSTREAM\nSTUDIES\nTimeseries 1\nSUBMITTER\nJay Moore\n\n____________________________________________________________________________\n\nAnalysis of Escherichia coli mutants with a linear respiratory chain(id: 216 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nDetermination of by-product formation and glucose uptake of mutants with linear respiratory chain at different aerobiosis levels\nDeternination of ArcA phosphroylation level in mutants with linear ETC at different defined aerobiosis levels\nDATA_FILES\nETCmutants_metabolites\nETC_mutants_ArcA\nPEOPLE\nKatja Bettenbrock\nStefan Stagge\nSonja Steinsiek\nPROJECTS\nSUMO\n\n____________________________________________________________________________\n\nAlkali-metal-cation influx and efflux systems in nonconventional yeast species(id: 118 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nHana Sychrova\nJoaquin Ario\nJose Ramos\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nGene knock-ins at the LACZ/Y locus by recombineering in E.coli(id: 19 | type: sops)\n\nDescription: Generating gene knock-ins using MJF618 expressing the defective lambdoid prophage recombination system  red.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nAnalysis of organic acids in the supernatant(id: 93 | type: sops)\n\nDescription: Method for analysis of various organic acids in the medium\n\n\nCREATORS\nMartijn Bekker\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\n\n____________________________________________________________________________\n\nHeiko Lemcke(id: 1289 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRostock University Medical Centre\nPROJECTS\niRhythmics\n\n____________________________________________________________________________\n\nBiophysical analysis of putative interactions between TCA cycle enzymes of B. subtilis(id: 7 | type: presentations)\n\nDescription: Presentation given by Colin Harwood presenting results about ITC experiments to confirm interactions. Preliminary crystallisation experiments commenced on suitable candidates.\n\n\nCREATORS\nColin Harwood\nEVENTS\nBacell-meeting 1109-Groningen\nPEOPLE\nUlf Liebal\nColin Harwood\nPROJECTS\nBaCell-SysMO\nSUBMITTER\nUlf Liebal\n\n____________________________________________________________________________\n\nTest created 1552503965203(id: 959 | type: assays)\n\nDescription: <br><br><strong>Related Publications</strong></strong><br>Me and Helena (2014). some book. <br>Moore A, Zielinski T, Millar AJ (2014). Online period estimation and determination of rhythmicity in circadian data, using the BioDare data infrastructure.. Methods in molecular biology (Clifton, N.J.). Retrieved from: doi.org/10.1007/978-1-4939-0700-7_2<br>Stein JM (1975). The effect of adrenaline and of alpha- and beta-adrenergic blocking agents on ATP concentration and on incorporation of 32Pi into ATP in rat fat cells.. Biochemical pharmacology. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/12<br><br><strong>Comments</strong></strong><p>\n\tNew comment after 2nd version</p>\n 2019-01-10 11:59:51 201 zielu tomasz.zielinski@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\r\n\tCommentsdfa</p>\r\n<br><br><strong>Originally submitted to PLaSMo on 2014-07-25 18:01:50</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nTomasz Zielinski\nINVESTIGATION\nZielinski, Tomasz\nPEOPLE\nTomasz Zielinski\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\namino acid auxotrophies S. pyogenes(id: 1049 | type: data_files)\n\nDescription: raw data OD600 of amino acid auxotrophy exps S. pyogenes\n\n\nCREATORS\nKoen Van Grinsven\nPEOPLE\nKoen Van Grinsven\nPROJECTS\nSysMO-LAB\nSUBMITTER\nKoen Van Grinsven\n\n____________________________________________________________________________\n\nNeurospora Circadian Clock 3-variable model - sinusoidal light oscillations - PLM_52, version 1, SIMPLIFIED(id: 488 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_52, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nNeurospora Circadian Clock 3-variable model - sinusoidal light oscillations - PLM_52, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nAdams, Richard\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nNeurospora Circadian Clock 3-variable model - sinusoidal light oscillations - PLM_52\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nIngemar Nrdal(id: 566 | type: people)\n\nDescription: Research Scientist in the Biotechnology and Nanomedicine department at the non-profit research institution, SINTEF, in Norway.  \n\n\nINSTITUTIONS\nSINTEF\nPROJECTS\nMetApp\n\n____________________________________________________________________________\n\nEszter Lakatos(id: 532 | type: people)\n\nDescription: I am a PhD student in the Theoretical Systems Biology group, based at Imperial College London.\r\nThe aim of my PhD is to understand how noise can be the driving force of decision-making processes (differentiation, self-renewal, apoptosis or tumorgenesis), and what are our chances to control them. So far I have been working on method development for stochastic models, a moment closure framework and a stochastic reachability method, to look into cell-to-cell-variability.\n\n\nINSTITUTIONS\nImperial College London\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nControl of transcription elongation by GreA determines rate of gene expression in Streptococcus pneumoniae(id: 227 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nRNA-Seq\nDATA_FILES\nRNA-seq wild type\nRNA-seq delta greA\nPEOPLE\nOscar Kuipers\nJan-Willem Veening\nNikolay Zenkin\nPamela Gamba\nYulia Yuzenkova\nSulman Shafeeq\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nJoaquin Dopazo(id: 752 | type: people)\n\nDescription: My scientific  interests revolve around functional genomics, systems biology and the development of algorithms and software for the analysis of high-throughput data (mainly, but not restricted to, Next Generation Sequencing) and its application to the relationship between genotype and phenotype, mainly oriented to personalized and precision medicine. I am especially interested in the study of disease mechanisms and drug action mechanisms, drug repositioning and the definition of mechanism-based biomarkers by modeling complex cellular systems in which signaling, regulation and metabolism information are integrated (see an example with http://hipathia.babelomics.org/).\r\nThe scope of my research has evolved in parallel to the penetration of the Big Data concept in the life sciences. I have been working on genomic data integration and, specifically, during the last years, focusing on massive sequence data analysis. As an example of the impact of the developments of Dopazos group, the software GenomeMaps (www.genomemaps.org) has been chosen as the genome viewer of the International Cancer Genome Consortium data analysis portal (dcc.icgc.org). \r\nI have promoted and coordinated genomic projects such as the FutureClinic (www.futureclinic.es), to prepare the scenario for the introduction of the genome in the electronic health record, or the Medical Genome Project (www.medicalgenomeproject.com), to sequence 1000 patients of inherited diseases to search for new biomarkers and disease genes. I was also involved in international projects such as the MAQC and SEQC (best practices in the use of microarrays and NGS, respectively, for finding diagnostic biomarkers) or the START consortium to characterize the variability of the rat genome. I also coordinate the HPC4Genomics (http://www.hpc4g.org) consortium, a joint initiative of the CIPF, several universities and companies that aim to combine genomic, bioinformatic and computing skills to address the new challenges posed by the technological advancement in genomics.\r\n\n\n\nINSTITUTIONS\nCentro de Investigacin Prncipe Felipe\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nDetermination of (cytosolic) enzyme activities(id: 98 | type: sops)\n\nDescription: Sample preparation - SOP for sampling, preparation of cell-free extracts, and determination of total extracted protein\n\n\nCREATORS\nFemke Mensonides\nPEOPLE\nFemke Mensonides\nMaksim Zakhartsev\nPROJECTS\nMOSES\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nMathematical modelling of the agr operon in Staphylococcus aureus(id: 35 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMathematical modelling of the agr operon in Staphylococcus aureus\nPEOPLE\nJohn King\nSara Jabbari\nPROJECTS\nCOSMIC\n\n____________________________________________________________________________\n\nCellular localization of choline-utilization proteins in Streptococcus pneumoniae using novel fluorescent reporter systems(id: 106 | type: publications)\n\nDescription: missing\n\n\nDATA_FILES\nRNA-seq wild type\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\r\n\r\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \r\n\r\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nGenomic Data Submission Excel Template (NimbleGen)(id: 934 | type: data_files)\n\nDescription: This template is for recording genome data from the NimbleGen platform.\r\nThis template was taken from the GEO website (http://www.ncbi.nlm.nih.gov/geo/info/spreadsheet.html) and modified to conform to the SysMO-JERM (Just enough Results Model) for transcriptomics.\r\nUsing these templates will mean easier submission to GEO/ArrayExpress and greater consistency of data in SEEK.\n\n\nASSAYS\nNimbleGen Transcriptomics Templates\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nGlobal transcript levels respond to small changes of the carbon status during progressive exhaustion of carbohydrates in Arabidopsis rosettes.(id: 352 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seaton\nPROJECTS\nMillar group\n\n____________________________________________________________________________\n\nCell volume_trk_(id: 180 | type: data_files)\n\nDescription: Cell volume_trk_\n\n\nCREATORS\nSilvia Petrezselyova\nPEOPLE\nSilvia Petrezselyova\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nUniversity of Copenhagen(id: 266 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nLene Juel Rasmussen\nAna Serrano Puebla\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nFlash-labeling of tetra-cyc-modified proteins in E.coli(id: 16 | type: sops)\n\nDescription: Experimental system that is designed to observe the in vivo stability and aggregation of a protein of interest.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nDynamics of extracellular metabolites during glucose pulse(id: 82 | type: data_files)\n\nDescription: Dynamics of extracellular metabolites (glc, pyr, suc, lac, gly, ac, etoh, fum, mal, cit, including loss of akg, g3p, 2pg, 3pg, r5p, f6p, g6p, 6pg) during glucose pulse. Glucose pulse was performed in anaerobically growing yeast Saccharomyces cerevisiae in steady state chemostat (D = 0.1 h-1) and transent concentrations of the extra- and intracellular metabolites from central carbon metabolism (e.g. glycolysis, PPP, glycerol, purines, etc) were measured.\r\n\r\n\r\n\n\n\nASSAYS\nDynamics of extracellular metabolites during glucose pulse\nMOSES: dynamic model of glucose pulse\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nMaximal specific growth rates of the three lactic acid bacteria and their ldh deletions.(id: 598 | type: data_files)\n\nDescription: .  L. lactis (NZ9000), E. faecalis (V538) and S. pyogenes (M49) wild type strain and their ldh- mutants were grown in batch cultures at 37C in anaerobic 96 wells plates in either TH-broth supplemented with 0.5% (w/v) yeast (THY) or a chemically defined medium for LAB (pH 7.4) (CDM-LAB (10)). Both media were buffered with either 100 mM MES buffer or 100 mM MOPS buffer for growth at pH 6.5 and 7.5 respectively. \n\n\nASSAYS\nMaximal specific growth rates of the three lactic acid bacteria and their ldh deletions\nCREATORS\nMartijn Bekker\nTomas Fiedler\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nMartijn Bekker\nTomas Fiedler\nPROJECTS\nSysMO-LAB\nPUBLICATIONS\nPhysiological characterization of three lactic acid bacteria and their isogenic ldh deletion mutants shows that these organisms are optimized for Yatp and not ATP formation rates at their physiological pH\nSTUDIES\nStudy of the physiological characterization of three lactic acid bacteria and their isogenic ldh deletion mutants\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nDynamics of receptor and protein transducer homodimerisation(id: 91 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlaf Wolkenhauer\nThomas Millat\nUlf Liebal\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nMOA2010-05_3C_sRNAomics_bioinformatics_sRNA-targets-in-silico(id: 2788 | type: data_files)\n\nDescription: Predicted targets of endogenous small RNAs (sRNAs) by in silico approach. For each predicted interaction, miRNAs/phasiRNA ID, the target transcript identifiers, representative gene identifier, short gene name, full descriptions, MapMan ontology annotations (GoMapMan; (Ramak et al., 2014)) and predicted target regulation (cleavage or translational repression) are shown. Short names for potato genes were inferred from Arabidopsis thaliana orthologs where applicable, else the StNIB_v1 gene identifier was set (Ramak et al., 2014).  For each sRNA-target interaction, alignment score value (based on the scoring schema: G: U wobble pairings - each 0.5 scores, insertions/ deletions (indels  2.0) and all other non-canonical Watson-Crick pairings  1.0.), UPE (unpaired energy required to open secondary structure around sRNAs target site on the mRNA), the position (miRNA_start, miRNA_end, target_start, target_end) and the aligned sequence (miRNA_aligned_fragment and target_aligned_fragment) have been obtained. Moreover, the information whether the interaction was also confirmed by degradome-sequencing was added (Y  YES; N  NO). Additionally, for each sRNA the log2FC and p-values among different comparisons are given and the log2FC ratios of target transcripts between PVYNTN and mock inoculated Dsire or NahG-Dsire plants; yellow  upregulated; blue  downregulated;  statistically not significant (FDR p > 0.05). NA  not analyzed or not available. D  Dsire; NahG  NahG-Dsire; M  mock; P  PVYNTN.\n\n\nASSAYS\nsRNAomics (bioinformatics)\nINVESTIGATIONS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nPEOPLE\nZiva Ramsak\nPROJECTS\nMOA - Multiomics analysis of potato response to Potato virus Y (PVY) infection\nSTUDIES\nMOA2010-05\nSUBMITTER\nZiva Ramsak\n\n____________________________________________________________________________\n\nBenchmarking various green fluorescent protein variants in Bacillus subtilis, Streptococcus pneumoniae, and Lactococcus lactis for live cell imaging(id: 219 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nThe circadian clock regulates the photoperiodic response of hypocotyl elongation through a coincidence mechanism in Arabidopsis thaliana(id: 307 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seaton\nPROJECTS\nMillar group\n\n____________________________________________________________________________\n\nXyloCut(id: 30 | type: projects)\n\nDescription: D-xylose is a major component of lignocellulose and is after D-glucose the most abundant monosaccharide on earth. However, D-xylose cannot be naturally utilised by several industrially relevant microorganisms. On the way to a strong bio-based economy in Europe, this widely available feedstock has to be made accessible for the sustainable microbial synthesis of value-added chemical building blocks to be used in a broad range of applications. The project aims at engineering Corynebacterium glutamicum and Saccharomyces cerevisiae to serve as flexible and carbon efficient microbial cell factories converting D-xylose derived from lignocellulosic material (LCM) into value added products covering diols, lactols and organic acids. Combination of genetic engineering and systems biology are used to demonstrate the potential of a rational system-level approach for metabolic engineering of C. glutamicum and S. cerevisiae. In particular, new fundamental knowledge is generated regarding the impact of industrial hydrolysates on the intracellular dynamics of proteins, metabolites and fluxes as well as global stress responses and production capabilities of the two platform organisms. Subsequently, model-driven optimization approaches are expected to show the feasibility of growth-decoupled production with the newly constructed producer strains on industrial hydrolysates. The project has a high potential for innovative industrial applications, and it is expected to contribute to improve the competitiveness of the European biotechnological industry.\n\n\nINSTITUTIONS\nForschungszentrum Jlich\nLund University\nDelft University of Technology (DUT)\nORGANISMS\nSaccharomyces cerevisiae\nCorynebacterium glutamicum\nPEOPLE\nSebastian Aljoscha Wahl\nStephan Noack\nGunnar Liden\nLisa Wasserstrom\nNiklas Tenhaef\nChristian Brsseler\nCelina Tufvegren\nHenrik Almqvist\nJinrui Zhang\nJan Marienhagen\nMarie-Francoise Gorwa-Grauslund\nPROGRAMMES\nERASysAPP\nPUBLICATIONS\nFormation of xylitol and xylitol-5-phosphate and its impact on growth of d-xylose-utilizing Corynebacterium glutamicum strains\nEngineering of Corynebacterium glutamicum for minimized carbon loss during utilization of D-xylose containing substrates\nMiniaturized and automated adaptive laboratory evolution: Evolving Corynebacterium glutamicum towards an improved d-xylose utilization.\nCharacterization of the Weimberg Pathway in Caulobacter crescentus\n\n____________________________________________________________________________\n\nTrygve Brautaset(id: 470 | type: people)\n\nDescription: Since 1st of January 2015 I am Professor in synthetic biology at the Norwegian University of Science and Technology (NTNU) institute of biotechnology. Before that I was research director in the non-profit research institution SINTEF. My major research activities are within microbial molecular biology, mainly combining metabolic engineering, synthetic biology and systems biology to develop microbial cell factories, and focusing both on the products and on the raw materials. The research includes both basic and applied sciences.\n\n\nINSTITUTIONS\nNorwegian University of Science and Technology\nSINTEF\nPROJECTS\nMetApp\nLEANPROT\n\n____________________________________________________________________________\n\nHK Kinetic characterisation(id: 192 | type: sops)\n\nDescription: SOP for measurement of HK activity in extracts.\n\n\nASSAYS\nHK\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nCultivation and K+ shift of the E. coli strains MG1655 (wildtype) and MG1655 (kdpA4), a (KdpFABC-inactive mutant)(id: 25 | type: sops)\n\nDescription: To induce kdpFABC expression, cells are cultivated in K10 minimal medium (10mM K+) were shifted into K+ limiting growth medium (K0), containing 20 M K+ by filtration.\n\n\nCREATORS\nLisbeth Lyngberg\nPEOPLE\nLisbeth Lyngberg\nPROJECTS\nKOSMOBAC\nSUBMITTER\nLisbeth Lyngberg\n\n____________________________________________________________________________\n\nA proteomic and transcriptional view of acidogenic and solventogenic steady-state cells of Clostridium acetobutylicum in a chemostat culture(id: 93 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nComparison of the transcriptome between pH 5.7 (acidogenesis) and pH 4.5 (solventogenesis)\nComparison of the proteome between pH 5.7 (acidogenesis) and pH 4.5 (solventogenesis).\nStudy of the end products of the acidogenesis and solventogenesis pathways\nDATA_FILES\nEnd products of Clostridium acetobutylicum growing at steady-state pH values 5.7 and 4.5.\nEnd products of Clostridium acetobutylicum growing at different steady-state pH values from 5.5 to 4.7.\nTranscriptome data of Clostridium acetobutylicum growing at steady-state pH 5.7 and pH 4.5.\nTranscriptome data of Clostridium acetobutylicum growing at steady-state pH 5.7, 5.5, 5.3, 5.1, 4.9, 4.7, and pH 4.5.\nProteome Reference maps of Clostridium acetobuytlicum growing at steady-state pH 5.7 and pH 4.5\nProteome of Clostridium acetobutylicum growing at steady-state pH 5.5 to pH 4.7\nMODELS\nModel of the pH-induced metabolic shift in phosphate-limited continuous cultures of C. acetobutylicum assuming a phenotypic switch\nPEOPLE\nArmin Ehrenreich\nHubert Bahl\nRalf-Jrg Fischer\nChristina Dring\nHolger Janssen\nSara Jabbari\nPRESENTATIONS\nThe pH-induced metabolic shift from acidogenesis to solventogenesis in Clostridium acetobutylicum - From experiments to models\nMathematical Modelling of the pH-induced Metabolic Shift in C. aceobutylicum Unravels a Heterogeneous Phase Transition\nPROJECTS\nCOSMIC\n\n____________________________________________________________________________\n\nConstruct of Arginine kinase gene knockout T. b. brucei 427(id: 943 | type: data_files)\n\nDescription:  Arginine kinase has been thought of as a potential stress marker (see 'Metabolic changes by oxidative stress in T. b. brucei 427'), the gene knockout cells have been constructed.\n\n\nCREATORS\nDong-Hyun Kim\nEduard Kerkhoven\nPEOPLE\nDong-Hyun Kim\nEduard Kerkhoven\nPROJECTS\nSilicoTryp\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nTowards system understanding of B. subtilis - integrating fluxes and cellular components under salt stress(id: 4 | type: presentations)\n\nDescription: Michael Kohlstedt presented flux data as integrated output of cellular components. Moreover, proteomics and metabolomics data reveal significant changes in intracellular protein and metabolite level and osmoprotection (supplementing glycine betaine) creates a novel metabolic state.\n\n\nCREATORS\nMichael Kohlstedt\nEVENTS\nBacell-meeting 1109-Groningen\nPEOPLE\nUlf Liebal\nMichael Kohlstedt\nPROJECTS\nBaCell-SysMO\nSUBMITTER\nUlf Liebal\n\n____________________________________________________________________________\n\nKollevg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nCellular size and granularity during glucose pulse(id: 79 | type: data_files)\n\nDescription: Cellular size and granularity (measured by FACS) during glucose pulse. Glucose pulse was performed in anaerobically growing yeast Saccharomyces cerevisiae in steady state chemostat (D = 0.1 h-1) and transent concentrations of the extra- and intracellular metabolites from central carbon metabolism (e.g. glycolysis, PPP, glycerol, purines, etc) were measured.\n\n\nASSAYS\nCellular size and granularity during glucose pulse\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nAnalysis quenching efficiency based on ATP measurements(id: 954 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nJohn Raedts\nWouter Kuit\nPEOPLE\nWouter Kuit\nJohn Raedts\nPROJECTS\nCOSMIC\nSUBMITTER\nJohn Raedts\n\n____________________________________________________________________________\n\nBarbara Mlody(id: 548 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMax-Planck Institute for Molecular Genetics\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nTest created 1552503978484(id: 960 | type: assays)\n\nDescription: <br><br><strong>Related Publications</strong></strong><br>Me and Helena (2014). some book. <br>Moore A, Zielinski T, Millar AJ (2014). Online period estimation and determination of rhythmicity in circadian data, using the BioDare data infrastructure.. Methods in molecular biology (Clifton, N.J.). Retrieved from: doi.org/10.1007/978-1-4939-0700-7_2<br>Stein JM (1975). The effect of adrenaline and of alpha- and beta-adrenergic blocking agents on ATP concentration and on incorporation of 32Pi into ATP in rat fat cells.. Biochemical pharmacology. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/12<br><br><strong>Comments</strong></strong><p>\n\tNew comment after 2nd version</p>\n 2019-01-10 11:59:51 201 zielu tomasz.zielinski@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\r\n\tCommentsdfa</p>\r\n<br><br><strong>Originally submitted to PLaSMo on 2014-07-25 18:01:50</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nTomasz Zielinski\nINVESTIGATION\nZielinski, Tomasz\nPEOPLE\nTomasz Zielinski\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nAuxSim full - PLM_30(id: 499 | type: studies)\n\nDescription: \tA cell-level model of the Arabidopsis root elongation zone. This spatial model is divided up into biological cells which are further divided into simulation boxes. The original model was designed to investigate how canal cells can accumulate auxin over time rather than to investigate the transport of auxin through the canal cells per se. The main outputs of the simulations in the original paper were the steady state ratios of auxin in the canal cell protoplasts to that in the parenchyma cell protoplasts for different cell sizes and AUX/LAX auxin carrier distributions.\tNote that the working version of the model is in the Simile (.sml) version rather than in XML (which is a temporary holding dummy file).&nbsp;<br><br><strong>Related Publications</strong></strong><br>Kramer E.M. (2004). PIN and AUX/LAX proteins: their role in auxin accumulation. . TRENDS in Plant Sciences, 9, 578-582 . <br><br><strong>Originally submitted to PLaSMo on 2011-01-11 20:52:13</strong></strong>\n\n\nASSAYS\nAuxSim full - PLM_30, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nDATA_FILES\nWorking Simile model, PLM_30_ 1\nSimile screenshot, PLM_30_ 1\nfiles to help specify tissue config for Simile model, PLM_30_ 1\nexample files to run Simile model, PLM_30_ 1\nINVESTIGATION\nMillar, Andrew\nMODELS\nAuxSim full - PLM_30, version 1, SUBMITTED\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nArabidopsis clock-associated pseudo-response regulators PRR9, PRR7 and PRR5 coordinately and positively regulate flowering time through the canonical CONSTANS-dependent photoperiodic pathway(id: 308 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seaton\nPROJECTS\nMillar group\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \r\nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nQuenching and Extraction of intracellular metabolites(id: 122 | type: sops)\n\nDescription: This method describes how one can quench metabolism of Escherichia coli and extract metabolites from many kinds of metabolite classes like: nucleotides, sugar-phosphates, organic acids .... \n\n\nASSAYS\nDetermination of intracellular metabolite concentrations\nCREATORS\nStefan Stagge\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nPEOPLE\nStefan Stagge\nPROJECTS\nSUMO\nSTUDIES\nDetermination of the impact of specific enzyme reactions and regulatory proteins using isogenic E. coli mutant strains\nSUBMITTER\nStefan Stagge\n\n____________________________________________________________________________\n\nThe dynamic architecture of the metabolic switch in Streptomyces coelicolor(id: 8 | type: publications)\n\nDescription: missing\n\n\nDATA_FILES\nFermentor 199 transcriptomics\nPEOPLE\nDavid Hodgson\nDavid Rand\nDavid Wild\nElizabeth Wellington\nEriko Takano\nGregory Challis\nJuan-Francisco Martin\nKay Nieselt\nLubbert Dijkhuizen\nMaggie Smith\nNigel Burroughs\nRainer Breitling\nTrond Ellingsen\nWolfgang Wohlleben\nWalid Omara\nM. Tauqeer Alam\nWilliam Gaze\nyvind Jakobsen\nRoxane Legaie\nJay Moore\nPer Bruheim\nR Jansen\nJens Reuther\nAntonio Rodrguez-Garca\nLouise Thomas\nHvard Sletta\nMaria Elena Merlo\nAlexander Wentzel\nPROJECTS\nSTREAM\nSTUDIES\nTimeseries 1\n\n____________________________________________________________________________\n\nMeasurement of cytochrome numbers(id: 49 | type: assays)\n\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Description: This .csv file shows the numbers of different cytochrome molecules per cell from steady-state continuously-grown cultures at various aerobiosis levels (0%, 31%, 56%, 85% and 115% AAU).\r\n\r\n\n\n\nINVESTIGATION\nSteady state studies for different oxygen availability in Escherichia coli\nORGANISMS\nEscherichia coli\nPEOPLE\nAlison Graham\nPROJECTS\nSUMO\nSTUDY\nThe Escherichia coli steady state response to oxygen: from molecular interactions to regulatory modules\nSUBMITTER\nAlison Graham\n\n____________________________________________________________________________\n\nChip-chip Excel template example(id: 931 | type: data_files)\n\nDescription: This Excel template is an example taken from the GEO web site (http://www.ncbi.nlm.nih.gov/geo/info/spreadsheet.html#GAtemplates) which has been modified to conform to the SysMO JERM (Just Enough Results Model).\r\nUsing templates helps with searching and comparing data as well as making it easier to submit data to public repositories for publications.\n\n\nASSAYS\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\r\n\r\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \r\n\r\nKey questions:\r\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\r\n\r\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\r\n\r\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\r\n\r\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharit University Medicine Berlin\nGriffin Discoveries BV\nUniversittsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Bhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nPDE model of oxygen diffusion near E. coli cells(id: 92 | type: models)\n\nDescription: This partial-differential equations model focuses on the oxygen gradients in consideration of the three-dimensional cell and environment.\n\n\nCREATORS\nSamantha Nolan\nPEOPLE\nDavid Knies\nSamantha Nolan\nPROJECTS\nSUMO\nSUBMITTER\nDavid Knies\n\n____________________________________________________________________________\n\nPGK Kinetic characterisation(id: 198 | type: sops)\n\nDescription: SOP for measurement of PGK activity in extracts.\n\n\nASSAYS\nPGK\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nAlkali metal cation transport and homeostasis in yeasts(id: 25 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nHana Sychrova\nJoaquin Ario\nJose Ramos\nFalko Krause\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nShinsuke Ohnuki(id: 545 | type: people)\n\nDescription: I am a postdoc in The University of Tokyo. My research interest is to know structure of biological systems by functional relationships between genes working in response to endogenous and/or exogenous disruptions. I'm trying to find functional relationships between genes by similarity of phenotype defined with high-dimensional morphological features in yeast gene deletion collection. I'm happy if you will talk with me in any topics.\n\n\nINSTITUTIONS\nUniversity of Tokyo\nPROJECTS\nICYSB 2015 - International Practical Course in Systems Biology\n\n____________________________________________________________________________\n\nbatchfermentation exp-starv01 transcriptome_Microarray Agilent platform_090204(id: 11 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nGene expression(Transcriptome)\nFermentation-BM5_SysMo\nCREATORS\nPraveen Kumar Sappa\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nPEOPLE\nPraveen Kumar Sappa\nPROJECTS\nBaCell-SysMO\nSTUDIES\nBatchfermentation exp-starv01_090204\n\n____________________________________________________________________________\n\nfermentation and product spectrum of Clostridium acetobutylicum ctfA::int(id: 1027 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nThomas Rimpf\nPEOPLE\nThomas Rimpf\nPROJECTS\nCOSMIC\nSUBMITTER\nThomas Rimpf\n\n____________________________________________________________________________\n\nModelling circadian regulation of flowering time and hypocotyl elongation, Seaton et al., 2015 - PLM_1010, version 1, SIMPLIFIED(id: 481 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_1010, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nModelling circadian regulation of flowering time and hypocotyl elongation, Seaton et al., 2015 - PLM_1010, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nSeaton, Daniel\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nModelling circadian regulation of flowering time and hypocotyl elongation, Seaton et al., 2015 - PLM_1010\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nDynamics of intracellular metabolites during glucose pulse(id: 83 | type: data_files)\n\nDescription: Dynamics of intracellular metabolites (pyr, suc, fum, mal, akg, pep, g3p, 2pg, 3pg, cit, r5p, f6p, g6p, 6pg, ATP, ADP, AMP, UTP, GTP, inosine, NAD+, IMP, UDP, NADP+, CTP, AdenyloSuccinate, NADPH, trehalose) during glucose pulse. Glucose pulse was performed in anaerobically growing yeast Saccharomyces cerevisiae in steady state chemostat (D = 0.1 h-1) and transent concentrations of the extra- and intracellular metabolites from central carbon metabolism (e.g. glycolysis, PPP, glycerol, purines, etc) were measured\r\n\n\n\nASSAYS\nDynamics of intracellular metabolites during glucose pulse\nMOSES: dynamic model of glucose pulse\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nPUBLICATIONS\nFast sampling for quantitative microbial metabolomics: new aspects on cold methanol quenching: metabolite co-precipitation\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nPenicillin-binding protein folding is dependent on the PrsA peptidyl-prolyl cis-trans isomerase in Bacillus subtilis(id: 46 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nUwe Voelker\nBogumila Marciniak\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nKinetic properties of Trk(id: 47 | type: studies)\n\nDescription: At external potassium concentrations in the range of 10uM to 1mM Trk is major cellular uptake system for potassium. This system responds to the activiy of the proton pump. Transmembrane fluxes of protons and potassium cations are analysed in a signal-response relationship.\n\n\nASSAYS\nModelling of the Trk-current\nDescribing of membrane potential changes in response to proton fluxes.\nDetermination of kinetic properties based on MIFE flux data and measurements of internal pH changes.\nINVESTIGATION\nTRK1,2 Transport Systems of Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nPablo Jos Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nsigB-activation-comic_vol1(id: 77 | type: presentations)\n\nDescription: The image file represents the activation of sigB in response to glucose starvation. It was drawn alongside the model 'sigb-response_starvation_shakeflask'.\r\nBiomass catalyses the conversion of glucose to biomass, while a low glucose concentration will negatively affect cell viability (inverted shown in the figure, where Glc inhibits cell death). \r\nW (RsbW) can form dimers with sigB (B) and dimers and trimers with V (RsbV)(WV, WV2) and is able to phosphorylate V to VP. A high glucose concentration inhibits the dephosphorylation of VP. sigB dependent transcription alongside sigA (A) dependent transcription stays constant.\r\nUpon glucose starvation dephosphorylation of VP increases V which performs partner switch with sigB on W. sigB is released to start the transcription of stress response genes.\n\n\nCREATORS\nUlf Liebal\nPEOPLE\nUlf Liebal\nPROJECTS\nBaCell-SysMO\nSUBMITTER\nUlf Liebal\n\n____________________________________________________________________________\n\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells(id: 106 | type: investigations)\n\nDescription: missing\n\n\nCREATORS\nLorenz Adlung\nAndreas Franke\nRainer Saffrich\nMarie-Christine Wagner\nMarcel Schilling\nPEOPLE\nMaja Rey\nLorenz Adlung\nAndreas Franke\nMarcel Schilling\nRainer Saffrich\nMarie-Christine Wagner\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\nPUBLICATIONS\nEvaluation of GMP-compliant culture media for in vitro expansion of human bone marrow mesenchymal stromal cells\nMicrocavity arrays as an in vitro model system of the bone marrow niche for hematopoietic stem cells\nMesenchymal stromal cells inhibit proliferation of virus-specific CD8(+) T cells\nStandardization of Good Manufacturing Practice-compliant production of bone marrow-derived human mesenchymal stromal cells for immunotherapeutic applications\nPlerixafor induces the rapid and transient release of stromal cell-derived factor-1 alpha from human mesenchymal stromal cells and influences the migration behavior of human hematopoietic progenitor cells\nDifferences between healthy hematopoietic progenitors and leukemia cells with respect to CD44 mediated rolling versus adherence behavior on hyaluronic acid coated surfaces\nFunctional potentials of human hematopoietic progenitor cells are maintained by mesenchymal stromal cells and not impaired by plerixafor\n[Mesenchymal stroma cells (MSCs) for the treatment of rheumatic disease]\nProtein abundance of AKT and ERK pathway components governs cell type-specific regulation of proliferation\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nB. subtilis_SysMo2_Chemostat_growhtrate-salt_cell count(id: 861 | type: data_files)\n\nDescription: B. subtilis was grown in minimal media in a chemostat at different growth rates (= max, =0.1, =0.4) and in the presence of 1.2M NaCl (=0.1) with or without glycinebetaine. Here you'll find cell titers for every sample.\n\n\nASSAYS\nAbsolute quantification of proteins by the AQUA-technology\nCREATORS\nSandra Maass\nINVESTIGATIONS\nMultiomics study of Bacillus subtilis under osmotic stress\nPEOPLE\nSandra Maass\nPROJECTS\nBaCell-SysMO\nSTUDIES\nB. subtilis_SysMo2_Chemostat_growthrate-salt\nSUBMITTER\nSandra Maass\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, CCA1 prod L - PLM_87, version 1, SIMPLIFIED(id: 518 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_87, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, CCA1 prod L - PLM_87, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, CCA1 prod L - PLM_87\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nSamantha Miller(id: 43 | type: people)\n\nDescription: I am a biologist by training.  My research career is focussed on the structure-function relationships of membrane transport systems in Escherichia coli. I am interested in understanding the mechanism of ion and solute transport across the membrane and how this influences bacterial cell survival.  My work mainly focusses on the ligand-gated potassium efflux systems which are crucial for cell survival during electrophile exposure and the mechanosensitive channels involved in hypoosmotic stress responses.\n\n\nINSTITUTIONS\nUniversity of Aberdeen\nPROJECTS\nKOSMOBAC\n\n____________________________________________________________________________\n\nSystems modelling age-related changes in the maintenance of dermal extra-cellular matrix: mechanisms and interventions(id: 118 | type: projects)\n\nDescription: Overall, we will measure in-vitro and model in-silico the short-term biochemical network dynamics of extra cellular matrix maintenance (ECM) in populations of young, old and senescent dermal fibroblasts. We have shown in previous work that differences in network dynamics are highly informative and provide a means to identify parts of the network that could be targeted to restore healthy function.\r\n\r\nThe work will be first carried out in 2D culture of human dermal fibroblasts then extended to a novel 3D in-vitro system which we have established and validation as a reliable surrogate for the human dermis. Intervention strategies will be explored in the computational model and subsequently tested in-vitro. The work will involve: generation of high throughput time series data; development and application of bioinformatics workflows to analyse the data and establish networks of molecular interactions governing the maintenance of ECM; building, calibration and validation of dynamic computational model(s) in vitro; use of the model(s) to identify intervention strategies to modify network behaviour; testing of molecules in-vitro that could be used in potential treatments to improve skin healthcare.\n\n\nINSTITUTIONS\nUniversity of Newcastle\nDurham University\nPEOPLE\nCiaran Welsh\nDaryl Shanley\nCraig Manning\nStefan Przyborski\nHannah O'Keefe\nJames Wordsworth\nPROGRAMMES\nNewcastle University Systems Modelling of Ageing\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2011 - PLM_64(id: 494 | type: studies)\n\nDescription: \tThis model is termed P2011 and derives from the article: <strong>The clock gene circuit in Arabidopsis includes a repressilator with additional feedback loops</strong><strong>. </strong>Alexandra Pokhilko, Aurora Pias Fernndez, Kieron D Edwards, Megan M Southern, Karen J Halliday &amp; Andrew J Millar <em>Mol. Syst. Biol. </em>2012; 8: 574, submitted 9 Aug 2011 and published 6 March 2012. <a href=\"http://www.nature.com/msb/journal/v8/n1/synopsis/msb20126.html\">Link</a>\tLink to <a href=\"http://www.nature.com/msb/journal/v8/n1/extref/msb20126-s1.pdf\">Supplementary Information</a>, including equations. Minor errors in the published Supplementary Information are described in a file attached to version 1 of this model (the published SBML is correct).\t&nbsp;The model describes the circuit depicted in Fig. 1 of the paper (GIF attached). It updates the Pokhilko et al. 2010 model (termed P2010), <a href=\"http://www.plasmo.ed.ac.uk/plasmo/models/model.shtml?accession=PLM_6\">PLM_6</a>, by including:<ol><li>\t\tthe Evening Complex genes (ELF4, ELF3, LUX),</li><li>\t\tlight-regulated degradation of ELF3 by COP1,</li><li>\t\tTOC1 as a repressor rather than an activator of LHY/CCA1.</li></ol>\tThese changes allowed the removal of hypothetical components TOC1mod (or X) and Y from the earlier models. They also reveal that the central loop of the model is a triple-repressor ring oscillator, or 'repressilator' (illustrated in Fig. 8, GIF attached).\tSBML curation notes (please see Comments for each version): Compared to the model version submitted to the Biomodels database, version 1 here slightly alters the names of some variables and uses an SBML AssignmentRule for the light input. A MATLAB version is attached to version 1. In version 2, the rule is used with a generic SBML StepFunction (Adams et al. J. Biol. Rhythms Aug 2012) to describe the light-dark cycle more flexibly. Versions 1 and 2 have multiple sinks and sources created by CellDesigner. These are removed in versions 3 &amp; 4. Version 3 has the published lightfunction, and an equivalent COPASI file attached. Version 4 has the StepFunction.<br><br><strong>Related Publications</strong></strong><br>Alexandra Pokhilko, Aurora Pias Fernndez, Kieron D Edwards, Megan M Southern, Karen J Halliday & Andrew J Millar (2012). The clock gene circuit in Arabidopsis includes a repressilator with additional feedback loops.. Mol. Syst. Biol. 8: 574.. Retrieved from: http://www.nature.com/msb/journal/v8/n1/synopsis/msb20126.html<br><br><strong>Originally submitted to PLaSMo on 2012-03-07 12:15:12</strong></strong>\n\n\nASSAYS\nArabidopsis_clock_P2011 - PLM_64, version 1\nArabidopsis_clock_P2011 - PLM_64, version 2\nArabidopsis_clock_P2011 - PLM_64, version 3\nArabidopsis_clock_P2011 - PLM_64, version 4\nCREATORS\nBioData SynthSys\nAndrew Millar\nDATA_FILES\nFig. 1, outline of P2011 model, with P2010 inset,PLM_64_ 1\nFig. 8, cartoon illustrating repressilator structure,PLM_64_ 1\nODE file for Matlab version of P2011 clock model, PLM_64_ 1\nMatlab version of P2011 clock model, PLM_64_ 1\nDescribes minor errors in published Supplementary Information (published SBML is correct)., PLM_64_ 1\nFig. 1, outline of P2011 model, with P2010 inset,PLM_64_ 3\nFig. 8, cartoon illustrating repressilator structure,PLM_64_ 3\nODE file for Matlab version of P2011 clock model, PLM_64_ 3\nMatlab version of P2011 clock model, PLM_64_ 3\nCopasi file corresponding to PLM_64 version 3, PLM_64_ 3\nFig. 1, outline of P2011 model, with P2010 inset,PLM_64_ 4\nFig. 8, cartoon illustrating repressilator structure,PLM_64_ 4\nODE file for Matlab version of P2011 clock model, PLM_64_ 4\nMatlab version of P2011 clock model, PLM_64_ 4\nINVESTIGATION\nMillar, Andrew\nMODELS\nArabidopsis_clock_P2011 - PLM_64, version 1, SUBMITTED\nArabidopsis_clock_P2011 - PLM_64, version 1, SIMPLIFIED\nArabidopsis_clock_P2011 - PLM_64, version 2, SUBMITTED\nArabidopsis_clock_P2011 - PLM_64, version 2, SIMPLIFIED\nArabidopsis_clock_P2011 - PLM_64, version 3, SUBMITTED\nArabidopsis_clock_P2011 - PLM_64, version 3, SIMPLIFIED\nArabidopsis_clock_P2011 - PLM_64, version 4, SUBMITTED\nArabidopsis_clock_P2011 - PLM_64, version 4, SIMPLIFIED\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nSalazar_et_al_2009_Photoperiodism_Model - PLM_74, version 1(id: 888 | type: assays)\n\nDescription: \tThis is the SimileXML for the Salazar2009_FloweringPhotoperiod model in PlaSMo. It corresponds to Model 3 in the publication of Salazar et al 2009. The Simile version of this model is also attached here.\t<strong>Instructions to run the Photoperiodism Model in Simile</strong>\t1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Save all the files into the same folder.\t2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Copy and paste the attached lightfunction.pl file in the following folder:\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Program File &gt; Simile6.0 (or other software version)&gt; Functions\t3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Download the attached Simile file or import the XML into Simile:\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; File &gt; Import &gt; XML Model Description\t4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; To run the model:\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Model &gt; Run or click on the Play (triangle) button\t5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; When you run the model for the first time, you will be prompted to load the parameter file. To do so:\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Load the Salazar model.spf file at the TOP LEVEL\t6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The time unit and time step also need to be adjusted as follows:\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The time step can be set in the run display window. The time step #1 should be set to 0.001\t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; b.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The time unit should be changed to hour, which can be done in the Run settings tab in the run display window<br><br><strong>Related Publications</strong></strong><br>Salazar JD, Saithong T, Brown PE, Foreman J, Locke JC, Halliday KJ, Carr IA, Rand DA, Millar AJ.  (2009). Prediction of photoperiodic regulators from quantitative gene circuit models . Cell. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/20005809  <br><br><strong>Originally submitted to PLaSMo on 2013-11-12 15:06:57</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nDATA_FILES\nSimile file for the Salazar et al 2009 Photoperiodism Model, PLM_74_ 1\nParameter file to run the model in Simile, PLM_74_ 1\nA Simile function file for light input into the clock model , PLM_74_ 1\nINVESTIGATION\nHoon, Yin\nMODELS\nSalazar_et_al_2009_Photoperiodism_Model - PLM_74, version 1, SUBMITTED\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nSalazar_et_al_2009_Photoperiodism_Model - PLM_74\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nMulti-omics Reveals the Lifestyle of the Acidophilic, Mineral-Oxidizing Model Species\n              Leptospirillum ferriphilum\n              T(id: 377 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nMulti -omics reveal lifestyle of acidophile, mineral-oxidizing model species Leptospirillum ferriphilumT\nPEOPLE\nMalte Herold\nMark Dopson\nWolfgang Sand\nPaul Wilmes\nIgor Pivkin\nAnsgar Poetsch\nAntoine Buetti-Dinh\nStephan Christel\nSoeren Bellenberg\nMohamed El Hajjami\nPROJECTS\nSysMetEx\n\n____________________________________________________________________________\n\nHana Sychrova(id: 96 | type: people)\n\nDescription: biochemistry, molecular biology and physiology of lower eukaryotes,\r\ncharacterization of cell membrane transporters\n\n\nINSTITUTIONS\nInstitute of Physiology Academy of Sciences\nPROJECTS\nTRANSLUCENT\nPUBLICATIONS\nLack of main K(+) uptake systems in Saccharomyces cerevisiae cells affects yeast performance in both potassium-sufficient and potassium-limiting conditions\nAlkali metal cation transport and homeostasis in yeasts\nSaccharomyces cerevisiae BY4741 and W303-1A laboratory strains differ in salt tolerance\nAlkali-metal-cation influx and efflux systems in nonconventional yeast species\nSystems Biology of Monovalent Cation Homeostasis in Yeast\n\n____________________________________________________________________________\n\nSABIO-RK: Finding, curating, and publishing data(id: 257 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nProtein purification from E. coli extracts(id: 187 | type: sops)\n\nDescription: Purification of gluconeogenic enzymes from S. solfataricus in recombinant E.coli extracts\n\n\nASSAYS\nPGK\nGAPDH\nTPI\nFBPAase\nCREATORS\nJacky Snoep\nTheresa Kouril\nINVESTIGATIONS\nCentral Carbon Metabolism of Sulfolobus solfataricus\nPEOPLE\nJacky Snoep\nTheresa Kouril\nPROJECTS\nSulfoSys\nSTUDIES\nModel Gluconeogenesis\nSUBMITTER\nJacky Snoep\n\n____________________________________________________________________________\n\nCopenhagen meeting 2012: Rostock presentation (id: 50 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nTomas Fiedler\nPEOPLE\nTomas Fiedler\nPROJECTS\nSysMO-LAB\nSUBMITTER\nTomas Fiedler\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nPerturbing the biological system: developmental adaptation and experimental manipulation of trypanosome biology(id: 40 | type: investigations)\n\nDescription: Multiply perturbations of trypanosome redox metabolism, closing the feedback loop between experimentation and in silioc modelling, allowing model refinement or, where there are unexpected outcomes, re-evaluation. \r\nProviding a dynamic picture of cell physiology by examining programmed metabolic changes during the developmental life-cycle of these parasites as they adapt to very different external milieus, including distinct levels of oxidative stress and unique adaptations of their redox balance system.\n\n\nASSAYS\nGeneration of RNAi cell lines in T.brucei brucei 2T1 cell line\nInduction of RNA interference in T.brucei brucei 2T1 cell line\nDATA_FILES\nInduction of RNA interference in T.brucei\nRNAi growth curves\nPEOPLE\nFederico Rojas\nPROJECTS\nSilicoTryp\nSOPS\nGeneration of RNAi cell lines in T.brucei brucei 2T1 cell line\nSTUDIES\nTargeted disruption and over expression of critical enzymes\nSUBMITTER\nFederico Rojas\n\n____________________________________________________________________________\n\n3' or Whole Gene Expression Array Template (Affymetrix)(id: 928 | type: data_files)\n\nDescription: This Excel template is for use with affy transcriptomics data in Matrix table format. It was created from a template on the GEO web site (http://www.ncbi.nlm.nih.gov/geo/info/geo_affy.html) and modified to conform to the SysMO JERM for transcriptomics.\n\n\nASSAYS\nAffy Transcriptomics Templates\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nAn Introduction to SEEK, JERM and ISA(id: 29 | type: presentations)\n\nDescription: This presentation gives an overview of the way we structure and share data in the SEEK using JERM templates, and how we use the ISA hierarchy of Investigations, Studies, and Assays to associate experiments and data sets together.\r\nThese slides were presented at the SEEK Users meeting in Berlin 2012.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nMeasurement of intracellular K+ content of cells lacking TRK1 an(id: 104 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nSilvia Petrezselyova\nPEOPLE\nSilvia Petrezselyova\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nYeast_deFBA Model(id: 555 | type: models)\n\nDescription: This SBML file uses the RAM extension and contains a minimal genome scaled model for Saccharomyces cerevisiae. The model is based of Yeast 6.06 and was published first in A.-M. Reimers Thesis \"Understanding metabolic regulation and cellular resource allocation through optimization\".\n\n\nCREATORS\nHenning Lindhorst\nAlexandra-M. Reimers\nPEOPLE\nHenning Lindhorst\nAlexandra-M. Reimers\nPROJECTS\nROBUSTYEAST\nSUBMITTER\nHenning Lindhorst\n\n____________________________________________________________________________\n\nSander Kersten(id: 756 | type: people)\n\nDescription: Dr. Sander Kersten received his PhD in Nutritional Biochemistry from Cornell University in 1997. After a postdoctoral stay in the laboratory of Dr. Walter Wahli at the University of Lausanne, Switzerland, he moved to Wageningen in 2000, initially as a fellow of the Royal Netherlands Academy of Arts and Sciences and later as Associate Professor. Since 2011 he is Full Professor in Molecular Nutrition and since 2014 chair of the Nutrition, Metabolism and Genomics group. His current research interests revolve around nutrition, lipid metabolism and metabolic diseases, with a special interest in key regulatory proteins such as PPARs and ANGPTL4.\n\n\nINSTITUTIONS\nWageningen University & Research\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nHeterogeneity of glycolytic oscillatory behaviour in individual yeast cells(id: 377 | type: studies)\n\nDescription: missing\n\n\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATION\nGlycolytic oscillations in individual isolated yeast cells\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nPUBLICATIONS\nHeterogeneity of glycolytic oscillatory behaviour in individual yeast cells\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nSuppression of indoleamine-2,3-dioxygenase 1 expression by promoter hypermethylation in ER-positive breast cancer.(id: 362 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nChristiane Opitz\nSarah Schott\nInes Heiland\nKarl-Heinz Kellner\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nRobOKoD: microbial strain design for (over)production of target compounds(id: 236 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nRobOKoD applied to e.coli for butanol production.\nOptKnock applied to e.coli for butanol production.\nRobustKnock applied to e.coli for butanol production.\nDATA_FILES\nFBA result of RobOKoD designed e.coli strain.\nMODELS\niNS142 RobOKoD Redesigned Butanol Producing.\nE.coli Core Model with Beta Oxidation Cycle.\nPEOPLE\nNatalie Stanford\nPROJECTS\nManchester Institute for Biotechnology\n\n____________________________________________________________________________\n\nmRNA decay assay(id: 156 | type: assays)\n\nDescription: This assay is designed to measure the decay kinetics of mRNA in T. brucei blood forms. T. brucei lacks the canonical transcriptional regulation employed by other eukaryotes through transcription factors, and relies almost entirely on regulation of mRNA decay and further downstream steps in order to control gene expression.  \r\n3 replicates of 8 time points were taken to measure mRNA abundance in the cell using RNA-seq. The first time point was fot WT, untreated cells; the second was 5 min after the addition of the splicing inhibitor Sinefungin; the following time points were after inhibition of transcription with Actinomycin D. \n\n\nDATA_FILES\nmRNA decay\nHalf life values for enzymes of redox and polyamine pathways as provided by Alejandro\nmRNA half-lives\nINVESTIGATION\nGene expression in Trypanosoma brucei\nORGANISMS\nTrypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSOPS\nSample preparation for mRNA decay study\nSTUDY\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\r\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nOverflow of a hyper-produced secretory protein from the Bacillus Sec pathway into the Tat pathway for protein secretion as revealed by proteogenomics(id: 67 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nNeurospora Circadian Clock 3-variable model - PLM_51, version 1, SUBMITTED(id: 485 | type: models)\n\nDescription: Originally submitted model file for PLaSMo accession ID PLM_51, version 1\n\n\nASSAYS\nNeurospora Circadian Clock 3-variable model - PLM_51, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nAdams, Richard\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nNeurospora Circadian Clock 3-variable model - PLM_51\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nSULFOSYSbiotech_Proteome analysis(id: 211 | type: sops)\n\nDescription: Protein extraction, iTRAQ labeling, peptides separation, mass spectrometry and data analysis\n\n\nASSAYS\nProteome analysis: d-fuc / l-glu\nProteome analysis: Casaminoacids versus D-Glc\nCREATORS\nTheresa Kouril\nPhil Wright\nTrong Khoa Pham\nINVESTIGATIONS\nL-fucose degradation in Sulfolobus solfataricus P2\nAmino acid degradation in Sulfolobus solfataricus P2\nPEOPLE\nPhil Wright\nTheresa Kouril\nTrong Khoa Pham\nPROJECTS\nSulfoSys - Biotec\nSTUDIES\nComparison of S. solfataricus grown on l-fucose and d-glucose\nComparison of Sulfolobus solfataricus P2 grown on caseinhydrolysate and D-glucose\nSUBMITTER\nTheresa Kouril\n\n____________________________________________________________________________\n\nExternal pH changes(id: 65 | type: assays)\n\nDescription: External pH changes for the conditions of Navarrete et al. (2010) will be estimated from the internal pH changes and the volume ratio of cell sample to medium.\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nKey Cell-Physiological Parameters\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nThe large mechanosensitive channel MscL determines bacterial susceptibility to the bacteriocin sublancin 168(id: 75 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nUwe Voelker\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nExtraction and quantitative analysis of proteins from FFPE tissue(id: 281 | type: sops)\n\nDescription: Extraction and quantitative analysis of proteins from FFPE tissue\n\n\nCREATORS\nOlga Krebs\nRuben Van Heck\nPEOPLE\nOlga Krebs\nRuben Van Heck\nPROJECTS\nEmPowerPutida\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nRalf Heermann(id: 162 | type: people)\n\nDescription: I am research assistant in the microbiology department at the Ludwig-Maximilians Universitt in Munich (Mnchen), working at the chair of Prof. Kirsten Jung. In our SysMO consortium we generate biological data and work in close cooperation with the workgroup of Dr. Andreas Kremling of the Max-Planck-Institut fr Dynamik komplexer technischer Systeme in Magdeburg who performs mathematical modeling. The topic of our workpackage deals with \"K+ homeostasis in Escherichia coli\", wherby the K+ transporters, the transcription of the corresponding genes, and the networks to other systems are analyzed. Thereby, biological data generation goes hand in hand with mathematical modeling.\n\n\nINSTITUTIONS\nLudwig-Maximilians University of Munich\nPROJECTS\nKOSMOBAC\n\n____________________________________________________________________________\n\nBatchfermentation_exp-starv01_2D-proteomics(id: 105 | type: data_files)\n\nDescription: batch fermatation - The transition from growing to non-growing Bacillus subtilis cells\n\n\nASSAYS\n2D-gelbased analysis of intracellular proteins\nCREATORS\nSandra Maass\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nPEOPLE\nSandra Maass\nPROJECTS\nBaCell-SysMO\nSTUDIES\nBatchfermentation exp-starv01_090204\nSUBMITTER\nSandra Maass\n\n____________________________________________________________________________\n\nNHA1 Transport System in Saccharomyces cerevisiae(id: 20 | type: investigations)\n\nDescription: Properties of cells lacking the NHA1 gene.\n\n\nASSAYS\nCurrent voltage relation for different external KCl\nCurrent-voltage relation for different internal potassium\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDIES\nCurrent-voltage relations\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nSustained glycolytic oscillations in individual isolated yeast cells - III(id: 82 | type: models)\n\nDescription: Using optical tweezers to position yeast cells in a microfluidic chamber, we were able to observe sustained oscillations in individual isolated cells. Using a detailed kinetic model for the cellular reactions, we simulated the heterogeneity in the response of the individual cells, assuming small differences in a single internal parameter. By operating at two different flow rates per experiment, we observe four of categories of cell behaviour. The present model (gustavsson3) predicts the steady-state to limit cycle transition observed in 40.2% of the cell population. \n\n\nASSAYS\nModelling sustained glycolytic oscillations in individual isolated yeast cells (Gustavsson models)\nCREATORS\nFranco Du Preez\nJacky Snoep\nINVESTIGATIONS\nYeast Glycolytic Oscillations\nPEOPLE\nJacky Snoep\nFranco Du Preez\nPROJECTS\nSysMO DB\nPUBLICATIONS\nSustained glycolytic oscillations in individual isolated yeast cells\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction\nSTUDIES\nSustained glycolytic oscillations in individual isolated yeast cells\nSUBMITTER\nFranco Du Preez\n\n____________________________________________________________________________\n\nFormation of xylitol and xylitol-5-phosphate and its impact on growth of d-xylose-utilizing Corynebacterium glutamicum strains(id: 323 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nStephan Noack\nNiklas Tenhaef\nPROJECTS\nXyloCut\n\n____________________________________________________________________________\n\nFigure 1C: Biphasic control can resist mutant invasion of feedback circuits.(id: 383 | type: studies)\n\nDescription: C Trajectories of Z from different initial concentrations of cells (Z) (i) or y (ii) for the circuit of (B). The healthy concentration Z = ZST is reached regardless of initial\r\nconcentration of Z, as long as it is nonzero, and regardless of the initial concentration of y.\r\n\r\nSED-ML simulation\r\nhttps://jjj.bio.vu.nl/models/experiments/karin2017_fig1c/simulate\n\n\nASSAYS\nBiphasic control can resist mutant invasion of feedback circuits.\nCREATORS\nJacky Snoep\nINVESTIGATION\nKarin et al (2017) Molecular Systems Biology\nPEOPLE\nJacky Snoep\nPROJECTS\nMolecular Systems Biology\nSUBMITTER\nJacky Snoep\n\n____________________________________________________________________________\n\nMembrane Potential(id: 83 | type: assays)\n\nDescription: To estimate the relative level of plasma-membrane potential in the wild type, single trk1 and trk2 or double trk1 trk2 mutants, the diSC3(3) fluorescent probe was added to cells grown under various conditions and the levels of cell fluorescence were monitored. \n\n\nINVESTIGATION\nTRK1,2 Transport Systems of Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nKey Cell-Physiological Parameters  of TRK1,2 Transport Systems Investigation\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nPGM Kinetic characterisation(id: 199 | type: sops)\n\nDescription: SOP for measurement of PGM activity in extracts.\n\n\nASSAYS\nPGM\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nTest created 1552504117107(id: 961 | type: assays)\n\nDescription: <br><br><strong>Related Publications</strong></strong><br>Me and Helena (2014). some book. <br>Moore A, Zielinski T, Millar AJ (2014). Online period estimation and determination of rhythmicity in circadian data, using the BioDare data infrastructure.. Methods in molecular biology (Clifton, N.J.). Retrieved from: doi.org/10.1007/978-1-4939-0700-7_2<br>Stein JM (1975). The effect of adrenaline and of alpha- and beta-adrenergic blocking agents on ATP concentration and on incorporation of 32Pi into ATP in rat fat cells.. Biochemical pharmacology. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/12<br><br><strong>Comments</strong></strong><p>\n\tNew comment after 2nd version</p>\n 2019-01-10 11:59:51 201 zielu tomasz.zielinski@ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\r\n\tCommentsdfa</p>\r\n<br><br><strong>Originally submitted to PLaSMo on 2014-07-25 18:01:50</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nTomasz Zielinski\nINVESTIGATION\nZielinski, Tomasz\nPEOPLE\nTomasz Zielinski\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nMOD-MTASK: Validating biochemical network reconstructions of Atlantic salmon(id: 423 | type: presentations)\n\nDescription: My graduation project presentation given on 22 of June at the Hanze University of Applied Sciences Groningen in The Netherlands. \n\n\nCREATORS\nWout Van Helvoirt\nPEOPLE\nWout Van Helvoirt\nPROJECTS\nDigiSal\nSUBMITTER\nWout Van Helvoirt\n\n____________________________________________________________________________\n\nAuxSim - PLM_27(id: 498 | type: studies)\n\nDescription: \tA cell-level model of the Arabidopsis root elongation zone. This spatial model is divided up into biological cells which are further divided into simulation boxes. The original model was designed to investigate how canal cells can accumulate auxin over time rather than to investigate the transport of auxin through the canal cells per se. The main outputs of the simulations in the original paper were the steady state ratios of auxin in the canal cell protoplasts to that in the parenchyma cell protoplasts for different cell sizes and AUX/LAX auxin carrier distributions.\tNote that the working version of the model is in the Simile (.sml) version rather than in XML (which is a temporary holding dummy file).&nbsp;\tThe publication is:\tKramer E.M. (2004) PIN and AUX/LAX proteins: their role in auxin accumulation. TRENDS in Plant Sciences, 9, 578-582 (+ supplementary material).<br><br><strong>Originally submitted to PLaSMo on 2011-01-07 21:54:18</strong></strong>\n\n\nASSAYS\nAuxSim - PLM_27, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATION\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nOff-gas monitoring during glucose pulse(id: 81 | type: data_files)\n\nDescription: Monitoring of partial pressure of CO2 in off-gas. Glucose pulse was performed in anaerobically growing yeast Saccharomyces cerevisiae in steady state chemostat (D = 0.1 h-1) and transent concentrations of the extra- and intracellular metabolites from central carbon metabolism (e.g. glycolysis, PPP, glycerol, purines, etc) were measured.\n\n\nASSAYS\nMOSES: dynamic model of glucose pulse\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nInformation on the draft transcription/translation model.(id: 1047 | type: data_files)\n\nDescription: Describes the assumptions made, how to integrate the new reactions to the rest of the glycolysis model, the parameters needed and the preliminary parameter values collected from the litterature.\n\n\nCREATORS\nFiona Achcar\nPEOPLE\nFiona Achcar\nPROJECTS\nSilicoTryp\nSUBMITTER\nFiona Achcar\n\n____________________________________________________________________________\n\nGlucose pulsed L. lactis 2(id: 73 | type: data_files)\n\nDescription: L. lactis was grown in LAB medium, strongly concentrated and glucose-pulsed in a MES buffer. Intracellular metabolite concentration is followed in time.\n\n\nASSAYS\nGlucose pulsed L. lactis\nCREATORS\nMartijn Bekker\nINVESTIGATIONS\nInvestigation of glycolysis and pyruvate branching of three lactic acid bacteria\nPEOPLE\nMartijn Bekker\nPROJECTS\nSysMO-LAB\nSTUDIES\nComparative modeling and phosphate dependence flux distributions and glucose consumption rates of two lactic acid bacteria S. pyogenes and L. lactis\nSUBMITTER\nMartijn Bekker\n\n____________________________________________________________________________\n\nTranscription of glucose transport systems under batch versus chemostat conditions.(id: 1073 | type: data_files)\n\nDescription: Mutants with defects in glucose transport systems were analyzed in a bioreactor and the transcription of certain uptake systems analyzed. Transcription profiles between batch and chemostat conditions were compared\n\n\nCREATORS\nSonja Steinsiek\nPEOPLE\nSonja Steinsiek\nPROJECTS\nSUMO\nPUBLICATIONS\nGlucose Transport in Escherichia coli Mutant Strains with Defects in Sugar Transport Systems\nSUBMITTER\nSonja Steinsiek\n\n____________________________________________________________________________\n\nGeneration of cell material from Streptomyces coelicolor culture for proteome analysis(id: 134 | type: sops)\n\nDescription: missing\n\n\nASSAYS\nOnline/offline measurements\nCREATORS\nPer Bruheim\nTrond Ellingsen\nSunniva Hoel\nyvind Jakobsen\nHvard Sletta\nAlexander Wentzel\nAnders verby\nINVESTIGATIONS\nMetabolism of Streptomyces coelicolor (SysMO STREAM)\nPEOPLE\nTrond Ellingsen\nyvind Jakobsen\nJay Moore\nPer Bruheim\nHvard Sletta\nAnders verby\nSunniva Hoel\nAlexander Wentzel\nPROJECTS\nSTREAM\nSTUDIES\nTimeseries 1\nSUBMITTER\nJay Moore\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, acc immediate - PLM_83, version 1, SIMPLIFIED(id: 508 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_83, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, acc immediate - PLM_83, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, acc immediate - PLM_83\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nPhotoreceptor effects on plant biomass, resource allocation, and metabolic state(id: 347 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nAndrew Romanowski\nPROJECTS\nPHYTOCAL: Phytochrome Control of Resource Allocation and Growth in Arabidopsis and in Brassicaceae crops\n\n____________________________________________________________________________\n\nTranscriptome data of Clostridium acetobutylicum growing at steady-state pH 5.7, 5.5, 5.3, 5.1, 4.9, 4.7, and pH 4.5.(id: 834 | type: data_files)\n\nDescription: Micro array analyses of all mRNA levels of cells growing at steady state pH values 5.7, 5.5, 5.3, 5.1, 4.9, 4.7 compared to the same reference steady-state pH 4.5.\n\n\nASSAYS\nComparison of the transcriptome between pH 5.7 (acidogenesis) and pH 4.5 (solventogenesis)\nCREATORS\nHolger Janssen\nINVESTIGATIONS\nThe effect of pH upon the metabolic shift in Clostridium acetobutylicum in continuous culture.\nPEOPLE\nHolger Janssen\nPROJECTS\nCOSMIC\nPUBLICATIONS\nIntegrative modelling of pH-dependent enzyme activity and transcriptomic regulation of the acetone-butanol-ethanol fermentation of\n              Clostridium acetobutylicum\n              in continuous culture\nSTUDIES\nEffect of pH upon the transcriptome\nSUBMITTER\nHolger Janssen\n\n____________________________________________________________________________\n\nRobert Poole(id: 34 | type: people)\n\nDescription: Work in my laboratory is focussed on microbial physiology - the study of how bacteria and other microorganisms work. Although rooted in the tradition of bacterial growth and intermediary metabolism, microbial physiology now embraces molecular biology, genetics, biochemistry, and indeed any discipline that can shed light on bacterial function. Much of our experimental work is conducted with Escherichia coli, the pre-eminent model organism with unrivalled ease of genetic and physiological manipulation. We are also work on its close relation, Salmonella (an intracellular pathogen and cause of enteritis), Campylobacter jejuni (the most common cause of food-borne disease), as well as Mycobacterium and Neisseria species. Special foci at present are bacterial respiratory chains and haemoglobins, nitrosative stress tolerance in bacteria, the biochemistry of CO interactions and oxygen metabolism.\r\n\r\nThe approaches used to study these problems range from molecular genetics, through microbial biochemistry and biophysics, to physiology and the properties of microbial populations in vitro and in vivo. Consequently, the tools we use are diverse; they include manipulations in vitro and in vivo of nucleic acids, bacterial molecular genetics (e.g. microarray technology, transposon-mediated and site-directed mutagenesis, gene and operon fusions), protein purification, immunochemical methods and characterisation using optical (dual- and multi-wavelength spectrophotometry) tools.\r\n\n\n\nINSTITUTIONS\nUniversity of Sheffield\nPRESENTATIONS\nSystems Understanding of Microbial Oxygen- Dependent and Independent Catabolism (SUMO2 ): A coordinators overview\nPROJECTS\nSUMO\nPUBLICATIONS\nTranscript profiling and inference of Escherichia coli K-12 ArcA activity across the range of physiologically relevant oxygen concentrations\nSystems analysis of transcription factor activities in environments with stable and dynamic oxygen concentrations\nReprogramming of Escherichia coli K-12 metabolism during the initial phase of transition from an anaerobic to a micro-aerobic environment\nModelling complex biological systems using an agent-based approach\nTowards a Systems Level Understanding of the Oxygen Response of Escherichia coli\nA mathematical model of metabolism and regulation provides a systems-level view of how Escherichia coli responds to oxygen\nAgent-based modeling of oxygen-responsive transcription factors in Escherichia coli\nSOPS\nSimons method for making monolysogenic lacZ operon fusions on the E. coli chromosome\nQuantifying cytochromes\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, acc on - PLM_84, version 1, SIMPLIFIED(id: 510 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_84, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, acc on - PLM_84, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, acc on - PLM_84\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nSULFOSYSbiotech_Protein extraction_iTRAQ labeling_Peptides separation_mass spectrometry_data analysis(id: 210 | type: sops)\n\nDescription: missing\n\n\nCREATORS\nTheresa Kouril\nPEOPLE\nTheresa Kouril\nSUBMITTER\nTheresa Kouril\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nSustained glycolytic oscillations in individual isolated yeast cells(id: 378 | type: studies)\n\nDescription: missing\n\n\nASSAYS\ngustavsson1-4 models\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATION\nGlycolytic oscillations in individual isolated yeast cells\nMODELS\ngustavsson1-4 models\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nYeast glycolytic oscillations\nPUBLICATIONS\nSustained glycolytic oscillations in individual isolated yeast cells\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nMscL of Bacillus subtilis prevents selective release of cytoplasmic proteins in a hypotonic environment(id: 68 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nIn vitro phosphorylation of key metabolic enzymes from Bacillus subtilis: PrkC phosphorylates enzymes from different branches of basic metabolism(id: 69 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoerg Stuelke\nMichael Hecker\nUwe Voelker\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nMark Musters(id: 52 | type: people)\n\nDescription: I'm a modeller, specialized in kinetic modeling of biochemical networks. My focus in the SysMO-LAB consortium is on creating models of Lactococcus lactis glycolysis and couple this to other related lactic acid bacteria like Streptococcus pyogenes and Enterococcus faecalis. Besides kinetic modeling, I'm also interested in combining various modeling techniques (genome-scale modeling, qualitative modeling).\n\n\nASSAYS\nGlobal sensitivity analysis\nModel of L. lactis glycolysis\nINSTITUTIONS\nWageningen University & Research\nMODELS\nCOPASI model of L. lactis glycolysis\nSBML file of L. lactis glycolysis\nPublished kinetic model of glycolysis in L. lactis\nPROJECTS\nSysMO-LAB\nPUBLICATIONS\nRole of phosphate in the central metabolism of two lactic acid bacteria - a comparative systems biology approach\n\n____________________________________________________________________________\n\nSalazar2009_FloweringPhotoperiod - PLM_9, version 2(id: 887 | type: assays)\n\nDescription: \tThe model shows how the <em>CONSTANS</em> gene and protein in <em>Arabidopsis thaliana</em> forms a day-length sensor. It corresponds to Model 3 in the publication of Salazar et al. 2009.\tMatlab versions of all the models in the paper are attached to this record as a ZIP archive, as are all the data waveforms curated from the literature to constrain the model. Further information may be available via links from the authors web site (<a href=\"http://www.amillar.org\">www.amillar.org</a>).\tSimulation notes for SBML version of Model3 from Salazar et al., Cell, 2009.\tThe interlocking-loop circadian clock in this model reaches its entrained limit cycle rather slowly. To ensure that simulations show the stable phase under any light:dark cycle, the model should be run for 720 hours (30 days) before evaluating its behaviour in light:dark cycles. Using the boundary value solver in Matlab, as described in the paper, avoids this issue.\tSBML prepared by Dr. Treenut Saithong, based on Matlab by Dr. Domingo Salazar. Model depositor, updates to version 2, Prof. Andrew Millar.\tNote comment from Rob Smith on Figure 5.<br><br><strong>Related Publications</strong></strong><br>Salazar JD, Saithong T, Brown PE, Foreman J, Locke JC, Halliday KJ, Carr IA, Rand DA, Millar AJ. (2009). Prediction of photoperiodic regulators from quantitative gene circuit models. Cell. Retrieved from: http://www.ncbi.nlm.nih.gov/pubmed/20005809<br><br><strong>Comments</strong></strong><p>\n\tThe <em>FT</em> expression in Figure 5 in Salazar et al. (2009) was simulated using Model 3b. Model 3b is a slight modification of Model 3. Instead of having <em>nTOC1</em> (nuclear TOC1 protein) replacing <em>CO</em> mRNA directly (as in equation 4 in the Supplementary), Model 3b contains an additional ODE for <em>CO</em> mRNA as follows:</p>\n<p>\n\t&nbsp;d<em>COm</em>/dt = d<em>nTOC1</em>/dt - 0.0004<em>L</em>.<em>vnTOC1</em>.<em>nTOC1</em></p>\n<p>\n\t&nbsp;<em>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</em></p>\n<p>\n\twhere <em>COm</em> = <em>CO</em> mRNA concentration, <em>L</em> = Light and <em>vnTOC1</em> = nuclear TOC1 protein degradation rate. The additional term represents <em>CO </em>mRNA degradation rate, which was slightly increased (by 0.04% of its original value) during the light interval. This was done to improve fit for training data set 8.</p>\n<p>\n\t&nbsp;</p>\n<p>\n\tThus, equation 4 in the Supplementary becomes</p>\n<p>\n\t&nbsp;d<em>COp</em>/dt = <em>vCOm</em>.<em>COm</em> - (1 - <em>L</em>).<em>vCOp</em>.<em>COp</em>/(<em>kCOp</em> + <em>COp</em>)</p>\n<p>\n\t&nbsp;</p>\n<p>\n\tAll parameter values remain the same.</p>\n 2011-08-19 12:35:48 121 s0921165 s0921165@sms.ed.ac.uk<br><p>\n\t&nbsp;The <em>FT</em> expression in Figure 5 in Salazar et al. (2009) was simulated using Model 3b. Model 3b is a slight modification of Model 3. Instead of having <em>nTOC1</em> (nuclear TOC1 protein) replacing <em>CO</em> mRNA directly (as in equation 4 in the Supplementary), Model 3b contains an additional ODE for <em>CO</em> mRNA as follows:</p>\n<p>\n\t&nbsp;</p>\n<p>\n\t</p>\n<p>\n\t&nbsp;</p>\n<p>\n\t<em>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</em></p>\n<p>\n\twhere <em>COm</em> = <em>CO</em> mRNA concentration, <em>L</em> = Light and <em>vnTOC1</em> = nuclear TOC1 protein degradation rate. The additional term represents <em>CO </em>mRNA degradation rate, which was slightly increased (by 0.04% of its original value) during the light interval. This was done to improve fit for training data set 8.</p>\n<p>\n\t&nbsp;</p>\n<p>\n\tThus, equation 4 in the Supplementary becomes</p>\n<p>\n\t&nbsp;</p>\n<p>\n\t</p>\n<p>\n\t&nbsp;</p>\n<p>\n\tAll parameter values remain the same.</p>\n 2011-08-19 12:23:46 121 s0921165 s0921165@sms.ed.ac.uk<br><br><strong>Version Comments</strong></strong><p>\n\tVersion 2 of the model does not have the Matlab files attached, these are only on version 1.</p>\n<p>\n\tVersion 2 removes the &#39;time&#39; variable in the lightFunction, replacing it with &#39;t&#39; for SBML compatibility.</p>\n<br><br><strong>Originally submitted to PLaSMo on 2010-05-05 15:16:35</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATION\nMillar, Andrew\nMODELS\nSalazar2009_FloweringPhotoperiod - PLM_9, version 2, SUBMITTED\nSalazar2009_FloweringPhotoperiod - PLM_9, version 2, SIMPLIFIED\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nSalazar2009_FloweringPhotoperiod - PLM_9\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nBSA115-sigB-adaptation(id: 24 | type: models)\n\nDescription: The zip file contains model files and an experiment file. Unpack it in a directory and navigate with matlab to there. Use the 'matlab_execution_guide.m' for simulation and visualisation of the model. This file is written in matlab cell mode, so it is not a stand alone function.\r\n\r\nThree models have been developed to test their capacity to reproduce the experimental data from Study: 'Controlled sigmaB induction in shake flask' with Assay: 'IPTG induction of sigmaB in BSA115'.\r\nOne model assumes a post-transcriptional inhibition (PTI) in the expression of the lacZ reporter gene. The PTI is either due to induced instability of lacZ mRNA or LacZ protein after activation of sigmaB. A hypothetical sigmaB dependent protein RegB is responsible for PTI. It is either a RNase or protease in this model.\r\nOne model assumes a transcription inhibitory effect on sigmaB dependent expression by a sigmaB dependent protein called RegB. This protein might bind to the sigma factor to decrease its efficacy in inducing transcription.\r\nOne model assumes that a protease called RegB acts specifically on sigmaB. This protease is also induced by sigmaB.\r\nAll models is based on mass-action (nearly so). Parameter for IPTG concentration is [100,200,1000], kbs, the expression parameter of Pspac-dependent sigmaB is fixed at 100. All other six parameters are estimated with two of the three experiments as training set (IPTG 0.1 and 0.2 mM) and one experiment as test set (IPTG 1mM). Parameter estimation is performed with SBToolbox2 using multiple rounds of particle swarm algorithm (pswarm).\r\n\n\n\nASSAYS\nIPTG induction of sigmaB in BSA115\nCREATORS\nUlf Liebal\nINVESTIGATIONS\nThe transition from growing to non-growing Bacillus subtilis cells\nPEOPLE\nUlf Liebal\nPROJECTS\nBaCell-SysMO\nPUBLICATIONS\nProteolysis of beta-galactosidase following SigmaB activation in Bacillus subtilis\nSTUDIES\nControlled sigmaB induction in shake flask\nSUBMITTER\nUlf Liebal\n\n____________________________________________________________________________\n\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana(id: 186 | type: models)\n\nDescription: The model presents a multi-compartmental (mesophyll, phloem and root) metabolic model of growing Arabidopsis thaliana. The flux balance analysis (FBA) of the model quantifies: sugar metabolism, central carbon and nitrogen metabolism, energy and redox metabolism, proton turnover, sucrose translocation from mesophyll to root and biomass growth under both dark- and light-growth conditions with corresponding growth either on starch (in darkness) or on CO2 (under light). The FBA predicts that photosynthetically active plastid [under fixed ratio of photorespiration / photosynthesis = 0.2] can operate in different energy modes by levering the cyclic electron flow through FQR reaction in light photosynthesis system: (i) when flux through FQR = 0, the plastid is forced to import ATP from cytosol to fulfil the ATP/NADPH requirement for carbon fixation; (ii) when the flux through FQR equals to 0.35 of non-cyclic photosynthesis, the plastid becomes self-sufficient in terms of the ATP balance; (iii) when the flux through FQR is >0.35 of non-cyclic photosynthesis, then the plastid produces ATP surplus, which is exported to cytosol. Thus, the interplay between cyclic photophosphorylation and photorespiration can play regulatory role in adapting of the energy state of plastid to various conditions to hold ATP/NADPH ratio sufficient for the carbon fixation in Calvin cycle. Under all tested light scenarios, NADPH is over-produced in light photosynthesis system, and therefore its surplus is exported from plastid through malate/oxaloacetate shuttle into mitochondria, where the reduced equivalents are used for ATP synthesis. From the energetic point of view, mitochondria is the main ATP provider for cytoplasmic processes together with plastid (when it is actual), glycolysis under both light- and dark-growth conditions; also, mitochondria is the main sink for reduced equivalents translocated both from plastid and from cytoplasm.\r\nThe model described all main metabolic processes participating in the proton metabolism. The translocation of sucrose among plant tissues was associated with an integral balance of protons, which in turn was partially defined by operational modes of the energy metabolism (photosynthesis, respiration). The proton fluxes predicted by FBA generally corresponds to the molecular mechanisms and functional peculiarity of the sucrose transporters SWEET and SUC/STP proton-symporters and net-flux of sucrose from source to sink tissue. Thus, our multi-compartmental model adequately describes the growth stoichiometry and quantifies sucrose translocation processes of Arabidopsis thaliana in light and dark conditions.\n\n\nASSAYS\nFlux Balance Analysis of multi-compartment metabolic model of growing Arabidopsis thaliana\nCREATORS\nMaksim Zakhartsev\nOlga Krebs\nIrina Medvedeva\nIlya Akberdin\nYuriy  Orlov\nINVESTIGATIONS\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nYuriy  Orlov\nIlya Akberdin\nIrina Medvedeva\nPROJECTS\nZucAt\nPUBLICATIONS\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nSTUDIES\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nKey Cell-Physiological Parameters(id: 30 | type: studies)\n\nDescription: How does the Volume, internal pH, membrane potential and intracellular cation content change in a time dependent scale during potassium limitation. \n\n\nASSAYS\nVolume determination during starvation at different times.\nTime courses of the internal pH changes\nHow does internal potassium change (/decrease) during several hours of potassium starvation.\nHow starvation affect protein content in yeast cells\nExternal Potassium Concentration\nExternal pH changes\nProtein Concentration\nMembrane Potential\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nTranscriptome data of Clostridium acetobutylicum growing at steady-state pH 5.7 and pH 4.5.(id: 833 | type: data_files)\n\nDescription: Micro array analysis of steady-state growing cells of Clostridium acetobutylicum at pH 5.7 (acidogenesis) in comparison to pH 4.5 (solventogenesis).\n\n\nASSAYS\nComparison of the transcriptome between pH 5.7 (acidogenesis) and pH 4.5 (solventogenesis)\nCREATORS\nHolger Janssen\nINVESTIGATIONS\nThe effect of pH upon the metabolic shift in Clostridium acetobutylicum in continuous culture.\nPEOPLE\nHolger Janssen\nPROJECTS\nCOSMIC\nPUBLICATIONS\nIntegrative modelling of pH-dependent enzyme activity and transcriptomic regulation of the acetone-butanol-ethanol fermentation of\n              Clostridium acetobutylicum\n              in continuous culture\nSTUDIES\nEffect of pH upon the transcriptome\nSUBMITTER\nHolger Janssen\n\n____________________________________________________________________________\n\nIan Booth(id: 40 | type: people)\n\nDescription: My research interests are in the physiology of bacteria subjected to stress.  The focus of my recent research has been the structure and function of regulated transport systems and ion channels involved in cellular homeostasis.  These transporters and channels respond to specific signals by a change in activity that either corrects the imposed stress or protects the cell during exposure to the stress.  Our systems biology interests are in the interplay of different enzymes systems and transporters in generating signals and in the behaviour of single cells in stress responses rather than cell populations. \n\n\nINSTITUTIONS\nUniversity of Aberdeen\nPROJECTS\nKOSMOBAC\n\n____________________________________________________________________________\n\nLEANPROT(id: 33 | type: projects)\n\nDescription: Escherichia coli is a well-established and the most widely used organism for the production of recombinant proteins (used in medical and industrial applications, as molecular biology reagents, etc.). Production of proteins is the most resource exhaustive process for the cells and therefore needs to be optimized to achieve maximal productivities. Natural environment of E. coli is much harsher compared to the near optimal growth conditions used in production processes. In order to survive cells produce many native proteins that could be considered unnecessary for the cells in industrial production conditions. In this project we aim to remove the most resource exhaustive unnecessary proteins from the host cells to free up resources for recombinant protein production. We will focus on this by using a novel metabolic modeling approach with constraints of protein production and cell geometry together with proteomics-based host cell physiology characterization. Novel lean-proteome strains with removed unnecessary proteins will be tested for the improved production capacity of several recombinant proteins used in research, industry and diagnostics.\n\n\nINSTITUTIONS\nNorwegian University of Science and Technology\nTechnische Universitt Berlin (TU Berlin)\nLatvia University of Agriculture\nCompetence Center of Food and Fermentation Technologies\nbiotechrabbit GmbH\nORGANISMS\nEscherichia coli\nPEOPLE\nTrygve Brautaset\nKlaus Bensch\nRaivo Vilu\nEgils Stalidzans\nPeter Neubauer\nKarl Peebo\nAgris Pentjuss\nSebastian Hans\nAgnieszka Gawin\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nAbsolute intracellular concentrations of metabolites extracted from T. b. brucei 427 wild type(id: 926 | type: data_files)\n\nDescription: Intracellular concentrations of 50 metabolites measured by LC-MS using isotope ratio based mass spectrometry technique\n\n\nASSAYS\nLC-MS based absolute quantification of intracellular metabolites\nCREATORS\nDong-Hyun Kim\nINVESTIGATIONS\nMetabolite profiling, quantification and flux quantification\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSTUDIES\nMetabolic flux measurement\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nSemanticSBML: a tool for annotating, checking, and merging of biochemical models in SBML format(id: 30 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nFalko Krause\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nDefining the robust behaviour of the plant clock gene circuit with absolute RNA timeseries and open infrastructure(id: 306 | type: publications)\n\nDescription: missing\n\n\nMODELS\nArabidopsis clock model P2011.1.2\nP2011.1.2_directupload\nPEOPLE\nAndrew Millar\nPROJECTS\nMillar group\nSTUDIES\nArabidopsis clock models P2011.1.2 and P2011.2.1 - PLM_71\n\n____________________________________________________________________________\n\nTranscription of glucose uptake transport systems in a mutant background under anaerobic conditions.(id: 1072 | type: data_files)\n\nDescription: Transcription of glucose uptake transport systems in a mutant background under anaerobic batch conditions.\n\n\nASSAYS\nAerobic and anaerobic characterization of MG1655 and mutant strains with regard to transcription profiles.\nCREATORS\nSonja Steinsiek\nINVESTIGATIONS\nAnalysis of the glucose transport in Escherichia coli under aerobic and anaerobic conditions\nPEOPLE\nSonja Steinsiek\nPROJECTS\nSUMO\nPUBLICATIONS\nGlucose Transport in Escherichia coli Mutant Strains with Defects in Sugar Transport Systems\nSTUDIES\nCharacterization of mutant strains with defects in sugar transport systems\nSUBMITTER\nSonja Steinsiek\n\n____________________________________________________________________________\n\nFigure 3A: Model calibration with experimental data of JAK2-STAT5 signaling obtained by different experimental techniques.(id: 148 | type: studies)\n\nDescription: For all experiments, primary CFU-E cells were starved and stimulated with 5 U/ml Epo. At the indicated time points, samples were subjected to quantitative immunoblotting. Experimental data (black circles) with estimated standard errors and trajectories of the best fit (solid lines) are represented. Mass spectrometry data represent replicates of four independent experiments.\n\n\nASSAYS\nSource data for Figure 3A: Experimental quantitative immunoblotting data.\nModel for JAK2/STAT5 signaling\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nDATA_FILES\nSource data for Figure 3A: Experimental quantitative immunoblotting data.\nINVESTIGATION\nBachmann et al (2011) Molecular Systems Biology 7:516\nMODELS\nJAK2/STAT5 model\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nMolecular Systems Biology\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation(id: 53 | type: presentations)\n\nDescription: Presented by Tobias Czauderna from Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) in Gatersleben at the SysMO PALs Meeting Heidelberg,29-11-2012\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\r\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nSustained glycolytic oscillations in individual isolated yeast cells - II(id: 83 | type: models)\n\nDescription: Using optical tweezers to position yeast cells in a microfluidic chamber, we were able to observe sustained oscillations in individual isolated cells. Using a detailed kinetic model for the cellular reactions, we simulated the heterogeneity in the response of the individual cells, assuming small differences in a single internal parameter. By operating at two different flow rates per experiment, we observe four of categories of cell behaviour. The present model (gustavsson2) predicts the damped oscillation to limit cycle transition observed in 13.6% of the cell population.\n\n\nASSAYS\nModelling sustained glycolytic oscillations in individual isolated yeast cells (Gustavsson models)\nCREATORS\nFranco Du Preez\nJacky Snoep\nINVESTIGATIONS\nYeast Glycolytic Oscillations\nPEOPLE\nJacky Snoep\nFranco Du Preez\nPROJECTS\nSysMO DB\nPUBLICATIONS\nSustained glycolytic oscillations in individual isolated yeast cells\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction\nSTUDIES\nSustained glycolytic oscillations in individual isolated yeast cells\nSUBMITTER\nFranco Du Preez\n\n____________________________________________________________________________\n\nMembrane Potential(id: 82 | type: assays)\n\nDescription: To estimate the relative level of plasma-membrane potential in the wild type, single trk1 and trk2 or double trk1 trk2 mutants, the diSC3(3) fluorescent probe was added to cells grown under various conditions and the levels of cell fluorescence were monitored. \n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nKey Cell-Physiological Parameters\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nArabidopsis - starch and the circadian clock, Model 2 (Seaton et al., 2013) - PLM_78, version 1, SIMPLIFIED(id: 376 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_78, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis - starch and the circadian clock, Model 2 (Seaton et al., 2013) - PLM_78, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nSeaton, Daniel\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis - starch and the circadian clock, Model 2 (Seaton et al., 2013) - PLM_78\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nTRK1,2 Transport Systems of Saccharomyces cerevisiae(id: 17 | type: investigations)\n\nDescription: Cellular parameters of strains lacking both TRK1 and TRK2 genes.\n\n\nASSAYS\nVolume\nInternal pH\nMembrane Potential\nProtein Concentration\nPotassium Concentration\nInsilico Promotor Anaysis\nCurrent voltage relation for different external KCl\nCurrent-voltage relation for different internal potassium\n2D Gel Analysis\nInteraction Database Analysis\nIn silico Promotor Anaysis\nModelling of the Trk-current\nDescribing of membrane potential changes in response to proton fluxes.\nDetermination of kinetic properties based on MIFE flux data and measurements of internal pH changes.\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDIES\nTranscriptional Profile\nKey Cell-Physiological Parameters  of TRK1,2 Transport Systems Investigation\nPromotor Anaysis\nCurrent-voltage relation\nProteomic analysis\nTranscriptomic Profiling of 14-3-3 Mutants\nGenetic analysis\nBioinformatic studies\nKinetic properties of Trk\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nTPI Kinetic characterisation(id: 196 | type: sops)\n\nDescription: SOP for measurement of TPI activity in extracts.\n\n\nASSAYS\nTPI\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, TOC1 degr D - PLM_90, version 1, SIMPLIFIED(id: 524 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_90, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, TOC1 degr D - PLM_90, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, TOC1 degr D - PLM_90\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nDevelopment of agent-based models describing the response of Escherichia coli to changes in oxygen availability(id: 18 | type: models)\n\nDescription: The agent-based model involves the representation of each individual molecule of interest as an autonomous agent that exists within the cellular environment and interacts with other molecules according to the biochemical situation. FLAME environmet has beem used for agent-based development. The FLAME framework is an enabling tool to create agent-based models that can be run on high performance computers (HPCs).  Models are created based upon extended finite state machines that include message input and output. This information is used by the frame work to automatically generate a simulation program that can run models  efficiently on HPCs.\r\nIn order to develop agent-based model in FLAME, the user need to define at least three xml  files as described below:\r\n\r\nModel.xml : X-Machine Agent Markup Modelling Language (XMML)\r\nA  description language for agent-based simulations. XMML should contain the whole structure of the  model\r\ni.e Agent descriptions, memory variables, functions, messages\r\n\r\nFunctions.c  should contain the implementations of the functions specified in Model.xml \r\n\r\n0.xml  should contain the initial states of the memory variables of the agents \r\ni.e Initializations of all parameters\r\n\r\nThe number of the resulting XML files depends on the number of iterations you specify to run your model (through Main.exe).\r\n\r\nEach important molecule is specified in the XMML language and information about its location (abstract co-ordinates), its state, its ability to interact with other molecules and the conditions under which this happens are encoded in its definition. \r\n\n\n\nCREATORS\nAfsaneh Maleki-Dizaji\nPEOPLE\nAfsaneh Maleki-Dizaji\nPROJECTS\nSUMO\nSUBMITTER\nAfsaneh Maleki-Dizaji\n\n____________________________________________________________________________\n\nSalazar2009_FloweringPhotoperiod - PLM_9, version 2, SIMPLIFIED(id: 502 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_9, version 2 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nSalazar2009_FloweringPhotoperiod - PLM_9, version 2\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nSalazar2009_FloweringPhotoperiod - PLM_9\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nGene position within a long transcript as a determinant for stochastic switching in bacteria(id: 103 | type: publications)\n\nDescription: missing\n\n\nDATA_FILES\nRNA-seq wild type\nPEOPLE\nOscar Kuipers\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\n\n____________________________________________________________________________\n\nTranscript profiling and inference of Escherichia coli K-12 ArcA activity across the range of physiologically relevant oxygen concentrations(id: 100 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nArcA phosphorylation at different aerobiosis levels (steady states)\nDATA_FILES\nSteady state transcriptional profiles at different oxygen concentrations\nMODELS\nTFInfer: a tool for probabilistic inference of transcription factor activities\nPEOPLE\nJeff Green\nJoost Teixeira De Mattos\nRobert Poole\nAlison Graham\nMatthew Rolfe\nAlexander Ter Beek\nPROJECTS\nSUMO\n\n____________________________________________________________________________\n\nGenetic or chemical protease inhibition causes significant changes in the Bacillus subtilis exoproteome(id: 63 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nSource data for Figure 3A: Experimental quantitative immunoblotting data.(id: 1329 | type: data_files)\n\nDescription: See 'Figure 3 legend' and 'Materials and methods - Time course experiments, cell lysis and quantitative immunoblotting' for details.\n\n\nASSAYS\nSource data for Figure 3A: Experimental quantitative immunoblotting data.\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nBachmann et al (2011) Molecular Systems Biology 7:516\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nMolecular Systems Biology\nSTUDIES\nFigure 3A: Model calibration with experimental data of JAK2-STAT5 signaling obtained by different experimental techniques.\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nSustained glycolytic oscillations in individual isolated yeast cells - I(id: 84 | type: models)\n\nDescription: Using optical tweezers to position yeast cells in a microfluidic chamber, we were able to observe sustained oscillations in individual isolated cells. Using a detailed kinetic model for the cellular reactions, we simulated the heterogeneity in the response of the individual cells, assuming small differences in a single internal parameter. By operating at two different flow rates per experiment, we observe four of categories of cell behaviour. The present model (gustavsson1) predicts the limit cycles observed in 16.7% of the cell population at both flow rates.\n\n\nASSAYS\nModelling sustained glycolytic oscillations in individual isolated yeast cells (Gustavsson models)\nCREATORS\nFranco Du Preez\nJacky Snoep\nINVESTIGATIONS\nYeast Glycolytic Oscillations\nPEOPLE\nJacky Snoep\nFranco Du Preez\nPROJECTS\nSysMO DB\nPUBLICATIONS\nSustained glycolytic oscillations in individual isolated yeast cells\nFrom steady-state to synchronized yeast glycolytic oscillations II: model validation\nFrom steady-state to synchronized yeast glycolytic oscillations I: model construction\nSTUDIES\nSustained glycolytic oscillations in individual isolated yeast cells\nSUBMITTER\nFranco Du Preez\n\n____________________________________________________________________________\n\nMeasured steady state concentrations of intracellular metabolites in yeast Saccharomyces cerevisiae anaerobic chemostat(id: 78 | type: data_files)\n\nDescription: Steady state concentrations of intracellular metabolites in yeast Saccharomyces cerevisiae anaerobic chemostat at D = 0.1 h-1 on minimal medium. All metabolite concentrations are in mmol/L(CV).\n\n\nASSAYS\nSteady state concentrations of intracellular metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nMOSES: dynamic model of glucose pulse\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nSteady state metabolic fluxes and metabolite concentrations in anaerobic chemostat of yeast Saccharomyces cerevisiae\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nPUBLICATIONS\nFast sampling for quantitative microbial metabolomics: new aspects on cold methanol quenching: metabolite co-precipitation\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSteady state concentrations of metabolites in yeast Saccharomyces cerevisiae in anaerobic chemostat\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\niKS1317.xml(id: 319 | type: models)\n\nDescription: The new GEM of S. coelicolor developed by Tjasa Kumelj, Snorre Sulheim, Alexander Wentzel and Eivind Almaas in 2017/2018\n\n\nCREATORS\nSnorre Sulheim\nTjasa Kumelj\nPEOPLE\nSnorre Sulheim\nTjasa Kumelj\nPROJECTS\nINBioPharm\nPUBLICATIONS\nPredicting strain engineering strategies using iKS1317: a genomescale metabolic model of Streptomyces coelicolor\nSUBMITTER\nSnorre Sulheim\n\n____________________________________________________________________________\n\nLF_proteomics_maxquant_perseus_unfiltered(id: 1803 | type: data_files)\n\nDescription: LFQ intensities from maxquant from combined search (mineral and continuous samples), log-transformed\n\n\nASSAYS\nProteomics\nCREATORS\nMalte Herold\nAnsgar Poetsch\nMohamed El Hajjami\nINVESTIGATIONS\nMulti -omics reveal lifestyle of acidophile, mineral-oxidizing model species Leptospirillum ferriphilumT\nPEOPLE\nMalte Herold\nAnsgar Poetsch\nMohamed El Hajjami\nPROJECTS\nSysMetEx\nSTUDIES\nOmics_data_analysis\nSUBMITTER\nMalte Herold\n\n____________________________________________________________________________\n\nHOTSOLUTE(id: 108 | type: projects)\n\nDescription: Background- Thermophilic organisms are composed of both bacterial and archaeal species. The enzymes isolated from these species and from other extreme habitats are more robust to temperature, organic solvents and proteolysis. They often have unique substrate specificities and originate from novel metabolic pathways. Thermophiles as well as their stable enzymes (thermozymes) are receiving increased attention for biotechnological applications.\r\n\r\nThe proposed project will establish thermophilic in vitro enzyme cascades as well as two new chassis, the thermophilic bacterium Thermus thermophilus (Tth, 65-75C, pH 7.0) and the thermoacidophilic archaeon Sulfolobus acidocaldarius (Saci, 75-80C, pH 2-4), as new thermophilic, bacterial and archaeal platforms for the production of novel high added-value products, i.e. extremolytes. Extremolytes are small molecular compatible solutes found naturally in the cells of thermophilic species that accumulate in the cell in response to multiple environmental stresses and stabilize cellular components (including proteins, membranes). Extremolytes offer an amazing so far unexploited potential for industrial applications including food, health, consumer care and cosmetics. However, their production in common mesophilic organisms (i.e. yeast, E. coli) is currently hampered by the hyperthermophilic origin of the respective metabolic pathways requiring a thermophilic cell factory.\n\n\nINSTITUTIONS\nUniversity Duisburg-Essen\nUniversity of Stellenbosch\nUniversitity Duisburg-Essen\nWinogradsky Institute of Microbiology, Federal Research Center Biotechnology\", Russian Academy of Sciences\nUniversity of Exeter\nEvonik Industries AG\nORGANISMS\nThermus thermophilus\nSulfolobus acidocaldarius\nEscherichia coli\nPEOPLE\nJacky Snoep\nBettina Siebers\nIlya Kublanov\nSimone Antonio De Rose\nSergey Gavrilov\nJennifer Littlechild\nChristina Stracke\nNicholas Harmer\nBenjamin Meyer\nFelix Mueller\nPROGRAMMES\nEra CoBioTech\n\n____________________________________________________________________________\n\nWetlab approach to transcription fidelity(id: 26 | type: investigations)\n\nDescription: Handling and manipulation of S. pneumoniae using molecular, cell biological and genetic tools.\n\n\nASSAYS\nParB-GFP ChIP on chip\nKinetics of misincorporation and proofreading by bacterial RNA polymerase\nRNA-Seq\nLive Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy.\nDATA_FILES\nChIP-chip ParB-GFP\nRNA-seq wild type\nRNA-seq delta greA\nPEOPLE\nJan-Willem Veening\nPROJECTS\nNoisy-Strep\nPUBLICATIONS\nSMC is recruited to oriC by ParB and promotes chromosome segregation in Streptococcus pneumoniae\nLive Cell Imaging of <em>Bacillus subtilis</em> and <em>Streptococcus pneumoniae</em> using Automated Time-lapse Microscopy\nControl of cell division in Streptococcus pneumoniae by the conserved Ser/Thr protein kinase StkP\nSingle cell analysis of gene expression patterns during carbon starvation in Bacillus subtilis reveals large phenotypic variation\nControl of transcription elongation by GreA determines rate of gene expression in Streptococcus pneumoniae\nSTUDIES\nChromosome segregation in S. pneumoniae\nInvestigation of bacterial transcription fidelity and processivity\nThe role of transcription factor GreA in transcription fidelity and processivity in Streptococcus pneumoniae\nAutomated time-lapse microscopy\nSUBMITTER\nJan-Willem Veening\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, TOC1 act D - PLM_92, version 1, SIMPLIFIED(id: 520 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_92, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, TOC1 act D - PLM_92, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, TOC1 act D - PLM_92\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nALD Kinetic characterisation(id: 195 | type: sops)\n\nDescription: SOP for measurement of ALD activity in extracts.\n\n\nASSAYS\nALD\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nSpontaneous spatiotemporal waves of gene expression from biological clocks in the leaf(id: 325 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nAndrew Millar\nPROJECTS\nMillar group\n\n____________________________________________________________________________\n\nAuxSim - PLM_27, version 1(id: 851 | type: assays)\n\nDescription: \tA cell-level model of the Arabidopsis root elongation zone. This spatial model is divided up into biological cells which are further divided into simulation boxes. The original model was designed to investigate how canal cells can accumulate auxin over time rather than to investigate the transport of auxin through the canal cells per se. The main outputs of the simulations in the original paper were the steady state ratios of auxin in the canal cell protoplasts to that in the parenchyma cell protoplasts for different cell sizes and AUX/LAX auxin carrier distributions.\tNote that the working version of the model is in the Simile (.sml) version rather than in XML (which is a temporary holding dummy file).&nbsp;\tThe publication is:\tKramer E.M. (2004) PIN and AUX/LAX proteins: their role in auxin accumulation. TRENDS in Plant Sciences, 9, 578-582 (+ supplementary material).<br><br><strong>Version Comments</strong></strong><p>\n\tSupplementary data files include PDF screenshot, SML model that works in Simile, configuration files to specify the tissue organisation, and example files to confirm correct operation.</p>\n<br><br><strong>Originally submitted to PLaSMo on 2011-01-07 21:54:18</strong></strong>\n\n\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATION\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDY\nAuxSim - PLM_27\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nSulfoSys(id: 11 | type: projects)\n\nDescription: Silicon cell model for the central carbohydrate metabolism of the archaeon Sulfolobus solfataricus under temperature variation\n\n\nASSAYS\nComparison of transcriptome of S. solfataricus at 70 and 80C\nComparison of proteome of S. solfataricus at 70 and 80C\nFermentation of S. solfataricus at 70 and 80C in a batch fermenter\nEnzyme activity tests for S. solfataricus\nIntracellular metabolomics of S. solfataricus at 70 and 80C\nPGK\nPGK Modelling\nGAPDH\nReconstituted Gluconeogenesis System\nModel Validation Gluconeogenesis in S. solfataricus\nGAPDH Modelling\nTPI\nTPI Modelling\nFBPAase\nFBPAase Modelling\nTemperature Degradation of Gluconeogenic Intermediates\nModelling Metabolite Degradation at High Temperature\nExperimental Validation Gluconeogenesis in S. solfataricus\nnon-enzymatic phosphoenolpyruvate (PEP) degradation at high temperature\nPGK 30C data\nPGK 30C model\nPGK 70 data\nBPG degradation\nPGK-GAPDH 30\nPGK-GAPDH 70\nBPG stability analysis\nPGK 70C model\nPGK - GAPDH models\nDATA_FILES\nPGK Kinetic Data\nPGK Kinetics Simulation\nFBPAase Kinetics Simulation\nFBPAase Kinetic data\nGAPDH Kinetic Data\nGAPDH Kinetics Simulation\nTPI Kinetic Data\nTPI Kinetics Simulation\nTemperature Degradation of Gluconeogenic Intermediates\nTemperature Degradation Kinetics Simulation\nExperimental Data for Reconstituted Gluconeogenic System\nSimulation for Reconstituted Gluconeogenic System\nPGK yeast with recycling of ATP\nPGK yeast without recycling of ATP\nSso PGK in presence of ATP recycling system at 70C, 3PG, BPG, PYR (enzymatic)\nSso PGK in absence of recycling system, 31P NMR data\nSso PGK (14 ug/ml) in presence of recycling system, 31P NMR data\nBPG degradation at 70\nPGK-GAPDH 30, 3PG, GAP & BPG\nPGK-GAPDH 70\nPGK-GAPDH 30, PYR\nSso PGK (28 ug/ml) in presence of recycling system, 31P NMR data\nINSTITUTIONS\nVU University Amsterdam\nUniversity of Amsterdam\nUniversity of Bergen\nUniversity of Braunschweig\nUniversity Duisburg-Essen\nUniversity of Groningen\nManchester Centre for Integrative Systems Biology, University of Manchester\nUniversity of Sheffield\nUniversity of Stavanger\nUniversity of Vienna\nWageningen University & Research\neGene Biotechnologie GmbH\nMax-Planck-Institute for terrestrial Microbiology\nINVESTIGATIONS\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature\nCentral Carbon Metabolism of Sulfolobus solfataricus\nPhosphoglycerate kinase acts as a futile cycle at high temperature\nMODELS\nModel for ED pathway in Sulfolobus solfataricus\nPGK Kinetic Model\nModel of reconstituted gluconeogenesis system in S. solfataricus\nFBPAase Kinetic Model\nGAPDH Kinetic Model\nTPI Kinetic Model\nKinetic Model of Temperature Degradation of Gluconeogenic Intermediates\nPGK yeast with/without recycling\nPGK yeast Fig1a\nBPG stability notebook\nPGK 70C model Fig. 3\nPGK 70C SBML\nPGK-GAPDH models yeast and Sulfolobus Fig. 4\nPGK-GAPDH model yeast kouril7\nPGK-GAPDH model Sulfolobus kouril8\nORGANISMS\nSulfolobus solfataricus\nPEOPLE\nJacky Snoep\nDietmar Schomburg\nHans Westerhoff\nPeter Ruoff\nRalf Steuer\nArnold Driessen\nBettina Siebers\nChrista Schleper\nJohn Van Der Oost\nNils-Kre Birkeland\nPhil Wright\nSonja-Verena Albers\nKieran Sharkey\nKerstin Schreiber\nAnita Elin-Fedoy\nSusanne Gertig\nPatrick Haferkamp\nAlexey Kolodkin\nTheresa Kouril\nAndrea Manica\nTrong Khoa Pham\nPawel Sierocinski\nMathias Von Jan\nPatricia Wieloch\nMelanie Zaparty\nHans-Peter Klenk\nPRESENTATIONS\nPublishing reproducible model construction and validation studies with SEEK and JWS Online\nPROGRAMMES\nSysMO\nPUBLICATIONS\nQuantitative proteomic analysis of Sulfolobus solfataricus membrane proteins\n\"Hot standards\" for the thermoacidophilic archaeon Sulfolobus solfataricus\nIntermediate instability at high temperature leads to low pathway efficiency for an in vitro reconstituted system of gluconeogenesis in Sulfolobus solfataricus\nSystems biology of the modified branched Entner-Doudoroff pathway in Sulfolobus solfataricus.\nPhosphoglycerate kinase acts as a futile cycle at high temperature.\nSOPS\nSulfosys - SOP proteomics\nSulfosys - SOP metabolomics\nSulfosys - SOP microarray\nSulfosys - SOP fermentation\nCloning and heterologous expression in E. coli\nProtein purification from E. coli extracts\nPreparation of cell extracts of recombinant E.coli\nSTUDIES\nPilot experiment - S. solfataricus grown at 70 and 80 C.\nModel Gluconeogenesis\nCarbon Loss at High Temperature\nPGK-30C\nPGK-70C\nBPG stability\nPGK-GAPDH 30C & 70C\n\n____________________________________________________________________________\n\nDynamics of macromolecules during glucose pulse(id: 84 | type: data_files)\n\nDescription: Dynamics of macromolecules (total RNA) during glucose pulse. Glucose pulse was performed in anaerobically growing yeast Saccharomyces cerevisiae in steady state chemostat (D = 0.1 h-1) and transent concentrations of the extra- and intracellular metabolites from central carbon metabolism (e.g. glycolysis, PPP, glycerol, purines, etc) were measured.\n\n\nASSAYS\nDynamics of macromolecules during glucose pulse\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nKinetic analysis of metabolic system using transient perturbations\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nMOSES\nSTUDIES\nMetabolic perturbation of the steady state culture by glucose pulse\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nRef2, a regulatory subunit of the yeast protein phosphatase 1, is a novel component of cation homoeostasis(id: 27 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoaquin Ario\nJose Ramos\nLina Patricia Barreto Parra\nAntonio Casamayor\nFalko Krause\nClara Navarrete\nPROJECTS\nTRANSLUCENT\n\n____________________________________________________________________________\n\nSulfoSys - Biotec(id: 16 | type: projects)\n\nDescription: Within the e:Bio - Innovationswettbewerb Systembiologie (Federal Ministry of Education and Research (BMBF)), the SulfoSYSBIOTECH consortium (10 partners), aim to unravel the complexity and regulation of the carbon metabolic network of the thermoacidophilic archaeon Sulfolobus solfataricus (optimal growth at 80C and pH 3) in order to provide new catalysts extremozymes for utilization in White Biotechnology.\r\n\r\nBased on the available S. solfataricus genome scale metabolic model (Ulas et al., 2012) the project will iteratively combine state of the art high throughput approaches (genomics, transcriptomics, proteomics, metabolomics), bioinformatics, modelling, data management and classical microbiology, biochemistry as well as genetic/ molecular biology techniques. The growth of S. solfataricus on alternative carbon sources (e.g. polymers, pentoses, sugar acids, aldehydes) will be analysed to elucidate the respective degradation pathways and to identify the involved enzymes; using this functional systems biology approach we expect to identify new enzymes of industrial interest. In collaboration with the industrial project partners target enzymes will be selected and screened for their potential for application (e.g. production efficiency). The initial focus will be on enzymes involved in the production of stereospecific base and fine chemicals and biomass conversion. The increasing knowledge of archaeal metabolic networks with respect to cellular blueprints, their complexity and regulation will be a major resource for a continuous stream of new applications.\r\n\n\n\nASSAYS\nRNA sequencing:l-fuc/d-glu\nintracellular metabolome analysis: l-fucose vs d-glucose\nProteome analysis: d-fuc / l-glu\nCell free extract activity measurements: L-fuc/d-glc\nMetabolic model of Sulfolobus solfataricus\nRNA sequencing: Casaminoacids vs D-glc\nProteome analysis: Casaminoacids versus D-Glc\nMetabolome analysis: Casaminoacids versus D-Glc\nMetabolic modelling of S. solfataricus during growth on casaminoacids\nnon-enzymatic phosphoenolpyruvate (PEP) degradation at high temperature\nPGK 30C data\nPGK 30C model\nPGK 70 data\nBPG degradation\nPGK-GAPDH 30\nPGK-GAPDH 70\nBPG stability analysis\nPGK 70C model\nPGK - GAPDH models\nDATA_FILES\nTranscriptomics L-fuc vs D-glc\nActivity assay L-fuc vs D-glc\nMetabolomics L-fuc vs D-glc\nProteomics L-fuc vs D-glc\nSSO Genome reference\nRNA-Seq raw data - Sulfolobus solfataricus P2 - casaminoacids and d-glucose\nRNA-Seq mappings - Sulfolobus solfataricus P2 - casaminoacids\nRNA-Seq mappings - Sulfolobus solfataricus P2 - d-glucose\nMetabolomics Casaminoacids vs Glc\nCalculation of amino acid uptake rates\nProteomics_Casaminoacids vs Glucose_Phenyx search output\nProteomics_Casaminoacids vs Glucose_results\nRaw MS data Casaminoacids vs glucose  - Main Zip archive\nRaw MS data Casaminoacids vs glucose - Supplementary file 1\nRaw MS data Casaminoacids vs Glucose - Supplementary file 2\nRaw MS data Casaminoacids vs Glucose - Supplementary file 3\nBiomass composition of S. solfataricus P2 grown on caseinhydrolysate\nDescription of changes made to the model compared to the first publication in 2012\nAmino acid depletion profile of S. solfataricus during growth on 1 % casaminoacids\nDistribution of enzymes involved in oxidative Stickland reactions among archaea\nnon-enzymatic phosphoenolpyruvate (PEP) degradation at 70C and 80C\nIntegrated summary of transcriptomics and proteomics results\nPGK yeast with recycling of ATP\nPGK yeast without recycling of ATP\nINSTITUTIONS\nVU University Amsterdam\nUniversity of Braunschweig\nUniversity Duisburg-Essen\nManchester Centre for Integrative Systems Biology, University of Manchester\nUniversity of Sheffield\nUniversity of Stellenbosch\nMax-Planck-Institute for terrestrial Microbiology\nOtto-von-Guericke University Magdeburg\nUniversity Bielefeld\nSigma Aldrich ( Swiss)\nOtto-von-Guericke-Universitt Magdeburg\nINVESTIGATIONS\nL-fucose degradation in Sulfolobus solfataricus P2\nAmino acid degradation in Sulfolobus solfataricus P2\nPhosphoglycerate kinase acts as a futile cycle at high temperature\nMODELS\nMetabolic model of Sulfolobus solfataricus\nMetabolic model of Sulfolobus solfataricus\nPGK yeast with/without recycling\nPGK yeast Fig1a\nBPG stability notebook\nPGK 70C model Fig. 3\nPGK 70C SBML\nPGK-GAPDH models yeast and Sulfolobus Fig. 4\nPGK-GAPDH model yeast kouril7\nPGK-GAPDH model Sulfolobus kouril8\nORGANISMS\nSulfolobus solfataricus\nPEOPLE\nJacky Snoep\nDietmar Schomburg\nHans Westerhoff\nBettina Siebers\nPhil Wright\nSonja-Verena Albers\nKerstin Schreiber\nTheresa Kouril\nTrong Khoa Pham\nStefan Albaum\nKatharina Kruse\nHelge Stark\nIda Schomburg\nJacqueline Wolf\nJrg Schaber\nRoland Wohlgemuth\nTobias Jakobi\nAntje Chang\nClaus-Rdiger Wallis\nAndreas Albersmeier\nSofia Figueiredo\nPROGRAMMES\ne:Bio\nPUBLICATIONS\nA systems biology approach reveals major metabolic changes in the thermoacidophilic archaeon Sulfolobus solfataricus in response to the carbon source L-fucose versus D-glucose\nOxidative Stickland reactions in an obligate aerobic organism - amino acid catabolism in the Crenarchaeon\n              Sulfolobus solfataricus\nSystems biology of the modified branched Entner-Doudoroff pathway in Sulfolobus solfataricus.\nPhosphoglycerate kinase acts as a futile cycle at high temperature.\nSOPS\nSulfoSYS_BIOTECH: RNA isolation and sequencing\nSULFOSYSbiotech_Proteome analysis\nSTUDIES\nComparison of S. solfataricus grown on l-fucose and d-glucose\nComparison of Sulfolobus solfataricus P2 grown on caseinhydrolysate and D-glucose\nPGK-30C\nPGK-70C\nBPG stability\nPGK-GAPDH 30C & 70C\n\n____________________________________________________________________________\n\nAre microRNAs key mediators of cartilage destruction in osteoarthritis?(id: 119 | type: projects)\n\nDescription: MicroRNAs are a recently discovered group of factors that control cell behaviour. There are thousands of microRNAs, but specific ones control the behaviour of cartilage cells and have key roles in osteoarthritis. Changing the level of a single microRNA can alter osteoarthritis, but we know that several microRNAs change during disease. The group has identified four microRNAs which have key roles in cartilage. This programme seeks:\r\n\r\nto show that these microRNAs have a role in osteoarthritis in the whole animal (not just the laboratory);\r\nto find the optimum way to deliver a microRNA to the joint to act as a therapy\r\nand to build a computer model of microRNA action in osteoarthritis to identify the outcomes of changing specific microRNAs therapeutically.\r\nThe programme of work aims to move the research towards clinical utility.\n\n\nINSTITUTIONS\nUniversity of Newcastle\nPEOPLE\nDaryl Shanley\nKrutik Patel\nPROGRAMMES\nNewcastle University Systems Modelling of Ageing\n\n____________________________________________________________________________\n\nFunctional analysis of the sortase YhcS in Bacillus subtilis(id: 154 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nLeif Steil\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\n2D-Gel Electrophoresis(id: 70 | type: assays)\n\nDescription: Is it possible to see changes in the proteome after starvation in 2D- Gels? Preparation of 2D Gels of cells incubated different periods of time in the absence of potassium.\n\n\nINVESTIGATION\nK+ Starvation in Saccharomyces cerevisiae\nPEOPLE\nFalko Krause\nPROJECTS\nTRANSLUCENT\nSTUDY\nProteomic Studies\nSUBMITTER\nFalko Krause\n\n____________________________________________________________________________\n\nENO Kinetic characterisation(id: 200 | type: sops)\n\nDescription: SOP for measurement of ENO activity in extracts.\n\n\nASSAYS\nENO\nCREATORS\nDawie Van Niekerk\nJacky Snoep\nINVESTIGATIONS\nGlucose metabolism in Plasmodium falciparum trophozoites\nPEOPLE\nJacky Snoep\nDawie Van Niekerk\nPROJECTS\nWhole body modelling of glucose metabolism in malaria patients\nSTUDIES\nModel construction\nSUBMITTER\nDawie Van Niekerk\n\n____________________________________________________________________________\n\nDomijanTS_AtClock2011 - PLM_50, version 2, SIMPLIFIED(id: 460 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_50, version 2 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nDomijanTS_AtClock2011 - PLM_50, version 2\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nDomijanTS_AtClock2011 - PLM_50\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nMonitoring of changes in the membrane proteome during stationary phase adaptation of Bacillus subtilis using in vivo labeling techniques(id: 53 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nintracellular metabolites LC/MS(id: 921 | type: data_files)\n\nDescription: intracellular metabolites measured by LC/MS\n\n\nASSAYS\nintracellular metabolites\nCREATORS\nHanna Meyer\nINVESTIGATIONS\nMultiomics study of Bacillus subtilis under osmotic stress\nPEOPLE\nHanna Meyer\nPROJECTS\nBaCell-SysMO\nSTUDIES\nB. subtilis_SysMo2_Chemostat_growthrate-salt\nSUBMITTER\nHanna Meyer\n\n____________________________________________________________________________\n\nThe JWS Online repository and modelling interface: SBGN schema generation, MIRIAM annotation and SBML model specification.(id: 45 | type: presentations)\n\nDescription: An overview of the JWS Online system, with emphasis on the latest model simulation interface and an update on the SBGN-ML and SED-ML support currently under development. \n\n\nEVENTS\nCOMBINE meeting\nPEOPLE\nFranco Du Preez\nPROJECTS\nSysMO DB\nSUBMITTER\nFranco Du Preez\n\n____________________________________________________________________________\n\nMetabolism of Streptomyces coelicolor (SysMO STREAM)(id: 16 | type: investigations)\n\nDescription: Antibiotics are made during the second phase of growth when there is a transition in metabolism from primary to secondary metabolism. Primary metabolism is growth related and involves all the normal cellular activities associated with cell growth and division. Whereas secondary metabolism is non-growth linked and is non-essential but many important activities occur during this phase which help the bacterium survive.\r\n\r\nOne of these activities is antibiotic production and is widespread in streptomycetes found in most soils. These bacteria have a fascinating life history and are abundant producers of biologically active compounds many of which have been exploited for their anti-tumour, anti-bacteria and anti-fungal activity.\r\n\r\nOur research will improve our understanding of how these bacteria regulate the transitions from primary to secondary metabolism and provide mathematical models to simulate the metabolic switch of life styles.\r\n\r\nThe approach is regarded as a systems analysis where a model is built of the whole metabolism and it can predict outcomes that will not have been determined previously by experimental methods. A range of modelling tools will be provided and these will be available for use with many other projects involved in a systems approach. The fuller understanding of the metabolic switch and the elucidation of how and why certain antibiotics are made under defined growth conditions will be vital for the exploitation of these bacteria. Many tools are available to manipulate bacterial genomes and with an understanding of the metabolism it will be possible to manipulate growth in order to produce novel antibiotics.\n\n\nASSAYS\nOnline/offline measurements\ntranscriptomics\nmetabolomics\nproteomics\nMetabolic pathway curation\nCREATORS\nAlexander Wentzel\nM. Tauqeer Alam\nSven Even Borgos\nRainer Breitling\nPer Bruheim\nNigel Burroughs\nGregory Challis\nLubbert Dijkhuizen\nTrond Ellingsen\nWilliam Gaze\nDavid Hodgson\nSunniva Hoel\nyvind Jakobsen\nR Jansen\nBrent Kiernan\nPreben Krabben\nRoxane Legaie\nJuan-Francisco Martin\nMaria Elena Merlo\nJay Moore\nKay Nieselt\nWalid Omara\nAnthony Palathingal\nDavid Rand\nJens Reuther\nAntonio Rodrguez-Garca\nHvard Sletta\nMaggie Smith\nMorris Swertz\nEriko Takano\nLouise Thomas\nElizabeth Wellington\nDavid Wild\nWolfgang Wohlleben\nAnders verby\nDATA_FILES\nFermentor 199 transcriptomics\nFermentor 199 cultivation data\nPEOPLE\nDavid Hodgson\nDavid Rand\nDavid Wild\nElizabeth Wellington\nEriko Takano\nGregory Challis\nJuan-Francisco Martin\nKay Nieselt\nLubbert Dijkhuizen\nMaggie Smith\nNigel Burroughs\nPreben Krabben\nRainer Breitling\nTrond Ellingsen\nWolfgang Wohlleben\nWalid Omara\nM. Tauqeer Alam\nWilliam Gaze\nyvind Jakobsen\nBrent Kiernan\nRoxane Legaie\nJay Moore\nPer Bruheim\nAnthony Palathingal\nR Jansen\nJens Reuther\nAntonio Rodrguez-Garca\nLouise Thomas\nHvard Sletta\nAnders verby\nSunniva Hoel\nSven Even Borgos\nMorris Swertz\nMaria Elena Merlo\nAlexander Wentzel\nPROJECTS\nSTREAM\nSYSTERACT\nPUBLICATIONS\nThe dynamic architecture of the metabolic switch in Streptomyces coelicolor\nA technical platform for generating reproducible expression data from Streptomyces coelicolor batch cultivations\nExtracting regulator activity profiles by integration of de novo motifs and expression data: characterizing key regulators of nutrient depletion responses in Streptomyces coelicolor\nMetabolic switches and adaptations deduced from the proteomes of Streptomyces coelicolor wild type and phoP mutant grown in batch culture\nThe PII protein GlnK is a pleiotropic regulator for morphological differentiation and secondary metabolism in Streptomyces coelicolor\nOptimized submerged batch fermentation strategy for systems scale studies of metabolic switching in Streptomyces coelicolor A3(2)\nIntracellular Metabolite Pool Changes in Response to Nutrient Depletion Induced Metabolic Switching in Streptomyces coelicolor\nSOPS\nPreparation of inoculum for Streptomyces coelicolor cultivation\nSampling from Streptomyces coelicolor culture for subsequent offline analyses\nCell dry weight (CDW) determination of Streptomyces coelicolor culture\nGeneration of cell material from Streptomyces coelicolor culture for proteome analysis\nGeneration of cell material from Streptomyces coelicolor culture for transcriptome analysis\nGeneration of cell material from Streptomyces coelicolor culture for metabolome analysis\nContamination test of Streptomyces coelicolor culture\nMCF derivatization\nMetabolome extraction\nSpore batch generation\nSTUDIES\nTimeseries 1\nScoCyc metabolic pathway curation\nSUBMITTER\nJay Moore\n\n____________________________________________________________________________\n\nComparison of proteome of S. solfataricus at 70 and 80C(id: 12 | type: assays)\n\nDescription: Samples obtained form the central fermentation facility of Sulfosys have been compared using iTRAQ (isobaric tag for relative and absolute quantification). A pilot experiment resulted in creation of SOP and initial data on cells grown at 70 and 80C\n\n\nINVESTIGATION\nAnalysis of Central Carbon Metabolism of Sulfolobus solfataricus under varying temperature\nORGANISMS\nSulfolobus solfataricus\nPEOPLE\nPawel Sierocinski\nPROJECTS\nSulfoSys\nSOPS\nSulfosys - SOP proteomics\nSTUDY\nPilot experiment - S. solfataricus grown at 70 and 80 C.\nSUBMITTER\nPawel Sierocinski\n\n____________________________________________________________________________\n\nStress-responsive systems set specific limits to the overproduction of membrane proteins in Bacillus subtilis(id: 83 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nUwe Voelker\nJan Maarten Van Dijl\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2011 - PLM_64, version 4, SIMPLIFIED(id: 430 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_64, version 4 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis_clock_P2011 - PLM_64, version 4\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis_clock_P2011 - PLM_64\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nData DNA microarray analyses of C. acetobutylcium trxB::int(29)(id: 1017 | type: data_files)\n\nDescription: WT (Cy3) vs. trxB (Cy5)_exp. growth phase\n\n\nCREATORS\nDaniel Hnicke\nPEOPLE\nDaniel Hnicke\nPROJECTS\nCOSMIC\nSUBMITTER\nDaniel Hnicke\n\n____________________________________________________________________________\n\npresentation Copenhagen Nov2012 SYSMO-LAB2 meeting(id: 76 | type: presentations)\n\nDescription: Koen van Grinsven\n\n\nCREATORS\nKoen Van Grinsven\nPEOPLE\nKoen Van Grinsven\nPROJECTS\nSysMO-LAB\nSUBMITTER\nKoen Van Grinsven\n\n____________________________________________________________________________\n\nDomijanTS_AtClock2011 - PLM_50, version 1, SIMPLIFIED(id: 458 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_50, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nDomijanTS_AtClock2011 - PLM_50, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nDomijanTS_AtClock2011 - PLM_50\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nSBEpo - Systems Biology of Erythropoietin(id: 54 | type: projects)\n\nDescription: Multi-level mathematical modeling of erythropoiesis for optimized expansion of erythroid progenitor cells and improved treatment regimes\r\n\r\n\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nUniversity Hospital Heidelberg\nRoche Diagnostics\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nWP2: Link of Signal Transduction to Gene Regulatory Network\nWP3: Cell Fate Decisions of Erythroid Progenitor Cells\nWP4: Cross-talk with Modulators - Epo with TGF and Novel Factors\nWP5: Translational Application for the Characterization and ex vivo Expansion of Human Erythroid Progenitor Cells\nData Management and Data Standardization\nORGANISMS\nHomo sapiens\nMus musculus\nPEOPLE\nWolfgang Mller\nMaja Rey\nSajib Chakraborty\nUlrike Wittig\nLorenz Adlung\nGeoffroy Andrieux\nHauke Busch\nMelania Barile\nMelanie Brries\nAndreas Franke\nClarissa Feuerstein\nBernhard Steiert\nMarcel Schilling\nPatricia Sauer\nRainer Saffrich\nChristian Tnsing\nMarie-Christine Wagner\nPROGRAMMES\ne:Bio\nPUBLICATIONS\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling\nEvaluation of GMP-compliant culture media for in vitro expansion of human bone marrow mesenchymal stromal cells\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation\nMicrocavity arrays as an in vitro model system of the bone marrow niche for hematopoietic stem cells\nData2Dynamics: a modeling environment tailored to parameter estimation in dynamical systems\nIdentification of isoform-specific dynamics in phosphorylation-dependent STAT5 dimerization by quantitative mass spectrometry and mathematical modeling\nMesenchymal stromal cells inhibit proliferation of virus-specific CD8(+) T cells\nStandardization of Good Manufacturing Practice-compliant production of bone marrow-derived human mesenchymal stromal cells for immunotherapeutic applications\nPlerixafor induces the rapid and transient release of stromal cell-derived factor-1 alpha from human mesenchymal stromal cells and influences the migration behavior of human hematopoietic progenitor cells\nDifferences between healthy hematopoietic progenitors and leukemia cells with respect to CD44 mediated rolling versus adherence behavior on hyaluronic acid coated surfaces\nFunctional potentials of human hematopoietic progenitor cells are maintained by mesenchymal stromal cells and not impaired by plerixafor\n[Mesenchymal stroma cells (MSCs) for the treatment of rheumatic disease]\nCovering a broad dynamic range: information processing at the erythropoietin receptor\nComputational processing and error reduction strategies for standardized quantitative data in biological networks\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nCause and cure of sloppiness in ordinary differential equation models\nIdentification of Cell Type-Specific Differences in Erythropoietin Receptor Signaling in Primary Erythroid and Lung Cancer Cells\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\nProtein abundance of AKT and ERK pathway components governs cell type-specific regulation of proliferation\n\n____________________________________________________________________________\n\nT2011 Ostreococcus clock, TOC1 degr L - PLM_89, version 2, SIMPLIFIED(id: 528 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_89, version 2 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nT2011 Ostreococcus clock, TOC1 degr L - PLM_89, version 2\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nT2011 Ostreococcus clock, TOC1 degr L - PLM_89\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nLactobacillus plantarum NC8 protein list(id: 1063 | type: data_files)\n\nDescription: Lactobacillus plantarum NC8 differentially expressed proteins, comparison between low and high growth rates\n\n\nCREATORS\nAnette McLeod\nPEOPLE\nAnette McLeod\nPROJECTS\nSysMO-LAB\nSUBMITTER\nAnette McLeod\n\n____________________________________________________________________________\n\nNoisy-Strep(id: 15 | type: projects)\n\nDescription: Bistable switches are the key elements of the regulatory networks governing cell development, differentiation and life-strategy decisions. Transcriptional noise is a main determinant that causes switching between different states in bistable systems. By using the human pathogen Streptococcus pneumoniae as a model bacterium, we will investigate how transcriptional fidelity and processivity influence (noisy) gene expression and participate in bistability. To study this question, we will use both natural and synthetic S. pneumoniae bistable switches as a highly sensitive probe for transcriptional noise. We will screen for mutations of the transcriptional apparatus that display altered bistability. A pure in vitro transcription system for S. pneumoniae will be set up and used to quantitatively characterize effects of these mutations on transcription. Detailed single-cell analysis using time-lapse microscopy will yield quantifiable data on the effects of the mutations on switching times and probabilities. Mathematical models that take transcription fidelity and processivity into account will be used to pinpoint parameters which most strongly affect the switching probabilities of our bistable networks. A global model encompassing all our in vivo and in vitro data will yield a high resolution systems-level understanding of the role of transcriptional noise in gene regulation of a human pathogen. Genetic and biochemical characterization of mutant RNAPs and/or accessory factors will yield molecular insights into the fundamental mechanisms of transcription. Furthermore, our results might lead to novel drug discovery projects specifically aimed to reduce or increase transcriptional noise to prevent unwanted development of pathogenic bacteria such as S. pneumoniae\n\n\nASSAYS\nParB-GFP ChIP on chip\nKinetics of misincorporation and proofreading by bacterial RNA polymerase\nRNA-Seq\nLive Cell Imaging of Bacillus subtilis and Streptococcus pneumoniae using Automated Time-lapse Microscopy.\nWhole genome sequencing (Illumina)\nDATA_FILES\nChIP-chip ParB-GFP\nRNA-seq wild type\nRNA-seq delta greA\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nAntibiotic-induced increase of origin-proximal gene copy numbers triggers bacterial competence\nWhole genome marker frequency analysis on strain MK350\nWhole genome marker frequency analysis on strain MK422\nWhole genome marker frequency analysis on strain MK423\nDCI15\nMK453\nMK454\nINSTITUTIONS\nUniversity of Groningen\nUniversity of Newcastle\nUniversity of Cologne\nINVESTIGATIONS\nWetlab approach to transcription fidelity\nModeling approach to transcription fidelity\nChromosome segregation and cell division in Streptococcus pneumoniae\nORGANISMS\nStreptococcus pneumoniae\nPEOPLE\nRobin Sorg\nJan-Willem Veening\nJohannes Berg\nNikolay Zenkin\nPamela Gamba\nYulia Yuzenkova\nPrasanna    Bhogale\nSulman Shafeeq\nPROGRAMMES\nSysMO\nPUBLICATIONS\nStepwise mechanism for transcription fidelity\nGene position within a long transcript as a determinant for stochastic switching in bacteria\nDnaA and ORC: more than DNA replication initiators\nTransformation of environmental Bacillus subtilis isolates by transiently inducing genetic competence\nCellular localization of choline-utilization proteins in Streptococcus pneumoniae using novel fluorescent reporter systems\nA mechanism for cell cycle regulation of sporulation initiation in Bacillus subtilis\nControlled interplay between trigger loop and Gre factor in the RNA polymerase active centre\nCentral role of the RNA polymerase trigger loop in intrinsic RNA hydrolysis\nSignificance analysis and statistical mechanics: an application to clustering\nSMC is recruited to oriC by ParB and promotes chromosome segregation in Streptococcus pneumoniae\nLive Cell Imaging of <em>Bacillus subtilis</em> and <em>Streptococcus pneumoniae</em> using Automated Time-lapse Microscopy\nA new basal promoter element recognized by RNA polymerase core enzyme\nFactor-independent transcription pausing caused by recognition of the RNA-DNA hybrid sequence\nIn vitro experimental system for analysis of transcription-translation coupling\nNonlinear fitness landscape of a molecular pathway\nControl of cell division in Streptococcus pneumoniae by the conserved Ser/Thr protein kinase StkP\nHypothesis: emergence of translation as a result of RNA helicase evolution\nMultiple active centers of multi-subunit RNA polymerases\nSingle cell analysis of gene expression patterns during carbon starvation in Bacillus subtilis reveals large phenotypic variation\nGraphAlignment: Bayesian pairwise alignment of biological networks\nBenchmarking various green fluorescent protein variants in Bacillus subtilis, Streptococcus pneumoniae, and Lactococcus lactis for live cell imaging\nControl of transcription elongation by GreA determines rate of gene expression in Streptococcus pneumoniae\nWhat makes the lac-pathway switch: identifying the fluctuations that trigger phenotype switching in gene regulatory systems\nSTUDIES\nChromosome segregation in S. pneumoniae\nInvestigation of bacterial transcription fidelity and processivity\nThe role of transcription factor GreA in transcription fidelity and processivity in Streptococcus pneumoniae\nAutomated time-lapse microscopy\nGenome wide marker frequency analysis on new strains to image genetic loci in Streptococcus pneumoniae\n\n____________________________________________________________________________\n\nThe unique trypanothione redox metabolism in the framework of the silicon trypanosome project.(id: 46 | type: presentations)\n\nDescription: Poster presented during the ICSB 2011 conference in Mannheim\n\n\nCREATORS\nAlejandro Leroux\nLuise Krauth-Siegel\nPEOPLE\nLuise Krauth-Siegel\nAlejandro Leroux\nPROJECTS\nSilicoTryp\nSUBMITTER\nAlejandro Leroux\n\n____________________________________________________________________________\n\nTroein Ostreococcus clock 1-loop - PLM_7, version 1, SIMPLIFIED(id: 540 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_7, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nTroein Ostreococcus clock 1-loop - PLM_7, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nTroein, Carl\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nTroein Ostreococcus clock 1-loop - PLM_7\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nArabidopsis_clock_extend - PLM_6, version 2, SIMPLIFIED(id: 420 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_6, version 2 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis_clock_extend - PLM_6, version 2\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nPokhilko, Alexandra\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis_clock_extend - PLM_6\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nTATAA biocenter workshop lecture \"Reverse Engineering Omics Data into Biological Networks\"(id: 535 | type: presentations)\n\nDescription: TATAA biocenter workshop lecture \n\n\nCREATORS\nAntoine Buetti-Dinh\nPEOPLE\nAntoine Buetti-Dinh\nPROJECTS\nSysMetEx\nSUBMITTER\nAntoine Buetti-Dinh\n\n____________________________________________________________________________\n\nMetabolites all experiments, relative measurements(id: 1726 | type: data_files)\n\nDescription: Contains:\r\n-Relative metabolite measurements at different time points from all experiments\r\n-Absolute metabolite measurements for amino-acid analysis of the proteome and the cytosol\r\n-Effect on adding CaCl2, KCl or NaCl to the medium on growth\r\n-Effect of spiking of growth medium with additional amino acids\r\n\r\n\n\n\nCREATORS\nNiels Zondervan\nLuis Serrano\nMaria Lluch\nEva Yus\nPEOPLE\nNiels Zondervan\nLuis Serrano\nMaria Lluch\nEva Yus\nPROJECTS\nMycoSynVac - Engineering Mycoplasma pneumoniae as a broad-spectrum animal vaccine\nPUBLICATIONS\nDissecting the energy metabolism in Mycoplasma pneumoniae through genome-scale metabolic modeling.\nLarge-scale metabolome analysis and quantitative integration with genomics and proteomics data in Mycoplasma pneumoniae.\nSUBMITTER\nNiels Zondervan\n\n____________________________________________________________________________\n\nRegulation of quinone detoxification by the thiol stress sensing DUF24/MarR-like repressor, YodB in Bacillus subtilis(id: 49 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\n\n____________________________________________________________________________\n\nProteome Reference maps of Clostridium acetobuytlicum growing at steady-state pH 5.7 and pH 4.5(id: 835 | type: data_files)\n\nDescription: All spots and identified proteins of Clostridium acetobutylicum growing at steady-state pH 5.7 (acidogenesis) and pH 4.5 (solventogenesis), respectively, using 2D PAGE and Maldi-TOF analysis. The focus were cytosolic proteins with an isoelectric point bewteen 4 and 7 as well as a molecular weight of 180-10 kDa.\n\n\nASSAYS\nComparison of the proteome between pH 5.7 (acidogenesis) and pH 4.5 (solventogenesis).\nCREATORS\nHolger Janssen\nINVESTIGATIONS\nThe effect of pH upon the metabolic shift in Clostridium acetobutylicum in continuous culture.\nPEOPLE\nHolger Janssen\nPROJECTS\nCOSMIC\nPUBLICATIONS\nIntegrative modelling of pH-dependent enzyme activity and transcriptomic regulation of the acetone-butanol-ethanol fermentation of\n              Clostridium acetobutylicum\n              in continuous culture\nSTUDIES\nEffect of pH on the proteome\nSUBMITTER\nHolger Janssen\n\n____________________________________________________________________________\n\nFAIRDOMhub: Managing data from Instrument to publication.(id: 120 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nCaterina Barillari\nNatalie Stanford\nRostyslav Kuzyakiv\nEVENTS\n1st EraSysApp PALs meeting\nPEOPLE\nStuart Owen\nOlga Krebs\nNatalie Stanford\nRostyslav Kuzyakiv\nCaterina Barillari\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nData DNA microarray analyses C. acetobutylicum::trxB(29)(id: 1019 | type: data_files)\n\nDescription: WT (Cy3) vs. trxB (Cy5)_stationary growth phase\n\n\nCREATORS\nDaniel Hnicke\nPEOPLE\nDaniel Hnicke\nPROJECTS\nCOSMIC\nSUBMITTER\nDaniel Hnicke\n\n____________________________________________________________________________\n\nGlutamate metabolism in Bacillus subtilis: gene expression and enzyme activities evolved to avoid futile cycles and to allow rapid responses to perturbations of the system(id: 52 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJoerg Stuelke\nUwe Voelker\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nSystems Understanding of Microbial Oxygen- Dependent and Independent Catabolism (SUMO2 ): A coordinators overview(id: 82 | type: presentations)\n\nDescription: Presentation by Robert Pool at the SysMO Conference (Feb 2013 in Berlin)\n\n\nCREATORS\nRobert Poole\nEVENTS\nSysMO Evaluation 2013\nPEOPLE\nRobert Poole\nMichael Ederer\nPROJECTS\nSUMO\nSUBMITTER\nMichael Ederer\n\n____________________________________________________________________________\n\narabidopsis_clock_biopepa - PLM_47, version 1, SIMPLIFIED(id: 416 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_47, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\narabidopsis_clock_biopepa - PLM_47, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nGuerriero, Maria-Luisa\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\narabidopsis_clock_biopepa - PLM_47\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nPSYSMO(id: 6 | type: projects)\n\nDescription: Systems analysis of process-induced stresses: towards a quantum increase in process performance of Pseudomonas putida as the cell factory of choice for white biotechnology.\r\n\r\nThe specific goal of this project is to exploit the full biotechnological efficacy of Pseudomonas putida KT2440 by developing new optimization strategies that increase its performance through a systems biology understanding of key metabolic and regulatory parameters that control callular responses to key stresses generated during biotechnological application of this versatile bacterium.\n\n\nINSTITUTIONS\nCSIC Granada\nCSIC Madrid\nHelmholtz Centre for Infection Research Braunscheig\nImperial College London\nMax Planck Institute for Dynamics of Complex Technical Systems\nUniversity of Braunschweig\nUniversity of Duesseldorf\nUniversity of Dortmund\nUniversity of Essex\nUniversity of Hannover\nManchester Centre for Integrative Systems Biology, University of Manchester\nUniversity of Stuttgart\nQueen's University Belfast\nORGANISMS\nPseudomonas putida\nPEOPLE\nErnst-Dieter Gilles\nAndreas Kremling\nJacky Snoep\nJohn Hallsworth\nMax Schobert\nAlfonso Valencia\nAndreas Schmid\nAndrew Livingston\nBurkhard Tuemmler\nDieter Jahn\nDietmar Schomburg\nEduardo Diaz\nEfstratios Pistikoupoulos\nHans Westerhoff\nJuan Ramos\nKarl-Erich Jaeger\nKenneth Timmis\nManuel Ferrer\nVictor De Lorenzo\nVitor Martins Dos Santos\nMatthias Reuss\nDaniel Samaga\nKieran Sharkey\nHeike Bauer\nPrashanth Bhaganna\nLars Blank\nBruno Buehler\nThorben Dammeyer\nEstrella Duque\nIsabel Fernndez\nSarah Frank\nHorst Funken\nIsam Haddad\nChristian Jaeger\nJens Klockgether\nMichalis Koutinas\nCarolyn Lam\nAntje Lemke\nSakis Mantalaris\nPaul McDermott\nTerry McGenity\nGloria Navarro-Aviles\nJuan Nogales\nAuxi Prieto\nBlair Prochnow\nJose Rodriguez-Herva\nFrank Rosenau\nKerstin Schreiber\nMartin Siemann-Herzberg\nChristoph Ulmer\nIgnacio Poblete Castro\nAlexandros Kiparissides\nPROGRAMMES\nSysMO\nPUBLICATIONS\nGenome-scale reconstruction and analysis of the Pseudomonas putida KT2440 metabolic network facilitates applications in biotechnology\nFunctional analysis of new transporters involved in stress tolerance inPseudomonas putidaDOT-T1E\nIdentification and characterization of the PhhR regulon inPseudomonas putida\nThe metabolic response of P. putida KT2442 producing high levels of polyhydroxyalkanoate under single- and multiple-nutrient-limited growth: highlights from a multi-level omics approach\nPseudomonas putida KT2440 genome update by cDNA sequencing and microarray transcriptomics\nFunctional genomics of the initial phase of cold adaptation of Pseudomonas putida KT2440\n\n____________________________________________________________________________\n\nData DNA microarray analyses C. acetobutylicum trxB::int(29)(id: 1020 | type: data_files)\n\nDescription: WT (Cy5) vs. trxB (Cy3)_stationary growth phase_DyeFlip\n\n\nCREATORS\nDaniel Hnicke\nPEOPLE\nDaniel Hnicke\nPROJECTS\nCOSMIC\nSUBMITTER\nDaniel Hnicke\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides(id: 608 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nWolfgang Mller\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nPEOPLE\nWolfgang Mller\nMaja Rey\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nConcentrations of extracellular metabolites from T. b. brucei 427(id: 1021 | type: data_files)\n\nDescription: Concentrations of 22 extracellular metabolites (major medium components) from T. b. brucei 427\n\n\nASSAYS\nLC-MS based absolute quantification of extracellular metabolites\nCREATORS\nDong-Hyun Kim\nINVESTIGATIONS\nMetabolite profiling, quantification and flux quantification\nPEOPLE\nDong-Hyun Kim\nPROJECTS\nSilicoTryp\nSTUDIES\nMetabolic flux measurement\nSUBMITTER\nDong-Hyun Kim\n\n____________________________________________________________________________\n\nGel-based proteomics of Gram-positive bacteria: a powerful tool to address physiological questions(id: 56 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\nArabidopsis_clock_extend - PLM_6, version 1, SIMPLIFIED(id: 418 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_6, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis_clock_extend - PLM_6, version 1\nCREATORS\nBioData SynthSys\nINVESTIGATIONS\nPokhilko, Alexandra\nPEOPLE\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis_clock_extend - PLM_6\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nBalancing the osmotic gradient: adaptation of the cytoplasmic solute pool via de novo synthesis and compatible solute uptake in response to hyper-osmolarity.(id: 3 | type: presentations)\n\nDescription: The talk shows that proline biosynthesis and accumulation is connected to glycine betaine availability. Glycine betaine complements proline function.\n\n\nCREATORS\nTamara Hoffmann\nEVENTS\nBacell-meeting 1109-Groningen\nPEOPLE\nUlf Liebal\nTamara Hoffmann\nPROJECTS\nBaCell-SysMO\nSUBMITTER\nUlf Liebal\n\n____________________________________________________________________________\n\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.(id: 102 | type: presentations)\n\nDescription: Presentation by Carole Goble at the International Semantic Web Conference 2013 in Sydney Australia about the use of semantic web technology in SEEK and RightField.\n\n\nCREATORS\nStuart Owen\nCarole Goble\nKaty Wolstencroft\nJacky Snoep\nWolfgang Mller\nOlga Krebs\nQuyen Nguyen\nPEOPLE\nJacky Snoep\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nQuyen Nguyen\nPROJECTS\nSysMO DB\nPUBLICATIONS\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nICYSB 2015 - International Practical Course in Systems Biology(id: 27 | type: projects)\n\nDescription: Systems Biology studies the properties and phenotypes that emerge from the interaction of biomolecules where such properties are not obvious from those of the individual molecules. By connecting fields such as genomics, proteomics, bioinformatics, mathematics, cell biology, genetics, mathematics, engineering and computer sciences, Systems Biology enables discovery of yet unknown principles underlying the functioning of living cells. At the same time, testable and predictive models of complex cellular pathways and eventually of whole cells and organisms are generated as tools for research. ICYSB will provide a hands-on experience in integrating experimentation and mathematical modelling with an emphasis on kinetic modelling of cellular pathways. The course focuses on yeast because the development of the emerging field of Systems Biology requires suitable model systems for data generation.\n\n\nDATA_FILES\nList of Participants\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nImperial College London\nHumboldt-Universitt zu Berlin\nVU University Amsterdam\nUniversity of Goettingen\nManchester Centre for Integrative Systems Biology, University of Manchester\nUniversity of Newcastle\nSchool of Computer Science, University of Manchester\nOtto-von-Guericke University Magdeburg\nETH Zurich\nUniversity of Luxembourg\nUniversity of Gothenburg\nUniversitt Konstanz\nOkayama University\nBuenos Aires\nUniversity of Milano-Bicocca\nUniversity of Tbingen\nUni Basel\nbo Akademi University\nInstitute of Cytology and Genetics\nInstitute for Experimental Internal Medicine\nInstitute for Biomechanics\nUniversity of Tokyo\nMax-Planck Institute for Molecular Genetics\nInstitut Curie\nStockholm University\nRadboud University Nijmegen\nInstitute of Pathology\nIFIBYNE - Physiology, Molecular Biology  and Neurosciences\nMODELS\niNS142 RobOKoD Redesigned Butanol Producing.\nORGANISMS\nSaccharomyces cerevisiae\nPEOPLE\nStuart Owen\nOlga Krebs\nWolfgang Mller\nQuyen Nguyen\nNatalie Stanford\nJrg Schaber\nRostyslav Kuzyakiv\nICYSB  Participant\nAnnika Hantusch\nKoji Ishikawa\nEszter Lakatos\nCecilia Brnnmark\nPaula Dunayevich\nAstrid Wachter\nRaffaele Nicastro\nPhillipp Schmidt\nThomas Hentrich\nArnaud Monnard\nCarolyn Alia Joko\nAlexandra Klimenko\nCiaran Welsh\nAnastasiya Lapytsko\nShinsuke Ohnuki\nScott Lovell\nSeverin Ehret\nBarbara Mlody\nKathiresan Natarajan\nDaniel Morgan\nMaaruthy Yelleswarapu\nSimone Keller\nYuan Qixia\nALEJANDRO  COLMAN-LERNER\nMarija  Cvijovic\nMarcus Krantz\nStefan Hohmann\nAriane Scheuren\nNiek Welkenhuysen\nPROGRAMMES\nIndependent Projects\nPUBLICATIONS\nModelling reveals novel roles of two parallel signalling pathways and homeostatic feedbacks in yeast\nNested autoinhibitory feedbacks alter the resistance of homeostatic adaptive biochemical networks\n\n____________________________________________________________________________\n\n091016 quinones complete data magdeburg(id: 65 | type: data_files)\n\nDescription: his document contains data for quinone concentrations and ratios of ubichinon and menachinon.These data have been generated for analysis of MG1655 at steady-state conditions in Infors-Multifors-Bioreactors. All measurements have been done in Amsterdam at the group of Prof. Joost Teixeira de Mattos.\r\n\r\nDuplicates of samples for quinones were taken after 20 hours of steady state conditions. Chemostat experiments were carried out in Infors-Bioreactors at 0, 20, 50, 80, 100 and 120% aerobiosis. All data for concentrations are shown in [mol / g DCW]\n\n\nASSAYS\nDetermination of intracellular redox state by means of ubiquinones (oxd/red-ratio)\nCREATORS\nStefan Stagge\nAlexander Ter Beek\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nPEOPLE\nStefan Stagge\nAlexander Ter Beek\nPROJECTS\nSUMO\nSTUDIES\nThe Escherichia coli steady state response to oxygen: from molecular interactions to regulatory modules\n\n____________________________________________________________________________\n\nDiamide triggers mainly S Thiolations in the cytoplasmic proteomes of Bacillus subtilis and Staphylococcus aureus(id: 80 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMichael Hecker\nUwe Voelker\nMichael Lalk\nPROJECTS\nBaCell-SysMO\n\n____________________________________________________________________________\n\niRhythmics(id: 28 | type: projects)\n\nDescription: The project addresses the generation and establishment of programmed pacemaker cells for an in vitro drug testing possibility to perform predictive tests. This may lead to an improved treatment of cardiac arrhythmias or an accurate identification of potential drug molecules at a very early stage of development. Important benefits will arise in verifying the safety of a wide variety of medicines while reducing animal testing.\r\n\r\nThis project is funded by the European Social Fund (ESF) program of the European Union (ESF/14-BM-A55-0027).  \n\n\nINSTITUTIONS\nUniversity of Rostock\nRostock University Medical Centre\nUniversity Medicine of Greifswald\nLeibniz Institute for Farm Animal Biology (FBN)\nORGANISMS\nHomo sapiens\nMus musculus\nPEOPLE\nOlaf Wolkenhauer\nMarkus Wolfien\nAli Salehzadeh-Yazdi\nJulien Sos\nAnne-Marie Galow\nAndreas Hflich\nHeiko Lemcke\nAnna Skorska\nSophie Kussauer\nStephan Marquardt\nPROGRAMMES\nIndependent Projects\n\n____________________________________________________________________________\n\nPresentation Serve Kengen(id: 89 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nServ Kengen\nEVENTS\nCOSMIC Final Meeting 2013\nPEOPLE\nServ Kengen\nSebastian Curth\nPROJECTS\nCOSMIC\nSUBMITTER\nSebastian Curth\n\n____________________________________________________________________________\n\nFAIRDOM user meeting(id: 51 | type: projects)\n\nDescription: Good data and model management improves the longevity and impact of your interdisciplinary research. FAIRDOM offers software and expertise to support you in better managing your interdisciplinary life-science projects, particularly in systems and synthetic biology. If you have never heard of data and model management, or are curious about it, or you are an expert keen to exchange ideas, our user meeting is the place for you!\r\n\r\nAt the meeting you will:\r\n\r\n - Learn why data and model management is important, and how it can improve the longevity and impact of your research.\r\n - Learn how other research groups formulate their data and model management using FAIRDOM software.\r\n - Have direct contact with the FAIRDOM Community and Tech Team, who can give you personalised advice on your data and model management, and using FAIRDOM.\r\n- Learn how users have extended the platforms to include e.g. electronic lab notebooks.\r\n - Be among the first people to hear about how to publish citable data and models using FAIRDOM!\r\n\r\nWorkshop agenda \r\n\r\n\r\n\r\n09:00  09:10\tWelcome\r\nExploring data and model management\r\n09:10  10:40\tInvited Talk: Gareth Knight on Rewarding open science practice in funded research\r\nResearch Data Manager, London School of Hygiene & Tropical Medicine\r\n09:40  10:10\tFAIRDOM introduction, Carole Goble\r\nThe University of Manchester, UK\r\n10:10  10:25\tFlash talks  3 minutes one slide (slots still available  register here)\r\nAlexey Kolodkin  Dynamic network modelling: from experimental data to design principles and personalised therapies\r\nNikolaus Sonnenschein  DD-DeCaF: Bioinformatics Services for Data-Driven Design of Cell Factories and Communities\r\nMarie Vidal  Upcoming CORBEL Open Call for advanced research projects\r\nIvelize Rocha Bernardo  Sharing and Reusing Spreadsheets through a Discovery of a Common Vocabulary/Schema\r\n10:25  10:35\tMartin Golebiewski\r\n10:35  11:00\tCoffee\r\nStar use cases\r\n11:00  11:15\tSiberian SEEK, Sergey Lashin, Institute of Cytology and Genetics SB RAS, Russia\r\n11:15  11:30\tSynthSys, Tomasz Zielinski, University of Edinburgh, UK\r\n11:30  11:45\tDigital Salmon, Jon Olav Vik, Norwegian University of Life Sciences, Norway\r\n11:45  12:00\tIMOMESIC, Martin Bhm, DKFZ, Heidelberg, Germany\r\n12.00  12.15\tLiSyM, Matthias Knig, Charite  Universittsmediz Berlin, Germany\r\n12:15  12:30\tMOSES, ZucAt and ExtremoPharm, Maksim Zakhartsev, Hohenheim University, Stuttgart, Germany\r\n12:30  13:00\tDiscussion\r\n13:00  13:45\tLunch\r\nParallel Session\r\n13:45  15:15\t\r\nFAIRDOM Training and helpdesk\r\nGroup discussions\r\n15:15  15:45\tcoffee\r\nLooking to the future\r\n15:45  16:15\tInvited talk: Yannick Wurm\r\nSenior Lecturer in Bioinformatics, Queen Mary University London, UK\r\n16:15  16:45\tPublishing a FAIR study with citable research assets, Jacky Snoep\r\nStellenbosch University, South Africa\r\n16:45  17:00\tWrap up\r\n18:00\tGroup dinner\r\n \r\n\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nGeneral Transcriptomics Templates\nProteomics Templates (Mass spectrometry)\nMetabolomics Master Template\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nNimbleGen Transcriptomics Templates\nProteomics Template (gel electrophoresis)\nHow to use templates to structure experimental data\nDATA_FILES\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nImperial College London\nVU University Amsterdam\nUniversity of Bergen\nUniversity of Leiden\nManchester Centre for Integrative Systems Biology, University of Manchester\nNorwegian University of Life Sciences\nUniversity of Edinburgh\nGerman Cancer Research Center (DKFZ)\nLund University\nLatvia University of Agriculture\nFAIRDOM User meeting\nSYSBIO - Centre of Systems Biology\nInstitute of Cytology and Genetics SB RAS\nHumboldt University Berlin\nUnicamp - University of Campinas\nMDC/BIMSB\nCSIC\nCentre for Digital Life Norway\nIDIBAPS\nIDAEA-CSIC\nVall d'Hebron Institut de Recerca (VHIR)\nPeter MacCallum Cancer Centre\nStandigm\nTianjin Institute of Industrial Biotechnology, Chinese Academy of Sciences\nInstitute of Microbiology and Biotechnology, University of Latvia\nNazarbayev University\nKAIST\nNovo Nordisk Foundation Center for Biosustainability, Technical University of Denmark\nZonMw\nUPF\nCaltech\nJAPAN TOBACCO INC\nIDIBELL\nCEIT\nNational Institute of Biology\nLeibniz Institute for Natural Product Research and Infection Biology - Hans Knll Institute (HKI)\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nORGANISMS\nHomo sapiens\nPEOPLE\nCarole Goble\nStuart Owen\nOlga Krebs\nAlexey Kolodkin\nNatalie Stanford\nRostyslav Kuzyakiv\nEgils Stalidzans\nJon Olav Vik\nMartin Golebiewski\nMatthias Knig\nMartin Bhm\nMassimiliano Borsani\nSergey Lashin\nTomasz Zielinski\nIvelize Rocha Bernardo\nFatemeh Zamanzad Ghavidel\nMarie Vidal\nJake Pitt\nRune Kleppe\nEkaterina Kotelnikova\nIgor Marn\nMarc Torrent\nAnna Trigos\nSang Ok Song\nIgor Goryanin\nTorsten Reimer\nNazgul Sakenova\nKatharina Heil\nMireia Arnedo\nJae Il Joo\nNikolaus Sonnenschein\nKristina Gruden\nSoyeong Jang\nAnne Boeter\nLeticia Galera\nSahand Hormoz\nGabriel Teku\nHiroaki Sasai\nLuis Palomero\nIigo Apaolaza\nTest Person\nDino Van Dissel\nThomas Wolf\nPRESENTATIONS\nInstructions for hands on session - Using SEEK to manage your research assets\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\nA major transition in social evolution & some data tidbits\nSiberian SEEK: Multiscale modeling of microbial communities\nFAIRDOM Introduction\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nPROGRAMMES\nFAIRDOM\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nCreating Templates for Proteomics\nExperimental study for hands on session\nTyrosine hydroxylase model\n\n____________________________________________________________________________\n\nSalazar2009_FloweringPhotoperiod - PLM_9, version 1, SIMPLIFIED(id: 500 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_9, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nSalazar2009_FloweringPhotoperiod - PLM_9, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nSalazar2009_FloweringPhotoperiod - PLM_9\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nZucAt: FBA solution of the model under dark growth conditions(id: 1297 | type: data_files)\n\nDescription: The solution of Flux Balance Analysis (FBA) represent metabolic flux distribution in the model under dark growth conditions (i.e. constraints)\n\n\nASSAYS\nFlux Balance Analysis of multi-compartment metabolic model of growing Arabidopsis thaliana\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nZucAt\nSTUDIES\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\nModeling the cell division cycle: cdc2 and cyclin interactions(id: 251 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nBIOMD0000000005 - Tyson1991 - Cell Cycle 6 var\nDATA_FILES\nSEDML for BIOMD0000000005\nMODELS\nBIOMD0000000005 from BioModels Database\nPEOPLE\nRon Henkel\nPROJECTS\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nUpdating the metabolic model of the Arabidopsis Framework Model, visit report(id: 473 | type: presentations)\n\nDescription: Outline report of joint research conducted during MSBnet-funded visit of Sanu Shameer to Millar lab\n\n\nCREATORS\nAndrew Millar\nSanu Shameer\nArgyris Zardilis\nPEOPLE\nAndrew Millar\nSanu Shameer\nArgyris Zardilis\nPROJECTS\nMillar group\nSUBMITTER\nAndrew Millar\n\n____________________________________________________________________________\n\nZucAt: FBA solution of the model under light growth conditions (V)(id: 1397 | type: data_files)\n\nDescription: The solution of Flux Balance Analysis (FBA) represents metabolic flux distribution in ZucAt model under light growth conditions. In this solution, (i) the ratio photorespiration / photosynthesis has been fixed to 0.25; and (ii) cyclic electron flow through FQR (ferredoxin-plastoquinone reductase) has been set 0.1 from non-cyclic flow through FRN (ferredoxin-NADP oxidoreductase). Under this constraints, ATP formed by non-cyclic photophosphorylation is not sufficient to fulfill ATP/NADPH ratio for carbon fixation, therefore plastid imports ATP from cytoplasm. Flux through FRN represents non-cyclic electron flow through the light photosynthesis system.\n\n\nASSAYS\nFlux Balance Analysis of multi-compartment metabolic model of growing Arabidopsis thaliana\nCREATORS\nMaksim Zakhartsev\nINVESTIGATIONS\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nPEOPLE\nMaksim Zakhartsev\nPROJECTS\nZucAt\nSTUDIES\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nSUBMITTER\nMaksim Zakhartsev\n\n____________________________________________________________________________\n\n\"Hot standards\" for the thermoacidophilic archaeon Sulfolobus solfataricus(id: 113 | type: publications)\n\nDescription: missing\n\n\nASSAYS\nEnzyme activity tests for S. solfataricus\nPEOPLE\nDietmar Schomburg\nArnold Driessen\nBettina Siebers\nChrista Schleper\nJohn Van Der Oost\nPhil Wright\nSonja-Verena Albers\nKerstin Schreiber\nSusanne Gertig\nPatrick Haferkamp\nTheresa Kouril\nAndrea Manica\nTrong Khoa Pham\nPawel Sierocinski\nMathias Von Jan\nPatricia Wieloch\nMelanie Zaparty\nHans-Peter Klenk\nPROJECTS\nSulfoSys\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2011 - PLM_64, version 2, SIMPLIFIED(id: 426 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_64, version 2 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis_clock_P2011 - PLM_64, version 2\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis_clock_P2011 - PLM_64\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nNAD + Replenishment Improves Lifespan and Healthspan in Ataxia Telangiectasia Models via Mitophagy and DNA Repair(id: 393 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMarianne B. Nilsen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nRightField: Semantic Enrichment of Systems Biology Data using Spreadsheets(id: 61 | type: presentations)\n\nDescription: This talk was presented at the IEEE e-Science conference, Chicago, USA (8-12 October 2012).\r\nhttp://www.ci.uchicago.edu/escience2012/\r\nIt describes the latest developments in RightField for both semantic data annotation and extraction to RDF.\n\n\nCREATORS\nKaty Wolstencroft\nStuart Owen\nCarole Goble\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nArabidopsis_clock_P2011 - PLM_64, version 1, SIMPLIFIED(id: 424 | type: models)\n\nDescription: Simplified model file for PLaSMo accession ID PLM_64, version 1 (use simplified if your software cannot read the file, e.g. Sloppy Cell)\n\n\nASSAYS\nArabidopsis_clock_P2011 - PLM_64, version 1\nCREATORS\nBioData SynthSys\nAndrew Millar\nINVESTIGATIONS\nMillar, Andrew\nPEOPLE\nAndrew Millar\nBioData SynthSys\nPROJECTS\nPlaSMo model repository\nSTUDIES\nArabidopsis_clock_P2011 - PLM_64\nSUBMITTER\nBioData SynthSys\n\n____________________________________________________________________________\n\nSUMO(id: 3 | type: projects)\n\nDescription: \"Systems Understanding of Microbial Oxygen responses\" (SUMO) investigates how Escherichia coli senses oxygen, or the associated changes in oxidation/reduction balance, via the Fnr and ArcA proteins, how these systems interact with other regulatory systems, and how the redox response of an E. coli population is generated from the responses of single cells. There are five sub-projects to determine system properties and behaviour and three sub-projects to employ different and complementary modelling approaches using published data sets and data emerging from our own work. We construct increasingly elaborate models of the system at different levels of detail, which are used to generate new hypotheses and influence further experimental design.\n\n\nASSAYS\nAnalysis of gene expression rates at different aerobiosis levels via RT-PCR\nAnalysis of by-product formation rates in MG1655\nFNR activity at different aerobiosis levels (steady state)\nDetermination of intracellular metabolite concentrations\nPhysiological measurements from Sheffield chemostat\nTranscriptional profiling of E. coli during anaerobic to aerobic and aerobic to anaerobic transitions\nTranscriptional profiling of steady states at different aerobiosis levels\nDetermination of intracellular redox state by means of NAD/NADH ratio\nDetermination of intracellular redox state by means of ubiquinones (oxd/red-ratio)\nMeasurement of cytochrome numbers\nKinetic modelling of Escherichia coli's electron transport chain\nKinetic modelling of Escherichia coli's electron transport chain coupled to a simplified growth model\nCharacterization of E. coli MG1655 and sdhC and frdA isogenic mutant strains in batch growth\nSteady State Oxygen Response of E. coli WT and two Electron Transport Chain Mutants\nAerobic and anaerobic characterization of MG1655 and mutant strains with regard to growth, by-product formation and cAMP concentrations.\nCharacterization of MG1655 and mutant strains under conditions of glucose excess and limitation.\nAerobic and anaerobic characterization of MG1655 and mutant strains with regard to transcription profiles.\nLiterature Data from Alexeeva et al., J. Bacteriol.,  2000, 2002, 2003\nArcA phosphorylation at different aerobiosis levels (steady states)\nTFinfer2\nDetermination of by-product formation and glucose uptake of mutants with linear respiratory chain at different aerobiosis levels\nGene expression analysis of mutants with linear electron transport chain at different aerobiosis levels\nDeternination of ArcA phosphroylation level in mutants with linear ETC at different defined aerobiosis levels\nDATA_FILES\nFNR activity at different aerobiosis levels RECALIBRATED .csv\nFNR reporter activity at different aerobiosis levels RECALIBRATED .pdf\nDissolved oxygen tension aerobic to anaerobic .pdf\nDissolved oxygen tension anaerobic to aerobic .pdf\nEntire transcriptional dataset - aerobic to anaerobic transition .csv\nEntire transcriptional dataset - anaerobic to aerobic .csv\nFiltered datset - aerobic to anaerobic transition .csv\nFiltered dataset - anaerobic to aerobic .csv\nSheffield chemostat parameters .csv\nSheffield chemostat parameters .pdf\nSteady state transcriptional profiles at different oxygen concentrations\nSheffield steady state cytochrome measurements\nSheffield steady state cytochrome measurements\n090714 chemostat overview fluxes & quinones wt\n091016 quinones complete data magdeburg\n091016 quinones complete data magdeburg\nNumeric Data from Alexeeva et al., J. Bacteriol.,  2000, 2002, 2003\nSimulation results of local sensitivity analysis for an Escherichia coli electron transport chain model.\nMap with simulation results\nGrowth, by-product formation and glucose consumption rates and cAMP-concentrations under aerobic conditions\nGrowth, yield and cAMP excretion under anaerobic conditions\nTranscription of glucose uptake transport systems in a mutant background under aerobic conditions\nTranscription of glucose uptake transport systems in a mutant background under anaerobic conditions.\nTranscription of glucose transport systems under batch versus chemostat conditions.\nArcA phosphorylation levels at different aerobiosis units\nETCmutants_metabolites\nETC_mutants_ArcA\nGene expression levels in mutants with linear respiratory chain\nETCmutants_mRNA\nGene expression rates at different aerobiosis levels via RT-PCR\nINSTITUTIONS\nMax Planck Institute for Dynamics of Complex Technical Systems\nUniversity of Amsterdam\nUniversity of Sheffield\nUniversity of Stuttgart\nUniversity of Edinburgh\nINVESTIGATIONS\nSteady state studies for different oxygen availability in Escherichia coli\nDynamical studies for different oxygen availability in Escherichia coli\nSystems Analysis of Anaerobic Systems Supplemented with Alternative Electron Acceptors\nAnalysis of the glucose transport in Escherichia coli under aerobic and anaerobic conditions\nModel prediction of FNR levels with different oxygen availability in E.coli Cyo/Cyd mutants\nAnalysis of Escherichia coli with linear electron transport chain\nAnalysis of population heterogeneity in chemostat cultures with limited oxygen supply\nMODELS\nBasic kinetic model of Escherichia coli's electron transport chain (ETC)\nDevelopment of agent-based models describing the response of Escherichia coli to changes in oxygen availability\nA Mathematical Model of Metabolism and Regulation Provides a Systems-Level View of the Oxygen Response of Escherichia coli\nKinetic model of Escherichia coli's electron transport chain (ETC) coupled to a simplified growth model\nCompartimented ODE model of oxygen diffusion near E. coli cells\nPDE model of oxygen diffusion near E. coli cells\nTFInfer: a tool for probabilistic inference of transcription factor activities\nAgent-based model of response of E.coli to different levels of oxygen supply (with FNR and ArcBA)\nTFinfer2\nORGANISMS\nEscherichia coli\nEscherichia coli K-12\nPEOPLE\nErnst-Dieter Gilles\nJeff Green\nJoost Teixeira De Mattos\nKatja Bettenbrock\nMichael Holcombe\nOliver Sawodny\nRobert Poole\nThomas Sauter\nMartijn Bekker\nAlison Graham\nMatthew Rolfe\nMichael Ederer\nSebastian Henkel\nStefan Stagge\nSonja Steinsiek\nAfsaneh Maleki-Dizaji\nAlexander Ter Beek\nTacita Nye\nMaikel Verouden\nWenjing Jia\nDavid Knies\nHao Bai\nGuido Sanguinetti\nBotond Cseke\nPoonam Sharma\nSamantha Nolan\nFrank Bruggerman\nKlaas Hellingwerf\nJohan  Van Beilen\nPRESENTATIONS\nReporting on genes, transcription factors and metabolites\nSystems Understanding of Microbial Oxygen- Dependent and Independent Catabolism (SUMO2 ): A coordinators overview\nModels of Redox and Carbon Metabolism in Escherichia coli: Integrating Different Types of Data\nPROGRAMMES\nSysMO\nPUBLICATIONS\nTFInfer: a tool for probabilistic inference of transcription factor activities\nTranscript profiling and inference of Escherichia coli K-12 ArcA activity across the range of physiologically relevant oxygen concentrations\nCharacterization of E. coli MG1655 and frdA and sdhC mutants at various aerobiosis levels\nSystems analysis of transcription factor activities in environments with stable and dynamic oxygen concentrations\nReprogramming of Escherichia coli K-12 metabolism during the initial phase of transition from an anaerobic to a micro-aerobic environment\nGlucose Transport in Escherichia coli Mutant Strains with Defects in Sugar Transport Systems\nModelling complex biological systems using an agent-based approach\nThe O2 sensitivity of the transcription factor FNR is controlled by Ser24 modulating the kinetics of [4Fe-4S] to [2Fe-2S] conversion\nBacterial sensors of oxygen\nUncoupling of substrate-level phosphorylation in Escherichia coli during glucose-limited growth\nOn the function of the various quinone species in\n              Escherichiacoli\nAnalysis of Escherichia coli mutants with a linear respiratory chain\nKinase activity of ArcB from Escherichia coli is subject to regulation by both ubiquinone and demethylmenaquinone\nTowards a Systems Level Understanding of the Oxygen Response of Escherichia coli\nBasic Regulatory Principles of Escherichia coli's Electron Transport Chain for Varying Oxygen Conditions\nA mathematical model of metabolism and regulation provides a systems-level view of how Escherichia coli responds to oxygen\nAgent-based modeling of oxygen-responsive transcription factors in Escherichia coli\nSOPS\nWriting SUMO SOPs\nSUMO RNA isolation from E. coli\nSUMO Transcriptional profiling of E. coli\nSUMO B-galactosidase assay\nSimons method for making monolysogenic lacZ operon fusions on the E. coli chromosome\nQuantifying cytochromes\nSUMO mRNA isolation Epicentre\nSUMO chemostat conditions\nDCW measurement\nEvans Medium\nMeasurement of extracellular metabolites\nQ/QH2 sampling\nQuenching and Extraction of intracellular metabolites\nGene doctoring method of making chromosomal fnr mutants\nDetermination of protein phosphorylation levels using Phos-tag gels\nSTUDIES\nThe Escherichia coli dynamical response to oxygen: from molecular interactions to regulatory modules\nThe Escherichia coli steady state response to oxygen: from molecular interactions to regulatory modules\nQuantitative analysis of catabolic carbon and electron fluxes in E. coli: impact of TCRS on the efficiency of energy conservation and physiology\nDetermination of the impact of specific enzyme reactions and regulatory proteins using isogenic E. coli mutant strains\nBasic regulatory principles of Escherichia colis electron transport chain for varying oxygen conditions in glucose-limited continuous cultures\nCharacterization of mutant strains with defects in sugar transport systems\nAnalysis of Escherichia coli strains with linear respiratory chain\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T13:18:45.274Z"
     },
     {
      "end_time": "2019-03-14T15:06:05.159Z",
      "execution_time": "10.5s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading Completed\n\n25 relationships found\nLoading Completed\n\nSubstituting relationships into original search results: \nLoading 88.24%\n\n                                        --SEARCH RESULTS--\n\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T15:05:54.659Z"
     },
     {
      "end_time": "2019-03-14T15:16:18.898Z",
      "execution_time": "6.49s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nLoading Completed\n\n25 relationships found\n"
       },
       {
        "ename": "TypeError",
        "evalue": "parallelRequest() got multiple values for argument 'requestPerThread'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     74\u001b[0m         \u001b[0;31m# Create a paralelized request for each relationship in the search results\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     75\u001b[0m         \u001b[0mPS\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mParallelSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 76\u001b[0;31m         \u001b[0mPS\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelationshipList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnumberOfRelationshipsNoDuplicates\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m20\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelationshipsPerThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     77\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     78\u001b[0m         \u001b[0;31m# Wait for the requests to finish in order to continue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: parallelRequest() got multiple values for argument 'requestPerThread'"
        ]
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T15:16:12.407Z"
     },
     {
      "end_time": "2019-03-14T15:38:22.823Z",
      "execution_time": "12.5s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n"
       },
       {
        "ename": "NameError",
        "evalue": "name 'numberOfRelationshipsNoDuplicates' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     66\u001b[0m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mremoveDuplicateRelationships\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     67\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 68\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\\n\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mnumberOfRelationshipsNoDuplicates\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\" relationships found\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     69\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     70\u001b[0m         \u001b[0;31m# Create a paralelized request for each relationship in the search results\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mNameError\u001b[0m: name 'numberOfRelationshipsNoDuplicates' is not defined"
        ]
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T15:38:10.332Z"
     },
     {
      "end_time": "2019-03-14T15:39:02.804Z",
      "execution_time": "10.8s",
      "outputs": [
       {
        "ename": "KeyboardInterrupt",
        "evalue": "",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36m_input_request\u001b[0;34m(self, prompt, ident, parent, password)\u001b[0m\n\u001b[1;32m    728\u001b[0m             \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 729\u001b[0;31m                 \u001b[0mident\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mreply\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstdin_socket\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    730\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mException\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/jupyter_client/session.py\u001b[0m in \u001b[0;36mrecv\u001b[0;34m(self, socket, mode, content, copy)\u001b[0m\n\u001b[1;32m    802\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 803\u001b[0;31m             \u001b[0mmsg_list\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msocket\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv_multipart\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmode\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    804\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mzmq\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mZMQError\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/zmq/sugar/socket.py\u001b[0m in \u001b[0;36mrecv_multipart\u001b[0;34m(self, flags, copy, track)\u001b[0m\n\u001b[1;32m    465\u001b[0m         \"\"\"\n\u001b[0;32m--> 466\u001b[0;31m         \u001b[0mparts\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrecv\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mflags\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcopy\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mcopy\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtrack\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mtrack\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    467\u001b[0m         \u001b[0;31m# have first part already, only loop while more to receive\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket.Socket.recv\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket.Socket.recv\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32mzmq/backend/cython/socket.pyx\u001b[0m in \u001b[0;36mzmq.backend.cython.socket._recv_copy\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/zmq/backend/cython/checkrc.pxd\u001b[0m in \u001b[0;36mzmq.backend.cython.checkrc._check_rc\u001b[0;34m()\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: ",
         "\nDuring handling of the above exception, another exception occurred:\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-3-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     31\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     32\u001b[0m         \u001b[0;31m# Begin the search by inputing the search term\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 33\u001b[0;31m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchTerm\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0minput\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Enter your search: \\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     34\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     35\u001b[0m         \u001b[0;31m# Choose the category of search\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36mraw_input\u001b[0;34m(self, prompt)\u001b[0m\n\u001b[1;32m    702\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_parent_ident\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    703\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_parent_header\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 704\u001b[0;31m             \u001b[0mpassword\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mFalse\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    705\u001b[0m         )\n\u001b[1;32m    706\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/ipykernel/kernelbase.py\u001b[0m in \u001b[0;36m_input_request\u001b[0;34m(self, prompt, ident, parent, password)\u001b[0m\n\u001b[1;32m    732\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    733\u001b[0m                 \u001b[0;31m# re-raise KeyboardInterrupt, to truncate traceback\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 734\u001b[0;31m                 \u001b[0;32mraise\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    735\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    736\u001b[0m                 \u001b[0;32mbreak\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
        ]
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T15:38:51.985Z"
     },
     {
      "end_time": "2019-03-14T15:39:26.579Z",
      "execution_time": "11.7s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n"
       },
       {
        "ename": "NameError",
        "evalue": "name 'numberOfRelationshipsNoDuplicates' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-5-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     66\u001b[0m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mremoveDuplicateRelationships\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     67\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 68\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\\n\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelationshipList\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\" relationships found\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     69\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     70\u001b[0m         \u001b[0;31m# Create a paralelized request for each relationship in the search results\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mNameError\u001b[0m: name 'numberOfRelationshipsNoDuplicates' is not defined"
        ]
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T15:39:14.830Z"
     },
     {
      "end_time": "2019-03-14T15:40:25.763Z",
      "execution_time": "45.6s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n\n557 relationships found\nLoading Completed\n\nSubstituting relationships into original search results: \nLoading 99.58%\n\n                                        --SEARCH RESULTS--\n\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\nMESI-STRAT Coordinator\nLeader WP8  Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universitt Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hnfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universitt zu Berlin\nUniversity of Freiburg\nCharit University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nMatthias Knig\nJohannes Bausch\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nKollevg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nOliver Ebenhh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\n\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Mller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhh\nDorota Kurek\nUte Spiekerktter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\n\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias Knig\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias Knig\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Brries\nBernhard Steiert\nChristian Tnsing\nUte Spiekerktter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\n\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\nMy current research deals with the integration of different metabolic pathways. Research lines include:\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\n- The relation between metabolism and biologial ageing;\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nCharit University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias Knig\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias Knig\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Sderstrm\nLibe Aranguren Abada\nSiri fsthus Goksyr\nOdd Andr Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\n\n<b>Topics include:</b>\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\n- Comprehensive and unambiguous data description and annotation\n- Standard formats and terminologies for data and metadata \n- Guidelines for reporting data (Minimum information standards) \n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\n- Using SEEK & FAIRDOMhub for integrated data management\n\n<b>Registration:</b> http://bit.ly/2ArOC7w\n(please state if you will attend on both days or only on one day)\n\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\n\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hnfeld will be refunded.\n\n<b>Organization:</b>\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\n\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\n\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nqPCR liver_Kollevg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nErik Schrder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nPAH metabolites in bile Kollevg\nSediment chemistry\nLiver chemistry_Kollevg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLeibniz-Institut fr Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\n\nMore information available from\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nUte Spiekerktter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUrsula Klingmller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nPablo Jos Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dchert\nMartin Bhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nTBARS_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemnde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP  EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemnde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\nSiberian SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias Knig\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Bhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Bhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Bhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nMartin Bhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nUrsula Klingmller\nJens Timmer\nMartin Golebiewski\nMartin Bhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1 axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigacin Prncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversit catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias Knig\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nTBARS_Kollevg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seil Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\n\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \n\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1g total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\n\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\n\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nMatthias Knig(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\nRead more about the LiSyM junior group at: www.livermetabolism.com\n\nJunior Group Leader\nDr. Matthias Knig\n\nInstitute for Theoretical Biology\nHumboldt-University Berlin\nInvalidenstrae 43, 10117 Berlin, Germany\nphone +49 30 2093-8450\nkonigmatt@googlemail.com\n\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\n\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universitt zu Berlin\nCharit University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias Knig\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias Knig\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias Knig\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\n\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \n\nKey questions:\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\n\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\n\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\n\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharit University Medicine Berlin\nGriffin Discoveries BV\nUniversittsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Bhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndr Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29  30 November 2012, Heidelberg\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevg biometrics\nqPCR ovaries_Kollevg\nqPCR liver_Kollevg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \n19th-20th of March, 2012, Vienna, Austria\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Mller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nHumboldt-Universitt zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias Knig\nJudith Wodke\nClemens Khn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nWolfgang Mller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\n\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\n\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Mller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Mller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hnicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Mller\nUrsula Klingmller\nJens Timmer\nMartin Golebiewski\nMartin Bhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Mller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hnfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR  The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universitt zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universitt Dresden\nGerman Cancer Research Center (DKFZ)\nCharit University Medicine Berlin\nHumboldt University Berlin\nUniversitt Leipzig - Medizinische Fakultt - Institut fr Biochemie\nLeibniz-Institut fr Arbeitsforschung (IfADo)\nUniversittsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Mller\nMaja Rey\nUrsula Klingmller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias Knig\nMartin Bhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schrder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schnfels\nPRESENTATIONS\nPAL report  Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlsung frSpezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\n\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmons ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmons body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\n\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and fed in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seil Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Rsg\nSahar Hassani\nMarius Andr Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Description: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Mller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen fr die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T15:39:40.140Z"
     },
     {
      "end_time": "2019-03-14T16:08:18.317Z",
      "execution_time": "11.9s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found \nLoading Completed\n"
       },
       {
        "ename": "TypeError",
        "evalue": "must be str, not int",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     61\u001b[0m             \u001b[0mthread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     62\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 63\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\\n\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestFails\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\" ommited results\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     64\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     65\u001b[0m         \u001b[0;31m# Get the total number of places where there is a relationship\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: must be str, not int"
        ]
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T16:08:06.462Z"
     },
     {
      "end_time": "2019-03-14T16:08:48.842Z",
      "execution_time": "10.9s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found \nLoading Completed\n"
       },
       {
        "ename": "TypeError",
        "evalue": "must be str, not int",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-3-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     61\u001b[0m             \u001b[0mthread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     62\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 63\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\\n\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestFails\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\" ommited results\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     64\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     65\u001b[0m         \u001b[0;31m# Get the total number of places where there is a relationship\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: must be str, not int"
        ]
       }
      ],
      "source": "s.search()",
      "start_time": "2019-03-14T16:08:37.935Z"
     }
    ],
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Enter your search: \n",
      "liver\n",
      "Please enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n",
      "\n",
      "135 results found \n",
      "Loading Completed\n"
     ]
    },
    {
     "ename": "TypeError",
     "evalue": "must be str, not int",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-3-c30eb7094d43>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0ms\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/Search.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     61\u001b[0m             \u001b[0mthread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     62\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 63\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\\n\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestFails\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\" ommited results\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     64\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     65\u001b[0m         \u001b[0;31m# Get the total number of places where there is a relationship\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: must be str, not int"
     ]
    }
   ],
   "source": [
    "s.search()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
