{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "provenance": [
     {
      "last_modified": "2019-03-17T16:36:08.563Z",
      "source": "#                                                Using SEEK API for writing reports"
     },
     {
      "last_modified": "2019-03-17T17:26:32.294Z",
      "source": "#                                                Using SEEK API for writing reports"
     },
     {
      "last_modified": "2019-03-17T17:57:03.473Z",
      "source": "#                                                Using SEEK API for writing reports"
     },
     {
      "last_modified": "2019-03-17T19:22:38.936Z",
      "source": "#                                                Using SEEK API for writing reports"
     },
     {
      "last_modified": "2019-03-17T19:31:23.871Z",
      "source": "#                                                Using SEEK API for writing reports"
     }
    ]
   },
   "source": [
    "#                                                Using SEEK API for writing reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecutionTime": {
     "end_time": "2019-03-17T20:46:45.429291Z",
     "start_time": "2019-03-17T20:46:45.315105Z"
    },
    "provenance": [
     {
      "end_time": "2019-03-17T19:25:38.315Z",
      "execution_time": "73ms",
      "outputs": [
       {
        "ename": "ImportError",
        "evalue": "cannot import name 'auth'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-3-94c59809e4fe>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mauth\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mImportError\u001b[0m: cannot import name 'auth'"
        ]
       }
      ],
      "source": "from SEEK import SEEK, auth",
      "start_time": "2019-03-17T19:25:38.242Z"
     },
     {
      "end_time": "2019-03-17T19:25:43.248Z",
      "execution_time": "94ms",
      "outputs": [],
      "source": "from SEEK import SEEK, auth",
      "start_time": "2019-03-17T19:25:43.154Z"
     },
     {
      "end_time": "2019-03-17T19:25:44.305Z",
      "execution_time": "8ms",
      "outputs": [],
      "source": "from SEEK import SEEK, auth",
      "start_time": "2019-03-17T19:25:44.297Z"
     },
     {
      "end_time": "2019-03-17T19:36:46.625Z",
      "execution_time": "116ms",
      "outputs": [],
      "source": "from SEEK import SEEK, auth",
      "start_time": "2019-03-17T19:36:46.509Z"
     },
     {
      "end_time": "2019-03-17T19:50:23.022Z",
      "execution_time": "19ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (SEEK.py, line 251)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-94c59809e4fe>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from SEEK import SEEK, auth\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\"\u001b[0;36m, line \u001b[0;32m251\u001b[0m\n\u001b[0;31m    requestPerThread = int(len(requests) / )\u001b[0m\n\u001b[0m                                           ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from SEEK import SEEK, auth",
      "start_time": "2019-03-17T19:50:23.003Z"
     },
     {
      "end_time": "2019-03-17T19:50:30.686Z",
      "execution_time": "154ms",
      "outputs": [],
      "source": "from SEEK import SEEK, auth",
      "start_time": "2019-03-17T19:50:30.532Z"
     },
     {
      "end_time": "2019-03-17T20:39:29.764Z",
      "execution_time": "112ms",
      "outputs": [],
      "source": "from SEEK import SEEK, auth",
      "start_time": "2019-03-17T20:39:29.652Z"
     },
     {
      "end_time": "2019-03-17T20:41:36.391Z",
      "execution_time": "359ms",
      "outputs": [],
      "source": "from SEEK import SEEK, auth",
      "start_time": "2019-03-17T20:41:36.032Z"
     },
     {
      "end_time": "2019-03-17T20:42:13.597Z",
      "execution_time": "96ms",
      "outputs": [],
      "source": "from SEEK import SEEK, auth",
      "start_time": "2019-03-17T20:42:13.501Z"
     },
     {
      "end_time": "2019-03-17T20:46:45.429Z",
      "execution_time": "114ms",
      "outputs": [],
      "source": "from SEEK import SEEK, auth",
      "start_time": "2019-03-17T20:46:45.315Z"
     }
    ]
   },
   "outputs": [],
   "source": [
    "from SEEK import SEEK, auth"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "provenance": [
     {
      "last_modified": "2019-03-17T19:31:23.871Z",
      "source": "### Authentication"
     }
    ]
   },
   "source": [
    "### Authentication"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "ExecutionTime": {
     "end_time": "2019-03-17T20:46:51.271497Z",
     "start_time": "2019-03-17T20:46:46.725731Z"
    },
    "provenance": [
     {
      "end_time": "2019-03-17T19:26:00.321Z",
      "execution_time": "8.42s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username: bogdan23a\nPassword: ········\n"
       }
      ],
      "source": "auth = auth()",
      "start_time": "2019-03-17T19:25:51.903Z"
     },
     {
      "end_time": "2019-03-17T19:36:56.857Z",
      "execution_time": "6.53s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username: bogdan23a\nPassword: ········\n"
       }
      ],
      "source": "auth = auth()",
      "start_time": "2019-03-17T19:36:50.331Z"
     },
     {
      "end_time": "2019-03-17T19:50:37.542Z",
      "execution_time": "4.86s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username: bogdan23a\nPassword: ········\n"
       }
      ],
      "source": "auth = auth()",
      "start_time": "2019-03-17T19:50:32.683Z"
     },
     {
      "end_time": "2019-03-17T20:40:12.873Z",
      "execution_time": "40.0s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username: bogdan23a\nPassword: ········\n"
       }
      ],
      "source": "auth = auth()",
      "start_time": "2019-03-17T20:39:32.848Z"
     },
     {
      "end_time": "2019-03-17T20:41:42.844Z",
      "execution_time": "5.45s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username: bogdan23a\nPassword: ········\n"
       }
      ],
      "source": "auth = auth()",
      "start_time": "2019-03-17T20:41:37.391Z"
     },
     {
      "end_time": "2019-03-17T20:42:22.444Z",
      "execution_time": "6.96s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username: bogdan23a\nPassword: ········\n"
       }
      ],
      "source": "auth = auth()",
      "start_time": "2019-03-17T20:42:15.479Z"
     },
     {
      "end_time": "2019-03-17T20:46:51.271Z",
      "execution_time": "4.55s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username: bogdan23a\nPassword: ········\n"
       }
      ],
      "source": "auth = auth()",
      "start_time": "2019-03-17T20:46:46.725Z"
     }
    ]
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Username: bogdan23a\n",
      "Password: ········\n"
     ]
    }
   ],
   "source": [
    "auth = auth()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "provenance": [
     {
      "last_modified": "2019-03-17T16:36:08.563Z",
      "source": "### Searching SEEK"
     },
     {
      "last_modified": "2019-03-17T17:26:32.294Z",
      "source": "### Searching SEEK"
     },
     {
      "last_modified": "2019-03-17T17:57:03.473Z",
      "source": "### Searching SEEK"
     },
     {
      "last_modified": "2019-03-17T19:22:38.936Z",
      "source": "### Searching SEEK"
     },
     {
      "last_modified": "2019-03-17T19:31:23.871Z",
      "source": "### Searching"
     }
    ]
   },
   "source": [
    "### Searching"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "ExecutionTime": {
     "end_time": "2019-03-17T19:31:25.219148Z",
     "start_time": "2019-03-17T19:31:24.319463Z"
    },
    "provenance": [
     {
      "end_time": "Unknown",
      "execution_time": "Unknown",
      "outputs": [],
      "source": "",
      "start_time": "Unknown"
     },
     {
      "end_time": "2019-03-17T16:19:34.728Z",
      "execution_time": "17ms",
      "outputs": [
       {
        "ename": "IndentationError",
        "evalue": "unexpected indent (SEEK.py, line 86)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-4b9d1aa18da9>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    import SEEK\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\"\u001b[0;36m, line \u001b[0;32m86\u001b[0m\n\u001b[0;31m    for relation in dir(self.data.relationships):\u001b[0m\n\u001b[0m    ^\u001b[0m\n\u001b[0;31mIndentationError\u001b[0m\u001b[0;31m:\u001b[0m unexpected indent\n"
        ]
       }
      ],
      "source": "import SEEK\n\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:19:34.711Z"
     },
     {
      "end_time": "2019-03-17T16:20:12.393Z",
      "execution_time": "150ms",
      "outputs": [
       {
        "ename": "NameError",
        "evalue": "name 'request' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-4b9d1aa18da9>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m: name 'request' is not defined"
        ]
       }
      ],
      "source": "import SEEK\n\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:20:12.243Z"
     },
     {
      "end_time": "2019-03-17T16:20:21.574Z",
      "execution_time": "23ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "request() missing 1 required positional argument: 'id'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-3-76a92958258d>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m: request() missing 1 required positional argument: 'id'"
        ]
       }
      ],
      "source": "import SEEK\n\nassay = SEEK.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:20:21.551Z"
     },
     {
      "end_time": "2019-03-17T16:20:37.060Z",
      "execution_time": "24ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "'module' object is not callable",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-4-5df159dd9646>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      4\u001b[0m \u001b[0massay\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: 'module' object is not callable"
        ]
       }
      ],
      "source": "import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:20:37.036Z"
     },
     {
      "end_time": "2019-03-17T16:20:55.735Z",
      "execution_time": "22ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "'module' object is not callable",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-5-5df159dd9646>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      4\u001b[0m \u001b[0massay\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: 'module' object is not callable"
        ]
       }
      ],
      "source": "import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:20:55.713Z"
     },
     {
      "end_time": "2019-03-17T16:21:00.571Z",
      "execution_time": "23ms",
      "outputs": [
       {
        "ename": "NameError",
        "evalue": "name 'request' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-6-4b9d1aa18da9>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m: name 'request' is not defined"
        ]
       }
      ],
      "source": "import SEEK\n\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:21:00.548Z"
     },
     {
      "end_time": "2019-03-17T16:22:48.932Z",
      "execution_time": "21ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "'module' object is not callable",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-7-086165628dbd>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      4\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: 'module' object is not callable"
        ]
       }
      ],
      "source": "import SEEK\n\nassay = SEEK()\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:22:48.911Z"
     },
     {
      "end_time": "2019-03-17T16:23:12.631Z",
      "execution_time": "164ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "'module' object is not callable",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-086165628dbd>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      4\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: 'module' object is not callable"
        ]
       }
      ],
      "source": "import SEEK\n\nassay = SEEK()\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:23:12.467Z"
     },
     {
      "end_time": "2019-03-17T16:23:18.907Z",
      "execution_time": "19ms",
      "outputs": [
       {
        "ename": "NameError",
        "evalue": "name 'request' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-559fc12f2c7b>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m: name 'request' is not defined"
        ]
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:23:18.888Z"
     },
     {
      "end_time": "2019-03-17T16:23:24.637Z",
      "execution_time": "914ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 3,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:23:23.723Z"
     },
     {
      "end_time": "2019-03-17T16:24:26.073Z",
      "execution_time": "22ms",
      "outputs": [
       {
        "ename": "NameError",
        "evalue": "name 'request' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-4-88fbb9262fda>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0;34m*\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m: name 'request' is not defined"
        ]
       }
      ],
      "source": "from SEEK import *\n\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:24:26.051Z"
     },
     {
      "end_time": "2019-03-17T16:24:30.316Z",
      "execution_time": "17ms",
      "outputs": [
       {
        "ename": "ImportError",
        "evalue": "cannot import name 'request'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-5-1cdaa7228921>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mImportError\u001b[0m: cannot import name 'request'"
        ]
       }
      ],
      "source": "from SEEK import request\n\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:24:30.299Z"
     },
     {
      "end_time": "2019-03-17T16:25:23.674Z",
      "execution_time": "20ms",
      "outputs": [
       {
        "ename": "ImportError",
        "evalue": "cannot import name 'request'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-6-1cdaa7228921>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mImportError\u001b[0m: cannot import name 'request'"
        ]
       }
      ],
      "source": "from SEEK import request\n\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:25:23.654Z"
     },
     {
      "end_time": "2019-03-17T16:25:50.117Z",
      "execution_time": "23ms",
      "outputs": [
       {
        "ename": "ImportError",
        "evalue": "cannot import name 'request'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-7-1cdaa7228921>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mImportError\u001b[0m: cannot import name 'request'"
        ]
       }
      ],
      "source": "from SEEK import request\n\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:25:50.094Z"
     },
     {
      "end_time": "2019-03-17T16:25:55.720Z",
      "execution_time": "159ms",
      "outputs": [
       {
        "ename": "NameError",
        "evalue": "name 'self' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-1cdaa7228921>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     14\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     15\u001b[0m \u001b[0msession\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequests\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mSession\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 16\u001b[0;31m \u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mheaders\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mupdate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mheaders\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     17\u001b[0m \u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mauth\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0;34m\"asda\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\"asda\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     18\u001b[0m \u001b[0mjson\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mNameError\u001b[0m: name 'self' is not defined"
        ]
       }
      ],
      "source": "from SEEK import request\n\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:25:55.561Z"
     },
     {
      "end_time": "2019-03-17T16:26:14.916Z",
      "execution_time": "28ms",
      "outputs": [
       {
        "ename": "NameError",
        "evalue": "name 'self' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-1cdaa7228921>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     14\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     15\u001b[0m \u001b[0msession\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequests\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mSession\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 16\u001b[0;31m \u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mheaders\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mupdate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mheaders\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     17\u001b[0m \u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mauth\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0;34m\"asda\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\"asda\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     18\u001b[0m \u001b[0mjson\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mNameError\u001b[0m: name 'self' is not defined"
        ]
       }
      ],
      "source": "from SEEK import request\n\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:26:14.888Z"
     },
     {
      "end_time": "2019-03-17T16:26:27.394Z",
      "execution_time": "981ms",
      "outputs": [
       {
        "ename": "RecursionError",
        "evalue": "maximum recursion depth exceeded while calling a Python object",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36mrequest\u001b[0;34m(type, id)\u001b[0m\n\u001b[1;32m     31\u001b[0m         \u001b[0mjson\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mr\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjson\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 32\u001b[0;31m         \u001b[0mloadJSON\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mjson\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'data'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     33\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mNameError\u001b[0m: name 'self' is not defined",
         "\nDuring handling of the above exception, another exception occurred:\n",
         "\u001b[0;31mRecursionError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-3-1cdaa7228921>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36mrequest\u001b[0;34m(type, id)\u001b[0m\n\u001b[1;32m     35\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     36\u001b[0m     \u001b[0;32mexcept\u001b[0m \u001b[0mException\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 37\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0me\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     38\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     39\u001b[0m \u001b[0;32mdef\u001b[0m \u001b[0mloadJSON\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mlayerName\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mlayer\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36mprint\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     70\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mprintRelationships\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     71\u001b[0m     \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 72\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Search item unavailable. Try again later.\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     73\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     74\u001b[0m \u001b[0;32mdef\u001b[0m \u001b[0mprintAttributes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "... last 1 frames repeated, from the frame below ...\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36mprint\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     70\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mprintRelationships\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     71\u001b[0m     \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 72\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"Search item unavailable. Try again later.\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     73\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     74\u001b[0m \u001b[0;32mdef\u001b[0m \u001b[0mprintAttributes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mRecursionError\u001b[0m: maximum recursion depth exceeded while calling a Python object"
        ]
       }
      ],
      "source": "from SEEK import request\n\nassay = request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:26:26.413Z"
     },
     {
      "end_time": "2019-03-17T16:28:40.414Z",
      "execution_time": "23ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "'module' object is not callable",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-4-5df159dd9646>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      4\u001b[0m \u001b[0massay\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: 'module' object is not callable"
        ]
       }
      ],
      "source": "import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:28:40.391Z"
     },
     {
      "end_time": "2019-03-17T16:28:45.125Z",
      "execution_time": "24ms",
      "outputs": [
       {
        "ename": "ImportError",
        "evalue": "cannot import name 'SEEK'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-5-20f9dd8707b3>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0massay\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mImportError\u001b[0m: cannot import name 'SEEK'"
        ]
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:28:45.101Z"
     },
     {
      "end_time": "2019-03-17T16:28:51.091Z",
      "execution_time": "19ms",
      "outputs": [
       {
        "ename": "IndentationError",
        "evalue": "unexpected indent (SEEK.py, line 1)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-20f9dd8707b3>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from SEEK import SEEK\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\"\u001b[0;36m, line \u001b[0;32m1\u001b[0m\n\u001b[0;31m    import json\u001b[0m\n\u001b[0m    ^\u001b[0m\n\u001b[0;31mIndentationError\u001b[0m\u001b[0;31m:\u001b[0m unexpected indent\n"
        ]
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:28:51.072Z"
     },
     {
      "end_time": "2019-03-17T16:28:57.127Z",
      "execution_time": "899ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 2,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:28:56.228Z"
     },
     {
      "end_time": "2019-03-17T16:36:13.517Z",
      "execution_time": "951ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:36:12.566Z"
     },
     {
      "end_time": "2019-03-17T16:38:59.878Z",
      "execution_time": "156ms",
      "outputs": [
       {
        "ename": "NameError",
        "evalue": "name 'PROT_REQUESTS_PER_THREAD' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-20f9dd8707b3>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0massay\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 5\u001b[0;31m \u001b[0;32mclass\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      6\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      7\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0m__init__\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36mSEEK\u001b[0;34m()\u001b[0m\n\u001b[1;32m    213\u001b[0m     \u001b[0;31m# return: none\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    214\u001b[0m     \u001b[0;31m# Fills the 'requestList' attribute with the response\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 215\u001b[0;31m     \u001b[0;32mdef\u001b[0m \u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequests\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mPROT_REQUESTS_PER_THREAD\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    216\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    217\u001b[0m         \u001b[0;31m# Compute the number of threads\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mNameError\u001b[0m: name 'PROT_REQUESTS_PER_THREAD' is not defined"
        ]
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:38:59.722Z"
     },
     {
      "end_time": "2019-03-17T16:39:38.815Z",
      "execution_time": "948ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:39:37.867Z"
     },
     {
      "end_time": "2019-03-17T16:41:18.562Z",
      "execution_time": "962ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:41:17.600Z"
     },
     {
      "end_time": "2019-03-17T16:42:18.471Z",
      "execution_time": "916ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:42:17.555Z"
     },
     {
      "end_time": "2019-03-17T16:45:49.499Z",
      "execution_time": "911ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T16:45:48.588Z"
     },
     {
      "end_time": "2019-03-17T17:06:38.748Z",
      "execution_time": "21ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "positional argument follows keyword argument (SEEK.py, line 163)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-20f9dd8707b3>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from SEEK import SEEK\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\"\u001b[0;36m, line \u001b[0;32m163\u001b[0m\n\u001b[0;31m    ps.parallelRequest(self = self, self.requestList, requestPerThread=self.searchResultsPerThread)\u001b[0m\n\u001b[0m                                   ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m positional argument follows keyword argument\n"
        ]
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T17:06:38.727Z"
     },
     {
      "end_time": "2019-03-17T17:07:35.208Z",
      "execution_time": "989ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T17:07:34.219Z"
     },
     {
      "end_time": "2019-03-17T17:12:11.394Z",
      "execution_time": "1.39s",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T17:12:10.006Z"
     },
     {
      "end_time": "2019-03-17T17:13:00.258Z",
      "execution_time": "840ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T17:12:59.418Z"
     },
     {
      "end_time": "2019-03-17T17:26:36.046Z",
      "execution_time": "1.03s",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T17:26:35.012Z"
     },
     {
      "end_time": "2019-03-17T17:38:15.980Z",
      "execution_time": "24ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (SEEK.py, line 416)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-20f9dd8707b3>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from SEEK import SEEK\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\"\u001b[0;36m, line \u001b[0;32m416\u001b[0m\n\u001b[0;31m    if string in request.data.attributes.title\u001b[0m\n\u001b[0m                                              ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T17:38:15.956Z"
     },
     {
      "end_time": "2019-03-17T17:38:21.705Z",
      "execution_time": "945ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 2,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T17:38:20.760Z"
     },
     {
      "end_time": "2019-03-17T17:44:19.586Z",
      "execution_time": "981ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T17:44:18.605Z"
     },
     {
      "end_time": "2019-03-17T19:17:31.079Z",
      "execution_time": "23ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (SEEK.py, line 22)",
        "output_type": "error",
        "traceback": [
         "Traceback \u001b[0;36m(most recent call last)\u001b[0m:\n",
         "  File \u001b[1;32m\"/home/bogdan/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\"\u001b[0m, line \u001b[1;32m2963\u001b[0m, in \u001b[1;35mrun_code\u001b[0m\n    exec(code_obj, self.user_global_ns, self.user_ns)\n",
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-20f9dd8707b3>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0;36m, in \u001b[0;35m<module>\u001b[0;36m\u001b[0m\n\u001b[0;31m    from SEEK import SEEK\u001b[0m\n",
         "\u001b[0;36m  File \u001b[0;32m\"/home/bogdan/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\"\u001b[0;36m, line \u001b[0;32m22\u001b[0m\n\u001b[0;31m    self.session.auth = (input(Username: ), getpass.getpass(\"Password: \"))\u001b[0m\n\u001b[0m                                       ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T19:17:31.056Z"
     },
     {
      "end_time": "2019-03-17T19:17:47.783Z",
      "execution_time": "7.31s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username:bogdan23a\nPassword: ········\n"
       },
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 2,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T19:17:40.473Z"
     },
     {
      "end_time": "2019-03-17T19:18:35.284Z",
      "execution_time": "22.7s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username:bogdan23a\nPassword: ········\n"
       },
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T19:18:12.540Z"
     },
     {
      "end_time": "2019-03-17T19:22:18.906Z",
      "execution_time": "143ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "__init__() missing 1 required positional argument: 'auth'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-1-20f9dd8707b3>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      4\u001b[0m \u001b[0massay\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: __init__() missing 1 required positional argument: 'auth'"
        ]
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T19:22:18.763Z"
     },
     {
      "end_time": "2019-03-17T19:22:20.610Z",
      "execution_time": "6ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "__init__() missing 1 required positional argument: 'auth'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-20f9dd8707b3>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfrom\u001b[0m \u001b[0mSEEK\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0massay\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      4\u001b[0m \u001b[0massay\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"assays\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m\"576\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: __init__() missing 1 required positional argument: 'auth'"
        ]
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T19:22:20.604Z"
     },
     {
      "end_time": "2019-03-17T19:22:46.482Z",
      "execution_time": "7.19s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username: bogdan23a\nPassword········\n"
       },
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 1,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "from SEEK import SEEK\n\nassay = SEEK()\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T19:22:39.295Z"
     },
     {
      "end_time": "2019-03-17T19:26:14.040Z",
      "execution_time": "877ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 4,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "assay = SEEK(auth)\nassay.request(\"assays\", \"576\")",
      "start_time": "2019-03-17T19:26:13.163Z"
     },
     {
      "end_time": "2019-03-17T19:31:25.219Z",
      "execution_time": "900ms",
      "outputs": [
       {
        "data": {
         "text/plain": "True"
        },
        "execution_count": 6,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "assay = SEEK(auth)\nassay.request(type=\"assays\", id=\"576\")",
      "start_time": "2019-03-17T19:31:24.319Z"
     }
    ]
   },
   "outputs": [],
   "source": [
    "assay = SEEK(auth)\n",
    "assay.request(type=\"assays\", id=\"576\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "provenance": [
     {
      "last_modified": "2019-03-17T16:36:08.563Z",
      "source": "### Browsing SEEK"
     },
     {
      "last_modified": "2019-03-17T17:26:32.294Z",
      "source": "### Browsing SEEK"
     },
     {
      "last_modified": "2019-03-17T17:57:03.473Z",
      "source": "### Browsing SEEK"
     },
     {
      "last_modified": "2019-03-17T19:22:38.936Z",
      "source": "### Browsing SEEK"
     },
     {
      "last_modified": "2019-03-17T19:31:23.871Z",
      "source": "### Browsing"
     }
    ]
   },
   "source": [
    "### Browsing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "ExecutionTime": {
     "end_time": "2019-03-17T20:47:11.659647Z",
     "start_time": "2019-03-17T20:46:52.893421Z"
    },
    "provenance": [
     {
      "end_time": "2019-03-17T16:36:17.793Z",
      "execution_time": "3.06s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n"
       },
       {
        "ename": "NameError",
        "evalue": "name 'ParallelSearch' is not defined",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-184293c31a76>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    150\u001b[0m         \u001b[0;31m# 2nd param: the total amount of request\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    151\u001b[0m         \u001b[0;31m# 3rd param: the number of request per thread for the paralelization\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 152\u001b[0;31m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mParallelSearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    153\u001b[0m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchResultsPerThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    154\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mNameError\u001b[0m: name 'ParallelSearch' is not defined"
        ]
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T16:36:14.738Z"
     },
     {
      "end_time": "2019-03-17T16:39:44.052Z",
      "execution_time": "3.08s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n"
       },
       {
        "ename": "AttributeError",
        "evalue": "'SEEK' object has no attribute 'threadList'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-184293c31a76>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    154\u001b[0m         \u001b[0;31m# 3rd param: the number of request per thread for the paralelization\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    155\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 156\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchResultsPerThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    157\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    158\u001b[0m         \u001b[0;31m# Wait for the requests to finish in order to continue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36mparallelRequest\u001b[0;34m(self, requests, requestPerThread)\u001b[0m\n\u001b[1;32m    237\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    238\u001b[0m             \u001b[0;31m# Add the thread to the list of threads\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 239\u001b[0;31m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mthreadList\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnewThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    240\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    241\u001b[0m     \u001b[0;31m# Each thread executes this method\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mAttributeError\u001b[0m: 'SEEK' object has no attribute 'threadList'"
        ]
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T16:39:40.967Z"
     },
     {
      "end_time": "2019-03-17T16:41:22.304Z",
      "execution_time": "2.61s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not definedname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not definedname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not definedname 'ReadObject' is not defined\n\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\n0 ommited results\n\n0 relationships found\n\n0 ommited results\n\nSubstituting relationships into original search results: \n\n\n                                        --SEARCH RESULTS--\n\n\n"
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T16:41:19.690Z"
     },
     {
      "end_time": "2019-03-17T16:42:11.341Z",
      "execution_time": "2.38s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nname 'ReadObject' is not definedname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not definedname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not definedname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\n0 ommited results\n\n0 relationships found\n\n0 ommited results\n\nSubstituting relationships into original search results: \n\n\n                                        --SEARCH RESULTS--\n\n\n"
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T16:42:08.960Z"
     },
     {
      "end_time": "2019-03-17T16:42:22.011Z",
      "execution_time": "2.88s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n[{'id': '405', 'type': 'assays'}, {'id': '754', 'type': 'assays'}, {'id': '753', 'type': 'assays'}, {'id': '776', 'type': 'assays'}, {'id': '751', 'type': 'assays'}, {'id': '771', 'type': 'assays'}, {'id': '2277', 'type': 'data_files'}, {'id': '2279', 'type': 'data_files'}, {'id': '1428', 'type': 'data_files'}, {'id': '1989', 'type': 'data_files'}, {'id': '2255', 'type': 'data_files'}, {'id': '2280', 'type': 'data_files'}, {'id': '1391', 'type': 'data_files'}, {'id': '2144', 'type': 'data_files'}, {'id': '2145', 'type': 'data_files'}, {'id': '2249', 'type': 'data_files'}, {'id': '951', 'type': 'data_files'}, {'id': '1320', 'type': 'data_files'}, {'id': '915', 'type': 'data_files'}, {'id': '9', 'type': 'data_files'}, {'id': '942', 'type': 'data_files'}, {'id': '64', 'type': 'events'}, {'id': '96', 'type': 'events'}, {'id': '69', 'type': 'events'}, {'id': '73', 'type': 'events'}, {'id': '120', 'type': 'events'}, {'id': '119', 'type': 'events'}, {'id': '182', 'type': 'institutions'}, {'id': '27', 'type': 'institutions'}, {'id': '198', 'type': 'institutions'}, {'id': '260', 'type': 'institutions'}, {'id': '261', 'type': 'institutions'}, {'id': '262', 'type': 'institutions'}, {'id': '63', 'type': 'institutions'}, {'id': '72', 'type': 'institutions'}, {'id': '203', 'type': 'institutions'}, {'id': '76', 'type': 'institutions'}, {'id': '79', 'type': 'institutions'}, {'id': '13', 'type': 'institutions'}, {'id': '23', 'type': 'institutions'}, {'id': '40', 'type': 'institutions'}, {'id': '89', 'type': 'institutions'}, {'id': '96', 'type': 'institutions'}, {'id': '35', 'type': 'institutions'}, {'id': '7', 'type': 'institutions'}, {'id': '96', 'type': 'investigations'}, {'id': '94', 'type': 'investigations'}, {'id': '226', 'type': 'models'}, {'id': '320', 'type': 'models'}, {'id': '760', 'type': 'people'}, {'id': '870', 'type': 'people'}, {'id': '816', 'type': 'people'}, {'id': '820', 'type': 'people'}, {'id': '678', 'type': 'people'}, {'id': '762', 'type': 'people'}, {'id': '1240', 'type': 'people'}, {'id': '1243', 'type': 'people'}, {'id': '1244', 'type': 'people'}, {'id': '1241', 'type': 'people'}, {'id': '1242', 'type': 'people'}, {'id': '753', 'type': 'people'}, {'id': '903', 'type': 'people'}, {'id': '1045', 'type': 'people'}, {'id': '1248', 'type': 'people'}, {'id': '1250', 'type': 'people'}, {'id': '1249', 'type': 'people'}, {'id': '1246', 'type': 'people'}, {'id': '1247', 'type': 'people'}, {'id': '1245', 'type': 'people'}, {'id': '1039', 'type': 'people'}, {'id': '1229', 'type': 'people'}, {'id': '474', 'type': 'people'}, {'id': '476', 'type': 'people'}, {'id': '759', 'type': 'people'}, {'id': '700', 'type': 'people'}, {'id': '907', 'type': 'people'}, {'id': '523', 'type': 'people'}, {'id': '1064', 'type': 'people'}, {'id': '335', 'type': 'people'}, {'id': '623', 'type': 'people'}, {'id': '205', 'type': 'people'}, {'id': '783', 'type': 'people'}, {'id': '136', 'type': 'people'}, {'id': '792', 'type': 'people'}, {'id': '100', 'type': 'presentations'}, {'id': '378', 'type': 'presentations'}, {'id': '604', 'type': 'presentations'}, {'id': '284', 'type': 'presentations'}, {'id': '295', 'type': 'presentations'}, {'id': '320', 'type': 'presentations'}, {'id': '105', 'type': 'presentations'}, {'id': '305', 'type': 'presentations'}, {'id': '87', 'type': 'presentations'}, {'id': '38', 'type': 'presentations'}, {'id': '11', 'type': 'presentations'}, {'id': '51', 'type': 'presentations'}, {'id': '500', 'type': 'presentations'}, {'id': '42', 'type': 'presentations'}, {'id': '9', 'type': 'programmes'}, {'id': '15', 'type': 'programmes'}, {'id': '44', 'type': 'projects'}, {'id': '41', 'type': 'projects'}, {'id': '46', 'type': 'projects'}, {'id': '42', 'type': 'projects'}, {'id': '45', 'type': 'projects'}, {'id': '130', 'type': 'projects'}, {'id': '24', 'type': 'projects'}, {'id': '34', 'type': 'projects'}, {'id': '366', 'type': 'publications'}, {'id': '338', 'type': 'publications'}, {'id': '267', 'type': 'publications'}, {'id': '389', 'type': 'publications'}, {'id': '390', 'type': 'publications'}, {'id': '391', 'type': 'publications'}, {'id': '288', 'type': 'publications'}, {'id': '310', 'type': 'publications'}, {'id': '266', 'type': 'publications'}, {'id': '309', 'type': 'publications'}, {'id': '312', 'type': 'publications'}, {'id': '242', 'type': 'publications'}, {'id': '311', 'type': 'publications'}, {'id': '359', 'type': 'publications'}, {'id': '363', 'type': 'publications'}, {'id': '194', 'type': 'publications'}, {'id': '272', 'type': 'publications'}, {'id': '394', 'type': 'publications'}, {'id': '239', 'type': 'publications'}, {'id': '274', 'type': 'publications'}, {'id': '70', 'type': 'sample_types'}, {'id': '68', 'type': 'sample_types'}, {'id': '15', 'type': 'sample_types'}, {'id': '58', 'type': 'sample_types'}, {'id': '75', 'type': 'sops'}, {'id': '255', 'type': 'studies'}, {'id': '170', 'type': 'studies'}]\n\n135 results found\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not definedname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not definedname 'ReadObject' is not defined\n\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not definedname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not definedname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\nname 'ReadObject' is not defined\n\n0 ommited results\n\n0 relationships found\n\n0 ommited results\n\nSubstituting relationships into original search results: \n\n\n                                        --SEARCH RESULTS--\n\n\n"
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T16:42:19.133Z"
     },
     {
      "end_time": "2019-03-17T16:46:34.410Z",
      "execution_time": "43.1s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n\n4 ommited results\n\n558 relationships found\nLoading Completed\n\n2 ommited results\n\nSubstituting relationships into original search results: \nLoading 99.58%\n\n                                        --SEARCH RESULTS--\n\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhöh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nErik Schröder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nPAH metabolites in bile Kollevåg\nSediment chemistry\nLiver chemistry_Kollevåg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universität Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUte Spiekerkötter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Böhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Böhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Böhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nLeibniz-Institut für Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hünfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut für Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1α axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nPablo José Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "SABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMatthias König(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\r\nRead more about the LiSyM junior group at: www.livermetabolism.com\r\n\r\nJunior Group Leader\r\nDr. Matthias König\r\n\r\nInstitute for Theoretical Biology\r\nHumboldt-University Berlin\r\nInvalidenstraße 43, 10117 Berlin, Germany\r\nphone +49 30 2093-8450\r\nkonigmatt@googlemail.com\r\n\r\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\r\n\r\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universität zu Berlin\nCharité University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\r\nMESI-STRAT Coordinator\r\nLeader WP8 – Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Börries\nBernhard Steiert\nChristian Tönsing\nUte Spiekerkötter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndré Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\r\n\r\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias König\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \r\n \r\nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\r\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nTBARS_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut für Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nWolfgang Müller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\r\n\r\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\r\n\r\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\r\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\r\n\r\nMore information available from\r\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\r\n\r\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\r\n\r\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \r\n19th-20th of March, 2012, Vienna, Austria\r\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Müller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nOliver Ebenhöh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\r\nMy current research deals with the integration of different metabolic pathways. Research lines include:\r\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\r\n- The relation between metabolism and biologial ageing;\r\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hönicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Müller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\r\n\r\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\r\n\r\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMartin Böhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevåg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nUrsula Klingmüller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\r\n\r\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \r\n\r\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\r\n\r\n<b>Topics include:</b>\r\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\r\n- Comprehensive and unambiguous data description and annotation\r\n- Standard formats and terminologies for data and metadata \r\n- Guidelines for reporting data (Minimum information standards) \r\n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\r\n- Using SEEK & FAIRDOMhub for integrated data management\r\n\r\n<b>Registration:</b> http://bit.ly/2ArOC7w\r\n(please state if you will attend on both days or only on one day)\r\n\r\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\r\n\r\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hünfeld will be refunded.\r\n\r\n<b>Organization:</b>\r\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\r\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\r\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevåg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevåg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\r\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skłodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\r\n\r\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised ‘Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Müller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhöh\nDorota Kurek\nUte Spiekerkötter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Müller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\r\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29 – 30 November 2012, Heidelberg\r\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nKollevåg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevåg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\r\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nqPCR ovaries_Kollevåg\nqPCR liver_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\r\n\r\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias König\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias König\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmüller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dächert\nMartin Böhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\r\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\r\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\r\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nqPCR liver_Kollevåg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nTBARS_Kollevåg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nCharité University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias König\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Freiburg\nCharité University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nMatthias König\nJohannes Bausch\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\r\n\r\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \r\n\r\nKey questions:\r\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\r\n\r\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\r\n\r\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\r\n\r\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nGriffin Discoveries BV\nUniversitätsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmüller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Böhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Müller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universität Dresden\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nHumboldt University Berlin\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nLeibniz-Institut für Arbeitsforschung (IfADo)\nUniversitätsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Müller\nMaja Rey\nUrsula Klingmüller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias König\nMartin Böhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schröder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schönfels\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nHumboldt-Universität zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias König\nJudith Wodke\nClemens Kühn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemünde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP – EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new ‘Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemünde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\r\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\n“Siberian” SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \r\n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigación Príncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversité catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kåre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksøyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Söderström\nLibe Aranguren Abadía\nSiri Øfsthus Goksøyr\nOdd André Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\r\n\r\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmon’s ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmon’s body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\r\n\r\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and “fed” in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Røsæg\nSahar Hassani\nMarius André Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T16:45:51.299Z"
     },
     {
      "end_time": "2019-03-17T17:07:40.310Z",
      "execution_time": "3.10s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n"
       },
       {
        "ename": "TypeError",
        "evalue": "parallelRequest() got multiple values for argument 'self'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-184293c31a76>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    161\u001b[0m         \u001b[0;31m# 3rd param: the number of request per thread for the paralelization\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    162\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 163\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequest\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchResultsPerThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    164\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    165\u001b[0m         \u001b[0;31m# Wait for the requests to finish in order to continue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: parallelRequest() got multiple values for argument 'self'"
        ]
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T17:07:37.209Z"
     },
     {
      "end_time": "2019-03-17T17:12:15.883Z",
      "execution_time": "3.02s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n<SEEK.SEEK object at 0x7efefc09b320>\n"
       },
       {
        "ename": "TypeError",
        "evalue": "parallelRequest() got an unexpected keyword argument 'request'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-2-184293c31a76>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    162\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    163\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 164\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequest\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchResultsPerThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    165\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    166\u001b[0m         \u001b[0;31m# Wait for the requests to finish in order to continue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: parallelRequest() got an unexpected keyword argument 'request'"
        ]
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T17:12:12.868Z"
     },
     {
      "end_time": "2019-03-17T17:12:52.990Z",
      "execution_time": "5.03s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n<SEEK.SEEK object at 0x7efeedabeba8>\n"
       },
       {
        "ename": "TypeError",
        "evalue": "parallelRequest() got an unexpected keyword argument 'request'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-3-184293c31a76>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    162\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    163\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 164\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchResultsPerThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    165\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    166\u001b[0m         \u001b[0;31m# Wait for the requests to finish in order to continue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: parallelRequest() got an unexpected keyword argument 'request'"
        ]
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T17:12:47.963Z"
     },
     {
      "end_time": "2019-03-17T17:13:46.941Z",
      "execution_time": "43.3s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n<SEEK.SEEK object at 0x7fe224273978>\n<SEEK.SEEK object at 0x7fe2154e6e10>\nLoading Completed\n\n4 ommited results\n\n558 relationships found\n<SEEK.SEEK object at 0x7fe2143da780>\nLoading Completed\n\n2 ommited results\n\nSubstituting relationships into original search results: \nLoading 99.58%\n\n                                        --SEARCH RESULTS--\n\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nUte Spiekerkötter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universität Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nErik Schröder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhöh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hünfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nPablo José Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nPAH metabolites in bile Kollevåg\nSediment chemistry\nLiver chemistry_Kollevåg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\nMESI-STRAT Coordinator\nLeader WP8 – Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Böhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Böhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Böhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut für Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nLeibniz-Institut für Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nOliver Ebenhöh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\n\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \n19th-20th of March, 2012, Vienna, Austria\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Müller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndré Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMatthias König(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\nRead more about the LiSyM junior group at: www.livermetabolism.com\n\nJunior Group Leader\nDr. Matthias König\n\nInstitute for Theoretical Biology\nHumboldt-University Berlin\nInvalidenstraße 43, 10117 Berlin, Germany\nphone +49 30 2093-8450\nkonigmatt@googlemail.com\n\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\n\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universität zu Berlin\nCharité University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\n\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\n\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut für Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Börries\nBernhard Steiert\nChristian Tönsing\nUte Spiekerkötter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1α axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nTBARS_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias König\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMartin Böhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\n\nMore information available from\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\nMy current research deals with the integration of different metabolic pathways. Research lines include:\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\n- The relation between metabolism and biologial ageing;\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nWolfgang Müller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\n\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\n\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hönicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nUrsula Klingmüller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevåg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevåg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\n\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\n\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Müller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\n\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \n\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\n\n<b>Topics include:</b>\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\n- Comprehensive and unambiguous data description and annotation\n- Standard formats and terminologies for data and metadata \n- Guidelines for reporting data (Minimum information standards) \n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\n- Using SEEK & FAIRDOMhub for integrated data management\n\n<b>Registration:</b> http://bit.ly/2ArOC7w\n(please state if you will attend on both days or only on one day)\n\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\n\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hünfeld will be refunded.\n\n<b>Organization:</b>\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29 – 30 November 2012, Heidelberg\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevåg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Müller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nKollevåg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevåg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skłodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\n\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised ‘Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Müller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhöh\nDorota Kurek\nUte Spiekerkötter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nqPCR ovaries_Kollevåg\nqPCR liver_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nTBARS_Kollevåg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmüller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dächert\nMartin Böhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nqPCR liver_Kollevåg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\n\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias König\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias König\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nCharité University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias König\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Freiburg\nCharité University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nMatthias König\nJohannes Bausch\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Müller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universität Dresden\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nHumboldt University Berlin\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nLeibniz-Institut für Arbeitsforschung (IfADo)\nUniversitätsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Müller\nMaja Rey\nUrsula Klingmüller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias König\nMartin Böhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schröder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schönfels\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nHumboldt-Universität zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias König\nJudith Wodke\nClemens Kühn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\n\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \n\nKey questions:\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\n\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\n\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\n\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nGriffin Discoveries BV\nUniversitätsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmüller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Böhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemünde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP – EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new ‘Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemünde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\n“Siberian” SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigación Príncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversité catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kåre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksøyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Söderström\nLibe Aranguren Abadía\nSiri Øfsthus Goksøyr\nOdd André Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\n\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmon’s ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmon’s body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\n\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and “fed” in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Røsæg\nSahar Hassani\nMarius André Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T17:13:03.633Z"
     },
     {
      "end_time": "2019-03-17T17:27:21.104Z",
      "execution_time": "44.0s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n\n4 ommited results\n\n558 relationships found\nLoading Completed\n\n2 ommited results\n\nSubstituting relationships into original search results: \nLoading 99.58%\n\n                                        --SEARCH RESULTS--\n\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nErik Schröder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Böhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Böhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Böhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universität Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nPAH metabolites in bile Kollevåg\nSediment chemistry\nLiver chemistry_Kollevåg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nUte Spiekerkötter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhöh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "SABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\r\n\r\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\r\nMESI-STRAT Coordinator\r\nLeader WP8 – Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nLeibniz-Institut für Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nPablo José Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hünfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \r\nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut für Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\r\n\r\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndré Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMatthias König(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\r\nRead more about the LiSyM junior group at: www.livermetabolism.com\r\n\r\nJunior Group Leader\r\nDr. Matthias König\r\n\r\nInstitute for Theoretical Biology\r\nHumboldt-University Berlin\r\nInvalidenstraße 43, 10117 Berlin, Germany\r\nphone +49 30 2093-8450\r\nkonigmatt@googlemail.com\r\n\r\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\r\n\r\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universität zu Berlin\nCharité University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Börries\nBernhard Steiert\nChristian Tönsing\nUte Spiekerkötter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\r\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nTBARS_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut für Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1α axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nOliver Ebenhöh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \r\n19th-20th of March, 2012, Vienna, Austria\r\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Müller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\r\n\r\nMore information available from\r\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias König\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \r\n \r\nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\r\n\r\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\r\n\r\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nWolfgang Müller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\r\n\r\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\r\n\r\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\r\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\r\nMy current research deals with the integration of different metabolic pathways. Research lines include:\r\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\r\n- The relation between metabolism and biologial ageing;\r\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Müller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nMartin Böhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevåg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nUrsula Klingmüller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\r\n\r\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\r\n\r\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hönicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevåg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevåg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\r\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\r\n\r\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \r\n\r\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\r\n\r\n<b>Topics include:</b>\r\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\r\n- Comprehensive and unambiguous data description and annotation\r\n- Standard formats and terminologies for data and metadata \r\n- Guidelines for reporting data (Minimum information standards) \r\n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\r\n- Using SEEK & FAIRDOMhub for integrated data management\r\n\r\n<b>Registration:</b> http://bit.ly/2ArOC7w\r\n(please state if you will attend on both days or only on one day)\r\n\r\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\r\n\r\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hünfeld will be refunded.\r\n\r\n<b>Organization:</b>\r\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\r\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\r\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Müller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nqPCR ovaries_Kollevåg\nqPCR liver_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skłodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\r\n\r\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised ‘Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Müller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhöh\nDorota Kurek\nUte Spiekerkötter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nKollevåg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevåg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nqPCR liver_Kollevåg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\r\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\r\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29 – 30 November 2012, Heidelberg\r\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Freiburg\nCharité University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nMatthias König\nJohannes Bausch\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\r\n\r\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias König\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias König\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\r\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmüller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dächert\nMartin Böhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nTBARS_Kollevåg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\r\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\r\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nCharité University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias König\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Müller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universität Dresden\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nHumboldt University Berlin\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nLeibniz-Institut für Arbeitsforschung (IfADo)\nUniversitätsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Müller\nMaja Rey\nUrsula Klingmüller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias König\nMartin Böhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schröder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schönfels\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\r\n\r\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \r\n\r\nKey questions:\r\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\r\n\r\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\r\n\r\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\r\n\r\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nGriffin Discoveries BV\nUniversitätsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmüller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Böhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nHumboldt-Universität zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias König\nJudith Wodke\nClemens Kühn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemünde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP – EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new ‘Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemünde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\r\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\n“Siberian” SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \r\n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigación Príncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversité catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kåre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksøyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Söderström\nLibe Aranguren Abadía\nSiri Øfsthus Goksøyr\nOdd André Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\r\n\r\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmon’s ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmon’s body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\r\n\r\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and “fed” in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Salmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Røsæg\nSahar Hassani\nMarius André Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T17:26:37.130Z"
     },
     {
      "end_time": "2019-03-17T17:39:05.867Z",
      "execution_time": "43.2s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n\n4 ommited results\n\n558 relationships found\nLoading Completed\n\n2 ommited results\n\nSubstituting relationships into original search results: \nLoading 99.58%\n\n                                        --SEARCH RESULTS--\n\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nUte Spiekerkötter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nErik Schröder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universität Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhöh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Böhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Böhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Böhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut für Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nPablo José Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1α axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nLeibniz-Institut für Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nOliver Ebenhöh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hünfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\nMESI-STRAT Coordinator\nLeader WP8 – Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nPAH metabolites in bile Kollevåg\nSediment chemistry\nLiver chemistry_Kollevåg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\n\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\n\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Börries\nBernhard Steiert\nChristian Tönsing\nUte Spiekerkötter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndré Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\n\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nTBARS_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut für Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nMatthias König(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\nRead more about the LiSyM junior group at: www.livermetabolism.com\n\nJunior Group Leader\nDr. Matthias König\n\nInstitute for Theoretical Biology\nHumboldt-University Berlin\nInvalidenstraße 43, 10117 Berlin, Germany\nphone +49 30 2093-8450\nkonigmatt@googlemail.com\n\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\n\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universität zu Berlin\nCharité University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \n19th-20th of March, 2012, Vienna, Austria\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Müller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nWolfgang Müller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\n\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\n\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\n\nMore information available from\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias König\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Matthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hönicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\r\nMy current research deals with the integration of different metabolic pathways. Research lines include:\r\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\r\n- The relation between metabolism and biologial ageing;\r\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Müller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \r\nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nMartin Böhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\r\n\r\n<b>Topics include:</b>\r\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\r\n- Comprehensive and unambiguous data description and annotation\r\n- Standard formats and terminologies for data and metadata \r\n- Guidelines for reporting data (Minimum information standards) \r\n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\r\n- Using SEEK & FAIRDOMhub for integrated data management\r\n\r\n<b>Registration:</b> http://bit.ly/2ArOC7w\r\n(please state if you will attend on both days or only on one day)\r\n\r\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\r\n\r\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hünfeld will be refunded.\r\n\r\n<b>Organization:</b>\r\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\r\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\r\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\r\n\r\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\r\n\r\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Müller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevåg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevåg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUrsula Klingmüller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\r\n\r\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \r\n\r\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nqPCR ovaries_Kollevåg\nqPCR liver_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skłodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\r\n\r\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised ‘Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Müller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhöh\nDorota Kurek\nUte Spiekerkötter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nKollevåg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevåg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\r\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\r\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\r\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29 – 30 November 2012, Heidelberg\r\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevåg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nTBARS_Kollevåg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\r\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\r\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmüller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dächert\nMartin Böhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nCharité University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias König\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\r\n\r\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias König\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias König\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\r\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Freiburg\nCharité University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nMatthias König\nJohannes Bausch\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nqPCR liver_Kollevåg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Müller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universität Dresden\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nHumboldt University Berlin\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nLeibniz-Institut für Arbeitsforschung (IfADo)\nUniversitätsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Müller\nMaja Rey\nUrsula Klingmüller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias König\nMartin Böhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schröder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schönfels\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\r\n\r\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \r\n\r\nKey questions:\r\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\r\n\r\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\r\n\r\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\r\n\r\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nGriffin Discoveries BV\nUniversitätsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmüller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Böhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nHumboldt-Universität zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias König\nJudith Wodke\nClemens Kühn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemünde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP – EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new ‘Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemünde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\r\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\n“Siberian” SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \r\n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigación Príncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversité catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kåre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksøyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Söderström\nLibe Aranguren Abadía\nSiri Øfsthus Goksøyr\nOdd André Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\r\n\r\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmon’s ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmon’s body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\r\n\r\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and “fed” in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Røsæg\nSahar Hassani\nMarius André Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T17:38:22.715Z"
     },
     {
      "end_time": "2019-03-17T17:45:02.618Z",
      "execution_time": "42.3s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n\n4 ommited results\n\n558 relationships found\nLoading Completed\n\n2 ommited results\n\nSubstituting relationships into original search results: \nLoading 99.58%\n\n                                        --SEARCH RESULTS--\n\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nErik Schröder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nUte Spiekerkötter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universität Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhöh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\nMESI-STRAT Coordinator\nLeader WP8 – Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hünfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nLeibniz-Institut für Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nPablo José Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut für Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Böhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Böhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Böhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Model Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1α axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nPAH metabolites in bile Kollevåg\nSediment chemistry\nLiver chemistry_Kollevåg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nMatthias König(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\r\nRead more about the LiSyM junior group at: www.livermetabolism.com\r\n\r\nJunior Group Leader\r\nDr. Matthias König\r\n\r\nInstitute for Theoretical Biology\r\nHumboldt-University Berlin\r\nInvalidenstraße 43, 10117 Berlin, Germany\r\nphone +49 30 2093-8450\r\nkonigmatt@googlemail.com\r\n\r\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\r\n\r\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universität zu Berlin\nCharité University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\r\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nTBARS_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\r\n\r\nMore information available from\r\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Börries\nBernhard Steiert\nChristian Tönsing\nUte Spiekerkötter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nOliver Ebenhöh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndré Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \r\n19th-20th of March, 2012, Vienna, Austria\r\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Müller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nWolfgang Müller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\r\n\r\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\r\n\r\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\r\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\r\n\r\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\r\n\r\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut für Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias König\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\r\n\r\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\r\n\r\n<b>Topics include:</b>\r\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\r\n- Comprehensive and unambiguous data description and annotation\r\n- Standard formats and terminologies for data and metadata \r\n- Guidelines for reporting data (Minimum information standards) \r\n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\r\n- Using SEEK & FAIRDOMhub for integrated data management\r\n\r\n<b>Registration:</b> http://bit.ly/2ArOC7w\r\n(please state if you will attend on both days or only on one day)\r\n\r\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\r\n\r\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hünfeld will be refunded.\r\n\r\n<b>Organization:</b>\r\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\r\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\r\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMartin Böhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\r\nMy current research deals with the integration of different metabolic pathways. Research lines include:\r\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\r\n- The relation between metabolism and biologial ageing;\r\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \r\nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\r\n\r\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\r\n\r\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\r\n\r\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \r\n\r\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUrsula Klingmüller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Müller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hönicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevåg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevåg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Müller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skłodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\r\n\r\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised ‘Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Müller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhöh\nDorota Kurek\nUte Spiekerkötter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\r\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\r\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nqPCR ovaries_Kollevåg\nqPCR liver_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\r\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\r\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29 – 30 November 2012, Heidelberg\r\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevåg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\r\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nTBARS_Kollevåg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\r\n\r\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias König\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias König\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmüller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dächert\nMartin Böhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Freiburg\nCharité University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nMatthias König\nJohannes Bausch\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\r\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nKollevåg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevåg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nqPCR liver_Kollevåg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nCharité University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias König\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Müller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universität Dresden\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nHumboldt University Berlin\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nLeibniz-Institut für Arbeitsforschung (IfADo)\nUniversitätsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Müller\nMaja Rey\nUrsula Klingmüller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias König\nMartin Böhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schröder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schönfels\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\r\n\r\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \r\n\r\nKey questions:\r\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\r\n\r\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\r\n\r\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\r\n\r\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nGriffin Discoveries BV\nUniversitätsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmüller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Böhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nHumboldt-Universität zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias König\nJudith Wodke\nClemens Kühn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemünde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP – EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new ‘Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemünde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\r\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\n“Siberian” SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \r\n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigación Príncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversité catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kåre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksøyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Söderström\nLibe Aranguren Abadía\nSiri Øfsthus Goksøyr\nOdd André Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\r\n\r\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmon’s ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmon’s body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\r\n\r\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and “fed” in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Røsæg\nSahar Hassani\nMarius André Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T17:44:20.319Z"
     },
     {
      "end_time": "2019-03-17T19:23:38.680Z",
      "execution_time": "47.5s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Username: bogdan23a\nPassword········\nEnter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n\n4 ommited results\n\n558 relationships found\nLoading Completed\n\n2 ommited results\n\nSubstituting relationships into original search results: \nLoading 99.58%\n\n                                        --SEARCH RESULTS--\n\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nErik Schröder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUte Spiekerkötter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universität Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhöh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\nMESI-STRAT Coordinator\nLeader WP8 – Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hünfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Böhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Böhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Böhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLeibniz-Institut für Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nMatthias König(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\nRead more about the LiSyM junior group at: www.livermetabolism.com\n\nJunior Group Leader\nDr. Matthias König\n\nInstitute for Theoretical Biology\nHumboldt-University Berlin\nInvalidenstraße 43, 10117 Berlin, Germany\nphone +49 30 2093-8450\nkonigmatt@googlemail.com\n\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\n\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universität zu Berlin\nCharité University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\n\nMore information available from\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nPablo José Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nPAH metabolites in bile Kollevåg\nSediment chemistry\nLiver chemistry_Kollevåg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\n\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nOliver Ebenhöh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut für Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1α axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nTBARS_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \n19th-20th of March, 2012, Vienna, Austria\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Müller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndré Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\n\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\n\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut für Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Börries\nBernhard Steiert\nChristian Tönsing\nUte Spiekerkötter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias König\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nWolfgang Müller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\n\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\n\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\n\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \n\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\n\n<b>Topics include:</b>\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\n- Comprehensive and unambiguous data description and annotation\n- Standard formats and terminologies for data and metadata \n- Guidelines for reporting data (Minimum information standards) \n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\n- Using SEEK & FAIRDOMhub for integrated data management\n\n<b>Registration:</b> http://bit.ly/2ArOC7w\n(please state if you will attend on both days or only on one day)\n\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\n\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hünfeld will be refunded.\n\n<b>Organization:</b>\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\nMy current research deals with the integration of different metabolic pathways. Research lines include:\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\n- The relation between metabolism and biologial ageing;\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nUrsula Klingmüller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nMartin Böhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\n\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\n\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Müller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hönicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevåg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevåg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Müller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nqPCR ovaries_Kollevåg\nqPCR liver_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skłodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\n\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised ‘Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Müller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhöh\nDorota Kurek\nUte Spiekerkötter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29 – 30 November 2012, Heidelberg\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Freiburg\nCharité University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nMatthias König\nJohannes Bausch\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\n\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias König\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias König\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevåg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nTBARS_Kollevåg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nKollevåg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevåg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmüller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dächert\nMartin Böhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nCharité University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias König\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nqPCR liver_Kollevåg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Wolfgang Müller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universität Dresden\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nHumboldt University Berlin\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nLeibniz-Institut für Arbeitsforschung (IfADo)\nUniversitätsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Müller\nMaja Rey\nUrsula Klingmüller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias König\nMartin Böhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schröder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schönfels\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\r\n\r\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \r\n\r\nKey questions:\r\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\r\n\r\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\r\n\r\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\r\n\r\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nGriffin Discoveries BV\nUniversitätsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmüller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Böhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nHumboldt-Universität zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias König\nJudith Wodke\nClemens Kühn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemünde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP – EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new ‘Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemünde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\r\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\n“Siberian” SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \r\n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigación Príncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversité catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kåre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksøyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Söderström\nLibe Aranguren Abadía\nSiri Øfsthus Goksøyr\nOdd André Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\r\n\r\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmon’s ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmon’s body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\r\n\r\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and “fed” in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Røsæg\nSahar Hassani\nMarius André Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "search = SEEK()\nsearch.search()",
      "start_time": "2019-03-17T19:22:51.226Z"
     },
     {
      "end_time": "2019-03-17T19:27:01.192Z",
      "execution_time": "44.4s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n\n4 ommited results\n\n558 relationships found\nLoading Completed\n\n2 ommited results\n\nSubstituting relationships into original search results: \nLoading 99.58%\n\n                                        --SEARCH RESULTS--\n\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUte Spiekerkötter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nErik Schröder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhöh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universität Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Böhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Böhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Böhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\nMESI-STRAT Coordinator\nLeader WP8 – Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut für Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hünfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nPAH metabolites in bile Kollevåg\nSediment chemistry\nLiver chemistry_Kollevåg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nLeibniz-Institut für Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nMatthias König(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\nRead more about the LiSyM junior group at: www.livermetabolism.com\n\nJunior Group Leader\nDr. Matthias König\n\nInstitute for Theoretical Biology\nHumboldt-University Berlin\nInvalidenstraße 43, 10117 Berlin, Germany\nphone +49 30 2093-8450\nkonigmatt@googlemail.com\n\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\n\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universität zu Berlin\nCharité University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nPablo José Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nOliver Ebenhöh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1α axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\n\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Börries\nBernhard Steiert\nChristian Tönsing\nUte Spiekerkötter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\n\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\n\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nWolfgang Müller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\n\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\n\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndré Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \n19th-20th of March, 2012, Vienna, Austria\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Müller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\n\nMore information available from\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias König\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut für Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nTBARS_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nMartin Böhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\nMy current research deals with the integration of different metabolic pathways. Research lines include:\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\n- The relation between metabolism and biologial ageing;\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUrsula Klingmüller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\n\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\n\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Müller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hönicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skłodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\n\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised ‘Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Müller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhöh\nDorota Kurek\nUte Spiekerkötter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevåg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevåg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\n\n<b>Topics include:</b>\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\n- Comprehensive and unambiguous data description and annotation\n- Standard formats and terminologies for data and metadata \n- Guidelines for reporting data (Minimum information standards) \n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\n- Using SEEK & FAIRDOMhub for integrated data management\n\n<b>Registration:</b> http://bit.ly/2ArOC7w\n(please state if you will attend on both days or only on one day)\n\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\n\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hünfeld will be refunded.\n\n<b>Organization:</b>\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\n\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \n\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevåg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nqPCR ovaries_Kollevåg\nqPCR liver_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nKollevåg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevåg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Müller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29 – 30 November 2012, Heidelberg\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Freiburg\nCharité University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nMatthias König\nJohannes Bausch\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\n\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias König\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias König\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nTBARS_Kollevåg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nqPCR liver_Kollevåg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nCharité University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias König\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmüller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dächert\nMartin Böhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\n\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \n\nKey questions:\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\n\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\n\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\n\n\n\n\nINSTITUTIONS\nVU University Amsterdam\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Chalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nGriffin Discoveries BV\nUniversitätsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmüller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Böhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Müller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universität Dresden\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nHumboldt University Berlin\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nLeibniz-Institut für Arbeitsforschung (IfADo)\nUniversitätsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Müller\nMaja Rey\nUrsula Klingmüller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias König\nMartin Böhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schröder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schönfels\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemünde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP – EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new ‘Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemünde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\r\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\r\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\n“Siberian” SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nHumboldt-Universität zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias König\nJudith Wodke\nClemens Kühn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \r\n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigación Príncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversité catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\r\n\r\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmon’s ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmon’s body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\r\n\r\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and “fed” in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Røsæg\nSahar Hassani\nMarius André Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kåre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksøyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Söderström\nLibe Aranguren Abadía\nSiri Øfsthus Goksøyr\nOdd André Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "search = SEEK(auth)\nsearch.search()",
      "start_time": "2019-03-17T19:26:16.780Z"
     },
     {
      "end_time": "2019-03-17T19:51:31.365Z",
      "execution_time": "50.1s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n\n4 ommited results\n\n558 relationships found\nLoading 38.89%\r"
       },
       {
        "ename": "KeyboardInterrupt",
        "evalue": "",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-4-ace431612635>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mauth\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    202\u001b[0m         \u001b[0;31m# Wait for the requests to finish in order to continue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    203\u001b[0m         \u001b[0;32mfor\u001b[0m \u001b[0mthread\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mPS\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mthreadList\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 204\u001b[0;31m             \u001b[0mthread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    205\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    206\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\\n\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mPS\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestFails\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\" ommited results\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/threading.py\u001b[0m in \u001b[0;36mjoin\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m   1054\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1055\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mtimeout\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1056\u001b[0;31m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_wait_for_tstate_lock\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1057\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1058\u001b[0m             \u001b[0;31m# the behavior of a negative timeout isn't documented, but\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/anaconda3/lib/python3.6/threading.py\u001b[0m in \u001b[0;36m_wait_for_tstate_lock\u001b[0;34m(self, block, timeout)\u001b[0m\n\u001b[1;32m   1070\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mlock\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m  \u001b[0;31m# already determined that the C code is done\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1071\u001b[0m             \u001b[0;32massert\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_is_stopped\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1072\u001b[0;31m         \u001b[0;32melif\u001b[0m \u001b[0mlock\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0macquire\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mblock\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1073\u001b[0m             \u001b[0mlock\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrelease\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1074\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_stop\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
        ]
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Loading Completed\n"
       }
      ],
      "source": "search = SEEK(auth)\nsearch.search()",
      "start_time": "2019-03-17T19:50:41.273Z"
     },
     {
      "end_time": "2019-03-17T20:41:15.586Z",
      "execution_time": "58.1s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nLoading Completed\n\n4 ommited results\n\n558 relationships found\nLoading Completed\n\n2 ommited results\n\nSubstituting relationships into original search results: \nLoading 99.58%\n\n                                        --SEARCH RESULTS--\n\n\nErik Schröder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nMatthias König(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\nRead more about the LiSyM junior group at: www.livermetabolism.com\n\nJunior Group Leader\nDr. Matthias König\n\nInstitute for Theoretical Biology\nHumboldt-University Berlin\nInvalidenstraße 43, 10117 Berlin, Germany\nphone +49 30 2093-8450\nkonigmatt@googlemail.com\n\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\n\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universität zu Berlin\nCharité University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\nMy current research deals with the integration of different metabolic pathways. Research lines include:\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\n- The relation between metabolism and biologial ageing;\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut für Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nTBARS_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhöh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nKollevåg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevåg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nLeibniz-Institut für Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\n\nMore information available from\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kåre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksøyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Söderström\nLibe Aranguren Abadía\nSiri Øfsthus Goksøyr\nOdd André Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\nMESI-STRAT Coordinator\nLeader WP8 – Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nUte Spiekerkötter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nqPCR ovaries_Kollevåg\nqPCR liver_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\n\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \n\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skłodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\n\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised ‘Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Müller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhöh\nDorota Kurek\nUte Spiekerkötter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nOliver Ebenhöh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29 – 30 November 2012, Heidelberg\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nUrsula Klingmüller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \n19th-20th of March, 2012, Vienna, Austria\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Müller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nWolfgang Müller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\n\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\n\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nPAH metabolites in bile Kollevåg\nSediment chemistry\nLiver chemistry_Kollevåg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndré Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\n\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \n\nKey questions:\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\n\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\n\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\n\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nGriffin Discoveries BV\nUniversitätsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmüller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Böhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\n\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\n\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\n\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hönicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nCharité University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias König\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nMartin Böhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevåg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1α axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nqPCR liver_Kollevåg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nPablo José Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Börries\nBernhard Steiert\nChristian Tönsing\nUte Spiekerkötter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\n\n<b>Topics include:</b>\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\n- Comprehensive and unambiguous data description and annotation\n- Standard formats and terminologies for data and metadata \n- Guidelines for reporting data (Minimum information standards) \n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\n- Using SEEK & FAIRDOMhub for integrated data management\n\n<b>Registration:</b> http://bit.ly/2ArOC7w\n(please state if you will attend on both days or only on one day)\n\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\n\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hünfeld will be refunded.\n\n<b>Organization:</b>\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemünde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP – EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new ‘Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemünde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\n“Siberian” SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\n"
       },
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Extremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Müller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universität Dresden\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nHumboldt University Berlin\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nLeibniz-Institut für Arbeitsforschung (IfADo)\nUniversitätsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Müller\nMaja Rey\nUrsula Klingmüller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias König\nMartin Böhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schröder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schönfels\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\r\n\r\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmon’s ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmon’s body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\r\n\r\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and “fed” in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Røsæg\nSahar Hassani\nMarius André Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \r\n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\r\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\r\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmüller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dächert\nMartin Böhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hünfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevåg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevåg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias König\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nHumboldt-Universität zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias König\nJudith Wodke\nClemens Kühn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nTBARS_Kollevåg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\r\n\r\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\r\n\r\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigación Príncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversité catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Müller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\r\n\r\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias König\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias König\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\r\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Böhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Böhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Böhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Freiburg\nCharité University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nMatthias König\nJohannes Bausch\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Müller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universität Dresden.\r\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\r\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\r\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut für Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "search = SEEK(auth)\nsearch.search()",
      "start_time": "2019-03-17T20:40:17.514Z"
     },
     {
      "end_time": "2019-03-17T20:41:47.517Z",
      "execution_time": "2.76s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\n"
       },
       {
        "ename": "TypeError",
        "evalue": "must be str, not int",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-3-ace431612635>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mauth\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    181\u001b[0m         \u001b[0;31m# 3rd param: the number of request per thread for the paralelization\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    182\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mauth\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 183\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchResultsPerThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    184\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    185\u001b[0m         \u001b[0;31m# Wait for the requests to finish in order to continue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36mparallelRequest\u001b[0;34m(self, requests, requestPerThread)\u001b[0m\n\u001b[1;32m    255\u001b[0m             \u001b[0mrequestPerThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m/\u001b[0m \u001b[0;36m25\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    256\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 257\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"r/total: \"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mrequestPerThread\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"/\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mend\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m''\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    258\u001b[0m         \u001b[0;31m# Compute the number of threads\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    259\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0mrequestPerThread\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: must be str, not int"
        ]
       }
      ],
      "source": "search = SEEK(auth)\nsearch.search()",
      "start_time": "2019-03-17T20:41:44.760Z"
     },
     {
      "end_time": "2019-03-17T20:43:20.029Z",
      "execution_time": "54.2s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allall\n\n135 results found\nr/total: 13/11\nLoading Completed\n\n4 ommited results\n\n558 relationships found\nr/total: 22/26\nLoading Completed\n\n2 ommited results\n\nSubstituting relationships into original search results: \nLoading 99.58%\n\n                                        --SEARCH RESULTS--\n\n\nAdelaide Raguin(id: 1249 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nErik Schröder(id: 820 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nSABIO-RK tasks for hands on training(id: 2144 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nMaja Rey\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nLiver transcriptome analysis of Atlantic cod (Gadus morhua) exposed to PCB 153 indicates effects on cell cycle regulation and lipid metabolism(id: 309 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon(id: 305 | type: presentations)\n\nDescription: Talk by Jon Olav Vik, Norwegian University of Life Sciences\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nJon Olav Vik\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)(id: 119 | type: events)\n\nDescription: The overarching integrating power of computational modelling, from systems biology to systems medicine\n\n\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nSara Tucci(id: 1246 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nBarbara Bakker(id: 335 | type: people)\n\nDescription: I am an Associate Professor in Systems Biology and the University Medical Centre Groningen. My research aims at understanding how energy metabolism is integrated and regulated. My work includes experimental, modelling as well as theoretical research. Recently I studied how metabolic regulation and gene-expression regulation work together towards in integrated response, e.g. when parasites are confronted with chemical inhibitors. Much of my work has been about the regulation of glycolysis and its various modes of action (steady-state, oscillations, turbo explosion).\nMy current research deals with the integration of different metabolic pathways. Research lines include:\n- Switching between lipids and carbohydrates and how this is affected in various metabolic diseases;\n- The relation between metabolism and biologial ageing;\n- The integration of redox and energy metabolism in African trypanosomes (SysMO project).\n\n\nINSTITUTIONS\nUniversity of Groningen\nMODELS\nModel of Trypanosoma brucei glycolysis as published in Achcar et al. (2012).\nPROJECTS\nSilicoTryp\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nThe silicon trypanosome\nMetabolomic systems biology of trypanosomes\nDynamic Modelling under Uncertainty: The Case of Trypanosoma brucei Energy Metabolism\nDissecting the Catalytic Mechanism of Trypanosoma brucei Trypanothione Synthetase by Kinetic Analysis and Computational Modelling\nThe silicon trypanosome: a test case of iterative model extension in systems biology\nHandling uncertainty in dynamic models: the pentose phosphate pathway in Trypanosoma brucei\nNaphthoquinone derivatives exert their antitrypanosomal activity via a multi-target mechanism\nProliferating bloodstream-form Trypanosoma brucei use a negligible part of consumed glucose for anabolic processes\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)(id: 45 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nLeibniz-Institut für Arbeitsforschung (IfADo)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nAhmed Ghallab\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nLiverpool John Moores University(id: 182 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAnton Wagenmakers\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nSABIO-RK results for hands on training(id: 2145 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMaja Rey\nUlrike Wittig\nEVENTS\nTools for systems biology modeling and data exchange 2018\nPEOPLE\nMaja Rey\nUlrike Wittig\nPROJECTS\nde.NBI-SysBio\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nStephan Goetz(id: 1247 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nIceni Diagnostics Ltd\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSharing data, models and procedures between Systems Biologists: technical and social engineering(id: 87 | type: presentations)\n\nDescription: Katy Wolstencroft's talk at the \"3rd Eagle Genomics Symposium:Will Big Data and Bigger Cuts Cripple Bioinformatics?\", presented on 21st March 2013.\n\nhttp://www.eaglegenomics.com/2013/01/eagles-third-symposium-how-the-efficient-bioinformatician-reduces-rd-costs/\n\n\nCREATORS\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153(id: 312 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nMatthias König(id: 678 | type: people)\n\nDescription: We are investigating liver metabolism and function with the help of computational models and methods.\nRead more about the LiSyM junior group at: www.livermetabolism.com\n\nJunior Group Leader\nDr. Matthias König\n\nInstitute for Theoretical Biology\nHumboldt-University Berlin\nInvalidenstraße 43, 10117 Berlin, Germany\nphone +49 30 2093-8450\nkonigmatt@googlemail.com\n\nThe liver is the central metabolic organ of our body playing a crucial role in the clearance of drugs, xenobiotics and numerous metabolites from the bloodstream. The quantification of these liver functions is essential to assess the status of the liver, obtain information about the degree of impairment, to evaluate treatment response or to select transplant recipients.\n\nWithin the Systems Medicine of the Liver (LiSyM) consortium we establish a standardized multiscale model framework of the human liver readily adaptable to various quantitative liver function tests currently used in the clinics. The resulting models will be personalized with individual imaging data from CT and PET in combination with anthropomorphic information like gender, age, body weight and height and used for the evaluation of quantitative liver function tests in a personalized context. \n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S1\ncy3sabiork-SupplementaryFile-S2\nINSTITUTIONS\nHumboldt-Universität zu Berlin\nCharité University Medicine Berlin\nHumboldt University Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPRESENTATIONS\nManagement of data, models, analyses and code for reproducible computational research: Present and Future\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nFAIRDOM user meeting\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHarmonizing semantic annotations for computational models in biology\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nTranscriptomics Template Guidelines(id: 75 | type: sops)\n\nDescription: An overview of creating MAGE-TAB compliant spreadsheets for transcriptomics data in SysMO SEEK\n\n\nASSAYS\nGeneral Transcriptomics Templates\nAffy Transcriptomics Templates\nRT-PCR Excel Template\nChip-chip Excel Template\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Transcriptomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nTBARS_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nRobert Field(id: 1245 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nJohn Innes Centre\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUniversity of Duesseldorf(id: 27 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nKarl-Erich Jaeger\nHorst Funken\nOliver Ebenhöh\nAdelaide Raguin\nPROJECTS\nPSYSMO\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nPatrick Schrauwen(id: 762 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity Maastricht\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nMartin Golebiewski(id: 623 | type: people)\n\nDescription: Data management and standardization expert for systems biology and systems medicine, responsible for the data management user requirements and user contacts within the German LiSyM network (Liver Systems Medicine: http://lisym.org/) and associated to the FAIRDOM team.\nInvolved in different standardization initiatives and committees, i.e. COMBINE (http://co.mbine.org), ISO/TC 276 Biotechnology (https://www.iso.org/committee/4514241.html), European COST action CHARME (http://www.cost-charme.eu) and others.\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n19th International Conference on Systems Biology (ICSB2018)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nMS_DILI\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nEnzymeML\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\n\n____________________________________________________________________________\n\nUniversity of Bergen(id: 23 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nNils-Kåre Birkeland\nAnita Elin-Fedoy\nInge Jonassen\nFatemeh Zamanzad Ghavidel\nFekadu Yadetie\nKarina Dale\nNello Blaser\nAnders Goksøyr\nXiaokang Zhang\nMorten Brun\nDorothy Dankel\nShirin Fallahi\nMathias Ziegler\nMarta Eide\nKidane Tekle\nSofie Söderström\nLibe Aranguren Abadía\nSiri Øfsthus Goksøyr\nOdd André Karlsen\nEileen Marie Hanna\nPROJECTS\nSulfoSys\nGenoSysFat\nDigiSal\nFAIRDOM user meeting\nSystems toxicology of Atlantic cod\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nSysMO-DB - Progress update and future activities(id: 38 | type: presentations)\n\nDescription: Carole Goble's presentation at the SysMO Evaluation conference in Tenerife - May 2012.\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nHuman Caffeine Metabolism(id: 255 | type: studies)\n\nDescription: Computational modeling of human caffeine clearance by the liver.\n\nMore information available from\nhttps://github.com/matthiaskoenig/caffeine\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nRecruitment and activation of a lipid kinase by hepatitis C virus NS5A is essential for integrity of the membranous replication compartment(id: 242 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nKollevåg biometrics(id: 2249 | type: data_files)\n\nDescription: Biometry data of fish from Kollevåg caging study\n\n\nASSAYS\nFish Biometrics\nEnzyme activity assays\nEnzyme immunoassay\nChemical analyses\nWestern blot - protein expression\nGene expression\nTBARS: Oxidative stress\nVitellogenin in blood plasma\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nKathrin Thedieck(id: 1039 | type: people)\n\nDescription: Prof. Dr. Kathrin Thedieck\nMESI-STRAT Coordinator\nLeader WP8 – Project Coordination\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nPUBLICATIONS\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\n\n____________________________________________________________________________\n\nMaaike H. Oosterveer(id: 1240 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation(id: 130 | type: projects)\n\nDescription: PoLiMeR is funded through the EU Marie Skłodowska-Curie Innovative Training Network (ITN), which drives scientific excellence and innovation. ITNs bring together universities, research institutes, industry and clinical partners from across the world to train researchers to doctorate level.\n\nMetabolic diseases are a burden on the European population and health care system. It is increasingly recognised that individual differences with respect to history, lifestyle, and genetic make-up affect disease progression and treatment response. A Systems Medicine approach, based on computational models fed with individual patient data, has the potential to provide the basis for a personalised diagnosis and treatment strategy. The PoLiMeR consortium (Polymers in the Liver: Metabolism and Regulation) has identified the inherited, liver-related diseases of glycogen and lipid metabolism as the ideal starting point for innovative research training in personalised ‘Systems Medicine'.\n\n\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Bergen\nUniversity of Duesseldorf\nUniversity of Groningen\nUniversity of Leiden\nChalmers University of Technology\nUniversity of Freiburg\nUniversity of Luxembourg\nIceni Diagnostics Ltd\nJohn Innes Centre\nMimetas\nORGANISMS\nHomo sapiens\nPEOPLE\nOlga Krebs\nWolfgang Müller\nBarbara Bakker\nAdil Mardinoglu\nInes Thiele\nKathrin Thedieck\nMathias Ziegler\nKaren Van Eunen\nMaaike H. Oosterveer\nOliver Ebenhöh\nDorota Kurek\nUte Spiekerkötter\nRonan Fleming\nRobert Field\nSara Tucci\nStephan Goetz\nThomas Hankemeier\nAdelaide Raguin\nTerry G.J. Derks\nPUBLICATIONS\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\n\n____________________________________________________________________________\n\nLeibniz-Institut für Arbeitsforschung (IfADo)(id: 198 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nAhmed Ghallab\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nqPCR ovaries_Kollevåg\nqPCR liver_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)(id: 11 | type: presentations)\n\nDescription: The presentation of new features in SEEK at last PALs meeing in Heidelberg on 27th of August by David, Lihua and Quyen\n\n\nCREATORS\nOlga Krebs\nEVENTS\nAll Hands SySMO PALs  meeting \nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nA Multiscale Computational Model of Human Galactose Metabolism(id: 170 | type: studies)\n\nDescription: Personalized liver function tests: A Multiscale Computational Model Predicts Individual Human Liver Function From Single-Cell Metabolism\n\nUnderstanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches. We report a multiscale mathematical model of the Human liver comprising the scales from single hepatocytes, over representation of ultra-structure and micro-circulation in the hepatic tissue, up to the entire organ integrated with perfusion. The model was validated against data on multiple spatial and temporal scales. Herein we describe the model construction and application to hepatic galactose metabolism demonstrating its utility via i) the personalization of liver function tests based on galactose elimination capacity (GEC), ii) the explanation of changes in liver function with aging, and iii) the prediction of population variability in liver function based on variability in liver volume and perfusion. We conclude that physiology- and morphology-based multiscale models can improve the evaluation of individual liver function. \n\nAvailability: Personalized prediction of liver function (GEC) was implemented in a proof-of-principle app available at https://www.livermetabolism.com/gec_app/. Source code and additional information at https://github.com/matthiaskoenig/multiscale-galactose\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nQuantitative analyses of the hepatic proteome of methylmercury-exposed Atlantic cod (Gadus morhua) suggest oxidative stress-mediated effects on cellular energy metabolism(id: 311 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nKaren Van Eunen(id: 1229 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Groningen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nIceni Diagnostics Ltd(id: 260 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nStephan Goetz\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUte Spiekerkötter(id: 1243 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nSEEK Users Meeting Report(id: 951 | type: data_files)\n\nDescription: This report gives an overview of the sessions and outcomes from the recent SEEK Users meeting in Berlin. \nPlease see related 'Presentation' entries for related slides and supplementary material.\n\n\nCREATORS\nKaty Wolstencroft\nEVENTS\nSySMO SEEK Users Meeting  2012\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nAn update on SEEK BioSamples\n(id: 51 | type: presentations)\n\nDescription: Olga's presentation at SySMO PALs meeting 29 – 30 November 2012, Heidelberg\n\n\n\nCREATORS\nOlga Krebs\nEVENTS\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\nPEOPLE\nOlga Krebs\nPROJECTS\nSysMO DB\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nJohn Innes Centre(id: 261 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRobert Field\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRonan Fleming(id: 1244 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUniversity of Leiden(id: 40 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nPaul Heusden\nKaty Wolstencroft\nWouter Hendriksen\nGilles P. Wezel Van\nClara Posthuma\nDino Van Dissel\nChao Du\nRonan Fleming\nThomas Hankemeier\nTerry G.J. Derks\nPROJECTS\nTRANSLUCENT\nSysMO DB\nFAIRDOM\nSysVirDrug\nSYSTERACT\nFAIRDOM user meeting\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUrsula Klingmüller(id: 474 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\n\n____________________________________________________________________________\n\nWolfgang Müller(id: 205 | type: people)\n\nDescription: I am group leader of the SDBV (Scientific Databases and Visualisation) group at the HITS gGmbH, the Heidelberg Institute for Theoretical Studies.\n\nI am interested in finding data. Starting with my master's thesis I have always worked on how to store data in a way that you can find it, and how to make sense out of data that has been stored.\n\nWithin FAIRDOM I find interesting to help people to store their data in a way that they make sense even after years.\n\n\n\nDATA_FILES\nList of Participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nINVESTIGATIONS\nData Management and Data Standardization\nPRESENTATIONS\nSEEK and Find:  Data Management for Systems Biology Projects\nSysMO-DB2 - Review and Update\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nReproducible and citable data and models: an introduction.\nMaking your data good enough for sharing.\nMaking the most of of the FAIRDOM: demo part\nIntroduction to FAIRDOMHub\nIntroduction into data management using FAIRDOMHub/SEEK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nMESI-STRAT\nINCOME\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nPUBLICATIONS\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nExcemplify: a flexible template based solution, parsing and managing data in spreadsheets for experimentalists\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nSABIO-RK: an updated resource for manually curated biochemical reaction kinetics.\n\n____________________________________________________________________________\n\nOliver Ebenhöh(id: 1241 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Duesseldorf\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nMimetas(id: 262 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDorota Kurek\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.(id: 359 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJon Olav Vik\nArne Gjuvsland\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nThomas Harvey\nTorgeir R. Hvidsten\nMagny Sidsel Thomassen\nYang Jin\nPROJECTS\nGenoSysFat\nDigiSal\n\n____________________________________________________________________________\n\nAnimating the Digital Salmon(id: 500 | type: presentations)\n\nDescription: How do you communicate a complex research endeavour in simple words and pictures using only two minutes? In my case that turned out to be scientifically impossible, but we managed to shave it down to three. This poster shows and tells how we went about it.\n\n\nCREATORS\nJon Olav Vik\nEVENTS\nDigital Life 2018 - DigiSal presentations\nPEOPLE\nJon Olav Vik\nPROJECTS\nDigiSal\nSUBMITTER\nJon Olav Vik\n\n____________________________________________________________________________\n\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria(id: 1320 | type: data_files)\n\nDescription: Report of the Data Management Foundry Workshop, \n19th-20th of March, 2012, Vienna, Austria\n\n\n\nCREATORS\nNatalie Stanford\nKaty Wolstencroft\nOlga Krebs\nStuart Owen\nCarole Goble\nWolfgang Müller\nPEOPLE\nKaty Wolstencroft\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nNatalie Stanford\nPROJECTS\nFAIRDOM\nSUBMITTER\nNatalie Stanford\n\n____________________________________________________________________________\n\nIMOMESIC(id: 24 | type: projects)\n\nDescription: IMOMESIC - Integrating Modelling of Metabolism and Signalling towards an Application in Liver Cancer\n\nOne of the most challenging questions in cancer research is currently the interconnection of metabolism and signalling. An understanding of mechanisms that facilitate the physiological shift towards a proliferative metabolism in cancer cells is considered a major upcoming topic in oncology and is a key activity for future drug development. Due to the complexity of interrelations, a systems biology approach would be most suited. However, rarely a systems biology approach is taken that: (i) works at standard conditions, (ii) combines modelling and quantitative data generation on metabolism and signalling, (iii) considers primary healthy and cancerous cells, and (iv) addresses patient heterogeneity. This project covers all these aspects. It is therefore timely, highly innovative and demonstrates the potential of systems biology towards an application in oncology. \n\nKey questions:\nWhich isoforms of glycolytic enzymes and key signalling components are relevant in hepatocytes and hepatoma cells?\n\nHow does growth factor and inflammatory signalling affect glycolysis in hepatocytes and hepatoma cells?\n\nCan we design strategies for combinatorial interventions of metabolism and signal transduction to inhibit proliferation of cancer cells but not affecting healthy cells?\n\n\n\n\nINSTITUTIONS\nVU University Amsterdam\nChalmers University of Technology\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nGriffin Discoveries BV\nUniversitätsklinikum Leipzig\nORGANISMS\nHomo sapiens\nPEOPLE\nBas Teusink\nJens Nielsen\nJurgen  Haanstra\nFrank Bruggerman\nUrsula Klingmüller\nDaniel Seehofer\nJens Timmer\nIwan De Esch\nMarkus Stepath\nAdil Mardinoglu\nAvlant Nilsson\nGeorg Damm\nJoep Vanlier\nMelanie Kiessig\nIrina Titkova\nMartin Böhm\nGerda Schicht\nPROGRAMMES\nERASysAPP\n\n____________________________________________________________________________\n\nJan Boren(id: 783 | type: people)\n\nDescription: POSITION:  Professor of Cardiovascular Research at University of Gothenburg, chair of Dept of Molecular and Clinical Medicine, Institute of Medicine, University of Gothenburg, and part-time physician at Clinical Chemistry, Sahlgrenska University Hospital, Gothenburg, Sweden\n\nRESEARCH:  My main long-term interest has been to understand the underlying mechanisms that lead to, and consequences of, lipid accumulation in the liver, arterial wall and heart, with the goal of translating this knowledge into effective treatment. My research program is translational and involves both in vitro and in vivo studies, including pathophysiological studies and mouse models, and kinetic studies in carefully phenotyped human volunteers.\n\nPUBLICATIONS:  Please see http://scholar.google.se/citations?user=OeiIGqcAAAAJ&hl=sv (Google Citations jan Boren)\n\n\nINSTITUTIONS\nUniversity of Gothenburg\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nJens Timmer(id: 476 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nMS_DILI\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\nCause and cure of sloppiness in ordinary differential equation models\nL1 regularization facilitates detection of cell type-specific parameters in dynamical systems\nDriving the Model to Its Limit: Profile Likelihood Based Model Reduction\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nPAH metabolites in bile Kollevåg\nSediment chemistry\nLiver chemistry_Kollevåg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nDorota Kurek(id: 1242 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nMimetas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nUniversity of Luxembourg(id: 89 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJorge Goncalves\nClemens Ostrowicz\nSigmar Lampe\nMalte Herold\nPaul Wilmes\nYuan Qixia\nInes Thiele\nPROJECTS\nSysMetEx\nCropClock\nICYSB 2015 - International Practical Course in Systems Biology\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.(id: 363 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nMathias Ziegler\nInes Heiland\nPROJECTS\nMESI-STRAT\n\n____________________________________________________________________________\n\nIf we build it will they come?(id: 42 | type: presentations)\n\nDescription: Keynote presentation by Professor Carole Goble at BOSC (Bioinformatics Open Source Conference) Long Beach, California, USA, July 14 2012. Co-located with ISMB, Intelligent Systems in Molecular Biology\n\nAlso available on slideshare at http://www.slideshare.net/mygrid/if-we-build-it-will-they-come-13652794\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nliterature values for allosteric regulation of pyruvate kinase(id: 915 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nStefan Henrich\nPEOPLE\nStefan Henrich\nPROJECTS\nSysMO-LAB\nSUBMITTER\nStefan Henrich\n\n____________________________________________________________________________\n\nChalmers University of Technology(id: 63 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Nielsen\nEduard Kerkhoven\nAndré Kelkkanen Innovationskontor\nAdil Mardinoglu\nAvlant Nilsson\nSimonas Marcisauskas\nBoyang Ji\nHao Wang\nPROJECTS\nSCaRAB\nSysMilk\nIMOMESIC\nSYSTERACT\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nJohannes Bausch(id: 759 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Freiburg\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\n\n____________________________________________________________________________\n\nInes Thiele(id: 753 | type: people)\n\nDescription: Associate Professor for Systems Biomedicine, Luxembourg Centre for Systems Biomedicine, University of Luxembourg\n\n\nINSTITUTIONS\nUniversity of Luxembourg\nPROJECTS\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nA mathematical modelling approach to assessing the reliability of biomarkers of glutathione metabolism(id: 194 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nJacky Snoep\nHans Westerhoff\nKaty Wolstencroft\nFranco Du Preez\nPROJECTS\nSysMO DB\n\n____________________________________________________________________________\n\nMax Luedemann(id: 903 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nTechnical University of Munich\nPROJECTS\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nMartin Böhm(id: 700 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nGerman Cancer Research Center (DKFZ)\nPRESENTATIONS\nPAL`s experience - IMOMESIC Project\nPROJECTS\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nPUBLICATIONS\nDivision of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range\n\n____________________________________________________________________________\n\nTechnical University of Munich(id: 72 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Hönicke\nMax Luedemann\nPROJECTS\nCOSMIC\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\n\n____________________________________________________________________________\n\nProteomics Template from PRIDE(id: 9 | type: data_files)\n\nDescription: An example of a completed data sheet for Mass spectrometry data. This is not SysMO specific, but it is an example of a MIAPE compliant data format taken from the PRIDE website (http://www.ebi.ac.uk/pride).\n\n\nASSAYS\nProteomics Templates (Mass spectrometry)\nCREATORS\nKaty Wolstencroft\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nPEOPLE\nKaty Wolstencroft\nPROJECTS\nSysMO DB\nSTUDIES\nCreating Templates for Proteomics\nSUBMITTER\nKaty Wolstencroft\n\n____________________________________________________________________________\n\nCharité University Medicine Berlin(id: 96 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nDaniel Seehofer\nGeorg Damm\nMelanie Kiessig\nMatthias König\nMarkus Morkel\nChristine Sers\nPamela Riemer\nCornelia Gieseler\nPROJECTS\nIMOMESIC\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\nCausalDB\nColosys\nMESI-STRAT\n\n____________________________________________________________________________\n\nLiver chemistry_Kollevåg(id: 2277 | type: data_files)\n\nDescription: Concentrations of various contaminants in liver from cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nChemical analyses\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nJanine Koehncke(id: 907 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nRWTH Aachen University Hospital - Department of Internal Medicine III\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nRWTH Aachen University Hospital - Department of Internal Medicine III(id: 203 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJanine Koehncke\nPROJECTS\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\n\n____________________________________________________________________________\n\nMathias Ziegler(id: 1045 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Bergen\nPROJECTS\nMESI-STRAT\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nNAD COMPARTMENTATION\nPUBLICATIONS\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\n\n____________________________________________________________________________\n\nDisentangling the Complexity of HGF Signaling by Combining Qualitative and Quantitative Modeling(id: 272 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nmRNA decay(id: 942 | type: data_files)\n\nDescription: missing\n\n\nASSAYS\nmRNA decay assay\nModelling the gene expression cascade with length-dependent processes\nCREATORS\nAbeer Fadda\nINVESTIGATIONS\nGene expression in Trypanosoma brucei\nPEOPLE\nAbeer Fadda\nPROJECTS\nSilicoTryp\nPUBLICATIONS\nTranscriptome-wide analysis of trypanosome mRNA decay reveals complex degradation kinetics and suggests a role for co-transcriptional degradation in determining mRNA levels\nSTUDIES\nDetermination and integration of abundance, processing efficiency, and degradation kinetics of the mRNAs encoding all T. brucei redox and pentose\nphosphate pathway enzymes\nSUBMITTER\nAbeer Fadda\n\n____________________________________________________________________________\n\nThomas Hankemeier(id: 1248 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nAdil Mardinoglu(id: 523 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nChalmers University of Technology\nPROJECTS\nIMOMESIC\nHUMET Startup\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nqPCR liver_Kollevåg(id: 2279 | type: data_files)\n\nDescription: Gene expression data of various genes from cod liver.\n\n\nASSAYS\nGene expression\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nRev1 contributes to proper mitochondrial function via the PARP-NAD+-SIRT1-PGC1α axis(id: 394 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nTim Rasmussen\nDaryl Shanley\nPROJECTS\nDynamic interplay between nuclei and mitochondria in ageing cells\n\n____________________________________________________________________________\n\nTerry G.J. Derks(id: 1250 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversity of Leiden\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nFirst LiSyM PostDoc and PhD student retreat(id: 64 | type: events)\n\nDescription: The meeting will provide a great opportunity for discussion between modellers, experimentalists and clinicians and to organize and harmonize our future work within the LiSyM network on the grassroot level. \n \nAll PhD students as well as all PostDocs of the LiSyM network are kindly requested to register by February 1st\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nPablo José Giraudi(id: 1064 | type: people)\n\nDescription: I am performing research in biomarkers discovery, particularly regarding liver fibrosis in non-alcoholic fatty liver disease. My research activities involved the use of bioinformatic platforms, the analysis of biological protein-protein interaction networks and the experimental validation of candidates by ELISA.\n\n\nObject has no relationships\n\n____________________________________________________________________________\n\nOlga Krebs(id: 136 | type: people)\n\nDescription: I am a researcher at the Scientific Databases and Visualization Group at Heidelberg Institute for Theoretical Studies (HITS)  , one of the  developers of SabioRK - System for the Analysis of Biochemical Pathways - Reaction Kinetics (http://sabiork.h-its.org/) . I am working on design  and maintenance of the information systems to store, query and analyse systems biology data; definition and implementation of methods for the integration of data from multiple sources.  In SySMO-DB project  (http://www.sysmo-db.org) I was responsible for metadata design (SEEK), conceptual and logical modelling of experimental data (JERM), now our team continue this work as  FAIRDOM project (http://fair-dom.org/).\n\n\nASSAYS\nStandard-based Excel template for metabolomics data\nMetabolomics Master Template\nHow to use templates to structure experimental data\nDATA_FILES\nExcel sheet template : concentrations of intracellular metabolites\nIntroduction to Lightning Talks and ISBE session at PALs meeting on 27th of August in Heidelberg\nMetabolomics Master Excel Template\nSysMO-SEEK Users Meeting\nThe map of Warnemuende\nAgenda: All Hands PALs meeting on 21-22 of May 2012 in Warnemünde/Rostock\nTravel Expense Report (HITS)\nMeeting location map\nTemplate for proteomics (2D gel)\nList of Participants\nList of participants\nReport of the Data Management Foundry Workshop,  19th-20th of March, 2012, Vienna, Austria\nAgenda for 3rd ERASysAPP – EXCHANGE Days\ntest file\nThe FAIR Guiding Principles for scientific data management and stewardship\nEVENTS\nAll Hands SySMO PALs  meeting \nSySMO SEEK Users Meeting  2012\n5th annual SySMO PALs All Hands meeting\nSEEK presentation at ICSB 2012 \nCOMBINE meeting\n6th All hands PALs meeting on 29 - 30 of November 2012 in Heidelberg\n7th All hands PALs + SEEK users meeting on 18 - 20 of September 2013 in Paris \nAdvanced Lecture Course on Systems Biology\nInternational SWAT4LS Workshop\n1st EraSysApp PALs meeting\n2nd FAIRDOM PALs meeting \nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nSummer school  SBB-2018\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nNorwegian University of Science and Technology\nFAIRDOM User meeting\nINVESTIGATIONS\nCreating data sheet template for 'omics data\nMetabolic analysis of effects of sucrose translocation on phenotypic traits of Arabidopsis thaliana\nUsing SEEK and I-S-A structure to integrate experimentation and mathematical modelling\nMODELS\nZucAt: multi-compartment metabolic model of growing Arabidopsis thaliana\nPRESENTATIONS\nNew SEEK features (presentation at last PALs meeing in Heidelberg on 27th of August)\nJERM and the new ‘Master Templates\n JERM templates presentation  at SEEK Users WS 2012 in Berlin\nJERM: new MASTER templates (at SySMO PALs meeting in Warnemünde/Rostock 21.05.2012)\nSEEK and Find:  Data Management for Systems Biology Projects\nSEEK and JERM - standard compliant integration of Systems Biology data\nAn update on SEEK BioSamples\n\nSBGN-ED - a tool for the Systems Biology Graphical Notation\nBiVeS & BudHat  - Version Control for Computational Models\nGraph based storage and retrieval of computational models\nPALs Meeting Report\nJERM templates: overview & recent development.\nRightField presentation for Virtual Liver Network\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform.\n FAIRDOM approach for semantic interoperability of systems biology data and models\nData and Model Management for ERASysAPP projects and Beyond\nFAIRDOMhub: Managing data from Instrument to publication.\nFAIRDOM: Improving data, model, and SOP management from creation to publication\nFAIRDOM Data and Model Management for ERASysAPP. Part I\n\nFAIRDOMHUB / SEEK user documentation\nMaking the most of of the FAIRDOM: demo part\nInstructions for hands on session - Using SEEK to manage your research assets\nEvaluation of SEEK and OpenBIS for data management on a centre-wide level\nPAL`s experience - IMOMESIC Project\nPublishing FAIR studies with citable research assets: a case study. Modelling glucose metabolism in malaria patients\n“Siberian” SEEK: Multiscale modeling of microbial communities\nData and model management needs for a knowledge base of salmon physiology. Use Case: The Digital Salmon\nFAIRDOMHub training\nIntroduction to FAIRDOMHub\nGuidelines on FAIR Data Management in Horizon 2020\nData and Model Management with FAIRDOM, SEEK, and other tools\nData and Model Management with FAIRDOM, SEEK, and other tools\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nZucAt\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nEraCoBiotech 2 nd call proposal preparation\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\nPUBLICATIONS\nStoring and annotating of kinetic data\nThe SEEK: a platform for sharing data and models in systems biology\nRightField: embedding ontology annotation in spreadsheets\nSABIO-RK--database for biochemical reaction kinetics\nStealthy annotation of experimental biology by spreadsheets\nSemantic Data and Models Sharing in Systems Biology: The Just Enough Results Model and the SEEK Platform\nMeeting report from the fourth meeting of the Computational Modeling in Biology Network (COMBINE)\nThe FAIR Guiding Principles for scientific data management and stewardship\nFAIRDOMHub: a repository and collaboration environment for sharing systems biology research\nMetabolic model of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nRightField: Scientific Knowledge Acquisition by Stealth through Ontology-Enabled Spreadsheets\nComparison of 432 Pseudomonas strains through integration of genomic, functional, metabolic and expression data.\nDigitoxin metabolism by rat liver microsomes.\nCopy number profiling by array comparative genomic hybridization identifies frequently occurring BRCA2-like male breast cancer.\nGene expression profiling of primary male breast cancers reveals two unique subgroups and identifies N-acetyltransferase-1 (NAT1) as a novel prognostic biomarker.\nModel of tryptophan metabolism, readily scalable using tissue-specific gene expression data.\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.\nHarmonizing semantic annotations for computational models in biology\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSOPS\nFluorescent probe labeling for microarrays\nGuidelines for Writing Standard Operating Procedures (SOP)\nExtraction and quantitative analysis of proteins from FFPE tissue\nExtraction and quantitative analysis of proteins from FFPE tissue\nSTUDIES\nCreating template for metabolomics data\nCreating Templates for Transcriptomics\nAnalysis of central carbon and energy metabolisms of growing Arabidopsis thaliana in relation to sucrose translocation\nExperimental study for hands on session\n\n____________________________________________________________________________\n\ncy3sabiork-SupplementaryFile-S2(id: 1428 | type: data_files)\n\nDescription: Supplementary File S2: Kinetic entries for human galactose metabolism.\nSBML for query: http://sabiork.h-its.org/sabioRestWebServices/searchKineticLaws/sbml?q=Pathway:%22galactose%20metabolism%22%20AND%20Organism:%22homo%20sapiens%22\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nIdentification of HNRNPK as regulator of hepatitis C virus particle production(id: 239 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nLars Kaderali\nMarco Binder\nRalf Bartenschlager\nPROJECTS\nSysVirDrug\n\n____________________________________________________________________________\n\nGenoSysFat(id: 34 | type: projects)\n\nDescription: Salmon farmed on modern feeds contains less of the healthy, long-chain fatty acids (EPA and DHA) than before. Up until the turn of the millennium, farmed salmon were fed fish oil as a replacement for their omega-3 rich natural prey. However, fish oil is now a scarce resource, and more than half of the fat in modern feeds comes from plant oils that are inexpensive, but devoid of long-chain omega-3 fatty acids. How can we increase the omega-3 content of salmon on sustainable feeds?\n\nOne option is to breed salmon that are well adapted to the feeds of the future. There is heritable variation in salmon’s ability to build EPA and DHA from shorter omega-3 fatty acids. The DNA sequence of salmon is now well known, allowing rapid characterization of heritable differences in nutrient utilization. A salmon family that appears promising on one feed, may not be the best on another. Therefore, we need to understand the salmon’s body as a system: a functional whole made up of parts that mutually affect, but also depend on, each other. A systems understanding of the interplay between feed and genetic factors will allow a tailoring of fish to feed and vice versa, which is robust to fluctuations in feedstuff availability and pricing.\n\nAs a first step towards such a systems understanding, the GenoSysFat project involves two biological experiments. 1) A traditional feeding experiment using high- vs low-omega-3 diets and salmon families that differ in feed utilization. 2) A novel study with pieces of liver kept alive and “fed” in laboratory dishes, studying for each fish how different feeds affect metabolism and gene activity. This allows faster and more detailed exploration of the interplay between genetics and feeds. Results will be interpreted with the help of mathematical models for the biochemical reaction networks, which are well established for other species and will be adapted to salmon based on the newly sequenced salmon genome.\n\n\nDATA_FILES\nSalmon genome\nCombined taxonomy table from freshwater and saltwater samples\nINSTITUTIONS\nNorwegian University of Science and Technology\nUniversity of Bergen\nUniversity of Rostock\nWageningen University & Research\nNorwegian University of Life Sciences\nSchool of Computer Science, University of Manchester\nCargill\nORGANISMS\nDanio rerio\nSalmo salar\nOncorhynchus mykiss\nPEOPLE\nVitor Martins Dos Santos\nMaksim Zakhartsev\nStuart Owen\nNatalie Stanford\nJon Olav Vik\nArne Gjuvsland\nDagmar Waltemath\nSandve Simen\nGareth Gillard\nJacob Seilø Torgersen\nHanne Hellerud Hansen\nKristil  Sundsaasen\nFabian Grammes\nThomas Harvey\nGraceline Tina Kirubakaran\nLars Snipen\nDominic Nanton\nJesse Van Dam\nTorgeir R. Hvidsten\nInga Leena Angell\nMagny Sidsel Thomassen\nTeshome Mulugeta\nTorfinn Nome\nFatemeh Zamanzad Ghavidel\nPer Bruheim\nMarit H Stafsnes\nJasper Koehorst\nZdenka Bartosova\nLiv Torunn Mydland\nFilip Rotnes\nYang Jin\nRenate Kvingedal\nLine Lieblein Røsæg\nSahar Hassani\nMarius André Strand\nPROGRAMMES\nThe Digital Salmon\nPUBLICATIONS\nReconstruction of Danio rerio metabolic model accounting for subcellular compartmentalisation\nQIIME allows analysis of high-throughput community sequencing data\nMinimum information about a microarray experiment (MIAME)-toward standards for microarray data\nSearch and clustering orders of magnitude faster than BLAST\nUPARSE: highly accurate OTU sequences from microbial amplicon reads\nStable Core Gut Microbiota across the Freshwater-to-Saltwater Transition for Farmed Atlantic Salmon\nLife-stage associated remodeling of lipid metabolism regulation in Atlantic salmon.\nAtlantic salmon raised with diets low in long-chain polyunsaturated n-3 fatty acids in freshwater have a Mycoplasma-dominated gut microbiota at sea\nSOPS\nISA structure for the Digital Salmon\n\n____________________________________________________________________________\n\nAnton Wagenmakers(id: 792 | type: people)\n\nDescription: Anton JM Wagenmakers is Professor of Exercise Metabolism and Academic Lead of the Exercise Metabolism and Adaptation Research Group (EMARG) at Liverpool John Moores University, UK. Anton is a leading researcher in skeletal muscle and integrative human physiology and metabolism with 185 peer reviewed journal publications and a Web of Science h-index of 50. His research is focussing on mechanisms by which impairments in fat metabolism lead to impaired glucose tolerance, insulin resistance, and muscle wasting (sarcopenia) in sedentary, obese, and elderly individuals, and on the design of optimal exercise/nutrition interventions that can be implemented in real life and reverse mechanisms that lead to development of the ageing phenotype and chronic disease. In these intervention studies the underlying molecular and physiological mechanisms are investigaetd as well. Anton is a leading expert in stable isotope methodology measuring glucose, fatty acid and protein kinetics in living human beings. Confocal immunohistochemical microscopy methods have been developed more recently to stain key enzymes playing a role in insulin resistance mechanisms in sections of skeletal muscle and its microvasculature to visualise the spatial distribution and co-localisation with membranes, mitochondria, lipid droplets and the endothelial layer of muscle arterioles and capillaries. This research has generated compelling evidence that insulin resistance occurs both in muscle fibers and their microvasculature. In obesity, metabolic syndrome and ageing insulin mediated dilation of the muscle microvasculature is impaired due to failure of insulin to activate eNOS (ser1177-P) and quenching of NO by superoxide anions (generated by NADPH oxidase). As a result of impaired arteriolar dilation the muscle fibres are starved of nutrients in well-fed bodies and meal derived glucose, lipids and amino acids are deposited in non-muscle tissues such as liver and adipose tissue. Anton was Editor for the Journal of Physiology (2007-14) and Chair of the Scientific Board of the European College of Sport Sciences (2007-15). Anton currently is Editor of the American Journal of Physiology - Endocrinology and Metabolism, and has a Web of Science h-index of 50. \n\n\n\nINSTITUTIONS\nLiverpool John Moores University\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nUniversity of Freiburg(id: 76 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nJens Timmer\nMatthias Meier\nJoep Vanlier\nJohannes Bausch\nGeoffroy Andrieux\nMelanie Börries\nBernhard Steiert\nChristian Tönsing\nUte Spiekerkötter\nSara Tucci\nStefan Lenz\nPROJECTS\nIMOMESIC\nRootBook\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\n\n____________________________________________________________________________\n\nLiSyM - Systems Medicine of the Liver(id: 9 | type: programmes)\n\nDescription: LiSyM (Liver Systems Medicine) represents a research network of German centers and institutions, brought together by a 20 Million Euro funding program of the German Government, in which mathematicians, modelers, pharmacologists, molecular biologists and clinical scientists work together to develop a Systems Medicine approach to study early and advanced liver disease. The aim of this unique research program is to acquire and use new experimental data and data from existing data bases to build computational models that facilitate decision making at the patient's bedsite and to predict the actions of new medicines in the treatment of metabolic liver disease.\n\n\nADMINISTRATORS\nMartin Golebiewski\nWolfgang Müller\nJohannes Bausch\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Rostock\nNorwegian University of Life Sciences\nTechnical University of Munich\nUniversity of Freiburg\nTechnische Universität Dresden\nGerman Cancer Research Center (DKFZ)\nCharité University Medicine Berlin\nHumboldt University Berlin\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nLeibniz-Institut für Arbeitsforschung (IfADo)\nUniversitätsklinikum Dresden - Medizinische Klinik I, Bereich Gastroenterologie & Hepatologie\nRWTH Aachen University Hospital - Department of Internal Medicine III\nUniversity Hospital Schleswig-Holstein, Kiel, General Surgery\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nMorpheus: open source modelling and simulation framework for multicellular systems\nMorpheus model of spatio-temporal Smo regulation in response to Hh signaling\nSpatio-temporal dynamics of Rab5 domains on the membrane of early endosomes\nPEOPLE\nMaksim Zakhartsev\nOlga Krebs\nWolfgang Müller\nMaja Rey\nUrsula Klingmüller\nJens Timmer\nUlrike Wittig\nMartin Golebiewski\nMatthias König\nMartin Böhm\nJohannes Bausch\nLutz Brusch\nMadlen Matz-Soja\nChristiane Rennert\nSteffen Brinkmann\nErik Schröder\nAhmed Ghallab\nVincent Moser\nMax Luedemann\nJanine Koehncke\nAida Ayoubzadeh\nWitigo Von Schönfels\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nNew LiSyM project\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nThe Human Physiome: how standards, software and innovative service infrastructures are providing the building blocks to make it achievable\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2016.\nSABIO-RK, von Daten in der Publikation zur Suchlösung für Spezialisten\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\npSSAlib: The partial-propensity stochastic chemical network simulator\nSpecifications of Standards in Systems and Synthetic Biology: Status and Developments in 2017\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)(id: 96 | type: events)\n\nDescription: <b>Target audience:</b> Wet-lab scientists and people dealing with experimental and/or clinical data for research\n\n<b>Topics include:</b>\n- Best practice data structuring (spreadsheets) for experimental and (clinical) samples data\n- Comprehensive and unambiguous data description and annotation\n- Standard formats and terminologies for data and metadata \n- Guidelines for reporting data (Minimum information standards) \n- Annotation tools (e.g. RightField: http://www.rightfield.org.uk)\n- Using SEEK & FAIRDOMhub for integrated data management\n\n<b>Registration:</b> http://bit.ly/2ArOC7w\n(please state if you will attend on both days or only on one day)\n\nTo demonstrate the practical relevance, attendees are encouraged to bring their own data and ideally share the data (or spreadsheet data templates) with the data management team beforehand.\n\nParticipation is free of charge. Participants from the LiSyM network will get free accommodation and full board and their travel expenses to get to Hünfeld will be refunded.\n\n<b>Organization:</b>\n<b>LiSyM (http://www.lisym.org) Data Management Team</b>, in conjunction with\n<b>FAIRDOM</b> (https://fair-dom.org) / <b>ERASysAPP</b> (European ERA-Net for Systems Biology Applications: https://www.erasysapp.eu) and\n<b>de.NBI</b> (German Network for Bioinformatics Infrastructure: https://www.denbi.de)\n\n\nDATA_FILES\nAgenda Data Structuring Course\nPRESENTATIONS\nIntroduction to FAIRDOM Data Management : SEEK and openBIS\nPROJECTS\nFAIRDOM\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.(id: 366 | type: publications)\n\nDescription: missing\n\n\nMODELS\nMorpheus: open source modelling and simulation framework for multicellular systems\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\n\n____________________________________________________________________________\n\nContext-specific flow through the MEK/ERK module produces cell- and ligand-specific patterns of ERK single and double phosphorylation(id: 274 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nWP1: Integration of Epo-induced Signal Transduction\nPEOPLE\nMaja Rey\nMarcel Schilling\nPROJECTS\nSBEpo - Systems Biology of Erythropoietin\n\n____________________________________________________________________________\n\nAgenda Data Structuring Course(id: 1989 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\nPEOPLE\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nUniversity of Groningen(id: 35 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nOscar Kuipers\nBert Poolman\nEriko Takano\nLubbert Dijkhuizen\nRainer Breitling\nArnold Driessen\nJacek Mika\nM. Tauqeer Alam\nAnthony Palathingal\nR Jansen\nMorris Swertz\nMaria Elena Merlo\nImke De Jong\nBogumila Marciniak\nBarbara Bakker\nRobin Sorg\nJan-Willem Veening\nRuben Mars\nPamela Gamba\nJurgen  Haanstra\nSulman Shafeeq\nFentaw Abegaz\nKathrin Thedieck\nErnst Wit\nJanina Leyk\nSebastian Groothuis\nPatricia Razquin Navas\nKaren Van Eunen\nMaaike H. Oosterveer\nAlexander Heberle\nPROJECTS\nBaCell-SysMO\nKOSMOBAC\nSTREAM\nSulfoSys\nSilicoTryp\nNoisy-Strep\nMESI-STRAT\nMultiscale modelling of state transitions in the host-microbiome-brain network\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nDigitoxin metabolism by rat liver microsomes.(id: 338 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nPROJECTS\nExample use cases\n\n____________________________________________________________________________\n\nJoint LiSyM & ERASysAPP Data Management PALs Meeting(id: 69 | type: events)\n\nDescription: This event is a jointly organized Data Management PALs Meeting that involves PALs from the German LiSyM (Liver Systems Medicine) network and from the ERASysAPP (ERA-Net for Systems Biology Applications) initiative. PALs (Project Area Liaisons) are the front line experts of the data management teams of these projects. They act as data management advocates and help gathering user requirements and testing potential data management solutions. An important part of the meeting will be to gather the user requirements from the LiSyM network and plan potential solutions. It is also the chance to get in direct contact with the data management experts and FAIRDOM/SEEK developers to discuss any data management issues.\nAfter the PALs meeting, there will be a tutorial data management session, including hands-on training for LiSyM SEEK and FAIRDOMhub, offered on the afternoon of May 3rd.\nParticipation is open to every LiSyM and EraSysApp project member who is interested to join and may want to act as PAL. Experienced ERASysAPP PALs will introduce LiSyM PALs to their tasks and responsibilities. Board and lodging for all attendees will be covered. If you are interested to attend please get in contact with Martin Golebiewski (martin.golebiewski@h-its.org) and Olga Krebs (olga.krebs@h-its.org) from HITS.\n\n\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nPROJECTS\nFAIRDOM\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nRightField presentation for Virtual Liver Network(id: 100 | type: presentations)\n\nDescription: Introduction to RightField and Demo were presented at the  Virtual Liver Network PALs meeting on 31st of October in Hünfeld, Germany\n\n\nCREATORS\nOlga Krebs\nStuart Owen\nKaty Wolstencroft\nPEOPLE\nKaty Wolstencroft\nStuart Owen\nOlga Krebs\nPROJECTS\nSysMO DB\nPUBLICATIONS\nRightField: embedding ontology annotation in spreadsheets\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nEnzyme activity Gst+Cat_Kollevåg(id: 2255 | type: data_files)\n\nDescription: Measurement of Gst and Cat enzyme activity in liver of fish caged in Kollevåg.\n\n\nASSAYS\nEnzyme activity assays\nCREATORS\nKarina Dale\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nToward community standards and software for whole-cell modeling(id: 267 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n\n____________________________________________________________________________\n\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests(id: 378 | type: presentations)\n\nDescription: Report about PAL experience and data management solutions in MM-PLF LiSyM.\n\n\nCREATORS\nMatthias König\nEVENTS\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\n1st LiSyM Jamboree & SAB Meeting(id: 73 | type: events)\n\nDescription: missing\n\n\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nHepatocytes contribute to residual glucose production in a mouse model for glycogen storage disease type Ia.(id: 389 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBert Groen\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nGerman Cancer Research Center (DKFZ)(id: 79 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nMarco Binder\nUrsula Klingmüller\nMarkus Stepath\nSajib Chakraborty\nIrina Titkova\nAhmed Sadik\nChristopher Dächert\nMartin Böhm\nLorenz Adlung\nHauke Busch\nMelania Barile\nClarissa Feuerstein\nMarcel Schilling\nPatricia Sauer\nMarie-Christine Wagner\nChristiane Opitz\nDyah Dewi\nMirja Tamara Prentzell\nArtyom Vlasov\nPROJECTS\nSysVirDrug\nIMOMESIC\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nFAIRDOM user meeting\nSBEpo - Systems Biology of Erythropoietin\nMS_DILI\nMESI-STRAT\n\n____________________________________________________________________________\n\nTBARS_Kollevåg(id: 2280 | type: data_files)\n\nDescription: Measurement of oxidative stress (TBARS) assay in liver of cod caged in Kollevåg and at reference location.\n\n\nASSAYS\nTBARS: Oxidative stress\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDIES\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction(id: 604 | type: presentations)\n\nDescription: missing\n\n\nCREATORS\nMartin Golebiewski\nEVENTS\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nPEOPLE\nMaja Rey\nMartin Golebiewski\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nMS_DILI\nSUBMITTER\nMaja Rey\n\n____________________________________________________________________________\n\nHUMET(id: 15 | type: programmes)\n\nDescription: Consortium of European research groups that aim to develop - in the forthcoming 10 years - a cluster of interlinked models that describe the whole-body human energy metabolism and its multi-dimensional regulation. Regulation includes interations with the brain, the immune system and the gut.\n\n\nADMINISTRATORS\nRoel Van Driel\nDATA_FILES\nlist of publications 2006 - 2016\nINSTITUTIONS\nUniversity of Amsterdam\nUniversity of Leiden\nUniversity of Rostock\nWageningen University & Research\nChalmers University of Technology\nLeiden University Medical Center\nUniversity of Luxembourg\nUniversity of Gothenburg\nHumboldt University Berlin\nCentro de Investigación Príncipe Felipe\nUniversity of Ulster\nEuropean Institute for Systems Biology and Medicine\nUniversitair ziekenhuis Antwerpen\nUniversity Maastricht\ngeneXplain GmbH\nUniversité catholique de Louvain\nUniversity Medical Center Hamburg - Eppendorf Hamburg\nLiverpool John Moores University\nInstitut Pasteur\nAstraZeneca\nUniversity of Lausanne\nUniversity of Zurich\nBilkent University\nPEOPLE\nOlaf Wolkenhauer\nWillem De Vos\nVitor Martins Dos Santos\nEdda Klipp\nKaty Wolstencroft\nAdil Mardinoglu\nNatal Van Riel\nRoel Van Driel\nBert Groen\nJoaquin Dopazo\nInes Thiele\nSteven Watterson\nAlexander Mazein\nSander Kersten\nSven Francque\nPatrick Schrauwen\nAlexander Kel\nJan Boren\nIlja Arts\nPatrice D. Cani\nJudith Wodke\nBart Staels\nJoerg Heeren\nJan Oscarsson\nAnton Wagenmakers\nAndries Kalsbeek\nErwin Zoetendal\nKo Willems Van Dijk\nChris Evelo\nBeatrice Desvergne\nArnold Von Eckardstein\nUgur Dogrusoz\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nSABIO-RK documentation user interface(id: 1391 | type: data_files)\n\nDescription: missing\n\n\nCREATORS\nUlrike Wittig\nPEOPLE\nUlrike Wittig\nPROJECTS\nKinetics on the move - Workshop 2016\nSUBMITTER\nUlrike Wittig\n\n____________________________________________________________________________\n\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice(id: 120 | type: events)\n\nDescription: Participants will learn how to structure and share complex data in a Findable, Accessible, Interoperable and Reusable (FAIR) manner. Hands-on sessions, lectures and software demonstrations will provide attendees with the necessary skills to apply components of the FAIRDOM infrastructure in their daily work for interrelating and exchanging data, models and corresponding information.\n\nFAIRDOM is a research infrastructure offering data management support. The SEEK software is designed as registry and storage place for data, models, biological samples, processes, publications and presentations, and at the same time as yellow pages for projects, people and events. SEEK is implemented as central data management platform in research networks such as LiSyM (https://seek.lisym.org), but also as cross-project platform FAIRDOMhub (https://www.fairdomhub.org) used e.g. by the European ERASysAPP initiative. The tutorial is jointly organized by LiSyM, ERASysAPP and the the German Network for Bioinformatics Infrastructure (de.NBI).\n\nParticipants need to bring their own laptop.\n\n\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction into SABIO-RK\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_Hands-on-tasks\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-slides_Hands-on\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK_api-slides\nFAIRDOM tutorial workshop SBMC2018 Bremen - SEEK-access_Jupyter_notebook\nPROJECTS\nFAIRDOM\nde.NBI-SysBio\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nSUBMITTER\nMartin Golebiewski\n\n____________________________________________________________________________\n\nA systems study reveals concurrent activation of AMPK and mTOR by amino acids.(id: 390 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nKathrin Thedieck\nDaryl Shanley\nPatricia Razquin Navas\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nManagement of data, models, analyses and code for reproducible computational research: Present and Future(id: 284 | type: presentations)\n\nDescription: Our group develops computational models of the liver within the German systems biology projects VLN (Virtual Liver Network) and LiSyM (Systems Medicine of the Liver). In this talk we present our experiences in data and model management in recent years and future directions towards reproducible computational research we plan to implement.\n\nWe give an overview over the role of Fairdom and SEEK within our data management solution and how this integrates with and complements other approaches such as code repositories, COMBINE archives, and Research Objects. We present challenges in data management within the typical workflows in model building, the integration of data with models and the preparation of results from computational analysis for publications.\n\n\nCREATORS\nMatthias König\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nMatthias König\nPROJECTS\nFAIRDOM user meeting\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nHumboldt-Universität zu Berlin(id: 13 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nEdda Klipp\nSimon Borger\nSusanne Gerber\nFalko Krause\nMamadou Bah\nSeverin Ehret\nMarcus Krantz\nMatthias König\nJudith Wodke\nClemens Kühn\nPROJECTS\nTRANSLUCENT\nICYSB 2015 - International Practical Course in Systems Biology\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nCOMBINE Multicellular Modelling\n\n____________________________________________________________________________\n\nTranslational Targeted Proteomics Profiling of Mitochondrial Energy Metabolic Pathways in Mouse and Human Samples.(id: 391 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nOlga Krebs\nBarbara Bakker\nPROJECTS\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\n\n____________________________________________________________________________\n\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)(id: 44 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nJohannes Bausch\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)(id: 7 | type: institutions)\n\nDescription: missing\n\n\nPEOPLE\nRebecca Wade\nRenate Kania\nIsabel Rojas\nOlga Krebs\nWolfgang Müller\nVlad Cojocaru\nStefan Henrich\nAnna Feldman-Salit\nMaja Rey\nQuyen Nguyen\nRon Henkel\nUlrike Wittig\nMartin Golebiewski\nAndreas Weidemann\nHadas Leonov\nSteffen Brinkmann\nDiego Sanchez\nXiaoming Hu\nPROJECTS\nSysMO-LAB\nSysMO DB\nFAIRDOM\nICYSB 2015 - International Practical Course in Systems Biology\nde.NBI-SysBio\nZucAt\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)\nLiSyM Core Infrastructure and Management (LiSyM-PD)\nLiver Function Diagnostics (LiSyM-LiFuDi - Pillar IV)\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nMolecular Steatosis - Imaging & Modeling (LiSyM-MSIM)\nKinetics on the move - Workshop 2016\nExample use cases\nFAIRDOM user meeting\nErasysApp Funders\nSBEpo - Systems Biology of Erythropoietin\nService to URV Tarragona, Spain with respect to their Safety Assessment of Endocrine Disrupting Chemicals model (Active NOW)\nMS_DILI\nEraCoBiotech 2 nd call proposal preparation\nCOMBINE Multicellular Modelling\nFAIRDOM & LiSyM & de.NBI Data Structuring Training\nSustainable co-production\nMESI-STRAT\nINCOME\nMultiscale modelling of state transitions in the host-microbiome-brain network\nBESTER\nTRALAMINOL\nExtremophiles metabolsim\nINDIE - Biotechnological production of sustainable indole\nNew LiSyM project\nEnzymeML\nPoLiMeR -   Polymers in the Liver: Metabolism and Regulation\nGMDS Project Group \"FAIRe Dateninfrastrukturen für die Biomedizinische Informatik\"\nGB-XMap: Assessing the risk of gut-brain cross-diseases Investigating the gut-brain-axis\nNAD COMPARTMENTATION\n\n____________________________________________________________________________\n\nPAL`s experience - IMOMESIC Project(id: 295 | type: presentations)\n\nDescription: Presented by Martin Böhm, German Cancer Research Center, Heidelberg, Germany\n\n\nCREATORS\nMarkus Stepath\nMartin Böhm\nEVENTS\nFirst FAIRDOM User Meeting held  as a satellite meeting of ICSB 2016 in Barcelona \nPEOPLE\nOlga Krebs\nMarkus Stepath\nMartin Böhm\nPROJECTS\nFAIRDOM\nFAIRDOM user meeting\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis(id: 288 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nMadlen Matz-Soja\nChristiane Rennert\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\n\n____________________________________________________________________________\n\nMM-PLF: Multiscale modeling for personalized liver function(id: 96 | type: investigations)\n\nDescription: Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\n\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways(id: 310 | type: publications)\n\nDescription: missing\n\n\nPEOPLE\nFekadu Yadetie\nPROJECTS\nSystems toxicology of Atlantic cod\n\n____________________________________________________________________________\n\nChronic Liver Disease Progression (LiSyM-DP - Pillar II)(id: 41 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nRWTH Aachen University Hospital - Department of Internal Medicine III\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nJanine Koehncke\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nTools and standards for computational modeling in biology(id: 94 | type: investigations)\n\nDescription: Tool and work flow development for computational biology.\n\n\nCREATORS\nMatthias König\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nIntroduction to FAIRDOMHub(id: 320 | type: presentations)\n\nDescription: Talk given by Olga Krebs at EmPowerPutida project meeting in Bruxeles  23rd November 2016\n\n\nCREATORS\nOlga Krebs\nCarole Goble\nRostyslav Kuzyakiv\nWolfgang Müller\nQuyen Nguyen\nStuart Owen\nBernd  Rinn\nJacky Snoep\nNatalie Stanford\nPEOPLE\nJacky Snoep\nCarole Goble\nStuart Owen\nOlga Krebs\nWolfgang Müller\nQuyen Nguyen\nNatalie Stanford\nRostyslav Kuzyakiv\nBernd  Rinn\nPROJECTS\nFAIRDOM\nSUBMITTER\nOlga Krebs\n\n____________________________________________________________________________\n\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK(id: 266 | type: publications)\n\nDescription: missing\n\n\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nKinetics on the move - Workshop 2016\n\n____________________________________________________________________________\n\nHepatic galactose model(id: 226 | type: models)\n\nDescription: missing\n\n\nASSAYS\nGalactose Modelling\nCREATORS\nMatthias König\nINVESTIGATIONS\nMM-PLF: Multiscale modeling for personalized liver function\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)(id: 46 | type: projects)\n\nDescription: We will contribute to the LiSyM Research Network an open source, freely available and reproducible multiscale model of the human liver from single cell metabolism to whole liver function. The model will be available in existing standards of systems biology, provide standardized interfaces for data integration and be fully annotated to available biological, medical and computational ontologies. All data, models and source code will be shared within the LiSyM Research Network and made available to the research community. We will fully support the de facto standard for computational models, the systems biology markup language (SBML).\n\n\nASSAYS\nGalactose Modelling\nDATA_FILES\ncy3sabiork-SupplementaryFile-S2\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nHumboldt-Universität zu Berlin\nUniversity of Freiburg\nCharité University Medicine Berlin\nINVESTIGATIONS\nTools and standards for computational modeling in biology\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nMartin Golebiewski\nMatthias König\nJohannes Bausch\nPRESENTATIONS\nPAL report – Data management LiSyM Multiscale Models for Liver Function Tests\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\nPUBLICATIONS\ncy3sabiork: A Cytoscape app for visualizing kinetic data from SABIO-RK\nToward community standards and software for whole-cell modeling\nSTUDIES\nA Multiscale Computational Model of Human Galactose Metabolism\nHuman Caffeine Metabolism\n\n____________________________________________________________________________\n\nThe SysMO SEEK - A Tale of Systems Biologists, Sharing and Stealth. Why every platform builder needs PALs(id: 105 | type: presentations)\n\nDescription: Carole Goble's KeyNote talk at the 9th Intl Symp on Integrative Bioinformatics 2013 from the 20th Martch 2013\n\n\n\nCREATORS\nCarole Goble\nPEOPLE\nCarole Goble\nStuart Owen\nPROJECTS\nSysMO DB\nSUBMITTER\nStuart Owen\n\n____________________________________________________________________________\n\nMorpheus: open source modelling and simulation framework for multicellular systems(id: 320 | type: models)\n\nDescription: Morpheus is the modelling and simulation framework for multicellular systems biology developed at Technische Universität Dresden.\nManual, examples and binaries for Windows, Linux, MacOS at: https://imc.zih.tu-dresden.de/wiki/morpheus\nOpen source code at: https://gitlab.com/morpheus.lab/morpheus\n\n\nCREATORS\nLutz Brusch\nPEOPLE\nLutz Brusch\nPROJECTS\nEarly Metabolic Injury (LiSyM-EMI - Pillar I)\nPUBLICATIONS\nMorpheus: a user-friendly modeling environment for multiscale and multicellular systems biology.\nA Predictive 3D Multi-Scale Model of Biliary Fluid Dynamics in the Liver Lobule.\nSUBMITTER\nLutz Brusch\n\n____________________________________________________________________________\n\nRegeneration and Repair in Acute-on-Chronic Liver Failure (LiSyM-ACLF - Pillar III)(id: 42 | type: projects)\n\nDescription: missing\n\n\nDATA_FILES\nAgenda Data Structuring Course\nEVENTS\nFirst LiSyM PostDoc and PhD student retreat\nJoint LiSyM & ERASysAPP Data Management PALs Meeting\n1st LiSyM Jamboree & SAB Meeting\nFAIRDOM-LiSyM-de.NBI Data Structuring Course - Nov 22-23 - Hünfeld (D)\n7th Conference on Systems Biology of Mammalian Cells (SBMC 2018)\nLiSyM/de.NBI/ERASysAPP Tutorial: How to Share FAIR — The FAIRDOM Data and Model Management Practice\nINSTITUTIONS\nHeidelberg Institute for Theoretical Studies (HITS gGmbH)\nTechnical University of Munich\nUniversity of Freiburg\nGerman Cancer Research Center (DKFZ)\nORGANISMS\nHomo sapiens\nMus musculus\nRattus norvegicus\nPEOPLE\nWolfgang Müller\nUrsula Klingmüller\nJens Timmer\nMartin Golebiewski\nMartin Böhm\nJohannes Bausch\nMax Luedemann\nPRESENTATIONS\nFAIRDOM tutorial workshop SBMC2018 Bremen - Introduction\nPROGRAMMES\nLiSyM - Systems Medicine of the Liver\n\n____________________________________________________________________________\n\nSven Francque(id: 760 | type: people)\n\nDescription: Sven Francque is currently Head of the Department of Gastroenterology and Hepatology of the Antwerp University Hospital and full professor of gastroenterology and hepatology at the University of Antwerpen, with a longtanding interest and expertise in the field of Non-Alcohloic Fatty Liver Disease. In collaboration with the Department of Endocrinology, Diabetes and Metabolic Diseases, a large cohort of well-phenotyped patients has been built up.\n\n\nINSTITUTIONS\nUniversitair ziekenhuis Antwerpen\nPROJECTS\nHUMET Startup\n\n____________________________________________________________________________\n\nAhmed Ghallab(id: 870 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nLeibniz-Institut für Arbeitsforschung (IfADo)\nPROJECTS\nModel Guided Pharmacotherapy In Chronic Liver Disease (LiSyM-MGP)\nMS_DILI\n\n____________________________________________________________________________\n\nMadlen Matz-Soja(id: 816 | type: people)\n\nDescription: missing\n\n\nINSTITUTIONS\nUniversität Leipzig - Medizinische Fakultät - Institut für Biochemie\nPROJECTS\nThe Hedgehog Signalling Pathway (LiSyM-JGMMS)\nPUBLICATIONS\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "search = SEEK(auth)\nsearch.search()",
      "start_time": "2019-03-17T20:42:25.812Z"
     },
     {
      "end_time": "2019-03-17T20:45:52.093Z",
      "execution_time": "4.92s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\n"
       },
       {
        "ename": "TypeError",
        "evalue": "object of type 'int' has no len()",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-4-ace431612635>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mauth\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    181\u001b[0m         \u001b[0;31m# 3rd param: the number of request per thread for the paralelization\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    182\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mauth\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 183\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchResultsPerThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    184\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    185\u001b[0m         \u001b[0;31m# Wait for the requests to finish in order to continue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36mparallelRequest\u001b[0;34m(self, requests, requestPerThread)\u001b[0m\n\u001b[1;32m    247\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    248\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m<\u001b[0m \u001b[0;36m20\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 249\u001b[0;31m             \u001b[0mrequestPerThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequestPerThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    250\u001b[0m         \u001b[0;32melif\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m<\u001b[0m \u001b[0;36m100\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    251\u001b[0m             \u001b[0mrequestPerThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m/\u001b[0m \u001b[0;36m2\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: object of type 'int' has no len()"
        ]
       }
      ],
      "source": "search = SEEK(auth)\nsearch.search()",
      "start_time": "2019-03-17T20:45:47.178Z"
     },
     {
      "end_time": "2019-03-17T20:46:38.487Z",
      "execution_time": "3.51s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\n"
       },
       {
        "ename": "TypeError",
        "evalue": "object of type 'int' has no len()",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-5-ace431612635>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mauth\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearch\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36msearch\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    181\u001b[0m         \u001b[0;31m# 3rd param: the number of request per thread for the paralelization\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    182\u001b[0m         \u001b[0mps\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mauth\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 183\u001b[0;31m         \u001b[0mps\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparallelRequest\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrequestPerThread\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchResultsPerThread\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    184\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    185\u001b[0m         \u001b[0;31m# Wait for the requests to finish in order to continue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;32m~/Work/ThirdYear/Seek/TheSeekNotebook/SEEK.py\u001b[0m in \u001b[0;36mparallelRequest\u001b[0;34m(self, requests, requestPerThread)\u001b[0m\n\u001b[1;32m    247\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    248\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m<\u001b[0m \u001b[0;36m20\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 249\u001b[0;31m             \u001b[0mrequestPerThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    250\u001b[0m         \u001b[0;32melif\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m<\u001b[0m \u001b[0;36m100\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    251\u001b[0m             \u001b[0mrequestPerThread\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mrequests\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m/\u001b[0m \u001b[0;36m2\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: object of type 'int' has no len()"
        ]
       }
      ],
      "source": "search = SEEK(auth)\nsearch.search()",
      "start_time": "2019-03-17T20:46:34.979Z"
     },
     {
      "end_time": "2019-03-17T20:47:11.659Z",
      "execution_time": "18.8s",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Enter your search: \nliver\nPlease enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n\n6 results found\nr/total: 6/1\nLoading Completed\n\n0 ommited results\n\n25 relationships found\nr/total: 12/3\nLoading Completed\n\n2 ommited results\n\nSubstituting relationships into original search results: \nLoading 88.24%\n\n                                        --SEARCH RESULTS--\n\n\nGalactose Modelling(id: 405 | type: assays)\n\nDescription: missing\n\n\nCREATORS\nMatthias König\nINVESTIGATION\nMM-PLF: Multiscale modeling for personalized liver function\nMODELS\nHepatic galactose model\nPEOPLE\nMatthias König\nPROJECTS\nMulti-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\nSTUDY\nA Multiscale Computational Model of Human Galactose Metabolism\nSUBMITTER\nMatthias König\n\n____________________________________________________________________________\n\nTBARS: Oxidative stress(id: 754 | type: assays)\n\nDescription: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\nPerformed in cod liver samples.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nTBARS_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nGene expression(id: 753 | type: assays)\n\nDescription: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nqPCR ovaries_Kollevåg\nqPCR liver_Kollevåg\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nFatty Acid Analysis(id: 776 | type: assays)\n\nDescription: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nInitial FAD KO_Liver phospholipid fatty acid profile\nInitial FAD KO_White muscle phospholipid fatty acid profile\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\nChemical analyses(id: 751 | type: assays)\n\nDescription: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n\n\nCREATORS\nKarina Dale\nDATA_FILES\nKollevåg biometrics\nPAH metabolites in bile Kollevåg\nSediment chemistry\nLiver chemistry_Kollevåg\nORGANISMS\nGadus morhua\nPEOPLE\nKarina Dale\nPROJECTS\nSystems toxicology of Atlantic cod\nSTUDY\nCaging study - Kollevåg\nSUBMITTER\nKarina Dale\n\n____________________________________________________________________________\n\nRNAseq(id: 771 | type: assays)\n\nDescription: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \nRNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n\n\nCREATORS\nSahar Hassani\nDATA_FILES\nCombined.counts\nCrispr_metadata\nPEOPLE\nSahar Hassani\nPROJECTS\nDigiSal\nSTUDY\nELOVL2 Knockout\nSUBMITTER\nSahar Hassani\n\n____________________________________________________________________________\n\n"
       }
      ],
      "source": "search = SEEK(auth)\nsearch.search()",
      "start_time": "2019-03-17T20:46:52.893Z"
     }
    ],
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Enter your search: \n",
      "liver\n",
      "Please enter one of: assays, data_files, events, institutions, investigations, models, organisms, people, presentations, programmes, projects, publications, sample_types, sops, studies, allassays\n",
      "\n",
      "6 results found\n",
      "r/total: 6/1\n",
      "Loading Completed\n",
      "\n",
      "0 ommited results\n",
      "\n",
      "25 relationships found\n",
      "r/total: 12/3\n",
      "Loading Completed\n",
      "\n",
      "2 ommited results\n",
      "\n",
      "Substituting relationships into original search results: \n",
      "Loading 88.24%\n",
      "\n",
      "                                        --SEARCH RESULTS--\n",
      "\n",
      "\n",
      "Galactose Modelling(id: 405 | type: assays)\n",
      "\n",
      "Description: missing\n",
      "\n",
      "\n",
      "CREATORS\n",
      "Matthias König\n",
      "INVESTIGATION\n",
      "MM-PLF: Multiscale modeling for personalized liver function\n",
      "MODELS\n",
      "Hepatic galactose model\n",
      "PEOPLE\n",
      "Matthias König\n",
      "PROJECTS\n",
      "Multi-Scale Models for Personalized Liver Function Tests (LiSyM-MM-PLF)\n",
      "STUDY\n",
      "A Multiscale Computational Model of Human Galactose Metabolism\n",
      "SUBMITTER\n",
      "Matthias König\n",
      "\n",
      "____________________________________________________________________________\n",
      "\n",
      "TBARS: Oxidative stress(id: 754 | type: assays)\n",
      "\n",
      "Description: TBARS assay measured oxidative stress as levels of malondialdehyde in samples.\n",
      "Performed in cod liver samples.\n",
      "\n",
      "\n",
      "CREATORS\n",
      "Karina Dale\n",
      "DATA_FILES\n",
      "Kollevåg biometrics\n",
      "TBARS_Kollevåg\n",
      "PEOPLE\n",
      "Karina Dale\n",
      "PROJECTS\n",
      "Systems toxicology of Atlantic cod\n",
      "STUDY\n",
      "Caging study - Kollevåg\n",
      "SUBMITTER\n",
      "Karina Dale\n",
      "\n",
      "____________________________________________________________________________\n",
      "\n",
      "Gene expression(id: 753 | type: assays)\n",
      "\n",
      "Description: Analyses of gene expression: qPCR analyses in liver (relative expression) and ovaries (absolute expression). \n",
      "\n",
      "\n",
      "CREATORS\n",
      "Karina Dale\n",
      "DATA_FILES\n",
      "Kollevåg biometrics\n",
      "qPCR ovaries_Kollevåg\n",
      "qPCR liver_Kollevåg\n",
      "PEOPLE\n",
      "Karina Dale\n",
      "PROJECTS\n",
      "Systems toxicology of Atlantic cod\n",
      "STUDY\n",
      "Caging study - Kollevåg\n",
      "SUBMITTER\n",
      "Karina Dale\n",
      "\n",
      "____________________________________________________________________________\n",
      "\n",
      "Fatty Acid Analysis(id: 776 | type: assays)\n",
      "\n",
      "Description: Total lipids are extracted from tissues of white muscle, liver and whole brain from three fish per dietary treatment. \n",
      "\n",
      "\n",
      "CREATORS\n",
      "Sahar Hassani\n",
      "DATA_FILES\n",
      "Initial FAD KO_Liver phospholipid fatty acid profile\n",
      "Initial FAD KO_White muscle phospholipid fatty acid profile\n",
      "PEOPLE\n",
      "Sahar Hassani\n",
      "PROJECTS\n",
      "DigiSal\n",
      "STUDY\n",
      "ELOVL2 Knockout\n",
      "SUBMITTER\n",
      "Sahar Hassani\n",
      "\n",
      "____________________________________________________________________________\n",
      "\n",
      "Chemical analyses(id: 751 | type: assays)\n",
      "\n",
      "Description: Chemical analyses in Kollevåg includes sediment contaminant concentrations, concentrations of various contaminants in cod liver and concentrations of PAH metabolites in cod bile.\n",
      "\n",
      "\n",
      "CREATORS\n",
      "Karina Dale\n",
      "DATA_FILES\n",
      "Kollevåg biometrics\n",
      "PAH metabolites in bile Kollevåg\n",
      "Sediment chemistry\n",
      "Liver chemistry_Kollevåg\n",
      "ORGANISMS\n",
      "Gadus morhua\n",
      "PEOPLE\n",
      "Karina Dale\n",
      "PROJECTS\n",
      "Systems toxicology of Atlantic cod\n",
      "STUDY\n",
      "Caging study - Kollevåg\n",
      "SUBMITTER\n",
      "Karina Dale\n",
      "\n",
      "____________________________________________________________________________\n",
      "\n",
      "RNAseq(id: 771 | type: assays)\n",
      "\n",
      "Description: Stranded RNAseq libraries were prepared from 1µg total RNA from liver tissue using TruSeq Stranded mRNA library preparation kit (Illumina, San Diego, USA) using double unique indices (#20022371), according to the manufacturer's instruction (Part 15031057 Rev.E). Libraries were sequenced at the Norwegian Sequencing Centre (NSC). All libraries were pooled, and the same pool was sequenced on 4 flow cell lanes on a HiSeq 3000 machine (Illumina), generating 100bp single-end reads. \n",
      "RNA sequencing files (.fastq) were processed in the following manner: (i) All reads from the same individual were merged into one fastq file. (ii) Reads were mapped to the Atlantic salmon genome (GenBank Accession number: GCA_0002333375.4) using STAR (v 2.6.0c) 50. (iii) Read alignments, recorded in BAM format were subsequently used to count uniquely mapped reads per gene using featurecounts (v1.4.4) 51, with the RefSeq gene_ids. Raw illumina reads as well as gene counts are publicly available through ArrayExpress 52 accession E-MTAB-7220.  \n",
      "\n",
      "\n",
      "CREATORS\n",
      "Sahar Hassani\n",
      "DATA_FILES\n",
      "Combined.counts\n",
      "Crispr_metadata\n",
      "PEOPLE\n",
      "Sahar Hassani\n",
      "PROJECTS\n",
      "DigiSal\n",
      "STUDY\n",
      "ELOVL2 Knockout\n",
      "SUBMITTER\n",
      "Sahar Hassani\n",
      "\n",
      "____________________________________________________________________________\n",
      "\n"
     ]
    }
   ],
   "source": [
    "search = SEEK(auth)\n",
    "search.search()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "ExecutionTime": {
     "end_time": "2019-03-17T18:03:58.185620Z",
     "start_time": "2019-03-17T18:03:58.171681Z"
    },
    "provenance": [
     {
      "end_time": "2019-03-17T16:48:21.811Z",
      "execution_time": "12ms",
      "outputs": [
       {
        "data": {
         "text/plain": "<object at 0x7f25e5ccdba0>"
        },
        "execution_count": 3,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.data",
      "start_time": "2019-03-17T16:48:21.799Z"
     },
     {
      "end_time": "2019-03-17T16:48:28.237Z",
      "execution_time": "112ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'object' object has no attribute 'id'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-4-23c723049cf6>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mid\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'object' object has no attribute 'id'"
        ]
       }
      ],
      "source": "search.data.id",
      "start_time": "2019-03-17T16:48:28.125Z"
     },
     {
      "end_time": "2019-03-17T16:48:32.779Z",
      "execution_time": "7ms",
      "outputs": [
       {
        "data": {
         "text/plain": "<object at 0x7f25e5ccdba0>"
        },
        "execution_count": 5,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.data",
      "start_time": "2019-03-17T16:48:32.772Z"
     },
     {
      "end_time": "2019-03-17T16:48:40.037Z",
      "execution_time": "6ms",
      "outputs": [
       {
        "data": {
         "text/plain": "['__class__',\n '__delattr__',\n '__dir__',\n '__doc__',\n '__eq__',\n '__format__',\n '__ge__',\n '__getattribute__',\n '__gt__',\n '__hash__',\n '__init__',\n '__init_subclass__',\n '__le__',\n '__lt__',\n '__ne__',\n '__new__',\n '__reduce__',\n '__reduce_ex__',\n '__repr__',\n '__setattr__',\n '__sizeof__',\n '__str__',\n '__subclasshook__']"
        },
        "execution_count": 6,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "dir(search.data)\n",
      "start_time": "2019-03-17T16:48:40.031Z"
     },
     {
      "end_time": "2019-03-17T16:49:29.840Z",
      "execution_time": "17ms",
      "outputs": [
       {
        "data": {
         "text/plain": "['__class__',\n '__delattr__',\n '__dict__',\n '__dir__',\n '__doc__',\n '__eq__',\n '__format__',\n '__ge__',\n '__getattribute__',\n '__gt__',\n '__hash__',\n '__init__',\n '__init_subclass__',\n '__le__',\n '__lt__',\n '__module__',\n '__ne__',\n '__new__',\n '__reduce__',\n '__reduce_ex__',\n '__repr__',\n '__setattr__',\n '__sizeof__',\n '__str__',\n '__subclasshook__',\n '__weakref__',\n 'base_url',\n 'createRelationshipList',\n 'createRequestList',\n 'data',\n 'headers',\n 'json',\n 'loadJSON',\n 'makeRequests',\n 'parallelRequest',\n 'percentageLoaded',\n 'print',\n 'printAttributes',\n 'printRelationships',\n 'relationshipList',\n 'relationshipsPerThread',\n 'removeDuplicateRelationships',\n 'request',\n 'requestFails',\n 'requestList',\n 'search',\n 'searchChoices',\n 'searchResultsPerThread',\n 'searchTerm',\n 'searchType',\n 'session',\n 'substituteRelationships',\n 'threadList']"
        },
        "execution_count": 7,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "dir(search)\n",
      "start_time": "2019-03-17T16:49:29.823Z"
     },
     {
      "end_time": "2019-03-17T16:51:01.517Z",
      "execution_time": "14ms",
      "outputs": [
       {
        "data": {
         "text/plain": "['__class__',\n '__delattr__',\n '__dir__',\n '__doc__',\n '__eq__',\n '__format__',\n '__ge__',\n '__getattribute__',\n '__gt__',\n '__hash__',\n '__init__',\n '__init_subclass__',\n '__le__',\n '__lt__',\n '__ne__',\n '__new__',\n '__reduce__',\n '__reduce_ex__',\n '__repr__',\n '__setattr__',\n '__sizeof__',\n '__str__',\n '__subclasshook__']"
        },
        "execution_count": 8,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "dir(search.data)\n",
      "start_time": "2019-03-17T16:51:01.503Z"
     },
     {
      "end_time": "2019-03-17T16:57:54.585Z",
      "execution_time": "6ms",
      "outputs": [],
      "source": "search.print()",
      "start_time": "2019-03-17T16:57:54.579Z"
     },
     {
      "end_time": "2019-03-17T16:58:28.918Z",
      "execution_time": "34ms",
      "outputs": [
       {
        "data": {
         "text/plain": "[{'id': '405', 'type': 'assays'},\n {'id': '754', 'type': 'assays'},\n {'id': '753', 'type': 'assays'},\n {'id': '776', 'type': 'assays'},\n {'id': '751', 'type': 'assays'},\n {'id': '771', 'type': 'assays'},\n {'id': '2277', 'type': 'data_files'},\n {'id': '2279', 'type': 'data_files'},\n {'id': '1428', 'type': 'data_files'},\n {'id': '1989', 'type': 'data_files'},\n {'id': '2255', 'type': 'data_files'},\n {'id': '2280', 'type': 'data_files'},\n {'id': '1391', 'type': 'data_files'},\n {'id': '2144', 'type': 'data_files'},\n {'id': '2145', 'type': 'data_files'},\n {'id': '2249', 'type': 'data_files'},\n {'id': '951', 'type': 'data_files'},\n {'id': '1320', 'type': 'data_files'},\n {'id': '915', 'type': 'data_files'},\n {'id': '9', 'type': 'data_files'},\n {'id': '942', 'type': 'data_files'},\n {'id': '64', 'type': 'events'},\n {'id': '96', 'type': 'events'},\n {'id': '69', 'type': 'events'},\n {'id': '73', 'type': 'events'},\n {'id': '120', 'type': 'events'},\n {'id': '119', 'type': 'events'},\n {'id': '182', 'type': 'institutions'},\n {'id': '27', 'type': 'institutions'},\n {'id': '198', 'type': 'institutions'},\n {'id': '260', 'type': 'institutions'},\n {'id': '261', 'type': 'institutions'},\n {'id': '262', 'type': 'institutions'},\n {'id': '63', 'type': 'institutions'},\n {'id': '72', 'type': 'institutions'},\n {'id': '203', 'type': 'institutions'},\n {'id': '76', 'type': 'institutions'},\n {'id': '79', 'type': 'institutions'},\n {'id': '13', 'type': 'institutions'},\n {'id': '23', 'type': 'institutions'},\n {'id': '40', 'type': 'institutions'},\n {'id': '89', 'type': 'institutions'},\n {'id': '96', 'type': 'institutions'},\n {'id': '35', 'type': 'institutions'},\n {'id': '7', 'type': 'institutions'},\n {'id': '96', 'type': 'investigations'},\n {'id': '94', 'type': 'investigations'},\n {'id': '226', 'type': 'models'},\n {'id': '320', 'type': 'models'},\n {'id': '760', 'type': 'people'},\n {'id': '870', 'type': 'people'},\n {'id': '816', 'type': 'people'},\n {'id': '820', 'type': 'people'},\n {'id': '678', 'type': 'people'},\n {'id': '762', 'type': 'people'},\n {'id': '1240', 'type': 'people'},\n {'id': '1243', 'type': 'people'},\n {'id': '1244', 'type': 'people'},\n {'id': '1241', 'type': 'people'},\n {'id': '1242', 'type': 'people'},\n {'id': '753', 'type': 'people'},\n {'id': '903', 'type': 'people'},\n {'id': '1045', 'type': 'people'},\n {'id': '1248', 'type': 'people'},\n {'id': '1250', 'type': 'people'},\n {'id': '1249', 'type': 'people'},\n {'id': '1246', 'type': 'people'},\n {'id': '1247', 'type': 'people'},\n {'id': '1245', 'type': 'people'},\n {'id': '1039', 'type': 'people'},\n {'id': '1229', 'type': 'people'},\n {'id': '474', 'type': 'people'},\n {'id': '476', 'type': 'people'},\n {'id': '759', 'type': 'people'},\n {'id': '700', 'type': 'people'},\n {'id': '907', 'type': 'people'},\n {'id': '523', 'type': 'people'},\n {'id': '1064', 'type': 'people'},\n {'id': '335', 'type': 'people'},\n {'id': '623', 'type': 'people'},\n {'id': '205', 'type': 'people'},\n {'id': '783', 'type': 'people'},\n {'id': '136', 'type': 'people'},\n {'id': '792', 'type': 'people'},\n {'id': '100', 'type': 'presentations'},\n {'id': '378', 'type': 'presentations'},\n {'id': '604', 'type': 'presentations'},\n {'id': '284', 'type': 'presentations'},\n {'id': '295', 'type': 'presentations'},\n {'id': '320', 'type': 'presentations'},\n {'id': '105', 'type': 'presentations'},\n {'id': '305', 'type': 'presentations'},\n {'id': '87', 'type': 'presentations'},\n {'id': '38', 'type': 'presentations'},\n {'id': '11', 'type': 'presentations'},\n {'id': '51', 'type': 'presentations'},\n {'id': '500', 'type': 'presentations'},\n {'id': '42', 'type': 'presentations'},\n {'id': '9', 'type': 'programmes'},\n {'id': '15', 'type': 'programmes'},\n {'id': '44', 'type': 'projects'},\n {'id': '41', 'type': 'projects'},\n {'id': '46', 'type': 'projects'},\n {'id': '42', 'type': 'projects'},\n {'id': '45', 'type': 'projects'},\n {'id': '130', 'type': 'projects'},\n {'id': '24', 'type': 'projects'},\n {'id': '34', 'type': 'projects'},\n {'id': '366', 'type': 'publications'},\n {'id': '338', 'type': 'publications'},\n {'id': '267', 'type': 'publications'},\n {'id': '389', 'type': 'publications'},\n {'id': '390', 'type': 'publications'},\n {'id': '391', 'type': 'publications'},\n {'id': '288', 'type': 'publications'},\n {'id': '310', 'type': 'publications'},\n {'id': '266', 'type': 'publications'},\n {'id': '309', 'type': 'publications'},\n {'id': '312', 'type': 'publications'},\n {'id': '242', 'type': 'publications'},\n {'id': '311', 'type': 'publications'},\n {'id': '359', 'type': 'publications'},\n {'id': '363', 'type': 'publications'},\n {'id': '194', 'type': 'publications'},\n {'id': '272', 'type': 'publications'},\n {'id': '394', 'type': 'publications'},\n {'id': '239', 'type': 'publications'},\n {'id': '274', 'type': 'publications'},\n {'id': '70', 'type': 'sample_types'},\n {'id': '68', 'type': 'sample_types'},\n {'id': '15', 'type': 'sample_types'},\n {'id': '58', 'type': 'sample_types'},\n {'id': '75', 'type': 'sops'},\n {'id': '255', 'type': 'studies'},\n {'id': '170', 'type': 'studies'}]"
        },
        "execution_count": 10,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.requestList",
      "start_time": "2019-03-17T16:58:28.884Z"
     },
     {
      "end_time": "2019-03-17T16:58:38.438Z",
      "execution_time": "12ms",
      "outputs": [
       {
        "data": {
         "text/plain": "[]"
        },
        "execution_count": 11,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.threadList",
      "start_time": "2019-03-17T16:58:38.426Z"
     },
     {
      "end_time": "2019-03-17T17:01:27.632Z",
      "execution_time": "3ms",
      "outputs": [
       {
        "data": {
         "text/plain": "{'id': '405', 'type': 'assays'}"
        },
        "execution_count": 12,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.requestList[0]",
      "start_time": "2019-03-17T17:01:27.629Z"
     },
     {
      "end_time": "2019-03-17T17:01:30.916Z",
      "execution_time": "22ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'dict' object has no attribute 'data'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-13-385aaf3a6ba4>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrequestList\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'dict' object has no attribute 'data'"
        ]
       }
      ],
      "source": "search.requestList[0].data",
      "start_time": "2019-03-17T17:01:30.894Z"
     },
     {
      "end_time": "2019-03-17T17:04:15.713Z",
      "execution_time": "4ms",
      "outputs": [
       {
        "data": {
         "text/plain": "<object at 0x7f25e5ccdba0>"
        },
        "execution_count": 16,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.data",
      "start_time": "2019-03-17T17:04:15.709Z"
     },
     {
      "end_time": "2019-03-17T17:05:34.321Z",
      "execution_time": "37ms",
      "outputs": [
       {
        "data": {
         "text/plain": "[{'id': '405', 'type': 'assays'},\n {'id': '754', 'type': 'assays'},\n {'id': '753', 'type': 'assays'},\n {'id': '776', 'type': 'assays'},\n {'id': '751', 'type': 'assays'},\n {'id': '771', 'type': 'assays'},\n {'id': '2277', 'type': 'data_files'},\n {'id': '2279', 'type': 'data_files'},\n {'id': '1428', 'type': 'data_files'},\n {'id': '1989', 'type': 'data_files'},\n {'id': '2255', 'type': 'data_files'},\n {'id': '2280', 'type': 'data_files'},\n {'id': '1391', 'type': 'data_files'},\n {'id': '2144', 'type': 'data_files'},\n {'id': '2145', 'type': 'data_files'},\n {'id': '2249', 'type': 'data_files'},\n {'id': '951', 'type': 'data_files'},\n {'id': '1320', 'type': 'data_files'},\n {'id': '915', 'type': 'data_files'},\n {'id': '9', 'type': 'data_files'},\n {'id': '942', 'type': 'data_files'},\n {'id': '64', 'type': 'events'},\n {'id': '96', 'type': 'events'},\n {'id': '69', 'type': 'events'},\n {'id': '73', 'type': 'events'},\n {'id': '120', 'type': 'events'},\n {'id': '119', 'type': 'events'},\n {'id': '182', 'type': 'institutions'},\n {'id': '27', 'type': 'institutions'},\n {'id': '198', 'type': 'institutions'},\n {'id': '260', 'type': 'institutions'},\n {'id': '261', 'type': 'institutions'},\n {'id': '262', 'type': 'institutions'},\n {'id': '63', 'type': 'institutions'},\n {'id': '72', 'type': 'institutions'},\n {'id': '203', 'type': 'institutions'},\n {'id': '76', 'type': 'institutions'},\n {'id': '79', 'type': 'institutions'},\n {'id': '13', 'type': 'institutions'},\n {'id': '23', 'type': 'institutions'},\n {'id': '40', 'type': 'institutions'},\n {'id': '89', 'type': 'institutions'},\n {'id': '96', 'type': 'institutions'},\n {'id': '35', 'type': 'institutions'},\n {'id': '7', 'type': 'institutions'},\n {'id': '96', 'type': 'investigations'},\n {'id': '94', 'type': 'investigations'},\n {'id': '226', 'type': 'models'},\n {'id': '320', 'type': 'models'},\n {'id': '760', 'type': 'people'},\n {'id': '870', 'type': 'people'},\n {'id': '816', 'type': 'people'},\n {'id': '820', 'type': 'people'},\n {'id': '678', 'type': 'people'},\n {'id': '762', 'type': 'people'},\n {'id': '1240', 'type': 'people'},\n {'id': '1243', 'type': 'people'},\n {'id': '1244', 'type': 'people'},\n {'id': '1241', 'type': 'people'},\n {'id': '1242', 'type': 'people'},\n {'id': '753', 'type': 'people'},\n {'id': '903', 'type': 'people'},\n {'id': '1045', 'type': 'people'},\n {'id': '1248', 'type': 'people'},\n {'id': '1250', 'type': 'people'},\n {'id': '1249', 'type': 'people'},\n {'id': '1246', 'type': 'people'},\n {'id': '1247', 'type': 'people'},\n {'id': '1245', 'type': 'people'},\n {'id': '1039', 'type': 'people'},\n {'id': '1229', 'type': 'people'},\n {'id': '474', 'type': 'people'},\n {'id': '476', 'type': 'people'},\n {'id': '759', 'type': 'people'},\n {'id': '700', 'type': 'people'},\n {'id': '907', 'type': 'people'},\n {'id': '523', 'type': 'people'},\n {'id': '1064', 'type': 'people'},\n {'id': '335', 'type': 'people'},\n {'id': '623', 'type': 'people'},\n {'id': '205', 'type': 'people'},\n {'id': '783', 'type': 'people'},\n {'id': '136', 'type': 'people'},\n {'id': '792', 'type': 'people'},\n {'id': '100', 'type': 'presentations'},\n {'id': '378', 'type': 'presentations'},\n {'id': '604', 'type': 'presentations'},\n {'id': '284', 'type': 'presentations'},\n {'id': '295', 'type': 'presentations'},\n {'id': '320', 'type': 'presentations'},\n {'id': '105', 'type': 'presentations'},\n {'id': '305', 'type': 'presentations'},\n {'id': '87', 'type': 'presentations'},\n {'id': '38', 'type': 'presentations'},\n {'id': '11', 'type': 'presentations'},\n {'id': '51', 'type': 'presentations'},\n {'id': '500', 'type': 'presentations'},\n {'id': '42', 'type': 'presentations'},\n {'id': '9', 'type': 'programmes'},\n {'id': '15', 'type': 'programmes'},\n {'id': '44', 'type': 'projects'},\n {'id': '41', 'type': 'projects'},\n {'id': '46', 'type': 'projects'},\n {'id': '42', 'type': 'projects'},\n {'id': '45', 'type': 'projects'},\n {'id': '130', 'type': 'projects'},\n {'id': '24', 'type': 'projects'},\n {'id': '34', 'type': 'projects'},\n {'id': '366', 'type': 'publications'},\n {'id': '338', 'type': 'publications'},\n {'id': '267', 'type': 'publications'},\n {'id': '389', 'type': 'publications'},\n {'id': '390', 'type': 'publications'},\n {'id': '391', 'type': 'publications'},\n {'id': '288', 'type': 'publications'},\n {'id': '310', 'type': 'publications'},\n {'id': '266', 'type': 'publications'},\n {'id': '309', 'type': 'publications'},\n {'id': '312', 'type': 'publications'},\n {'id': '242', 'type': 'publications'},\n {'id': '311', 'type': 'publications'},\n {'id': '359', 'type': 'publications'},\n {'id': '363', 'type': 'publications'},\n {'id': '194', 'type': 'publications'},\n {'id': '272', 'type': 'publications'},\n {'id': '394', 'type': 'publications'},\n {'id': '239', 'type': 'publications'},\n {'id': '274', 'type': 'publications'},\n {'id': '70', 'type': 'sample_types'},\n {'id': '68', 'type': 'sample_types'},\n {'id': '15', 'type': 'sample_types'},\n {'id': '58', 'type': 'sample_types'},\n {'id': '75', 'type': 'sops'},\n {'id': '255', 'type': 'studies'},\n {'id': '170', 'type': 'studies'}]"
        },
        "execution_count": 17,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.requestList",
      "start_time": "2019-03-17T17:05:34.284Z"
     },
     {
      "end_time": "2019-03-17T17:27:26.909Z",
      "execution_time": "29ms",
      "outputs": [
       {
        "data": {
         "text/plain": "[<SEEK.SEEK at 0x7f8fdf47add8>,\n <SEEK.SEEK at 0x7f8fdc424cc0>,\n <SEEK.SEEK at 0x7f8fdc3e47f0>,\n <SEEK.SEEK at 0x7f8fdc448dd8>,\n <SEEK.SEEK at 0x7f8fdc3f1198>,\n <SEEK.SEEK at 0x7f8fdf47a4a8>,\n <SEEK.SEEK at 0x7f8fdc4135f8>,\n <SEEK.SEEK at 0x7f8fdc3d44a8>,\n <SEEK.SEEK at 0x7f8fdc454cf8>,\n <SEEK.SEEK at 0x7f8fdf471208>,\n <SEEK.SEEK at 0x7f8fdc4483c8>,\n <SEEK.SEEK at 0x7f8fdf471ac8>,\n <SEEK.SEEK at 0x7f8fdf488748>,\n <SEEK.SEEK at 0x7f8fdc4548d0>,\n <SEEK.SEEK at 0x7f8fdc3bacc0>,\n <SEEK.SEEK at 0x7f8fdc424390>,\n <SEEK.SEEK at 0x7f8fdc3f1be0>,\n <SEEK.SEEK at 0x7f8fdf49b9e8>,\n <SEEK.SEEK at 0x7f8fdc25f4e0>,\n <SEEK.SEEK at 0x7f8fdc3fd6a0>,\n <SEEK.SEEK at 0x7f8fdc3d4ef0>,\n <SEEK.SEEK at 0x7f8fdc3ca080>,\n <SEEK.SEEK at 0x7f8fdc3a3978>,\n <SEEK.SEEK at 0x7f8fdf49b0f0>,\n <SEEK.SEEK at 0x7f8fdf488550>,\n <SEEK.SEEK at 0x7f8fdc3618d0>,\n <SEEK.SEEK at 0x7f8fdc413f98>,\n <SEEK.SEEK at 0x7f8fdc448080>,\n <SEEK.SEEK at 0x7f8fdc2ed128>,\n <SEEK.SEEK at 0x7f8fdc267908>,\n <SEEK.SEEK at 0x7f8fdc2fcc50>,\n <SEEK.SEEK at 0x7f8fdc3af7f0>,\n <SEEK.SEEK at 0x7f8fdc376b70>,\n <SEEK.SEEK at 0x7f8fdc1cb198>,\n <SEEK.SEEK at 0x7f8fdc2fc5f8>,\n <SEEK.SEEK at 0x7f8fdc2bacc0>,\n <SEEK.SEEK at 0x7f8fdc29bd30>,\n <SEEK.SEEK at 0x7f8fdc409b70>,\n <SEEK.SEEK at 0x7f8fdc256da0>,\n <SEEK.SEEK at 0x7f8fdc37f518>,\n <SEEK.SEEK at 0x7f8fdc3ba2b0>,\n <SEEK.SEEK at 0x7f8fdc2ceda0>,\n <SEEK.SEEK at 0x7f8fdc2fc748>,\n <SEEK.SEEK at 0x7f8fdc294780>,\n <SEEK.SEEK at 0x7f8fdc2b52b0>,\n <SEEK.SEEK at 0x7f8fdc3af4a8>,\n <SEEK.SEEK at 0x7f8fdc2f9da0>,\n <SEEK.SEEK at 0x7f8fdc430668>,\n <SEEK.SEEK at 0x7f8fdc2edc88>,\n <SEEK.SEEK at 0x7f8fdc43c048>,\n <SEEK.SEEK at 0x7f8fdc407fd0>,\n <SEEK.SEEK at 0x7f8fdc20ec88>,\n <SEEK.SEEK at 0x7f8fdc2ad550>,\n <SEEK.SEEK at 0x7f8fdc174898>,\n <SEEK.SEEK at 0x7f8fdc193860>,\n <SEEK.SEEK at 0x7f8fdc2b8550>,\n <SEEK.SEEK at 0x7f8fdc3214e0>,\n <SEEK.SEEK at 0x7f8fdc0b3f28>,\n <SEEK.SEEK at 0x7f8fdc2ed780>,\n <SEEK.SEEK at 0x7f8fdc3531d0>,\n <SEEK.SEEK at 0x7f8fdc2fce10>,\n <SEEK.SEEK at 0x7f8fdc2ed940>,\n <SEEK.SEEK at 0x7f8fdc07cc18>,\n <SEEK.SEEK at 0x7f8fdc2b59b0>,\n <SEEK.SEEK at 0x7f8fdc1b9748>,\n <SEEK.SEEK at 0x7f8fdc2ed9b0>,\n <SEEK.SEEK at 0x7f8fdc3fdfd0>,\n <SEEK.SEEK at 0x7f8fdc21f2e8>,\n <SEEK.SEEK at 0x7f8fdc314240>,\n <SEEK.SEEK at 0x7f8fc07c9ac8>,\n <SEEK.SEEK at 0x7f8fdc2f12b0>,\n <SEEK.SEEK at 0x7f8fdc1e2668>,\n <SEEK.SEEK at 0x7f8fdc15ed68>,\n <SEEK.SEEK at 0x7f8fdc153d30>,\n <SEEK.SEEK at 0x7f8fdc408400>,\n <SEEK.SEEK at 0x7f8fdc0c53c8>,\n <SEEK.SEEK at 0x7f8fc0793b00>,\n <SEEK.SEEK at 0x7f8fdc3c1978>,\n <SEEK.SEEK at 0x7f8fdc1b09e8>,\n <SEEK.SEEK at 0x7f8fdc334b38>,\n <SEEK.SEEK at 0x7f8fdc2a2b00>,\n <SEEK.SEEK at 0x7f8fdc3996a0>,\n <SEEK.SEEK at 0x7f8fdc0940f0>,\n <SEEK.SEEK at 0x7f8fc070d1d0>,\n <SEEK.SEEK at 0x7f8fdc2c4cf8>,\n <SEEK.SEEK at 0x7f8fdc1e98d0>,\n <SEEK.SEEK at 0x7f8fdc0bec50>,\n <SEEK.SEEK at 0x7f8fdc099588>,\n <SEEK.SEEK at 0x7f8fc07bcf60>,\n <SEEK.SEEK at 0x7f8fdc20ed30>,\n <SEEK.SEEK at 0x7f8fdc1e9978>,\n <SEEK.SEEK at 0x7f8fdc0b3fd0>,\n <SEEK.SEEK at 0x7f8fdc153be0>,\n <SEEK.SEEK at 0x7f8fdc14a828>,\n <SEEK.SEEK at 0x7f8fdc43c7f0>,\n <SEEK.SEEK at 0x7f8fdc0d2588>,\n <SEEK.SEEK at 0x7f8fdc3037f0>,\n <SEEK.SEEK at 0x7f8fdc275f98>,\n <SEEK.SEEK at 0x7f8fc070d710>,\n <SEEK.SEEK at 0x7f8fdc43c9e8>,\n <SEEK.SEEK at 0x7f8fdc1b05f8>,\n <SEEK.SEEK at 0x7f8fdc2a2ef0>,\n <SEEK.SEEK at 0x7f8fdc347e48>,\n <SEEK.SEEK at 0x7f8fdc04f1d0>,\n <SEEK.SEEK at 0x7f8fc06e4550>,\n <SEEK.SEEK at 0x7f8fc06b9dd8>,\n <SEEK.SEEK at 0x7f8fc06ce6a0>,\n <SEEK.SEEK at 0x7f8fc075ec18>,\n <SEEK.SEEK at 0x7f8fc07d06a0>,\n <SEEK.SEEK at 0x7f8fdc0739b0>,\n <SEEK.SEEK at 0x7f8fdc448d30>,\n <SEEK.SEEK at 0x7f8fdc17e908>,\n <SEEK.SEEK at 0x7f8fc0656470>,\n <SEEK.SEEK at 0x7f8fdc303c18>,\n <SEEK.SEEK at 0x7f8fdc2fc160>,\n <SEEK.SEEK at 0x7f8fdc106cf8>,\n <SEEK.SEEK at 0x7f8fdc407128>,\n <SEEK.SEEK at 0x7f8fdc106160>,\n <SEEK.SEEK at 0x7f8fdc021d68>,\n <SEEK.SEEK at 0x7f8fdc2940f0>,\n <SEEK.SEEK at 0x7f8fc06e4470>,\n <SEEK.SEEK at 0x7f8fdc04f898>,\n <SEEK.SEEK at 0x7f8fc37e1828>,\n <SEEK.SEEK at 0x7f8fc37f3978>,\n <SEEK.SEEK at 0x7f8fc07f44e0>,\n <SEEK.SEEK at 0x7f8fc06ebb00>,\n <SEEK.SEEK at 0x7f8fc0698d68>,\n <SEEK.SEEK at 0x7f8fc37fe0b8>,\n <SEEK.SEEK at 0x7f8fdc17e940>,\n <SEEK.SEEK at 0x7f8fdf49b0b8>,\n <SEEK.SEEK at 0x7f8fc0676080>]"
        },
        "execution_count": 3,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.requestList",
      "start_time": "2019-03-17T17:27:26.880Z"
     },
     {
      "end_time": "2019-03-17T17:27:35.401Z",
      "execution_time": "11ms",
      "outputs": [
       {
        "data": {
         "text/plain": "<function SEEK.SEEK.loadJSON.<locals>.<lambda>()>"
        },
        "execution_count": 4,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.requestList[0].data",
      "start_time": "2019-03-17T17:27:35.390Z"
     },
     {
      "end_time": "2019-03-17T17:27:38.736Z",
      "execution_time": "13ms",
      "outputs": [
       {
        "data": {
         "text/plain": "'1391'"
        },
        "execution_count": 5,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.requestList[0].data.id",
      "start_time": "2019-03-17T17:27:38.723Z"
     },
     {
      "end_time": "2019-03-17T17:27:41.595Z",
      "execution_time": "14ms",
      "outputs": [
       {
        "data": {
         "text/plain": "'data_files'"
        },
        "execution_count": 6,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.requestList[0].data.type",
      "start_time": "2019-03-17T17:27:41.581Z"
     },
     {
      "end_time": "2019-03-17T17:40:54.530Z",
      "execution_time": "16ms",
      "outputs": [
       {
        "data": {
         "text/plain": "<SEEK.SEEK at 0x7f8af0599438>"
        },
        "execution_count": 4,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.find(\"liver\")",
      "start_time": "2019-03-17T17:40:54.514Z"
     },
     {
      "end_time": "2019-03-17T17:41:04.491Z",
      "execution_time": "5ms",
      "outputs": [
       {
        "data": {
         "text/plain": "'1241'"
        },
        "execution_count": 5,
        "metadata": {},
        "output_type": "execute_result"
       }
      ],
      "source": "search.find(\"liver\").data.id\n",
      "start_time": "2019-03-17T17:41:04.486Z"
     },
     {
      "end_time": "2019-03-17T17:45:48.868Z",
      "execution_time": "85ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'list' object has no attribute 'data'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-3-85f02afb2d65>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mid\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'list' object has no attribute 'data'"
        ]
       }
      ],
      "source": "search.find(\"liver\").data.id\n",
      "start_time": "2019-03-17T17:45:48.783Z"
     },
     {
      "end_time": "2019-03-17T17:46:33.185Z",
      "execution_time": "7ms",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "338\n1241\n312\n288\n310\n96\n2279\n"
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.id)",
      "start_time": "2019-03-17T17:46:33.178Z"
     },
     {
      "end_time": "2019-03-17T17:46:37.966Z",
      "execution_time": "8ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'function' object has no attribute 'title'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-5-01f013872303>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtitle\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'function' object has no attribute 'title'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.title)",
      "start_time": "2019-03-17T17:46:37.958Z"
     },
     {
      "end_time": "2019-03-17T17:46:46.625Z",
      "execution_time": "28ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'function' object has no attribute 'attribute'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-6-b00b195dc770>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattribute\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtitle\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'function' object has no attribute 'attribute'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.attribute.title)",
      "start_time": "2019-03-17T17:46:46.597Z"
     },
     {
      "end_time": "2019-03-17T17:46:50.169Z",
      "execution_time": "12ms",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Digitoxin metabolism by rat liver microsomes.\nOliver Ebenhöh\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways\nMM-PLF: Multiscale modeling for personalized liver function\nqPCR liver_Kollevåg\n"
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.attributes.title)",
      "start_time": "2019-03-17T17:46:50.157Z"
     },
     {
      "end_time": "2019-03-17T17:55:23.735Z",
      "execution_time": "24ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'function' object has no attribute 'description'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-8-ec075ef164d8>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdescription\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'function' object has no attribute 'description'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T17:55:23.711Z"
     },
     {
      "end_time": "2019-03-17T17:55:33.093Z",
      "execution_time": "19ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'function' object has no attribute 'description'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-9-ec075ef164d8>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdescription\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'function' object has no attribute 'description'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T17:55:33.074Z"
     },
     {
      "end_time": "2019-03-17T17:55:35.879Z",
      "execution_time": "12ms",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Digitoxin metabolism by rat liver microsomes.\n\nOliver Ebenhöh\n\nQuantitative proteomics analysis reveals perturbation of lipid metabolic pathways in the liver of Atlantic cod (Gadus morhua) treated with PCB 153\n\nHedgehog signaling is a potent regulator of liver lipid metabolism and reveals a GLI-code associated with steatosis\n\nGlobal transcriptome analysis of Atlantic cod (Gadus morhua) liver after in vivo methylmercury exposure suggests effects on energy metabolism pathways\n\nMM-PLF: Multiscale modeling for personalized liver function\n\nqPCR liver_Kollevåg\n\n"
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.attributes.title + \"\\n\")",
      "start_time": "2019-03-17T17:55:35.867Z"
     },
     {
      "end_time": "2019-03-17T17:55:42.233Z",
      "execution_time": "26ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'function' object has no attribute 'description'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-11-ec075ef164d8>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdescription\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'function' object has no attribute 'description'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T17:55:42.207Z"
     },
     {
      "end_time": "2019-03-17T17:55:48.204Z",
      "execution_time": "9ms",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "publications\n\npeople\n\npublications\n\npublications\n\npublications\n\ninvestigations\n\ndata_files\n\n"
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.type + \"\\n\")",
      "start_time": "2019-03-17T17:55:48.195Z"
     },
     {
      "end_time": "2019-03-17T17:55:58.521Z",
      "execution_time": "25ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "unsupported operand type(s) for +: 'function' and 'str'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-13-02f13d7c023c>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m: unsupported operand type(s) for +: 'function' and 'str'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.attributes + \"\\n\")",
      "start_time": "2019-03-17T17:55:58.496Z"
     },
     {
      "end_time": "2019-03-17T17:56:09.750Z",
      "execution_time": "12ms",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "<function SEEK.loadJSON.<locals>.<lambda> at 0x7f2a6f531ae8>\n\n<function SEEK.loadJSON.<locals>.<lambda> at 0x7f2a6c191a60>\n\n<function SEEK.loadJSON.<locals>.<lambda> at 0x7f2a6c1598c8>\n\n<function SEEK.loadJSON.<locals>.<lambda> at 0x7f2a6c0eef28>\n\n<function SEEK.loadJSON.<locals>.<lambda> at 0x7f2a6c143d08>\n\n<function SEEK.loadJSON.<locals>.<lambda> at 0x7f2a507bc620>\n\n<function SEEK.loadJSON.<locals>.<lambda> at 0x7f2a53789840>\n\n"
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(str(item.data.attributes) + \"\\n\")",
      "start_time": "2019-03-17T17:56:09.738Z"
     },
     {
      "end_time": "2019-03-17T17:57:03.786Z",
      "execution_time": "23ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'SEEK' object has no attribute 'attributes'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-15-b714741efcaa>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m     \u001b[0;32mif\u001b[0m \u001b[0mdir\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;36m2\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0;34m\"__\"\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      3\u001b[0m         \u001b[0mdir\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0;31m#         print(dir() \"\\n\")\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mAttributeError\u001b[0m: 'SEEK' object has no attribute 'attributes'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    if dir(item.attributes)[:2] != \"__\":\n        dir(item.attributes)\n#         print(dir() \"\\n\")",
      "start_time": "2019-03-17T17:57:03.763Z"
     },
     {
      "end_time": "2019-03-17T17:57:12.546Z",
      "execution_time": "9ms",
      "outputs": [],
      "source": "for item in search.find(\"liver\"):\n    if dir(item.data.attributes)[:2] != \"__\":\n        dir(item.data.attributes)\n#         print(dir() \"\\n\")",
      "start_time": "2019-03-17T17:57:12.537Z"
     },
     {
      "end_time": "2019-03-17T17:57:33.892Z",
      "execution_time": "25ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "'bool' object is not iterable",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-17-c7c6c2c1134f>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m     \u001b[0;32mfor\u001b[0m \u001b[0mi\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mdir\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;36m2\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0;34m\"__\"\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      3\u001b[0m         \u001b[0mdir\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0;31m#         print(dir() \"\\n\")\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mTypeError\u001b[0m: 'bool' object is not iterable"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    for i in dir(item.data.attributes)[:2] != \"__\":\n        dir(item.data.attributes)\n#         print(dir() \"\\n\")",
      "start_time": "2019-03-17T17:57:33.867Z"
     },
     {
      "end_time": "2019-03-17T17:57:58.326Z",
      "execution_time": "14ms",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "abstract\nauthors\ncitation\ndoi\njournal\nlink_to_pub\npublished_date\npubmed_id\ntitle\navatar\ndescription\nexpertise\nfirst_name\nlast_name\nmbox_sha1sum\norcid\nproject_positions\ntitle\ntools\nabstract\nauthors\ncitation\ndoi\njournal\nlink_to_pub\npublished_date\npubmed_id\ntitle\nabstract\nauthors\ncitation\ndoi\njournal\nlink_to_pub\npublished_date\npubmed_id\ntitle\nabstract\nauthors\ncitation\ndoi\njournal\nlink_to_pub\npublished_date\npubmed_id\ntitle\ndescription\nother_creators\nsnapshots\ntitle\ncontent_blobs\ncreated_at\ndescription\nlatest_version\nlicense\nother_creators\nrevision_comments\ntags\ntitle\nupdated_at\nversion\nversions\n"
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    for i in dir(item.data.attributes):\n        if i[:2] != \"__\":\n            print(i)\n#         print(dir() \"\\n\")",
      "start_time": "2019-03-17T17:57:58.312Z"
     },
     {
      "end_time": "2019-03-17T17:58:22.405Z",
      "execution_time": "26ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'function' object has no attribute 'description'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-19-03f035bd4b4d>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      3\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mi\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;36m2\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0;34m\"__\"\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m             \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mi\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 5\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdescription\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'function' object has no attribute 'description'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    for i in dir(item.data.attributes):\n        if i[:2] != \"__\":\n            print(i)\n        print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T17:58:22.379Z"
     },
     {
      "end_time": "2019-03-17T17:58:27.158Z",
      "execution_time": "19ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'function' object has no attribute 'description'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-20-ec075ef164d8>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdescription\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'function' object has no attribute 'description'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T17:58:27.139Z"
     },
     {
      "end_time": "2019-03-17T17:59:06.403Z",
      "execution_time": "11ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "unsupported operand type(s) for +: 'NoneType' and 'str'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-21-e49bb8171d02>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mhasattr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'description'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdescription\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m: unsupported operand type(s) for +: 'NoneType' and 'str'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    if hasattr(item.data.attributes, 'description'):\n        print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T17:59:06.392Z"
     },
     {
      "end_time": "2019-03-17T18:02:51.008Z",
      "execution_time": "11ms",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Oliver Ebenhöh\n\nMM-PLF: Multiscale modeling for personalized liver function\n\nqPCR liver_Kollevåg\n\n"
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    if hasattr(item.data.attributes, 'description'):\n        print(item.data.attributes.title + \"\\n\")",
      "start_time": "2019-03-17T18:02:50.997Z"
     },
     {
      "end_time": "2019-03-17T18:03:01.237Z",
      "execution_time": "24ms",
      "outputs": [
       {
        "ename": "TypeError",
        "evalue": "unsupported operand type(s) for +: 'NoneType' and 'str'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-23-e49bb8171d02>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mhasattr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'description'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdescription\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mTypeError\u001b[0m: unsupported operand type(s) for +: 'NoneType' and 'str'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    if hasattr(item.data.attributes, 'description'):\n        print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T18:03:01.213Z"
     },
     {
      "end_time": "2019-03-17T18:03:27.507Z",
      "execution_time": "25ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'function' object has no attribute 'description'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-24-16e1918f6f5b>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mitem\u001b[0m \u001b[0;32min\u001b[0m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfind\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"liver\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m     \u001b[0;32mif\u001b[0m \u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdescription\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      3\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mattributes\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdescription\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
         "\u001b[0;31mAttributeError\u001b[0m: 'function' object has no attribute 'description'"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    if item.data.attributes.description != None:\n        print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T18:03:27.482Z"
     },
     {
      "end_time": "2019-03-17T18:03:51.628Z",
      "execution_time": "16ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (<ipython-input-25-15f92b1d9590>, line 2)",
        "output_type": "error",
        "traceback": [
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-25-15f92b1d9590>\"\u001b[0;36m, line \u001b[0;32m2\u001b[0m\n\u001b[0;31m    if hasattr(item.data.attributes, 'description') && item.data.attributes.description != None:\u001b[0m\n\u001b[0m                                                     ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    if hasattr(item.data.attributes, 'description') && item.data.attributes.description != None:\n        print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T18:03:51.612Z"
     },
     {
      "end_time": "2019-03-17T18:03:55.233Z",
      "execution_time": "13ms",
      "outputs": [
       {
        "ename": "SyntaxError",
        "evalue": "invalid syntax (<ipython-input-26-22dd371b9e1c>, line 2)",
        "output_type": "error",
        "traceback": [
         "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-26-22dd371b9e1c>\"\u001b[0;36m, line \u001b[0;32m2\u001b[0m\n\u001b[0;31m    if hasattr(item.data.attributes, 'description')  item.data.attributes.description != None:\u001b[0m\n\u001b[0m                                                        ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
        ]
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    if hasattr(item.data.attributes, 'description')  item.data.attributes.description != None:\n        print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T18:03:55.220Z"
     },
     {
      "end_time": "2019-03-17T18:03:58.185Z",
      "execution_time": "14ms",
      "outputs": [
       {
        "name": "stdout",
        "output_type": "stream",
        "text": "Understanding how liver function arises from the complex interaction of morphology, perfusion, and metabolism from single cells up to the entire organ requires systems-levels computational approaches.\r\n\n\nGene expression data of various genes from cod liver.\n\n"
       }
      ],
      "source": "for item in search.find(\"liver\"):\n    if hasattr(item.data.attributes, 'description') and item.data.attributes.description != None:\n        print(item.data.attributes.description + \"\\n\")",
      "start_time": "2019-03-17T18:03:58.171Z"
     }
    ]
   },
   "outputs": [],
   "source": [
    "for item in search.find(\"liver\"):\n",
    "    if hasattr(item.data.attributes, 'description') and item.data.attributes.description != None:\n",
    "        print(item.data.attributes.description + \"\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "ExecutionTime": {
     "end_time": "2019-03-17T19:36:36.971943Z",
     "start_time": "2019-03-17T19:36:36.933591Z"
    },
    "provenance": [
     {
      "end_time": "2019-03-17T19:35:28.141Z",
      "execution_time": "905ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'SEEK' object has no attribute 'advancedSearchSetup'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-7-dbfc43d5408c>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mauth\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0madvancedSearchSetup\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'SEEK' object has no attribute 'advancedSearchSetup'"
        ]
       }
      ],
      "source": "search = SEEK(auth)\nsearch.advancedSearchSetup()",
      "start_time": "2019-03-17T19:35:27.236Z"
     },
     {
      "end_time": "2019-03-17T19:36:36.971Z",
      "execution_time": "38ms",
      "outputs": [
       {
        "ename": "AttributeError",
        "evalue": "'SEEK' object has no attribute 'searchAdvancedSetup'",
        "output_type": "error",
        "traceback": [
         "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
         "\u001b[0;32m<ipython-input-8-de8efdc2b9ae>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0msearch\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSEEK\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mauth\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0msearch\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msearchAdvancedSetup\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
         "\u001b[0;31mAttributeError\u001b[0m: 'SEEK' object has no attribute 'searchAdvancedSetup'"
        ]
       }
      ],
      "source": "search = SEEK(auth)\nsearch.searchAdvancedSetup()",
      "start_time": "2019-03-17T19:36:36.933Z"
     }
    ]
   },
   "outputs": [],
   "source": [
    "search = SEEK(auth)\n",
    "search.searchAdvancedSetup()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
